--- reddit-safe-after-reseed@20260804T090120Z
+++ reddit-safe-after-reseed@20260804T144232Z
@@ -154,3 +154,67 @@
 A) if signed tx exported by coldcard is leaking something private 
 
 B) if the coldcard gets in the wrong hand and some way to exploit is found to access your seed + passphrase 
+
+comment: p1no81j
+parent: t1_p10q59k
+author: Solid_Wolverine1639
+created_utc: 1785853786
+edited: false
+body:
+Why are you suggesting coinkite can survive in any capacity?
+
+comment: p1nocbm
+parent: t1_p15ll2d
+author: Solid_Wolverine1639
+created_utc: 1785853818
+edited: false
+body:
+Why would you even suggest they could survive?  I wouldn't give them 1 oz of possibility
+
+comment: p1noj02
+parent: t1_p18o9ey
+author: Solid_Wolverine1639
+created_utc: 1785853868
+edited: false
+body:
+In other words trash it and don't trust it
+
+comment: p1nolde
+parent: t1_p15xqru
+author: Solid_Wolverine1639
+created_utc: 1785853885
+edited: false
+body:
+It's amazing enough of the people making comments need to hear this
+
+comment: p1np5j6
+parent: t1_p0zz9nt
+author: Solid_Wolverine1639
+created_utc: 1785854035
+edited: false
+body:
+And what other possible attack vectors are there beyond the the random generator not being so random?
+
+Whether you know or not asking that question should be more than enough to give you the answer based on what just happened days ago...
+
+The devices themselves can be compromised... Beyond the generator... In fact this hack might have started with backup plans for the stubborn people that want to keep their cold card Bitcoin... Even with changes, like new seed...
+
+Some back door?  
+
+comment: p1npdko
+parent: t1_p0zzqui
+author: Solid_Wolverine1639
+created_utc: 1785854094
+edited: false
+body:
+Amazing that you could have a negative count on your comment...  
+
+Cold card maxis have been revealed!
+
+comment: p1npl8o
+parent: t1_p14257u
+author: Solid_Wolverine1639
+created_utc: 1785854151
+edited: false
+body:
+Anything is better now!
