--- reddit-coinkite-email-data-retention@20260804T094309Z
+++ reddit-coinkite-email-data-retention@20260805T142623Z
@@ -81,14 +81,6 @@
 
 I agree that an end user shouldn't just rely on a sender's address for determining spam from non-spam because anyone's email account can still be compromised and used to send malicious emails. But positioning email like it is easy to spoof any email address these days and actually make it into a user's inbox ignores many of the protections that email has nowadays.
 
-comment: p1gf4fu
-parent: t1_p1g7n8m
-author: OldHamburger7923
-created_utc: 1785768056
-edited: false
-body:
-Correct. We just need phone lines to implement the same. Cause it is still the wild west there. 
-
 comment: p1gmg4g
 parent: t1_p1fushx
 author: Optionbulls
