--- reddit-coinkite-email-data-retention@20260805T142623Z
+++ reddit-coinkite-email-data-retention@20260806T163400Z
@@ -65,11 +65,11 @@
 
 comment: p1g7n8m
 parent: t1_p1fx195
-author: IllllIIlIllIllllIlll
+author: [deleted]
 created_utc: 1785766022
-edited: false
+edited: 1786027883
 body:
-Well, as a guy whose job involved crafting phishing campaigns, any decent mail provider will reject spoofed emails. With SPF, DKIM and DMARC, spoofed emails are trivial to spot. It's not 2010 anymore. If you try to spoof from a legit domain, if it even lands in the mailbox, it will be highlighted it in bright red and tell you this email is spoofed.
+[deleted]
 
 comment: p1gdo4w
 parent: t1_p1fx195
