--- bitcoindevs-explainer-thread@20260815T063146Z
+++ bitcoindevs-explainer-thread@20260815T130038Z
@@ -33,6 +33,15 @@
 @Bitcoin_Devs
  for next post on part 2 on Mk4/Mk5/Q
 
+post: 2044582580754792577
+role: reply
+author: LodgeIT
+name: LodgeiT
+created: 
+media: 0
+body:
+Watch tax tasks disappear. Submit to ATO faster with LodgeiT. Company tax lodgements from $60. Try for free.
+
 post: 2083949593260073000
 role: reply
 author: OikosBack
@@ -278,15 +287,6 @@
 body:
 I know the numbers are staggering but what’s stops someone from just guessing and checking bip39 words until they get a wallet? Sure you can’t select which wallet you find but you could guess and check and possible find someone’s wallet. Correct?
 
-post: 2084569938015224291
-role: reply
-author: musclebooster_
-name: musclebooster
-created: 
-media: 0
-body:
-Level up your training with bodyweight exercises at home. Receive a custom calisthenics plan + easy tracking. Build strength and stay on track. Discover your level and start today. Take the test!
-
 post: 2084849733705560486
 role: reply
 author: Bitcoin_Devs
