--- bitcoindevs-explainer-thread@20260813T143444Z
+++ bitcoindevs-explainer-thread@20260813T210356Z
@@ -149,6 +149,17 @@
 body:
 this is great!  How does anyone know what an individual chip UID is for the micropython PRNG?
 
+post: 2084036698325815570
+role: reply
+author: meirelindo
+name: @meirelindo
+created: 2026-08-02T22:00:43Z
+media: 0
+body:
+With the new software, is it safe to create a new address using dice? 
+
+Or did this affect other things in Coldcard Q?
+
 post: 2084041637852787155
 role: reply
 author: joinduelcom
@@ -267,6 +278,15 @@
 body:
 I know the numbers are staggering but what’s stops someone from just guessing and checking bip39 words until they get a wallet? Sure you can’t select which wallet you find but you could guess and check and possible find someone’s wallet. Correct?
 
+post: 2084849733705560486
+role: reply
+author: Bitcoin_Devs
+name: Bitcoin Dev Project
+created: 2026-08-05T03:51:25Z
+media: 1
+body:
+Yes, it is safe to create a new set of keys with the new firmware, but we are hearing reports of the firmware update bricking devices. Be careful!
+
 post: 2084850669001847071
 role: reply
 author: Bitcoin_Devs
