--- reddit-secure-btc-after-incident@20260805T210127Z
+++ reddit-secure-btc-after-incident@20260806T033027Z
@@ -441,6 +441,14 @@
 edited: false
 body:
 2 of 2 multisig with 2 different brands of hardware wallets. Also use strong bip39 passphrases on both wallets. 
+
+comment: p1lt2ay
+parent: t3_1veuo5u
+author: MulberryWorldly9904
+created_utc: 1785828182
+edited: false
+body:
+Sell
 
 comment: p1lto5h
 parent: t3_1veuo5u
@@ -761,3 +769,11 @@
 edited: false
 body:
 Thanks for the laugh this morning. 
+
+comment: p1zmpdc
+parent: t3_1veuo5u
+author: ORANGEisthenewGOLD
+created_utc: 1785986506
+edited: false
+body:
+Bitkey and chill
