--- reddit-secure-btc-after-incident@20260804T134505Z
+++ reddit-secure-btc-after-incident@20260804T164752Z
@@ -583,3 +583,11 @@
 edited: false
 body:
 The seed still has a lot more entropy than a passphrase unless you make that passphrase really huge. If you really don't want to rely at all on a Hw entropy would would use the dice. Though I'm confident that Trezor has a pretty reliable entropy already.
+
+comment: p1odvwy
+parent: t1_p1k6do4
+author: curiousengineer601
+created_utc: 1785860419
+edited: false
+body:
+Great insight here. A couple months from now I expect to read many stories about “moved my coins to x, now I can’t access them for reasons”
