--- reddit-secure-btc-after-incident@20260804T181540Z
+++ reddit-secure-btc-after-incident@20260804T184643Z
@@ -532,9 +532,9 @@
 parent: t1_p1m19j9
 author: wegwerfi08
 created_utc: 1785833139
-edited: false
-body:
-What you mean is FOSS (https://en.wikipedia.org/wiki/Free\_and\_open-source\_software). Coldcard is OSS (https://en.wikipedia.org/wiki/Open-source\_software) — without the F. 
+edited: 1785868106
+body:
+What you mean is FOSS ([https://en.wikipedia.org/wiki/Free\_and\_open-source\_software](https://en.wikipedia.org/wiki/Free_and_open-source_software)). Coldcard is OSS ([https://en.wikipedia.org/wiki/Open-source\_software](https://en.wikipedia.org/wiki/Open-source_software)) — without the F, “buddy”.
 
 comment: p1m534h
 parent: t1_p1k6do4
@@ -614,3 +614,73 @@
 body:
 ColdCard: 40 bit entropy is one grain of sand  
 Usual 128 bit entropy is 300k earths full of sand
+
+comment: p1p73yr
+parent: t1_p1oujx5
+author: H5Sooner
+created_utc: 1785867658
+edited: false
+body:
+Looks like most of them where using taproot. bc1qnk4zh9qcnap2mycp56qjrgza3cc8ylrh8fecp0
+
+comment: p1p8955
+parent: t1_p1p73yr
+author: DreamingTooLong
+created_utc: 1785867943
+edited: false
+body:
+Taproot addresses began with:
+
+bc1p
+
+Native Segwit addresses begin with:
+
+bc1q
+
+comment: p1p8rpm
+parent: t1_p1p8955
+author: H5Sooner
+created_utc: 1785868072
+edited: false
+body:
+Learned something new today. Thanks
+
+comment: p1p96ep
+parent: t1_p1p8rpm
+author: DreamingTooLong
+created_utc: 1785868173
+edited: 1785868653
+body:
+The address you provided only had 32 stolen BTC in it
+
+This address has 562 stolen BTC in it
+
+bc1qq85v2c926eg6pgxhwp6q7lf6cnsz80qs3fcu9r
+
+Taproot is cheaper when doing transactions that include multiple inputs. It’s also a little bit more private because it forces all transactions to look more alike. I use a platform that allows me to purchase BTC every 60 minutes and when the balance reaches a certain level, it automatically withdrawals to a taproot address on my nano x.
+
+comment: p1pbifi
+parent: t1_p1p96ep
+author: H5Sooner
+created_utc: 1785868756
+edited: false
+body:
+I was just checking sparrow and you can't do muli sig taproot unfortunately.
+
+comment: p1pbtll
+parent: t1_p1pbifi
+author: DreamingTooLong
+created_utc: 1785868834
+edited: false
+body:
+Oh, OK
+
+Then it’s no different than using electrum on tails OS unless you have your own node running
+
+comment: p1pclj4
+parent: t1_p1pbtll
+author: H5Sooner
+created_utc: 1785869031
+edited: false
+body:
+Looks like Nunchuk allows multi sig taproot but not familiar with them. 
