--- reddit-secure-btc-after-incident@20260805T011747Z
+++ reddit-secure-btc-after-incident@20260805T075759Z
@@ -218,14 +218,6 @@
 edited: false
 body:
 Because it's technical and if you have no idea what you are doing you will mess up and your seed will get stolen 
-
-comment: p1kddzk
-parent: t1_p1kbmdr
-author: idekwutp
-created_utc: 1785807226
-edited: false
-body:
-Yeah security is inversely related to access. If something is 100% secure it’s 100% inaccessible
 
 comment: p1kdpme
 parent: t3_1veuo5u
@@ -712,3 +704,11 @@
 edited: false
 body:
 Just keep in an exchange that offers insurance limits 🤷‍♂️. 
+
+comment: p1szcd1
+parent: t3_1veuo5u
+author: krispykye
+created_utc: 1785911211
+edited: false
+body:
+I thought the best way was too move it to a hard-drive and keep in unplugged from any computers?
