--- reddit-secure-btc-after-incident@20260804T221719Z
+++ reddit-secure-btc-after-incident@20260805T011747Z
@@ -486,13 +486,13 @@
 parent: t1_p1ke1ah
 author: bustaphlex
 created_utc: 1785831282
-edited: false
+edited: 1785891594
 body:
 No one is panicking about it with cash and banks because traditional commerce has had thousands of years to mature and become regulated.
 
 No one is panicking about it with cash and banks because the government and the banks ultimately own the fiat they’ve let you hold temporarily, and they have a vested interest in the stability of those systems because they rely on it to keep you and I motivated to work. 
 
-People absolutely did lose money to scams and security issues in the early days of currency. commerce, banking and fiat, but those holes have been plugged, the legislation written, the systems progressively more centralised, customers progressively more known… you see where this is going…
+People absolutely did lose money to scams and security issues in the early days of currency, commerce, banking and fiat, but those holes have been plugged, the legislation written, the systems progressively more centralised, customers progressively more known… you see where this is going…
 
 Part of the value of Bitcoin is its de-centralisation. With that comes some risk. If it’s outside your risk appetite, go buy indexes.
 
