--- reddit-secure-btc-after-incident@20260804T094317Z
+++ reddit-secure-btc-after-incident@20260804T134505Z
@@ -555,3 +555,31 @@
 edited: false
 body:
 safest way is to create your own  cryptographic hash function and then make your own cryptocurrency
+
+comment: p1mczba
+parent: t1_p1k36m2
+author: One-Adhesiveness-138
+created_utc: 1785838062
+edited: false
+body:
+A 2 min Google search will give you such answers.
+
+I'm afraid if you require such spoonfeeding then self custody may not be your best option.
+
+comment: p1mhv16
+parent: t3_1veuo5u
+author: CiaranCarroll
+created_utc: 1785840188
+edited: false
+body:
+I like Cold Card Q for this. Combined with Ian Coleman's website (and probably others) I can verify core functionality works, minimises trust in developers.
+
+Mine worked exactly as intended.
+
+comment: p1n7dmj
+parent: t1_p1kjiao
+author: skr_replicator
+created_utc: 1785849101
+edited: false
+body:
+The seed still has a lot more entropy than a passphrase unless you make that passphrase really huge. If you really don't want to rely at all on a Hw entropy would would use the dice. Though I'm confident that Trezor has a pretty reliable entropy already.
