--- reddit-secure-btc-after-incident@20260804T184643Z
+++ reddit-secure-btc-after-incident@20260804T191603Z
@@ -684,3 +684,23 @@
 edited: false
 body:
 Looks like Nunchuk allows multi sig taproot but not familiar with them. 
+
+comment: p1pelp4
+parent: t1_p1pclj4
+author: DreamingTooLong
+created_utc: 1785869533
+edited: false
+body:
+Nunchuck supports the liquid network
+
+It’s a second layer similar to lightning network
+
+Liquid is private and it also supports swapping back-and-forth between BTC and USDT
+
+How it’s private is it still reports that a transaction happened but it doesn’t say the coin type or the quantity.
+
+There’s a wallet called aqua wallet that supports spending liquid network bitcoin at lightning network bitcoin addresses. Doing a transaction like that is two different layers of privacy. When you send a lightning network payment, they can’t look on a block explorer and see how much you have or where it came from. Same with liquid.
+
+This site lets you swap between all the different bitcoin layers.
+
+[https://boltz.exchange](https://boltz.exchange)
