--- reddit-secure-btc-after-incident@20260804T164752Z
+++ reddit-secure-btc-after-incident@20260804T174610Z
@@ -591,3 +591,17 @@
 edited: false
 body:
 Great insight here. A couple months from now I expect to read many stories about “moved my coins to x, now I can’t access them for reasons”
+
+comment: p1oujx5
+parent: t1_p1k9px9
+author: DreamingTooLong
+created_utc: 1785864561
+edited: 1785864811
+body:
+You can do the same thing with electrum built-in on tails OS
+
+I like sparrow more though, because it allows me to easily connect to my own node and I can’t quite figure out how to generate a taproot address on tails OS electrum.
+
+Taproot is very easy on sparrow.
+
+I’m kind of curious how many people that lost all their BTC on coldcard were using taproot addresses. Probably very few. The bots that were sweeping people clean were probably looking at their native segwit addresses.
