--- reddit-secure-btc-after-incident@20260804T174610Z
+++ reddit-secure-btc-after-incident@20260804T181540Z
@@ -605,3 +605,12 @@
 Taproot is very easy on sparrow.
 
 I’m kind of curious how many people that lost all their BTC on coldcard were using taproot addresses. Probably very few. The bots that were sweeping people clean were probably looking at their native segwit addresses.
+
+comment: p1p44wx
+parent: t3_1veuo5u
+author: MrDryGuy_
+created_utc: 1785866923
+edited: false
+body:
+ColdCard: 40 bit entropy is one grain of sand  
+Usual 128 bit entropy is 300k earths full of sand
