--- reddit-coldcard-never-again@20260808T083358Z
+++ reddit-coldcard-never-again@20260809T165407Z
@@ -149,14 +149,6 @@
 body:
 why don’t you just use trezor instead? Ledger also owns your seed btw
 
-comment: p1bu274
-parent: t1_p1bopi0
-author: 0fWhomIAmChief
-created_utc: 1785705472
-edited: false
-body:
-This is what we have come to, OGs on Bitcointalk always swore by the 256 coin flips so i guess this is the 2026 version of that lol
-
 comment: p1buuph
 parent: t1_p1boq3w
 author: NorthSky6
