--- reddit-coldcard-never-again@20260806T100306Z
+++ reddit-coldcard-never-again@20260808T083358Z
@@ -417,3 +417,11 @@
 edited: false
 body:
 Just like when ledger introduced Ledger Recover.. Where are the Cold Card tough guys now?
+
+comment: p2et2wr
+parent: t3_1vdsqhy
+author: Legitimate_Form5449
+created_utc: 1786168023
+edited: false
+body:
+Y are u doing this? Y are u destroying the evidence? take them to small claims court
