--- reddit-coldcard-never-again@20260804T134604Z
+++ reddit-coldcard-never-again@20260806T100306Z
@@ -362,14 +362,6 @@
 body:
 So those with 24 word seed on coldcard are safe? It was those with 12 words generated on coldcard that were hacked?
 
-comment: p1e80go
-parent: t3_1vdsqhy
-author: DonTheHolder
-created_utc: 1785736346
-edited: false
-body:
-Office Space 🏬 🏢 
-
 comment: p1e9s7k
 parent: t3_1vdsqhy
 author: setec404
