--- reddit-self-custody-dagger@20260804T204835Z
+++ reddit-self-custody-dagger@20260805T210356Z
@@ -181,12 +181,11 @@
 
 comment: p1nfwr8
 parent: t1_p1mgpuc
-author: mrAech219
+author: [deleted]
 created_utc: 1785851533
-edited: false
-body:
-Even more unpopular/unemotional take: most of this issue is user error.   
-People not fully understanding how seed phrases work, how cold storage works and how important security due diligence is when practicing self custody. 
+edited: 1785962610
+body:
+[deleted]
 
 comment: p1nk5az
 parent: t3_1vf5gj4
@@ -251,12 +250,11 @@
 
 comment: p1nvd8h
 parent: t1_p1npetz
-author: mrAech219
+author: [deleted]
 created_utc: 1785855667
-edited: false
-body:
-The user trust IS the error.   
-I’m not saying that it’s not Coinkite’s fault. Two things can be true at the same time. 
+edited: 1785962578
+body:
+[deleted]
 
 comment: p1nvtgi
 parent: t1_p1nuj9b
@@ -301,16 +299,11 @@
 
 comment: p1o8jf2
 parent: t1_p1o3rid
-author: mrAech219
+author: [deleted]
 created_utc: 1785859067
-edited: false
-body:
-I do get your point. I’m not disagreeing with you at all. Imo Coinkite should not continue to exist after this. 
-
-However, it should be well known and understood in the bitcoin space that putting trust in any company for any reason whatsoever introduces third party risk and that trust should be minimized at all cost. Especially when there is already a solution to that specific risk.  
-   
-I have nothing but empathy for anyone affected by this but it really should be a wakeup call to all of us.   
-RNGs shouldn’t be trusted. I don’t care which wallet you’re using. 
+edited: 1785962555
+body:
+[deleted]
 
 comment: p1ofxzc
 parent: t1_p1o8jf2
