--- reddit-coldcard-letter-db-leak@20260803T110745Z
+++ reddit-coldcard-letter-db-leak@20260803T114241Z
@@ -11,6 +11,13 @@
 Stay safe out there and don’t ship Bitcoin hardware to your door, regardless on how safe the company might seem.
 
 Share
+huntresslabs
+•
+Promoted
+To tell this story, we needed to bring in one of the highest ranking cybercrime law enforcement officials in the world...
+Learn More
+huntress.com
+Collapse video player
 Sort by:
 Comments Section
 HodlDee
@@ -168,6 +175,12 @@
 
 Why are you in denial
 
+u/Bitcoin_Conference
+•
+Promoted
+This ad is for Bitcoin Asia. You're welcome.
+Learn More
+asia.b.tc
 ArmchairCryptologist
 •
 1mo ago
@@ -214,8 +227,12 @@
 
 Yes it has😐 &for some strange reason My emails to Coinkite or saying “an error, mail not sent”😐 &yes I performed a send mail test to My other email account😐
 
-Reply
-Share
+u/moneykineticssg
+•
+Promoted
+Tired of endless loan searches? Money Kinetics compares rates in 3 mins via Singpass. Start saving.
+Learn More
+moneykinetics.sg
 Mr_Rozay
 •
 1mo ago
@@ -242,24 +259,24 @@
 Share
 Lord_Chthulu
 •
+1mo ago
+
+I don't have any crypto and have no idea what Coinkite is but got an email today that said thanks for joining.
+
+Reply
+Share
+Artistic_Call_5234
+•
 <relative-time>
 
-I don't have any crypto and have no idea what Coinkite is but got an email today that said thanks for joining.
-
-Reply
-Share
-Artistic_Call_5234
+Welp, so much for that.
+
+Reply
+Share
+nymobster
 •
 <relative-time>
 
-Welp, so much for that.
-
-Reply
-Share
-nymobster
-•
-<relative-time>
-
 got one too. knew it was trash immediately. CK will never mail you!
 
 Reply
