--- reddit-coldcard-letter-db-leak@20260804T184002Z
+++ reddit-coldcard-letter-db-leak@20260804T211034Z
@@ -300,4 +300,12 @@
   
 Saying we did not leak anything is without a proper investigation of each claim is asking for trouble. Hopefully you can read this 1 month later. 
 
+comment: p1q6fdu
+parent: t3_1uiix4b
+author: olivesandparmesan
+created_utc: 1785876834
+edited: false
+body:
+Aged like ass milk huh? 🤔 
+
 more-stub: parent t1_oui9ob6 count 9
