--- reddit-coldcard-letter-db-leak@20260803T101900Z
+++ reddit-coldcard-letter-db-leak@20260803T103653Z
@@ -179,7 +179,7 @@
 •
 Promoted
 This ad is for Bitcoin Asia. You're welcome.
-Learn More
+Sign Up
 asia.b.tc
 ArmchairCryptologist
 •
@@ -227,12 +227,12 @@
 
 Yes it has😐 &for some strange reason My emails to Coinkite or saying “an error, mail not sent”😐 &yes I performed a send mail test to My other email account😐
 
-u/ClickUp_App
+u/CodeRabbitAI
 •
 Promoted
-Projects scattered across tools and spreadsheets? ClickUp brings tasks, timelines, and team communication into one place so you can keep projects moving.
-Learn More
-clickup.com
+The only tool that reviews everywhere you work. Sign up for a free trial!
+View More
+coderabbit.ai
 Mr_Rozay
 •
 1mo ago
