--- reddit-coldcard-letter-db-leak@20260803T131441Z
+++ reddit-coldcard-letter-db-leak@20260803T134328Z
@@ -178,10 +178,9 @@
 u/IBKR_Official
 •
 Promoted
-Find trading opportunities instantly with Interactive Brokers' AI-powered stock screener. Get Started.
+Stocks, options, futures, bonds & more. 100+ order types. AI tools. One platform.
 Sign Up
 interactivebrokers.com.sg
-Collapse video player
 ArmchairCryptologist
 •
 1mo ago
@@ -228,12 +227,12 @@
 
 Yes it has😐 &for some strange reason My emails to Coinkite or saying “an error, mail not sent”😐 &yes I performed a send mail test to My other email account😐
 
-u/Bitcoin_Conference
+u/moneykineticssg
 •
 Promoted
-The biggest Bitcoin conference is back. Get tickets.
+Money Kinetics compares loan rates instantly. Get started.
 Learn More
-asia.b.tc
+moneykinetics.sg
 Mr_Rozay
 •
 1mo ago
