--- reddit-coldcard-letter-db-leak@20260804T221058Z
+++ reddit-coldcard-letter-db-leak@20260804T224123Z
@@ -316,4 +316,12 @@
 body:
 Yep. Just in case anyone thought it could not get worse.
 
+comment: p1qrzaj
+parent: t3_1uiix4b
+author: Otherwise-Adagio4201
+created_utc: 1785882868
+edited: false
+body:
+Like I said to start this thread…Cold  Card has a problem.   
+
 more-stub: parent t1_oui9ob6 count 9
