--- reddit-ccq-stuck-splash-support@20260807T130016Z
+++ reddit-ccq-stuck-splash-support@20260807T193358Z
@@ -289,3 +289,19 @@
 There are ways to flip a coin for entropy and map that too the word list. Then you just need a device (could be offline laptop or old phone) to generate the 24th word options (and flip a coin 3 more times to select one).
 
 Hardware wallets aren't required for generating an offline seed, they just make it easier to stay airgapped.
+
+comment: p29ikpx
+parent: t1_p293m2p
+author: danialzo
+created_utc: 1786109937
+edited: false
+body:
+Yeah I bought a cheap MacBook Air (2017) without a battery for couple of bucks. Gonna format it and remove the wifi card to do this. 
+
+comment: p2aska0
+parent: t1_p2502q1
+author: Intelligent_Map_246
+created_utc: 1786122036
+edited: false
+body:
+Still use it as a offline seed generator and password manager 
