--- reddit-audit-cold-wallets-ai@20260808T050246Z
+++ reddit-audit-cold-wallets-ai@20260812T044318Z
@@ -129,3 +129,23 @@
 Reddit have some auto translation at least if you google something in some language and it will give result in that language, im not sure if it can be activated somehow so it does it without google too.
 
 [See this current post were are in](https://www.reddit.com/r/Bitcoin/comments/1vhsp5a/audit_of_cold_wallets/?tl=es-419), but I added ES to the url
+
+comment: p2v6avf
+parent: t1_p27s42w
+author: JumpProfessional3372
+created_utc: 1786380148
+edited: false
+body:
+With Coldcard you can pull the firmware code, compile the firmware update file and check that the bytes matches the one from the vendor (signed by them). I remember I did 2 years ago with the 5.2.2 firmware before flashing it.
+
+But I can't remember if i followed the process they put in their github to do this (Reproducible builds section) or a custom way of doing it. 
+
+[https://github.com/Coldcard/firmware](https://github.com/Coldcard/firmware)
+
+comment: p2vd2p6
+parent: t3_1vhsp5a
+author: No-Wrap3568
+created_utc: 1786381852
+edited: false
+body:
+Wallet with Two independent sources of randomness could be an answer to this
