--- reddit-bought-last-week@20260806T100138Z
+++ reddit-bought-last-week@20260806T163404Z
@@ -652,3 +652,27 @@
 edited: false
 body:
 Just use it as a signing device! Once you put your seed into it though, if you rly don’t trust coldcard, make a new seed phrase
+
+comment: p21gmrs
+parent: t1_p1ha3r3
+author: NoBrosCrypto
+created_utc: 1786016112
+edited: false
+body:
+With cold cards you do dice rolls to increase the randomness of the seed phrase generation. The more rolls the greater the entropy. So 99+ dice rolls will give you 256 bits of entropy for a 24 word seed phrase. Something like Ledger does that max entropy randomness for you without dice rolls. That 256 bits represents 2 to the power of 256 possible permutations (more atoms than in the known universe) for somebody to try to guess.
+
+comment: p21y41n
+parent: t1_p21gmrs
+author: Bionic_Push
+created_utc: 1786021780
+edited: false
+body:
+So why is this not done by default? Does it need a lot of computing power or something like that? 
+
+comment: p223fum
+parent: t1_p21y41n
+author: NoBrosCrypto
+created_utc: 1786023301
+edited: false
+body:
+I don’t know. It sounds stupid as hell. I never used a cold card so I’ve only learnt about it since the hack. Why you wouldn’t just dice roll the heck out of everything by default is beyond me.
