--- reddit-bought-last-week@20260804T214655Z
+++ reddit-bought-last-week@20260805T142627Z
@@ -67,14 +67,6 @@
 body:
 This. The bug/weakness has been identified but for psychological reasons I'd get a ledger or a trezor. 
 
-comment: p1gdyye
-parent: t1_p1gc61j
-author: OldHamburger7923
-created_utc: 1785767751
-edited: false
-body:
-No, don't rely on it to make your seed at all. Make your own seed with dice then import it. 
-
 comment: p1ge12a
 parent: t3_1veessi
 author: Emergency-Warthog-56
@@ -87,13 +79,11 @@
 
 comment: p1ge8gt
 parent: t1_p1gdbo7
-author: OldHamburger7923
+author: [deleted]
 created_utc: 1785767822
 edited: false
 body:
-You have no clue what ledger is doing. They could have backdoor code generating seeds and you wouldn't know till the rug pull. Don't recommend nonsense. Roll your own seed. Don't trust them to do it.
-
-The bigger issue is, if they let a major issue go for five years in critical code, what other code has issues? I was initially thinking the same as you but now I don't think I'd be comfortable with using it anymore.
+[deleted]
 
 comment: p1gei7t
 parent: t1_p1gbv0a
@@ -295,13 +285,11 @@
 
 comment: p1gumai
 parent: t1_p1gtro8
-author: OldHamburger7923
+author: [deleted]
 created_utc: 1785772095
 edited: false
 body:
-When I made my new wallet I loaded it into my old nano s and my new jade plus. I can see the same addresses in both after entering my passphrase. 
-
-I don't trust any of them further than I can throw them.
+[deleted]
 
 comment: p1gvwja
 parent: t3_1veessi
@@ -342,16 +330,6 @@
 edited: false
 body:
 Customs saved your bacon. Return that POS and use anything else, or keep your btc on coinbase. 
-
-comment: p1gygkc
-parent: t1_p1gw1kc
-author: OldHamburger7923
-created_utc: 1785773076
-edited: false
-body:
-Your previous post was spot on - we don't know what ledger is doing. They could be transferring our seed to their backend when we use their desktop client. 
-
-If their secure element really prevented the seed from ever leaving the device, then that would have made people feel a lot more secure, but its all smoke and mirrors. They plainly told us its not only possible, but they'll charge you to export your seed into the cloud. The exact opposite of what anyone with half a brain would want.
 
 comment: p1h0zjl
 parent: t1_p1gei7t
