--- reddit-bought-last-week@20260805T210121Z
+++ reddit-bought-last-week@20260806T033022Z
@@ -652,3 +652,11 @@
 Why are you against getting the ETF? Do you hate having a good night sleep or something? 🤣
 
 Unless you're in it for the tech then I apologize but I'm just in it for the price action which the ETF delivers the same and more, as you can right covered calls on your ETFs and generate 1 to 2% every month. That is a massive amount of money that very few asset classes can replicate 🤑🤑😮
+
+comment: p1yf7q3
+parent: t3_1veessi
+author: jake2422
+created_utc: 1785971994
+edited: false
+body:
+Just use it as a signing device! Once you put your seed into it though, if you rly don’t trust coldcard, make a new seed phrase
