--- reddit-bought-last-week@20260804T134501Z
+++ reddit-bought-last-week@20260804T161717Z
@@ -676,3 +676,11 @@
 FYI "stolen because the generated seeds were weak due to Coldcard using a software PRNG." means nothing to most people...
 
 Kinda like leaving your bike unlocked in a dark alley still requires a thief to steal it... The bike itself would be just fine if no thieves came along... Kinda like Coldcard woulda been just fine if no hackers came along... 
+
+comment: p1ob7md
+parent: t1_p1nciwv
+author: riscten
+created_utc: 1785859746
+edited: false
+body:
+You're actually right. I was under the impression that "hacking" implied remote access and/or code modification. I double checked and it does not. Just figuring out the weak pattern and deriving private keys is enough to be considered hacking. My apologies.
