--- reddit-bought-last-week@20260804T164748Z
+++ reddit-bought-last-week@20260804T194643Z
@@ -686,3 +686,15 @@
 You're actually right. I was under the impression that "hacking" implied remote access and/or code modification. I double checked and it does not. Just figuring out the weak pattern and deriving private keys is enough to be considered hacking. My apologies.
 
 Edit: Back to your original point, sure, Coldcard was hacked, but the hackable flaw was fixed. It's not like being hackable is a permanent state.
+
+comment: p1pme86
+parent: t3_1veessi
+author: DifficultSquash1517
+created_utc: 1785871535
+edited: false
+body:
+No one in this community will be satisfied until everybody's wallets are drained 
+
+Why are you against getting the ETF? Do you hate having a good night sleep or something? 🤣
+
+Unless you're in it for the tech then I apologize but I'm just in it for the price action which the ETF delivers the same and more, as you can right covered calls on your ETFs and generate 1 to 2% every month. That is a massive amount of money that very few asset classes can replicate 🤑🤑😮
