--- reddit-bought-last-week@20260804T194643Z
+++ reddit-bought-last-week@20260804T214655Z
@@ -617,26 +617,10 @@
 body:
 Keep it and receipts. Maybe you can get it on the lawsuit later.
 
-comment: p1kkdvc
-parent: t3_1veessi
-author: thelongestusernameee
-created_utc: 1785809555
-edited: false
-body:
-Keep it and receipts. Maybe you can get it on the lawsuit later.
-
 comment: p1kklji
 parent: t1_p1gqfba
 author: thelongestusernameee
 created_utc: 1785809627
-edited: false
-body:
-The shark is well fed and happy now :D
-
-comment: p1kkn1g
-parent: t1_p1gqfba
-author: thelongestusernameee
-created_utc: 1785809640
 edited: false
 body:
 The shark is well fed and happy now :D
