--- reddit-bought-last-week@20260804T161717Z
+++ reddit-bought-last-week@20260804T164748Z
@@ -681,6 +681,8 @@
 parent: t1_p1nciwv
 author: riscten
 created_utc: 1785859746
-edited: false
+edited: 1785860383
 body:
 You're actually right. I was under the impression that "hacking" implied remote access and/or code modification. I double checked and it does not. Just figuring out the weak pattern and deriving private keys is enough to be considered hacking. My apologies.
+
+Edit: Back to your original point, sure, Coldcard was hacked, but the hackable flaw was fixed. It's not like being hackable is a permanent state.
