--- reddit-bought-last-week@20260806T033022Z
+++ reddit-bought-last-week@20260806T100138Z
@@ -166,14 +166,6 @@
 Your seed phrase is your wallet basically. Coldcard just signs transactions with your seed.  The issue was 100% just letting Coldcard generate the seed for you. Use the dice roll method to generate your new seed instead and you’ll be fine.   
 Airgapped transaction signing is still super secure. Use it with confidence.   
 That being said, I use a Bitbox02 for convenience. 
-
-comment: p1gn0u0
-parent: t3_1veessi
-author: DonTheHolder
-created_utc: 1785770136
-edited: false
-body:
-Trash that shit bro. 🗑 
 
 comment: p1gnbs7
 parent: t3_1veessi
