--- reddit-bought-last-week@20260804T094313Z
+++ reddit-bought-last-week@20260804T134501Z
@@ -664,3 +664,15 @@
 edited: false
 body:
 The details surrounding how this bug happened in the first place don't paint a picture of a company you should trust. They don't seem to take security or software development seriously. Why would you assume such a company only had this one single bug? There could easily be others now, in the past, or in the future.
+
+comment: p1nciwv
+parent: t1_p1ha236
+author: DavidssonA
+created_utc: 1785850585
+edited: false
+body:
+But it took hacking to get it... Didn't it?   
+  
+FYI "stolen because the generated seeds were weak due to Coldcard using a software PRNG." means nothing to most people...
+
+Kinda like leaving your bike unlocked in a dark alley still requires a thief to steal it... The bike itself would be just fine if no thieves came along... Kinda like Coldcard woulda been just fine if no hackers came along... 
