--- reddit-bought-last-week@20260805T142627Z
+++ reddit-bought-last-week@20260805T210121Z
@@ -132,14 +132,6 @@
 edited: false
 body:
 Fait toi rembourser. Perte de confiance et achète un Trezor à la place 
-
-comment: p1gi7iv
-parent: t3_1veessi
-author: Expert-Evening9303
-created_utc: 1785768874
-edited: false
-body:
-It's fine. Just make sure you update the firmware, generate your seed phrase with dice rolls, and add a passphrase.
 
 comment: p1gi8fs
 parent: t3_1veessi
