--- newsbitcom-class-action-threat@20260812T115812Z
+++ newsbitcom-class-action-threat@20260812T182813Z
@@ -82,39 +82,28 @@
 © 2026 Saint Bitts LLC Bitcoin.com. All rights reserved
 Support
 support@bitcoin.com
-MOST POPULAR
-Toledo, Ohio Resident Wins Two Scratch-off Jackpots in a Single Day, Nets $384,562 After Taxes
-1 day ago
-Buried in the FCC’s New Robot Import Ban: The 35% Foreign-Parts Rule
-7 hours ago
-Blackrock Cuts IBIT Bitcoin Conversion Minimum From $25M to $1M
-1 day ago
-White House to Senate: Pass Clarity Act by Sept. 15 or Never
-1 day ago
-The Brutal Truth About Crypto Infrastructure and M&A
-1 day ago
 LATEST NEWS
-Strategy CEO: Firm Will Resume Bitcoin Buying Later This Year
-28 minutes ago
-Metaplanet, Hut8 Move $125 Million in Bitcoin off Exchanges
+Blackrock’s $50.2M Inflow Offsets Selling as Bitcoin ETFs Turn Green
 1 hour ago
-Bitwise Cuts 14% of Staff as Its Own CIO Calls a Bitcoin Bottom
-2 hours ago
-Glassnode: Bitcoin Sellers Showing Exhaustion Around $64,000, But No Bottom Yet
+Tether’s USDT Supply Shrinks by $4B in 60 Days as Crypto Demand Cools
 3 hours ago
-XRP Millionaire Wallets Rise Again as Token Price Falls 29% in 3 Months
+Inflation Cools, Bitcoin Stalls as Fed Faces Its Next Big Test
 4 hours ago
+Ethereum’s Oldest Whales Wake up as Millions Leave Deep Storage
+4 hours ago
+Harmony’s ONE Plunges 30% After Attacker Mints 4 Billion Tokens
+5 hours ago
 LATEST NEWS
-Strategy CEO: Firm Will Resume Bitcoin Buying Later This Year
-28 minutes ago
-Metaplanet, Hut8 Move $125 Million in Bitcoin off Exchanges
+Blackrock’s $50.2M Inflow Offsets Selling as Bitcoin ETFs Turn Green
 1 hour ago
-Bitwise Cuts 14% of Staff as Its Own CIO Calls a Bitcoin Bottom
-2 hours ago
-Glassnode: Bitcoin Sellers Showing Exhaustion Around $64,000, But No Bottom Yet
+Tether’s USDT Supply Shrinks by $4B in 60 Days as Crypto Demand Cools
 3 hours ago
-XRP Millionaire Wallets Rise Again as Token Price Falls 29% in 3 Months
+Inflation Cools, Bitcoin Stalls as Fed Faces Its Next Big Test
 4 hours ago
+Ethereum’s Oldest Whales Wake up as Millions Leave Deep Storage
+4 hours ago
+Harmony’s ONE Plunges 30% After Attacker Mints 4 Billion Tokens
+5 hours ago
 LATEST PODCASTS
 Can Bitcoin Mining Save Renewable Energy in the AI Era?
 50:35Aug 11, 2026
@@ -126,6 +115,17 @@
 57:02Aug 07, 2026
 Inside Bittensor: The Race to Decentralize AI
 53:12Aug 04, 2026
+MOST POPULAR
+Buried in the FCC’s New Robot Import Ban: The 35% Foreign-Parts Rule
+14 hours ago
+White House to Senate: Pass Clarity Act by Sept. 15 or Never
+1 day ago
+XRP Millionaire Wallets Rise Again as Token Price Falls 29% in 3 Months
+11 hours ago
+Arthur Hayes: Yen Rescue Could Fuel a Bitcoin Rally
+1 day ago
+Anthropic Emerges as 'Frontier AI Lab' Behind Riot’s $9.1B AI Deal
+1 day ago
 Related articles
 Jul 11, 2026
 Bitcoin Policy Institute Fights $293B Wallet Grab as Noah Doe Court Date Nears
