--- newsbitcom-class-action-threat@20260807T142524Z
+++ newsbitcom-class-action-threat@20260807T205359Z
@@ -83,38 +83,38 @@
 Support
 support@bitcoin.com
 LATEST NEWS
-BIP-110 Supporters Prepare PoW Switch If Miners Refuse Soft Fork Plan
-51 minutes ago
-Cathie Wood's Ark Buys $21M in Block, $2.3M in SpaceX
-3 hours ago
-Bitcoin Red Team Finds 4,962 Flaws After Coldcard Hack
+Bitcoin Lightning Nodes Hit as BTCPay Signals Emergency 2.4.2 Fix
+44 minutes ago
+ForumPay Brings Crypto Payments to Shopify Merchants
+54 minutes ago
+Bitcoin Tops $65,340 as BIP 110 Fight Raises Hard Fork Risk
+2 hours ago
+Trezor: Someone Always Holds Your Keys. It Should Be You.
 4 hours ago
-Tesla, SpaceX Pick Texas Site for Musk's $16.8B Chip Plant
-5 hours ago
-MARA Reports $611M Loss While Miners Deposit 581 BTC to NYDIG
-6 hours ago
+Wintermute Registers as US Broker-Dealer, Eyes Tokenized Stocks
+4 hours ago
 LATEST NEWS
-BIP-110 Supporters Prepare PoW Switch If Miners Refuse Soft Fork Plan
-51 minutes ago
-Cathie Wood's Ark Buys $21M in Block, $2.3M in SpaceX
-3 hours ago
-Bitcoin Red Team Finds 4,962 Flaws After Coldcard Hack
+Bitcoin Lightning Nodes Hit as BTCPay Signals Emergency 2.4.2 Fix
+44 minutes ago
+ForumPay Brings Crypto Payments to Shopify Merchants
+54 minutes ago
+Bitcoin Tops $65,340 as BIP 110 Fight Raises Hard Fork Risk
+2 hours ago
+Trezor: Someone Always Holds Your Keys. It Should Be You.
 4 hours ago
-Tesla, SpaceX Pick Texas Site for Musk's $16.8B Chip Plant
-5 hours ago
-MARA Reports $611M Loss While Miners Deposit 581 BTC to NYDIG
-6 hours ago
+Wintermute Registers as US Broker-Dealer, Eyes Tokenized Stocks
+4 hours ago
 MOST POPULAR
 Google Scraps Google Earth’s AI-Generated Imagery Feature After Misinformation Warnings
-9 hours ago
-China Says It Cracked the Chipmaking Tech the West Spent Billions Trying to Keep From It
+16 hours ago
+Report: Crypto Holders Lose $30M as Wrench Attacks Spiral Worldwide
 1 day ago
-Report: Crypto Holders Lose $30M as Wrench Attacks Spiral Worldwide
-23 hours ago
 Bitcoin Nears Chain Split as BIP-110 Rebels Defy Global Hashpower
 1 day ago
 One Day Left as Senate Faces Final Push for CLARITY Act Crypto Vote
-14 hours ago
+21 hours ago
+Solo Bitcoin Miner Defies the Odds, Lands $200K Block Reward Jackpot
+1 day ago
 LATEST PODCASTS
 Who Really Owns Crypto Users? Bitcoin Self-Custody, Ethereum Issuance & the App vs. Chain Debate
 57:02Aug 07, 2026
@@ -139,7 +139,7 @@
 May 28, 2026
 Anonymous Plaintiff Claims $293B Bitcoin Haul, Targets Satoshi's Dormant Wallets in NY Court Case
 Regulation & Legal
-2 days ago
+3 days ago
 Dutch Court Hears Crypto Dispute Kidnapping Case
 Regulation & Legal
 4 days ago
