--- reddit-wild-west-sentiment@20260805T142650Z
+++ reddit-wild-west-sentiment@20260806T033054Z
@@ -959,3 +959,13 @@
 edited: false
 body:
 All things you don't have to worry about with a bank and fiat currency. That's why it will never go mainstream.
+
+comment: p1ypknv
+parent: t1_p1l6l0b
+author: Ok-Astronaut4952
+created_utc: 1785975392
+edited: false
+body:
+Wether or not infesting investing in an ETF meets your definition of using crypto or not, the existence of an ETF means Bitcoin is mainstream as hell
+
+And yeah that is probably the full extent of what crypto will do
