--- reddit-wild-west-sentiment@20260806T100212Z
+++ reddit-wild-west-sentiment@20260806T163436Z
@@ -31,11 +31,11 @@
 
 comment: p1g3w9n
 parent: t3_1vedx0e
-author: IllllIIlIllIllllIlll
+author: [deleted]
 created_utc: 1785764977
-edited: false
-body:
-Insert "first time?" meme
+edited: 1786027907
+body:
+[deleted]
 
 comment: p1g3xsr
 parent: t3_1vedx0e
@@ -592,11 +592,11 @@
 
 comment: p1hvrfo
 parent: t1_p1h1k3m
-author: CraftyBrother6974
+author: [deleted]
 created_utc: 1785781350
 edited: 1785781689
 body:
-The average person also doesn't know shit about the mechanics of the traditional financial system either. Why even would you there's many things that are completely irrelevant to know unless you work in the space. There's plenty of people who buy stocks, but I doubt they know the fundamentals of how their share is custodied, how voting rights works, etc. You don't even really need to know why it works you just need to know a well diversified group of stocks is going to outperform the alternative (cash) over a period of greater than 3-5 years 99% of the time. Considering most people are also averaging in as they are paid periodically, all you really need to know is that you will get richer by buying some stocks everytime you get a paycheck and the drawdowns won't even be that crazy until you've already seen your savings massively multiply.
+[removed]
 
 comment: p1hzpd0
 parent: t1_p1hbek1
@@ -844,11 +844,11 @@
 
 comment: p1je4cp
 parent: t1_p1inw2w
-author: CraftyBrother6974
+author: [deleted]
 created_utc: 1785795951
 edited: 1785796138
 body:
-yeah but they also don't understand the basics of the current monetary system either is my point. I'd say at this point, most people that do understand monetary economics also understand the value/how bitcoin works at least a high level. Obviously it's a very small group of people that will understand how it works mathematically. I don't think we will actually ever reach a point of significantly greater saturation there. That's at least my take after working in financial markets for the last decade. I've been a currency trader for my whole career and have done a bit of crypto/stables at banks, asset management companies and a start up and I never actually had to understand how the technology worked. I'd prefer to outsource that to someone smarter than me. This is even included in CFA and MBA curriculums now.
+[removed]
 
 comment: p1jek3d
 parent: t3_1vedx0e
