--- reddit-wild-west-sentiment@20260806T033054Z
+++ reddit-wild-west-sentiment@20260806T100212Z
@@ -969,3 +969,11 @@
 Wether or not infesting investing in an ETF meets your definition of using crypto or not, the existence of an ETF means Bitcoin is mainstream as hell
 
 And yeah that is probably the full extent of what crypto will do
+
+comment: p1zp15k
+parent: t1_p1ypknv
+author: Zaytion_
+created_utc: 1785987377
+edited: false
+body:
+Then we have different definitions of mainstream.
