--- reddit-wild-west-sentiment@20260806T163436Z
+++ reddit-wild-west-sentiment@20260807T061754Z
@@ -977,3 +977,19 @@
 edited: false
 body:
 Then we have different definitions of mainstream.
+
+comment: p23bxj9
+parent: t3_1vedx0e
+author: Left_Entrepreneur918
+created_utc: 1786034787
+edited: false
+body:
+No Normies or Noobs here is much better. I myself am a HUGE crypto bro and technology enthusiast. I’m in my basement with dice making seeds all day all night just blasting seeds. If anyone doesn’t use 99 dice rolls in the dark, with no phones, cameras, or any electronics they are lame. I also usually wait for my mom to go to work before blasting my seeds all over the basement. 
+
+comment: p249gfe
+parent: t1_p1gjz5t
+author: markphillips401
+created_utc: 1786043085
+edited: false
+body:
+It's not going to be mainstream if you can't just use it :)
