--- reddit-wild-west-sentiment@20260804T095513Z
+++ reddit-wild-west-sentiment@20260804T134524Z
@@ -935,3 +935,37 @@
 edited: false
 body:
 ETFs isn't using crypto. If all crypto ever does is end up in an ETF, it has failed.
+
+comment: p1mcefn
+parent: t3_1vedx0e
+author: Javanaut018
+created_utc: 1785837792
+edited: false
+body:
+This space is a bunch of frightened chicken running all over the place currently 
+
+comment: p1mz3k2
+parent: t1_p1k5vn2
+author: tekmiester
+created_utc: 1785846536
+edited: false
+body:
+To make use of their totally reliable off grid Internet access? Or to own an asset they will likely be lost when they pass away so their family gets nothing?
+
+comment: p1n0qdv
+parent: t1_p1hvrfo
+author: tekmiester
+created_utc: 1785847062
+edited: false
+body:
+They understand if they put their money in a bank, it will still be there in 6 months or the government will reimburse them. They understand if they but stock in Amazon it will likely go up in value over time. 
+
+More importantly they understand there is a lot of ways you can lose your crypto investment no matter how careful you are.
+
+comment: p1n1ago
+parent: t1_p1gu75s
+author: tekmiester
+created_utc: 1785847238
+edited: false
+body:
+All things you don't have to worry about with a bank and fiat currency. That's why it will never go mainstream.
