--- therage-coldcard-hack-guide@20260805T022428Z
+++ therage-coldcard-hack-guide@20260807T003321Z
@@ -88,6 +88,11 @@
 To complete Subscribe, click the confirmation link in your inbox. If it doesn’t arrive within 3 minutes, check your spam folder.
 Ok, Thanks
 Latest posts
+Policy
+Is Debanking The New Immigration Policy?
+by
+Nicholas Anthony
+August 06, 2026
 Bitcoin
 Everything You Need to Know About the COLDCARD Hack
 Featured post
@@ -106,12 +111,6 @@
 by
 L0la L33tz
 July 22, 2026
-Policy
-French Finance Ministry Sued Over Global Surveillance Database
-Featured post
-by
-L0la L33tz
-July 08, 2026
 Copyright ©
 2026
 The Rage. Published with Ghost and
