--- reddit-am-i-affected-dice@20260804T134036Z
+++ reddit-am-i-affected-dice@20260804T164250Z
@@ -225,3 +225,19 @@
 edited: false
 body:
 it sounds like your firmware didn't have this bug and the dice rolls and passphrase are added peace of mind. Your good. And don't do multisig without understanding everything thats required to back it up
+
+comment: p1oi1sy
+parent: t1_p1lhbkk
+author: HodlDee
+created_utc: 1785861464
+edited: false
+body:
+DM me your 16 digit invoice ID. I’ll do my best to help.
+
+comment: p1oinjj
+parent: t1_p1oi1sy
+author: AntZealousideal3728
+created_utc: 1785861616
+edited: false
+body:
+Just DMd you! 
