--- reddit-am-i-affected-dice@20260804T164250Z
+++ reddit-am-i-affected-dice@20260804T194231Z
@@ -241,3 +241,11 @@
 edited: false
 body:
 Just DMd you! 
+
+comment: p1po8lz
+parent: t3_1vewq6y
+author: kenorb
+created_utc: 1785872021
+edited: false
+body:
+It seems all Mk3 are affected, so don't take any chances. It doesn't matter how many dice rolls you've, it's a matter of time for attackers. [Someone else reported](https://www.reddit.com/r/Bitcoin/comments/1ja3uua/coldcard_mk4_ux_flaw/) on Reddit 50+ rolls and it was already gone like 2+ years ago.
