--- reddit-am-i-affected-dice@20260804T194231Z
+++ reddit-am-i-affected-dice@20260804T214239Z
@@ -249,3 +249,11 @@
 edited: false
 body:
 It seems all Mk3 are affected, so don't take any chances. It doesn't matter how many dice rolls you've, it's a matter of time for attackers. [Someone else reported](https://www.reddit.com/r/Bitcoin/comments/1ja3uua/coldcard_mk4_ux_flaw/) on Reddit 50+ rolls and it was already gone like 2+ years ago.
+
+comment: p1qe3zh
+parent: t1_p1po8lz
+author: Individual_Gate9375
+created_utc: 1785878908
+edited: false
+body:
+Apparently I created my wallet before the bugged firmware ever existed. Saw that mentioned here and went to look. March 2021 was the first time the bugged firmware was pushed out and I had my first transaction on my wallet in January 2021. So I generated my seed with the actual randomizer chip on the device. I would have apparently been unaffected even if I didn't have the dice rolls or the passphrase.
