--- reddit-am-i-affected-dice@20260804T214239Z
+++ reddit-am-i-affected-dice@20260804T224338Z
@@ -257,3 +257,11 @@
 edited: false
 body:
 Apparently I created my wallet before the bugged firmware ever existed. Saw that mentioned here and went to look. March 2021 was the first time the bugged firmware was pushed out and I had my first transaction on my wallet in January 2021. So I generated my seed with the actual randomizer chip on the device. I would have apparently been unaffected even if I didn't have the dice rolls or the passphrase.
+
+comment: p1qrgcs
+parent: t1_p1po8lz
+author: adequate_redditor
+created_utc: 1785882710
+edited: false
+body:
+I don’t think that’s true. If you did 100 rolls you did not used the flawed random generator on the device. That doesn’t mean that the device is safe from other vulnerabilities though…
