--- reddit-am-i-affected-dice@20260804T082409Z
+++ reddit-am-i-affected-dice@20260804T134036Z
@@ -177,3 +177,51 @@
 edited: false
 body:
 Technically you are safe but I would sleep better if I create a new seed phrase with passphrase on the Trezor wallet.
+
+comment: p1m72rh
+parent: t1_p1l5ogr
+author: Silent_Ad_9963
+created_utc: 1785835214
+edited: false
+body:
+And he generated his seed before the broken firmware was even released .
+
+On top of that the code has now been audited by multiple sources more than any other code out there 
+
+I 100% agree that moving just because of emotional feeling and panic while all info say seed is fine is risky , possibly more risky than staying where he is now. 
+
+Unfortunately now this "can't trust anything this dev ever wrote or done" risk to create more problems down the road ... Is an emotional response and it should be treated as such
+
+comment: p1mg22g
+parent: t3_1vewq6y
+author: JunketTurbulent2114
+created_utc: 1785839418
+edited: false
+body:
+The issue is coldcard likely won't be around in the future to support firmware upgrades and what not.  Probably a good idea to go back to trezor. But the dice I think saved your ass. 
+
+comment: p1mg5nu
+parent: t1_p1kyeov
+author: JunketTurbulent2114
+created_utc: 1785839461
+edited: false
+body:
+They told us to not trust them and we didn't bother to verify.  Our fault. 
+
+comment: p1mjocd
+parent: t1_p1m72rh
+author: grraarr
+created_utc: 1785840938
+edited: false
+body:
+Thank you! I didn't even calculate the firmware timeline but that's a great point.
+
+I also see a bunch of people not only insisting on switching wallets, but also recommending passphrases and multisig blindly without knowing what they're meant to protect against or if they're appropriate for the individual's inheritance model. I understand those are important security measures, but lot of people are hardening and will be locked out.
+
+comment: p1movkz
+parent: t3_1vewq6y
+author: No_Position_8581
+created_utc: 1785842980
+edited: false
+body:
+it sounds like your firmware didn't have this bug and the dice rolls and passphrase are added peace of mind. Your good. And don't do multisig without understanding everything thats required to back it up
