--- newsbitcom-who-lost-who-at-risk@20260803T000343Z
+++ newsbitcom-who-lost-who-at-risk@20260803T003323Z
@@ -110,17 +110,27 @@
 Support
 support@bitcoin.com
 LATEST NEWS
+Bitgo CEO Funds 100 BTC Wallet, Dares Anthropic's AI to Steal It
+32 seconds ago
 Samson Mow Shares 5 Urgent Steps for Coldcard Users Facing Losses
-33 minutes ago
+1 hour ago
 Bitcoin Miners Face August Showdown After Revenue Rebound
+2 hours ago
+Coldcard Hacker Gets Brazen Bitcoin Laundering Offer Onchain
+3 hours ago
+Coinkite Faces Class Action Threat as Bitcoin Wallet Bug Costs Users Over 1,300 BTC
+4 hours ago
+LATEST NEWS
+Bitgo CEO Funds 100 BTC Wallet, Dares Anthropic's AI to Steal It
+32 seconds ago
+Samson Mow Shares 5 Urgent Steps for Coldcard Users Facing Losses
 1 hour ago
+Bitcoin Miners Face August Showdown After Revenue Rebound
+2 hours ago
 Coldcard Hacker Gets Brazen Bitcoin Laundering Offer Onchain
-LATEST NEWS
-Samson Mow Shares 5 Urgent Steps for Coldcard Users Facing Losses
-33 minutes ago
-Bitcoin Miners Face August Showdown After Revenue Rebound
-1 hour ago
-Coldcard Hacker Gets Brazen Bitcoin Laundering Offer Onchain
+3 hours ago
+Coinkite Faces Class Action Threat as Bitcoin Wallet Bug Costs Users Over 1,300 BTC
+4 hours ago
 PRESS RELEASES
 CoinMortgage Launches Bitcoin Purchase Financing Services
 4 days ago
@@ -132,16 +142,6 @@
 5 days ago
 TRX Futures Listing Launches on Bitnomial, Broadening Regulated U.S. Derivatives Access to TRON
 6 days ago
-3 hours ago
-Coinkite Faces Class Action Threat as Bitcoin Wallet Bug Costs Users Over 1,300 BTC
-4 hours ago
-BTC Plunge Triggers Altcoin Sell-off as ADA Bucks the Trend
-5 hours ago
-3 hours ago
-Coinkite Faces Class Action Threat as Bitcoin Wallet Bug Costs Users Over 1,300 BTC
-4 hours ago
-BTC Plunge Triggers Altcoin Sell-off as ADA Bucks the Trend
-5 hours ago
 LATEST PODCASTS
 Franklin Templeton: The $Trillion Tokenization Opportunity Explained
 32:16Aug 01, 2026
