--- newsbitcom-who-lost-who-at-risk@20260802T223233Z
+++ newsbitcom-who-lost-who-at-risk@20260802T230333Z
@@ -110,26 +110,26 @@
 Support
 support@bitcoin.com
 LATEST NEWS
+Bitcoin Miners Face August Showdown After Revenue Rebound
+33 minutes ago
 Coldcard Hacker Gets Brazen Bitcoin Laundering Offer Onchain
-58 minutes ago
+1 hour ago
 Coinkite Faces Class Action Threat as Bitcoin Wallet Bug Costs Users Over 1,300 BTC
-1 hour ago
+3 hours ago
 BTC Plunge Triggers Altcoin Sell-off as ADA Bucks the Trend
-3 hours ago
+4 hours ago
 Iran Rejects Trump Deal as Hormuz Tensions Grip Markets Again
-4 hours ago
-Coinkite Under Fire for Retaining Customer Emails After $88M Coldcard Hack
 5 hours ago
 LATEST NEWS
+Bitcoin Miners Face August Showdown After Revenue Rebound
+33 minutes ago
 Coldcard Hacker Gets Brazen Bitcoin Laundering Offer Onchain
-58 minutes ago
+1 hour ago
 Coinkite Faces Class Action Threat as Bitcoin Wallet Bug Costs Users Over 1,300 BTC
-1 hour ago
+3 hours ago
 BTC Plunge Triggers Altcoin Sell-off as ADA Bucks the Trend
-3 hours ago
+4 hours ago
 Iran Rejects Trump Deal as Hormuz Tensions Grip Markets Again
-4 hours ago
-Coinkite Under Fire for Retaining Customer Emails After $88M Coldcard Hack
 5 hours ago
 PRESS RELEASES
 CoinMortgage Launches Bitcoin Purchase Financing Services
@@ -137,16 +137,16 @@
 TRX Spot and Perpetuals Listing Launches on Backpack, Expanding Access to the TRON Ecosystem
 4 days ago
 Chainstack is the First to Support Robinhood Chain With Both Managed RPC and Self-Hosted Nodes
+4 days ago
+Nigeria's First SEC-Licensed Exchange, Quidax, Expands Stablecoin Infrastructure to Over 21 Countries
+5 days ago
+TRX Futures Listing Launches on Bitnomial, Broadening Regulated U.S. Derivatives Access to TRON
+6 days ago
 LATEST PODCASTS
 Franklin Templeton: The $Trillion Tokenization Opportunity Explained
 32:16Aug 01, 2026
 Has crypto finally reached the end of its bear market?
 47:57Jul 31, 2026
-4 days ago
-Nigeria's First SEC-Licensed Exchange, Quidax, Expands Stablecoin Infrastructure to Over 21 Countries
-5 days ago
-TRX Futures Listing Launches on Bitnomial, Broadening Regulated U.S. Derivatives Access to TRON
-6 days ago
 Why Fidelity Says Institutions Are Finally Buying Bitcoin
 35:29Jul 28, 2026
 The Great Capitulation: Mallers Exits, Satsuma Liquidates & BitMEX Shuts Down
