--- newsbitcom-who-lost-who-at-risk@20260802T213222Z
+++ newsbitcom-who-lost-who-at-risk@20260802T220321Z
@@ -129,23 +129,27 @@
 Another Set of Long-Silent Bitcoin Wallets Move Millions During BTC Decline
 Featured
 LATEST NEWS
+Coldcard Hacker Gets Brazen Bitcoin Laundering Offer Onchain
+33 minutes ago
 Coinkite Faces Class Action Threat as Bitcoin Wallet Bug Costs Users Over 1,300 BTC
 1 hour ago
 BTC Plunge Triggers Altcoin Sell-off as ADA Bucks the Trend
-2 hours ago
-Iran Rejects Trump Deal as Hormuz Tensions Grip Markets Again
 LATEST NEWS
+Coldcard Hacker Gets Brazen Bitcoin Laundering Offer Onchain
+33 minutes ago
 Coinkite Faces Class Action Threat as Bitcoin Wallet Bug Costs Users Over 1,300 BTC
 1 hour ago
 BTC Plunge Triggers Altcoin Sell-off as ADA Bucks the Trend
-2 hours ago
+3 hours ago
 Iran Rejects Trump Deal as Hormuz Tensions Grip Markets Again
-PRESS RELEASES
-CoinMortgage Launches Bitcoin Purchase Financing Services
-4 days ago
-TRX Spot and Perpetuals Listing Launches on Backpack, Expanding Access to the TRON Ecosystem
-4 days ago
-Chainstack is the First to Support Robinhood Chain With Both Managed RPC and Self-Hosted Nodes
+4 hours ago
+Coinkite Under Fire for Retaining Customer Emails After $88M Coldcard Hack
+5 hours ago
+3 hours ago
+Iran Rejects Trump Deal as Hormuz Tensions Grip Markets Again
+4 hours ago
+Coinkite Under Fire for Retaining Customer Emails After $88M Coldcard Hack
+5 hours ago
 LATEST PODCASTS
 Franklin Templeton: The $Trillion Tokenization Opportunity Explained
 32:16Aug 01, 2026
@@ -157,16 +161,12 @@
 43:26Jul 27, 2026
 Gauntlet CEO Tarun Chitra on DeFi Risk, RWAs, Japan & Crypto’s AI Moment
 46:01Jul 24, 2026
-3 hours ago
-Coinkite Under Fire for Retaining Customer Emails After $88M Coldcard Hack
-4 hours ago
-Stablecoin Supply Sheds $15 Billion in Biggest Drop Since Terra
-5 hours ago
-3 hours ago
-Coinkite Under Fire for Retaining Customer Emails After $88M Coldcard Hack
-4 hours ago
-Stablecoin Supply Sheds $15 Billion in Biggest Drop Since Terra
-5 hours ago
+PRESS RELEASES
+CoinMortgage Launches Bitcoin Purchase Financing Services
+4 days ago
+TRX Spot and Perpetuals Listing Launches on Backpack, Expanding Access to the TRON Ecosystem
+4 days ago
+Chainstack is the First to Support Robinhood Chain With Both Managed RPC and Self-Hosted Nodes
 4 days ago
 Nigeria's First SEC-Licensed Exchange, Quidax, Expands Stablecoin Infrastructure to Over 21 Countries
 5 days ago
