--- newsbitcom-who-lost-who-at-risk@20260803T023430Z
+++ newsbitcom-who-lost-who-at-risk@20260803T030614Z
@@ -128,28 +128,6 @@
 May 27, 2026
 Another Set of Long-Silent Bitcoin Wallets Move Millions During BTC Decline
 Featured
-LATEST NEWS
-CLARITY Act Faces New Senate Threat Over Trump Crypto Gains
-1 hour ago
-Bitgo CEO Funds 100 BTC Wallet, Dares Anthropic's AI to Steal It
-2 hours ago
-Samson Mow Shares 5 Urgent Steps for Coldcard Users Facing Losses
-3 hours ago
-Bitcoin Miners Face August Showdown After Revenue Rebound
-4 hours ago
-Coldcard Hacker Gets Brazen Bitcoin Laundering Offer Onchain
-5 hours ago
-LATEST NEWS
-CLARITY Act Faces New Senate Threat Over Trump Crypto Gains
-1 hour ago
-Bitgo CEO Funds 100 BTC Wallet, Dares Anthropic's AI to Steal It
-2 hours ago
-Samson Mow Shares 5 Urgent Steps for Coldcard Users Facing Losses
-3 hours ago
-Bitcoin Miners Face August Showdown After Revenue Rebound
-4 hours ago
-Coldcard Hacker Gets Brazen Bitcoin Laundering Offer Onchain
-5 hours ago
 PRESS RELEASES
 CoinMortgage Launches Bitcoin Purchase Financing Services
 4 days ago
@@ -172,3 +150,25 @@
 43:26Jul 27, 2026
 Gauntlet CEO Tarun Chitra on DeFi Risk, RWAs, Japan & Crypto’s AI Moment
 46:01Jul 24, 2026
+LATEST NEWS
+Bitcoin Wasn’t Hacked in Coldcard Attack, Pompliano Explains
+17 minutes ago
+CLARITY Act Faces New Senate Threat Over Trump Crypto Gains
+1 hour ago
+Bitgo CEO Funds 100 BTC Wallet, Dares Anthropic's AI to Steal It
+2 hours ago
+Samson Mow Shares 5 Urgent Steps for Coldcard Users Facing Losses
+4 hours ago
+Bitcoin Miners Face August Showdown After Revenue Rebound
+5 hours ago
+LATEST NEWS
+Bitcoin Wasn’t Hacked in Coldcard Attack, Pompliano Explains
+17 minutes ago
+CLARITY Act Faces New Senate Threat Over Trump Crypto Gains
+1 hour ago
+Bitgo CEO Funds 100 BTC Wallet, Dares Anthropic's AI to Steal It
+2 hours ago
+Samson Mow Shares 5 Urgent Steps for Coldcard Users Facing Losses
+4 hours ago
+Bitcoin Miners Face August Showdown After Revenue Rebound
+5 hours ago
