--- newsbitcom-who-lost-who-at-risk@20260803T003323Z
+++ newsbitcom-who-lost-who-at-risk@20260803T010354Z
@@ -111,26 +111,26 @@
 support@bitcoin.com
 LATEST NEWS
 Bitgo CEO Funds 100 BTC Wallet, Dares Anthropic's AI to Steal It
-32 seconds ago
+31 minutes ago
 Samson Mow Shares 5 Urgent Steps for Coldcard Users Facing Losses
 1 hour ago
 Bitcoin Miners Face August Showdown After Revenue Rebound
-2 hours ago
+3 hours ago
 Coldcard Hacker Gets Brazen Bitcoin Laundering Offer Onchain
-3 hours ago
+4 hours ago
 Coinkite Faces Class Action Threat as Bitcoin Wallet Bug Costs Users Over 1,300 BTC
-4 hours ago
+5 hours ago
 LATEST NEWS
 Bitgo CEO Funds 100 BTC Wallet, Dares Anthropic's AI to Steal It
-32 seconds ago
+31 minutes ago
 Samson Mow Shares 5 Urgent Steps for Coldcard Users Facing Losses
 1 hour ago
 Bitcoin Miners Face August Showdown After Revenue Rebound
-2 hours ago
+3 hours ago
 Coldcard Hacker Gets Brazen Bitcoin Laundering Offer Onchain
-3 hours ago
+4 hours ago
 Coinkite Faces Class Action Threat as Bitcoin Wallet Bug Costs Users Over 1,300 BTC
-4 hours ago
+5 hours ago
 PRESS RELEASES
 CoinMortgage Launches Bitcoin Purchase Financing Services
 4 days ago
