--- newsbitcom-who-lost-who-at-risk@20260803T013304Z
+++ newsbitcom-who-lost-who-at-risk@20260803T020403Z
@@ -109,44 +109,44 @@
 © 2026 Saint Bitts LLC Bitcoin.com. All rights reserved
 Support
 support@bitcoin.com
-LATEST NEWS
-Bitgo CEO Funds 100 BTC Wallet, Dares Anthropic's AI to Steal It
-1 hour ago
-Samson Mow Shares 5 Urgent Steps for Coldcard Users Facing Losses
-2 hours ago
-Bitcoin Miners Face August Showdown After Revenue Rebound
-LATEST NEWS
-Bitgo CEO Funds 100 BTC Wallet, Dares Anthropic's AI to Steal It
-1 hour ago
-Samson Mow Shares 5 Urgent Steps for Coldcard Users Facing Losses
-2 hours ago
-Bitcoin Miners Face August Showdown After Revenue Rebound
 PRESS RELEASES
 CoinMortgage Launches Bitcoin Purchase Financing Services
 4 days ago
 TRX Spot and Perpetuals Listing Launches on Backpack, Expanding Access to the TRON Ecosystem
 4 days ago
 Chainstack is the First to Support Robinhood Chain With Both Managed RPC and Self-Hosted Nodes
-3 hours ago
-Coldcard Hacker Gets Brazen Bitcoin Laundering Offer Onchain
-4 hours ago
-Coinkite Faces Class Action Threat as Bitcoin Wallet Bug Costs Users Over 1,300 BTC
-5 hours ago
-3 hours ago
-Coldcard Hacker Gets Brazen Bitcoin Laundering Offer Onchain
-4 hours ago
-Coinkite Faces Class Action Threat as Bitcoin Wallet Bug Costs Users Over 1,300 BTC
-5 hours ago
 5 days ago
 Nigeria's First SEC-Licensed Exchange, Quidax, Expands Stablecoin Infrastructure to Over 21 Countries
 5 days ago
 TRX Futures Listing Launches on Bitnomial, Broadening Regulated U.S. Derivatives Access to TRON
-6 days ago
+Jul 27, 2026
+LATEST NEWS
+CLARITY Act Faces New Senate Threat Over Trump Crypto Gains
+33 minutes ago
+Bitgo CEO Funds 100 BTC Wallet, Dares Anthropic's AI to Steal It
+1 hour ago
+Samson Mow Shares 5 Urgent Steps for Coldcard Users Facing Losses
+LATEST NEWS
+CLARITY Act Faces New Senate Threat Over Trump Crypto Gains
+33 minutes ago
+Bitgo CEO Funds 100 BTC Wallet, Dares Anthropic's AI to Steal It
+1 hour ago
+Samson Mow Shares 5 Urgent Steps for Coldcard Users Facing Losses
 LATEST PODCASTS
 Franklin Templeton: The $Trillion Tokenization Opportunity Explained
 32:16Aug 01, 2026
 Has crypto finally reached the end of its bear market?
 47:57Jul 31, 2026
+3 hours ago
+Bitcoin Miners Face August Showdown After Revenue Rebound
+4 hours ago
+Coldcard Hacker Gets Brazen Bitcoin Laundering Offer Onchain
+5 hours ago
+3 hours ago
+Bitcoin Miners Face August Showdown After Revenue Rebound
+4 hours ago
+Coldcard Hacker Gets Brazen Bitcoin Laundering Offer Onchain
+5 hours ago
 Why Fidelity Says Institutions Are Finally Buying Bitcoin
 35:29Jul 28, 2026
 The Great Capitulation: Mallers Exits, Satsuma Liquidates & BitMEX Shuts Down
@@ -154,7 +154,7 @@
 Gauntlet CEO Tarun Chitra on DeFi Risk, RWAs, Japan & Crypto’s AI Moment
 46:01Jul 24, 2026
 Related articles
-1 day ago
+2 days ago
 Was AI Responsible for Finding the Coldcard Security Flaw?
 Featured
 1 day ago
