--- reddit-88m-hack-not-over@20260804T093459Z
+++ reddit-88m-hack-not-over@20260804T134450Z
@@ -402,3 +402,29 @@
 edited: false
 body:
 It went down a bit but is picking up again.
+
+comment: p1mhgkn
+parent: t1_p1g10xq
+author: CiaranCarroll
+created_utc: 1785840017
+edited: false
+body:
+Who would pay them for the compute? Would they be guaranteed a payout from the original owners? Wouldn't that look a little bit like extortion?
+
+comment: p1n1sey
+parent: t1_p1mhgkn
+author: dj_destroyer
+created_utc: 1785847395
+edited: false
+body:
+Ask the white hat hackers -- there's already a ton of them. A lot of people already have the compute and have no problem donating the electricity. All in the name of being a good person. Furthermore, if they made their seeds without entropy (dice rolls) of at least like 25 dice rolls then it doesn't really take that much power/energy to crack them. 
+
+These white hats also probably have sizeable BTC holdings if I had to guess. Drop in the bucket in the grand scheme. 
+
+comment: p1n4ioz
+parent: t1_p1n1sey
+author: CiaranCarroll
+created_utc: 1785848239
+edited: false
+body:
+Fair enough
