--- reddit-safely-store-btc-without-hww@20260809T104106Z
+++ reddit-safely-store-btc-without-hww@20260813T125740Z
@@ -354,17 +354,11 @@
 
 comment: p2ifvlb
 parent: t1_p2gbkv6
-author: MiguelLancaster
+author: [deleted]
 created_utc: 1786216154
-edited: false
-body:
-not robust against crushing
-
-those little letter pieces fall out and you've lost your seed
-
-trash device, overpriced
-
-you can get a whole stack of stainless steel 'credit card' blanks and an engraving pen for like 20 bucks on amazon and have 10 seed backups for like a tenth of the price of that gimmicky thing
+edited: 1786604878
+body:
+[deleted]
 
 comment: p2jh966
 parent: t1_p2ifvlb
