--- reddit-safely-store-btc-without-hww@20260809T020514Z
+++ reddit-safely-store-btc-without-hww@20260809T104106Z
@@ -19,17 +19,11 @@
 
 comment: p2e9vcv
 parent: t3_1vil2qs
-author: Ok-Call-5864
+author: [deleted]
 created_utc: 1786159904
 edited: false
 body:
-DONT REACT TO DM‘s. They‘ll try to scam you.  
-TAKE YOUR TIME!  
-DONOT ENTER YOUR SEEDPHRASE ON ANY ONLINE DEVICE
-
-My 2 sats: i think you still need a HW wallet. In my opinion, the risks by messing  everything up by yourself is way higher than „trusting“ a decent HW company. Make sure to have a good passphrase and you‘ll do just fine :)
-
-Good luck
+[deleted]
 
 comment: p2eahts
 parent: t3_1vil2qs
