--- kloaec-fork-scam-warning@20260815T000513Z
+++ kloaec-fork-scam-warning@20260815T130941Z
@@ -116,15 +116,6 @@
 body:
 as long as it's not on a Coldcard, all good :D
 
-post: 2085494871574605950
-role: reply
-author: Ultra_Mega_BTC
-name: BigDutch
-created: 2026-08-06T22:34:58Z
-media: 0
-body:
-I’m gonna coldcard and chill. 
-
 post: 2085495013299925430
 role: reply
 author: KLoaec
@@ -146,15 +137,6 @@
 
 During this fork, should those that are migrating coins due to the CC  have anything to worry about when transacting on-chain to new setups ?
 
-post: 2085500652273213718
-role: reply
-author: eINstigator
-name: Iain S
-created: 2026-08-06T22:57:56Z
-media: 0
-body:
-The timing says more about the BIP110 crowd than anything.  How tone deaf are they.
-
 post: 2085502464086647018
 role: reply
 author: mrwaldenwoods
@@ -276,15 +258,6 @@
 body:
 so you're saying to tell them to "fork-off"
 
-post: 2085518127257850256
-role: reply
-author: pudgypengweng
-name: Pengweng.sol
-created: 2026-08-07T00:07:23Z
-media: 0
-body:
-I’m team do nothing, just hodling my coldcard
-
 post: 2085520989127889268
 role: reply
 author: AshmanAFP
@@ -293,15 +266,6 @@
 media: 0
 body:
 Why or how was it safe to sell bcash but not this time?
-
-post: 2085520994958225702
-role: reply
-author: lifeonbeta1224
-name: lifeonbeta
-created: 2026-08-07T00:18:46Z
-media: 0
-body:
-Just do nothing with this then. Sit tight.
 
 post: 2085523305088635178
 role: reply
