--- reddit-vendor-counterfactual@20260807T061342Z
+++ reddit-vendor-counterfactual@20260808T015559Z
@@ -336,3 +336,15 @@
 edited: false
 body:
 By now, I am no longer excluding the ‘retirement attack’ scenario. 😢
+
+comment: p2crga2
+parent: t3_1vduxyg
+author: _gianlucag_
+created_utc: 1786141225
+edited: 1786141414
+body:
+The only viable option would have been to "steal" all the btc by themselves, effectively whitehat-ing their way out, then returning the bitcoins to the original owners.
+
+Or
+
+go partially "closed source" by removing the github account, dont disclose the issue, ask people, one by one, to recreate the seed and move their funds there
