--- reddit-vendor-counterfactual@20260804T191137Z
+++ reddit-vendor-counterfactual@20260806T162955Z
@@ -108,7 +108,7 @@
 parent: t1_p1ccc66
 author: Charming-Designer944
 created_utc: 1785717372
-edited: false
+edited: 1786024058
 body:
 It's another issue entirely, but with a very similar result.
 
@@ -117,7 +117,7 @@
  - no minimum requirement on the number of dice rolls
  - easy to mix up the options of mixing dice rolls into the intenal entropy vs building the entropy entirely from the dice rolls alone.
 
-Combined it can result in users selecting the option of only usinng the dice rolls, and then only perform a handful number of dice roolls thinking that the dice rolls are just additional security to the internal entropy, not replacing it entirely and not realize that they created a vere weak seed.
+Combined it can result in users selecting the option of only usinng the dice rolls, and then only perform a handful number of dice roolls thinking that the dice rolls are just additional security to the internal entropy, not replacing it entirely and not realize that they created a very weak seed.
 
 I believe this issue is partially fixed in later firmwares, presenting a warning in main seed generation when using weak dice rolls, but still present when dice rolling a temporary seed.
 
@@ -320,3 +320,11 @@
 edited: false
 body:
 I would guess we are beyond that stage at this point, unfortunately. The awareness of the attack is widespread and sweeps are still occurring which means if there were a white hat phase, it would be concluded by now and the remainder we are seeing is black hat, or at least mixed with black hat.
+
+comment: p2279bk
+parent: t1_p1c3w9z
+author: Charming-Designer944
+created_utc: 1786024345
+edited: false
+body:
+Its game over as soon there is a whisper about there being an issue with seed generation entropy. It does not take long to find the weakness when knowing there is one.
