--- reddit-dice-seed-generation@20260806T163537Z
+++ reddit-dice-seed-generation@20260808T020143Z
@@ -751,3 +751,11 @@
 Actually the opposite is true. Casino dice, if not carefully chosen, may be intentionally made to be biased for some very obvious purposes. In the future if a hacker know that some one is using these kind of casino dicss to generate the seed phrase that will also greatly reduce the entropy of seed phrase. Casino dices may not even be as good as a regular dices.
 
 "Always being unpredictable" is the way to go. If you are gonna use dice, go ahead, but don't tell others that you use which brand of dice.
+
+comment: p2dqket
+parent: t1_p1bniq6
+author: WeekendQuant
+created_utc: 1786152914
+edited: false
+body:
+Huh? $30 for two dice is nothing compared to what I lose in $ each year to the banking system.
