--- reddit-dice-seed-generation@20260804T214818Z
+++ reddit-dice-seed-generation@20260805T142751Z
@@ -733,3 +733,21 @@
 edited: false
 body:
 There are 2048 bip39 words so 1d32+1d64 would be most efficient of all.
+
+comment: p1tkw0r
+parent: t1_p1ani3y
+author: ibrewcoffeeonthemoon
+created_utc: 1785921646
+edited: false
+body:
+If you have this level of luck, you should just start picking random 24 words and you will land on someone's wallet very soon. 😁
+
+comment: p1tp9x9
+parent: t1_p1aqkwi
+author: ibrewcoffeeonthemoon
+created_utc: 1785923730
+edited: false
+body:
+Actually the opposite is true. Casino dice, if not carefully chosen, may be intentionally made to be biased for some very obvious purposes. In the future if a hacker know that some one is using these kind of casino dicss to generate the seed phrase that will also greatly reduce the entropy of seed phrase. Casino dices may not even be as good as a regular dices.
+
+"Always being unpredictable" is the way to go. If you are gonna use dice, go ahead, but don't tell others that you use which brand of dice.
