--- reddit-dice-seed-generation@20260805T142751Z
+++ reddit-dice-seed-generation@20260806T163537Z
@@ -268,11 +268,11 @@
 
 comment: p1b7u44
 parent: t3_1vdl4h3
-author: cleankiwii
+author: [deleted]
 created_utc: 1785699164
 edited: false
 body:
-i want to do it, how do i convert the 99 1-6 ranged numbers into words? btw why is 99 instead of 100 for 24 words since 12 require 50
+[deleted]
 
 comment: p1b7u5v
 parent: t1_p19uydy
@@ -633,11 +633,11 @@
 
 comment: p1fk0rq
 parent: t1_p1fiob4
-author: cleankiwii
+author: [deleted]
 created_utc: 1785758752
 edited: false
 body:
-it doesn’t seem right, how can up have a 4 digit numbers using only 4 dice rolls? 50/12=4,167 little more than 4 rolls for each word?
+[deleted]
 
 comment: p1flzta
 parent: t3_1vdl4h3
