--- cryptonews-build-error-38m@20260802T230331Z
+++ cryptonews-build-error-38m@20260802T233241Z
@@ -33,180 +33,180 @@
 Search
 Close search
 Bitcoin (BTC)
-$63,377.00
-0.95156
+$63,446.00
+1.16639
 Bitcoin price
 Ethereum (ETH)
-$1,880.66
-1.87242
+$1,881.40
+2.12297
 Ethereum price
 XRP (XRP)
-$1.09
-2.42767
+$1.08
+2.30075
 XRP price
 BNB (BNB)
-$587.65
-2.14013
+$587.52
+2.13802
 BNB price
 Solana (SOL)
-$73.66
-2.49199
+$73.51
+2.40932
 Solana price
 Hyperliquid (HYPE)
-$52.55
-0.87468
+$52.60
+1.05158
 Hyperliquid price
 Cardano (ADA)
-$0.190259
-9.6804
+$0.188931
+8.74026
 Cardano price
 Chainlink (LINK)
-$8.39
-4.00622
+$8.36
+3.87618
 Chainlink price
 POL (ex-MATIC) (POL)
-$0.072741
-1.15026
+$0.072733
+1.69475
 POL (ex-MATIC) price
 Gram (prev. Toncoin) (GRAM)
-$1.43
-2.66625
+$1.42
+2.24404
 Gram (prev. Toncoin) price
 Asteroid Shiba (ASTEROID)
 $0.0000669
-2.8287
+2.97746
 Asteroid Shiba price
 Bitcoin (BTC)
-$63,377.00
-0.95156
+$63,446.00
+1.16639
 Bitcoin price
 Ethereum (ETH)
-$1,880.66
-1.87242
+$1,881.40
+2.12297
 Ethereum price
 XRP (XRP)
-$1.09
-2.42767
+$1.08
+2.30075
 XRP price
 BNB (BNB)
-$587.65
-2.14013
+$587.52
+2.13802
 BNB price
 Solana (SOL)
-$73.66
-2.49199
+$73.51
+2.40932
 Solana price
 Hyperliquid (HYPE)
-$52.55
-0.87468
+$52.60
+1.05158
 Hyperliquid price
 Cardano (ADA)
-$0.190259
-9.6804
+$0.188931
+8.74026
 Cardano price
 Chainlink (LINK)
-$8.39
-4.00622
+$8.36
+3.87618
 Chainlink price
 POL (ex-MATIC) (POL)
-$0.072741
-1.15026
+$0.072733
+1.69475
 POL (ex-MATIC) price
 Gram (prev. Toncoin) (GRAM)
-$1.43
-2.66625
+$1.42
+2.24404
 Gram (prev. Toncoin) price
 Asteroid Shiba (ASTEROID)
 $0.0000669
-2.8287
+2.97746
 Asteroid Shiba price
 Bitcoin (BTC)
-$63,377.00
-0.95156
+$63,446.00
+1.16639
 Bitcoin price
 Ethereum (ETH)
-$1,880.66
-1.87242
+$1,881.40
+2.12297
 Ethereum price
 XRP (XRP)
-$1.09
-2.42767
+$1.08
+2.30075
 XRP price
 BNB (BNB)
-$587.65
-2.14013
+$587.52
+2.13802
 BNB price
 Solana (SOL)
-$73.66
-2.49199
+$73.51
+2.40932
 Solana price
 Hyperliquid (HYPE)
-$52.55
-0.87468
+$52.60
+1.05158
 Hyperliquid price
 Cardano (ADA)
-$0.190259
-9.6804
+$0.188931
+8.74026
 Cardano price
 Chainlink (LINK)
-$8.39
-4.00622
+$8.36
+3.87618
 Chainlink price
 POL (ex-MATIC) (POL)
-$0.072741
-1.15026
+$0.072733
+1.69475
 POL (ex-MATIC) price
 Gram (prev. Toncoin) (GRAM)
-$1.43
-2.66625
+$1.42
+2.24404
 Gram (prev. Toncoin) price
 Asteroid Shiba (ASTEROID)
 $0.0000669
-2.8287
+2.97746
 Asteroid Shiba price
 Bitcoin (BTC)
-$63,377.00
-0.95156
+$63,446.00
+1.16639
 Bitcoin price
 Ethereum (ETH)
-$1,880.66
-1.87242
+$1,881.40
+2.12297
 Ethereum price
 XRP (XRP)
-$1.09
-2.42767
+$1.08
+2.30075
 XRP price
 BNB (BNB)
-$587.65
-2.14013
+$587.52
+2.13802
 BNB price
 Solana (SOL)
-$73.66
-2.49199
+$73.51
+2.40932
 Solana price
 Hyperliquid (HYPE)
-$52.55
-0.87468
+$52.60
+1.05158
 Hyperliquid price
 Cardano (ADA)
-$0.190259
-9.6804
+$0.188931
+8.74026
 Cardano price
 Chainlink (LINK)
-$8.39
-4.00622
+$8.36
+3.87618
 Chainlink price
 POL (ex-MATIC) (POL)
-$0.072741
-1.15026
+$0.072733
+1.69475
 POL (ex-MATIC) price
 Gram (prev. Toncoin) (GRAM)
-$1.43
-2.66625
+$1.42
+2.24404
 Gram (prev. Toncoin) price
 Asteroid Shiba (ASTEROID)
 $0.0000669
-2.8287
+2.97746
 Asteroid Shiba price
 Home → Feature → A build error in Coldcard’s firmware drained $38 million in bitcoin in 25 minutes
 A build error in Coldcard’s firmware drained $38 million in bitcoin in 25 minutes
@@ -362,7 +362,7 @@
 Trump Media transfers 2,628 BTC as holdings shrink
 BTC
 Bitcoin
-10 hours ago
+11 hours ago
 Sign up for crypto news and market insights
 Get crypto market analysis and curated news delivered right to your inbox every week.
 Email
