--- cryptonews-build-error-38m@20260802T220319Z
+++ cryptonews-build-error-38m@20260802T223230Z
@@ -33,180 +33,180 @@
 Search
 Close search
 Bitcoin (BTC)
-$63,624.00
-1.37446
+$63,611.00
+1.33617
 Bitcoin price
 Ethereum (ETH)
-$1,893.52
-2.73182
+$1,890.97
+2.49102
 Ethereum price
 XRP (XRP)
 $1.09
-2.6104
+2.84808
 XRP price
 BNB (BNB)
-$589.61
-2.4996
+$589.13
+2.37591
 BNB price
 Solana (SOL)
-$74.03
-3.1864
+$74.02
+2.98069
 Solana price
 Hyperliquid (HYPE)
-$52.91
-1.36559
+$52.85
+1.39985
 Hyperliquid price
 Cardano (ADA)
-$0.189513
-9.02569
+$0.189501
+8.54985
 Cardano price
 Chainlink (LINK)
-$8.38
-4.07659
+$8.40
+4.10211
 Chainlink price
 POL (ex-MATIC) (POL)
-$0.073024
-1.43714
+$0.072881
+1.2979
 POL (ex-MATIC) price
 Gram (prev. Toncoin) (GRAM)
 $1.43
-2.29989
+2.52184
 Gram (prev. Toncoin) price
 Asteroid Shiba (ASTEROID)
-$0.0000667
--0.89771
+$0.0000679
+3.28861
 Asteroid Shiba price
 Bitcoin (BTC)
-$63,624.00
-1.37446
+$63,611.00
+1.33617
 Bitcoin price
 Ethereum (ETH)
-$1,893.52
-2.73182
+$1,890.97
+2.49102
 Ethereum price
 XRP (XRP)
 $1.09
-2.6104
+2.84808
 XRP price
 BNB (BNB)
-$589.61
-2.4996
+$589.13
+2.37591
 BNB price
 Solana (SOL)
-$74.03
-3.1864
+$74.02
+2.98069
 Solana price
 Hyperliquid (HYPE)
-$52.91
-1.36559
+$52.85
+1.39985
 Hyperliquid price
 Cardano (ADA)
-$0.189513
-9.02569
+$0.189501
+8.54985
 Cardano price
 Chainlink (LINK)
-$8.38
-4.07659
+$8.40
+4.10211
 Chainlink price
 POL (ex-MATIC) (POL)
-$0.073024
-1.43714
+$0.072881
+1.2979
 POL (ex-MATIC) price
 Gram (prev. Toncoin) (GRAM)
 $1.43
-2.29989
+2.52184
 Gram (prev. Toncoin) price
 Asteroid Shiba (ASTEROID)
-$0.0000667
--0.89771
+$0.0000679
+3.28861
 Asteroid Shiba price
 Bitcoin (BTC)
-$63,624.00
-1.37446
+$63,611.00
+1.33617
 Bitcoin price
 Ethereum (ETH)
-$1,893.52
-2.73182
+$1,890.97
+2.49102
 Ethereum price
 XRP (XRP)
 $1.09
-2.6104
+2.84808
 XRP price
 BNB (BNB)
-$589.61
-2.4996
+$589.13
+2.37591
 BNB price
 Solana (SOL)
-$74.03
-3.1864
+$74.02
+2.98069
 Solana price
 Hyperliquid (HYPE)
-$52.91
-1.36559
+$52.85
+1.39985
 Hyperliquid price
 Cardano (ADA)
-$0.189513
-9.02569
+$0.189501
+8.54985
 Cardano price
 Chainlink (LINK)
-$8.38
-4.07659
+$8.40
+4.10211
 Chainlink price
 POL (ex-MATIC) (POL)
-$0.073024
-1.43714
+$0.072881
+1.2979
 POL (ex-MATIC) price
 Gram (prev. Toncoin) (GRAM)
 $1.43
-2.29989
+2.52184
 Gram (prev. Toncoin) price
 Asteroid Shiba (ASTEROID)
-$0.0000667
--0.89771
+$0.0000679
+3.28861
 Asteroid Shiba price
 Bitcoin (BTC)
-$63,624.00
-1.37446
+$63,611.00
+1.33617
 Bitcoin price
 Ethereum (ETH)
-$1,893.52
-2.73182
+$1,890.97
+2.49102
 Ethereum price
 XRP (XRP)
 $1.09
-2.6104
+2.84808
 XRP price
 BNB (BNB)
-$589.61
-2.4996
+$589.13
+2.37591
 BNB price
 Solana (SOL)
-$74.03
-3.1864
+$74.02
+2.98069
 Solana price
 Hyperliquid (HYPE)
-$52.91
-1.36559
+$52.85
+1.39985
 Hyperliquid price
 Cardano (ADA)
-$0.189513
-9.02569
+$0.189501
+8.54985
 Cardano price
 Chainlink (LINK)
-$8.38
-4.07659
+$8.40
+4.10211
 Chainlink price
 POL (ex-MATIC) (POL)
-$0.073024
-1.43714
+$0.072881
+1.2979
 POL (ex-MATIC) price
 Gram (prev. Toncoin) (GRAM)
 $1.43
-2.29989
+2.52184
 Gram (prev. Toncoin) price
 Asteroid Shiba (ASTEROID)
-$0.0000667
--0.89771
+$0.0000679
+3.28861
 Asteroid Shiba price
 Home → Feature → A build error in Coldcard’s firmware drained $38 million in bitcoin in 25 minutes
 A build error in Coldcard’s firmware drained $38 million in bitcoin in 25 minutes
@@ -362,7 +362,7 @@
 Trump Media transfers 2,628 BTC as holdings shrink
 BTC
 Bitcoin
-9 hours ago
+10 hours ago
 Sign up for crypto news and market insights
 Get crypto market analysis and curated news delivered right to your inbox every week.
 Email
