--- cryptonews-build-error-38m@20260802T233241Z
+++ cryptonews-build-error-38m@20260803T000341Z
@@ -33,180 +33,180 @@
 Search
 Close search
 Bitcoin (BTC)
-$63,446.00
-1.16639
+$63,453.00
+1.10217
 Bitcoin price
 Ethereum (ETH)
-$1,881.40
-2.12297
+$1,881.96
+2.09956
 Ethereum price
 XRP (XRP)
 $1.08
-2.30075
+2.31557
 XRP price
 BNB (BNB)
-$587.52
-2.13802
+$587.51
+2.22587
 BNB price
 Solana (SOL)
-$73.51
-2.40932
+$73.43
+2.17027
 Solana price
 Hyperliquid (HYPE)
-$52.60
-1.05158
+$52.52
+0.76354
 Hyperliquid price
 Cardano (ADA)
-$0.188931
-8.74026
+$0.189296
+8.62245
 Cardano price
 Chainlink (LINK)
-$8.36
-3.87618
+$8.37
+3.87817
 Chainlink price
 POL (ex-MATIC) (POL)
-$0.072733
-1.69475
+$0.072877
+1.77399
 POL (ex-MATIC) price
 Gram (prev. Toncoin) (GRAM)
 $1.42
-2.24404
+2.31873
 Gram (prev. Toncoin) price
 Asteroid Shiba (ASTEROID)
-$0.0000669
-2.97746
+$0.000067
+2.70136
 Asteroid Shiba price
 Bitcoin (BTC)
-$63,446.00
-1.16639
+$63,453.00
+1.10217
 Bitcoin price
 Ethereum (ETH)
-$1,881.40
-2.12297
+$1,881.96
+2.09956
 Ethereum price
 XRP (XRP)
 $1.08
-2.30075
+2.31557
 XRP price
 BNB (BNB)
-$587.52
-2.13802
+$587.51
+2.22587
 BNB price
 Solana (SOL)
-$73.51
-2.40932
+$73.43
+2.17027
 Solana price
 Hyperliquid (HYPE)
-$52.60
-1.05158
+$52.52
+0.76354
 Hyperliquid price
 Cardano (ADA)
-$0.188931
-8.74026
+$0.189296
+8.62245
 Cardano price
 Chainlink (LINK)
-$8.36
-3.87618
+$8.37
+3.87817
 Chainlink price
 POL (ex-MATIC) (POL)
-$0.072733
-1.69475
+$0.072877
+1.77399
 POL (ex-MATIC) price
 Gram (prev. Toncoin) (GRAM)
 $1.42
-2.24404
+2.31873
 Gram (prev. Toncoin) price
 Asteroid Shiba (ASTEROID)
-$0.0000669
-2.97746
+$0.000067
+2.70136
 Asteroid Shiba price
 Bitcoin (BTC)
-$63,446.00
-1.16639
+$63,453.00
+1.10217
 Bitcoin price
 Ethereum (ETH)
-$1,881.40
-2.12297
+$1,881.96
+2.09956
 Ethereum price
 XRP (XRP)
 $1.08
-2.30075
+2.31557
 XRP price
 BNB (BNB)
-$587.52
-2.13802
+$587.51
+2.22587
 BNB price
 Solana (SOL)
-$73.51
-2.40932
+$73.43
+2.17027
 Solana price
 Hyperliquid (HYPE)
-$52.60
-1.05158
+$52.52
+0.76354
 Hyperliquid price
 Cardano (ADA)
-$0.188931
-8.74026
+$0.189296
+8.62245
 Cardano price
 Chainlink (LINK)
-$8.36
-3.87618
+$8.37
+3.87817
 Chainlink price
 POL (ex-MATIC) (POL)
-$0.072733
-1.69475
+$0.072877
+1.77399
 POL (ex-MATIC) price
 Gram (prev. Toncoin) (GRAM)
 $1.42
-2.24404
+2.31873
 Gram (prev. Toncoin) price
 Asteroid Shiba (ASTEROID)
-$0.0000669
-2.97746
+$0.000067
+2.70136
 Asteroid Shiba price
 Bitcoin (BTC)
-$63,446.00
-1.16639
+$63,453.00
+1.10217
 Bitcoin price
 Ethereum (ETH)
-$1,881.40
-2.12297
+$1,881.96
+2.09956
 Ethereum price
 XRP (XRP)
 $1.08
-2.30075
+2.31557
 XRP price
 BNB (BNB)
-$587.52
-2.13802
+$587.51
+2.22587
 BNB price
 Solana (SOL)
-$73.51
-2.40932
+$73.43
+2.17027
 Solana price
 Hyperliquid (HYPE)
-$52.60
-1.05158
+$52.52
+0.76354
 Hyperliquid price
 Cardano (ADA)
-$0.188931
-8.74026
+$0.189296
+8.62245
 Cardano price
 Chainlink (LINK)
-$8.36
-3.87618
+$8.37
+3.87817
 Chainlink price
 POL (ex-MATIC) (POL)
-$0.072733
-1.69475
+$0.072877
+1.77399
 POL (ex-MATIC) price
 Gram (prev. Toncoin) (GRAM)
 $1.42
-2.24404
+2.31873
 Gram (prev. Toncoin) price
 Asteroid Shiba (ASTEROID)
-$0.0000669
-2.97746
+$0.000067
+2.70136
 Asteroid Shiba price
 Home → Feature → A build error in Coldcard’s firmware drained $38 million in bitcoin in 25 minutes
 A build error in Coldcard’s firmware drained $38 million in bitcoin in 25 minutes
@@ -350,13 +350,13 @@
 News
 Read more - Coldcard users face urgent seed migration warning
 Coldcard users face urgent seed migration warning
-6 hours ago
+7 hours ago
 News
 Read more - Bitcoin ETFs gain fresh case after $89M Coldcard drain, says Balchunas
 Bitcoin ETFs gain fresh case after $89M Coldcard drain, says Balchunas
 BTC
 Bitcoin
-7 hours ago
+8 hours ago
 News
 Read more - Trump Media transfers 2,628 BTC as holdings shrink
 Trump Media transfers 2,628 BTC as holdings shrink
