--- cryptonews-build-error-38m@20260803T013302Z
+++ cryptonews-build-error-38m@20260803T020401Z
@@ -33,180 +33,180 @@
 Search
 Close search
 Bitcoin (BTC)
-$63,148.00
-0.36559
+$63,107.00
+0.26262
 Bitcoin price
 Ethereum (ETH)
-$1,869.47
-0.89403
+$1,864.35
+0.47492
 Ethereum price
 XRP (XRP)
-$1.08
-0.8342
+$1.07
+0.78893
 XRP price
 BNB (BNB)
-$584.07
-1.44947
+$583.46
+1.11477
 BNB price
 Solana (SOL)
-$72.97
-0.74559
+$72.91
+0.50076
 Solana price
 Hyperliquid (HYPE)
-$52.41
--0.03281
+$52.52
+0.31452
 Hyperliquid price
 Cardano (ADA)
-$0.185038
-5.01718
+$0.185346
+5.21538
 Cardano price
 Chainlink (LINK)
-$8.35
-2.43886
+$8.32
+2.12598
 Chainlink price
 POL (ex-MATIC) (POL)
-$0.07276
-1.14321
+$0.072673
+0.89036
 POL (ex-MATIC) price
 Gram (prev. Toncoin) (GRAM)
-$1.42
-1.502
+$1.41
+1.07092
 Gram (prev. Toncoin) price
 Asteroid Shiba (ASTEROID)
-$0.0000655
--1.82855
+$0.0000643
+-2.52702
 Asteroid Shiba price
 Bitcoin (BTC)
-$63,148.00
-0.36559
+$63,107.00
+0.26262
 Bitcoin price
 Ethereum (ETH)
-$1,869.47
-0.89403
+$1,864.35
+0.47492
 Ethereum price
 XRP (XRP)
-$1.08
-0.8342
+$1.07
+0.78893
 XRP price
 BNB (BNB)
-$584.07
-1.44947
+$583.46
+1.11477
 BNB price
 Solana (SOL)
-$72.97
-0.74559
+$72.91
+0.50076
 Solana price
 Hyperliquid (HYPE)
-$52.41
--0.03281
+$52.52
+0.31452
 Hyperliquid price
 Cardano (ADA)
-$0.185038
-5.01718
+$0.185346
+5.21538
 Cardano price
 Chainlink (LINK)
-$8.35
-2.43886
+$8.32
+2.12598
 Chainlink price
 POL (ex-MATIC) (POL)
-$0.07276
-1.14321
+$0.072673
+0.89036
 POL (ex-MATIC) price
 Gram (prev. Toncoin) (GRAM)
-$1.42
-1.502
+$1.41
+1.07092
 Gram (prev. Toncoin) price
 Asteroid Shiba (ASTEROID)
-$0.0000655
--1.82855
+$0.0000643
+-2.52702
 Asteroid Shiba price
 Bitcoin (BTC)
-$63,148.00
-0.36559
+$63,107.00
+0.26262
 Bitcoin price
 Ethereum (ETH)
-$1,869.47
-0.89403
+$1,864.35
+0.47492
 Ethereum price
 XRP (XRP)
-$1.08
-0.8342
+$1.07
+0.78893
 XRP price
 BNB (BNB)
-$584.07
-1.44947
+$583.46
+1.11477
 BNB price
 Solana (SOL)
-$72.97
-0.74559
+$72.91
+0.50076
 Solana price
 Hyperliquid (HYPE)
-$52.41
--0.03281
+$52.52
+0.31452
 Hyperliquid price
 Cardano (ADA)
-$0.185038
-5.01718
+$0.185346
+5.21538
 Cardano price
 Chainlink (LINK)
-$8.35
-2.43886
+$8.32
+2.12598
 Chainlink price
 POL (ex-MATIC) (POL)
-$0.07276
-1.14321
+$0.072673
+0.89036
 POL (ex-MATIC) price
 Gram (prev. Toncoin) (GRAM)
-$1.42
-1.502
+$1.41
+1.07092
 Gram (prev. Toncoin) price
 Asteroid Shiba (ASTEROID)
-$0.0000655
--1.82855
+$0.0000643
+-2.52702
 Asteroid Shiba price
 Bitcoin (BTC)
-$63,148.00
-0.36559
+$63,107.00
+0.26262
 Bitcoin price
 Ethereum (ETH)
-$1,869.47
-0.89403
+$1,864.35
+0.47492
 Ethereum price
 XRP (XRP)
-$1.08
-0.8342
+$1.07
+0.78893
 XRP price
 BNB (BNB)
-$584.07
-1.44947
+$583.46
+1.11477
 BNB price
 Solana (SOL)
-$72.97
-0.74559
+$72.91
+0.50076
 Solana price
 Hyperliquid (HYPE)
-$52.41
--0.03281
+$52.52
+0.31452
 Hyperliquid price
 Cardano (ADA)
-$0.185038
-5.01718
+$0.185346
+5.21538
 Cardano price
 Chainlink (LINK)
-$8.35
-2.43886
+$8.32
+2.12598
 Chainlink price
 POL (ex-MATIC) (POL)
-$0.07276
-1.14321
+$0.072673
+0.89036
 POL (ex-MATIC) price
 Gram (prev. Toncoin) (GRAM)
-$1.42
-1.502
+$1.41
+1.07092
 Gram (prev. Toncoin) price
 Asteroid Shiba (ASTEROID)
-$0.0000655
--1.82855
+$0.0000643
+-2.52702
 Asteroid Shiba price
 Home → Feature → A build error in Coldcard’s firmware drained $38 million in bitcoin in 25 minutes
 A build error in Coldcard’s firmware drained $38 million in bitcoin in 25 minutes
@@ -350,7 +350,7 @@
 News
 Read more - Coldcard users face urgent seed migration warning
 Coldcard users face urgent seed migration warning
-8 hours ago
+9 hours ago
 News
 Read more - Bitcoin ETFs gain fresh case after $89M Coldcard drain, says Balchunas
 Bitcoin ETFs gain fresh case after $89M Coldcard drain, says Balchunas
