--- cryptonews-build-error-38m@20260803T023427Z
+++ cryptonews-build-error-38m@20260803T030612Z
@@ -33,180 +33,180 @@
 Search
 Close search
 Bitcoin (BTC)
-$63,065.00
--0.49359
+$63,071.00
+-0.46895
 Bitcoin price
 Ethereum (ETH)
-$1,864.32
--0.50214
+$1,862.29
+-0.63995
 Ethereum price
 XRP (XRP)
 $1.07
-0.15681
+-0.16455
 XRP price
 BNB (BNB)
-$584.57
-0.80385
+$584.95
+0.79863
 BNB price
 Solana (SOL)
-$72.94
--0.05901
+$72.95
+-0.14498
 Solana price
 Hyperliquid (HYPE)
-$52.29
--0.93823
+$52.31
+-1.09487
 Hyperliquid price
 Cardano (ADA)
-$0.186231
-3.95235
+$0.186728
+3.67719
 Cardano price
 Chainlink (LINK)
-$8.30
-0.73869
+$8.29
+0.30423
 Chainlink price
 POL (ex-MATIC) (POL)
-$0.072658
-0.61081
+$0.072706
+0.61893
 POL (ex-MATIC) price
 Gram (prev. Toncoin) (GRAM)
 $1.41
-0.61199
+0.51653
 Gram (prev. Toncoin) price
 Asteroid Shiba (ASTEROID)
-$0.0000649
--3.44703
+$0.0000645
+-3.59712
 Asteroid Shiba price
 Bitcoin (BTC)
-$63,065.00
--0.49359
+$63,071.00
+-0.46895
 Bitcoin price
 Ethereum (ETH)
-$1,864.32
--0.50214
+$1,862.29
+-0.63995
 Ethereum price
 XRP (XRP)
 $1.07
-0.15681
+-0.16455
 XRP price
 BNB (BNB)
-$584.57
-0.80385
+$584.95
+0.79863
 BNB price
 Solana (SOL)
-$72.94
--0.05901
+$72.95
+-0.14498
 Solana price
 Hyperliquid (HYPE)
-$52.29
--0.93823
+$52.31
+-1.09487
 Hyperliquid price
 Cardano (ADA)
-$0.186231
-3.95235
+$0.186728
+3.67719
 Cardano price
 Chainlink (LINK)
-$8.30
-0.73869
+$8.29
+0.30423
 Chainlink price
 POL (ex-MATIC) (POL)
-$0.072658
-0.61081
+$0.072706
+0.61893
 POL (ex-MATIC) price
 Gram (prev. Toncoin) (GRAM)
 $1.41
-0.61199
+0.51653
 Gram (prev. Toncoin) price
 Asteroid Shiba (ASTEROID)
-$0.0000649
--3.44703
+$0.0000645
+-3.59712
 Asteroid Shiba price
 Bitcoin (BTC)
-$63,065.00
--0.49359
+$63,071.00
+-0.46895
 Bitcoin price
 Ethereum (ETH)
-$1,864.32
--0.50214
+$1,862.29
+-0.63995
 Ethereum price
 XRP (XRP)
 $1.07
-0.15681
+-0.16455
 XRP price
 BNB (BNB)
-$584.57
-0.80385
+$584.95
+0.79863
 BNB price
 Solana (SOL)
-$72.94
--0.05901
+$72.95
+-0.14498
 Solana price
 Hyperliquid (HYPE)
-$52.29
--0.93823
+$52.31
+-1.09487
 Hyperliquid price
 Cardano (ADA)
-$0.186231
-3.95235
+$0.186728
+3.67719
 Cardano price
 Chainlink (LINK)
-$8.30
-0.73869
+$8.29
+0.30423
 Chainlink price
 POL (ex-MATIC) (POL)
-$0.072658
-0.61081
+$0.072706
+0.61893
 POL (ex-MATIC) price
 Gram (prev. Toncoin) (GRAM)
 $1.41
-0.61199
+0.51653
 Gram (prev. Toncoin) price
 Asteroid Shiba (ASTEROID)
-$0.0000649
--3.44703
+$0.0000645
+-3.59712
 Asteroid Shiba price
 Bitcoin (BTC)
-$63,065.00
--0.49359
+$63,071.00
+-0.46895
 Bitcoin price
 Ethereum (ETH)
-$1,864.32
--0.50214
+$1,862.29
+-0.63995
 Ethereum price
 XRP (XRP)
 $1.07
-0.15681
+-0.16455
 XRP price
 BNB (BNB)
-$584.57
-0.80385
+$584.95
+0.79863
 BNB price
 Solana (SOL)
-$72.94
--0.05901
+$72.95
+-0.14498
 Solana price
 Hyperliquid (HYPE)
-$52.29
--0.93823
+$52.31
+-1.09487
 Hyperliquid price
 Cardano (ADA)
-$0.186231
-3.95235
+$0.186728
+3.67719
 Cardano price
 Chainlink (LINK)
-$8.30
-0.73869
+$8.29
+0.30423
 Chainlink price
 POL (ex-MATIC) (POL)
-$0.072658
-0.61081
+$0.072706
+0.61893
 POL (ex-MATIC) price
 Gram (prev. Toncoin) (GRAM)
 $1.41
-0.61199
+0.51653
 Gram (prev. Toncoin) price
 Asteroid Shiba (ASTEROID)
-$0.0000649
--3.44703
+$0.0000645
+-3.59712
 Asteroid Shiba price
 Home → Feature → A build error in Coldcard’s firmware drained $38 million in bitcoin in 25 minutes
 A build error in Coldcard’s firmware drained $38 million in bitcoin in 25 minutes
@@ -350,7 +350,7 @@
 News
 Read more - Coldcard users face urgent seed migration warning
 Coldcard users face urgent seed migration warning
-9 hours ago
+10 hours ago
 News
 Read more - Bitcoin ETFs gain fresh case after $89M Coldcard drain, says Balchunas
 Bitcoin ETFs gain fresh case after $89M Coldcard drain, says Balchunas
