--- cryptonews-build-error-38m@20260802T213220Z
+++ cryptonews-build-error-38m@20260802T220319Z
@@ -33,180 +33,180 @@
 Search
 Close search
 Bitcoin (BTC)
-$63,449.00
-1.1956
+$63,624.00
+1.37446
 Bitcoin price
 Ethereum (ETH)
-$1,883.63
-2.40447
+$1,893.52
+2.73182
 Ethereum price
 XRP (XRP)
-$1.08
-2.32457
+$1.09
+2.6104
 XRP price
 BNB (BNB)
-$588.96
-2.41423
+$589.61
+2.4996
 BNB price
 Solana (SOL)
-$73.63
-2.81269
+$74.03
+3.1864
 Solana price
 Hyperliquid (HYPE)
-$52.53
-0.81723
+$52.91
+1.36559
 Hyperliquid price
 Cardano (ADA)
-$0.188201
-8.356
+$0.189513
+9.02569
 Cardano price
 Chainlink (LINK)
-$8.35
-3.83992
+$8.38
+4.07659
 Chainlink price
 POL (ex-MATIC) (POL)
-$0.072957
-1.55788
+$0.073024
+1.43714
 POL (ex-MATIC) price
 Gram (prev. Toncoin) (GRAM)
-$1.42
-2.20011
+$1.43
+2.29989
 Gram (prev. Toncoin) price
 Asteroid Shiba (ASTEROID)
-$0.0000672
--1.11009
+$0.0000667
+-0.89771
 Asteroid Shiba price
 Bitcoin (BTC)
-$63,449.00
-1.1956
+$63,624.00
+1.37446
 Bitcoin price
 Ethereum (ETH)
-$1,883.63
-2.40447
+$1,893.52
+2.73182
 Ethereum price
 XRP (XRP)
-$1.08
-2.32457
+$1.09
+2.6104
 XRP price
 BNB (BNB)
-$588.96
-2.41423
+$589.61
+2.4996
 BNB price
 Solana (SOL)
-$73.63
-2.81269
+$74.03
+3.1864
 Solana price
 Hyperliquid (HYPE)
-$52.53
-0.81723
+$52.91
+1.36559
 Hyperliquid price
 Cardano (ADA)
-$0.188201
-8.356
+$0.189513
+9.02569
 Cardano price
 Chainlink (LINK)
-$8.35
-3.83992
+$8.38
+4.07659
 Chainlink price
 POL (ex-MATIC) (POL)
-$0.072957
-1.55788
+$0.073024
+1.43714
 POL (ex-MATIC) price
 Gram (prev. Toncoin) (GRAM)
-$1.42
-2.20011
+$1.43
+2.29989
 Gram (prev. Toncoin) price
 Asteroid Shiba (ASTEROID)
-$0.0000672
--1.11009
+$0.0000667
+-0.89771
 Asteroid Shiba price
 Bitcoin (BTC)
-$63,449.00
-1.1956
+$63,624.00
+1.37446
 Bitcoin price
 Ethereum (ETH)
-$1,883.63
-2.40447
+$1,893.52
+2.73182
 Ethereum price
 XRP (XRP)
-$1.08
-2.32457
+$1.09
+2.6104
 XRP price
 BNB (BNB)
-$588.96
-2.41423
+$589.61
+2.4996
 BNB price
 Solana (SOL)
-$73.63
-2.81269
+$74.03
+3.1864
 Solana price
 Hyperliquid (HYPE)
-$52.53
-0.81723
+$52.91
+1.36559
 Hyperliquid price
 Cardano (ADA)
-$0.188201
-8.356
+$0.189513
+9.02569
 Cardano price
 Chainlink (LINK)
-$8.35
-3.83992
+$8.38
+4.07659
 Chainlink price
 POL (ex-MATIC) (POL)
-$0.072957
-1.55788
+$0.073024
+1.43714
 POL (ex-MATIC) price
 Gram (prev. Toncoin) (GRAM)
-$1.42
-2.20011
+$1.43
+2.29989
 Gram (prev. Toncoin) price
 Asteroid Shiba (ASTEROID)
-$0.0000672
--1.11009
+$0.0000667
+-0.89771
 Asteroid Shiba price
 Bitcoin (BTC)
-$63,449.00
-1.1956
+$63,624.00
+1.37446
 Bitcoin price
 Ethereum (ETH)
-$1,883.63
-2.40447
+$1,893.52
+2.73182
 Ethereum price
 XRP (XRP)
-$1.08
-2.32457
+$1.09
+2.6104
 XRP price
 BNB (BNB)
-$588.96
-2.41423
+$589.61
+2.4996
 BNB price
 Solana (SOL)
-$73.63
-2.81269
+$74.03
+3.1864
 Solana price
 Hyperliquid (HYPE)
-$52.53
-0.81723
+$52.91
+1.36559
 Hyperliquid price
 Cardano (ADA)
-$0.188201
-8.356
+$0.189513
+9.02569
 Cardano price
 Chainlink (LINK)
-$8.35
-3.83992
+$8.38
+4.07659
 Chainlink price
 POL (ex-MATIC) (POL)
-$0.072957
-1.55788
+$0.073024
+1.43714
 POL (ex-MATIC) price
 Gram (prev. Toncoin) (GRAM)
-$1.42
-2.20011
+$1.43
+2.29989
 Gram (prev. Toncoin) price
 Asteroid Shiba (ASTEROID)
-$0.0000672
--1.11009
+$0.0000667
+-0.89771
 Asteroid Shiba price
 Home → Feature → A build error in Coldcard’s firmware drained $38 million in bitcoin in 25 minutes
 A build error in Coldcard’s firmware drained $38 million in bitcoin in 25 minutes
@@ -350,19 +350,19 @@
 News
 Read more - Coldcard users face urgent seed migration warning
 Coldcard users face urgent seed migration warning
-4 hours ago
+5 hours ago
 News
 Read more - Bitcoin ETFs gain fresh case after $89M Coldcard drain, says Balchunas
 Bitcoin ETFs gain fresh case after $89M Coldcard drain, says Balchunas
 BTC
 Bitcoin
-5 hours ago
+6 hours ago
 News
 Read more - Trump Media transfers 2,628 BTC as holdings shrink
 Trump Media transfers 2,628 BTC as holdings shrink
 BTC
 Bitcoin
-8 hours ago
+9 hours ago
 Sign up for crypto news and market insights
 Get crypto market analysis and curated news delivered right to your inbox every week.
 Email
