--- reddit-opreturn-laundering-offer@20260804T134552Z
+++ reddit-opreturn-laundering-offer@20260807T193034Z
@@ -297,3 +297,35 @@
 edited: false
 body:
 Dude type likes he did meth and never left his mom’s basement in St. Petersburg 
+
+comment: p29nbu9
+parent: t1_p1fspma
+author: CompetitionDouble420
+created_utc: 1786111266
+edited: false
+body:
+Reef.... ....er
+
+comment: p29nt3w
+parent: t1_p1f1h3d
+author: CompetitionDouble420
+created_utc: 1786111399
+edited: 1786112230
+body:
+Enter the block height of the block you're interested in, in an advanced block explorer like tokenview or blockchair.
+
+comment: p29prfx
+parent: t1_p29nt3w
+author: RedditTooAddictive
+created_utc: 1786111937
+edited: false
+body:
+Thanks!
+
+comment: p29r9nz
+parent: t1_p29prfx
+author: CompetitionDouble420
+created_utc: 1786112347
+edited: false
+body:
+Gotchu bro!
