--- reddit-1400-btc-hacked-report@20260804T191449Z
+++ reddit-1400-btc-hacked-report@20260805T075702Z
@@ -1352,4 +1352,14 @@
 body:
 I mean, if Crowdstrike can take the world down and survive, surely a hacked wallet can?
 
+comment: p1sjkus
+parent: t1_p1fuqps
+author: FudgeAdventurous3875
+created_utc: 1785904196
+edited: false
+body:
+Seriously everyone is acting like dice rolls and passphrases are standard. 5 years ago it was common thought that using these advanced options increased user error and chance to lock yourself out, like the people that dice rolled one time and lost their funds because they didnt know what they were doing. 5 years ago there was VERY little instructions on how to even use ColdCard. CoinKites official setup video from 4 years ago doesnt even mention it and uses the generated seed phrase. 
+
+In 5 years people could be calling each other idiots for only using 50 dice rolls instead of 100 or only using 2/3 multisig instead of 3/5.
+
 more-stub: parent t1_p1isqg5 count <live-count>
