--- reddit-1400-btc-hacked-report@20260804T134401Z
+++ reddit-1400-btc-hacked-report@20260804T154602Z
@@ -1328,4 +1328,20 @@
 body:
 That's even worse... 
 
+comment: p1nzxz0
+parent: t3_1veauv2
+author: NakedNick_ballin
+created_utc: 1785856873
+edited: false
+body:
+And now people will learn how trash open source can be
+
+comment: p1o0heq
+parent: t1_p1gtrpq
+author: NakedNick_ballin
+created_utc: 1785857015
+edited: false
+body:
+Their code was not tested at all, and extremely poorly designed.  This wasn't a mistake that could happen to someone with proper care, and actually espousing the "strong safety" practices they were so loud about publicly.
+
 more-stub: parent t1_p1isqg5 count 0
