--- reddit-1400-btc-hacked-report@20260805T142526Z
+++ reddit-1400-btc-hacked-report@20260806T163258Z
@@ -29,11 +29,11 @@
 
 comment: p1fgmt9
 parent: t3_1veauv2
-author: IllllIIlIllIllllIlll
+author: [deleted]
 created_utc: 1785757519
-edited: false
-body:
-Damn, it's definitely my first time hearing about this one! How come nobody has ever made a post about it?
+edited: 1786028003
+body:
+[deleted]
 
 comment: p1fgnie
 parent: t3_1veauv2
@@ -242,11 +242,11 @@
 
 comment: p1fs6ok
 parent: t1_p1fl2sz
-author: IllllIIlIllIllllIlll
+author: [deleted]
 created_utc: 1785761480
-edited: false
-body:
-Nah, fuck the s. Obvious sarcasm is obvious.
+edited: 1786027967
+body:
+[deleted]
 
 comment: p1fsjqm
 parent: t1_p1fki7l
