--- reddit-1400-btc-hacked-report@20260805T075702Z
+++ reddit-1400-btc-hacked-report@20260805T142526Z
@@ -437,16 +437,6 @@
 body:
 Why are there no white hats stepping in??
 
-comment: p1g55ol
-parent: t1_p1fuqps
-author: OldHamburger7923
-created_utc: 1785765331
-edited: false
-body:
-Dumb. We all use passwords for every account we use, even the ones with 2fa. Even if it's just your mom's dob, that helps.
-
-I've always used a passphrase. But on reddit a ton of people kept saying it makes losing your coins easier. Group think can be wrong.
-
 comment: p1g5ugv
 parent: t1_p1fpybc
 author: Jonassydney
@@ -472,14 +462,6 @@
 edited: false
 body:
 Anyone with Claude code can find the vulnerability now that someone else found it and it's been published on ten thousand sites on the internet for Claude to copy/paste from.
-
-comment: p1g7lnq
-parent: t1_p1fgyng
-author: OldHamburger7923
-created_utc: 1785766010
-edited: false
-body:
-If it was one minor issue or bug, but this is a major flaw on a critical system and it was in place for 5 years while they made other random BS products.
 
 comment: p1g8bwb
 parent: t1_p1fuqps
@@ -828,17 +810,11 @@
 
 comment: p1hb9bc
 parent: t1_p1h4nwd
-author: yet-again-temporary
+author: [deleted]
 created_utc: 1785776266
 edited: false
 body:
->Yea, just because we want our money decentralized does not mean you can be a bad actor in said space.
-
-I mean it quite literally does lmao. That's the free market at work, if you don't like a company then vote with your wallet and stop financially supporting them.
-
-Calling for regulations every time there's a situation like this goes against the very principles that crypto was founded on, and defeats the entire point. No regulations means no regulations, good *and* bad.
-
-"Those who would give up essential Liberty, to purchase a little temporary Safety, ***deserve neither Liberty nor Safety***."
+[deleted]
 
 comment: p1hc260
 parent: t1_p1gtmpf
