--- reddit-1400-btc-hacked-report@20260804T093404Z
+++ reddit-1400-btc-hacked-report@20260804T134401Z
@@ -1304,4 +1304,28 @@
 body:
 Your governments. You know that thing that runs the county you are in. I don't want my government to have total control over a currency, however I don't want a lawless hell scape either. We can have a decentralized yet regulated asset.
 
+comment: p1mk0jj
+parent: t3_1veauv2
+author: calambacle
+created_utc: 1785841074
+edited: false
+body:
+Not your softwares not your shit
+
+comment: p1n7p0u
+parent: t1_p1hn4t5
+author: PracticalPianist6189
+created_utc: 1785849195
+edited: false
+body:
+It was just “runs”. What a shame.
+
+comment: p1nd73d
+parent: t1_p1n7p0u
+author: jmeador42
+created_utc: 1785850777
+edited: false
+body:
+That's even worse... 
+
 more-stub: parent t1_p1isqg5 count 0
