--- reddit-1400-btc-hacked-report@20260804T154602Z
+++ reddit-1400-btc-hacked-report@20260804T191449Z
@@ -1344,4 +1344,12 @@
 body:
 Their code was not tested at all, and extremely poorly designed.  This wasn't a mistake that could happen to someone with proper care, and actually espousing the "strong safety" practices they were so loud about publicly.
 
+comment: p1pf8li
+parent: t1_p1fgnie
+author: Fangletron
+created_utc: 1785869692
+edited: false
+body:
+I mean, if Crowdstrike can take the world down and survive, surely a hacked wallet can?
+
 more-stub: parent t1_p1isqg5 count 0
