--- reddit-cash-out-forensics-question@20260804T181501Z
+++ reddit-cash-out-forensics-question@20260804T194614Z
@@ -1615,3 +1615,15 @@
 edited: false
 body:
 Of a friend who knows a guy that knows a guy's cousin who met a guy that heard about....
+
+comment: p1po56m
+parent: t1_p1k1kjd
+author: Euphoric-Language695
+created_utc: 1785871995
+edited: false
+body:
+Are those systems automatic and also immediately flag any ither wallet sent from the hacker wallet? Because otherwise you're just a new wallet away from moving funds.
+
+If they are automatic, this could be weaponized by dumping funds into a bunch of wallets to disable them.
+
+The way I see it is this is an arms race and the hackers can lijely move faster than these exchanges. 
