--- reddit-cash-out-forensics-question@20260805T142554Z
+++ reddit-cash-out-forensics-question@20260805T210041Z
@@ -1739,6 +1739,14 @@
 body:
 Dumb
 
+comment: p1tauj3
+parent: t3_1vegndz
+author: Visible_Abroad9810
+created_utc: 1785916741
+edited: false
+body:
+Very easy you can buy expensive watches with the crypto or gold nowadays turning crypto into fiat is piss easy 
+
 comment: p1tiucp
 parent: t1_p1h1f0v
 author: PensionConscious6752
@@ -1754,3 +1762,19 @@
 edited: false
 body:
 yea i didn't claim it would be easy or safe for the people with that legitimate BTC.
+
+comment: p1wncqy
+parent: t1_p1hbm95
+author: m39583
+created_utc: 1785954357
+edited: false
+body:
+But how much cocaine does one person need?!
+
+comment: p1xdzc6
+parent: t1_p1kxub0
+author: Dear_Combination_805
+created_utc: 1785961198
+edited: false
+body:
+Nicely trying.
