--- reddit-cash-out-forensics-question@20260804T201642Z
+++ reddit-cash-out-forensics-question@20260804T204618Z
@@ -1635,3 +1635,13 @@
 edited: false
 body:
 I'm not sure, but it would not be difficult to code some software that would automatically track the transfer from wallet to wallet. Transfers are not even close to being instantaneous either by virtue of how block chains work so its possible  for such software to stay ahead. Will they do it? I have no idea. But it's possible. 
+
+comment: p1pytk1
+parent: t1_p1pxg35
+author: Euphoric-Language695
+created_utc: 1785874813
+edited: false
+body:
+Yes, but if it came out publicly you could now use this to force account flags in exchanges by transferring funds to them. If the funds are useless anyway, might as well cause havoc. Could also use all the confusion tot ry to get some percentage of funds cleared somehow.
+
+My main point is that the assumption that because the wallet with the funds is well known it's easy to mitigate further misuse is at best wishful thinking. 
