--- reddit-cash-out-forensics-question@20260804T221646Z
+++ reddit-cash-out-forensics-question@20260804T232229Z
@@ -1364,6 +1364,21 @@
 
 
 
+comment: p1kkapi
+parent: t1_p1jwcws
+author: Numerous-Annual-721
+created_utc: 1785809526
+edited: 1785809970
+body:
+not true. they'd have to prove you still have the keys to the coins.  
+you cannot be sentenced to jail indefinitely for theft, but you can be held in jail for contempt of court / non compliance with an order to return the stolen property.  
+but that part is limited based on your ability to actually return said property.  
+if you don't have the keys, they can't hold you.  
+so far the record for being held in such contempt is 14 years [https://en.wikipedia.org/wiki/H.\_Beatty\_Chadwick](https://en.wikipedia.org/wiki/H._Beatty_Chadwick)
+
+edit: bottom line, they'd have to prove quite a lot.  
+the theft part that affected lots of people is the serious one in terms of sentence because you stole from 1000 people so you can get thousands of years stacked in prison. but theft has the shortest statute of limitations. just a few years. if you don't do anything stupid until that statute passes, the next thing would be money laundering (as you haven't actually hacked any computer system, but maybe there's something there too). but money laundering would possibly be a single offense and we'd be pretty far away from the original crime. but the money laundering offense is only relevant to the money that has been laundered, and that would presumably be in the form of cash/gold/cars/house, which they'd take. any coins you still have have not been laundered...
+
 comment: p1kppkn
 parent: t1_p1ilaoq
 author: anonymous-12358
