--- reddit-cash-out-forensics-question@20260804T232229Z
+++ reddit-cash-out-forensics-question@20260805T001708Z
@@ -1688,3 +1688,21 @@
 5. Profit
 
 
+
+comment: p1r6bql
+parent: t1_p1my2i5
+author: RetiredAvocado
+created_utc: 1785887305
+edited: false
+body:
+Fro stealing a literal car example people get more than just equivalent value to pay back. There are lawyer's fees to get a more comfortable sentence deal, probation, community service hours, many serve jail terms. If you steal someone's car or bitcoin, you should go to jail, as a stronger discouragement than just giving back what you stole when caught.
+
+comment: p1r93mh
+parent: t1_p1ksmg8
+author: RetiredAvocado
+created_utc: 1785888196
+edited: false
+body:
+There's a difference between taking unknown gold vs taking one that everyone can see was there and then was taken. Original owner would have a hard time proving it was theirs, nor would even notice it missing for a long long time. On bitcoin blockchain it's highly visible.
+
+But yeah, you generally have to turn in big findings into authorities and maybe seek a finder's fee. Those are the rules where I live though. In other societies with billions of people that type of regulation/law may not be a factor in their decision making.
