--- reddit-cash-out-forensics-question@20260805T075726Z
+++ reddit-cash-out-forensics-question@20260805T142554Z
@@ -1738,3 +1738,19 @@
 edited: false
 body:
 Dumb
+
+comment: p1tiucp
+parent: t1_p1h1f0v
+author: PensionConscious6752
+created_utc: 1785920644
+edited: false
+body:
+i could be wrong but wouldnt that take cooperations of large numbers of people holding legitimate btc to do that?
+
+comment: p1tzk9u
+parent: t1_p1tiucp
+author: skr_replicator
+created_utc: 1785928104
+edited: false
+body:
+yea i didn't claim it would be easy or safe for the people with that legitimate BTC.
