--- reddit-cash-out-forensics-question@20260804T194614Z
+++ reddit-cash-out-forensics-question@20260804T201642Z
@@ -1627,3 +1627,11 @@
 If they are automatic, this could be weaponized by dumping funds into a bunch of wallets to disable them.
 
 The way I see it is this is an arms race and the hackers can lijely move faster than these exchanges. 
+
+comment: p1pxg35
+parent: t1_p1po56m
+author: scrandlle
+created_utc: 1785874452
+edited: false
+body:
+I'm not sure, but it would not be difficult to code some software that would automatically track the transfer from wallet to wallet. Transfers are not even close to being instantaneous either by virtue of how block chains work so its possible  for such software to stay ahead. Will they do it? I have no idea. But it's possible. 
