--- reddit-cash-out-forensics-question@20260807T061703Z
+++ reddit-cash-out-forensics-question@20260808T150252Z
@@ -1804,3 +1804,11 @@
 edited: false
 body:
 Whomever did this was not an ordinary scammer from India or Nigeria. I m sure they know how to be unnoticed. 
+
+comment: p2g5bqf
+parent: t1_p1gwnlg
+author: bravedave109
+created_utc: 1786191370
+edited: false
+body:
+How do you like dem apples
