--- reddit-cash-out-forensics-question@20260804T204618Z
+++ reddit-cash-out-forensics-question@20260804T221646Z
@@ -1645,3 +1645,31 @@
 Yes, but if it came out publicly you could now use this to force account flags in exchanges by transferring funds to them. If the funds are useless anyway, might as well cause havoc. Could also use all the confusion tot ry to get some percentage of funds cleared somehow.
 
 My main point is that the assumption that because the wallet with the funds is well known it's easy to mitigate further misuse is at best wishful thinking. 
+
+comment: p1qiabm
+parent: t3_1vegndz
+author: _gianlucag_
+created_utc: 1785880071
+edited: false
+body:
+Fee rebating. 
+
+By colluding with a miner, you could prepare a transaction of few sats but a humongous fee, let's say a fee of 1 btc. The miner gets the fee and the usual block reward, and then it will send you a transaction of 1 btc back. You now have 1 perfectly clean btc. Problem is, you must trust the miner to give you back the 1 btc, or lets say 0.7 btc, depending on the agreement.
+
+comment: p1qjj8j
+parent: t3_1vegndz
+author: choppadonmiss
+created_utc: 1785880421
+edited: false
+body:
+1. Find a foreign government that opposes the US
+
+2. Offer bitcoin for a steep discount
+
+3. Get threatened by the foreign government
+
+4. Give up bitcoin for nothing in exchange
+
+5. Profit
+
+
