--- reddit-cash-out-forensics-question@20260805T001708Z
+++ reddit-cash-out-forensics-question@20260805T075726Z
@@ -1706,3 +1706,35 @@
 There's a difference between taking unknown gold vs taking one that everyone can see was there and then was taken. Original owner would have a hard time proving it was theirs, nor would even notice it missing for a long long time. On bitcoin blockchain it's highly visible.
 
 But yeah, you generally have to turn in big findings into authorities and maybe seek a finder's fee. Those are the rules where I live though. In other societies with billions of people that type of regulation/law may not be a factor in their decision making.
+
+comment: p1rv47r
+parent: t1_p1oar4q
+author: saltyfoot73
+created_utc: 1785895341
+edited: false
+body:
+yeah I thought that was the case but why doesn't whichever branch of government has the keys transfer it back to bitfinex if someone steals your car and then the police get it back do the police just keep your car i realize not a great analogy because insurance but you know what I am saying right 
+
+comment: p1rxglm
+parent: t1_p1krumf
+author: JoeMillersHat
+created_utc: 1785896120
+edited: false
+body:
+because every transaction lives on the chain forever
+
+comment: p1sivtz
+parent: t1_p1jm32c
+author: Remarkable_Gap_3418
+created_utc: 1785903911
+edited: false
+body:
+Just go to SEA country and pay some local a few weeks of meals worth of money to go withdraw on your behalf. He never even needs to see your face. However yes, it's incredibly inefficient and not feasible in this scenario. It would take multiple lifetimes to withdraw 100+mil this way lol.
+
+comment: p1ssdoo
+parent: t1_p1jkjq8
+author: blackrack
+created_utc: 1785908006
+edited: false
+body:
+Dumb
