--- reddit-cash-out-forensics-question@20260804T144424Z
+++ reddit-cash-out-forensics-question@20260804T161644Z
@@ -1591,3 +1591,11 @@
 edited: false
 body:
 You misunderstand me. By sending to just a few wallets it's possible to see who the victims are. If they sent to many wallets, there would be no way to see that. I agree that tracking isn't affected - but only if you can see what to track.
+
+comment: p1oar4q
+parent: t1_p1ksqzg
+author: 00MacDonald
+created_utc: 1785859629
+edited: false
+body:
+Coins up until now have not been returned to bitfinex. They made their customer base good because it was only around 50M USD at the time, after BTC dumped because of the hack. They leveraged assets to pay the customers back, and BTC increase on it's own healed their accounts pretty quickly.
