--- reddit-cash-out-forensics-question@20260804T161644Z
+++ reddit-cash-out-forensics-question@20260804T181501Z
@@ -1599,3 +1599,19 @@
 edited: false
 body:
 Coins up until now have not been returned to bitfinex. They made their customer base good because it was only around 50M USD at the time, after BTC dumped because of the hack. They leveraged assets to pay the customers back, and BTC increase on it's own healed their accounts pretty quickly.
+
+comment: p1p1y0y
+parent: t1_p1jhjhn
+author: Soulists_Shadow
+created_utc: 1785866378
+edited: false
+body:
+You could turn satoshi into the bad guy by doing that. A if he is still alive, he didnt return three coins. You were holding onto stolen property. 
+
+comment: p1p2so3
+parent: t1_p1hr7gd
+author: Healthy_Committee888
+created_utc: 1785866591
+edited: false
+body:
+Of a friend who knows a guy that knows a guy's cousin who met a guy that heard about....
