--- reddit-unused-coldcard-collectible@20260804T101414Z
+++ reddit-unused-coldcard-collectible@20260804T104256Z
@@ -761,3 +761,11 @@
 edited: false
 body:
 Talk about dodging bullets lol
+
+comment: p1mgz6u
+parent: t1_p1gg5is
+author: FavorableMadness
+created_utc: 1785839811
+edited: false
+body:
+I am with you. I don’t understand why this event makes the device less good than any other. Yes they had an issue, they may have more. It’s naive to think that all the other devices are perfect. Has anyone looked to see how often a Microsoft Windows security update has to be released?! Do we think there are no vulnerabilities in banking software?!
