--- reddit-unused-coldcard-collectible@20260805T080619Z
+++ reddit-unused-coldcard-collectible@20260805T211055Z
@@ -803,3 +803,19 @@
 Well, the whole point is to just trust the code of the firmware, not the company. The code is all that matters when it comes to hardware wallets. If the firmware is good, I dont care how the company is behaving. 
 
 Problem is, very few people can inspect the patch by themselves and check the fix before upgrading the coldcard.
+
+comment: p1wuozb
+parent: t3_1vecxwh
+author: Big-Cheetah5159
+created_utc: 1785956188
+edited: false
+body:
+Sell it too me
+
+comment: p1wxd1i
+parent: t3_1vecxwh
+author: Big-Cheetah5159
+created_utc: 1785956868
+edited: false
+body:
+Use dice rolls and verify online that seeds match. This tells you if the ColdCard is generating the right mathematical results. Once confirmed, it’s safe to use your dice rolls to create your actual seed. 
