--- reddit-unused-coldcard-collectible@20260806T101358Z
+++ reddit-unused-coldcard-collectible@20260806T164223Z
@@ -129,14 +129,6 @@
 body:
 Well the good news is that with the latest firmware it’s using a valid entropy source for the seed.
 
-comment: p1fzkr8
-parent: t1_p1fvpmq
-author: IllllIIlIllIllllIlll
-created_utc: 1785763733
-edited: false
-body:
-Rng, not encryption
-
 comment: p1fzkyz
 parent: t3_1vecxwh
 author: OkWave5583
@@ -835,3 +827,11 @@
 edited: false
 body:
 Ah yes, I also disassemble my car and inspect every single nut and bolt for defects before driving it. Makes sense.
+
+comment: p21svnp
+parent: t3_1vecxwh
+author: No_Feed1906
+created_utc: 1786020203
+edited: false
+body:
+It says it right there on the package.
