--- reddit-unused-coldcard-collectible@20260804T172434Z
+++ reddit-unused-coldcard-collectible@20260805T015609Z
@@ -692,14 +692,6 @@
 
 The inverse of this is they say to not trust, verify. What’s it say about the end user who didn’t over the years. If it’s lazy to not catch it it’s even lazier to not have listened to the people you trusted up until a few days ago. 
 
-comment: p1kaopy
-parent: t1_p1gg5is
-author: MCHappster1
-created_utc: 1785806329
-edited: false
-body:
-I agree. I won't recommend CC again. I won't buy from Coinkite again. I'll keep using my existing CC since I trust my high entropy PP and 100+ dice rolls. I will however move to multi-sig ASAP.
-
 comment: p1kvzut
 parent: t1_p1hwjnk
 author: ModtheMods420
