--- reddit-unused-coldcard-collectible@20260804T114314Z
+++ reddit-unused-coldcard-collectible@20260804T172434Z
@@ -785,3 +785,11 @@
 edited: false
 body:
 same as me, never heard, went to website, really liked the style and models
+
+comment: p1opio7
+parent: t3_1vecxwh
+author: LV_HiLife
+created_utc: 1785863327
+edited: false
+body:
+when I was doing research on cold wallets Mk3 cold card caught my attention because I saw it on a Btc magazine but I went with Trezor because the mk3 cold card just sounded complex for a beginner
