--- reddit-unused-coldcard-collectible@20260805T015609Z
+++ reddit-unused-coldcard-collectible@20260805T080619Z
@@ -785,3 +785,21 @@
 edited: false
 body:
 when I was doing research on cold wallets Mk3 cold card caught my attention because I saw it on a Btc magazine but I went with Trezor because the mk3 cold card just sounded complex for a beginner
+
+comment: p1t0ngw
+parent: t3_1vecxwh
+author: Few-Marsupial7659
+created_utc: 1785911832
+edited: false
+body:
+Jsut open it, upgrade to the latest firmware, dice roll your new seed if you want and you'll be fine. 
+
+comment: p1t70kh
+parent: t1_p1g702h
+author: _gianlucag_
+created_utc: 1785914877
+edited: false
+body:
+Well, the whole point is to just trust the code of the firmware, not the company. The code is all that matters when it comes to hardware wallets. If the firmware is good, I dont care how the company is behaving. 
+
+Problem is, very few people can inspect the patch by themselves and check the fix before upgrading the coldcard.
