--- stackernews-retirement-attack-question@20260804T204826Z
+++ stackernews-retirement-attack-question@20260804T234906Z
@@ -25,6 +25,13 @@
             }
           },
           {
+            "createdAt": "2026-08-04T23:29:21.360Z",
+            "text": "The coins were not mixed and tumbled like the usual suspect hacker operations do, but consolidated onchain, so my guess would be strategic Bitcoin reserve.",
+            "user": {
+              "name": "teemupleb"
+            }
+          },
+          {
             "createdAt": "2026-08-04T19:01:34.183Z",
             "text": "Inside job is what I have been thinking too.",
             "user": {
