--- stackernews-retirement-attack-question@20260805T075931Z
+++ stackernews-retirement-attack-question@20260806T100352Z
@@ -3,6 +3,13 @@
     "item": {
       "comments": {
         "comments": [
+          {
+            "createdAt": "2026-08-04T23:29:21.360Z",
+            "text": "The coins were not mixed and tumbled like the usual suspect hacker operations do, but consolidated onchain, so my guess would be strategic Bitcoin reserve.",
+            "user": {
+              "name": "teemupleb"
+            }
+          },
           {
             "createdAt": "2026-08-04T14:13:52.190Z",
             "text": "Biggest argument against it is they should also be sophisticated enough to know there's no great way to wash those coins, might be the most toxic coins ever with customers all over the globe and therefore law enforcement risk anywhere you go. \n\nThe business destroyed and potential legal costs likely outweigh whatever small amounts that could be dripped out at a time. \n\nAround the time of that post he had a hair across his ass for Foundation, who had only recently pivoted from shitcoin scamming. Understandable. The persona fits someone who believed in what they were selling more than just marketing, but was ultimately just too busy huffing their own farts.",
@@ -39,13 +46,6 @@
             }
           },
           {
-            "createdAt": "2026-08-04T23:29:21.360Z",
-            "text": "The coins were not mixed and tumbled like the usual suspect hacker operations do, but consolidated onchain, so my guess would be strategic Bitcoin reserve.",
-            "user": {
-              "name": "teemupleb"
-            }
-          },
-          {
             "createdAt": "2026-08-04T19:01:34.183Z",
             "text": "Inside job is what I have been thinking too.",
             "user": {
