--- stackernews-retirement-attack-question@20260804T234906Z
+++ stackernews-retirement-attack-question@20260805T004916Z
@@ -3,6 +3,13 @@
     "item": {
       "comments": {
         "comments": [
+          {
+            "createdAt": "2026-08-04T20:35:42.952Z",
+            "text": "Update: I am now on team Retirement Attack:\n\n![](https://m.stacker.news/150841)\n\n![](https://m.stacker.news/150842)\n@justin_shocknet",
+            "user": {
+              "name": "UncleJim21"
+            }
+          },
           {
             "createdAt": "2026-08-04T14:13:52.190Z",
             "text": "Biggest argument against it is they should also be sophisticated enough to know there's no great way to wash those coins, might be the most toxic coins ever with customers all over the globe and therefore law enforcement risk anywhere you go. \n\nThe business destroyed and potential legal costs likely outweigh whatever small amounts that could be dripped out at a time. \n\nAround the time of that post he had a hair across his ass for Foundation, who had only recently pivoted from shitcoin scamming. Understandable. The persona fits someone who believed in what they were selling more than just marketing, but was ultimately just too busy huffing their own farts.",
@@ -11,17 +18,17 @@
             }
           },
           {
-            "createdAt": "2026-08-04T20:35:42.952Z",
-            "text": "Update: I am now on team Retirement Attack:\n\n![](https://m.stacker.news/150841)\n\n![](https://m.stacker.news/150842)\n@justin_shocknet",
-            "user": {
-              "name": "UncleJim21"
-            }
-          },
-          {
             "createdAt": "2026-08-04T14:15:11.446Z",
             "text": "Adding their response, or lack thereof, to those reports from years ago does paint a pretty damning picture.",
             "user": {
               "name": "Undisciplined"
+            }
+          },
+          {
+            "createdAt": "2026-08-05T00:20:33.193Z",
+            "text": "I lean into a joke which didn't end well.\nAlso people are angry and want a scapegoat. Combine the two you get this.\n\nI was happy at first when I saw the coldcard bug because I wouldn't hear about nvk in the near future. But now when I see the scale of the disaster I would not like to be in his shoes, psychologically it must be tough.",
+            "user": {
+              "name": "john_doe"
             }
           },
           {
