--- reddit-64-btc-mixing-discussion@20260807T125922Z
+++ reddit-64-btc-mixing-discussion@20260807T193252Z
@@ -3044,6 +3044,46 @@
 
 They certainly didn't sweat. Very few animals have ever evolved sweat glands.
 
+comment: p2a50jq
+parent: t1_p253wbc
+author: kawfeeman68
+created_utc: 1786115998
+edited: false
+body:
+Around $3.8M isn't too bad?
+
+comment: p2aybt0
+parent: t1_p27wkcg
+author: RedshiftOTF
+created_utc: 1786123508
+edited: false
+body:
+Dinosaurs didn't pee, neither do birds or reptiles. When mammals and reptiles diverged, mammals did the peeing and reptiles did not. Check it out. :)
+
+comment: p2azeog
+parent: t1_p1zl295
+author: nerpish-inkdots
+created_utc: 1786123784
+edited: false
+body:
+Because banks are private. If the tax man or the bank man is who you want to keep away from your money, then it’s crime. Or conspiracy.
+
+comment: p2b9xm0
+parent: t3_1vg1rlc
+author: Intrepid_Interest912
+created_utc: 1786126484
+edited: false
+body:
+All your doing is lying fuck off 
+
+comment: p2bfngu
+parent: t1_p1xe8m7
+author: kennymac6969
+created_utc: 1786127955
+edited: false
+body:
+More US dollars are used for criminal activity than bitcoin. 
+
 more-stub: parent t1_p20rqux count <live-count>
 
 more-stub: parent t1_p1uultr count <live-count>
