--- reddit-64-btc-mixing-discussion@20260808T020312Z
+++ reddit-64-btc-mixing-discussion@20260808T083539Z
@@ -695,11 +695,11 @@
 
 comment: p1tsgzl
 parent: t1_p1tr1ol
-author: idekwutp
+author: [deleted]
 created_utc: 1785925165
 edited: false
 body:
-2010-2024, the new kids are called Gen beta now as well
+[deleted]
 
 comment: p1tskwr
 parent: t1_p1tr1ol
@@ -1608,11 +1608,11 @@
 
 comment: p1ukq15
 parent: t1_p1tskwr
-author: idekwutp
+author: [deleted]
 created_utc: 1785935262
 edited: false
 body:
-I had a Gen alpha kid I was babysitting last week explain to me in detail what munting and felching is
+[deleted]
 
 comment: p1ul12c
 parent: t1_p1u8von
@@ -2116,14 +2116,6 @@
 
 Why would they choose to take payments in dollar/euro/whatever bill if they might accidentally get radioactive bills?
 
-comment: p1vdvj1
-parent: t1_p1v2ww6
-author: idekwutp
-created_utc: 1785943124
-edited: false
-body:
-If the crew is your kids please do not 😭
-
 comment: p1vfr56
 parent: t3_1vg1rlc
 author: f08g
@@ -3100,6 +3092,14 @@
 body:
 Cold wallet stories
 
+comment: p2f9r35
+parent: t1_p1uaqwm
+author: New_Atmosphere_2315
+created_utc: 1786176285
+edited: false
+body:
+With any luck, he’s got the grail already 🥂
+
 more-stub: parent t1_p20rqux count <live-count>
 
 more-stub: parent t1_p1uultr count <live-count>
