--- reddit-64-btc-mixing-discussion@20260808T083539Z
+++ reddit-64-btc-mixing-discussion@20260808T213622Z
@@ -1468,11 +1468,11 @@
 
 comment: p1ug5ji
 parent: t3_1vg1rlc
-author: Just_Bluebird_5268
+author: [deleted]
 created_utc: 1785933886
 edited: false
 body:
-It's the future of money. The ultimate safe haven asset, hyperbitcoinization is just around the corner
+[removed]
 
 comment: p1ugacl
 parent: t1_p1u0uav
@@ -1489,16 +1489,6 @@
 edited: false
 body:
 Words nobody has ever said. 
-
-comment: p1ugxac
-parent: t3_1vg1rlc
-author: riseandride69
-created_utc: 1785934119
-edited: false
-body:
-Yeah, it was pretty naive to think “the hacker will never be able to use/cash out” 
-
-He will cash out allright.
 
 comment: p1uhgi4
 parent: t1_p1tlimh
@@ -2845,14 +2835,6 @@
 body:
 Wrong
 
-comment: p210qjd
-parent: t1_p1zxjd6
-author: Just_Bluebird_5268
-created_utc: 1786009398
-edited: false
-body:
-Transferring wealth from the gullible into the hands of the predatory? Absolutely it works!
-
 comment: p214fha
 parent: t1_p20mnr8
 author: Pirulax
