--- reddit-64-btc-mixing-discussion@20260807T193252Z
+++ reddit-64-btc-mixing-discussion@20260808T020312Z
@@ -3084,6 +3084,22 @@
 body:
 More US dollars are used for criminal activity than bitcoin. 
 
+comment: p2c1ffk
+parent: t1_p2azeog
+author: Some-btc-name
+created_utc: 1786133735
+edited: false
+body:
+It's not a crime to have money or transactions outside of the banking system. What a dumb take. 
+
+comment: p2dufx3
+parent: t3_1vg1rlc
+author: Upstairs_Ad_1998
+created_utc: 1786154268
+edited: false
+body:
+Cold wallet stories
+
 more-stub: parent t1_p20rqux count <live-count>
 
 more-stub: parent t1_p1uultr count <live-count>
