--- reddit-64-btc-mixing-discussion@20260809T040818Z
+++ reddit-64-btc-mixing-discussion@20260812T044016Z
@@ -3090,6 +3090,14 @@
 body:
 Laundering 
 
+comment: p2td6m1
+parent: t1_p1ti8u0
+author: Real-Bite9293
+created_utc: 1786361151
+edited: false
+body:
+I’m wondering. I had the feeling that nowadays agencies were able to track quite deeply into the blockchain and would be able to track millions of transactions if needed to find someone. Is it that just as easy to mix for one or two years and then slowly cash out?
+
 more-stub: parent t1_p20rqux count <live-count>
 
 more-stub: parent t1_p1uultr count <live-count>
