--- reddit-64-btc-mixing-discussion@20260806T050838Z
+++ reddit-64-btc-mixing-discussion@20260806T100824Z
@@ -2793,6 +2793,76 @@
 body:
 People did find it and flagged it and the ceo/cto ignored it. This got blasted for being ai writing earlier today but all the things in the timeline did actually happen so it’s not just slop https://x.com/inverse\_hanlon/status/2084689208627925384
 
+comment: p20c4bv
+parent: t1_p1zoe78
+author: Rabid_Mexican
+created_utc: 1785997283
+edited: 1785999210
+body:
+No sorry it's not - the source of the money remains illegitimate on paper whereas laundering would show it coming from a business and show the taxes being paid on it.
+
+Mixing can allow you to launder the money by then running it through a business - this is the laundering part.
+
+comment: p20ii6l
+parent: t1_p1y2pbl
+author: Ill-Party8305
+created_utc: 1786000314
+edited: false
+body:
+How do you track it then
+
+comment: p20j38q
+parent: t3_1vg1rlc
+author: GME_looooong
+created_utc: 1786000598
+edited: false
+body:
+Say I work for cold card and used this exploit. I would hit whales first. Wait for them to complain to me at customer service then I can put a hit on them (paid for with bitcoin how cute) and keep my activities on the down low. You guys don’t think like ceo’s. This mass attack was probably the swan song 
+
+comment: p20mnr8
+parent: t1_p1tur02
+author: crakked21
+created_utc: 1786002359
+edited: false
+body:
+But if I had good vbucks, and I got back stolen cold card vbucks, wouldn’t that be net negative for me since i got vbucks that have everyone looking at them?
+
+comment: p20rqux
+parent: t1_p20c4bv
+author: 00MacDonald
+created_utc: 1786004933
+edited: false
+body:
+I assume you are not familiar with the FATF definition of money laundering.
+
+Activity as defined by the previous commenter is 100% within the scope of FATFs definition 
+
+comment: p20yelz
+parent: t1_p1uv3fs
+author: Efriminiz
+created_utc: 1786008265
+edited: false
+body:
+Wrong
+
+comment: p20ygnx
+parent: t1_p1zoe78
+author: Efriminiz
+created_utc: 1786008291
+edited: false
+body:
+Wrong
+
+comment: p210qjd
+parent: t1_p1zxjd6
+author: Just_Bluebird_5268
+created_utc: 1786009398
+edited: false
+body:
+Transferring wealth from the gullible into the hands of the predatory? Absolutely it works!
+
+more-stub: parent t1_p20rqux count <live-count>
+
 more-stub: parent t1_p1uultr count <live-count>
 
 more-stub: parent t1_p1yhb18 count <live-count>
