--- bleepingcomputer-88m-report@20260803T030502Z
+++ bleepingcomputer-88m-report@20260803T081658Z
@@ -115,15 +115,15 @@
 Not a member yet? Register Now
 You may also like:
 Popular Stories
-Anthropic's Claude breached 3 orgs, uploaded PyPI malware during tests
 Arch Linux disables AUR package adoption to stop malware flood
-Microsoft Teams vishing attacks lead to Chaos ransomware attacks
+Hacker uses DeepSeek AI to autonomously attack vulnerable servers
+Amgen says cloud data breach exposed patient health, proprietary info
 Sponsor Posts
+Explore ESET’s H1 2026 Threat Report to uncover the latest AI, phishing, and ransomware trends.
+Overdue a password health-check? Audit your Active Directory for free
+Pixellot discovered and secured hundreds of unmanaged AI agent identities in weeks, not months. Download the case study for how.
+Uncover shadow AI apps, agents, and risky data sharing. Get started in 5 min.
 Stay one step ahead of new threats in the new year. Join Huntress for the monthly Tradecraft Tuesday.
-Explore ESET’s H1 2026 Threat Report to uncover the latest AI, phishing, and ransomware trends.
-Uncover shadow AI apps, agents, and risky data sharing. Get started in 5 min.
-Pixellot discovered and secured hundreds of unmanaged AI agent identities in weeks, not months. Download the case study for how.
-Overdue a password health-check? Audit your Active Directory for free
 Follow us:
 Main Sections
 News
