--- bleepingcomputer-88m-report@20260803T022643Z
+++ bleepingcomputer-88m-report@20260803T030502Z
@@ -119,11 +119,11 @@
 Arch Linux disables AUR package adoption to stop malware flood
 Microsoft Teams vishing attacks lead to Chaos ransomware attacks
 Sponsor Posts
+Stay one step ahead of new threats in the new year. Join Huntress for the monthly Tradecraft Tuesday.
+Explore ESET’s H1 2026 Threat Report to uncover the latest AI, phishing, and ransomware trends.
 Uncover shadow AI apps, agents, and risky data sharing. Get started in 5 min.
-Explore ESET’s H1 2026 Threat Report to uncover the latest AI, phishing, and ransomware trends.
+Pixellot discovered and secured hundreds of unmanaged AI agent identities in weeks, not months. Download the case study for how.
 Overdue a password health-check? Audit your Active Directory for free
-Pixellot discovered and secured hundreds of unmanaged AI agent identities in weeks, not months. Download the case study for how.
-Stay one step ahead of new threats in the new year. Join Huntress for the monthly Tradecraft Tuesday.
 Follow us:
 Main Sections
 News
