--- reddit-mk4-bricked-hotfix@20260806T095815Z
+++ reddit-mk4-bricked-hotfix@20260807T061425Z
@@ -640,3 +640,11 @@
 edited: false
 body:
 Oops forgot to do regression testing
+
+comment: p27i4b1
+parent: t1_p1q1psp
+author: nicolbolas10
+created_utc: 1786079926
+edited: false
+body:
+did this problem happen again? I updated my CC with the new firmware 5.6.0 last week, used it without any problems for 2 days. Opened it after 6 days and the device bricked(MK4). After trying to open it for 30 min, the device turned on and now it's working. However I think this bug can still happen again with the current firmware so not sure what to do.
