--- reddit-mk4-bricked-hotfix@20260807T125341Z
+++ reddit-mk4-bricked-hotfix@20260808T213008Z
@@ -672,3 +672,15 @@
 edited: false
 body:
 still using coldcard or thinking about moving your funds?
+
+comment: p2hekmj
+parent: t3_1vct6m2
+author: bje332013
+created_utc: 1786205567
+edited: false
+body:
+I tried to start a new thread in this subreddit asking about the firmware and whether people are getting 'BRICKED' messages as often as is suggested on X / Twitter. However, my thread was automatically blocked by Reddit's filters.
+
+"If you are running the new firmware and messing with passphrases, PLEASE BE CAREFUL."
+
+What exactly do you mean by "be careful"? It sounds like you did everything right, so I don't get what action(s) you're advising others against taking other than maybe avoiding the new firmware.
