--- reddit-mk4-bricked-hotfix@20260809T103219Z
+++ reddit-mk4-bricked-hotfix@20260809T164917Z
@@ -718,3 +718,21 @@
 edited: false
 body:
 That message was before the edit. After carching the why, the problem is not the passphrase, it’s the keyboard press. Coldcard uses HW RNG to randomize the scan order. So every press could potentially send it in a soft bricked state.
+
+comment: p2migmm
+parent: t1_p2mcn5g
+author: bje332013
+created_utc: 1786273333
+edited: false
+body:
+"the problem is not the passphrase, it’s the keyboard press. Coldcard uses HW RNG to randomize the scan order. So every press could potentially send it in a soft bricked state."
+
+Sorry, I don't understand your explanation. If the Coldcard device is working correctly (which it reportedly is not once the new firmware gets flashed), no sequence of valid keyboard/button presses should 'brick' the device.
+
+comment: p2mn4fd
+parent: t1_p2migmm
+author: lucapocchio
+created_utc: 1786275400
+edited: false
+body:
+U are 100% right, and this is not intended. In the edits i’ve added the urls of the PR’s to verify the fix. But still until when a new firmware doesn’t include it this can happen.
