--- reddit-mk4-bricked-hotfix@20260804T134142Z
+++ reddit-mk4-bricked-hotfix@20260804T201356Z
@@ -23,7 +23,16 @@
 
 ———-EDIT 2 DAYS LATER, BUG CONFIRMED ———-
 
-https://github.com/Coldcard/firmware/pull/692
+[https://github.com/Coldcard/firmware/pull/692](https://github.com/Coldcard/firmware/pull/692)
+
+
+
+———-EDIT 3 DAYS LATER, COLD CARD REPORT  ———-
+
+  
+[https://x.com/COLDCARDwallet/status/2084632863756955661](https://x.com/COLDCARDwallet/status/2084632863756955661)
+
+I will try to test in the described way and update here about results for everyone.
 
 comment: p13tui2
 parent: t3_1vct6m2
