--- reddit-mk4-bricked-hotfix@20260807T061425Z
+++ reddit-mk4-bricked-hotfix@20260807T125341Z
@@ -648,3 +648,27 @@
 edited: false
 body:
 did this problem happen again? I updated my CC with the new firmware 5.6.0 last week, used it without any problems for 2 days. Opened it after 6 days and the device bricked(MK4). After trying to open it for 30 min, the device turned on and now it's working. However I think this bug can still happen again with the current firmware so not sure what to do.
+
+comment: p27w6vj
+parent: t1_p27i4b1
+author: lucapocchio
+created_utc: 1786086324
+edited: false
+body:
+Yes it can happen. I wrote that also on the edit. Until the fix is not in a new version like 5.6.1.
+
+comment: p28mha5
+parent: t3_1vct6m2
+author: zatsnotmyname
+created_utc: 1786098868
+edited: false
+body:
+Works. 'x'
+
+comment: p28rrb0
+parent: t1_p27w6vj
+author: nicolbolas10
+created_utc: 1786101040
+edited: false
+body:
+still using coldcard or thinking about moving your funds?
