--- reddit-mk4-bricked-hotfix@20260806T032704Z
+++ reddit-mk4-bricked-hotfix@20260806T095815Z
@@ -124,19 +124,11 @@
 
 comment: p13yo6g
 parent: t3_1vct6m2
-author: DonTheHolder
+author: [deleted]
 created_utc: 1785607309
 edited: false
 body:
-Bricked and hacked. Lovely. Investors putting trust in companies who program their crypto as its worh 63k a coin right now for a $100 wallet. So silly.
-
-No migration. Get off the hardware wallets and mitigate the issue by going to the CEX such as Coinbase, ETF stock, etc. 
-
-You're going to pay taxes regardless, unless you're in another country that has none or low taxes.
-
-Safer for you, your family, and being wrenched 🔧 
-
-Not to mention there is a beneficiary path versus some posted note that your wife finds before she divorces you and has a boating accident. 
+[removed]
 
 comment: p13yrfv
 parent: t1_p13xt0j
@@ -231,14 +223,6 @@
 body:
 Nice try 😄
 
-comment: p158e0x
-parent: t1_p1560p6
-author: DonTheHolder
-created_utc: 1785620944
-edited: false
-body:
-Thats what the leftover Cold Card holders said. The bag holders are eliminated. 
-
 comment: p15ai96
 parent: t1_p13tui2
 author: SpareEconomy1849
@@ -443,14 +427,6 @@
 body:
 Its like a dog that got beat. Still gotta eat lol
 
-comment: p1cv0qw
-parent: t1_p15pe4v
-author: DonTheHolder
-created_utc: 1785717135
-edited: false
-body:
-All garbage 🗑 CEX. Coinbase or bust. Stop shopping at Walmart and shop at Whole Foods.
-
 comment: p1fx8i6
 parent: t1_p16gdco
 author: buzzkillb
