--- reddit-mk4-dice-seed-safety@20260804T083734Z
+++ reddit-mk4-dice-seed-safety@20260807T061406Z
@@ -200,11 +200,11 @@
 
 comment: p1a0ib2
 parent: t3_1vdge71
-author: Warrior_witha_Garden
+author: [deleted]
 created_utc: 1785687410
 edited: false
 body:
-No it is not safe. ALL the cold cards are cooked ! more is coming out. get a different wallet. You can use Tails and Electrum in a pinch. save that seed. Seed Signer is the move right now. Get some dice and get to work.
+[removed]
 
 comment: p1aqb2r
 parent: t3_1vdge71
@@ -241,14 +241,6 @@
 What else is coming out ?
 
 
-
-comment: p1be4h8
-parent: t1_p1as272
-author: Warrior_witha_Garden
-created_utc: 1785700926
-edited: false
-body:
-cold cards are getting bricked with new firmware. other exploits are being found. 1) if you have a usb you can use ian colemans seed generator using tails OS. free and secure. 2) you dont want a cold card for anything at this point. to much risk. if they were this sloppy its a matter of time. save your cold card for lawsuits and civil action. 
 
 comment: p1bmgr9
 parent: t1_p1ar3d9
