--- reddit-passphrase-entropy-ten-char@20260808T050250Z
+++ reddit-passphrase-entropy-ten-char@20260809T184851Z
@@ -121,3 +121,21 @@
 edited: false
 body:
 Checkout this site for good info on this. [https://drop.amboss.tech/entropy-explainer.html](https://drop.amboss.tech/entropy-explainer.html)
+
+comment: p2n9p1e
+parent: t1_p2bwmui
+author: Londonskaterboi
+created_utc: 1786283480
+edited: false
+body:
+yes but some wallets only accept bip39 words
+
+comment: p2ncvz8
+parent: t1_p2n9p1e
+author: na3than
+created_utc: 1786284458
+edited: false
+body:
+1. Picking "four words that are easy to remember" from a list of only 2048 words is absolutely not random.
+
+2. Every wallet that accepts a passphrase accepts any combination of letters, numbers and special characters as a passphrase. That's what a passphrase is. Check the specification. It's NOT meant to be a few more words from the canonical list of 2048. If you've found a wallet that stupidly constrains your passphrase that way, stop using that shitty wallet. There's no telling what other dumb decisions the authors made is implementing the specifications.
