--- reddit-throw-away-coldcard@20260804T121449Z
+++ reddit-throw-away-coldcard@20260804T124355Z
@@ -252,3 +252,19 @@
 edited: false
 body:
 That's why I only have minimal funds at banks
+
+comment: p1mz4l2
+parent: t3_1veuene
+author: DoctorDownvotesDelux
+created_utc: 1785846545
+edited: false
+body:
+The problem was with how it generated your seed. If you generate your seed elsewhere (like on a trezor) and import it to the coldcard you should be fine. Would I? Probably not
+
+comment: p1mziox
+parent: t1_p1k0lef
+author: NuggedClarp
+created_utc: 1785846672
+edited: false
+body:
+So, does that mean storing Bitcoin in a cold wallet ultimately depends on a third-party company staying in business forever?
