--- reddit-throw-away-coldcard@20260804T124355Z
+++ reddit-throw-away-coldcard@20260804T212613Z
@@ -268,3 +268,19 @@
 edited: false
 body:
 So, does that mean storing Bitcoin in a cold wallet ultimately depends on a third-party company staying in business forever?
+
+comment: p1qajqy
+parent: t1_p1l4n9p
+author: R7SOA19281
+created_utc: 1785877947
+edited: false
+body:
+Don’t they have like an extra 100 million now? /s
+
+comment: p1qb7r9
+parent: t1_p1mziox
+author: First-Check-164
+created_utc: 1785878126
+edited: false
+body:
+Depends on if you want updated hardware or not.  Things change with time.  If your 24 words have proper entropy, you are good.  
