--- reddit-throw-away-coldcard@20260804T094320Z
+++ reddit-throw-away-coldcard@20260804T111427Z
@@ -228,3 +228,19 @@
 edited: false
 body:
 You can generate the codes yourself using dice, regardless of the device you will eventually use, OMG. Otherwise, rolling dice and not relying on HW generators (Trezor and others have more than one) is stupidity. Literally no one ever cared about this before. But now? Suddenly it's mandatory and necessary? LOL.
+
+comment: p1mlxpk
+parent: t3_1veuene
+author: FavorableMadness
+created_utc: 1785841845
+edited: false
+body:
+Keep the device. It is superior in so many other ways. Patched, it is as good as any other. Acting like someone else built a better mouse trap is probably incorrect.
+
+comment: p1mmcif
+parent: t1_p1lai0l
+author: FavorableMadness
+created_utc: 1785842007
+edited: false
+body:
+Do you feel the tellers at your bank are highly skilled money managers?
