--- reddit-seed-collision-odds@20260804T144609Z
+++ reddit-seed-collision-odds@20260808T020129Z
@@ -232,3 +232,11 @@
 Yo fuck a coinkite. 
 
 So glad ColdCard wasn't my first device. 
+
+comment: p2cj9wr
+parent: t3_1vdr94r
+author: satoshisfeverdream
+created_utc: 1786138738
+edited: false
+body:
+Supposedly there were some thefts reported back in 2021-2022 timeframe that could have simply been organic collisions.  
