--- reddit-ledger-official-incident-statement@20260809T121952Z
+++ reddit-ledger-official-incident-statement@20260809T170048Z
@@ -264,3 +264,19 @@
 edited: false
 body:
 For folks who are comfortable to handle passphrase, any tips when setting and managing one?
+
+comment: p2n08xn
+parent: t1_p2jrvtd
+author: btchip
+created_utc: 1786280395
+edited: false
+body:
+It can be anything you want, if you want to just create more accounts it could be something as simple as "ac1" "ac2" ... and you can have one associated to a PIN which is nice as a duress code
+
+comment: p2ne6id
+parent: t1_p2n08xn
+author: _speedoflight_
+created_utc: 1786284853
+edited: false
+body:
+Thanks, some people say it needs to be a very long and strong word or collection of words. Having “ac1”, “ac2” will be easy to brute force if someone happens to get their hands on the recovery pass phrase isn’t? Isn’t it recommended to be at least 12 characters long?
