--- clay-attribution@20260809T053309Z
+++ clay-attribution@20260809T120504Z
@@ -50,7 +50,7 @@
 post: 2083249035864154259
 role: reply
 author: humble21m
-name: HODL ∞/21M
+name: ∞/21M (BIP448)
 created: 2026-07-31T17:50:49Z
 media: 0
 body:
@@ -274,6 +274,24 @@
 media: 0
 body:
 great work. keep us updated on findings.
+
+post: 2083261908908495216
+role: reply
+author: r0ckstardev
+name: Uncle Rockstar Developer
+created: 2026-07-31T18:41:58Z
+media: 0
+body:
+let it be a credit card payment 
+
+post: 2083262290854388055
+role: reply
+author: MctoshiW
+name: McToshi Whoppamoto
+created: 2026-07-31T18:43:29Z
+media: 0
+body:
+Any guess on provider??
 
 post: 2083262795207041083
 role: reply
@@ -538,6 +556,15 @@
 body:
 
 
+post: 2083276133613584491
+role: reply
+author: ggggggg13131313
+name: Nero
+created: 2026-07-31T19:38:30Z
+media: 0
+body:
+Awesome! Well done.
+
 post: 2083277255191429287
 role: reply
 author: leafdelpino
@@ -750,6 +777,15 @@
 body:
 @ojedabtc
 
+post: 2083330414869905443
+role: reply
+author: bitchimlying
+name: -.-- --- ..- / .- .-. . / -... --- .-. . -..
+created: 2026-07-31T23:14:11Z
+media: 0
+body:
+how do they even know which addresses to target? cold card leave some kind of footprint?
+
 post: 2083331018170233053
 role: reply
 author: loblawbob2
@@ -1154,6 +1190,16 @@
 body:
 Should be available for shipment to Ukraine next week.
 
+post: 2083462272291340458
+role: reply
+author: IphigenAleksy
+name: Aleksy
+created: 2026-08-01T07:58:09Z
+media: 0
+body:
+@grok
+ Is this how the attacker knew how to hone in the addresses to  brute force? What I don't get yet is, if those low entropy addresses were sitting on the chain for that long, why weren't they attacked earlier? Low entropy brain wallets get emptied instantly.
+
 post: 2083470017132187992
 role: reply
 author: observerofdecay
@@ -1240,6 +1286,15 @@
 body:
 Cheering for you all to catch this scumbag! 
 
+post: 2083530023320142052
+role: reply
+author: RNHairman
+name: Plebtitioner RN
+created: 2026-08-01T12:27:22Z
+media: 0
+body:
+I smell CIA
+
 post: 2083531203974402095
 role: reply
 author: bokiko
@@ -1409,6 +1464,15 @@
 @grok
  quais as últimas notícias?
 
+post: 2084038620806328398
+role: reply
+author: FiatisF00lsgold
+name: April Snow
+created: 2026-08-02T22:08:21Z
+media: 0
+body:
+GET THEM 
+
 post: 2084403700957872525
 role: reply
 author: pseudoramdom
