--- clay-attribution@20260808T162954Z
+++ clay-attribution@20260808T230229Z
@@ -275,6 +275,15 @@
 body:
 great work. keep us updated on findings.
 
+post: 2083262290854388055
+role: reply
+author: MctoshiW
+name: McToshi Whoppamoto
+created: 2026-07-31T18:43:29Z
+media: 0
+body:
+Any guess on provider??
+
 post: 2083262795207041083
 role: reply
 author: darrenahunter
@@ -538,6 +547,15 @@
 body:
 
 
+post: 2083276133613584491
+role: reply
+author: ggggggg13131313
+name: Nero
+created: 2026-07-31T19:38:30Z
+media: 0
+body:
+Awesome! Well done.
+
 post: 2083277255191429287
 role: reply
 author: leafdelpino
@@ -1096,15 +1114,6 @@
 body:
 Lol
 
-post: 2083427608629326307
-role: reply
-author: Zenul_Abidin
-name: Ali Sherief
-created: 2026-08-01T05:40:24Z
-media: 0
-body:
-Did you get a name and contact details?
-
 post: 2083435429756563708
 role: reply
 author: mannan25195
@@ -1381,15 +1390,6 @@
 body:
 For something is so basic and an upmost important you guys screwed up… providing a random seed phrase generator that is not random
 
-post: 2083805166940127666
-role: reply
-author: mmni99inc
-name: Adam Charles Maxwell
-created: 2026-08-02T06:40:41Z
-media: 0
-body:
-Nice work 
-
 post: 2083823213042192384
 role: reply
 author: OurielOhayon
