--- reddit-self-custody-to-etf@20260804T134555Z
+++ reddit-self-custody-to-etf@20260804T232425Z
@@ -1516,6 +1516,16 @@
 body:
 True indeed. Those who bought pre covid are trapped lol.
 
+comment: p1l7khc
+parent: t3_1ve1fai
+author: Zaytion_
+created_utc: 1785818207
+edited: false
+body:
+If the technology of Bitcoin is too unsafe for people to use, then the thesis is gone right? Why own it at that point? Just sell your Bitcoin and go buy some physical gold or something. 
+
+If everyone just buys the ETF, you don't have a decentralized currency of the future.
+
 comment: p1l7ryc
 parent: t1_p1do8mg
 author: Zaytion_
