--- reddit-self-custody-to-etf@20260805T142722Z
+++ reddit-self-custody-to-etf@20260805T210240Z
@@ -1591,6 +1591,14 @@
 It's there/built, if the world ends and I need it.  
 
 
+comment: p1scqg1
+parent: t3_1ve1fai
+author: Maximum_Stretch1465
+created_utc: 1785901504
+edited: false
+body:
+I lost some gold also. Very little but how when all my settings where for immediate verification
+
 comment: p1slni0
 parent: t1_p1fu8s9
 author: More-Translator9074
@@ -1622,3 +1630,11 @@
 edited: false
 body:
 Why?
+
+comment: p1vti63
+parent: t3_1ve1fai
+author: Nervous-Fox-7103
+created_utc: 1785947014
+edited: false
+body:
+Absolutely not
