--- reddit-self-custody-to-etf@20260805T210240Z
+++ reddit-self-custody-to-etf@20260806T100255Z
@@ -47,15 +47,11 @@
 
 comment: p1dl6os
 parent: t3_1ve1fai
-author: DonTheHolder
+author: [deleted]
 created_utc: 1785726389
 edited: false
 body:
-Already done
-
-They are broken and broke. They trusted their bag with a hardware wallet creator instead of security professionals and other safety factors and switches that come into play that are secured with stocks as ETFS and CEX COINBASES.
-
-You're not nerd enough to protect your coin. Let a professional company manage them properly. 
+[removed]
 
 comment: p1dl82p
 parent: t3_1ve1fai
