--- btcpp-dettmer-commit-history@20260803T110725Z
+++ btcpp-dettmer-commit-history@20260804T173940Z
@@ -92,15 +92,15 @@
 Subscribe to Dusty
 Discussion about this post
 CommentsRestacks
-Lynne Bairstow
-<comment-age>
-Exceptional explanation—thanks, Dusty. I believe anyone impacted by this (and, aren’t we all?) wants to understand what the heck happened; how it could have possibly happened—and this unravels the mystery. Your closing remarks are the critical take-away.
-Reply
-Share
 Curious George
 <comment-age>
 Very well said. As a retired dev I appreciate this analysis. Good God - how can you ship code you don't understand, especially involving money? People's lives are at risk!!
 Complexity **IS** the enemy.
+Reply
+Share
+Lynne Bairstow
+<comment-age>
+Exceptional explanation—thanks, Dusty. I believe anyone impacted by this (and, aren’t we all?) wants to understand what the heck happened; how it could have possibly happened—and this unravels the mystery. Your closing remarks are the critical take-away.
 Reply
 Share
 2 more comments...
