--- btcpp-dettmer-commit-history@20260809T231639Z
+++ btcpp-dettmer-commit-history@20260812T043842Z
@@ -93,7 +93,7 @@
 Discussion about this post
 CommentsRestacks
 Curious George
-<comment-age>
+Aug 2
 Liked by ~nifty~
 Very well said. As a retired dev I appreciate this analysis. Good God - how can you ship code you don't understand, especially involving money? People's lives are at risk!!
 Complexity **IS** the enemy.
