--- btcpp-dettmer-commit-history@20260802T140112Z
+++ btcpp-dettmer-commit-history@20260802T200240Z
@@ -4,9 +4,9 @@
 What a commit message tells us about the recent COLDCARD bug
 Dusty Daemon
 Aug 01, 2026
-36
+46
 4
-4
+6
 Share
 This is a guest post from noted Core-Lightning developer, ddustin, who dug into the Coldcard firmware commit history to uncover what happened and why the code failed.
 Intro
@@ -86,9 +86,9 @@
 I hope that we, as an industry, can learn from this and lean on each other to ship secure code, as a community of developers.
 Thanks for reading bitcoin++'s Insider Edition! Click below to share this post with your GC
 Share
-36
+46
 4
-4
+6
 Share
 A guest post by
 Dusty Daemon
@@ -96,16 +96,15 @@
 Subscribe to Dusty
 Discussion about this post
 CommentsRestacks
-Boomberg
-4h
-https://github.com/switck/libngu
-is to blame. The bad code came from here
+Curious George
+18h
+Very well said. As a retired dev I appreciate this analysis. Good God - how can you ship code you don't understand, especially involving money? People's lives are at risk!!
+Complexity **IS** the enemy.
 Reply
 Share
-Curious George
-12h
-Very well said. As a retired dev I appreciate this analysis. Good God - how can you ship code you don't understand, especially involving money? People's lives are at risk!!
-Complexity **IS** the enemy.
+Lynne Bairstow
+19h
+Exceptional explanation—thanks, Dusty. I believe anyone impacted by this (and, aren’t we all?) wants to understand what the heck happened; how it could have possibly happened—and this unravels the mystery. Your closing remarks are the critical take-away.
 Reply
 Share
 2 more comments...
