--- trustwallet-wasm-update@20260809T120353Z
+++ trustwallet-wasm-update@20260814T235227Z
@@ -173,6 +173,15 @@
 body:
 $VRA
 
+post: 1649731250876473344
+role: reply
+author: getasriram
+name: Dr.Sriram
+created: 2023-04-22T11:05:55Z
+media: 0
+body:
+Why Elrond logo there ? Please advise
+
 post: 1649745484649750530
 role: reply
 author: AdityaPoco
@@ -181,6 +190,32 @@
 media: 0
 body:
 its centralisation rather than decentralisation
+
+post: 1649754081186004993
+role: reply
+author: Anoxxxy
+name: Anoxy (Wally.id)
+created: 2023-04-22T12:36:38Z
+media: 0
+body:
+Should Trustwallet really be trusted?
+
+
+http://
+Wally.id has top notch security in a totally decentralized environment,  users funds are safe 
+
+#decentralized #crypto #wallet
+
+@WallyIDOfficial
+
+post: 1649758372496035842
+role: reply
+author: japrisx
+name: VanderWaals
+created: 2023-04-22T12:53:41Z
+media: 0
+body:
+I've observed that the dapp browser was so slow at times. Maube this was why.
 
 post: 1649765931768483840
 role: reply
@@ -230,6 +265,21 @@
 @BoDjimou
  lisez le fil en dessous
 
+post: 1649809753860653059
+role: reply
+author: Robacoin
+name: Robacoin
+created: 2023-04-22T16:17:51Z
+media: 0
+body:
+@TrustWallet
+
+ somedays ago I had issues transferring MATIC on the polygon network. All my transfer txs were not processed, just pending. Had to import wallet to Metamask and did the transfer. However, today I deposited MATIC to my 
+
+@TrustWallet
+
+,
+
 post: 1649812800695422977
 role: reply
 author: HentonsShop
