--- reddit-entropy-bug-commits-review@20260804T184658Z
+++ reddit-entropy-bug-commits-review@20260804T191619Z
@@ -1003,3 +1003,37 @@
 edited: false
 body:
 Most big names are all hat and no cattle. What exactly did he do to warrant that reverence?
+
+comment: p1pduwo
+parent: t1_p1da4ay
+author: magicmulder
+created_utc: 1785869345
+edited: false
+body:
+\> with that "squashing commits" approach, you will never make a "code change"
+
+No, that is literally how it works. The reviewer looks at your PR with its commits and commit messages, and once the PR is accepted, nobody ever sees those commits again because nobody wants 825 commit messages "fixed dumb bug from yesterday" from a single PR in the dev or main branch, that's just noise.
+
+comment: p1pk4cd
+parent: t1_p1pduwo
+author: DyehuthyTV
+created_utc: 1785870947
+edited: false
+body:
+ Have you ever read a PR in a big project? lol xD  
+  
+Example, from Bitcoin Core (Node) repo on Github: [Pull requests · bitcoin/bitcoin](https://github.com/bitcoin/bitcoin/pulls) 👀 
+
+Commit: <type/scope (dir)>: <description> 👇🏻
+
+`util: set os-level thread names on Windows- #number`
+
+You cant put a commits like this 👇🏻
+
+`Windows` or `Crap`  
+  
+xD lol, is non-professional 
+
+Plus a comment explaining the the reason (bug, fix, error, typo, etc) and the logic of your PR.
+
+Read the CONTRIBUTING file, before talking about it lol
