--- reddit-entropy-bug-commits-review@20260804T191619Z
+++ reddit-entropy-bug-commits-review@20260804T194701Z
@@ -1018,11 +1018,11 @@
 parent: t1_p1pduwo
 author: DyehuthyTV
 created_utc: 1785870947
-edited: false
-body:
- Have you ever read a PR in a big project? lol xD  
-  
-Example, from Bitcoin Core (Node) repo on Github: [Pull requests · bitcoin/bitcoin](https://github.com/bitcoin/bitcoin/pulls) 👀 
+edited: 1785871211
+body:
+Have you ever read a PR in a big project? lol xD
+
+Example, from Bitcoin Core (Node) repo on Github: [Pull requests · bitcoin/bitcoin](https://github.com/bitcoin/bitcoin/pulls) (Github Link) 👀
 
 Commit: <type/scope (dir)>: <description> 👇🏻
 
@@ -1030,10 +1030,12 @@
 
 You cant put a commits like this 👇🏻
 
-`Windows` or `Crap`  
-  
-xD lol, is non-professional 
-
-Plus a comment explaining the the reason (bug, fix, error, typo, etc) and the logic of your PR.
-
-Read the CONTRIBUTING file, before talking about it lol
+`Windows` or `Crap`
+
+xD lol, is non-professional
+
+Plus a comment explaining the reason (bug, fix, error, typo, etc) and the logic of your PR.
+
+**There is much more descriptive content (commits + comments) in a PR than people who have never worked as developers believe ;)**
+
+**Read the CONTRIBUTING file, before talking about it lol**
