--- reddit-mcu-uid-seed-correction@20260806T050852Z
+++ reddit-mcu-uid-seed-correction@20260806T101616Z
@@ -104,3 +104,11 @@
 It would be prudent to remind people here that there are still a lot of vulnerable funds that could be saved and that it would be anti-social to explain how to reduce the search space of coldcard devices just to brag about how smart you are, or show that some rando was wrong on the internet.
 
 Plenty of other people could be posting about the *exact* search space needed to enumerate vulnerable seeds and are kindly refraining from doing so.  Just because you could figure out that doesn't mean that every would be coin-thief is going to figure it out before the owners sweep their coins.
+
+comment: p20heev
+parent: t1_p1y70jo
+author: opossum_cz
+created_utc: 1785999781
+edited: false
+body:
+Sounds plausible.
