--- reddit-fake-trezor-google-ad@20260809T104026Z
+++ reddit-fake-trezor-google-ad@20260809T232738Z
@@ -1868,6 +1868,14 @@
 body:
 Yah I scroll past the sponsored links now and never click on them.  They are full of scams. 
 
+comment: p2leu1s
+parent: t1_p2cn99x
+author: blueshoesrcool
+created_utc: 1786253487
+edited: false
+body:
+Including btc itself
+
 comment: p2llcxh
 parent: t3_1vi2jv0
 author: racingmonster1234
@@ -1883,3 +1891,23 @@
 edited: false
 body:
 😂😂
+
+comment: p2o2tja
+parent: t3_1vi2jv0
+author: corporate-citizen
+created_utc: 1786291962
+edited: false
+body:
+Cardinal rule for cold wallet newbies. It’s simpler than the four rules of gun safety:
+
+# 1. Never type your seed phrase onto any keyboard, EVER.
+
+2. See no. 1.
+
+comment: p2ok7uw
+parent: t1_p2ag6hq
+author: abitofcoinOTC
+created_utc: 1786296838
+edited: false
+body:
+I agree that they should be liable. Same with Youtube and all of those fake Elon Musk scam videos. However, Section 230 immunity has protected them for the most part. I don't agree with this, particularly with advertising, as even the most basic of screening would prevent this, but most of the laws were designed and lobbied for by big corporations to protect themselves, so it doesn't surprise me.
