--- reddit-fake-trezor-google-ad@20260808T083713Z
+++ reddit-fake-trezor-google-ad@20260808T150803Z
@@ -809,16 +809,6 @@
 body:
 lol wdym? ColdCard was chilled by everyone as the top best way for everyone to self custody. Don’t come acting like people who used it were wreckless or dumb or ill informed. Any other more complicated wallet is just out of reach for the average holder. For me SC just failed to democratize and get safe and easy enough after all those years. I tried it, but swapped it for ETF now (for ease of use and tax incentives) and I am so glad I did.
 
-comment: p2ayy7o
-parent: t1_p2aff1u
-author: Peewee223
-created_utc: 1786123667
-edited: false
-body:
-Google runs one of the biggest web crawlers in the world.
-
-They could detect that an advertised site changed significantly and disable the ad until it's reviewed again.
-
 comment: p2az52q
 parent: t1_p2ad13b
 author: Free-Way-9220
@@ -1685,3 +1675,81 @@
 edited: false
 body:
 I’m kinda new at this so please forgive my ignorance. How would one access their wallet if they lose their Trezor device? Don’t you essentially have to type your seed phrase into somewhere, like Trezor website or suite? 
+
+comment: p2fdhx5
+parent: t1_p2bfpkm
+author: ComaVN
+created_utc: 1786178244
+edited: false
+body:
+You really should check the pgp signature . Then it doesn't matter where you downloaded it from.
+
+comment: p2fj6fd
+parent: t1_p2fdhx5
+author: nycteris91
+created_utc: 1786181213
+edited: false
+body:
+I did as well.
+
+I don't trust anything. I miss when they published the hash on the website, which is the easiest to verify.
+
+It seems to be outdated.
+
+comment: p2fq014
+parent: t1_p2az52q
+author: Whole-Respond4782
+created_utc: 1786184701
+edited: false
+body:
+yeah they've shown countless times that they don't care at all sadly
+
+google wants that money, people paying for ads is how they get it, i'm sure its intentional that they're not very strict
+
+comment: p2frw8e
+parent: t3_1vi2jv0
+author: nyphoebe
+created_utc: 1786185624
+edited: false
+body:
+damn I need to learn how to scam
+
+comment: p2fvhlv
+parent: t1_p2a3fwe
+author: MannerismsBot8000
+created_utc: 1786187285
+edited: false
+body:
+Victims of crypto scams become unable to have children!? 
+
+comment: p2fvt9x
+parent: t3_1vi2jv0
+author: MigBuscles
+created_utc: 1786187430
+edited: false
+body:
+Scam website irl:
+
+Please enter your 🔑 
+
+…okay 
+
+🪙🪙🪙👻
+
+comment: p2gab0v
+parent: t1_p2d0268
+author: tenor_tymir
+created_utc: 1786193204
+edited: false
+body:
+They just said on X that they are going to in a reply to the victim. 
+
+comment: p2gyu1x
+parent: t1_p2ag6hq
+author: jesuswithoutabeard
+created_utc: 1786201024
+edited: false
+body:
+This has been going on for years, I once very nearly got scammed when I googled for the Sirius XM contact number and the sponsored topresult was a scam number. 
+
+Google is absolutely complicit.
