--- reddit-fake-trezor-google-ad@20260809T041034Z
+++ reddit-fake-trezor-google-ad@20260809T104026Z
@@ -1089,11 +1089,11 @@
 
 comment: p2bqhvh
 parent: t1_p2b88vc
-author: jesusrambo
+author: [deleted]
 created_utc: 1786130798
-edited: false
-body:
-This is called GofundMe lol
+edited: 1786256684
+body:
+[deleted]
 
 comment: p2bsskv
 parent: t1_p2aleaq
@@ -1859,3 +1859,27 @@
 edited: false
 body:
 But what if you lose the hardware wallet itself? 
+
+comment: p2l7knr
+parent: t1_p2a3zmh
+author: Time_after_Time_67
+created_utc: 1786250163
+edited: false
+body:
+Yah I scroll past the sponsored links now and never click on them.  They are full of scams. 
+
+comment: p2llcxh
+parent: t3_1vi2jv0
+author: racingmonster1234
+created_utc: 1786256633
+edited: false
+body:
+Almost did that myself 
+
+comment: p2m0a3u
+parent: t3_1vi2jv0
+author: GiveMeSomeOfThatWow
+created_utc: 1786264111
+edited: false
+body:
+😂😂
