--- reddit-fake-trezor-google-ad@20260808T150803Z
+++ reddit-fake-trezor-google-ad@20260808T213758Z
@@ -1087,14 +1087,6 @@
 body:
 This is on Google , they have had this issue forever but as long as you pay for adds they will put it up without content checking for phishing
 
-comment: p2boli1
-parent: t3_1vi2jv0
-author: Just_Bluebird_5268
-created_utc: 1786130295
-edited: false
-body:
-It's the future of money
-
 comment: p2bqhvh
 parent: t1_p2b88vc
 author: jesusrambo
@@ -1753,3 +1745,53 @@
 This has been going on for years, I once very nearly got scammed when I googled for the Sirius XM contact number and the sponsored topresult was a scam number. 
 
 Google is absolutely complicit.
+
+comment: p2h2lcr
+parent: t1_p2a7s3c
+author: Souprshooter
+created_utc: 1786202127
+edited: false
+body:
+Take this tele tab my main bournos is waiting in Camelot
+
+comment: p2hf2ed
+parent: t1_p2aee0h
+author: Dmoan
+created_utc: 1786205709
+edited: false
+body:
+This is nothing new Google has refused to take steps to prevent spammers from exploiting it.
+
+I used to report these like the fake insurance companies masquering as real insurance companies popping up in adverts appearing in search results. Google would rarely take action.. 
+
+comment: p2hp1tr
+parent: t1_p2cms7u
+author: electromage
+created_utc: 1786208558
+edited: false
+body:
+2.5 BTC, A "sat" is 0.00000001 BTC.
+
+comment: p2i08cv
+parent: t1_p2ey1we
+author: True-Lychee
+created_utc: 1786211689
+edited: false
+body:
+Only into the hardware wallet itself. Never into Windows or any other operating system that could have a keylogger, compromised website or phishing site.
+
+comment: p2i7ls6
+parent: t3_1vi2jv0
+author: MiamiSkeet69
+created_utc: 1786213772
+edited: false
+body:
+Imagine fake banking apps and websites that drain your bank account, i will forever stay away from bitcoin it does nothing it was originally created for 
+
+comment: p2ijwwh
+parent: t1_p29yr27
+author: MPolanyi
+created_utc: 1786217338
+edited: false
+body:
+Rule #0 don't put life savings in any crypto facility
