--- reddit-fake-trezor-google-ad@20260809T232738Z
+++ reddit-fake-trezor-google-ad@20260812T044253Z
@@ -508,16 +508,6 @@
 body:
 I used RBF since the transaction was still unconfirmed when I caught it.
 
-comment: p2aitav
-parent: t1_p2a96u3
-author: Todo_es
-created_utc: 1786119540
-edited: false
-body:
-Not true. Self custody is safe as long as you are able to keep your private key safe, which is not difficult if you follow some very basic safety rules.
-
-Once I tested the freedom and power of sovereignty over my money, I will never go back to give it away to central powers.
-
 comment: p2aiywn
 parent: t3_1vi2jv0
 author: sing2nite
@@ -1911,3 +1901,43 @@
 edited: false
 body:
 I agree that they should be liable. Same with Youtube and all of those fake Elon Musk scam videos. However, Section 230 immunity has protected them for the most part. I don't agree with this, particularly with advertising, as even the most basic of screening would prevent this, but most of the laws were designed and lobbied for by big corporations to protect themselves, so it doesn't surprise me.
+
+comment: p2s4wel
+parent: t3_1vi2jv0
+author: EmuFlaky2922
+created_utc: 1786339699
+edited: false
+body:
+Still ok w Coinbase here. No one is perfect but they’re more perfect than me when it goes to protecting shit. I just know Coinbase will never call me cuz realistically their support is awful 🤣 but that’s usually the scam
+
+comment: p2t16au
+parent: t1_p2l2x6z
+author: Own-Veterinarian4611
+created_utc: 1786355901
+edited: false
+body:
+Buy a new one, and enter it only into the new hardware wallet, never ever anywhere else.
+
+comment: p2xl8we
+parent: t1_p2af7ag
+author: trufin2038
+created_utc: 1786403278
+edited: false
+body:
+There is no case for that horrible design
+
+comment: p2xlfnk
+parent: t1_p2ahu2k
+author: trufin2038
+created_utc: 1786403335
+edited: false
+body:
+Scams are a dime a dozen. I'm disappointed in people who trust search engine and websites and enter their root mnemonic into anything that asks for it.
+
+comment: p350kdl
+parent: t3_1vi2jv0
+author: No-Reading-4384
+created_utc: 1786491121
+edited: false
+body:
+I got clipped for 160k. Sucks!!!
