--- reddit-fake-trezor-google-ad@20260808T213758Z
+++ reddit-fake-trezor-google-ad@20260809T041034Z
@@ -1795,3 +1795,67 @@
 edited: false
 body:
 Rule #0 don't put life savings in any crypto facility
+
+comment: p2jzzjs
+parent: t1_p29yiko
+author: ChironXII
+created_utc: 1786233633
+edited: false
+body:
+Hmm, but then again some people lose all their money and conclude only that they need to get more to lose 
+
+comment: p2k3idw
+parent: t1_p2aee0h
+author: Big80sweens
+created_utc: 1786234855
+edited: false
+body:
+I won’t take as long if you keep buying bitcoin
+
+comment: p2k6ghn
+parent: t1_p2ag6hq
+author: CalvinsStuffedTiger
+created_utc: 1786235902
+edited: false
+body:
+Half of the YouTube ads I get are from fuckin scam companies 
+
+comment: p2kefbg
+parent: t3_1vi2jv0
+author: griswaldwaldwald
+created_utc: 1786238762
+edited: false
+body:
+Why would anyone ever enter their hardware seed into a website rather than into the device?
+
+comment: p2kffwi
+parent: t3_1vi2jv0
+author: BitcoinMD
+created_utc: 1786239122
+edited: false
+body:
+This is self custody
+
+comment: p2ki697
+parent: t1_p2ag6hq
+author: professional_burrito
+created_utc: 1786240117
+edited: false
+body:
+As of 6 months ago, if you search “vanilla visa gift card balance” on Bing, the first three results were all phishing pages. Not sure if they fixed it yet.
+
+comment: p2kj9l4
+parent: t1_p2i7ls6
+author: True-Lychee
+created_utc: 1786240516
+edited: false
+body:
+What are you talking about? People get their bank accounts drained by scams all the time.
+
+comment: p2l2x6z
+parent: t1_p2i08cv
+author: olivehue1
+created_utc: 1786248177
+edited: false
+body:
+But what if you lose the hardware wallet itself? 
