--- reddit-forrest-video-coldcard@20260813T210008Z
+++ reddit-forrest-video-coldcard@20260814T032834Z
@@ -367,3 +367,17 @@
 edited: false
 body:
 Don't remember who it was but there was one wallet a decade ago that kept getting hacked.  It turned out they were using random.org to generate random numbers and random.org went down or changed urls or something resulting in the number 404  being the random number that was always used.
+
+comment: p3j3sim
+parent: t1_p3e8i0w
+author: PaperPigGolf
+created_utc: 1786657922
+edited: false
+body:
+How do you suggest converting into 24 words? How do you verify that the software you are using to do that conversion is working propery.
+
+I'm not... arguing, I think this is ultimately what the entire community is scratching their heads right now.
+
+Basically, if you are trusting software to do the right thing, how do you come about trusting that it has done the right thing. 
+
+In your case, the software is some conversion from actualy seed to the BIP-39. 
