--- reddit-coinkite-solvency-class-action@20260804T084605Z
+++ reddit-coinkite-solvency-class-action@20260804T134147Z
@@ -499,4 +499,14 @@
 body:
 I just want to understand why everybody says that, Ledger is a peace of shit, because not open source. On the other side, they produced over 7 million wallets, and no one is compromised so far. I was work for a multi-million dollar financial company for years. I work with Thales HSM cards. Never heard from any security expert that: "it is not safe, because closed source". I don't think who say it in youtube/reddit/X knows mutch more about the IT security than the experts I know.
 
+comment: p1mqbpw
+parent: t1_p17fzvj
+author: magma_lakes
+created_utc: 1785843516
+edited: false
+body:
+... said ColdCard users about a week ago.
+
+You don't know if there's a bug if you can't inspect the code.
+
 more-stub: parent t1_p1igcfs count 0
