--- reddit-coinkite-solvency-class-action@20260806T095821Z
+++ reddit-coinkite-solvency-class-action@20260808T150010Z
@@ -533,4 +533,26 @@
 body:
 Amazing that a single line of code caused all this.
 
+comment: p2fkl6u
+parent: t1_p1mqbpw
+author: sasankhatibi
+created_utc: 1786181955
+edited: false
+body:
+ColdCard was/is open source. What's your excuse? Did YOU inspect it, or are you just repeating what others are telling you?
+
+comment: p2flj1q
+parent: t1_p2fkl6u
+author: magma_lakes
+created_utc: 1786182441
+edited: false
+body:
+I've been a programmer for many years, and I don't use any hardware wallets.
+
+I have a bunch of Python scripts that I wrote myself many years ago that generate a Bitcoin wallet, from scratch.
+
+I've copied them over to my air-gapped Linux PC, in order to generate my Bitcoin wallets.
+
+So, I don't need to inspect any hardware wallet's code, because I don't use them. I only need to inspect and trust my own code, which I do.
+
 more-stub: parent t1_p1igcfs count <live-count>
