--- ledger-open-source-security-position@20260809T122007Z
+++ ledger-open-source-security-position@20260814T015834Z
@@ -235,14 +235,14 @@
 Those who equated Coldcard’s public source (not really open source, by the way) with security just learned, the…
 — Charles Guillemet (@P3b7_) August 5, 2026
 More articles
+Product | 08/14/2026
+Unveiling the Ledger Flex™ Hidden Edition
+Read more
 Thought leadership | 08/02/2026
 The Coldcard Incident: How Did This Happen?
 Read more
 Company | 07/24/2026
 Ledger & MoonPay Bring Digital Ownership to the X Games...
-Read more
-Product | 07/08/2026
-Ledger Wallet™: Upgrade Your Digital Life, Free From Compromise...
 Read more
 Stay in touch
 Announcements can be found in our blog. Press contact:
