--- reddit-0-7-btc-drained@20260805T011614Z
+++ reddit-0-7-btc-drained@20260805T014620Z
@@ -2843,4 +2843,28 @@
 body:
 "Not your keys; not tour cryto." They said.
 
+comment: p1roblq
+parent: t1_p1q6bh4
+author: RelevantAmbition2433
+created_utc: 1785893109
+edited: false
+body:
+I just used it to see if the funds were indeed stolen. Never used the app before this. 
+
+comment: p1rqq8k
+parent: t3_1vewzn4
+author: Yohbaba
+created_utc: 1785893899
+edited: false
+body:
+Omg really feel for you brother, I was a victim of Celsius debacle but I've recovered quite a bit. It just sucks when these things happen and coinkite is 100% responsible for it. Horrible company that always marketed their product as a Bitcoin safe haven. So sorry this happened to you but it just infuriates me how grass root companies are so messed up. I can only imagine and I've been through some what similar thing. Hang in there man, hope this company finds a way to compensate you to whatever degree possible.
+
+comment: p1rr8ui
+parent: t3_1vewzn4
+author: Yohbaba
+created_utc: 1785894071
+edited: false
+body:
+Set a "PASSPHRASE" folks please, small nusance with high upside.
+
 more-stub: parent t1_p1l2enr count <live-count>
