--- reddit-0-7-btc-drained@20260805T004611Z
+++ reddit-0-7-btc-drained@20260805T011614Z
@@ -2827,4 +2827,20 @@
 body:
 lol
 
+comment: p1rhzy7
+parent: t1_p1kvwdh
+author: Plus-Barber-6171
+created_utc: 1785891050
+edited: false
+body:
+And you sign your transactions with what exactly? 
+
+comment: p1rl1ow
+parent: t3_1vewzn4
+author: _GAT_in_the_HAT_
+created_utc: 1785892040
+edited: false
+body:
+"Not your keys; not tour cryto." They said.
+
 more-stub: parent t1_p1l2enr count 0
