--- reddit-0-7-btc-drained@20260804T134325Z
+++ reddit-0-7-btc-drained@20260804T141416Z
@@ -2347,4 +2347,38 @@
 body:
 Use Trezor. 
 
+comment: p1ngutk
+parent: t3_1vewzn4
+author: cozmicraven
+created_utc: 1785851792
+edited: false
+body:
+I hate reading all the posts about this hack. Cold storage isn't as cold as we were promised. I sure hope other companies like Ledger and Trezor and etc. are paying attention.
+
+At some point BTC holders will have to decide if the security of holding your coins at an exchange or even buying into an etf is worth trading your ownership for. Presumably corporate security (Coinbase, Kraken etc.) is better and they have private insurance to at least get you some of your value back in the event of a hack. I'm fully aware of the crappy cs all these companies share but all the wallet owners that got attacked here have nowhere to go.
+
+comment: p1nihln
+parent: t1_p1ko8i6
+author: KELVALL
+created_utc: 1785852244
+edited: false
+body:
+It's from the old cowboy movies...White hat good...Black hat bad.
+
+comment: p1nj04v
+parent: t3_1vewzn4
+author: ILurkReddi
+created_utc: 1785852385
+edited: false
+body:
+fuck
+
+comment: p1njwxp
+parent: t3_1vewzn4
+author: kftnyc
+created_utc: 1785852634
+edited: false
+body:
+Want a cold wallet? Write your private key on a piece of paper and put it in a safe.
+
 more-stub: parent t1_p1l2enr count 0
