--- reddit-0-7-btc-drained@20260806T163222Z
+++ reddit-0-7-btc-drained@20260809T040413Z
@@ -3065,9 +3065,11 @@
 parent: t1_p1qe4en
 author: KeyAdvanced1032
 created_utc: 1785970211
-edited: false
+edited: 1786226770
 body:
 Seconded.
+
+Edit: Close-sourced since 2021. Yikes.
 
 comment: p1zfyrq
 parent: t1_p1km2f8
@@ -3111,4 +3113,12 @@
 body:
 I am so sorry.
 
+comment: p2kcbmy
+parent: t3_1vewzn4
+author: lookn4a304
+created_utc: 1786238000
+edited: false
+body:
+Guess what nationality is behind CC. Every fucking time…
+
 more-stub: parent t1_p1l2enr count <live-count>
