--- reddit-0-7-btc-drained@20260804T151413Z
+++ reddit-0-7-btc-drained@20260804T154435Z
@@ -2433,4 +2433,12 @@
 body:
 if it was an insider job those 100M might turn into 500M in btc in a few years then coldcard could pay the 100M back to its hacked users
 
+comment: p1o0d1v
+parent: t1_p1lhxen
+author: fllannell
+created_utc: 1785856983
+edited: false
+body:
+And also, to complicate things more often than not scammers claim to be white hat hackers when they have no real ability to recover funds/assets and are really just acting out a secondary scam to victims of crime and other scams. They do this by identifying victims, then offering their services for a fee, then another fee, then another fee... until the victim finally (hopefully) gives up.
+
 more-stub: parent t1_p1l2enr count 0
