--- reddit-0-7-btc-drained@20260804T211532Z
+++ reddit-0-7-btc-drained@20260804T214524Z
@@ -2726,4 +2726,37 @@
 body:
 This is why I use IBIT etf for my bitcoin, so sorry and hope your 2.0 investments work out 
 
+comment: p1qbwpl
+parent: t1_p1l261a
+author: KungLa0
+created_utc: 1785878314
+edited: false
+body:
+It's been dead in the water for so long IMO. It is kept alive by speculation and "get rich quick" investors. 
+
+comment: p1qcrr4
+parent: t3_1vewzn4
+author: GiveMeSomeOfThatWow
+created_utc: 1785878545
+edited: false
+body:
+People, please stop using bitcoin as some sort of safety nets of your lives.  
+We're gonna see a number of suicides because of this
+
+comment: p1qe4en
+parent: t1_p1le9im
+author: captn03
+created_utc: 1785878911
+edited: false
+body:
+I highly doubt its an insider. I hear anyone could have reviewed their code in the last 4 years, why wasnt this revealed by the community ? This seems like a massive miss in their development process. No dedicated security or QA team reviewing the code, how can that happen in a company that specializes in hardware technology. 
+
+comment: p1qh4to
+parent: t3_1vewzn4
+author: Blake_a12
+created_utc: 1785879749
+edited: false
+body:
+Decentralized? They’re a company who held your keys, right? And now can be held liable, too 
+
 more-stub: parent t1_p1l2enr count 0
