--- reddit-0-7-btc-drained@20260804T154435Z
+++ reddit-0-7-btc-drained@20260804T161529Z
@@ -2441,4 +2441,84 @@
 body:
 And also, to complicate things more often than not scammers claim to be white hat hackers when they have no real ability to recover funds/assets and are really just acting out a secondary scam to victims of crime and other scams. They do this by identifying victims, then offering their services for a fee, then another fee, then another fee... until the victim finally (hopefully) gives up.
 
+comment: p1o6ri1
+parent: t1_p1nubop
+author: H3adshotfox77
+created_utc: 1785858618
+edited: false
+body:
+Some ai models hallucinate bugs and flaws and if you know nothing about code you won't know it hasn't found a real bug.
+
+Auditing with ai needs a human to verify the bug to he effective.
+
+Chatgpt is fairly bad at finding coding bugs compared to Claude which generally does better at it.
+
+But the process still requires human intuition and the ability to prompt it properly as well as audit the data after the ai.
+
+comment: p1o8288
+parent: t1_p1lftl8
+author: Fil3toFishy69
+created_utc: 1785858947
+edited: false
+body:
+Getting fucked daily. She loves cock.
+
+comment: p1o86dd
+parent: t1_p1kwujo
+author: Fil3toFishy69
+created_utc: 1785858975
+edited: false
+body:
+T R A D E R
+
+comment: p1o8ffn
+parent: t1_p1lcgcl
+author: Fil3toFishy69
+created_utc: 1785859039
+edited: false
+body:
+100% fucking peons 
+
+comment: p1o8qdy
+parent: t1_p1l5w1n
+author: Fil3toFishy69
+created_utc: 1785859115
+edited: false
+body:
+Hope he loses his house and car next. Really put shit into perspective.
+
+comment: p1o8t2b
+parent: t1_p1kktoe
+author: Fil3toFishy69
+created_utc: 1785859134
+edited: false
+body:
+😆 🤣 😂 
+
+comment: p1o9k4h
+parent: t1_p1lxolw
+author: sluuuurp
+created_utc: 1785859325
+edited: false
+body:
+There’s no way to prove you have the address of someone else has the private key right? I guess you could try to prove that you sent money from an exchange to this address and therefore it must be yours.
+
+comment: p1oa2jb
+parent: t1_p1km2f8
+author: sluuuurp
+created_utc: 1785859457
+edited: false
+body:
+No, I couldn’t be in their shoes, and they did do something wrong. I generated the randomness with dice like everyone should.
+
+I definitely have empathy though that they didn’t realize they had done something wrong. I admit there could be something I’ve done wrong without realizing it, in that sense I could be in their shoes one day.
+
+comment: p1obzo4
+parent: t1_p1kqq5c
+author: TheDisapprovingBrit
+created_utc: 1785859941
+edited: false
+body:
+And for the scale involved here, "most of the $$$" is still probably less than 1% of the actual money stolen. There's no way Coldcard can afford to reimburse anywhere even close to this, they're just going to go broke and fold the company. 
+
 more-stub: parent t1_p1l2enr count 0
