--- reddit-0-7-btc-drained@20260804T204515Z
+++ reddit-0-7-btc-drained@20260804T211532Z
@@ -2692,4 +2692,38 @@
 body:
 I never got the benefit of cold wallets. To me it’s more secure to generate it yourself and take care of the private key how ever you want. Why do you need hardware?
 
+comment: p1q6bh4
+parent: t3_1vewzn4
+author: ContentBlackberry0
+created_utc: 1785876806
+edited: false
+body:
+Why do people continue to use sparrow with all the fake apps and stuff that they have copying it, Makes no sense. I would personally stay away from all of those wallets. 
+
+comment: p1q7mov
+parent: t1_p1n663a
+author: lubdeptrai
+created_utc: 1785877156
+edited: false
+body:
+Both Trezor and Ledger have EAL certificates and have already been audited by professionals.
+
+Also, I believe that if anything happens, the company will take responsibility properly. Not like the shitty coinkite.
+
+comment: p1q8boy
+parent: t3_1vewzn4
+author: Not-the-best-name
+created_utc: 1785877344
+edited: false
+body:
+This is why we use banks that need to be insured and obey your country's laws. Its really sucks.
+
+comment: p1q97ec
+parent: t3_1vewzn4
+author: MilkNutty
+created_utc: 1785877584
+edited: false
+body:
+This is why I use IBIT etf for my bitcoin, so sorry and hope your 2.0 investments work out 
+
 more-stub: parent t1_p1l2enr count 0
