--- reddit-0-7-btc-drained@20260804T191410Z
+++ reddit-0-7-btc-drained@20260804T194517Z
@@ -2641,4 +2641,22 @@
 body:
 Are you a total buffoon? What’s to stop me from generating a seed that belongs to someone else through the same attack vector, loading it onto a coldcard, and then pretending that I was the original owner when asking for funds to be returned?
 
+comment: p1pm817
+parent: t1_p1ox6pj
+author: Novice89
+created_utc: 1785871491
+edited: false
+body:
+Oh yeah, no it’s not happening. I was just saying even the best dream scenario people who lost out will still lose
+
+comment: p1pnpne
+parent: t1_p1phujn
+author: shoebertdoubert
+created_utc: 1785871882
+edited: false
+body:
+you're an idіоt lol
+
+go ahead and try 
+
 more-stub: parent t1_p1l2enr count 0
