--- reddit-0-7-btc-drained@20260804T092531Z
+++ reddit-0-7-btc-drained@20260804T134325Z
@@ -1888,11 +1888,11 @@
 parent: t1_p1kzp55
 author: opossum_cz
 created_utc: 1785831267
-edited: false
+edited: 1785838860
 body:
 It was not compiler issue.
 
-Somebody (cough Doc Hex cough) was check if the macro was defined instead of macro being set to 1.
+Somebody (cough Doc Hex cough) was checking if the macro was defined instead of macro being set to 1.
 
 `#define MICROPY_HW_ENABLE_RNG (0)`
 
@@ -1976,4 +1976,375 @@
 body:
 OP, I sent you a DM 
 
+comment: p1m8tqr
+parent: t1_p1l3eaf
+author: keypusher
+created_utc: 1785836080
+edited: false
+body:
+when you are dealing with cryptography, there are very few things more important than the RNG.  this wasn’t a subtle off by one error deep in the code.  not saying it was intentional, but it was at least negligent 
+
+comment: p1m9t0g
+parent: t1_p1kz140
+author: Background_Pause34
+created_utc: 1785836557
+edited: false
+body:
+Ok fair enough
+
+comment: p1macls
+parent: t3_1vewzn4
+author: iloverunning11
+created_utc: 1785836820
+edited: false
+body:
+Fuckcoldcard tab 😀that’s the spirit. I’m so sorry for your loss, hope you can recover at least some of your funds.. Good luck! 
+
+comment: p1mae37
+parent: t3_1vewzn4
+author: Acceptable-Ant-9231
+created_utc: 1785836838
+edited: false
+body:
+how did it happen?
+
+comment: p1masnz
+parent: t1_p1kwn4f
+author: monxas
+created_utc: 1785837029
+edited: false
+body:
+You say that like you go through life trusting nobody. Thats impossible. Do you test your food for poison every time you eat? To you trust the tests you use to check if there’s poison on your food? Do you trust your car or you had it dismantled piece by piece to make sure they’re all good? And do you trust who put the car back together? 
+
+comment: p1mbc6t
+parent: t3_1vewzn4
+author: ivanjurman
+created_utc: 1785837289
+edited: false
+body:
+Ummm it was never the best, Trezor was and always will be the best, most trusted hardware wallet
+
+comment: p1mbgw0
+parent: t3_1vewzn4
+author: Aeschbacher15
+created_utc: 1785837351
+edited: false
+body:
+This is turning out to be a real scandal lol
+
+comment: p1mbtyp
+parent: t3_1vewzn4
+author: Bionic_Push
+created_utc: 1785837525
+edited: false
+body:
+Did you have a passphrase? Every single post that claims to have lost with coldcard never answers when i ask this for some reason.
+
+comment: p1mecqh
+parent: t1_p1masnz
+author: DyehuthyTV
+created_utc: 1785838677
+edited: false
+body:
+If you get food poisoning, it’s not as expensive as losing your Bitcoins (Self-custody your wealth, 'life savings') lol
+
+I think the immense laziness that people like you have doesn’t even allow them to put their priorities in order, regarding which things actually require you to do a minimum amount of “checks” (verify!) and research before using them (devices, wallets, exchanges, etc)
+
+comment: p1meka3
+parent: t3_1vewzn4
+author: BTC_90210
+created_utc: 1785838768
+edited: false
+body:
+Your keys, their coins 
+
+comment: p1melmy
+parent: t1_p1lu6ya
+author: BTC_90210
+created_utc: 1785838785
+edited: false
+body:
+😂
+
+comment: p1mfvde
+parent: t3_1vewzn4
+author: magic-shroomman
+created_utc: 1785839340
+edited: false
+body:
+lmao sooo glad I went with Trezor
+
+comment: p1mhfa8
+parent: t1_p1lryp2
+author: MiguelLancaster
+created_utc: 1785840002
+edited: false
+body:
+I don't know how many coldcards exist in the world, but I highly doubt it's anywhere even remotely close to a million
+
+comment: p1mhjbh
+parent: t1_p1mbtyp
+author: MiguelLancaster
+created_utc: 1785840050
+edited: false
+body:
+why are you asking a question that you already know the answer to?
+
+comment: p1mhofw
+parent: t3_1vewzn4
+author: OscarCorterz_20
+created_utc: 1785840110
+edited: false
+body:
+Fuck them a hundred times
+
+comment: p1mhp7o
+parent: t1_p1mhjbh
+author: Bionic_Push
+created_utc: 1785840120
+edited: false
+body:
+i don't know the answer. And also a lot of these posts seem fake. If they want to help the community why don't they at least share what their setup was?
+
+comment: p1mi55n
+parent: t1_p1mhp7o
+author: MiguelLancaster
+created_utc: 1785840307
+edited: false
+body:
+read any single one of the quite numerous posts/comment threads/articles about the attack and you'll know what their setup was
+
+no, there wasn't a passphrase
+
+comment: p1milcr
+parent: t1_p1mi55n
+author: Bionic_Push
+created_utc: 1785840494
+edited: false
+body:
+I did read it and there's a post here in reddit claiming that a wallet with a two word passphrase was hacked, you can look it up yourself. I want to read the answer from OP not from some other random user.
+
+comment: p1mj0wv
+parent: t1_p1kzlzx
+author: lsumoose
+created_utc: 1785840672
+edited: false
+body:
+They did it with offline models. 
+
+comment: p1mkuwp
+parent: t3_1vewzn4
+author: SkidMarkShark
+created_utc: 1785841415
+edited: false
+body:
+I pray you are fake, but when it comes down to it you should have done more research and learned how to actually store your BTC. 
+
+Never let the wallet generate your seed words.
+
+Use a non BIP 39 uncommon words passphrase with Upper and Lower case letters with numbers and characters.
+
+The Coldcard or any wallet is just a keysigner. There is nothing wrong with the device. I'm sorry for your loss, but as soon as you realize your mistake the faster you'll move on
+
+comment: p1mlkq7
+parent: t3_1vewzn4
+author: pplgg4445
+created_utc: 1785841702
+edited: false
+body:
+The dream of bitcoin was mass adoption. Be your own bank. And while banks do get robbed mass adoption doesn’t mean everyone needs to be a cypherpunk security expert to use the technology safely.  Wallet vendors took on much of that role and charged money for it.  Coinkit needs to do better here in terms of restitution 
+
+comment: p1mlphe
+parent: t1_p1lgvtk
+author: GiveMeYourSmile
+created_utc: 1785841754
+edited: false
+body:
+OMG, guys, if you trust your safety to AI, you are even bigger idiots than the people who trust people. AI doesn't make you a security expert if you don't understand anything about it and can't find a bug yourself - AI will simply create the illusion of an audit - it will find you a lot of bugs and flaws that are not a problem, among which you won't even be able to identify the real one. I'm not saying AI is useless for security auditing, I'm saying it's useless for the average person, just like open source code.
+
+comment: p1mltcw
+parent: t3_1vewzn4
+author: pplgg4445
+created_utc: 1785841797
+edited: false
+body:
+I wonder if Claude actually took the bitcoin and now controls the addresses involved. lol.   Skynet begins 
+
+comment: p1mmci6
+parent: t3_1vewzn4
+author: SoggyGrayDuck
+created_utc: 1785842007
+edited: false
+body:
+Yeah it really is to the point you should hold no more than 10% on each hardware wallet just to be safe. 
+
+Someone please convince me why people will still use hardware wallets in a generation or two? Especially if people get tax discounts by using things like holding in a Roth etf? 
+
+They're going to turn it into fiat, they Probably already have full control over the value but I say we have to wait until the global stage clears itself up a bit more. 
+
+comment: p1mpvih
+parent: t3_1vewzn4
+author: iWearSkinyTies
+created_utc: 1785843353
+edited: false
+body:
+If you get a lawyer to sue them yourself, you will get more than joining a class action.
+
+comment: p1mrqcx
+parent: t3_1vewzn4
+author: comp21
+created_utc: 1785844020
+edited: false
+body:
+We all need to collectively learn the following truth: 
+
+If someone screams loudly about themselves, they're not to be trusted and probably massive sacks of shit.
+
+The crap pile from Celsius, Voyager, coldcard, Andrew Tate, most of our gov, TV preachers...
+
+comment: p1mryub
+parent: t3_1vewzn4
+author: McBurger
+created_utc: 1785844104
+edited: false
+body:
+*“And since it’s decentralized, there’s no recourse for getting any of it back.  It’s gone.”*
+
+*“Where’s the class action lawsuit? How do I sign up?”*
+
+There’s your recourse, my friend.
+
+comment: p1ms90d
+parent: t1_p1lb6k1
+author: Satelite_of_Love
+created_utc: 1785844206
+edited: false
+body:
+Not a bad point and maybe it just me that's changed. Im actually a member of the mt got class action and remembered that well... but this one hits different. 
+
+I dunno could just be my risk tolerance for shenanigans has lowered as ive gotten older or my personal use case for crypto has changed since I dont look at it as a currency at ALL anymore where in the distant past I would. 
+
+Not exactly sure. What I know for SURE it this sucks for those invovled. 
+
+comment: p1msd0e
+parent: t1_p1mlphe
+author: uniicorn77
+created_utc: 1785844246
+edited: false
+body:
+right you are correct, thanks for correcting me 
+
+comment: p1msrpt
+parent: t1_p1koz0y
+author: pjb1999
+created_utc: 1785844391
+edited: false
+body:
+Yeah this whole thing has sort of turned me off to crypto maybe for good. I started buying BTC and some other coins nearly 10 years ago but Im seriously thinking about selling at least half of everything I have at the next BTC ATH and just dumping that money into my VOO ETF. The stress of self custody is just too much honestly. Especially after something like this happens. 
+
+comment: p1mtrep
+parent: t1_p1krk6p
+author: pjb1999
+created_utc: 1785844739
+edited: false
+body:
+Yep I just made a comment about this. I'm likely selling at least half of my BTC at the next ATH and putting that money into my ETFs. 
+
+comment: p1muus2
+parent: t1_p1l2430
+author: stranix13
+created_utc: 1785845124
+edited: false
+body:
+If its government/conspiracy related its a scheme to help push etfs and get people away from self custody, and for that its not so much about the total amount stolen but the public perception 
+
+comment: p1n2cjp
+parent: t3_1vewzn4
+author: Huge-Artichoke-1376
+created_utc: 1785847571
+edited: false
+body:
+I won’t lie, being lazy and just keeping it on River because I didn’t trust a wallet made sense. 
+
+comment: p1n2qyd
+parent: t3_1vewzn4
+author: LamarJacksonIsMyHero
+created_utc: 1785847697
+edited: false
+body:
+Start buying VOO
+
+comment: p1n41l1
+parent: t1_p1ku4tc
+author: RefrigeratorRough727
+created_utc: 1785848095
+edited: false
+body:
+Sorry for you man. Through which exchange did you do dca every week for 100? 
+
+comment: p1n5s4f
+parent: t3_1vewzn4
+author: DarthBen_in_Chicago
+created_utc: 1785848620
+edited: false
+body:
+I’m sorry for your loss. I can feel your anger and frustration. Just know you are not alone which I realize doesn’t solve this for you. 
+
+comment: p1n663a
+parent: t1_p1lfpdu
+author: hrad95
+created_utc: 1785848738
+edited: false
+body:
+I use Trezor. Since their code is open-source, shouldn't we, the community, be using AI to audit it?
+
+comment: p1n6hhq
+parent: t3_1vewzn4
+author: nakedlunch2
+created_utc: 1785848834
+edited: false
+body:
+Could have saved a lot of money by switching to Geico
+
+comment: p1n8npl
+parent: t1_p1kskpc
+author: ZeroDollars
+created_utc: 1785849473
+edited: false
+body:
+A typical business of this size wouldn't have the assets to cover a tiny fraction of the losses.  And their IP and inventory is worthless now.  They will declare bankruptcy and customers will be left holding the bag. 
+
+Only way it could turn out different is if, given the industry, they happened to be holding a significant cache of bitcoin at the legal entity level and it could be liquidated to pay damages.  Very improbable though.  Would mean the owners were multimillionaires many times over and still were f'ing around with this little e-commerce business.
+
+comment: p1n9a1l
+parent: t1_p1m1t6n
+author: notanigaa
+created_utc: 1785849653
+edited: false
+body:
+13 year pro trader ig
+
+comment: p1n9rkf
+parent: t1_p1kyw0b
+author: ctzn2000
+created_utc: 1785849794
+edited: false
+body:
+Look at what Coldcard currently claims below :)
+
+"COLDCARD is a Bitcoin-only hardware wallet made by Coinkite since 2017. It protects private keys with dual secure elements, supports air-gapped signing, and runs open-source firmware you can verify yourself. It can also generate and vault BIP-39/BIP-85 seeds, recovery phrases, passwords, and private notes for other wallets. Unlike Trezor and Ledger, COLDCARD supports only Bitcoin to reduce the attack surface. No companion app, no cloud account."
+
+**TRUSTED BY BITCOINERS, EDUCATORS, AND INDEPENDENT REVIEWERS**
+
+[**“Passed all 10 tests”**](https://walletscrutiny.com/hardware/coldcardQ1/)  
+[**WALLETSCRUTINY**](https://walletscrutiny.com/hardware/coldcardQ1/)
+
+comment: p1nc7gt
+parent: t3_1vewzn4
+author: Cerlog
+created_utc: 1785850494
+edited: false
+body:
+Use Trezor. 
+
 more-stub: parent t1_p1l2enr count 0
