--- reddit-0-7-btc-drained@20260804T164606Z
+++ reddit-0-7-btc-drained@20260804T171418Z
@@ -2561,4 +2561,28 @@
 body:
 I wonder how that bug left undetected for 5 years on an open source repo
 
+comment: p1olqph
+parent: t1_p1kskpc
+author: striata
+created_utc: 1785862381
+edited: false
+body:
+There is literally no way for the original owners to prove ownership, so of course they cannot be reimbursed. 
+
+comment: p1olwh3
+parent: t3_1vewzn4
+author: WarPlanMango
+created_utc: 1785862421
+edited: false
+body:
+Man this really sucks. Why would they even release a product that they would say is safe if this can happen?? The company needs to be held accountable for even implementing and selling this unsecure device
+
+comment: p1omma9
+parent: t1_p1ly94p
+author: striata
+created_utc: 1785862599
+edited: false
+body:
+You can recover any compromised seed to the device.
+
 more-stub: parent t1_p1l2enr count 0
