--- reddit-0-7-btc-drained@20260804T224631Z
+++ reddit-0-7-btc-drained@20260804T234555Z
@@ -1723,6 +1723,16 @@
 body:
 As a Celsius survivor, I can understand how you feel. What's appalling is that I was speculating, believing in something too good to be true, while using "cold" case was probably the most diligent crypto activity possible. Seeing this really makes me hesitant to buy back Bitcoin with the fraction of my money we got back. Hope you all get some justice.
 
+comment: p1loabs
+parent: t3_1vewzn4
+author: Knarz97
+created_utc: 1785825867
+edited: false
+body:
+Do your son a favor and save for your son’s college outside of an extremely speculative asset. This is not Bitcoin hate. I’d be saying the same thing if your savings were in Gold Bars or NVIDIA stock as well. College Savings is perfectly fine in a HYSA, CDs, Bonds, or if you must do something like ETFs in a 529 account. 
+
+It’s irresponsible to literally gamble with his future.  
+
 comment: p1lph9t
 parent: t1_p1krk6p
 author: AppropriateCan4064
@@ -2084,11 +2094,11 @@
 
 comment: p1mhjbh
 parent: t1_p1mbtyp
-author: MiguelLancaster
+author: [deleted]
 created_utc: 1785840050
-edited: false
-body:
-why are you asking a question that you already know the answer to?
+edited: 1785885347
+body:
+[deleted]
 
 comment: p1mhofw
 parent: t3_1vewzn4
@@ -2108,13 +2118,11 @@
 
 comment: p1mi55n
 parent: t1_p1mhp7o
-author: MiguelLancaster
+author: [deleted]
 created_utc: 1785840307
-edited: false
-body:
-read any single one of the quite numerous posts/comment threads/articles about the attack and you'll know what their setup was
-
-no, there wasn't a passphrase
+edited: 1785885337
+body:
+[deleted]
 
 comment: p1milcr
 parent: t1_p1mi55n
@@ -2795,4 +2803,20 @@
 body:
 none of these companies could ever afford to reimburse the total value stored on their hardware. it’s just not possible economically 
 
+comment: p1qwm3g
+parent: t3_1vewzn4
+author: ElRojo22
+created_utc: 1785884268
+edited: false
+body:
+Could you tell if the seed was created with RNG or you make roll dice? Also if you have a passphrase on it?
+
+comment: p1r0i0p
+parent: t3_1vewzn4
+author: I_iron_my_t-shirts
+created_utc: 1785885479
+edited: false
+body:
+Between the “not your keys, not your coin” crowd and the Coinbase haters, I considered moving my crypto to a cold wallet. But after this, I might as well leave them on an exchange. 
+
 more-stub: parent t1_p1l2enr count 0
