--- reddit-0-7-btc-drained@20260804T214524Z
+++ reddit-0-7-btc-drained@20260804T221546Z
@@ -2759,4 +2759,32 @@
 body:
 Decentralized? They’re a company who held your keys, right? And now can be held liable, too 
 
+comment: p1qiucm
+parent: t1_p1p3672
+author: Fil3toFishy69
+created_utc: 1785880227
+edited: false
+body:
+Just getting started my boy.
+
+comment: p1qnmv7
+parent: t1_p1n663a
+author: FauxReal
+created_utc: 1785881592
+edited: false
+body:
+People are already auditing their code.
+
+[https://sqmagazine.co.uk/trezor-safe-7-chip-vulnerability-security-audit/](https://sqmagazine.co.uk/trezor-safe-7-chip-vulnerability-security-audit/)
+
+[https://www.nathanfox.net/p/trezor-seed-phrase-generation-security](https://www.nathanfox.net/p/trezor-seed-phrase-generation-security)
+
+  
+And at least one redditor trying to get a commuinity effort to audit general Bitcoin ecosystem code.
+
+[https://www.reddit.com/r/Bitcoin/comments/1vejkbl/help\_review\_bitcoin\_ecosystem\_code/](https://www.reddit.com/r/Bitcoin/comments/1vejkbl/help_review_bitcoin_ecosystem_code/)
+
+  
+I'm sure there are other efforts underway in light of the ColdCard hack.
+
 more-stub: parent t1_p1l2enr count 0
