--- reddit-0-7-btc-drained@20260804T194517Z
+++ reddit-0-7-btc-drained@20260804T201541Z
@@ -2659,4 +2659,29 @@
 
 go ahead and try 
 
+comment: p1puij8
+parent: t3_1vewzn4
+author: BlondDeutcher
+created_utc: 1785873672
+edited: false
+body:
+Not “like” a loser. 
+
+comment: p1pxfrn
+parent: t3_1vewzn4
+author: CowDogRatGoose
+created_utc: 1785874450
+edited: false
+body:
+Not to rub salt in the wound, and I'm sorry this happened to you. Totally sucky. But why didn't you use dice?
+
+comment: p1pxpy6
+parent: t1_p1kktoe
+author: CowDogRatGoose
+created_utc: 1785874524
+edited: false
+body:
+Lol. Of course they did. 
+They also encouraged people to use dice when generating their phrase.
+
 more-stub: parent t1_p1l2enr count 0
