--- reddit-0-7-btc-drained@20260804T221546Z
+++ reddit-0-7-btc-drained@20260804T224631Z
@@ -2787,4 +2787,12 @@
   
 I'm sure there are other efforts underway in light of the ColdCard hack.
 
+comment: p1qtvl5
+parent: t1_p1q7mov
+author: davvblack
+created_utc: 1785883438
+edited: false
+body:
+none of these companies could ever afford to reimburse the total value stored on their hardware. it’s just not possible economically 
+
 more-stub: parent t1_p1l2enr count 0
