--- reddit-0-7-btc-drained@20260804T184503Z
+++ reddit-0-7-btc-drained@20260804T191410Z
@@ -2633,4 +2633,12 @@
 body:
 Yeah they'll use fully audited AI to do it as well 🙄
 
+comment: p1phujn
+parent: t1_p1ly94p
+author: FirstAmendmentIsDead
+created_utc: 1785870362
+edited: false
+body:
+Are you a total buffoon? What’s to stop me from generating a seed that belongs to someone else through the same attack vector, loading it onto a coldcard, and then pretending that I was the original owner when asking for funds to be returned?
+
 more-stub: parent t1_p1l2enr count 0
