--- reddit-0-7-btc-drained@20260804T171418Z
+++ reddit-0-7-btc-drained@20260804T174435Z
@@ -2585,4 +2585,36 @@
 body:
 You can recover any compromised seed to the device.
 
+comment: p1ovr4q
+parent: t1_p1o9k4h
+author: peterwilli
+created_utc: 1785864860
+edited: false
+body:
+Yeah I think that proof of purchase + showing some form of control of the first address that sent money to the hacked one on ColdCard (i.e. sending money from the same exchange again or sending a new tx if the money came from an existing wallet) is the most reliable way though still not ideal
+
+comment: p1ow8kf
+parent: t3_1vewzn4
+author: Sensitive-Rule-5563
+created_utc: 1785864979
+edited: false
+body:
+Reason number 1 million I will only hold my money FDIC insured. If it’s an amount I can’t fathom losing I’m not trusting it to one of these random startups. For all we know it’s an inside job. 
+
+comment: p1owqcb
+parent: t3_1vewzn4
+author: KewlKicks_
+created_utc: 1785865100
+edited: false
+body:
+iToddler btfo
+
+comment: p1ox6pj
+parent: t1_p1ku1ci
+author: AngusDagnabit
+created_utc: 1785865212
+edited: false
+body:
+I find it hard to believe the company would survive this and have enough assets to pay everyone back, even at the current value or less.
+
 more-stub: parent t1_p1l2enr count 0
