--- reddit-0-7-btc-drained@20260804T161529Z
+++ reddit-0-7-btc-drained@20260804T164606Z
@@ -2521,4 +2521,44 @@
 body:
 And for the scale involved here, "most of the $$$" is still probably less than 1% of the actual money stolen. There's no way Coldcard can afford to reimburse anywhere even close to this, they're just going to go broke and fold the company. 
 
+comment: p1oe8qn
+parent: t1_p1o8288
+author: generichandel
+created_utc: 1785860510
+edited: false
+body:
+It's always good to have interests in common with your spouse. 
+
+comment: p1ofpb1
+parent: t1_p1oe8qn
+author: Fil3toFishy69
+created_utc: 1785860876
+edited: false
+body:
+100% that's the motto of life.
+
+comment: p1oirsx
+parent: t3_1vewzn4
+author: eimattz
+created_utc: 1785861645
+edited: false
+body:
+what was your passphrase?
+
+comment: p1ojp09
+parent: t1_p1lattd
+author: RandomKnifeBro
+created_utc: 1785861875
+edited: false
+body:
+So even less chance anything is done.
+
+comment: p1ojxwy
+parent: t3_1vewzn4
+author: Firm-Ad-2446
+created_utc: 1785861936
+edited: false
+body:
+I wonder how that bug left undetected for 5 years on an open source repo
+
 more-stub: parent t1_p1l2enr count 0
