--- optech-416@20260801T025320Z
+++ optech-416@20260805T210513Z
@@ -261,12 +261,6 @@
 Newsletter #409 for an LDK workaround that addresses LND’s
 behavior of receiving but not forwarding onion messages from non-channel
 peers.
-Want more?
-For more discussion about the topics mentioned in this newsletter, join us for
-the weekly Bitcoin Optech Recap on Riverside.fm at
-16:30 UTC on August 4. The
-discussion is also recorded and will be available from our podcasts
-page.
 Subscribe to our newsletter
 Bitcoin Optech
 Bitcoin Optech
