--- reddit-future-of-bitcoin-debate@20260805T142733Z
+++ reddit-future-of-bitcoin-debate@20260805T210258Z
@@ -4091,14 +4091,6 @@
 body:
 https://www.qastlewallet.com
 
-comment: p1p489j
-parent: t3_1vdds1l
-author: Moonrunner04
-created_utc: 1785866946
-edited: false
-body:
-Everyone needs to check out real-random. This whole thing would have been prevented if they were using them 
-
 comment: p1pi7af
 parent: t3_1vdds1l
 author: marshyr3d1and
