--- reddit-future-of-bitcoin-debate@20260804T201816Z
+++ reddit-future-of-bitcoin-debate@20260804T221822Z
@@ -2691,14 +2691,6 @@
 edited: false
 body:
 I didn't quit, I'm just not playing anymore
-
-comment: p1b731i
-parent: t3_1vdds1l
-author: [deleted]
-created_utc: 1785698954
-edited: false
-body:
-I think it's interesting this is happening after US government moves it's BTC to exchange and during time Clarity Act is being pushed. Just interesting..
 
 comment: p1b95iy
 parent: t1_p19qh4q
