--- reddit-future-of-bitcoin-debate@20260808T020120Z
+++ reddit-future-of-bitcoin-debate@20260808T083403Z
@@ -314,16 +314,6 @@
 
 - the trade-off between full self custody/ more risk and third party custody / more trust is not a new topic, but there haven't been good advances that drive adoption. The Coldcard case sets us back substantially 
 
-comment: p18oxtw
-parent: t1_p18e1as
-author: the_bitcoin_kid
-created_utc: 1785672638
-edited: false
-body:
-There isn't really a passphrase bug that could cause your coins to get stolen.
-
-The passphrase _adds_ security to your seed, it doesn't take it away.
-
 comment: p18qg22
 parent: t1_p189e6p
 author: JohnnyTreemain
@@ -571,16 +561,6 @@
 edited: false
 body:
 Bitkey is one 
-
-comment: p191bju
-parent: t3_1vdds1l
-author: arcrad
-created_utc: 1785677063
-edited: false
-body:
-This hack doesn't change a goddamned thing about Bitcoin. 
-
-Y'all are sensationalists 
 
 comment: p191ccu
 parent: t1_p18hbu5
@@ -1578,22 +1558,6 @@
 
 Choose your poison. It’s still better than any fiat currency where you can only choose number 3.
 
-comment: p1a6exs
-parent: t1_p19p1g4
-author: binary_quasar
-created_utc: 1785689027
-edited: false
-body:
->Coinbase doesn't provide proof of reserves. It's only a matter of time before they pull an FTX or an MtGox.
-
-Laughably braindead take. The largest asset managers in the world, who have a fiduciary responsibility to every single person that invests with them, chose Coinbase to custody their crypto assets. 
-
-Do you think BlackRock and their ilk would make that decision without consistent access to Coinbase's financials? Do you think they would choose a custodian lightly, knowing that even if Coinbase fucks up, BlackRock would *still have a fiduciary responsibility to their clients and would be responsible for customer deposits?*
-
-They did an elaborate audit of Coinbase's books before making that decision and chose Coinbase precisely because their company was financially healthy and have top-tier asset security practices. 
-
-Furthermore, Coinbase is a publicly traded company on the NYSE, meaning their books are public as well, dumbass.
-
 comment: p1a6l3g
 parent: t1_p19x63o
 author: gtwooh
@@ -1992,16 +1956,6 @@
 body:
 That goes for most everything in life though: your car, your utilities, your food, etc etc. Bitcoin provides the ability for truly sovereign money, but the implementation ultimately depends on some level of infrastructure. Unless you are a cyber security and hardware engineer expert that can build a wallet entirely on their own, you are forced to depend on some level of others. That is the flaw here, and exactly why we need better consumer protection (e.g. insurance). The average person shouldn't be expected to know what multi signature or RNG entropy is. This is solvable, but we are not there yet.
 
-comment: p1ah949
-parent: t1_p1a6exs
-author: KontoOficjalneMR
-created_utc: 1785691949
-edited: false
-body:
-> The largest asset managers in the world
-
-Bernie Madoff. Lehman Brothers. AIG
-
 comment: p1ahjja
 parent: t1_p196tc1
 author: LPC_Rebuilder
@@ -2262,6 +2216,14 @@
 I don't think this has the potential to crack the shiny surface, I think this situation more so is showing us what the true surface looks like if you get what I'm saying lol. Someone told us the surface is super clean and shiny and we just trusted it, but the reality is the surface may be a bit rough and you need to be more vigilant when it comes to securing your money.
 
 This whole situation sucks and I agree it's a blow to alot of hardcore bitcoiners but at the end of the day Bitcoin doesn't care and it still functions the same way and the network is just as secure as it was before this event. 
+
+comment: p1as8fe
+parent: t1_p1aj3e4
+author: confusedguy1212
+created_utc: 1785694876
+edited: false
+body:
+Explain please. I’m interested in this take. 
 
 comment: p1asd1x
 parent: t3_1vdds1l
@@ -2598,6 +2560,14 @@
 body:
 If this is true, then holy crap……
 
+comment: p1bb9en
+parent: t1_p1as8fe
+author: Possible-Mud-1380
+created_utc: 1785700120
+edited: false
+body:
+It's not really much different then the current system, just faster and cheaper transactions on the blockchain. Control will be mostly centralized between say a number of financial institutions, corps, and/or nation states. Look at the Unit by brics. Commodity backed stablecoin basically.
+
 comment: p1bc4f6
 parent: t3_1vdds1l
 author: moonkingdome
@@ -2698,6 +2668,14 @@
 
 Hope that helps 
 
+comment: p1bkkvq
+parent: t1_p1bb9en
+author: confusedguy1212
+created_utc: 1785702741
+edited: false
+body:
+So you’re saying the whole crypto experiment literally yielded not more than cheaper financial infrastructure for the old and tested currencies? 
+
 comment: p1blqau
 parent: t1_p18hbu5
 author: coffeelover9457
@@ -2715,6 +2693,14 @@
 edited: false
 body:
 That’s a great point. 
+
+comment: p1bpoe2
+parent: t1_p1bkkvq
+author: Possible-Mud-1380
+created_utc: 1785704199
+edited: false
+body:
+Pretty close. There will be new currency though, backed by other currencies and commodities. But your average person needs centralized institutional backing and insurance. And end of the day they are better keeping gold in safe for savings, then using a technology whose encryption protocols and safeguards could become obsolete without warning. It was a good first run. Maybe we'll get a second go at it in the future. That's my take. Could be wrong of course.
 
 comment: p1bposd
 parent: t3_1vdds1l
@@ -3078,18 +3064,6 @@
 body:
 Bitcoin has failed at everything it had hoped to achieve. 
 
-comment: p1cn25f
-parent: t1_p1a6exs
-author: halt_spell
-created_utc: 1785714495
-edited: false
-body:
-> Do you think BlackRock and their ilk would make that decision without consistent access to Coinbase's financials? Do you think they would choose a custodian lightly, knowing that even if Coinbase fucks up, BlackRock would still have a fiduciary responsibility to their clients and would be responsible for customer deposits?
-
-It's all people at the end of the day. And yes people fuck up.
-
-Let me ask you a question, since you have all this confidence, when you say "consistent access" what does that look like? Perhaps a _proof of reserves_?
-
 comment: p1cn3su
 parent: t1_p19xwck
 author: Charming-Designer944
@@ -3920,6 +3894,16 @@
 edited: false
 body:
 I've seen several comments on this post blaming the users
+
+comment: p1l2qh7
+parent: t1_p1bkkvq
+author: Kinslayer817
+created_utc: 1785816226
+edited: false
+body:
+The infrastructure isn't even cheaper than existing systems. BTC costs an insane amount of money and energy to run and the only reason people are paying that cost is that they're making tons of money off of the people using it 
+
+All that crypto accomplished is creating a new way to scam and steal from people
 
 comment: p1l60ir
 parent: t3_1vdds1l
@@ -4181,14 +4165,12 @@
 
 more-stub: parent t1_p1b161g count <live-count>
 
-more-stub: parent t1_p1a6exs count <live-count>
+more-stub: parent t1_p19p1g4 count <live-count>
 
 more-stub: parent t1_p18x932 count <live-count>
 
 more-stub: parent t1_p19u32s count <live-count>
 
-more-stub: parent t1_p1aj3e4 count <live-count>
-
 more-stub: parent t1_p18djp7 count <live-count>
 
 more-stub: parent t1_p189e6p count <live-count>
@@ -4209,12 +4191,12 @@
 
 more-stub: parent t1_p18lzey count <live-count>
 
+more-stub: parent t1_p18oefs count <live-count>
+
 more-stub: parent t1_p1a23gf count <live-count>
 
 more-stub: parent t1_p19xg7j count <live-count>
 
-more-stub: parent t1_p18oefs count <live-count>
-
 more-stub: parent t1_p188oc0 count <live-count>
 
 more-stub: parent t1_p18cebt count <live-count>
@@ -4223,7 +4205,7 @@
 
 more-stub: parent t1_p18rx5l count <live-count>
 
-more-stub: parent t1_p18oxtw count <live-count>
+more-stub: parent t1_p18e1as count <live-count>
 
 more-stub: parent t1_p18ctae count <live-count>
 
