--- reddit-future-of-bitcoin-debate@20260808T083403Z
+++ reddit-future-of-bitcoin-debate@20260808T213454Z
@@ -1429,14 +1429,6 @@
 edited: false
 body:
 This has nothing to do with Bitcoin. People can generate their own seed with sufficient entropy and not have to leave it to trust - if they choose. It’s just easier to assume the device is using a TRNG.
-
-comment: p1a1z9n
-parent: t1_p19vhma
-author: The25thUser
-created_utc: 1785687812
-edited: false
-body:
-I think it's this: https://www.reddit.com/r/ledgerwallet/s/gIdWMgCelJ
 
 comment: p1a20ry
 parent: t1_p19uizw
@@ -2798,11 +2790,11 @@
 
 comment: p1bwdnt
 parent: t3_1vdds1l
-author: thegunky72
+author: [deleted]
 created_utc: 1785706146
 edited: false
 body:
-right. exactly.  just like "a bank with bad security got robbed therefore all banks can't be trusted". also, cars will never replace horse and carriage
+[removed]
 
 comment: p1bwfvq
 parent: t3_1vdds1l
