--- reddit-future-of-bitcoin-debate@20260807T193054Z
+++ reddit-future-of-bitcoin-debate@20260808T020120Z
@@ -1833,14 +1833,6 @@
 edited: false
 body:
 It's the warm-up before quantum enters the room.
-
-comment: p1abbp5
-parent: t1_p1a8spe
-author: NiagaraBTC
-created_utc: 1785690358
-edited: false
-body:
-No, it wasn't turned off. It worked fine but wasn't being asked to participate in seed creation. I guess that's functionally the same thing. 
 
 comment: p1abg2k
 parent: t1_p189e6p
@@ -4179,6 +4171,14 @@
 body:
 So has many banks 🤷🏽‍♂️
 
+comment: p2ck51g
+parent: t1_p1d18sa
+author: Chickenjon
+created_utc: 1786138991
+edited: false
+body:
+Yeah I would have never thought about doing that
+
 more-stub: parent t1_p1b161g count <live-count>
 
 more-stub: parent t1_p1a6exs count <live-count>
@@ -4199,7 +4199,7 @@
 
 more-stub: parent t1_p19qh4q count <live-count>
 
-more-stub: parent t1_p1abbp5 count <live-count>
+more-stub: parent t1_p1a8spe count <live-count>
 
 more-stub: parent t1_p19nkiy count <live-count>
 
