--- reddit-future-of-bitcoin-debate@20260804T184745Z
+++ reddit-future-of-bitcoin-debate@20260804T191705Z
@@ -356,14 +356,6 @@
 
 - the trade-off between full self custody/ more risk and third party custody / more trust is not a new topic, but there haven't been good advances that drive adoption. The Coldcard case sets us back substantially 
 
-comment: p18om9k
-parent: t1_p18mgt8
-author: dvondurden
-created_utc: 1785672514
-edited: false
-body:
-Thank you, trying my best here to pour some oil into the fire
-
 comment: p18oxtw
 parent: t1_p18e1as
 author: the_bitcoin_kid
@@ -1209,14 +1201,6 @@
 user interface, sturdiness, open source code, aesthetics, independent and internal audits, resistance to attacks, abilities to use multiple numbers of seed phrases, theres plenty. 
 
 no idea why cold cards were considered the best, i never even considered them. it was always trezor. 
-
-comment: p19pzle
-parent: t1_p18mgt8
-author: cliff_smiff
-created_utc: 1785684505
-edited: false
-body:
-I'm gonna take this chance to say that FUD is a braindead term reflective of a braindead mentality. Fear? Uncertainty? Doubt? Yes these are real factors in life. Dismissing doubt is the absolute height of arrogance.
 
 comment: p19pzrr
 parent: t1_p19nlwt
@@ -4189,6 +4173,23 @@
 body:
 Everyone needs to check out real-random. This whole thing would have been prevented if they were using them 
 
+comment: p1pi7af
+parent: t3_1vdds1l
+author: marshyr3d1and
+created_utc: 1785870453
+edited: false
+body:
+Why the fuck can't we all just use stainless plates in multiple locations?
+Trust NO ONE!
+
+comment: p1piit3
+parent: t1_p1b1vie
+author: marshyr3d1and
+created_utc: 1785870535
+edited: false
+body:
+Err backed up in another location 🙄
+
 more-stub: parent t1_p1b161g count 3
 
 more-stub: parent t1_p1a6exs count 1
@@ -4249,4 +4250,6 @@
 
 more-stub: parent t1_p186jye count 2
 
+more-stub: parent t1_p18mgt8 count 2
+
 more-stub: parent t3_1vdds1l count 59
