--- reddit-future-of-bitcoin-debate@20260804T134608Z
+++ reddit-future-of-bitcoin-debate@20260804T161824Z
@@ -483,14 +483,6 @@
 edited: false
 body:
 No it doesn’t , coldcard had one line of shit code that fucked the whole thing up 
-
-comment: p18uww5
-parent: t1_p18tlp9
-author: dvondurden
-created_utc: 1785674859
-edited: false
-body:
-It's reflecting the space. A highly regarded and recommended product by many had faulty code for years for everyone to see, yet nobody noticed. Do we really lack self awareness so badly? I'm including myself in this
 
 comment: p18w5ya
 parent: t3_1vdds1l
@@ -4267,4 +4259,6 @@
 
 more-stub: parent t1_p18rw92 count 2
 
+more-stub: parent t1_p18tlp9 count 1
+
 more-stub: parent t3_1vdds1l count 54
