--- reddit-dice-rolls-trust@20260804T090131Z
+++ reddit-dice-rolls-trust@20260804T201457Z
@@ -258,3 +258,13 @@
 edited: false
 body:
 Same. I did 50 live dice rolls when I setup my CCQ. Then a strong passphrase as well. Then I have multiple wallets both single and multisig under that also using bip-85 so I think im safe, but I still feel like I should move everything to a new seed. It’s a lot though, so I don’t really want to.
+
+comment: p1pwfvl
+parent: t3_1vc55eb
+author: thelaurelhighlander
+created_utc: 1785874185
+edited: false
+body:
+My two cents.....I had an unopened MK4 that I wasn't using when this all went down. I updated the firmware and I created a seed phrase with 100 dice rolls. I did a test set of rolls first to see if the CC provided the same value for the rolls as on bitcoiner.guide/seed/, it did. I then created a new seed the same way (without checking online) and added a passphrase.  
+
+
