--- reddit-dice-rolls-trust@20260804T201457Z
+++ reddit-dice-rolls-trust@20260804T211445Z
@@ -268,3 +268,11 @@
 My two cents.....I had an unopened MK4 that I wasn't using when this all went down. I updated the firmware and I created a seed phrase with 100 dice rolls. I did a test set of rolls first to see if the CC provided the same value for the rolls as on bitcoiner.guide/seed/, it did. I then created a new seed the same way (without checking online) and added a passphrase.  
 
 
+
+comment: p1q69xq
+parent: t1_p1pwfvl
+author: Intelligent_Map_246
+created_utc: 1785876794
+edited: false
+body:
+Yes. Many have confirmed this method to. Probably use it for seed generation and password storage.  Nothing else 
