--- bitcointalk-device-entropy-trust@20260807T100536Z
+++ bitcointalk-device-entropy-trust@20260808T020410Z
@@ -180,7 +180,7 @@
 The insurance isn't foolproof solution, it's always better if you can take care your things than depending on insurance but in case you need one, understand the clause.
 Title: Re: Possible all Bitcoin Hardware Wallets be hacked due to 'Trust in Device Entropy?
 Post by: BlackBoss_ on August 07, 2026, 05:02:11 AM
-Quote from: X-ray on Today at 03:49:34 AM
+Quote from: X-ray on August 07, 2026, 03:49:34 AM
 If you ever thinking of one, be sure to do your extensive research about what losses they cover and properly learn about how to file a claim so that you don't give them free money for nothing.
 Crypto insurance that covers holding often exclude the real risk that people usually face like phishing, social engineering, or even pure technical defects which in this case might very well be referred to incident like Coldcard.
 The insurance isn't foolproof solution, it's always better if you can take care your things than depending on insurance but in case you need one, understand the clause.
@@ -190,5 +190,25 @@
 https://en.wikipedia.org/wiki/Enron
 https://en.wikipedia.org/wiki/Enron
 Bitcoin is designed to give everyone private keys that have very strong security, so they just have to find a good Bitcoin wallet to use and secure their wallets as well as their bitcoins by themselves.
+Title: Re: Possible all Bitcoin Hardware Wallets be hacked due to 'Trust in Device Entropy?
+Post by: dim_mak5 on August 07, 2026, 10:09:06 PM
+I have a feeling BlackRock Wall Street in the near future is going to fund a new hardware wallet company and release a new closed source hardware btc wallet that is so called 'Government Approved' that is insured up to max 0.1 BTC for retail customers and 1 BTC for commercial customers kinda like FDIC insurance because they can afford to. Only by doing this they can trick the nearly all btc hodlers to give up self custody. They will try to corner the bitcoin hardware wallet and be like like Iphone in the cell phone market.
+They will sponsor most of the hacks to get hodlers to fear self custody.
+True Entropy is suppose to be more random that the lottery number being drawn out.
+Can anyone confirm whether existing hardware wallets entropy will still be secure in 5-10 years time? Never mind quantum computers.
+I still cant understand someone paying money for a hardware wallet for Entropy yet still expect to roll the dice for added randomness is insane. It is like paying for a cheeseburger but buying your own cheese slice and adding it in yourself in the burger. The customers paid for Entropy so why they rolling the dice? The hardware wallet manufacturers are implying are entropy chips are not as good as your old fashioned dice rolling?
+What if wall st sponsored hackers or Ai hackers figured out the randomness of lets say Ledger and Trezor devices and most BTC in cold storage gets hacked.
+Why Michael Saylor not stash any of his bitcoins in a hardware wallet and instead use a third party insured Custodian like Coinbase?
+Satoshi Wallet that has a million bitcoins in it, that must be truly random generated as it hasn't been hacked yet. What randomness Entropy Satoshi used?
+Title: Re: Possible all Bitcoin Hardware Wallets be hacked due to 'Trust in Device Entropy?
+Post by: dim_mak5 on August 07, 2026, 10:44:02 PM
+Also Hardware Wallets should not have a back up built in software entropy randomness generator if the hardware entropy chip in the hardware wallet fails. Also firmware software updates should be isolated from the software entropy generator as a simple bug in the code in the update could compromise the software entropy generator.
+What's the point of a hardware wallet when it uses software for entropy  ??? ???. Why call it hardware when its really a cheap software? Coldcard should have called their wallet a hardware/software hybrid wallet They misled their customers.
+Title: Re: Possible all Bitcoin Hardware Wallets be hacked due to 'Trust in Device Entropy?
+Post by: lombok on August 07, 2026, 11:57:28 PM
+Quote from: dim_mak5 on August 07, 2026, 10:44:02 PM
+Also Hardware Wallets should not have a back up built in software entropy randomness generator if the hardware entropy chip in the hardware wallet fails. Also firmware software updates should be isolated from the software entropy generator as a simple bug in the code in the update could compromise the software entropy generator.
+What's the point of a hardware wallet when it uses software for entropy  ??? ???. Why call it hardware when its really a cheap software? Coldcard should have called their wallet a hardware/software hybrid wallet They misled their customers.
+Indeed, the susceptibility of the physical randomness chip is also true risk on it is own. It is exactly to avoid the single point of failure in the electronic elements that a hybrid software based approach is used. The name physical wallet does not lose its validity by the fact that its main purpose is not process algorithms but to isolated offline transactions. The alternative of randomization of dice separately has shown to be a workable solution in the dissolution of doubts in tampering with inner codes.
 Powered by SMF 1.1.19 |
 SMF © 2006-2009, Simple Machines
