--- stackernews-2023-hww-paranoia@20260807T205418Z
+++ stackernews-2023-hww-paranoia@20260808T162835Z
@@ -3,6 +3,13 @@
     "item": {
       "comments": {
         "comments": [
+          {
+            "createdAt": "2023-10-25T22:12:30.108Z",
+            "text": "In cryptography (the original 'crypto'), it is said: don't roll your own cryptography.\n\nWe should also normalise 'don't roll your own dice': https://crypto.stackexchange.com/a/87982",
+            "user": {
+              "name": "sime"
+            }
+          },
           {
             "createdAt": "2026-08-07T05:52:53.033Z",
             "text": "\"I know you can do dice rolls and all that but you are still trusting that it is a genuine random seed that is being generated.\"\n\nWhat exactly are you 'trusting'? That physics works correctly in your apartment? Even if dice is not ideally balanced it is not a big deal.\n\nIf you refer to 'does device use the rolls I typed in?' question - with seedsigner you can verify it yourself. They publish code that reproduces deterministic on-device process and you can verify that for given rolls you get given address.",
@@ -44,13 +51,6 @@
             "user": {
               "name": "CruncherDefi"
             }
-          },
-          {
-            "createdAt": "2023-10-25T22:12:30.108Z",
-            "text": "In cryptography (the original 'crypto'), it is said: don't roll your own cryptography.\n\nWe should also normalise 'don't roll your own dice': https://crypto.stackexchange.com/a/87982",
-            "user": {
-              "name": "sime"
-            }
           }
         ]
       },
