--- reddit-safe-seed-without-hardware@20260804T164816Z
+++ reddit-safe-seed-without-hardware@20260804T184706Z
@@ -741,4 +741,20 @@
 body:
 Ikr, global adoption my ass
 
+comment: p1p7o5y
+parent: t1_p1jz3fx
+author: Porcellanidae
+created_utc: 1785867798
+edited: false
+body:
+Throw away, as you checked your formula for that one on your computer 
+
+comment: p1p7y56
+parent: t1_p1i1dkv
+author: Porcellanidae
+created_utc: 1785867866
+edited: false
+body:
+Where can i order ?
+
 more-stub: parent t1_p1mxyeo count 0
