--- reddit-safe-seed-without-hardware@20260804T134528Z
+++ reddit-safe-seed-without-hardware@20260804T141614Z
@@ -725,4 +725,12 @@
 
 BUT, it will also complicate your operation with the wallet, as you have to type the passphrase to use it
 
+comment: p1njmoq
+parent: t1_p1ir21r
+author: doctrgiggles
+created_utc: 1785852557
+edited: false
+body:
+Almost the opposite. I'm far enough into these spaces that I'm confident in my ability to create and operate secure machines. I'm obviously cautious about key management, but there are plenty of people out there using encryption technologies similar to Bitcoin with higher stakes and without the luxury of hardware wallets to keep them safe. Dissidents and reporters in plenty of places are targeted by state-level actors and still they need to be able to keep their machines secure while still actively using them. 
+
 more-stub: parent t1_p1mxyeo count 0
