--- reddit-mk3-advisory-relay@20260804T090149Z
+++ reddit-mk3-advisory-relay@20260805T205911Z
@@ -309,14 +309,6 @@
   
 [https://engineering.block.xyz/blog/predictable-rng-fallback-and-32-bit-reseed-in-coldcard-firmware](https://engineering.block.xyz/blog/predictable-rng-fallback-and-32-bit-reseed-in-coldcard-firmware)
 
-comment: p0t0wt2
-parent: t1_p0sivl4
-author: ASilverSpartan
-created_utc: 1785469295
-edited: false
-body:
-Was tails electrum ever unsafe? That’s how I protect mine and it’s lasted for almost a decade now. Do I need to change my security posture?
-
 comment: p0t53se
 parent: t1_p0soaak
 author: cilicia3k3
@@ -373,11 +365,11 @@
 
 comment: p0u14mh
 parent: t3_1vb9doi
-author: smaakversterker
+author: [deleted]
 created_utc: 1785485305
 edited: false
 body:
-what about the MK1?
+[deleted]
 
 comment: p0u2otq
 parent: t1_p0tqbhn
