--- unchained-help-replace-keys@20260803T194632Z
+++ unchained-help-replace-keys@20260805T205511Z
@@ -16,9 +16,9 @@
 Log in to your Unchained account.
 Ensure you have two secure, uncompromised keys available to build the new vault. You may need to setup and add new keys to your account.
 Navigate to the vault creation flow and set up your new vault.
-Ensure you select the option to transfer your existing subscription to this new vault. This will activate your 30-day grace period.
-Once your new vault is active, initiate a transaction sending all of the bitcoin from your old vault to your new vault.
-When initiating the withdrawal on your old vault, you can choose “Internal transfer” and select the new vault.
+Ensure you select the option to transfer your existing subscription to this new vault. This will activate your 30-day grace period. Some clients will not see this option due to being grandfathered into an account-level billing plan.
+Once your new vault is active, initiate a withdrawal transaction sending all of the bitcoin from your old vault to your new vault.
+When initiating the withdrawal on your old vault, you can choose Internal transfer and select the new vault. If you don't see the internal transfer option, then navigate to your new vault and copy the deposit address. Then return to your old vault, create the withdrawal, and paste the address in the address field. Double check that it looks the same as the address you copied.
 If you have 2 Coldcard-generated keys and are affected by the July/August 2026 Coldcard vulnerability, be aware of the option to broadcast your transaction using Slipstream. This is unnecessary if you only have 1 Coldcard-generated key.
 Once the transaction is confirmed, your funds are safely secured in the new vault. You can then close the old vault.
 If you encounter any issues or have questions regarding your specific vault configuration, please contact our support team for guidance before proceeding.
