--- unchained-help-replace-keys@20260803T161456Z
+++ unchained-help-replace-keys@20260803T164641Z
@@ -10,7 +10,7 @@
 You can replace multiple keys with one step by building a new vault
 Creating a new vault is our recommended alternative to using our key replacement feature, especially if you need to rotate multiple keys. By following these instructions to create a new vault and manually sweeping your funds into it, you gain a cleaner, safer, and more flexible process.
 Note: If you have an IRA vault, you may use this method. Ensure that you are building the new vault within the same IRA account. Once finished, please close the old IRA vault. You should only have one active vault per IRA account.
-You may see a big red warning on the deposit address for your new IRA vault. The warning is intended to prevent an outside deposit from being sent directly to an IRA vault without going through the approved contribution or rollover process. You may proceed when you are using Unchained's workflow to move bitcoin between vaults in the same IRA account. Do not disregard the warning for bitcoin coming from a personal wallet, exchange, or any other source outside the IRA.
+You may see a big red warning on the deposit address for your new IRA vault. The warning is intended to prevent an outside deposit from being sent directly to an IRA vault without going through the approved contribution or rollover process. You may proceed when you are using this workflow to move bitcoin between vaults in the same IRA account. Do not disregard the warning for bitcoin coming from a personal wallet, exchange, or any other source outside the IRA.
 How to create a new vault and move your funds
 Log in to your Unchained account.
 Ensure you have two secure, uncompromised keys available to build the new vault. You may need to setup and add new keys to your account.
