--- casa-blog-index@20260804T214102Z
+++ casa-blog-index@20260804T224206Z
@@ -6,6 +6,11 @@
 Sign up
 Log in
 Casa Blog - Bitcoin Security Made Easy
+Featured
+The Coldcard Vulnerability, Explained: Who's at Risk and What to Do Now
+by
+Team Casa
+6 hours ago
 Featured
 Evolving Casa's defenses against social engineering
 by
@@ -39,7 +44,7 @@
 The Coldcard Vulnerability, Explained: Who's at Risk and What to Do Now
 by
 Team Casa
-5 hours ago
+6 hours ago
 From Single Point of Failure to Full Sovereignty: A Portfolio Manager’s Story
 by
 Gui
