--- casa-blog-index@20260807T061149Z
+++ casa-blog-index@20260809T164647Z
@@ -10,7 +10,7 @@
 The Coldcard Vulnerability, Explained: Who's at Risk and What to Do Now
 by
 Team Casa
-2 days ago
+5 days ago
 Featured
 Evolving Casa's defenses against social engineering
 by
@@ -44,16 +44,16 @@
 The Rise of the Machines
 by
 Jameson Lopp
-6 hours ago
+3 days ago
 The Coldcard Vulnerability, Explained: Who's at Risk and What to Do Now
 by
 Team Casa
-2 days ago
+5 days ago
 Bitcoin security
 From Single Point of Failure to Full Sovereignty: A Portfolio Manager’s Story
 by
 Gui
-22 days ago
+25 days ago
 Bitcoin
 A Social Engineering Story
 by
