--- casa-blog-index@20260805T001144Z
+++ casa-blog-index@20260805T142022Z
@@ -10,7 +10,7 @@
 The Coldcard Vulnerability, Explained: Who's at Risk and What to Do Now
 by
 Team Casa
-7 hours ago
+21 hours ago
 Featured
 Evolving Casa's defenses against social engineering
 by
@@ -44,12 +44,12 @@
 The Coldcard Vulnerability, Explained: Who's at Risk and What to Do Now
 by
 Team Casa
-7 hours ago
+21 hours ago
 Bitcoin security
 From Single Point of Failure to Full Sovereignty: A Portfolio Manager’s Story
 by
 Gui
-20 days ago
+21 days ago
 Bitcoin
 A Social Engineering Story
 by
