--- reddit-bitcoin-org-still-recommends@20260807T192851Z
+++ reddit-bitcoin-org-still-recommends@20260808T015908Z
@@ -3,15 +3,7 @@
 created_utc: 1785775201
 title: Bitcoin.org still recommends coldcard
 body:
-[Bitcoin.org](http://Bitcoin.org) still recommends coldcard.
-
-"Very secure environment" = GOOD 
-
-"Control over your money" = GOOD 
-
-Sauce: [https://bitcoin.org/en/wallets/hardware/coldcard/?platform=hardware&step=5](https://bitcoin.org/en/wallets/hardware/coldcard/?platform=hardware&step=5)
-
-
+[removed]
 
 comment: p1h9sbz
 parent: t3_1veine5
