--- reddit-bitcoin-org-still-recommends@20260806T163309Z
+++ reddit-bitcoin-org-still-recommends@20260807T125608Z
@@ -627,6 +627,26 @@
 body:
 Corrupted compromised
 
+comment: p28n4p6
+parent: t1_p1hol9l
+author: JumpProfessional3372
+created_utc: 1786099146
+edited: false
+body:
+And they also have a nice paranoid guide for this. I learned a lot from that guide. 
+
+comment: p28nwep
+parent: t1_p1hbn75
+author: JumpProfessional3372
+created_utc: 1786099469
+edited: false
+body:
+I recently created a new wallet Blixt ligthning wallet, i just checked that it is an open source self custody wallet and I 100% trusted the wallet to randomly generate a seed. Why? Because I only have a few sats in there. I accept the risk here.
+
+But if I had to put money I cannot afford to lose, I would not use any RNG, I prefer rolling 100+ dices to generate the seed myself and adding a good passphrase. I would do this no matter the wallet manufacturer.
+
+Any HW wallet is a dangerous device if the user is not paranoid enough.
+
 more-stub: parent t1_p1m4y3j count <live-count>
 
 more-stub: parent t1_p1lqpae count <live-count>
