--- reddit-bitcoin-org-still-recommends@20260804T194602Z
+++ reddit-bitcoin-org-still-recommends@20260806T163309Z
@@ -41,11 +41,11 @@
 
 comment: p1hf7ct
 parent: t1_p1hbn75
-author: IllllIIlIllIllllIlll
+author: [deleted]
 created_utc: 1785777230
-edited: false
-body:
-Even if they stop support, people can simply buy another hardware wallet. Unless they were stupid enough to not backup their seed.
+edited: 1786027775
+body:
+[deleted]
 
 comment: p1hgx5k
 parent: t1_p1hf7ct
