--- reddit-casino-dice-myths@20260807T193420Z
+++ reddit-casino-dice-myths@20260808T020438Z
@@ -679,3 +679,27 @@
 edited: false
 body:
 True and as others have said it doesn't rule out the chance of still coming up with any possible seed. Whereas the PRNG of CC device was actually limiting what could be generated so making things very easy for hackers 
+
+comment: p2bzy57
+parent: t1_p23nnpp
+author: pythosynthesis
+created_utc: 1786133332
+edited: false
+body:
+Tree fiddy.
+
+comment: p2cahhj
+parent: t1_p21atmg
+author: Deto
+created_utc: 1786136202
+edited: false
+body:
+yeah, the more rules and process around this people emphasize the more likely it is that someone doesn't bother.  Even rolling the dice in a way that they bounce...I doubt a person could really reduce their entropy by that much unless they were _trying_ to roll in a dumb way.
+
+comment: p2cn6a3
+parent: t1_p21c9cf
+author: Gesha24
+created_utc: 1786139900
+edited: false
+body:
+Even if it causes a half of your throws to land on the same number, until somebody knows that you have used this specific dice with this specific imperfection and starts targeting specifically you - it's still secure.
