--- reddit-casino-dice-myths@20260807T104502Z
+++ reddit-casino-dice-myths@20260807T130041Z
@@ -633,3 +633,11 @@
 Digital signing is where you have some open kimono going on,  but you can still preplan this and prepare it offline.
 
 I've done 90% of my crypto-related interactions this way.  It's time-consuming, mundane, and annoying.  But ultimately, my stacks of hand written index cards have kept me safer than hardware wallets. 
+
+comment: p29069t
+parent: t1_p229hwi
+author: 10kpizza
+created_utc: 1786104195
+edited: false
+body:
+You think you're very funny
