--- reddit-casino-dice-myths@20260807T130041Z
+++ reddit-casino-dice-myths@20260807T193420Z
@@ -641,3 +641,41 @@
 edited: false
 body:
 You think you're very funny
+
+comment: p29hy5u
+parent: t1_p23sl1p
+author: EclecticKant
+created_utc: 1786109760
+edited: false
+body:
+Everything you add to a truly random event can only make it less random, a perfectly balanced dice that rolls correctly is theoretically perfectly random, every imperfection that changes that can only make things worse.  
+In this case those imperfections are too hard to predict, so the event stays random anyway, but it van have some patterns that we can't detect
+
+comment: p29md9v
+parent: t1_p226q1e
+author: Sitonyourhandsnclap
+created_utc: 1786111002
+edited: false
+body:
+Yea I wondered this too. Wouldn't someone have to actually take your dice and do analysis on them like 10k, 100k maybe even 1million rolls in order to determine an observable bias then use that information to determine what seeds are more likely to have been generated with those dice? Or is it simply the case that they tweak settings for say a dice that rolls a 1 slightly more (say 10%) than any other number, then look for seeds. Then adjust percentages, look again for seeds. Adjust for the number 2 being more favourable etc. 
+
+comment: p29nqep
+parent: t1_p29md9v
+author: togetherwem0m0
+created_utc: 1786111379
+edited: false
+body:
+yes someone would have to get the dice used and conduct their own rolls to determine any bias. even then, the test rolls would occur after the original seed generation and because of physical imperfections created by the analysis itself, the analysis would drift from the original rolls.
+
+even then, dice rolling involves rolling more dice than you "use" because when you roll 6 dice you select 1 randomly, so whatever bias there is in a dice set is reduced/spread across the number of dice you use.
+
+  
+its a very effective source of real entropy.
+
+comment: p29pauf
+parent: t1_p29nqep
+author: Sitonyourhandsnclap
+created_utc: 1786111811
+edited: false
+body:
+True and as others have said it doesn't rule out the chance of still coming up with any possible seed. Whereas the PRNG of CC device was actually limiting what could be generated so making things very easy for hackers 
