--- reddit-casino-dice-myths@20260808T083645Z
+++ reddit-casino-dice-myths@20260809T165706Z
@@ -711,3 +711,12 @@
 edited: false
 body:
 what's wrong with /dev/random?
+
+comment: p2ma8fq
+parent: t3_1vh0tzo
+author: BrokenHefaistos
+created_utc: 1786269300
+edited: false
+body:
+I can think of better ways to generate a seed, lol
+If this was perfect then why arent there tiny dice in computers ? Maybe because a dice has only 6 values introducing predictability.
