--- reddit-coldcard-open-source-question@20260804T202805Z
+++ reddit-coldcard-open-source-question@20260804T215744Z
@@ -94,3 +94,13 @@
 > or was it available to inspect but no-one bothered?
 
 I'd say some people bothered for the same reason the attacker last week did, they just failed for 5 years until last week, for the reason I just gave above.
+
+comment: p1qh56a
+parent: t1_p1m97b1
+author: Level-Set5770
+created_utc: 1785879752
+edited: false
+body:
+Just another midwit regurgitating shit takes.
+
+Since when did **"trust but verify"** become **"trust but only verify if I can fork it to start my own project"**
