--- reddit-coldcard-open-source-question@20260804T175640Z
+++ reddit-coldcard-open-source-question@20260804T182622Z
@@ -68,3 +68,11 @@
 edited: false
 body:
 It was "source available software". Means everyone can view it, but cannot make a product using that code and start selling it, offering services, etc.
+
+comment: p1p3m2g
+parent: t3_1vf4z1b
+author: EyesFor1
+created_utc: 1785866794
+edited: false
+body:
+Ledger is not 100% open source, parts of the ledger os that interact with the Secure Element chip are closed. Trezor is closer to 100% but not 100%. You have to trust a company at some point.
