--- reddit-coldcard-open-source-question@20260804T233443Z
+++ reddit-coldcard-open-source-question@20260805T080814Z
@@ -114,3 +114,11 @@
 It always been don't trust. But if you are trusting and not verifying. Then don't complain that no one verified because checking buggy code is hard work and testing and the people willing to do that is because they want to build on top of it. If you can't fork it, then you have less testing.
 
 By the way, haven't you seen all the "firmware upgrade bricked my cc", "my cc doesn't boot after I..",  etc? That kind of bugs are easy to fix when another company tests the firmware on different hardware.
+
+comment: p1tbkzb
+parent: t3_1vf4z1b
+author: _gianlucag_
+created_utc: 1785917100
+edited: false
+body:
+Being open source is totally irrelevant if you are unable to compile and flash the binary into the device. You can inspect the code on github or whatever but that's non sense if you just trust that software is really the one running on your device.
