--- reddit-coldcard-open-source-question@20260804T215744Z
+++ reddit-coldcard-open-source-question@20260804T233443Z
@@ -104,3 +104,13 @@
 Just another midwit regurgitating shit takes.
 
 Since when did **"trust but verify"** become **"trust but only verify if I can fork it to start my own project"**
+
+comment: p1qy08f
+parent: t1_p1qh56a
+author: RevolutionaryPick241
+created_utc: 1785884699
+edited: false
+body:
+It always been don't trust. But if you are trusting and not verifying. Then don't complain that no one verified because checking buggy code is hard work and testing and the people willing to do that is because they want to build on top of it. If you can't fork it, then you have less testing.
+
+By the way, haven't you seen all the "firmware upgrade bricked my cc", "my cc doesn't boot after I..",  etc? That kind of bugs are easy to fix when another company tests the firmware on different hardware.
