--- reddit-coldcard-open-source-question@20260805T080814Z
+++ reddit-coldcard-open-source-question@20260807T142509Z
@@ -3,15 +3,7 @@
 created_utc: 1785834128
 title: Is Coldcard Really Open Source or Not?
 body:
-Is Coldcard really open-source? I'm confused. I'm getting mixed messages about this. My definition of 'Open Source' for a a hardware wallet is:  Can anyone download the code for free without permission and pay a software engineer or an AI to check for bugs?
-
-Why did no-one spot the error in Coldcard's programing?
-
-[https://insider.btcpp.dev/p/when-randombytes-runs-but-doesnt](https://insider.btcpp.dev/p/when-randombytes-runs-but-doesnt)
-
-Was it that the code was locked up and unavailable for inspection
-
-or was it available to inspect but no-one bothered?
+[removed]
 
 comment: p1m5tc6
 parent: t3_1vf4z1b
