--- reddit-white-hat-drains-relay@20260804T224638Z
+++ reddit-white-hat-drains-relay@20260804T234603Z
@@ -1013,6 +1013,14 @@
 body:
 Yeah but since the keys are compromised, how would you prove you are the original owner and not the hacker?
 
+comment: p1oc0ft
+parent: t3_1veo0yh
+author: 00-SilverShot
+created_utc: 1785859947
+edited: false
+body:
+I call bullshit.  Whitehat my ass.  These hackers are breaking into someones wallet and draining funds.  That's theft, through and through.  Imagine if I broke into my neighbors house and removed all their gold from their safe, then the police caught me.  Would the police let me go because I was "trying to prevent someone else from stealing it from them".  It's so insanely stupid to think that's somehow not theft.  Handcuffs and jail time for you thief!
+
 comment: p1pjfbm
 parent: t3_1veo0yh
 author: locotx
