--- reddit-white-hat-drains-relay@20260805T205943Z
+++ reddit-white-hat-drains-relay@20260806T163229Z
@@ -142,14 +142,6 @@
 body:
 Device ID of the physical device. That's why they are telling people to not destroy their device.
 
-comment: p1iqrx2
-parent: t1_p1il4yb
-author: IllllIIlIllIllllIlll
-created_utc: 1785789452
-edited: false
-body:
-Lol maybe *you* should read again. It's not so hard to understand.
-
 comment: p1ir29g
 parent: t3_1veo0yh
 author: Slow-Childhood-5671
@@ -160,11 +152,11 @@
 
 comment: p1iro8r
 parent: t1_p1ippvk
-author: IllllIIlIllIllllIlll
+author: [deleted]
 created_utc: 1785789689
-edited: false
-body:
-That doesn't really answer the question, there is nothing tying an address to a specific device id no? A black hat could just give the device id their own cold card and claim that they are the person who had 18 BTC on it when in reality they had 0.01
+edited: 1786027693
+body:
+[deleted]
 
 comment: p1is3d8
 parent: t1_p1iihxv
@@ -186,13 +178,11 @@
 
 comment: p1iut2w
 parent: t1_p1isqlx
-author: IllllIIlIllIllllIlll
+author: [deleted]
 created_utc: 1785790519
-edited: false
-body:
-If the black hat was able to drain the wallet, it means that he is able to correctly derive addresses from the seed already, which means he already has all the info needed. If the attacker was able to brute force the seed, it means that he can also brute force a valid device id that generates this seed (I assume, maybe not)
-
-Like isn't the whole vulnerability stemming from the fact that the attacker could brute force the device ID?
+edited: 1786027681
+body:
+[deleted]
 
 comment: p1iv56s
 parent: t1_p1iut2w
@@ -214,11 +204,11 @@
 
 comment: p1iwhhq
 parent: t1_p1iv56s
-author: IllllIIlIllIllllIlll
+author: [deleted]
 created_utc: 1785790972
-edited: false
-body:
-Why does it matter? Or will the recovery imply physically shipping your device to the white hat?
+edited: 1786027669
+body:
+[deleted]
 
 comment: p1iwktp
 parent: t1_p1invwp
@@ -1055,4 +1045,12 @@
 
 A single transaction wouldn't be enough, but if multiple derived addresses got transactions from your exchange account, that's pretty strong evidence.
 
+comment: p21z557
+parent: t1_p1kezat
+author: Personal-Time-9993
+created_utc: 1786022082
+edited: false
+body:
+Supposedly, I’ve not even seen anything concrete, only rumors.
+
 more-stub: parent t1_p1j79qs count <live-count>
