--- stackernews-boltz-zeuslsp-shutdown@20260804T134018Z
+++ stackernews-boltz-zeuslsp-shutdown@20260804T224319Z
@@ -4,17 +4,17 @@
       "comments": {
         "comments": [
           {
+            "createdAt": "2026-08-03T23:27:05.176Z",
+            "text": "Hope Boltz can sort this out.",
+            "user": {
+              "name": "OT"
+            }
+          },
+          {
             "createdAt": "2026-08-03T17:08:13.373Z",
             "text": "Any speculation about what’s up?",
             "user": {
               "name": "Undisciplined"
-            }
-          },
-          {
-            "createdAt": "2026-08-03T23:27:05.176Z",
-            "text": "Hope Boltz can sort this out.",
-            "user": {
-              "name": "OT"
             }
           },
           {
