--- stackernews-boltz-zeuslsp-shutdown@20260804T224319Z
+++ stackernews-boltz-zeuslsp-shutdown@20260805T075327Z
@@ -8,6 +8,13 @@
             "text": "Hope Boltz can sort this out.",
             "user": {
               "name": "OT"
+            }
+          },
+          {
+            "createdAt": "2026-08-03T16:59:48.260Z",
+            "text": "we are still having @SwapMarket",
+            "user": {
+              "name": "supratic"
             }
           },
           {
@@ -50,13 +57,6 @@
             "text": "coincidence, I was just coming here to report this error and saw your post. Is that why Shockwallet’s swaps disabled?\n\n![](https://m.stacker.news/150672)",
             "user": {
               "name": "0xbitcoiner"
-            }
-          },
-          {
-            "createdAt": "2026-08-03T16:59:48.260Z",
-            "text": "we are still having @SwapMarket",
-            "user": {
-              "name": "supratic"
             }
           },
           {
