--- stackernews-boltz-zeuslsp-shutdown@20260804T041709Z
+++ stackernews-boltz-zeuslsp-shutdown@20260804T071019Z
@@ -3,13 +3,6 @@
     "item": {
       "comments": {
         "comments": [
-          {
-            "createdAt": "2026-08-03T19:30:34.330Z",
-            "text": "Ark, Liquid, Spark are literally fucked right now.\nI hope people will learn the lesson and do not consider sidechains as LN...\n\n![](https://m.stacker.news/150688)\n\nenjoy your fakery",
-            "user": {
-              "name": "DarthCoin"
-            }
-          },
           {
             "createdAt": "2026-08-03T17:08:13.373Z",
             "text": "Any speculation about what’s up?",
@@ -22,6 +15,13 @@
             "text": "![](https://m.stacker.news/150723)\n\n[source](https://x.com/alexbosworth/status/2084379577095446624)",
             "user": {
               "name": "Scoresby"
+            }
+          },
+          {
+            "createdAt": "2026-08-03T19:30:34.330Z",
+            "text": "Ark, Liquid, Spark are literally fucked right now.\nI hope people will learn the lesson and do not consider sidechains as LN...\n\n![](https://m.stacker.news/150688)\n\nenjoy your fakery",
+            "user": {
+              "name": "DarthCoin"
             }
           },
           {
