--- stackernews-boltz-zeuslsp-shutdown@20260804T071019Z
+++ stackernews-boltz-zeuslsp-shutdown@20260804T134018Z
@@ -11,10 +11,10 @@
             }
           },
           {
-            "createdAt": "2026-08-03T23:46:45.202Z",
-            "text": "![](https://m.stacker.news/150723)\n\n[source](https://x.com/alexbosworth/status/2084379577095446624)",
+            "createdAt": "2026-08-03T23:27:05.176Z",
+            "text": "Hope Boltz can sort this out.",
             "user": {
-              "name": "Scoresby"
+              "name": "OT"
             }
           },
           {
@@ -25,10 +25,10 @@
             }
           },
           {
-            "createdAt": "2026-08-03T16:51:00.529Z",
-            "text": "https://twiiit.com/ZeusLN/status/2084316041673347138",
+            "createdAt": "2026-08-03T23:46:45.202Z",
+            "text": "![](https://m.stacker.news/150723)\n\n[source](https://x.com/alexbosworth/status/2084379577095446624)",
             "user": {
-              "name": "nitter"
+              "name": "Scoresby"
             }
           },
           {
@@ -36,6 +36,13 @@
             "text": "When the APIs go dark, the value of P2P shines:\n\nhttps://github.com/ElementsProject/peerswap",
             "user": {
               "name": "nullcount"
+            }
+          },
+          {
+            "createdAt": "2026-08-03T16:51:00.529Z",
+            "text": "https://twiiit.com/ZeusLN/status/2084316041673347138",
+            "user": {
+              "name": "nitter"
             }
           },
           {
@@ -64,13 +71,6 @@
             "text": "This is a regrettable turn of events; ‘Boltz’ was doing an excellent job.",
             "user": {
               "name": "Wellknow35"
-            }
-          },
-          {
-            "createdAt": "2026-08-03T23:27:05.176Z",
-            "text": "Hope Boltz can sort this out.",
-            "user": {
-              "name": "OT"
             }
           },
           {
