--- ledger-primary-incident-article@20260803T123857Z
+++ ledger-primary-incident-article@20260804T153945Z
@@ -1,5 +1,5 @@
-Ledger Wallet™ just got another major upgrade
-Take control today
+Switching hardware wallets? Migrate to Ledger safely in a few steps.
+Learn more
 Do you want to change language to ?
 Yes, please
 No, thanks
@@ -182,6 +182,7 @@
 Ledger is not affected by the recently published Coldcard Mk3 advisory.
 Ledger devices use a certified True Random Number Generator (TRNG) built directly into our Secure Element chip, generating full 256 bits of entropy for every 24-word Secret Recovery Phrase. Please refer to this article for more information.
 Coinkite has published a security advisory regarding certain Coldcard Mk3 firmware versions – you can read their notice here.
+Get a Ledger signer
 The Coldcard Incident: The Foundation of Randomness
 On July 31st, 2026, Coldcard reported that a firmware bug had weakened how some devices generated seed phrases leading to significant user losses. Back in late 2022, the Ledger Donjon discovered a similar vulnerability in the Trust Wallet browser extension.
 The bug made every wallet created by the extension recoverable from its public address. In practice, an attacker could derive the private keys and steal the funds without any user interaction. The search space was small enough to brute force.
