--- reddit-multisig-only-way@20260806T101559Z
+++ reddit-multisig-only-way@20260806T164423Z
@@ -103,3 +103,13 @@
 edited: false
 body:
 Depends on how super huge the entropy of the extra password is....
+
+comment: p22fbub
+parent: t3_1vf9oh5
+author: Blackcherry777
+created_utc: 1786026511
+edited: false
+body:
+https://benma.github.io/2020/11/05/multisig-xpubs-verification.html
+
+No actually read this article about multisig you could argue it’s a bit more dangerous and most people’s setups are at risk!  I think there’s nothing better than a Seedsigner you build yourself from rasberry pi ordered from a separate vendor.  Make sure you verify and flash a new SD card with the verified software and roll dice 100 times.  Add a passphrase on top off that and you are golden!  
