--- reddit-multisig-only-way@20260806T164423Z
+++ reddit-multisig-only-way@20260807T142513Z
@@ -113,3 +113,19 @@
 https://benma.github.io/2020/11/05/multisig-xpubs-verification.html
 
 No actually read this article about multisig you could argue it’s a bit more dangerous and most people’s setups are at risk!  I think there’s nothing better than a Seedsigner you build yourself from rasberry pi ordered from a separate vendor.  Make sure you verify and flash a new SD card with the verified software and roll dice 100 times.  Add a passphrase on top off that and you are golden!  
+
+comment: p252sqi
+parent: t3_1vf9oh5
+author: ElderMight
+created_utc: 1786050742
+edited: false
+body:
+I think the calls for multi-sig are an over reaction. 
+
+ONE implementation of self custody failed by a specific vendor. A properly generated private key is still extremely secure. 
+
+Pushing everyone to do multi sig is dangerous because it is more likely to lock out users from their funds than protect them from theft. 
+
+I didn't know this before and recently found out that to do a 2 of 3 multi-sig, you need at least 2 of 3 keys AND all 3 public keys. If you lose 1 public key you lost your money...
+
+Dice generated private key + strong passphrase should be plenty enough security. 
