--- reddit-multisig-only-way@20260805T143746Z
+++ reddit-multisig-only-way@20260805T211303Z
@@ -87,3 +87,11 @@
 Im assuming all attackers brute force 12 - 24 keywords which is hard enough.
 
 How will they even know on which of these wallets to run another brute force for the passphrase? Isnt it increasing their attack surfsce by a super huge margin?
+
+comment: p1wr4di
+parent: t3_1vf9oh5
+author: CiaranCarroll
+created_utc: 1785955292
+edited: false
+body:
+No, just do what the people with Coldcards who didn't lose their funds did.
