--- reddit-multisig-only-way@20260804T202811Z
+++ reddit-multisig-only-way@20260805T080819Z
@@ -50,3 +50,19 @@
 edited: false
 body:
 this is how i was setup. i had a multi-sig but all child seeds were derived from the master, and each signer was derived from it. thankfully i got funds moved before any loss. i still like the device. IF i use it in the future, it will be with my own entropy.
+
+comment: p1ruqju
+parent: t3_1vf9oh5
+author: SuccessfulPlenty942
+created_utc: 1785895215
+edited: false
+body:
+A 2 of 3 multisig is not that hard if you've taken some time to learn how to use bitcoin. Id recommend it 
+
+comment: p1s9pp7
+parent: t3_1vf9oh5
+author: Old_Instruction_6004
+created_utc: 1785900369
+edited: false
+body:
+Look into Liana wallet. It's multi-sig with redundant keys. It's free and open source.
