--- reddit-multisig-only-way@20260805T080819Z
+++ reddit-multisig-only-way@20260805T143746Z
@@ -66,3 +66,24 @@
 edited: false
 body:
 Look into Liana wallet. It's multi-sig with redundant keys. It's free and open source.
+
+comment: p1td3id
+parent: t3_1vf9oh5
+author: SnooCookies9506
+created_utc: 1785917845
+edited: false
+body:
+Adding a passphrase is basically a 2 of 2 multi-sig if you think about it. 
+
+comment: p1tiqbd
+parent: t3_1vf9oh5
+author: Massive-Reception161
+created_utc: 1785920589
+edited: false
+body:
+Trezor for example has the 24 seed + passphrase.
+Isnt it sufficient?
+
+Im assuming all attackers brute force 12 - 24 keywords which is hard enough.
+
+How will they even know on which of these wallets to run another brute force for the passphrase? Isnt it increasing their attack surfsce by a super huge margin?
