--- reddit-coldcard-mk4-ux-flaw-continued@20260809T120336Z
+++ reddit-coldcard-mk4-ux-flaw-continued@20260809T184908Z
@@ -31,3 +31,11 @@
 edited: false
 body:
 The first part of the manual (easy setup guide) doesn't mention dice rolls. You have to go out of your way to find any mention of dice 
+
+comment: p2oc6um
+parent: t3_1viixzj
+author: Economy-Cash6726
+created_utc: 1786294593
+edited: false
+body:
+I did 12 word no dice roll option and then hit 4 to mix in dice rolls. Didn’t enforce shit for me
