--- reddit-coldcard-mk4-ux-flaw-continued@20260809T064148Z
+++ reddit-coldcard-mk4-ux-flaw-continued@20260809T120336Z
@@ -23,3 +23,11 @@
 edited: false
 body:
 Lol. Several others got robbed like me a few years later. I was one of the users who was blocked and  i mixed in my dice rolls so not sure what to say other than nvk is a joker who should stop making wallets
+
+comment: p2lozfu
+parent: t1_p2dwafi
+author: SpareEconomy1849
+created_utc: 1786258418
+edited: false
+body:
+The first part of the manual (easy setup guide) doesn't mention dice rolls. You have to go out of your way to find any mention of dice 
