--- stackernews-matt-levine-heist@20260804T175633Z
+++ stackernews-matt-levine-heist@20260804T195718Z
@@ -3,6 +3,13 @@
     "item": {
       "comments": {
         "comments": [
+          {
+            "createdAt": "2026-08-04T17:26:29.690Z",
+            "text": "> *Not your keys, not your coins,”*\\* but if your keys were not generated using robust randomization techniques, also not your coins. Oops!\"\\*\n\nWell... yea.\nI mean, what are we supposed to say? If I make a seed phrase I make myself, I'd also get swept. This has in fact [happened to people who tried brain wallets](https://www.securityweek.com/most-bitcoin-brain-wallets-drained-attackers/) back in the day.",
+            "user": {
+              "name": "Aeneas"
+            }
+          },
           {
             "createdAt": "2026-08-04T10:42:46.357Z",
             "text": "well, I suppose it is pieces with this sort of tone that make me feel better about Bitcoin. I get scared when they don't have anything bad to say. \n\nOf course, if they have enough bad stuff to say for long enough, perhaps the bitcoin train remains a unstoppable, yet minor intercity line doing the milk run from backwater A to bumfuck B. Still seems like a good train on which to ride.",
@@ -16,13 +23,6 @@
             "user": {
               "name": "billytheked"
             }
-          },
-          {
-            "createdAt": "2026-08-04T17:26:29.690Z",
-            "text": "> *Not your keys, not your coins,”*\\* but if your keys were not generated using robust randomization techniques, also not your coins. Oops!\"\\*\n\nWell... yea.\nI mean, what are we supposed to say? If I make a seed phrase I make myself, I'd also get swept. This has in fact [happened to people who tried brain wallets](https://www.securityweek.com/most-bitcoin-brain-wallets-drained-attackers/) back in the day.",
-            "user": {
-              "name": "Aeneas"
-            }
           }
         ]
       },
