--- reddit-attack-timing-speculation@20260804T212554Z
+++ reddit-attack-timing-speculation@20260805T080625Z
@@ -215,3 +215,22 @@
 edited: false
 body:
 I think the thing people are missing is - what does fixing the vulnerability even look like? So they fix the code and have to announce to everyone “hey if you generated a key with the old code you have to move all your coins now!” Just alerting bad actors. That is better than doing nothing (they still would have been raked through the coals), I don’t see any world where this ended nicely. In all honesty if they knew about it after selling thousands of cold cards with that version of firmware they probably would have quietly dismantled the company before it blew up in their faces.
+
+comment: p1suw17
+parent: t3_1vetc6x
+author: Crcex86
+created_utc: 1785909153
+edited: false
+body:
+If that's true, they were extremely patient and cool, because the exploit could have been resolved easily at any time.
+
+comment: p1t82tj
+parent: t3_1vetc6x
+author: Laukess
+created_utc: 1785915393
+edited: false
+body:
+If you had known about this bug for 5 years, why not use the time to find exposed keys?   
+That way you could target way more wallets.  
+The day they wants to execute the transfer, they already got a list of private keys.  
+I also find it hard to believe someone finding this bug would wait 5 years. 
