--- reddit-steel-plates-wasted@20260813T193122Z
+++ reddit-steel-plates-wasted@20260815T122907Z
@@ -290,3 +290,11 @@
 Like [https://a.co/d/0gTJZSu6](https://a.co/d/0gTJZSu6) or [https://a.co/d/0bHdnPqR](https://a.co/d/0bHdnPqR) [https://a.co/d/0i4oInDJ](https://a.co/d/0i4oInDJ)
 
 I don't use steel for my backup seed but learning from coldcard experience I will use a steel plate I can change my seed  moving forward
+
+comment: p3tm055
+parent: t1_p2yp3dd
+author: Oxymorix
+created_utc: 1786792876
+edited: false
+body:
+You should have rolled your own. 
