--- reddit-steel-plates-wasted@20260812T173320Z
+++ reddit-steel-plates-wasted@20260813T193122Z
@@ -183,11 +183,11 @@
 
 comment: p30wf6i
 parent: t3_1vkqgos
-author: Armadillodillodillo
+author: [deleted]
 created_utc: 1786451094
 edited: false
 body:
-They were always a waste, cause they do not solve anything. It protects from fire, but doesn't protect from theft. So you would still need to store same phrase in two different places, then the protection from fire becomes irrelevant.
+[deleted]
 
 comment: p314aqf
 parent: t1_p2wot6t
@@ -225,11 +225,11 @@
 
 comment: p37yae0
 parent: t1_p34nv75
-author: Armadillodillodillo
+author: [deleted]
 created_utc: 1786534153
 edited: false
 body:
-And your other copy in other place would be completely safe. Your reading comprehension is bit lacking.
+[deleted]
 
 comment: p3884b2
 parent: t1_p2x7rxj
@@ -246,3 +246,47 @@
 edited: false
 body:
 Where would you store the other copy besides your home? Please don't tell me you keep a copy in a safety deposit box or your comprehension is a bit lacking
+
+comment: p3fxias
+parent: t1_p38b8iz
+author: [deleted]
+created_utc: 1786628209
+edited: false
+body:
+[deleted]
+
+comment: p3g82og
+parent: t1_p3fxias
+author: lysergicpoptarts
+created_utc: 1786631090
+edited: false
+body:
+Ah yes. Tell us adults here with dead parents. Wait till you grow up and you don't have anymore family or people you trust to leave seed phrases with 10's-100's of thousands of dollars at their house. Top tier security.
+
+comment: p3g8e8l
+parent: t1_p3g82og
+author: PeroniBites
+created_utc: 1786631174
+edited: false
+body:
+This guy is a complete fool. Dont listen to him 
+
+comment: p3gezy3
+parent: t1_p3g8e8l
+author: lysergicpoptarts
+created_utc: 1786632900
+edited: false
+body:
+You're right. He said you have a reading comprehension problem and his method is %100 percent secure only to delete his comment when called out and proven wrong. That's reddit for ya.
+
+comment: p3hl6oo
+parent: t3_1vkqgos
+author: xirvin
+created_utc: 1786643555
+edited: false
+body:
+Op  you should look at cheaper options to update your steel plate seeds. There are some steel plates you can reuse if you change your seeds
+
+Like [https://a.co/d/0gTJZSu6](https://a.co/d/0gTJZSu6) or [https://a.co/d/0bHdnPqR](https://a.co/d/0bHdnPqR) [https://a.co/d/0i4oInDJ](https://a.co/d/0i4oInDJ)
+
+I don't use steel for my backup seed but learning from coldcard experience I will use a steel plate I can change my seed  moving forward
