--- reddit-coldcard-misinformation-warning@20260807T125854Z
+++ reddit-coldcard-misinformation-warning@20260808T150601Z
@@ -275,11 +275,11 @@
 
 comment: p22xe52
 parent: t1_p1n10lr
-author: evgeniy_pp
+author: [deleted]
 created_utc: 1786031172
 edited: false
 body:
-Who are these people who selected 24 words and dice, but was too lazy to finish the job and stoped halfway rolling?
+[deleted]
 
 comment: p281ntn
 parent: t1_p22xe52
