--- reddit-coldcard-misinformation-warning@20260806T163624Z
+++ reddit-coldcard-misinformation-warning@20260807T125854Z
@@ -280,3 +280,11 @@
 edited: false
 body:
 Who are these people who selected 24 words and dice, but was too lazy to finish the job and stoped halfway rolling?
+
+comment: p281ntn
+parent: t1_p22xe52
+author: ukieninger
+created_utc: 1786088966
+edited: false
+body:
+you are completely missing the point here
