--- reddit-coldcard-misinformation-warning@20260805T210426Z
+++ reddit-coldcard-misinformation-warning@20260806T163624Z
@@ -272,3 +272,11 @@
 edited: false
 body:
 Away at work for another couple months. No way to check anything. Used 300 dice rolls…. Hoping I’m ok. If I’m not coinkite will be getting an in person visit. 
+
+comment: p22xe52
+parent: t1_p1n10lr
+author: evgeniy_pp
+created_utc: 1786031172
+edited: false
+body:
+Who are these people who selected 24 words and dice, but was too lazy to finish the job and stoped halfway rolling?
