--- bitcoinmag-red-team-390-repos@20260806T055814Z
+++ bitcoinmag-red-team-390-repos@20260807T205409Z
@@ -102,17 +102,19 @@
 Copy URL
 Previous article
 Russia Passes Law to Regulate Crypto Exchanges, Keeps Payment Ban in Place
+Next article
+White House Now Reviewing Crypto Clarity Act Ethics Text Ahead of August Deadline: Report
 Juan Galt
 RELATED ARTICLES
-CULTURE
-Self Custody Is Dead. Long Live Self Custody
-August 4, 2026
+TECHNICAL
+Bitcoin’s BIP110 Moment: Three Possible Scenarios
+August 7, 2026
 BUSINESS
-Coinkite Releases Fixed Firmware After Coldcard Bug; AI Likely Involved In The Breach
-July 31, 2026
-NEWS
-COLDCARD SECURITY RISK: IMMEDIATE ACTION REQUIRED
-July 31, 2026
+Breez Announces Glow, an Open Source Bitcoin to Stablecoins Progressive Web App
+August 6, 2026
+TECHNICAL
+The End of the Closed-Source Era Is at Hand: Obscurity Was Never Security
+August 6, 2026
 Bitcoin Portfolio Tracker & Media Updates
 Bitcoin BTC/USD
 $0.00
@@ -125,14 +127,14 @@
 Error loading data. Check console for details.
 VIEW 150+ BITCOIN CHARTS
 LATEST NEWS
-Russia Passes Law to Regulate Crypto Exchanges, Keeps Payment Ban in Place
-August 5, 2026
-‘If Clarity Dies, Democrats Killed It’: Lummis Urges Senate to Act on Crypto Bill Before Recess
-August 5, 2026
-Bitcoin Treasury Firm Strategy Pledges $250 a Year to Employee Trump Accounts
-August 5, 2026
-Self Custody Is Dead. Long Live Self Custody
-August 4, 2026
+Coldcard Bitcoin Hack: Victims Report Median Loss of 1 BTC as Theft Tops $111 Million
+August 7, 2026
+Bitcoin Shrugs off Coldcard Hack and Clarity Act Delays, Price Chops Higher as Investors Buy ETFs
+August 7, 2026
+Bitcoin ETFs Add Nearly $800 Million in the Wake of Coldcard Exploit
+August 7, 2026
+Crucial Vote On Crypto Clarity Act Delayed Until September, Trump Says More People Are Paying in Bitcoin: Reports
+August 7, 2026
 Load more
 EDITOR PICKS
 President Harris Should Buy Bitcoin to Pay Black Americans Reparations
@@ -149,11 +151,11 @@
 New Pi Cycle Top Prediction Chart Identifies Bitcoin Price Market Peaks with Precision
 January 6, 2025
 CATEGORIES
-BUSINESS4305
+BUSINESS4306
 CULTURE3586
 MARKETS2428
-NEWS1488
-TECHNICAL1340
+NEWS1499
+TECHNICAL1342
 INDUSTRY EVENTS366
 PRESS RELEASES292
 LEGAL206
