--- reddit-not-your-keys-not-your-coins@20260804T181737Z
+++ reddit-not-your-keys-not-your-coins@20260804T184837Z
@@ -827,3 +827,37 @@
 edited: false
 body:
 Seems entirely reasonable! But so to did trusting the cold card to generate a seed…
+
+comment: p1p7vi5
+parent: t1_p1o46gb
+author: WeekendQuant
+created_utc: 1785867848
+edited: false
+body:
+Dice can never be cracked.
+
+comment: p1p9ogc
+parent: t1_p1p7vi5
+author: piejlucas
+created_utc: 1785868298
+edited: false
+body:
+The hash algorithm that uses the dice results can always be faulty
+
+comment: p1p9tux
+parent: t1_p1p1suo
+author: andrew869
+created_utc: 1785868335
+edited: false
+body:
+Right, how could I forget the magical money printer 
+
+comment: p1pa1xy
+parent: t1_p1nwtf6
+author: cipherjones
+created_utc: 1785868391
+edited: false
+body:
+It was a methodology that failed, you can't say only .01% of wallets were affected with the information given. 
+
+One in 33800 holders. So it would equate to 140,000, Visa accounts or 90,000 MasterCard accounts being drained.
