--- reddit-not-your-keys-not-your-coins@20260804T225003Z
+++ reddit-not-your-keys-not-your-coins@20260804T232554Z
@@ -357,6 +357,14 @@
 body:
 Not a human drawing, not a human meme
 
+comment: p1o7sj4
+parent: t1_p1njymf
+author: GeeEyeDoe
+created_utc: 1785858878
+edited: false
+body:
+Carved seed phrase into sandstone blocks located in the middle of a booby trapped pyramid. Like how the ancient Egyptians used to store their seed phrase. 
+
 comment: p1o8iyc
 parent: t1_p1o50jm
 author: tribepride25
