--- reddit-not-your-keys-not-your-coins@20260806T163615Z
+++ reddit-not-your-keys-not-your-coins@20260807T061929Z
@@ -1048,11 +1048,11 @@
 
 comment: p1pum56
 parent: t1_p1plxvu
-author: Nick700
+author: [deleted]
 created_utc: 1785873698
 edited: false
 body:
-Yeah they aren't at risk of 100% of their funds disappearing but in less than 10 years 25% of their value has been siphoned away
+[deleted]
 
 comment: p1px9q0
 parent: t1_p1po2bg
@@ -1098,11 +1098,11 @@
 
 comment: p1q62u3
 parent: t1_p1q3kyo
-author: Nick700
+author: [deleted]
 created_utc: 1785876742
 edited: false
 body:
-Unfortunate for those who gambled their emergency fund on BTC otherwise who cares BTC/USD is up 1300% in that same timeframe USD inflated 25%
+[deleted]
 
 comment: p1qcs6z
 parent: t1_p1pnd0i
@@ -1557,6 +1557,22 @@
 
 So what are people to do? I don't have any answers.
 
+comment: p26kncc
+parent: t1_p1tniak
+author: puck2
+created_utc: 1786067330
+edited: false
+body:
+mainstream adoption can do the other things (coinbase, ETF, robinhood, cashapp, etrade, fidelity, circle, etc)
+
+comment: p26l4xc
+parent: t1_p1u3uqn
+author: puck2
+created_utc: 1786067491
+edited: false
+body:
+is that a 256 bits of entropy able to be hacked by a superintelligent AI??
+
 more-stub: parent t1_p1v7iiv count <live-count>
 
 more-stub: parent t1_p1u76ef count <live-count>
