--- reddit-not-your-keys-not-your-coins@20260804T194839Z
+++ reddit-not-your-keys-not-your-coins@20260804T201909Z
@@ -983,3 +983,27 @@
 edited: false
 body:
 ohhh my sweet sweet summer child...
+
+comment: p1pum56
+parent: t1_p1plxvu
+author: Nick700
+created_utc: 1785873698
+edited: false
+body:
+Yeah they aren't at risk of 100% of their funds disappearing but in less than 10 years 25% of their value has been siphoned away
+
+comment: p1px9q0
+parent: t1_p1po2bg
+author: lapideous
+created_utc: 1785874405
+edited: false
+body:
+“Actual dollars”
+
+comment: p1py1mj
+parent: t1_p1px9q0
+author: Forded_Fiction24
+created_utc: 1785874609
+edited: false
+body:
+What are you getting at? Once you spend dollars on any stock or asset it's no longer "actual dollars" either, but the represented valuation of the underlying asset notated in a dollar amount. However, it's shares of a company (company ownership) convertible to "actual dollars" upon sale of the shares. This isn't really any different from Bitcoin, real estate etc
