--- reddit-not-your-keys-not-your-coins@20260804T184837Z
+++ reddit-not-your-keys-not-your-coins@20260804T191758Z
@@ -861,3 +861,63 @@
 It was a methodology that failed, you can't say only .01% of wallets were affected with the information given. 
 
 One in 33800 holders. So it would equate to 140,000, Visa accounts or 90,000 MasterCard accounts being drained.
+
+comment: p1pe2wy
+parent: t1_p1odd7g
+author: TheresNoSecondBest
+created_utc: 1785869402
+edited: false
+body:
+That would be a step backwards.
+
+comment: p1pfcuw
+parent: t1_p1p9ogc
+author: WeekendQuant
+created_utc: 1785869722
+edited: false
+body:
+Verify your seeds with ian Coleman's BIP-39
+
+comment: p1pft6f
+parent: t1_p1p07iv
+author: tribepride25
+created_utc: 1785869836
+edited: false
+body:
+Fidelity has a policy that if your funds are lost and it’s their fault ie from hacking (not just crypto but IRA, 401k etc),  then they pay you back. That’s one of the reasons I feel comfortable with them. They also keep your crypto 1:1 without lending like Voyager, FTX did. I don’t worry about my Fidelity IRA getting stolen or hacked so I said why worry about my Fidelity crypto getting stolen or hacked
+
+comment: p1pghwe
+parent: t1_p1oye81
+author: infernal_celery
+created_utc: 1785870012
+edited: false
+body:
+NooOoOOoo!!!!!1110101010001…2.
+
+comment: p1pgsq5
+parent: t1_p1p2418
+author: nmhaas
+created_utc: 1785870091
+edited: false
+body:
+No, it has been 'source verified' since 2021, I believe. Coincidentally the same year that new seeds started being worthless. 
+
+comment: p1phntd
+parent: t1_p1oznit
+author: tribepride25
+created_utc: 1785870314
+edited: false
+body:
+This. And it’s kinda ridiculous to just hear people talk about it and I understand why Buttcoiners joke on us. It’s like “what? you didn’t roll the dice, create a pin, set a passphrase, multi sig, air gap, shake your left leg, update firm ware, say a prayer, download the latest version, put your keys in a metal plate, put keys in lock box, verify code using AI?” What did you expect? Of course you got hacked and lost your life savings. How could you be this stupid? /s
+
+comment: p1phq80
+parent: t1_p1njymf
+author: LurkerFromTheVoid
+created_utc: 1785870332
+edited: false
+body:
+So...Diversify your Storage.
+ 
+Trust No One with Everything.
+
+Just like Stocks.
