--- reddit-not-your-keys-not-your-coins@20260804T191758Z
+++ reddit-not-your-keys-not-your-coins@20260804T194839Z
@@ -101,9 +101,9 @@
 parent: t1_p1nqb7z
 author: AvailableTie6834
 created_utc: 1785855484
-edited: false
-body:
-and also deny people from withdrawning their money during a bank run
+edited: 1785872376
+body:
+and also deny people from withdrawing their money during a bank run
 
 comment: p1nupmi
 parent: t1_p1nqb7z
@@ -921,3 +921,65 @@
 Trust No One with Everything.
 
 Just like Stocks.
+
+comment: p1pkvlp
+parent: t1_p1phq80
+author: Forded_Fiction24
+created_utc: 1785871144
+edited: false
+body:
+I do.  I have separate wallets and addresses for my self-custody Bitcoin. That doesn't give me enough comfort in of itself though still which is why I hold some on a couple different exchanges and also have some shares of IBIT and FBTC as well
+
+comment: p1plxvu
+parent: t1_p1njymf
+author: Time_Jump8047
+created_utc: 1785871419
+edited: false
+body:
+Meanwhile all the people who use actual dollars and have the security and backing of regulated banks have a much closer to perfect way of holding their money. This shit is ridiculous people are losing their life savings
+
+comment: p1pnd0i
+parent: t1_p1o66eq
+author: D_Anargyre
+created_utc: 1785871790
+edited: false
+body:
+Electrum wallet on tails using tor on a no hard drive pc.
+Manual entropy.
+
+Keys noted on paper and nothing else.
+The human holding that paper owns those btc.
+
+comment: p1pnhnu
+parent: t1_p1nljav
+author: D_Anargyre
+created_utc: 1785871824
+edited: false
+body:
+Best hardware wallet is a piece of paper
+
+comment: p1po2bg
+parent: t1_p1plxvu
+author: Forded_Fiction24
+created_utc: 1785871974
+edited: false
+body:
+I mean not with Bitcoin holdings they don't. None of it is sipc insured and all is hosted on exchanges regardless If you're buying a BTC spot ETF, leaving it on an exchange or taking custody yourself. Regular bank insurance policies don't cover digital assets. These custodians probably have much more security than an individual, but exchanges have been taken down in the past too. 
+
+Now if what you're saying is having all your life savings in Bitcoin, however you're storing it, yeah I agree that's stupid. I would never have all my life savings in Bitcoin or another single basket otherwise. Very short-sighted indeed, so yeah the people that made these claims that they lost everything were being reckless. Not saying they deserve it, Play stupid games, win stupid prizes though
+
+comment: p1ppi3p
+parent: t1_p1p5wc2
+author: AvailableTie6834
+created_utc: 1785872355
+edited: false
+body:
+iancoleman seems to have more eyes than coldcard really
+
+comment: p1ppoj6
+parent: t1_p1p2ed5
+author: AvailableTie6834
+created_utc: 1785872402
+edited: false
+body:
+ohhh my sweet sweet summer child...
