--- reddit-not-your-keys-not-your-coins@20260804T232554Z
+++ reddit-not-your-keys-not-your-coins@20260804T234926Z
@@ -381,6 +381,14 @@
 body:
 Building your own hardware wallet and software lol
 
+comment: p1o8qje
+parent: t1_p1nerqy
+author: SingularityCentral
+created_utc: 1785859116
+edited: false
+body:
+You can put gold coins under your bed and "be your own bank". Doesn't mean it is a good idea. 
+
 comment: p1o9q1q
 parent: t1_p1o24p6
 author: shabusnelik
@@ -473,13 +481,13 @@
 body:
 Yeah I guess if there is another 100 year crash (referring to the Great Depression and then the Great Recession 100 years later), then my funds may be a risk. I do feel better knowing they survived the Great Recession so hopefully that means they aren’t run by a bunch of assholes 
 
-comment: p1ogi2o
-parent: t3_1vfak2k
-author: Adius_Omega
-created_utc: 1785861075
-edited: false
-body:
-There has to be a point where you have to just accept that this whole Bitcoin thing just didn't quite work out the way we all thought it would.
+comment: p1of3ep
+parent: t3_1vfak2k
+author: JackPembroke
+created_utc: 1785860724
+edited: false
+body:
+...maaaybe gold coins arent so bad looking after all lol
 
 comment: p1ogxu4
 parent: t1_p1o6gaw
@@ -1198,3 +1206,27 @@
 If the coins were physical, would you store them all in one place?
 
 You need to hide a small portion in each storage method
+
+comment: p1r2733
+parent: t1_p1nerqy
+author: Lopitoz07
+created_utc: 1785886008
+edited: false
+body:
+Be your own bank ? Like Celsius!!! 
+
+comment: p1r3mwl
+parent: t1_p1nvrit
+author: BZ852
+created_utc: 1785886454
+edited: false
+body:
+Be your own bank! Be your own FDIC insurance!
+
+comment: p1r4md8
+parent: t3_1vfak2k
+author: Informal-Special-984
+created_utc: 1785886762
+edited: false
+body:
+what is mpc?
