--- reddit-not-your-keys-not-your-coins@20260807T061929Z
+++ reddit-not-your-keys-not-your-coins@20260808T213544Z
@@ -405,14 +405,6 @@
 body:
 Svb and lehman are banks that went tits up. If an exchange went tits up (i.e. all their coins got stolen) you won't see a reimbursement either, no matter what their policy is.
 
-comment: p1ob81j
-parent: t3_1vfak2k
-author: Just_Bluebird_5268
-created_utc: 1785859749
-edited: false
-body:
-It's the future of money
-
 comment: p1obi5k
 parent: t1_p1nz6f7
 author: AvailableTie6834
