--- reddit-not-your-keys-not-your-coins@20260804T214854Z
+++ reddit-not-your-keys-not-your-coins@20260804T221913Z
@@ -1102,3 +1102,19 @@
 edited: false
 body:
 That non existent hard drive ? 
+
+comment: p1qklpg
+parent: t1_p1ok9s6
+author: metalzip
+created_utc: 1785880723
+edited: false
+body:
+what are you rambling on about - just use a secure way to store keys. try the multisignature method
+
+comment: p1qlkgt
+parent: t1_p1q3kyo
+author: Letterhead-Warm
+created_utc: 1785880998
+edited: false
+body:
+Lol
