--- reddit-not-your-keys-not-your-coins@20260804T211901Z
+++ reddit-not-your-keys-not-your-coins@20260804T214854Z
@@ -1051,3 +1051,54 @@
 edited: false
 body:
 Unfortunate for those who gambled their emergency fund on BTC otherwise who cares BTC/USD is up 1300% in that same timeframe USD inflated 25%
+
+comment: p1qcs6z
+parent: t1_p1pnd0i
+author: Not-the-best-name
+created_utc: 1785878549
+edited: false
+body:
+What a terrible way to secure late sums of money
+
+comment: p1qdpjl
+parent: t3_1vfak2k
+author: El_Zilcho
+created_utc: 1785878799
+edited: false
+body:
+Not your AI, not your content.
+
+comment: p1qeppp
+parent: t1_p1pnd0i
+author: Gosat
+created_utc: 1785879075
+edited: false
+body:
+That harddrive can crash. 
+
+That paper can get lost or stolen. 
+
+comment: p1qh4cn
+parent: t3_1vfak2k
+author: Dbear_son
+created_utc: 1785879745
+edited: false
+body:
+The rolling dice part is wild
+
+
+comment: p1qhooz
+parent: t1_p1pum56
+author: Tebasaki
+created_utc: 1785879904
+edited: false
+body:
+I think people are forgetting how much pizza you could buy in 2012 with btc vs now. In cash the system steals your money if you dont donate leadt two things: don't lose it and beat inflation every year. 
+
+comment: p1qi0zv
+parent: t1_p1qeppp
+author: D_Anargyre
+created_utc: 1785879999
+edited: false
+body:
+That non existent hard drive ? 
