--- stackernews-first-drain-report@20260805T004218Z
+++ stackernews-first-drain-report@20260805T142051Z
@@ -281,6 +281,13 @@
             "text": "![](https://m.stacker.news/150227)",
             "user": {
               "name": "npub1zapsats"
+            }
+          },
+          {
+            "createdAt": "2026-08-05T13:11:10.781Z",
+            "text": "I know a person who builds drainers if you guys want I can urge him to sell that drainer to someone who can review the drainer and update the coldcard wallet to prevent such drains... I have that person in contact",
+            "user": {
+              "name": "QuietHorizon"
             }
           },
           {
