--- reddit-dice-seed-mistakes@20260808T050242Z
+++ reddit-dice-seed-mistakes@20260808T150816Z
@@ -145,3 +145,21 @@
 These recent hacks are happening because the attacker knows the _exact algorithm_ that was used to create a very, very, very narrow range of possible SPECIFIC sequences, NOT because anyone didn't roll their dice right.
 
 I sympathize with anyone who has any form of spectrum disorder that's causing this kind of hyper-fixation, but exactly how you roll any given dice absolutely will not result in any degree of "insufficient entropy" that would make your phrase any more meaningfully vulnerable to attack. 
+
+comment: p2fkmo4
+parent: t1_p28bx4k
+author: TechnologyGrouchy679
+created_utc: 1786181975
+edited: false
+body:
+in Celsius or Fahrenheit?
+
+comment: p2fl2dz
+parent: t1_p2fkmo4
+author: Skinny_Human
+created_utc: 1786182202
+edited: false
+body:
+Celsius of course!!
+
+*continues licking window of bus*
