--- reddit-coinkite-next-steps@20260804T083751Z
+++ reddit-coinkite-next-steps@20260804T151218Z
@@ -319,3 +319,27 @@
 edited: false
 body:
 Minus the one dumbass who made this bug
+
+comment: p1nrvgt
+parent: t1_p16iduc
+author: Solid_Wolverine1639
+created_utc: 1785854755
+edited: false
+body:
+Happy for the criminals... Imagine it could just be one guy... Who's got to be a certain amount of theft that should warrant capital punishment...  This is more than enough of an example
+
+comment: p1ns5qi
+parent: t1_p1akf4h
+author: Solid_Wolverine1639
+created_utc: 1785854830
+edited: false
+body:
+What about seedless like tangem
+
+comment: p1nsfms
+parent: t1_p1bls4t
+author: Solid_Wolverine1639
+created_utc: 1785854902
+edited: false
+body:
+They should have paid for audits...tangem has...
