--- reddit-coinkite-next-steps@20260807T192610Z
+++ reddit-coinkite-next-steps@20260808T145950Z
@@ -157,11 +157,11 @@
 
 comment: p18cagf
 parent: t3_1vd5f5h
-author: ToneCapwn
+author: [deleted]
 created_utc: 1785667092
 edited: false
 body:
-I will keep buying their product because of their open source and transparency. Ledger is not open source 
+[deleted]
 
 comment: p18ir6m
 parent: t1_p188xlt
@@ -351,3 +351,21 @@
 edited: false
 body:
 Lol smooooooooooth brain 
+
+comment: p2fxw0d
+parent: t1_p1bls4t
+author: Skinny_Human
+created_utc: 1786188349
+edited: false
+body:
+This should be the top comment 
+
+All the CC fanboys in this thread, human psychology is fascinating 
+
+comment: p2g41w5
+parent: t1_p1ajhbj
+author: Skinny_Human
+created_utc: 1786190879
+edited: false
+body:
+That’s like saying ‘the food in this restaurant is safe,provided you don’t eat there ’
