--- reddit-plot-thickens-coldcard-hack@20260815T060010Z
+++ reddit-plot-thickens-coldcard-hack@20260815T123052Z
@@ -198,3 +198,11 @@
 edited: false
 body:
 The fact this issue was known for so long. Was dismissed and minimized internally it feels 100% like a retirement plan
+
+comment: p3tl6na
+parent: t1_p3phdsl
+author: Ok-Courage-5115
+created_utc: 1786792514
+edited: false
+body:
+It can well be classed as leak. It's like someone leaking 10 documents from some goverment office, saying: "You'll find one of them very interesting".
