--- reddit-plot-thickens-coldcard-hack@20260815T015837Z
+++ reddit-plot-thickens-coldcard-hack@20260815T060010Z
@@ -101,21 +101,19 @@
 
 comment: p3qhs7p
 parent: t1_p3p6y36
-author: FTP_FTP_
+author: [deleted]
 created_utc: 1786745934
 edited: false
 body:
-You don't know what the word leak means and that's not opinion. We can tell you don't work in threat intelligence or infosec. 
+[deleted]
 
 comment: p3qi0q3
 parent: t1_p3plbxk
-author: FTP_FTP_
+author: [deleted]
 created_utc: 1786746007
 edited: false
 body:
-That's not what happened at all. 
-
-"Got the keys" 😂 😂 😂 
+[deleted]
 
 comment: p3qi82x
 parent: t1_p3qhs7p
@@ -137,11 +135,11 @@
 
 comment: p3qiov0
 parent: t1_p3qi82x
-author: FTP_FTP_
+author: [deleted]
 created_utc: 1786746210
 edited: false
 body:
-You don't know what the word leak means. Any other questions? 
+[deleted]
 
 comment: p3qix5j
 parent: t1_p3qiov0
@@ -150,17 +148,6 @@
 edited: false
 body:
 I didn't ask you anything. Can you read?
-
-comment: p3qj64d
-parent: t1_p3qix5j
-author: FTP_FTP_
-created_utc: 1786746360
-edited: false
-body:
-"
-What are you people even arguing"
-
-You were confused and asked what we are arguing. I set you correct. Dumbass
 
 comment: p3qjurq
 parent: t1_p3qix5j
@@ -195,3 +182,19 @@
 edited: false
 body:
 I’ll forward this comment to the other researcher, keep looking 
+
+comment: p3rmxpi
+parent: t3_1vobf3s
+author: data_diver
+created_utc: 1786759856
+edited: 1786760234
+body:
+Well if researchers couldn’t do it in a week, I’m sure a good hacking collective couldn’t have done it in the years they had.  Must have been an inside job.
+
+comment: p3rov9p
+parent: t1_p3qu621
+author: lobhater
+created_utc: 1786760578
+edited: false
+body:
+The fact this issue was known for so long. Was dismissed and minimized internally it feels 100% like a retirement plan
