--- stackernews-spouse-response@20260804T041513Z
+++ stackernews-spouse-response@20260804T101059Z
@@ -3,6 +3,13 @@
     "item": {
       "comments": {
         "comments": [
+          {
+            "createdAt": "2026-07-31T12:08:49.164Z",
+            "text": "Breakfast this morning turned into a tutorial on generating sufficient entropy.",
+            "user": {
+              "name": "billytheked"
+            }
+          },
           {
             "createdAt": "2026-07-31T12:00:26.399Z",
             "text": "Sell everything!!!\n\nBut she says that every bear market. Could be a bottom signal!",
@@ -22,13 +29,6 @@
             "text": "This whole thing reminds me of linux neck-bearded nerds: they act like if not knowing how to use the computer by interacting directly with the kernel is a sin and assume everyone should know, and scoff on those who don't know. They assume their world must be everyone's. But 99.99% of people just want to send an email and buy that new appliance from amazon. The exact same scenario applies to bitcoin.",
             "user": {
               "name": "didiplaywell"
-            }
-          },
-          {
-            "createdAt": "2026-07-31T12:08:49.164Z",
-            "text": "Breakfast this morning turned into a tutorial on generating sufficient entropy.",
-            "user": {
-              "name": "billytheked"
             }
           },
           {
