--- reddit-still-trust-coinkite@20260805T075533Z
+++ reddit-still-trust-coinkite@20260806T163126Z
@@ -710,15 +710,15 @@
 parent: t3_1vc9jqu
 author: Flowa-Powa
 created_utc: 1785785896
-edited: false
-body:
-My Coldcard Q has a broken usb reader, and now this omnishambles.
+edited: 1786018845
+body:
+My Coldcard Q has a broken SD card reader, and now this omnishambles.
 
 It was a genuine error though. There was a very well designed randomiser built in, but it wasn't being used, and was deferring to a very low quality randomiser that the developer didn't even know existed.
 
 The real error was not empirically testing the output for entropy.
 
-I don't know. I'm conflicted. If you update the firmware, generate with dice and layer a passphrase on top you really should be alright. 
+I don't know. I'm conflicted. If you update the firmware, generate with dice and layer a passphrase on top you really should be alright.
 
 comment: p1ryc7f
 parent: t3_1vc9jqu
