--- reddit-trezor-community-audit-call@20260806T032917Z
+++ reddit-trezor-community-audit-call@20260806T163252Z
@@ -700,3 +700,11 @@
 edited: false
 body:
 What is a good tool or tutorial to use for flipping the coin method that is trusted? After rolling dice or flipping a coin, how does that translate to BIP 39 seed and passphrase? 
+
+comment: p22lwaj
+parent: t1_p1j6frl
+author: Ok-Mammoth552
+created_utc: 1786028234
+edited: false
+body:
+AI is better at finding exploits than securing them for the very specific reason that cybersecurity is inherently assymetrical, even for humans. It is always harder to secure a system than break it, just like it's always harder to make a card castle than knock one down. The security professional has to protect against all imaginable threats; the hacker just has to find one vulnerability.
