--- reddit-trezor-community-audit-call@20260804T232156Z
+++ reddit-trezor-community-audit-call@20260804T234626Z
@@ -662,3 +662,11 @@
 To systemically go through the codebase requires organization and knowledge. The people with the skills are probably working real jobs for money which leaves the trezor validation team to check the code.
 
 For some reason anytime someone yells audit I think of those first amendment auditors that annoy me by the post office
+
+comment: p1r47ee
+parent: t1_p1qx2hy
+author: Kooriki
+created_utc: 1785886632
+edited: false
+body:
+IM NOT ENCRYPTING I'M TRAVELLING. I DO NOT CONSENT TO CREATING JOINDER WITH THE BLOCKCHAIN
