--- reddit-trezor-community-audit-call@20260804T134355Z
+++ reddit-trezor-community-audit-call@20260804T144352Z
@@ -558,3 +558,19 @@
 edited: false
 body:
 How safe is electrum ?
+
+comment: p1nlffv
+parent: t3_1veoo6t
+author: 00-SilverShot
+created_utc: 1785853042
+edited: false
+body:
+I'd like the help, but unfortunately, I have zero experience coding software.  I'm just a mid level IT admin and tech support guy who dropped out of community college.
+
+comment: p1nm10f
+parent: t1_p1m7jed
+author: naked_number_one
+created_utc: 1785853204
+edited: false
+body:
+[ Removed by Reddit ]
