--- reddit-trezor-community-audit-call@20260804T093400Z
+++ reddit-trezor-community-audit-call@20260804T134355Z
@@ -508,3 +508,53 @@
 edited: false
 body:
 Wouldn't (or shouldn't) frontier models refuse to do this?
+
+comment: p1m9mu8
+parent: t1_p1jeplf
+author: jungle
+created_utc: 1785836473
+edited: false
+body:
+Also go back the git history. If previous versions had a weak random generation, no updates would have fixed those wallets.
+
+comment: p1mfq6d
+parent: t1_p1lektv
+author: opossum_cz
+created_utc: 1785839277
+edited: false
+body:
+Do you know what is great? You don't need to, don't generate seeds on the devices at all. Flip a fucking coin.
+
+comment: p1mk60k
+parent: t3_1veoo6t
+author: Murky_Ad6160
+created_utc: 1785841136
+edited: false
+body:
+I use OneKey and I never see it mentioned. Is it not good?
+
+comment: p1msfe7
+parent: t3_1veoo6t
+author: SuchTrezorVeryCrypto
+created_utc: 1785844270
+edited: false
+body:
+Hi from Trezor,
+
+do it
+
+comment: p1n5byu
+parent: t1_p1leqxd
+author: Final_Bite_7228
+created_utc: 1785848484
+edited: false
+body:
+Hey! Community!
+
+comment: p1n9g84
+parent: t3_1veoo6t
+author: Beatrix_0000
+created_utc: 1785849702
+edited: false
+body:
+How safe is electrum ?
