--- reddit-trezor-community-audit-call@20260805T001639Z
+++ reddit-trezor-community-audit-call@20260805T142520Z
@@ -678,3 +678,17 @@
 edited: false
 body:
 So funny
+
+comment: p1tbbty
+parent: t3_1veoo6t
+author: SmartPipe3882
+created_utc: 1785916975
+edited: 1785917173
+body:
+What about if, instead of trusting each other - completely without vetting, on Reddit of all places - instead of Trezor-et-al, we just don’t generate seed phrases on device if we’re that concerned that the generation process is compromised?
+
+Use genuine physical randomness to select from the 1024 words.
+
+I’ve seen these things on Star Wars, Watto called them chance cubes, I’m sure that we as a community could come up with some sort of IRL proxy for this sort of thing? Perhaps we could make these cubes even more random by numbering the sides? Or, if we like binary chance, maybe we just streamline the physical product by redesigning it with only two faces?
+
+Then all we really have to worry about it making sure we generate our seed phrases well away from any Jedi. And I feel like we can all do that much easier and more reliably than trusting each other to learn cryptography.
