--- reddit-trezor-community-audit-call@20260804T191443Z
+++ reddit-trezor-community-audit-call@20260804T194546Z
@@ -610,3 +610,11 @@
 edited: false
 body:
 Thats done absolutely zilch to harm its security. 
+
+comment: p1pp5ky
+parent: t1_p1pf488
+author: Laukess
+created_utc: 1785872262
+edited: false
+body:
+So it's air gapped because you don't see the value in it, even though it's not actually air gapped?
