--- reddit-trezor-community-audit-call@20260804T201612Z
+++ reddit-trezor-community-audit-call@20260804T232156Z
@@ -483,6 +483,14 @@
 body:
 [removed]
 
+comment: p1llcdn
+parent: t1_p1lektv
+author: Full-Atmosphere-4818
+created_utc: 1785824472
+edited: false
+body:
+According to someone on another thread, Ledger AND Tangem have only one way to make the entropy vs several methods for other options. The third one that had only one method was ColdCard. 
+
 comment: p1lp6nq
 parent: t3_1veoo6t
 author: joeyx22lm
@@ -494,6 +502,22 @@
 And if you're not money laundering (using bitcoin?), you're probably safest leaving it in a US based exchange, where there is regulatory oversight.
 
 Want to be a cyberpunk defi tech boi, store your seed phrase alongside your PGP key.
+
+comment: p1ltzte
+parent: t1_p1ipx3q
+author: Spy008
+created_utc: 1785828649
+edited: false
+body:
+Not wasting my time responding to each comment. My point is there is A LOT of eyes on Trezor’s code.
+
+For open source to be secure you need two things Time + target. The longer the time (history of working) the better the security is, the more appealing/lucrative the target is the better the security is.
+
+Think of it this way you’re a researcher- finding a flaw in Trezor’s code would get you on the front page news. Finding a flaw in SuperDuperSecurityTheater wallet might have some random guy mention it on Reddit. (Hell even the constant repetition of the physical vulnerability of model 1 and T get a front page news article every few years someone rediscovers it)
+
+Audit the code, idc, but the chance that some random Redditor is going to find anything that experienced professionals in the field havent is near 0.
+
+But but coldcard… no one was looking at this crap. Literally the incentive (Target) was incredibly small, outside of this sub where it was constantly peddled. coldcard’s user base was small and their business practices did nothing to encourage outside scrutiny.
 
 comment: p1m7jed
 parent: t1_p1ja9ho
@@ -626,3 +650,15 @@
 edited: false
 body:
 Are we even in the same discussion here?? You said the Safe 5 is air gapped. It’s not. Full stop. 
+
+comment: p1qx2hy
+parent: t1_p1iotom
+author: curiousengineer601
+created_utc: 1785884407
+edited: false
+body:
+Lol. The vast majority of people don’t have any of the skills needed to contribute to any ‘audit’.
+
+To systemically go through the codebase requires organization and knowledge. The people with the skills are probably working real jobs for money which leaves the trezor validation team to check the code.
+
+For some reason anytime someone yells audit I think of those first amendment auditors that annoy me by the post office
