--- reddit-trezor-community-audit-call@20260805T142520Z
+++ reddit-trezor-community-audit-call@20260806T032917Z
@@ -692,3 +692,11 @@
 I’ve seen these things on Star Wars, Watto called them chance cubes, I’m sure that we as a community could come up with some sort of IRL proxy for this sort of thing? Perhaps we could make these cubes even more random by numbering the sides? Or, if we like binary chance, maybe we just streamline the physical product by redesigning it with only two faces?
 
 Then all we really have to worry about it making sure we generate our seed phrases well away from any Jedi. And I feel like we can all do that much easier and more reliably than trusting each other to learn cryptography.
+
+comment: p1z14hs
+parent: t1_p1mfq6d
+author: privacymatterznow
+created_utc: 1785979194
+edited: false
+body:
+What is a good tool or tutorial to use for flipping the coin method that is trusted? After rolling dice or flipping a coin, how does that translate to BIP 39 seed and passphrase? 
