--- reddit-attacker-execution-critique@20260808T020425Z
+++ reddit-attacker-execution-critique@20260808T083636Z
@@ -1752,7 +1752,7 @@
 
 comment: p23v33d
 parent: t3_1vh47pj
-author: RevolutionaryQuit487
+author: [deleted]
 created_utc: 1786039468
 edited: false
 body:
@@ -3196,4 +3196,36 @@
 body:
 how do we know his is using a public node and not a local node to broadcast the transactions behind Tor?
 
+comment: p2e9vt2
+parent: t3_1vh47pj
+author: OrangePillar
+created_utc: 1786159909
+edited: false
+body:
+Not a bitcoiner 
+
+comment: p2eqo6a
+parent: t1_p28flwi
+author: ZascandileandoAndo
+created_utc: 1786166897
+edited: false
+body:
+Kraken was hacked?
+
+comment: p2ew5bv
+parent: t1_p2e9vt2
+author: ineedanamegenerator
+created_utc: 1786169489
+edited: false
+body:
+Relevance?
+
+comment: p2f48lc
+parent: t1_p2eqo6a
+author: Arphinator
+created_utc: 1786173476
+edited: false
+body:
+My account in Kraken was, 2018. Police was laughing. Kraken was laughing. A few BTC was gone.
+
 more-stub: parent t1_p23f9pw count <live-count>
