--- reddit-mk3-seed-entropy-worse@20260808T083641Z
+++ reddit-mk3-seed-entropy-worse@20260808T150731Z
@@ -1721,6 +1721,14 @@
 body:
 Why do you need new wallets? New adress would be sufficient, no?
 
+comment: p2fz73e
+parent: t1_p21r3af
+author: ItsMeMulbear
+created_utc: 1786188909
+edited: false
+body:
+Both things that were considered advanced schizo stuff at the time. Entering a passphrase on an MK3 would also be a royal pain in the ass. 
+
 more-stub: parent t1_p22wv5v count <live-count>
 
 more-stub: parent t1_p22ti4j count <live-count>
