--- reddit-mk3-seed-entropy-worse@20260813T125620Z
+++ reddit-mk3-seed-entropy-worse@20260813T192749Z
@@ -1052,18 +1052,6 @@
 
 Cool
 
-comment: p22uu6m
-parent: t1_p21yp2i
-author: 1n5aN1aC
-created_utc: 1786030521
-edited: false
-body:
-With Coldcard, you could use dicerolls, and a python script on a computer to verify the hash matches the generated.
-
-And since the python script is very very simple, it wouldn't be too hard to audit that, then generate 1-2 keys on your card, verifying it matches, then generate the real one, now confident the key is secure and not leaked.
-
-That does solve pretty much all possible vulnerabilities related to key generation, but it is harder for a non-technical user.
-
 comment: p22v0fg
 parent: t1_p22mvk0
 author: WhenIntegralsAttack2
