--- reddit-mk3-seed-entropy-worse@20260808T150731Z
+++ reddit-mk3-seed-entropy-worse@20260809T165657Z
@@ -1468,14 +1468,6 @@
 
 But that’s all unnecessary… if your key generation is good.
 
-comment: p24ys9j
-parent: t1_p220iyg
-author: TheGamingGallifreyan
-created_utc: 1786049669
-edited: false
-body:
-Pretty much every one of them was removed by the moderators lol
-
 comment: p24zxdu
 parent: t1_p21mxwf
 author: LeatherMine
@@ -1729,6 +1721,14 @@
 body:
 Both things that were considered advanced schizo stuff at the time. Entering a passphrase on an MK3 would also be a royal pain in the ass. 
 
+comment: p2kcuhh
+parent: t3_1vh0lm7
+author: Tahueisin
+created_utc: 1786238191
+edited: false
+body:
+Fewer and fewer will understand...
+
 more-stub: parent t1_p22wv5v count <live-count>
 
 more-stub: parent t1_p22ti4j count <live-count>
