--- reddit-mk3-seed-entropy-worse@20260807T104457Z
+++ reddit-mk3-seed-entropy-worse@20260807T193415Z
@@ -176,11 +176,11 @@
 
 comment: p21guy2
 parent: t3_1vh0lm7
-author: kurtzsp
+author: [deleted]
 created_utc: 1786016196
 edited: false
 body:
-This post explain well why it's not possible to reach mass adoption. You almost need a degree in physics to just be sure that your keys are safe...
+[deleted]
 
 comment: p21hjd5
 parent: t1_p21cu31
@@ -1693,6 +1693,18 @@
 
 The CEO dismissed it.
 
+comment: p29ax2e
+parent: t1_p219g6c
+author: Psychological_Mode98
+created_utc: 1786107689
+edited: false
+body:
+This is gold. Understanding that different entropy sources „added up“ can only increase the entropy or keep it equal to the best factor‘s entropy and can never result in worse.
+
+This knowledge would have been so easy to put in one sentence in all that cc setup edu that it could have saved so many more people because it would have reduced their fear of messing something up.
+
+Don‘t say „by adding dice rolls you can optionally add your own entropy which is cool because you don‘t have to trust the device‘s“ say „by adding dice rolls you can never mess up the devices security you can not make anything worse. Even if you mess it up completely and make the rolls up yourself, only use one roll or post your rolls online: It doesn‘t decrease the safety of your coins. So roll the damn dice!“
+
 more-stub: parent t1_p22wv5v count <live-count>
 
 more-stub: parent t1_p22ti4j count <live-count>
