--- reddit-mk3-seed-entropy-worse@20260809T232706Z
+++ reddit-mk3-seed-entropy-worse@20260812T044139Z
@@ -844,14 +844,6 @@
 What? Of course it can. Why couldn't they?
 Pre-compute all possible wallets first and rank them.
 
-comment: p22fhmu
-parent: t3_1vh0lm7
-author: broskibrokovski21
-created_utc: 1786026553
-edited: false
-body:
-This story keeps getting crazier by the minute...
-
 comment: p22fj2c
 parent: t1_p22eqfq
 author: ineedanamegenerator
@@ -1251,11 +1243,11 @@
 
 comment: p240ksc
 parent: t3_1vh0lm7
-author: serious_discussion5
+author: [deleted]
 created_utc: 1786040838
 edited: false
 body:
-This is an absolutely ridiculous bug , the fact it made it to production is stupid
+[removed]
 
 comment: p241rce
 parent: t1_p23thyp
@@ -1267,11 +1259,11 @@
 
 comment: p2420ku
 parent: t1_p21upfp
-author: Todo_es
+author: [deleted]
 created_utc: 1786041201
 edited: false
 body:
-It has, look up "Passphrase".
+[deleted]
 
 comment: p2421nk
 parent: t1_p21nnox
@@ -1343,11 +1335,11 @@
 
 comment: p244jxf
 parent: t1_p2447sm
-author: serious_discussion5
+author: [deleted]
 created_utc: 1786041838
 edited: false
 body:
-And now my bitcoin is stuck on a time locked seed that'll probably get stolen and I can't do anything about it.
+[removed]
 
 comment: p246gwx
 parent: t1_p243ftx
@@ -1367,11 +1359,11 @@
 
 comment: p248d39
 parent: t1_p246o8q
-author: serious_discussion5
+author: [deleted]
 created_utc: 1786042806
 edited: false
 body:
-I'm talking about mini script time locked vaults. I've got some bitcoin locked up long term there and I hate that it could be compromised right now.
+[removed]
 
 comment: p248mvb
 parent: t3_1vh0lm7
@@ -1411,14 +1403,6 @@
 But honestly, that would still be a much better situation than today. If they had added just one factory generated UUID in the mix they'd be protected against a hack like this.
 
 This is common design choice: if you get hacked or a flaw is exposed, try to make it impact only one device at a time. In this case all devices are exposed to the single flaw.
-
-comment: p24c7mr
-parent: t1_p24aezd
-author: serious_discussion5
-created_utc: 1786043794
-edited: false
-body:
-Exactly. I'll have to wait until the exact moment it's unlocked then try to choose a fee that's higher than what they'll choose and hope they can't RBF it. Luckily I have a while to plan for this event and hopefully the hackers won't scan for my type of wallet or will forget by then.
 
 comment: p24d62o
 parent: t1_p24abxx
@@ -1737,6 +1721,14 @@
 body:
 Mk3 or 4/5/q?
 
+comment: p2tg535
+parent: t1_p2ppo8e
+author: orbag
+created_utc: 1786362294
+edited: false
+body:
+mk3
+
 more-stub: parent t1_p22wv5v count <live-count>
 
 more-stub: parent t1_p22ti4j count <live-count>
