--- reddit-mk3-seed-entropy-worse@20260807T193415Z
+++ reddit-mk3-seed-entropy-worse@20260808T020432Z
@@ -1705,6 +1705,14 @@
 
 Don‘t say „by adding dice rolls you can optionally add your own entropy which is cool because you don‘t have to trust the device‘s“ say „by adding dice rolls you can never mess up the devices security you can not make anything worse. Even if you mess it up completely and make the rolls up yourself, only use one roll or post your rolls online: It doesn‘t decrease the safety of your coins. So roll the damn dice!“
 
+comment: p2ca3z4
+parent: t1_p279vs9
+author: tridentgum
+created_utc: 1786136099
+edited: false
+body:
+you wouldn't need to tho.
+
 more-stub: parent t1_p22wv5v count <live-count>
 
 more-stub: parent t1_p22ti4j count <live-count>
