--- afilini-2085269060028170742@20260814T235921Z
+++ afilini-2085269060028170742@20260815T063535Z
@@ -126,47 +126,3 @@
 media: 0
 body:
 Does my tweet sound like I’m defending them?
-
-post: 2088230096306065429
-role: reply
-author: lukechilds
-name: Luke Childs
-created: 2026-08-14T11:43:47Z
-media: 1
-body:
-*slaps roof of trezor*
-
-This bad boy can fit so many pre-signed transactions in it
-
-post: 2088278202078699936
-role: reply
-author: Rob1Ham
-name: Rob Hamilton
-created: 2026-08-14T14:54:56Z
-media: 0
-body:
-I know I've spent the last 2 weeks crashing out about being restricted in accessing cyber programs from the American frontier AI labs.
-
-For about a day now I've had access to 
-@OpenAI
-'s Daybreak Blue, and its the best model by far I've interacted with for security research.
-
-post: 2088305623318298903
-role: reply
-author: intangiblecoins
-name: Alex Thorn
-created: 2026-08-14T16:43:54Z
-media: 0
-body:
-an update on the coldcard investgiation, including top-line numbers, victim reports, location of some funds moving onchain
-
-and i give my high level thoughts on the impact of the event on bitcoin culture going forward
-
-post: 2088331320074314204
-role: reply
-author: stutxo
-name: stu
-created: 2026-08-14T18:26:00Z
-media: 2
-body:
-come and take it (like actually pls)
