--- afilini-2085269060028170742@20260806T081129Z
+++ afilini-2085269060028170742@20260806T164808Z
@@ -57,3 +57,45 @@
 media: 0
 body:
 I’m not the first to point out the sloppy work (I consider the one-letter commit messages in libngu also sloppy work) and this just adds to that
+
+post: 2085311458879971338
+role: reply
+author: peterktodd
+name: Peter Todd
+created: 2026-08-06T10:26:09Z
+media: 0
+body:
+No need to write an emulator. Just design a TRNG that you can actually test and independently verify on real hardware: 
+https://
+petertodd.org/2014/push-butt
+on-rng
+…
+
+Everything else can be done deterministically.
+
+post: 2085315585479483716
+role: reply
+author: bigshiny0
+name: Shiny
+created: 2026-08-06T10:42:33Z
+media: 0
+body:
+Yes they could have. I took an afternoon after the new firmware was released to test the old and new firmware - specifically to test that the hardware rang was being used, by testing the firmware on a real device (I just added some debug functions that allowed me to query what the hardware was doing).took me a few hours to do.  No emulator needed.
+
+post: 2085316907658330372
+role: reply
+author: afilini
+name: Alekos Filini
+created: 2026-08-06T10:47:48Z
+media: 0
+body:
+Yes but that is very specific instrumentation added for one single feature. And presumably you removed them after confirming it worked. An emulator allows ongoing checks as the codebase progresses
+
+post: 2085348260122677395
+role: reply
+author: FreedomIsntSafe
+name: LazyNinja
+created: 2026-08-06T12:52:23Z
+media: 0
+body:
+1000%
