--- coindesk-70m-no-device-touch@20260813T192905Z
+++ coindesk-70m-no-device-touch@20260814T015759Z
@@ -60,35 +60,35 @@
 Storing a key safely is now the easier half of the problem.
 Latest Crypto News
 1
+U.S. SEC to again delay 'innovation exemption' for tokenization amid Wall Street, White House concerns
+2 hrs ago
+2
+SEC cancels long-awaited proposal of Reg Crypto, postponing meeting without new date
+3 hrs ago
+3
+Tether says it completed long-promised 'Big Four' audit of finances behind $180 billion USDT stablecoin
+7 hrs ago
+4
+Trezor warns 14,000 customers after fulfilment partner suffers data breach
+8 hrs ago
+5
 Crypto for Advisors: The crypto advice gap
-4 hrs ago
-2
+10 hrs ago
+6
 B2C2 taps Schroders veteran to chase Asia’s growing crypto wealth
-4 hrs ago
-3
+11 hrs ago
+7
 Kalshi in talks with Sequoia, Wellington for $750 million fund raise at $40 billion valuation
-5 hrs ago
-4
+11 hrs ago
+8
 Swissquote cuts full-year guidance as 1H crypto income plunges, shares slide
-5 hrs ago
-5
+12 hrs ago
+9
 Crypto platform operator gets 15-year prison sentence in South Korea due to $50 million scam
-5 hrs ago
-6
+12 hrs ago
+10
 Figure revenue doubles as blockchain loan marketplace volumes surge
-5 hrs ago
-7
-Bullish shares surge 14% as subscription revenue offsets digital asset slowdown
-7 hrs ago
-8
-Strategy, Metaplanet unrealized bitcoin losses highlight risk of concentrating on just one token
-7 hrs ago
-9
-Bitcoin holds steady near $64,000 as monero, hyperliquid outperform
-8 hrs ago
-10
-MUFG to test real-time blockchain settlement for Japanese government bond trades
-9 hrs ago
+12 hrs ago
 Latest Research
 Building the Zcash Machine: Tachyon and Quantum Readiness
 Building the Zcash Machine: Tachyon and Quantum Readiness
@@ -166,4 +166,4 @@
 EthicsPrivacyTerms of UseCookie SettingsDo Not Sell My InfoBug Bounty Program
 © 2026 CoinDesk, Inc.
 Disclosure & Polices: CoinDesk is an award-winning media outlet that covers the cryptocurrency industry. Its journalists abide by a strict set of editorial policies. CoinDesk has adopted a set of principles aimed at ensuring the integrity, editorial independence and freedom from bias of its publications. CoinDesk is part of Bullish (NYSE:BLSH), an institutionally focused global digital asset platform that provides market infrastructure and information services. Bullish owns and invests in digital asset businesses and digital assets and CoinDesk employees, including journalists, may receive Bullish equity-based compensation.
-CryptoCD20$1,738.68CD20 up 0.11 percent0.11%BTC$63,356.72BTC down 0.010 percent0.010%ETH$1,885.52ETH up 0.40 percent0.40%XRP$1.01XRP up 0.21 percent0.21%SOL$76.05SOL up 0.47 percent0.47%
+CryptoCD20$1,740.98CD20 up 0.29 percent0.29%BTC$63,470.41BTC up 0.10 percent0.10%ETH$1,886.54ETH up 0.55 percent0.55%XRP$1.01XRP up 0.63 percent0.63%SOL$76.01SOL up 0.57 percent0.57%
