--- coindesk-70m-no-device-touch@20260808T213820Z
+++ coindesk-70m-no-device-touch@20260809T041105Z
@@ -61,31 +61,31 @@
 Latest Crypto News
 1
 Bitcoin hits block 961,632 as the controversial BIP-110 soft fork attempt begins
-1 hr ago
+8 hrs ago
 2
 Hardware wallet sales in Russia more than double as new crypto rules near
-3 hrs ago
+10 hrs ago
 3
 Brazil's central bank orders exchanges to delay large crypto transfers abroad
-6 hrs ago
+12 hrs ago
 4
 Trillions in institutional money to flow into bitcoin, says Bitwise's Matt Hougan
-7 hrs ago
+14 hrs ago
 5
 Why trillion-dollar asset manager T. Rowe Price put memecoins in its crypto ETF
-8 hrs ago
+15 hrs ago
 6
 U.S. Senate opens first stage of crypto Clarity Act voting to give bill a chance next month
-12 hrs ago
+19 hrs ago
 7
 Another Bitcoin infrastructure exploit hits, this time draining Lightning payment servers
-13 hrs ago
+20 hrs ago
 8
 New XRP Ledger amendments target $530 million in tokenized Wall Street assets
-18 hrs ago
+1 day ago
 9
 Bitcoin holders risk losing real BTC if they sell coins from BIP-110 fork, says developer
-19 hrs ago
+1 day ago
 10
 Trump Media pulls back from crypto, scraps Crypto.com's CRO token treasury deal
 1 day ago
@@ -164,4 +164,4 @@
 EthicsPrivacyTerms of UseCookie SettingsDo Not Sell My InfoBug Bounty Program
 © 2026 CoinDesk, Inc.
 Disclosure & Polices: CoinDesk is an award-winning media outlet that covers the cryptocurrency industry. Its journalists abide by a strict set of editorial policies. CoinDesk has adopted a set of principles aimed at ensuring the integrity, editorial independence and freedom from bias of its publications. CoinDesk is part of Bullish (NYSE:BLSH), an institutionally focused global digital asset platform that provides market infrastructure and information services. Bullish owns and invests in digital asset businesses and digital assets and CoinDesk employees, including journalists, may receive Bullish equity-based compensation.
-CryptoCD20$1,768.89CD20 up 0.96 percent0.96%BTC$65,004.30BTC up 0.16 percent0.16%ETH$1,921.19ETH up 0.29 percent0.29%XRP$1.04XRP up 1.96 percent1.96%SOL$76.23SOL up 3.55 percent3.55%
+CryptoCD20$1,761.95CD20 up 0.22 percent0.22%BTC$64,755.26BTC down 0.33 percent0.33%ETH$1,912.87ETH down 0.16 percent0.16%XRP$1.04XRP up 0.28 percent0.28%SOL$76.02SOL up 1.91 percent1.91%
