--- coindesk-70m-no-device-touch@20260814T015759Z
+++ coindesk-70m-no-device-touch@20260814T082857Z
@@ -60,35 +60,35 @@
 Storing a key safely is now the easier half of the problem.
 Latest Crypto News
 1
+JPMorgan shuttered its banking relationship with predictions platform Polymarket: FT
+3 hrs ago
+2
 U.S. SEC to again delay 'innovation exemption' for tokenization amid Wall Street, White House concerns
-2 hrs ago
-2
+9 hrs ago
+3
 SEC cancels long-awaited proposal of Reg Crypto, postponing meeting without new date
-3 hrs ago
-3
+10 hrs ago
+4
 Tether says it completed long-promised 'Big Four' audit of finances behind $180 billion USDT stablecoin
-7 hrs ago
-4
+14 hrs ago
+5
 Trezor warns 14,000 customers after fulfilment partner suffers data breach
-8 hrs ago
-5
+15 hrs ago
+6
 Crypto for Advisors: The crypto advice gap
-10 hrs ago
-6
+17 hrs ago
+7
 B2C2 taps Schroders veteran to chase Asia’s growing crypto wealth
-11 hrs ago
-7
+17 hrs ago
+8
 Kalshi in talks with Sequoia, Wellington for $750 million fund raise at $40 billion valuation
-11 hrs ago
-8
+18 hrs ago
+9
 Swissquote cuts full-year guidance as 1H crypto income plunges, shares slide
-12 hrs ago
-9
+18 hrs ago
+10
 Crypto platform operator gets 15-year prison sentence in South Korea due to $50 million scam
-12 hrs ago
-10
-Figure revenue doubles as blockchain loan marketplace volumes surge
-12 hrs ago
+18 hrs ago
 Latest Research
 Building the Zcash Machine: Tachyon and Quantum Readiness
 Building the Zcash Machine: Tachyon and Quantum Readiness
@@ -166,4 +166,4 @@
 EthicsPrivacyTerms of UseCookie SettingsDo Not Sell My InfoBug Bounty Program
 © 2026 CoinDesk, Inc.
 Disclosure & Polices: CoinDesk is an award-winning media outlet that covers the cryptocurrency industry. Its journalists abide by a strict set of editorial policies. CoinDesk has adopted a set of principles aimed at ensuring the integrity, editorial independence and freedom from bias of its publications. CoinDesk is part of Bullish (NYSE:BLSH), an institutionally focused global digital asset platform that provides market infrastructure and information services. Bullish owns and invests in digital asset businesses and digital assets and CoinDesk employees, including journalists, may receive Bullish equity-based compensation.
-CryptoCD20$1,740.98CD20 up 0.29 percent0.29%BTC$63,470.41BTC up 0.10 percent0.10%ETH$1,886.54ETH up 0.55 percent0.55%XRP$1.01XRP up 0.63 percent0.63%SOL$76.01SOL up 0.57 percent0.57%
+CryptoCD20$1,727.16CD20 down 1.17 percent1.17%BTC$62,739.11BTC down 1.52 percent1.52%ETH$1,868.33ETH down 1.05 percent1.05%XRP$1.00XRP down 0.74 percent0.74%SOL$75.56SOL down 0.75 percent0.75%
