--- coindesk-70m-no-device-touch@20260815T055616Z
+++ coindesk-70m-no-device-touch@20260815T122655Z
@@ -61,34 +61,34 @@
 Latest Crypto News
 1
 Trump expected to attend White House meeting with crypto CEOs, sources say
-6 hrs ago
+12 hrs ago
 2
 Trump-backed World Liberty wins conditional bank charter from federal regulator
-8 hrs ago
+15 hrs ago
 3
 Tokenization stocks slip as SEC delay puts 'speed bump' in crypto’s Wall Street push
-12 hrs ago
+18 hrs ago
 4
 Strategy says MSCI should measure markets, not dictate corporate assets
-15 hrs ago
+22 hrs ago
 5
 Israel’s largest bank to offer crypto trading with Galaxy
-16 hrs ago
+22 hrs ago
 6
 Fear is fading across markets, be it bitcoin, stocks, gold or bonds
-18 hrs ago
+1 day ago
 7
 Trump family's World Liberty Financial delay plans to sell Maldives resort token
-18 hrs ago
+1 day ago
 8
 Bitcoin slips as U.S. inflation fails to spark gains, ETFs see August's first two-day drawdown
-18 hrs ago
+1 day ago
 9
 RedotPay said to put $1 billion U.S. IPO plan on hold: Bloomberg
-19 hrs ago
+1 day ago
 10
 Live updates: Bitcoin slips back to $63,000; MSCI threatens to exclude Strategy from indices
-21 hrs ago
+1 day ago
 Latest Research
 Building the Zcash Machine: Tachyon and Quantum Readiness
 Building the Zcash Machine: Tachyon and Quantum Readiness
@@ -166,5 +166,4 @@
 EthicsPrivacyTerms of UseCookie SettingsDo Not Sell My InfoBug Bounty Program
 © 2026 CoinDesk, Inc.
 Disclosure & Polices: CoinDesk is an award-winning media outlet that covers the cryptocurrency industry. Its journalists abide by a strict set of editorial policies. CoinDesk has adopted a set of principles aimed at ensuring the integrity, editorial independence and freedom from bias of its publications. CoinDesk is part of Bullish (NYSE:BLSH), an institutionally focused global digital asset platform that provides market infrastructure and information services. Bullish owns and invests in digital asset businesses and digital assets and CoinDesk employees, including journalists, may receive Bullish equity-based compensation.
-CryptoCD20$1,734.69CD20 down 0.11 percent0.11%BTC$62,997.10BTC down 0.50 percent0.50%ETH$1,878.57ETH down 0.18 percent0.18%XRP$1.00XRP down 0.55 percent0.55%SOL$75.27SOL down 0.76 percent0.76%
-How bitcoin cold wallets lost $70 million in an attack that never touched the devices
+CryptoCD20$1,731.46CD20 up 0.41 percent0.41%BTC$62,921.07BTC up 0.38 percent0.38%ETH$1,877.01ETH up 0.29 percent0.29%XRP$1.00XRP up 0.12 percent0.12%SOL$75.17SOL down 0.28 percent0.28%
