--- bitcointalk-learn-from-incident@20260807T193124Z
+++ bitcointalk-learn-from-incident@20260808T020149Z
@@ -439,7 +439,7 @@
 Exactly and the price if compared to buying a brand new hardware wallet, almost roughly the same if you don't have old laptops lying around, even then old laptop might have security flaw on its old hardware.
 Title: Re: We can all learn one or two from this ColdCard incident
 Post by: hd49728 on August 07, 2026, 03:34:16 AM
-Quote from: X-ray on Today at 01:24:02 AM
+Quote from: X-ray on August 07, 2026, 01:24:02 AM
 Setting it up, installing OS etc is easy, the hard part is exactly what you mentioned, keeping up with the exploit updates and verify that you got the right files.
 People must avoid using cracked OS because it is very dangerous in security. If it's not feasible for them to buy it to use, they can consider to use open source OS like Linux.
 https://linuxmint.com/
@@ -482,13 +482,13 @@
 Sure, a lot of those that are affected will not return back to bitcoin investment, I believe 80% of those that we were affected will not talk about bitcoin investment ever again. Just like you have said, a lot of them that was affected has been accumulating Bitcoin since the creation of Bitcoin. The pain of losing everything will never allow them to come back to Bitcoin.
 Title: Re: We can all learn one or two from this ColdCard incident
 Post by: Dogedegen on August 07, 2026, 07:24:33 PM
-Quote from: hd49728 on Today at 03:34:16 AM
-Quote from: X-ray on Today at 01:24:02 AM
+Quote from: hd49728 on August 07, 2026, 03:34:16 AM
+Quote from: X-ray on August 07, 2026, 01:24:02 AM
 Setting it up, installing OS etc is easy, the hard part is exactly what you mentioned, keeping up with the exploit updates and verify that you got the right files.
 People must avoid using cracked OS because it is very dangerous in security. If it's not feasible for them to buy it to use, they can consider to use open source OS like Linux.
 https://linuxmint.com/
 This is true, but this is very rare so we must be careful what we write and how we write about it. There have been only a few times when this kind of case ended up in the news and this was mainly because the source of those cracked versions was not reputable, and if you think that there have been billions of installs of cracked OS and software the risk is very low. Actually the much higher risk comes from other attack vectors like malicious software, using very weak wallets like browser wallets as your main wallets and stuff like that. Anyway, using Windows as a primary OS for keeping cryptocurrency safe is very bad for both security and privacy and using a real license does not help with this. So you are very right to suggest Linux Mint, I have been using that and Ubuntu for a very long time and I have never had any issues of any kind when it comes to security. Simply by switching to such an OS the risk of a hack decreases considerably!
-Quote from: Supreme Donvic on Today at 03:27:01 PM
+Quote from: Supreme Donvic on August 07, 2026, 03:27:01 PM
 Quote from: KiaKia on August 01, 2026, 12:17:17 PM
 The only lesson I learnt here is that nowhere is safe, putting all your eggs in one basket is totally wrong, as for that passphrase thing, hardware wallets are the only ones offering it?
 If any mobile wallet has passphrase+ recovery seeds then there is really no reason to run after hardware wallets anymore, I am really disappointed in this ColdCard company and their team together.
