--- bitcointalk-learn-from-incident@20260806T033223Z
+++ bitcointalk-learn-from-incident@20260806T100339Z
@@ -383,5 +383,24 @@
 I've used airgapped computers for my wallet for quite sometime, I don't find it any more riskier than hardware wallet. You installed latest OS to an airgapped computer, you never connect it to the Internet and the wallet stays on the computer.
 No supply chain attack which hardware wallet could be infected from, no accidental seed phrase leak because the internet isn't there in the first place, your wallet stay safe until the day you decided to withdraw it. I think it's robust enough solution for me, but it's just my opinion.
 To be honest though, setting up one is a pain to do, you need to verify the stack and keep updated on latest exploit which going to take a lot of your time, which is why I also own hardware wallet.
+Title: Re: We can all learn one or two from this ColdCard incident
+Post by: Outhue on August 06, 2026, 07:35:59 AM
+Quote from: X-ray on Today at 03:23:13 AM
+Quote from: PostQuantumBTC on August 02, 2026, 12:36:35 AM
+Quote from: bitmover on August 01, 2026, 04:59:06 PM
+Airgapped computers are risky to setup and use.
+How is it risky?
+Airgapped computers means you are not sending anything out of the wallet in a way that you can be affected. If you have airgapped wallet, you are safe so far it stayed airgapped but stay away from crypto clipper.
+But I will likely if you correct me because my understanding about the cold airgapped device may be limited.
+I've used airgapped computers for my wallet for quite sometime, I don't find it any more riskier than hardware wallet. You installed latest OS to an airgapped computer, you never connect it to the Internet and the wallet stays on the computer.
+No supply chain attack which hardware wallet could be infected from, no accidental seed phrase leak because the internet isn't there in the first place, your wallet stay safe until the day you decided to withdraw it. I think it's robust enough solution for me, but it's just my opinion.
+To be honest though, setting up one is a pain to do, you need to verify the stack and keep updated on latest exploit which going to take a lot of your time, which is why I also own hardware wallet.
+Let's say that you can handle this perfectly, I am not new to Bitcoin but I can't do this same thing that you are doing, because of few reasons like I am not living alone, my computer isn't private and others, you can go out and someone startup your computer and connect it to the internet, unless you live alone all by yourself.
+Also airgapped devices comes with extra security, if anyone tries to tamper with em they can self wiped themselves, talking for Keystone wallet though, I don't know about the rest, but if someone wants to tampered with that PC or Laptop behind your back they would successfully do it, but like I've just said, if you can handle it right then no problem, only that it's not for everyone.
+Title: Re: We can all learn one or two from this ColdCard incident
+Post by: PostQuantumBTC on August 06, 2026, 09:13:25 AM
+Quote from: X-ray on Today at 03:23:13 AM
+To be honest though, setting up one is a pain to do, you need to verify the stack and keep updated on latest exploit which going to take a lot of your time, which is why I also own hardware wallet.
+I do not see any difficulty in setting up a wallet on an airgapped device, it was simple for me to do. Another thing is that if you are also not updated with exploit updates and knowing how to avoid hackers, hardware wallet can not save your coins from the hackers. Hackers also target hardware wallet users specifically.
 Powered by SMF 1.1.19 |
 SMF © 2006-2009, Simple Machines
