--- bitcointalk-learn-from-incident@20260806T100339Z
+++ bitcointalk-learn-from-incident@20260806T163541Z
@@ -402,5 +402,20 @@
 Quote from: X-ray on Today at 03:23:13 AM
 To be honest though, setting up one is a pain to do, you need to verify the stack and keep updated on latest exploit which going to take a lot of your time, which is why I also own hardware wallet.
 I do not see any difficulty in setting up a wallet on an airgapped device, it was simple for me to do. Another thing is that if you are also not updated with exploit updates and knowing how to avoid hackers, hardware wallet can not save your coins from the hackers. Hackers also target hardware wallet users specifically.
+Title: Re: We can all learn one or two from this ColdCard incident
+Post by: bitmover on August 06, 2026, 10:12:18 AM
+Quote from: X-ray on Today at 03:23:13 AM
+Quote from: PostQuantumBTC on August 02, 2026, 12:36:35 AM
+Quote from: bitmover on August 01, 2026, 04:59:06 PM
+Airgapped computers are risky to setup and use.
+How is it risky?
+Airgapped computers means you are not sending anything out of the wallet in a way that you can be affected. If you have airgapped wallet, you are safe so far it stayed airgapped but stay away from crypto clipper.
+But I will likely if you correct me because my understanding about the cold airgapped device may be limited.
+I've used airgapped computers for my wallet for quite sometime, I don't find it any more riskier than hardware wallet. You installed latest OS to an airgapped computer, you never connect it to the Internet and the wallet stays on the computer.
+No supply chain attack which hardware wallet could be infected from, no accidental seed phrase leak because the internet isn't there in the first place, your wallet stay safe until the day you decided to withdraw it. I think it's robust enough solution for me, but it's just my opinion.
+To be honest though, setting up one is a pain to do, you need to verify the stack and keep updated on latest exploit which going to take a lot of your time, which is why I also own hardware wallet.
+There is a risk of supply chain attack, accidental seed phrase leak, everything.
+The risk is exactly in the step you called a "pain to do". Setting it up. If you dont know exactly what you are doing,  you can expose yourself to those risks
+When you buy a good hardware wallet (not coldcard), you are theoretically free of all those risks.
 Powered by SMF 1.1.19 |
 SMF © 2006-2009, Simple Machines
