--- reddit-case-against-passphrases@20260813T133357Z
+++ reddit-case-against-passphrases@20260813T193225Z
@@ -526,11 +526,3 @@
 \>I have no clue what you said, and for that, Bitcoin will never be mainstream.
 
 That was the reason I mentioned ssl. At some point you don't need to understand a technology to use it.
-
-comment: p3di0en
-parent: t3_1vm56ou
-author: SmokeAndSkate
-created_utc: 1786591517
-edited: false
-body:
-You make some really great points tbh. I’ve been thinking a lot of the same thoughts. 
