--- reddit-coinkite-done-for@20260806T032715Z
+++ reddit-coinkite-done-for@20260808T015653Z
@@ -891,4 +891,12 @@
 body:
 The CFO of coinkite was behide all of this, he’s going down.
 
+comment: p2ca3e3
+parent: t3_1vcfugv
+author: piejlucas
+created_utc: 1786136095
+edited: false
+body:
+To some extent it also reveals how flawed it is to rely on open source security.  This code was sitting in a public container for years and only now has this “not complex” vulnerability surfaced. 
+
 more-stub: parent t1_p139si9 count <live-count>
