--- reddit-coinkite-done-for@20260805T205807Z
+++ reddit-coinkite-done-for@20260806T032715Z
@@ -875,4 +875,20 @@
 
 Ledger doesn't just support BTC; it also supports many other cryptocurrencies. I'm not sure whether that would make it incompatible with a seed phrase that the Coldcard revealed after being fed dice roll values, but since the BTC-only Coldcard isn't relying on its own RNG in that case, I have a feeling that it should be possible.
 
+comment: p1z1xpb
+parent: t3_1vcfugv
+author: bears196
+created_utc: 1785979460
+edited: false
+body:
+Yep, the people behind coldcard are responsible. They should go to prison for a very long time.
+
+comment: p1z2g8q
+parent: t3_1vcfugv
+author: bears196
+created_utc: 1785979631
+edited: false
+body:
+The CFO of coinkite was behide all of this, he’s going down.
+
 more-stub: parent t1_p139si9 count <live-count>
