--- reddit-where-store-bitcoin-now@20260804T224833Z
+++ reddit-where-store-bitcoin-now@20260804T232351Z
@@ -634,3 +634,19 @@
 edited: false
 body:
 Completely different situation. He’s talking about mandatory reporting the funds have to make that anyone can check on the public blockchain. No similarity to people getting loans they can’t pay back. 
+
+comment: p1qx113
+parent: t1_p1l170j
+author: 1-gill
+created_utc: 1785884395
+edited: false
+body:
+Listen to me bro. I’m am not a bot. Ok this is EXACTLY how I lost .22 btc. Please write it down or stamp it in steel. That what I’ve learned  ….the hard way. 
+
+comment: p1qx9ht
+parent: t3_1veyt5w
+author: 1-gill
+created_utc: 1785884468
+edited: false
+body:
+Stamped steel. Guess what….It’s still there.
