--- reddit-where-store-bitcoin-now@20260804T214726Z
+++ reddit-where-store-bitcoin-now@20260804T221747Z
@@ -607,3 +607,22 @@
 edited: false
 body:
 Create a seed with an open source wallet on an air gapped VM.
+
+comment: p1qifer
+parent: t1_p1q4juw
+author: Fil3toFishy69
+created_utc: 1785880111
+edited: false
+body:
+I bet my boy. PLOW IT BABY
+
+comment: p1qnzoj
+parent: t3_1veyt5w
+author: RulePersonal
+created_utc: 1785881695
+edited: false
+body:
+Someone please fill me in as a casual. I was under the impression that this issue was isolated to coldcard users. Someone in their company made a big whoopie and cost some billions. Is the issue actually with any wallet with a generated seed phrase or what am  
+I missing? I store mostly on cake wallet on my phone. It’s a 12 word phrase. How safe is that? 
+
+If it’s not safe, what can I do to protect myself? Willing to send a tip for help. This makes me thank my lucky stars I converted most to usdt back in nov. 
