--- reddit-where-store-bitcoin-now@20260804T201741Z
+++ reddit-where-store-bitcoin-now@20260804T204716Z
@@ -575,3 +575,19 @@
 edited: false
 body:
 Cold storage. But educate yourself to do the process yourself.
+
+comment: p1pyfxy
+parent: t1_p1p3u7q
+author: uhohthrowawayyyyyy
+created_utc: 1785874714
+edited: false
+body:
+People like this are conspiracy types, they’re into bitcoin for a different reason than yourself hahah
+
+comment: p1q4juw
+parent: t1_p1o8akz
+author: Interesting-Lime3031
+created_utc: 1785876337
+edited: false
+body:
+Better than ur mother treated me last night 
