--- reddit-where-store-bitcoin-now@20260804T161748Z
+++ reddit-where-store-bitcoin-now@20260804T164821Z
@@ -517,3 +517,19 @@
 What happened with Coldcard was the result of incredibly low entropy during the seed generation process, something the rest of manufacturers ([us included](https://github.com/BitBoxSwiss/bitbox02-firmware/blob/c838d7fd80190a02531ba30e4a904240e4485e1f/src/rust/bitbox02-rust/src/keystore.rs#L577)) take very seriously.
 
 If you can't trust an RNG anymore you can always [roll your own seed](https://blog.bitbox.swiss/en/roll-the-dice-generate-your-own-seed/).
+
+comment: p1ogga3
+parent: t3_1veyt5w
+author: BearlyBearishBear
+created_utc: 1785861063
+edited: false
+body:
+Cash out and store in a real bank 
+
+comment: p1ojjbw
+parent: t3_1veyt5w
+author: No-Reading-4384
+created_utc: 1785861836
+edited: false
+body:
+Lost 160k….nowhere.
