--- reddit-where-store-bitcoin-now@20260804T134532Z
+++ reddit-where-store-bitcoin-now@20260804T141617Z
@@ -481,3 +481,11 @@
 A guy just made a list of all hardware wallets.
 
 https://www.reddit.com/r/Bitcoin/comments/1vehzv8/i_compared_every_major_bitcoin_hardware_wallet/
+
+comment: p1ngddn
+parent: t1_p1mpobj
+author: swaggercatr
+created_utc: 1785851660
+edited: false
+body:
+Have passphrase on your trezor for safety
