--- reddit-where-store-bitcoin-now@20260804T095521Z
+++ reddit-where-store-bitcoin-now@20260804T134532Z
@@ -388,3 +388,96 @@
 edited: false
 body:
 Cold storage 
+
+comment: p1mpobj
+parent: t3_1veyt5w
+author: DK4E2XFpbETJrj
+created_utc: 1785843278
+edited: false
+body:
+I'm seriously questioning my Trezor One. I bought it and forgot about it for so many years. I mean, it's been great. But what if, right? 
+
+comment: p1msoi8
+parent: t1_p1l87yx
+author: AAAdamKK
+created_utc: 1785844360
+edited: false
+body:
+I'll hold it for you and send you 2 BTC back for every 1 BTC you send to me!
+
+comment: p1mu15i
+parent: t3_1veyt5w
+author: Btcyoda
+created_utc: 1785844836
+edited: false
+body:
+On the blockchain.
+
+comment: p1muvp3
+parent: t3_1veyt5w
+author: InevitableSalty4366
+created_utc: 1785845133
+edited: false
+body:
+Create your own paper wallet. Use coinflips, use dicerolls, count the number of different coloured cars. Be your own rng.
+
+
+https://www.bitaddress.org/bitaddress.org-v3.3.0-SHA256-dec17c07685e1870960903d8f58090475b25af946fe95a734f88408cef4aa194.html
+
+comment: p1mzr2u
+parent: t3_1veyt5w
+author: tnethacker
+created_utc: 1785846744
+edited: false
+body:
+I've heard nice things about mtgox 
+
+comment: p1mzye5
+parent: t1_p1l170j
+author: Delicious_Mousse6210
+created_utc: 1785846812
+edited: false
+body:
+low iq or bait?
+
+comment: p1n4yqq
+parent: t1_p1l40uo
+author: iloverunning11
+created_utc: 1785848374
+edited: false
+body:
+That is dumb af
+
+comment: p1n7kcb
+parent: t3_1veyt5w
+author: Inevitable-Pay-7159
+created_utc: 1785849157
+edited: false
+body:
+Use Qastle wallet available on App Store. This is ultimate full proof unhackable wallet with quantum pure entropy
+
+comment: p1n831n
+parent: t1_p1lbx2m
+author: Interesting-Lime3031
+created_utc: 1785849307
+edited: false
+body:
+I don’t really believe in etf version of bitcoin, I believe in owning bitcoin not trusting some hedge fund manager to own it bitcoin 
+
+comment: p1n88vf
+parent: t1_p1l2xbu
+author: Interesting-Lime3031
+created_utc: 1785849354
+edited: false
+body:
+Ur a Moron. I don’t even use cold card lol 
+
+comment: p1nbdwg
+parent: t3_1veyt5w
+author: Gh0mri
+created_utc: 1785850260
+edited: false
+body:
+A guy just made a list of all hardware wallets.
+
+https://www.reddit.com/r/Bitcoin/comments/1vehzv8/i_compared_every_major_bitcoin_hardware_wallet/
