--- reddit-where-store-bitcoin-now@20260804T141617Z
+++ reddit-where-store-bitcoin-now@20260804T161748Z
@@ -489,3 +489,31 @@
 edited: false
 body:
 Have passphrase on your trezor for safety
+
+comment: p1o7n6m
+parent: t3_1veyt5w
+author: HODL_Dawg
+created_utc: 1785858840
+edited: false
+body:
+I keep mine in an old coffee can buried under the back porch.
+
+comment: p1o8akz
+parent: t1_p1n88vf
+author: Fil3toFishy69
+created_utc: 1785859005
+edited: false
+body:
+How is your Micro brain treating you today?
+
+comment: p1oa4i5
+parent: t3_1veyt5w
+author: pako-bitbox
+created_utc: 1785859470
+edited: false
+body:
+You can still do self-custody, in fact you should.
+
+What happened with Coldcard was the result of incredibly low entropy during the seed generation process, something the rest of manufacturers ([us included](https://github.com/BitBoxSwiss/bitbox02-firmware/blob/c838d7fd80190a02531ba30e4a904240e4485e1f/src/rust/bitbox02-rust/src/keystore.rs#L577)) take very seriously.
+
+If you can't trust an RNG anymore you can always [roll your own seed](https://blog.bitbox.swiss/en/roll-the-dice-generate-your-own-seed/).
