--- reddit-where-store-bitcoin-now@20260804T184710Z
+++ reddit-where-store-bitcoin-now@20260804T201741Z
@@ -559,3 +559,19 @@
 edited: false
 body:
 Ledger 
+
+comment: p1px4jn
+parent: t1_p1oykr1
+author: No-Reading-4384
+created_utc: 1785874367
+edited: false
+body:
+Yep was on exodus for 10 years because it was “secure”
+
+comment: p1pxnse
+parent: t3_1veyt5w
+author: Charmed-paper345
+created_utc: 1785874509
+edited: false
+body:
+Cold storage. But educate yourself to do the process yourself.
