--- reddit-where-store-bitcoin-now@20260804T181606Z
+++ reddit-where-store-bitcoin-now@20260804T184710Z
@@ -107,11 +107,11 @@
 
 comment: p1l40uo
 parent: t1_p1l0ijo
-author: Some-btc-name
+author: [deleted]
 created_utc: 1785816739
 edited: false
 body:
-Not sure why this is that complicated for ppl. Like BIP-39 has 2048 words. Choose one. Then do that another 23x. Now you have your seed. Cold card can still be used to sign but don't have it generate a seed. Also. If you want to quantum proof go multisig.
+[deleted]
 
 comment: p1l5x7x
 parent: t3_1veyt5w
@@ -551,3 +551,11 @@
 Spot Bitcoin ETFs are legally required to hold the Bitcoin that backs the shares, and those holdings are audited and disclosed…
 
 You’re not trusting an individual but rules/regulations that have been around a long time and current enforcement.
+
+comment: p1p60e6
+parent: t3_1veyt5w
+author: nick_c8_vegas
+created_utc: 1785867387
+edited: false
+body:
+Ledger 
