--- libngu-pr-64@20260808T015236Z
+++ libngu-pr-64@20260809T102811Z
@@ -244,6 +244,60 @@
 August 7, 2026 21:18
 test: gate external STM32 rng_get provider
 ab26f08
+scgbckbone
+approved these changes
+Aug 9, 2026
+View reviewed changes
+scgbckbone
+left a comment
+Copy link
+Copy Markdown
+Contributor
+There was a problem hiding this comment.
+Choose a reason for hiding this comment
+The reason will be displayed to describe this comment to others. Learn more.
+Choose a reason
+Spam
+Abuse
+Off Topic
+Outdated
+Duplicate
+Resolved
+Low Quality
+Hide comment
+nice improvements, thanks!
+Sorry, something went wrong.
+Uh oh!
+There was an error while loading. Please reload this page.
+All reactions
+Comment thread
+ngu/random_backend.h
+return true;
+}
+#elif MICROPY_PY_STM
+scgbckbone
+Aug 9, 2026
+Copy link
+Copy Markdown
+Contributor
+There was a problem hiding this comment.
+Choose a reason for hiding this comment
+The reason will be displayed to describe this comment to others. Learn more.
+Choose a reason
+Spam
+Abuse
+Off Topic
+Outdated
+Duplicate
+Resolved
+Low Quality
+Hide comment
+#if defined(MICROPY_PY_STM)
+nit: just a module flag which can be 0 for stm32 build
+Sorry, something went wrong.
+Uh oh!
+There was an error while loading. Please reload this page.
+All reactions
 This file contains hidden or bidirectional Unicode text that may be interpreted or compiled differently than what appears below. To review, open the file in an editor that reveals hidden Unicode characters.
 Learn more about bidirectional Unicode characters
 Show hidden characters
@@ -252,7 +306,10 @@
 Already have an account?
 Sign in to comment
 Reviewers
-No reviews
+1 more reviewer
+scgbckbone
+scgbckbone approved these changes
+Reviewers whose approvals may not affect merge requirements
 Assignees
 No one assigned
 Labels
@@ -265,7 +322,7 @@
 Successfully merging this pull request may close these issues.
 Uh oh!
 There was an error while loading. Please reload this page.
-1 participant
+2 participants
 Add this suggestion to a batch that can be applied as a single commit.This suggestion is invalid because no changes were made to the code.Suggestions cannot be applied while the pull request is closed.Suggestions cannot be applied while viewing a subset of changes.Only one suggestion per line can be applied in a batch.Add this suggestion to a batch that can be applied as a single commit.Applying suggestions on deleted lines is not supported.You must change the existing code in this line in order to create a valid suggestion.Outdated suggestions cannot be applied.This suggestion has been applied or marked resolved.Suggestions cannot be applied from pending reviews.Suggestions cannot be applied on multi-line comments.Suggestions cannot be applied while the pull request is queued to merge.Suggestion cannot be applied right now. Please check back later.
 Footer
 © 2026 GitHub, Inc.
