--- reddit-glacier-protocol-20-call@20260814T234743Z
+++ reddit-glacier-protocol-20-call@20260815T055945Z
@@ -464,3 +464,31 @@
 I prefer a good old fashioned Bitcoin node, but...  With all the nodes being deprecated for the latest node.. and since the Core and Knot nodes are at war, and giving the fact that nodes are being used to fork Bitcoin to whomever they please.. I really don't trust the core or knot devs...  I kind of trust schildbach's mobile app.  But ok.  How long will that trust last?
 
 So my best answer is to found and establish a new node, a better model, that's going to be difficult without help.  But thanks to AI, I can find, understand, and learn hundreds of times faster than I can, but, I wish I was able to talk to the devs again, that was fastest.  
+
+comment: p3rfh2o
+parent: t3_1vnu1eb
+author: FreezedPeachNow
+created_utc: 1786757153
+edited: false
+body:
+There should be an "official" open source Bitcoin announcement page.
+
+How do you get what you want to say onto it?
+
+You write a BLIP. Bitcoin language improvement protocol.
+
+Then 51% of node operators approve the announcement and it gets published to the medium of choice.  Like a webpage, or maybe even the blockchain
+
+This becones the official unofficial place for Bitcoin information.  So in 500 years students can look back at it and like the 10 commandments.
+
+Or some shit like that, I dunno, don't listen to me
+
+Almost like 
+
+comment: p3rgbfz
+parent: t1_p3rfh2o
+author: monstane
+created_utc: 1786757453
+edited: false
+body:
+yes i think there should be bitcoin news
