--- reddit-inside-job-speculation@20260806T034239Z
+++ reddit-inside-job-speculation@20260806T164414Z
@@ -220,3 +220,12 @@
 edited: false
 body:
 Thanks. I think I also did 100 dice rolls and was able to move my balance to an updated, new wallet, yesterday. 🤷‍♂️
+
+comment: p22oqgf
+parent: t3_1vfdmtf
+author: monstane
+created_utc: 1786028969
+edited: false
+body:
+Why would he mention it to Matt Odell in 2021 and basically give away that the bug is a vulnerability in the RNG?  
+There's a video of matt talking about it out there.
