--- reddit-inside-job-speculation@20260804T212742Z
+++ reddit-inside-job-speculation@20260804T233438Z
@@ -90,3 +90,11 @@
 This is a tricky one either way since you can’t fix already generated seeds.
 
 UPDATE: This actually has me thinking now lol… maybe they were aware of the issue and the introduction of dice rolls feature was their silent way out without announcing it and ruining their company??!?
+
+comment: p1qza2c
+parent: t1_p1okmrn
+author: grraarr
+created_utc: 1785885096
+edited: false
+body:
+They could also just have a social media person who has no idea what's going on with the code.
