--- reddit-inside-job-speculation@20260806T164414Z
+++ reddit-inside-job-speculation@20260807T142504Z
@@ -229,3 +229,13 @@
 body:
 Why would he mention it to Matt Odell in 2021 and basically give away that the bug is a vulnerability in the RNG?  
 There's a video of matt talking about it out there.
+
+comment: p263krk
+parent: t1_p1soeju
+author: Charming-Designer944
+created_utc: 1786061774
+edited: false
+body:
+50 dice rolls are required for 128 bits of entropy, which is considered the minimum secure level, and also the amount of entropy that is saved in a 12 word Bip-39 seed phrase.
+
+For 256 bits (24 word bip-39 seed phrase) 100 dice rolls are needed. (256/log2(6) = 99.034).
