--- reddit-inside-job-speculation@20260805T015807Z
+++ reddit-inside-job-speculation@20260805T080810Z
@@ -106,3 +106,93 @@
 edited: false
 body:
 Jews right 
+
+comment: p1rvgbd
+parent: t1_p1rtkru
+author: Dense_Egg_5858
+created_utc: 1785895453
+edited: false
+body:
+God fuck right off 
+
+comment: p1s0gho
+parent: t3_1vfdmtf
+author: [deleted]
+created_utc: 1785897115
+edited: false
+body:
+[removed]
+
+comment: p1sa5s6
+parent: t1_p1rvgbd
+author: DavidssonA
+created_utc: 1785900535
+edited: false
+body:
+New here?
+
+comment: p1sf3pp
+parent: t3_1vfdmtf
+author: Lost-Bowl3269
+created_utc: 1785902410
+edited: false
+body:
+Provavelmente. 
+
+comment: p1shfyk
+parent: t1_p1s0gho
+author: moviemaker2
+created_utc: 1785903332
+edited: false
+body:
+No they have not, If you'd even watched the first 20 seconds you'd have gotten to the part where the user just used a single dice roll, so they weren't even using the TRNG properly.
+
+Why are people so allergic to watching/reading the links they post as "evidence" for this nonsense?
+
+comment: p1shpln
+parent: t1_p1shfyk
+author: [deleted]
+created_utc: 1785903438
+edited: false
+body:
+[removed]
+
+comment: p1sler3
+parent: t1_p1shpln
+author: moviemaker2
+created_utc: 1785904962
+edited: false
+body:
+You presented that video as evidence that this exploit was called out 2 years ago.  An improper number of dice rolls is not related in any way, shape or form to this vulnerability.  You have no idea what you're talking about.
+
+comment: p1soeju
+parent: t3_1vfdmtf
+author: 4r4nd0mninj4
+created_utc: 1785906238
+edited: false
+body:
+Does anyone know how many dice rolls were needed to be safe from this?
+
+comment: p1t02th
+parent: t1_p1qbmgs
+author: The25thUser
+created_utc: 1785911560
+edited: false
+body:
+This is a good point, announcing publicly would have meant an attacker could have gotten to it first.
+
+comment: p1t8r82
+parent: t3_1vfdmtf
+author: TraditionalAd7423
+created_utc: 1785915721
+edited: false
+body:
+Can't spell crypto without "scam"
+
+comment: p1tbvhx
+parent: t1_p1obza0
+author: No-Aardvark-3840
+created_utc: 1785917244
+edited: false
+body:
+Yeah he’s going to prison 
