--- reddit-inside-job-speculation@20260805T211254Z
+++ reddit-inside-job-speculation@20260806T034239Z
@@ -204,3 +204,19 @@
 edited: false
 body:
 The one thing that makes me think it could have been an inside job is that their CTO wrote that shady crypto library under a pseudonym.
+
+comment: p1y0rnt
+parent: t1_p1soeju
+author: aureliorramos
+created_utc: 1785967532
+edited: false
+body:
+I think it was like 100 dice rolls when I set mine up several years ago
+
+comment: p1y5emo
+parent: t1_p1y0rnt
+author: 4r4nd0mninj4
+created_utc: 1785968926
+edited: false
+body:
+Thanks. I think I also did 100 dice rolls and was able to move my balance to an updated, new wallet, yesterday. 🤷‍♂️
