--- passport-not-affected@20260801T004619Z
+++ passport-not-affected@20260801T160953Z
@@ -54,6 +54,10 @@
 qna:
 We plan to allow for manual firmware updates outside of Envoy yes.
 When?
+qna
+August 1, 2026, 10:24am
+5
+This is currently scheduled for the next but one release, but our roadmap is always subject to change and as of right now, I do not have definitive time frame for this.
 Home
 Categories
 Guidelines
