--- reddit-btc-price-holding-up@20260804T202619Z
+++ reddit-btc-price-holding-up@20260804T212607Z
@@ -1041,3 +1041,11 @@
 edited: false
 body:
 For sowm reason I doubt that. Trump and his sons have a shared interest in BTC, they worn allow anything that prevents that interest to hurt BTC. Not because they care about us, like I said they aren't going to pass something that will hurt their pocket books. 
+
+comment: p1q9cpc
+parent: t1_p1puhpn
+author: FuturePrimitiv3
+created_utc: 1785877624
+edited: false
+body:
+😂 no down vote from me! I'm not a BTC "true believer", just a nerd who thinks the technology of BTC is interesting, but still struggles to find a valid use case. 
