--- reddit-btc-price-holding-up@20260804T135342Z
+++ reddit-btc-price-holding-up@20260804T142415Z
@@ -732,3 +732,27 @@
 edited: false
 body:
 It's good to see that someone at least has put some thought into this. Etfs do the same thing to stocks, they get very stretched thin the value gets manipulated heavily by large institutions. BTC would be no different and it would be ggs to breaking it's all-time high if the government passes the clarity act
+
+comment: p1ngcqr
+parent: t3_1vezvlv
+author: korean_kracka
+created_utc: 1785851655
+edited: false
+body:
+Everyone wants to call the bottom so badly lol. The bottom is in when people like you give up on calling the bottom 
+
+comment: p1nib03
+parent: t3_1vezvlv
+author: lumiosengineering
+created_utc: 1785852193
+edited: false
+body:
+This is very similar to the previous bottom, i hate to say it.  But then youll start seeing insane pumps out of nowhere wondering why is this happening, makes no sense.  And it doesnt.  But this is the market doing its thing.
+
+comment: p1nigut
+parent: t1_p1lsdd8
+author: korean_kracka
+created_utc: 1785852238
+edited: false
+body:
+Why do you think it can only get better? It can get much much MUCH worse lol
