--- reddit-btc-price-holding-up@20260805T080638Z
+++ reddit-btc-price-holding-up@20260805T211115Z
@@ -1161,3 +1161,13 @@
 edited: false
 body:
 不知道，可能年底吧
+
+comment: p1wp8s5
+parent: t3_1vezvlv
+author: Equivalent_Offer_67
+created_utc: 1785954822
+edited: false
+body:
+Because the price is manipulated to hell and back.
+
+Big players will raise the price, or drop the price when they feel like it.
