--- reddit-btc-price-holding-up@20260804T111422Z
+++ reddit-btc-price-holding-up@20260804T114332Z
@@ -506,3 +506,43 @@
 I do wish we will get a small relief to \~90k until  
 Next halving.  
 But emotionally it does feel rock bottom
+
+comment: p1mmip1
+parent: t1_p1lyb54
+author: Different_Umpire9003
+created_utc: 1785842076
+edited: false
+body:
+I’m so mad at myself for not buying then 
+
+comment: p1mnbty
+parent: t3_1vezvlv
+author: theoretical_hipster
+created_utc: 1785842389
+edited: false
+body:
+Bad News for CoinKite. Bitcoin is fine. 
+
+comment: p1moawe
+parent: t1_p1lgjce
+author: Particular-Fall-3194
+created_utc: 1785842761
+edited: false
+body:
+I can envision Mark Karpeles' fat ass sipping his iced coffee somewhere laughing at the coldcard hack
+
+comment: p1mp9h7
+parent: t1_p1mk3e1
+author: zante2033
+created_utc: 1785843126
+edited: false
+body:
+Not of any meaningful tier
+
+comment: p1mpd4p
+parent: t3_1vezvlv
+author: other_acc_banned
+created_utc: 1785843163
+edited: false
+body:
+I'm convinced it's the bottom
