--- reddit-btc-price-holding-up@20260804T114332Z
+++ reddit-btc-price-holding-up@20260804T121443Z
@@ -546,3 +546,35 @@
 edited: false
 body:
 I'm convinced it's the bottom
+
+comment: p1mrtlf
+parent: t3_1vezvlv
+author: youcantexterminateme
+created_utc: 1785844051
+edited: false
+body:
+yeah its hit the bottom, but who knows, maybe a little dip to come. the bottom line has always been predictable and I think thats basically because the market runs out of sellers and the holders are holding at that point.
+
+comment: p1muu9s
+parent: t1_p1moawe
+author: sonicode
+created_utc: 1785845119
+edited: false
+body:
+*"Whenever my code breaks, I just remember: 'There is a bug in Bitcoin."*
+
+comment: p1mv6gc
+parent: t1_p1lyh0e
+author: DistributionOk2111
+created_utc: 1785845236
+edited: false
+body:
+If so i take a small 20k loan and get another half
+
+comment: p1mvumk
+parent: t3_1vezvlv
+author: DavidssonA
+created_utc: 1785845463
+edited: false
+body:
+Normies have not heard of this coldcard thing.... I asked my wife... Not even I news story hit her screen
