--- reddit-btc-price-holding-up@20260804T182444Z
+++ reddit-btc-price-holding-up@20260804T185612Z
@@ -993,3 +993,27 @@
 edited: false
 body:
 Holds true.  Last year we had tons of great BTC news and it didn't move much.   This year tons of bad BTC news and it didn't move much.  BTC is now a stable coin.
+
+comment: p1p92eu
+parent: t1_p1na372
+author: MrKittenz
+created_utc: 1785868146
+edited: false
+body:
+Haha true 
+
+comment: p1pa81q
+parent: t1_p1lb884
+author: mlhender
+created_utc: 1785868433
+edited: false
+body:
+Dems feel they have a better chance of passing their version of the clarity act with much much higher capital gains tax. They’re going to pass on it this year
+
+comment: p1pazs0
+parent: t3_1vezvlv
+author: Ok-Refrigerator7284
+created_utc: 1785868626
+edited: false
+body:
+Best wishes to all of you who are buying now!  If this is the bottom, you will catch it and I will be missing out.  Is the normal cycle broken?  It just might be...The nature of past cycles suggest a plunge to 30K area should still happen toward November and if it doesn't then things have indeed changed and I'll be missing this altogether.  If price does drop to around 30K near the end of the year I will be joining the game then!!
