--- reddit-btc-price-holding-up@20260804T142415Z
+++ reddit-btc-price-holding-up@20260804T145344Z
@@ -756,3 +756,35 @@
 edited: false
 body:
 Why do you think it can only get better? It can get much much MUCH worse lol
+
+comment: p1nosj0
+parent: t3_1vezvlv
+author: Just_Bluebird_5268
+created_utc: 1785853938
+edited: false
+body:
+SPY ripping to new highs while cryptocoin stays flat? I mean it's not great is it
+
+comment: p1np27z
+parent: t3_1vezvlv
+author: IndependenceTop6501
+created_utc: 1785854010
+edited: false
+body:
+A lot of these hacks are going to wallets that don't know how to withdraw and cash out without getting caught.  Some will obviously, but much of it will be frozen for years to come.
+
+comment: p1nqi8q
+parent: t3_1vezvlv
+author: mrlandlord
+created_utc: 1785854394
+edited: false
+body:
+I am gonna feel even worse for people when BTC goes back to 100K
+
+comment: p1ntf2u
+parent: t1_p1mmip1
+author: XXsforEyes
+created_utc: 1785855160
+edited: false
+body:
+I wish I’d bought more then, which is exactly why I’m buying more now!
