--- reddit-btc-price-holding-up@20260804T192647Z
+++ reddit-btc-price-holding-up@20260804T202619Z
@@ -581,7 +581,7 @@
 
 comment: p1my2au
 parent: t1_p1lklla
-author: EnvironmentalYard467
+author: [deleted]
 created_utc: 1785846198
 edited: false
 body:
@@ -1025,3 +1025,19 @@
 edited: false
 body:
 That's a good sign 
+
+comment: p1puhpn
+parent: t1_p1n77dc
+author: [deleted]
+created_utc: 1785873666
+edited: false
+body:
+Retail is broke and institutional money is what drives btc at the moment. Im not saying that it's a good thing. Yes it's more manipulatable but maybe I already see how bitcoin is flawed in that way. Im only here to make a profit on it, down vote me if you want. I'll still hold some long term and sell some when institutional money comes in. 
+
+comment: p1pwj7k
+parent: t1_p1nebg7
+author: [deleted]
+created_utc: 1785874210
+edited: false
+body:
+For sowm reason I doubt that. Trump and his sons have a shared interest in BTC, they worn allow anything that prevents that interest to hurt BTC. Not because they care about us, like I said they aren't going to pass something that will hurt their pocket books. 
