--- reddit-btc-price-holding-up@20260804T225739Z
+++ reddit-btc-price-holding-up@20260804T233304Z
@@ -117,17 +117,11 @@
 
 comment: p1lbkro
 parent: t3_1vezvlv
-author: ThreeTonChonker
+author: [deleted]
 created_utc: 1785819936
 edited: false
 body:
-The result of events like this is that people realize the problem isn’t Bitcoin, it’s the products surrounding Bitcoin.
-
-If the hackers had only stolen Ethereum, would that mean the death of ETH?
-
-No. The coins are not getting hacked. Inferior third party products made for the coins are getting hacked.
-
-It means BTC is scarce, valuable, and requires defense. Thus the price reflects that and if anything it will go up.
+[removed]
 
 comment: p1lbmk7
 parent: t1_p1lbant
@@ -393,6 +387,16 @@
 body:
 It is a hard year for Bitcoin for sure…
 
+comment: p1lx2i3
+parent: t1_p1l5789
+author: Rino-Sensei
+created_utc: 1785830214
+edited: false
+body:
+"Coldcard was a black swan event" 
+
+No it isn't ... Because this news didn't go mainstream ...
+
 comment: p1lxb8d
 parent: t3_1vezvlv
 author: Thin_Needleworker795
@@ -595,6 +599,22 @@
 body:
 [removed]
 
+comment: p1mygz3
+parent: t1_p1l5avj
+author: JTHM8008
+created_utc: 1785846333
+edited: false
+body:
+And Cramer said he was selling all of his…. Or something. 
+
+comment: p1n0at3
+parent: t3_1vezvlv
+author: Aggravating-Owl-7050
+created_utc: 1785846923
+edited: false
+body:
+well I think because we are in our reddit Bubble, I nearly heard nothing about in besides this subreddit 
+
 comment: p1n0f8l
 parent: t1_p1lsh0t
 author: Aggravating-Owl-7050
