--- reddit-btc-price-holding-up@20260804T222637Z
+++ reddit-btc-price-holding-up@20260804T225739Z
@@ -433,6 +433,14 @@
 body:
 Seller exhaustion and dead bear momentum. There is no hope left for bears. We either go sideways from here or take off from demand surplus at some point.
 
+comment: p1m3c5f
+parent: t3_1vezvlv
+author: lexicon_riot
+created_utc: 1785833364
+edited: false
+body:
+It's because Cramer sold his BTC, inverse Cramer is never wrong
+
 comment: p1m7nfu
 parent: t3_1vezvlv
 author: Simple_Window21
@@ -882,6 +890,14 @@
 body:
 Feel so bad for anyone holding on cold card. Could this possibly be black swan of the bear ? Expecting lower lows from here til October personally 
 
+comment: p1o7s0b
+parent: t3_1vezvlv
+author: JordanPetterPans
+created_utc: 1785858874
+edited: false
+body:
+Because it literally have nothing to do with Bitcoin lol
+
 comment: p1o93iy
 parent: t3_1vezvlv
 author: originalgainster
@@ -1042,6 +1058,18 @@
 body:
 [removed]
 
+comment: p1q1owa
+parent: t3_1vezvlv
+author: FreezedPeachNow
+created_utc: 1785875576
+edited: false
+body:
+Does the rest of the world even know of, or understand the severity of, the cold card hack yet?
+
+And by the rest of the world, I mean people who don't think about Bitcoin 24/7 outside of this sub or Bitcoin Twitter or podcasts?
+
+I tried to make a similar post about this on Sunday and the moderators didn't allow it for whatever stupid reason.  
+
 comment: p1q9cpc
 parent: t1_p1puhpn
 author: FuturePrimitiv3
