--- reddit-btc-price-holding-up@20260804T131451Z
+++ reddit-btc-price-holding-up@20260804T135342Z
@@ -668,3 +668,67 @@
 edited: false
 body:
 If CLARITY helps institutions do you think thats gonna helps retail BTC investors? I'm gonna go out on a very short limb here and suggest that inviting corporate and institutional money only makes BTC more manipulatable under a thin the veil of legitimacy which will generate more volatility (more organized pump and dumps) and ultimately hurt BTC in the long run. 
+
+comment: p1n80y0
+parent: t3_1vezvlv
+author: UpstairsCheetah235
+created_utc: 1785849290
+edited: false
+body:
+Feels bottomy. However, hoping bitcoin stays low so strategy has to greatly reduce their stake over the next few years. And I can continue DCA at $60k instead of $120k. 
+
+comment: p1n9nzh
+parent: t1_p1m255j
+author: Astronaut6735
+created_utc: 1785849765
+edited: false
+body:
+My 3rd rodeo too. I was already expecting bottom to be closer to the end of the year, and nothing has changed my mind about that. 
+
+comment: p1na372
+parent: t1_p1lyb54
+author: revanevan7
+created_utc: 1785849888
+edited: false
+body:
+Ha $150 in 2015
+
+comment: p1na4ut
+parent: t1_p1lbant
+author: massn87
+created_utc: 1785849901
+edited: false
+body:
+By that logic, the same can be said about bull markets
+
+comment: p1nbia4
+parent: t1_p1li3fh
+author: Difficult-Repair1295
+created_utc: 1785850295
+edited: false
+body:
+This is nothing. 2015 was terrible. You literally thought Bitty was over and everyone arguing over block size. Crazy how they got the UASF passed and then Roger and Jihan had a tantrum.
+
+comment: p1ncq7b
+parent: t1_p1l9dzx
+author: confuzzledfather
+created_utc: 1785850643
+edited: false
+body:
+And yet the magic internet money is still worth $60k+ each. 
+
+comment: p1ncvd9
+parent: t3_1vezvlv
+author: Saibazz
+created_utc: 1785850684
+edited: false
+body:
+BTC holding up while everything else struggles? That's strength
+
+comment: p1nebg7
+parent: t1_p1n77dc
+author: Novel-Yak1927
+created_utc: 1785851091
+edited: false
+body:
+It's good to see that someone at least has put some thought into this. Etfs do the same thing to stocks, they get very stretched thin the value gets manipulated heavily by large institutions. BTC would be no different and it would be ggs to breaking it's all-time high if the government passes the clarity act
