--- reddit-btc-price-holding-up@20260804T093435Z
+++ reddit-btc-price-holding-up@20260804T101433Z
@@ -440,3 +440,27 @@
 edited: false
 body:
 MORE
+
+comment: p1m93eo
+parent: t1_p1l5avj
+author: broskibrokovski21
+created_utc: 1785836209
+edited: false
+body:
+It does feel and smell like it. Would love to see a hard bounce here just to see the faces of the "buying the October lows" gang
+
+comment: p1m96ds
+parent: t3_1vezvlv
+author: Madsen13140
+created_utc: 1785836250
+edited: false
+body:
+Refinance where , Michael ?
+
+comment: p1ma5fc
+parent: t3_1vezvlv
+author: Disavowed_Rogue
+created_utc: 1785836725
+edited: false
+body:
+It's not weird it's because the fundamentals are strong get it right
