--- reddit-btc-price-holding-up@20260804T212607Z
+++ reddit-btc-price-holding-up@20260804T215604Z
@@ -1049,3 +1049,19 @@
 edited: false
 body:
 😂 no down vote from me! I'm not a BTC "true believer", just a nerd who thinks the technology of BTC is interesting, but still struggles to find a valid use case. 
+
+comment: p1qdzc1
+parent: t1_p1lgjce
+author: Gloomy_Dependent_985
+created_utc: 1785878873
+edited: false
+body:
+It’s not the physical market impact that matters, but the phycological impact. The fact that it’s been a number of days and the market is flat is very bullish imo
+
+comment: p1qe8q3
+parent: t3_1vezvlv
+author: ContentBlackberry0
+created_utc: 1785878945
+edited: false
+body:
+70 million for bitcoin is a drop in the bucket. And it was not like it was all sold it was just moved either way amount would barely move it 1-2%
