--- reddit-btc-price-holding-up@20260804T172458Z
+++ reddit-btc-price-holding-up@20260804T175500Z
@@ -977,3 +977,11 @@
 edited: false
 body:
 什么时候能涨回来
+
+comment: p1ozm0z
+parent: t1_p1lxb8d
+author: Express_Living2264
+created_utc: 1785865804
+edited: false
+body:
+i mean we have numbers. ~1200 addresses were hacked. I assume most people using a cold card will have relied on the rng for the wallet creation. That's less than 1200 affected users world wide. crazy low imho.
