--- reddit-btc-price-holding-up@20260804T175500Z
+++ reddit-btc-price-holding-up@20260804T182444Z
@@ -985,3 +985,11 @@
 edited: false
 body:
 i mean we have numbers. ~1200 addresses were hacked. I assume most people using a cold card will have relied on the rng for the wallet creation. That's less than 1200 affected users world wide. crazy low imho.
+
+comment: p1p7c8r
+parent: t3_1vezvlv
+author: locotx
+created_utc: 1785867716
+edited: false
+body:
+Holds true.  Last year we had tons of great BTC news and it didn't move much.   This year tons of bad BTC news and it didn't move much.  BTC is now a stable coin.
