--- reddit-btc-price-holding-up@20260804T121443Z
+++ reddit-btc-price-holding-up@20260804T124349Z
@@ -578,3 +578,27 @@
 edited: false
 body:
 Normies have not heard of this coldcard thing.... I asked my wife... Not even I news story hit her screen
+
+comment: p1my2au
+parent: t1_p1lklla
+author: EnvironmentalYard467
+created_utc: 1785846198
+edited: false
+body:
+I feel this, bur part of me likes the possible pumps institutional money can give it. Like Fidelty and Black Rock, they want the clarity act to pass. Not because they care about us lol, but it helps them. I'd honestly prefer governments leave it alone too, but we need to realize governments will do whatever the fuck they want. Regardless bitcoin will carry on, tick tok..
+
+comment: p1n0f8l
+parent: t1_p1lsh0t
+author: Aggravating-Owl-7050
+created_utc: 1785846962
+edited: false
+body:
+can't wait to buy my btc 4 free /s
+
+comment: p1n0nfa
+parent: t3_1vezvlv
+author: Clear_Item_922
+created_utc: 1785847036
+edited: false
+body:
+The Cold Card fiasco has only affected 0.1% of Bitcoin.
