--- reddit-btc-price-holding-up@20260804T235644Z
+++ reddit-btc-price-holding-up@20260805T080638Z
@@ -647,14 +647,6 @@
 body:
 Most people are not even aware lmao
 
-comment: p1n4ihp
-parent: t1_p1lgopc
-author: trademarktower
-created_utc: 1785848237
-edited: false
-body:
-People with money hold their crypto along with all their other assets like 401k with OG brokers like Fidelity which have fraud protection guarantees.  
-
 comment: p1n53w7
 parent: t3_1vezvlv
 author: Key-Session6216
@@ -1137,3 +1129,35 @@
 edited: false
 body:
 Coldcard only a Black Swan on Reddit. It’s a tiny drop in the sea of Bitcoin 
+
+comment: p1rw14a
+parent: t1_p1li3fh
+author: Hugh108
+created_utc: 1785895646
+edited: false
+body:
+This bear market is like a stroll in the park compared to 2018, 2019 and the first half of 2020! 
+
+comment: p1rz2d3
+parent: t1_p1nosj0
+author: ximogui
+created_utc: 1785896651
+edited: false
+body:
+True
+
+comment: p1rzdgp
+parent: t1_p1ngcqr
+author: ximogui
+created_utc: 1785896753
+edited: false
+body:
+Each time, somebody somewhere gets it right. Let me shoot for my moment in the sun!
+
+comment: p1rzg9y
+parent: t1_p1oqi3l
+author: ximogui
+created_utc: 1785896779
+edited: false
+body:
+不知道，可能年底吧
