--- reddit-btc-price-holding-up@20260804T104315Z
+++ reddit-btc-price-holding-up@20260804T111422Z
@@ -472,3 +472,37 @@
 edited: false
 body:
 The longer we stay here the more support it builds at this level
+
+comment: p1mk3e1
+parent: t1_p1luhjo
+author: Efficient_Range1156
+created_utc: 1785841106
+edited: false
+body:
+The protocol has sustained an ongoing attack for the past 17 years.
+
+comment: p1mkgmk
+parent: t1_p1lgjce
+author: Acrobatic_News_4860
+created_utc: 1785841254
+edited: false
+body:
+It’s more about the fud and not sell pressure 
+
+comment: p1mklau
+parent: t3_1vezvlv
+author: Forded_Fiction24
+created_utc: 1785841307
+edited: false
+body:
+Macroeconomics are holding it up. Look at the broader picture
+
+comment: p1mkndo
+parent: t3_1vezvlv
+author: Acrobatic_News_4860
+created_utc: 1785841330
+edited: false
+body:
+I do wish we will get a small relief to \~90k until  
+Next halving.  
+But emotionally it does feel rock bottom
