--- reddit-btc-price-holding-up@20260804T155846Z
+++ reddit-btc-price-holding-up@20260804T162711Z
@@ -881,3 +881,67 @@
 edited: false
 body:
 Feel so bad for anyone holding on cold card. Could this possibly be black swan of the bear ? Expecting lower lows from here til October personally 
+
+comment: p1o93iy
+parent: t3_1vezvlv
+author: originalgainster
+created_utc: 1785859208
+edited: false
+body:
+this is pretty much the bottom. we might see a little lower for brief period but good luck catching that.
+
+comment: p1oagke
+parent: t1_p1o6dgo
+author: TheHollowed_Knight
+created_utc: 1785859555
+edited: false
+body:
+Sure, I honestly don’t give a shit what you think anyways. 
+
+comment: p1objko
+parent: t1_p1na372
+author: Different_Umpire9003
+created_utc: 1785859829
+edited: false
+body:
+Yeah, I know. I briefly worked in real estate and showed a house to a bitcoin millionaire in maybe 2019? He told me he’d made 4 million 
+
+comment: p1obqr8
+parent: t1_p1ntf2u
+author: Different_Umpire9003
+created_utc: 1785859879
+edited: false
+body:
+Yeah I mean I’m DCA-ing but it’s hardly anything. Oh well.
+
+comment: p1oc018
+parent: t1_p1obqr8
+author: TheRadishBros
+created_utc: 1785859944
+edited: false
+body:
+When it’s $1m per coin your DCA will look much more impressive.
+
+comment: p1ocfqw
+parent: t1_p1l5avj
+author: MainConcentrate2707
+created_utc: 1785860054
+edited: false
+body:
+Too optimistic i smell lower
+
+comment: p1oe4lg
+parent: t3_1vezvlv
+author: cryptoopotamus
+created_utc: 1785860481
+edited: false
+body:
+An unknown hardware wallet that only redditors used getting hacked isn’t even remotely close enough to move the needle for BTC. 
+
+comment: p1oeksq
+parent: t1_p1lyb54
+author: HungryCurrent7901
+created_utc: 1785860595
+edited: false
+body:
+Don’t remind me I’m just trying to make it to the end of this year lol
