--- reddit-btc-price-holding-up@20260804T215604Z
+++ reddit-btc-price-holding-up@20260804T222637Z
@@ -585,7 +585,7 @@
 created_utc: 1785846198
 edited: false
 body:
-I feel this, bur part of me likes the possible pumps institutional money can give it. Like Fidelty and Black Rock, they want the clarity act to pass. Not because they care about us lol, but it helps them. I'd honestly prefer governments leave it alone too, but we need to realize governments will do whatever the fuck they want. Regardless bitcoin will carry on, tick tok..
+[removed]
 
 comment: p1n0f8l
 parent: t1_p1lsh0t
@@ -1032,7 +1032,7 @@
 created_utc: 1785873666
 edited: false
 body:
-Retail is broke and institutional money is what drives btc at the moment. Im not saying that it's a good thing. Yes it's more manipulatable but maybe I already see how bitcoin is flawed in that way. Im only here to make a profit on it, down vote me if you want. I'll still hold some long term and sell some when institutional money comes in. 
+[removed]
 
 comment: p1pwj7k
 parent: t1_p1nebg7
@@ -1040,7 +1040,7 @@
 created_utc: 1785874210
 edited: false
 body:
-For sowm reason I doubt that. Trump and his sons have a shared interest in BTC, they worn allow anything that prevents that interest to hurt BTC. Not because they care about us, like I said they aren't going to pass something that will hurt their pocket books. 
+[removed]
 
 comment: p1q9cpc
 parent: t1_p1puhpn
@@ -1065,3 +1065,19 @@
 edited: false
 body:
 70 million for bitcoin is a drop in the bucket. And it was not like it was all sold it was just moved either way amount would barely move it 1-2%
+
+comment: p1qkria
+parent: t1_p1pwj7k
+author: FuturePrimitiv3
+created_utc: 1785880768
+edited: false
+body:
+They have no interest in BTC other than how they can profit from it in the short term. It's just another scam to them, nothing more. 
+
+comment: p1qolnn
+parent: t3_1vezvlv
+author: assclown356
+created_utc: 1785881872
+edited: false
+body:
+It was an inside job. 
