--- reddit-bricked-mid-transfer@20260804T082515Z
+++ reddit-bricked-mid-transfer@20260804T124220Z
@@ -32,3 +32,19 @@
 edited: false
 body:
 I initiated an test transaction using my coldcard Q with a multisig using the 1.5.0q firmware with no issues. Bricking is another CCQ issue that's separate from the RNG bug that was patched.
+
+comment: p1n0tyz
+parent: t1_p1h319n
+author: Soft-Spring9843
+created_utc: 1785847093
+edited: false
+body:
+Except it wasn’t 
+
+comment: p1n0yli
+parent: t1_p1gyxpt
+author: Soft-Spring9843
+created_utc: 1785847133
+edited: false
+body:
+Casa doesn’t allow transfer seed to sparrow…. I don’t understand why….. thank u
