--- reddit-do-not-dispose-device@20260804T082416Z
+++ reddit-do-not-dispose-device@20260804T134041Z
@@ -198,3 +198,11 @@
 Regardless, I wouldn't be expecting much from this company by the time vendors are paid off. Not like they wouldve been sitting on millions on profits. I doubt it at least. 
 
 
+
+comment: p1n3e5u
+parent: t1_p1iwtvv
+author: kawfeeman68
+created_utc: 1785847896
+edited: false
+body:
+Coldcard is from Canada. What will the process be for Coldcard device users from the US ?  Do you think everyone with many BTC's are going to be able to easily prove their funds? Once the gov gets involved,  will they ask you to not only prove your the owner funds but also to prove the source of the funds. 
