--- reddit-do-not-dispose-device@20260804T134041Z
+++ reddit-do-not-dispose-device@20260805T205655Z
@@ -206,3 +206,11 @@
 edited: false
 body:
 Coldcard is from Canada. What will the process be for Coldcard device users from the US ?  Do you think everyone with many BTC's are going to be able to easily prove their funds? Once the gov gets involved,  will they ask you to not only prove your the owner funds but also to prove the source of the funds. 
+
+comment: p1v95jj
+parent: t1_p1gyf31
+author: pwinne
+created_utc: 1785941912
+edited: false
+body:
+Depends they worked it in their favour with Celsius
