--- reddit-do-not-dispose-device@20260808T145906Z
+++ reddit-do-not-dispose-device@20260808T212906Z
@@ -218,3 +218,11 @@
 edited: false
 body:
 Easy to verify funds. Transaction history will show that
+
+comment: p2gzcym
+parent: t1_p2gxhbs
+author: kawfeeman68
+created_utc: 1786201178
+edited: false
+body:
+i was just considering the issues and difficulties for citizens of Countries other than Canada. Certainly, losses of non Canadian nationals will not be considered as important as those of Canadian Nationals..
