--- reddit-do-not-dispose-device@20260806T032602Z
+++ reddit-do-not-dispose-device@20260806T095716Z
@@ -95,18 +95,6 @@
 body:
 AFAIK, no need to update firmware if you rolled 100+ dice rolls neither. 
 
-comment: p1go4gs
-parent: t3_1ve8hve
-author: DonTheHolder
-created_utc: 1785770423
-edited: false
-body:
-Thats a lie.
-
-Them funds are in the coldcard owners inbox.
-
-You don't have to lie to kick it
-
 comment: p1gsh5a
 parent: t1_p1g03pq
 author: Many-Blueberry968
