--- reddit-self-custody-custodian-debate@20260804T234620Z
+++ reddit-self-custody-custodian-debate@20260805T011639Z
@@ -999,3 +999,11 @@
 edited: false
 body:
 hey when you got a beefcake on the block to try attract attention, got to do what you got to do
+
+comment: p1rmdkn
+parent: t3_1ve9or3
+author: Incoherentp00rnoises
+created_utc: 1785892475
+edited: false
+body:
+I’m not a boomer,but live a catastrophically busy life. ADHD maxing and multi hobby while also trying to start a farm and work full time shift work. I could never invest the brain time to self custody. Custodian has been best for me up until now. 
