--- reddit-self-custody-custodian-debate@20260804T184529Z
+++ reddit-self-custody-custodian-debate@20260804T194540Z
@@ -957,3 +957,21 @@
 edited: false
 body:
 "It takes 20 years to build a reputation and five minutes to ruin it. If you think about that, you'll do things differently". - Warren Buffett.   That old man was right, look at the Cold Card situation.  Also, mature adult men know you don't have to yell, you can whisper something and if what you have to say is important - people will listen.  All good thought, we like watching idiot kids make dumb mistakes we already learned from.  =)
+
+comment: p1po9st
+parent: t3_1ve9or3
+author: TypicalSalt5360
+created_utc: 1785872030
+edited: false
+body:
+He's CoinBASE'd
+
+comment: p1ppucd
+parent: t1_p1o7cpp
+author: Bourgey
+created_utc: 1785872445
+edited: false
+body:
+I didn't set one up until I heard of this ColdCard nonsense... Foolish of me yes, but it's quite easy and your security greatly increases. 
+
+On top of that, not forgetting the 25th word is as simple as writing down that made up word/term/phrase/numbers on the same paper given to you by your wallet brand with the other 24 words on it. 
