--- reddit-self-custody-custodian-debate@20260805T210001Z
+++ reddit-self-custody-custodian-debate@20260806T032911Z
@@ -1007,3 +1007,19 @@
 edited: false
 body:
 Sam Bankman fried me.
+
+comment: p1ys9h2
+parent: t3_1ve9or3
+author: Walla_Walla_26
+created_utc: 1785976285
+edited: false
+body:
+My shorts would be wayyyyy shorter
+
+comment: p1zal1x
+parent: t1_p1p6jll
+author: Catoutofthebag69
+created_utc: 1785982311
+edited: false
+body:
+It was just a meme from the Simpsons bud. You didn’t have to type all of that 
