--- reddit-self-custody-custodian-debate@20260804T161558Z
+++ reddit-self-custody-custodian-debate@20260804T184529Z
@@ -949,3 +949,11 @@
 edited: false
 body:
 Its honestly unfathomable to me that people still use wallets with no custom passphrase
+
+comment: p1p6jll
+parent: t1_p1ih3q5
+author: locotx
+created_utc: 1785867518
+edited: false
+body:
+"It takes 20 years to build a reputation and five minutes to ruin it. If you think about that, you'll do things differently". - Warren Buffett.   That old man was right, look at the Cold Card situation.  Also, mature adult men know you don't have to yell, you can whisper something and if what you have to say is important - people will listen.  All good thought, we like watching idiot kids make dumb mistakes we already learned from.  =)
