--- reddit-where-to-transfer@20260804T090145Z
+++ reddit-where-to-transfer@20260806T095924Z
@@ -33,17 +33,11 @@
 
 comment: p10q77q
 parent: t3_1vbmuet
-author: DonTheHolder
+author: [deleted]
 created_utc: 1785563240
 edited: false
 body:
-No migration. Get off the hardware wallets and mitigate the issue by going to the CEX such as Coinbase, ETF stock, etc. 
-
-You're going to pay taxes regardless, unless you're in another country that has none or low taxes.
-
-Safer for you, your family, and being wrenched 🔧 
-
-Not to mention there is a beneficiary path versus some posted note that your wife finds before she divorces you and has a boating accident. 
+[removed]
 
 comment: p10r3i8
 parent: t3_1vbmuet
@@ -78,11 +72,3 @@
 edited: false
 body:
 Yeah I don't know how they're going to get out of this one. And there's no way to contact them for refunds... 
-
-comment: p1cv2hp
-parent: t1_p16n8mb
-author: DonTheHolder
-created_utc: 1785717152
-edited: false
-body:
-Lies.
