--- reddit-still-using-coldcard@20260806T032638Z
+++ reddit-still-using-coldcard@20260807T061401Z
@@ -478,3 +478,29 @@
 edited: false
 body:
 No fucking way I’ll use my coldcards again.  One of them is brand new, never used. 
+
+comment: p23hbwp
+parent: t3_1vdnfov
+author: shoebertdoubert
+created_utc: 1786036106
+edited: false
+body:
+>I like the device and don't really want to switch
+
+your choice, but sear those words into your brain 
+
+comment: p23hj97
+parent: t1_p1alhus
+author: shoebertdoubert
+created_utc: 1786036156
+edited: false
+body:
+Insane cope 
+
+comment: p23wpwl
+parent: t3_1vdnfov
+author: FeedSeparate
+created_utc: 1786039875
+edited: false
+body:
+F**k that!!!!🤣🤣🤣🤣🤣🤣🤣
