--- reddit-still-using-coldcard@20260805T142240Z
+++ reddit-still-using-coldcard@20260806T032638Z
@@ -470,3 +470,11 @@
 And figure out a way to make your selection *random*. Roll dice, flip cards, cut the paper up into the individual words and draw them out of a hat, etc. 
 
 Six words make an extremely strong passphrase. Eight is essentially unbreakable. Twelve of the first list is the same strength as a Bitcoin private key. 
+
+comment: p1xmeu4
+parent: t3_1vdnfov
+author: WBDubya
+created_utc: 1785963453
+edited: false
+body:
+No fucking way I’ll use my coldcards again.  One of them is brand new, never used. 
