--- reddit-still-using-coldcard@20260804T083723Z
+++ reddit-still-using-coldcard@20260805T075424Z
@@ -440,3 +440,19 @@
 ...might wanna mix in some lowercase letters when you do the passphrase thing
 
 JUST SAYIN!!!!!!@!!21
+
+comment: p1salhc
+parent: t1_p1acf3j
+author: PeroniBites
+created_utc: 1785900696
+edited: false
+body:
+There’s no such thing as random words. You gotta mix the wordswith89457and@$$-/?!. ….like that. 
+
+comment: p1savxa
+parent: t1_p1cnvrw
+author: PeroniBites
+created_utc: 1785900803
+edited: false
+body:
+I did 300 dice rolls. Funds were safu. Might keep it as a signing tool, but that’s it. 
