--- coldcard-downloads@20260808T135400Z
+++ coldcard-downloads@20260812T000517Z
@@ -20,7 +20,6 @@
 Temporary Seeds
 Get in touch
 X / Twitter
-Telegram
 Support
 Company
 About
@@ -80,7 +79,6 @@
 Temporary Seeds
 Get in touch
 X / Twitter
-Telegram
 Support
 Company
 About
@@ -170,7 +168,6 @@
 Privacy Policy
 STAY CONNECTED
 @COLDCARDwallet
-Telegram
 Blog
 Newsletter
 Copyright © 2026 Coinkite Inc.
