--- reddit-randomness-collision-question@20260812T173325Z
+++ reddit-randomness-collision-question@20260814T234630Z
@@ -104,3 +104,25 @@
 160 bits = comparable to atoms in Earth's moon
 
 Spoiler alert, with traditional computing we will never find a collision that corresponds with a specific 160bit bitcoin address. Practically impossible. So to answer the OPs question, it is practically impossible for a proper RNG function to create one of the hacked cold-card PRNG deterministic wallets. It is THEORETICALLY possible, but it will never happen.
+
+comment: p3mstg0
+parent: t1_p32cujo
+author: phoebeethical
+created_utc: 1786709621
+edited: false
+body:
+So a passphrase can convert a master password to the identical master password generated with a different set of words?  
+
+comment: p3ngkn5
+parent: t1_p3mstg0
+author: logan-807128
+created_utc: 1786716648
+edited: false
+body:
+I know what you mean although there is a technical difference in your wording, what you said is correct.
+
+Say you used the seed words "hello world blah ... ", it would create the private key 1747293... (a super big number). Once you add a passphrase, the private key would now become a completely different number say 4827282..., there exists another seed words "somewhere out there..." without passphrase that would generate the exact same private key 4827282...
+
+There is a reason why some people call the passphrase the 13th word (12 word seed) or 25th word (24 word seed) because in many ways that's more correct than thinking the passphrase as password for your seed words. 
+
+
