--- stackernews-dear-podcasters@20260804T121228Z
+++ stackernews-dear-podcasters@20260804T124132Z
@@ -4,17 +4,17 @@
       "comments": {
         "comments": [
           {
+            "createdAt": "2026-08-03T13:34:52.978Z",
+            "text": ">Every time anyone tried to poke at ColdCard or even just ask questions nvk would always dismiss them, call it FUD and attack their character.\n\n>Nvm the absolute abysmal attitude he had against competitor, who now still acted like gentlemen despite this generational dunking opportunity.\n\nNo one is infallible point blank. Dunk on NVK today trust me a day will come when someone will dunk on you! Way of the universe. We all make mistakes some are more graceful about it than others. \n\nCoinkite still makes cool products some I will continue to use like the block clock and sat cards. Others I was excited about (Arca box) I may fade now. But when when podcasters shill products we still have to take responsibility for using them for better or for worse.",
+            "user": {
+              "name": "BlokchainB"
+            }
+          },
+          {
             "createdAt": "2026-08-03T19:20:06.579Z",
             "text": "[![people-are-stupid-news.jpg](https://i.postimg.cc/brGryYg8/people-are-stupid-news.jpg)](https://postimg.cc/30TY9Tzb)",
             "user": {
               "name": "DarthCoin"
-            }
-          },
-          {
-            "createdAt": "2026-08-03T13:34:52.978Z",
-            "text": ">Every time anyone tried to poke at ColdCard or even just ask questions nvk would always dismiss them, call it FUD and attack their character.\n\n>Nvm the absolute abysmal attitude he had against competitor, who now still acted like gentlemen despite this generational dunking opportunity.\n\nNo one is infallible point blank. Dunk on NVK today trust me a day will come when someone will dunk on you! Way of the universe. We all make mistakes some are more graceful about it than others. \n\nCoinkite still makes cool products some I will continue to use like the block clock and sat cards. Others I was excited about (Arca box) I may fade now. But when when podcasters shill products we still have to take responsibility for using them for better or for worse.",
-            "user": {
-              "name": "BlokchainB"
             }
           },
           {
