--- bitcointalk-dice-bias-entropy@20260807T100538Z
+++ bitcointalk-dice-bias-entropy@20260807T130012Z
@@ -149,5 +149,20 @@
 but if you then keep it on a sheet of paper... bad
 if you don't put the passphrase, bad
 it's a whole journey, not a single mega paranoid action
+Title: Re: dice or not dice
+Post by: Satofan44 on August 07, 2026, 12:13:21 PM
+Quote from: babo on Today at 08:05:49 AM
+These are all interesting discussions, and they are all more or less paranoid methods.
+However, as a cybersecurity expert I always remind people that the strength of a chain is always measured by weighing its weakest link.
+that's the strength index of a chain
+if you generated the seed using fairy farts which are random, good
+but if you then keep it on a sheet of paper... bad
+if you don't put the passphrase, bad
+it's a whole journey, not a single mega paranoid action
+Sure, but you need to think about it more simply and especially in the context of what happened recently. One should not invest $100k to create a vault that takes 5 days of verification to enter in order to protect $5000. Most users should neither do dice setups, nor multisignatures. The security setup must be as practical as it is safe in relation to the value that it is supposed to be protecting, everything else is wrong for one reason or another. A good passphrase solves:
+1) Weak seed generation by the user.
+2) Weak seed generation by the company.
+3) Any other potential human error that could be introduced in these advanced setups.
+It just works, and it does not need any special knowledge. The same knowledge regarding the backing up of your seed applies to the seedphrase, redundancy, avoid keeping the whole thing in a single place, etc. This is the best setup for most users.Being paranoid is never warranted, advanced setups should stay primarily for hobby users who are personally interested in them and those that are protecting very high values.
 Powered by SMF 1.1.19 |
 SMF © 2006-2009, Simple Machines
