--- bitcointalk-dice-bias-entropy@20260807T130012Z
+++ bitcointalk-dice-bias-entropy@20260808T020412Z
@@ -151,7 +151,7 @@
 it's a whole journey, not a single mega paranoid action
 Title: Re: dice or not dice
 Post by: Satofan44 on August 07, 2026, 12:13:21 PM
-Quote from: babo on Today at 08:05:49 AM
+Quote from: babo on August 07, 2026, 08:05:49 AM
 These are all interesting discussions, and they are all more or less paranoid methods.
 However, as a cybersecurity expert I always remind people that the strength of a chain is always measured by weighing its weakest link.
 that's the strength index of a chain
