--- reddit-price-mockery@20260806T101239Z
+++ reddit-price-mockery@20260806T164104Z
@@ -230,3 +230,23 @@
 edited: false
 body:
 i can’t get over someone smart enough to be able to retire on bitcoin before it’s even reached its full potential not having the sensibility to move their bitcoin to cold storage sober…..i don’t mean that in a condescending way i’m just genuinely nonplussed. 
+
+comment: p21sbp4
+parent: t1_p20uif5
+author: ZICRON_ULTRA
+created_utc: 1786020031
+edited: false
+body:
+You and me both. 
+
+I wasn't drunk or blitz, but I forgot to write down a couple of key things that I thought that I would remember months or years later that I clearly did not, hence my cautionary tale for the class. 
+
+Trust me, for about 24 hours, my wife was VERY not happy with me. 😔 
+
+comment: p22eod7
+parent: t1_p21sbp4
+author: tigercublondon
+created_utc: 1786026339
+edited: false
+body:
+please tell me it’s not gone forever?
