--- reddit-price-mockery@20260806T033910Z
+++ reddit-price-mockery@20260806T101239Z
@@ -214,3 +214,19 @@
 edited: false
 body:
 I wouldn't take em for free
+
+comment: p20tpdf
+parent: t1_p1hei0c
+author: tigercublondon
+created_utc: 1786005912
+edited: false
+body:
+what would you use them for?
+
+comment: p20uif5
+parent: t1_p1nvv80
+author: tigercublondon
+created_utc: 1786006316
+edited: false
+body:
+i can’t get over someone smart enough to be able to retire on bitcoin before it’s even reached its full potential not having the sensibility to move their bitcoin to cold storage sober…..i don’t mean that in a condescending way i’m just genuinely nonplussed. 
