--- reddit-im-out@20260804T224944Z
+++ reddit-im-out@20260804T232530Z
@@ -380,3 +380,11 @@
 Being humbled isn't what I would call it. People put trust in a company that should have done things properly but didn't. Now a bunch of people got ripped off because ColdCard messed up. If this taught something to everyone it's DO NOT TRUST anybody or any company and generate your seed with 99 dice roll and make sure you do that right and include a 8 word or more paraphrase on top of it. This was a horrible situation for many people who didn't deserve it.
 
 This was some epic BS and Cold Card is 100% to blame for this.
+
+comment: p1r0li2
+parent: t3_1vfbi4w
+author: RandyJohnsonsBird
+created_utc: 1785885509
+edited: false
+body:
+The people that dont think like you seem to like getting bent over all day long
