--- reddit-im-out@20260804T232530Z
+++ reddit-im-out@20260805T001917Z
@@ -388,3 +388,23 @@
 edited: false
 body:
 The people that dont think like you seem to like getting bent over all day long
+
+comment: p1r7t0w
+parent: t1_p1q0eis
+author: moviemaker2
+created_utc: 1785887779
+edited: false
+body:
+Did all the world's cybercriminals just leave 80 million dollars on the table for 5 years out of their goodness of their hearts?
+
+Why did Coinkite not patch this easily fixable and company-destroying bug until after the wallets were drained, then patch it in a few hours?
+
+comment: p1rbsb1
+parent: t1_p1r7t0w
+author: adequate_redditor
+created_utc: 1785889059
+edited: false
+body:
+I’m not saying coldcard is not at fault. Their randomness was not random. It’s definitely a big flaw.
+
+I’m just saying that such attacks were used before many times with other wallet providers. I don’t think anyone should consider the 100 dice rolls “above and beyond”, regardless of the wallet maker. 
