--- reddit-im-out@20260804T201853Z
+++ reddit-im-out@20260804T204829Z
@@ -326,3 +326,23 @@
 edited: false
 body:
 Ledger is Number One ☝️
+
+comment: p1pzxcz
+parent: t3_1vfbi4w
+author: StageComprehensive81
+created_utc: 1785875104
+edited: false
+body:
+Is not the device, but the software. They updated the software. You can make a new seed from it and a strong passphrase no need to throw away the Coldcard. 
+
+comment: p1q0eis
+parent: t1_p1oxxr8
+author: adequate_redditor
+created_utc: 1785875229
+edited: false
+body:
+Yes, but there were similar hacks before. Many YouTube videos from years ago go over this exact situation. The hardware is sufficient for entropy, but if it’s flawed in anyway then you get something like the coldcard hack…
+
+https://youtu.be/D9j5y3tyMo8
+
+Wallets that don’t offer the dice roll function could be at risk too if similar vulnerabilities are uncovered.
