--- reddit-im-out@20260804T191741Z
+++ reddit-im-out@20260804T194823Z
@@ -294,3 +294,11 @@
 body:
 Even scarier to think it wasnt incompetente at all. Deliberate, planned way in advance:
 https://www.reddit.com/r/CryptoCurrency/s/s5fNgg9k4X
+
+comment: p1prhh8
+parent: t1_p1oy5k1
+author: ady1583
+created_utc: 1785872875
+edited: false
+body:
+…. And I am here expecting the company to pay back the $$ I spent to get 2 CCs that I got in 2023
