--- reddit-im-out@20260805T210343Z
+++ reddit-im-out@20260806T033244Z
@@ -632,3 +632,43 @@
 edited: false
 body:
 Those are valid points. I'm not expecting to rely on any future fixes or support. I am not even installing their emergency patches, so if/when they go under it won't impact my use of the cold card. But...yes, it is likely worth looking into other solutions that will be reliable and eventually take cold card out of the picture entirely. 
+
+comment: p1yao12
+parent: t3_1vfbi4w
+author: FerdaStonks
+created_utc: 1785970553
+edited: false
+body:
+Don’t throw away your coldcard, it still works well as an airgapped way to calculate your 24th word if you make a wallet by flipping a coin 256 times.
+
+Use it to get that word and then use another wallet to actually transact with. I trust it to calculate 1 word and then turn off but wouldn’t trust it to actually enter dice rolls and get a legit seed or to transact with.
+
+comment: p1yeg9r
+parent: t1_p1v78j6
+author: Ok-Information-2428
+created_utc: 1785971750
+edited: false
+body:
+If the attacker was a highly sophisticated operation it would be more understandable- in this case though it was their own total incompetence and complete lack of concern with the most critical code path in their wallet
+
+comment: p1yu0bw
+parent: t1_p1v78j6
+author: rockorangebear
+created_utc: 1785976860
+edited: false
+body:
+Honestly, any competent senior developer from a top tech company could've done it. A wallet's functionality isn't that large in scope, in fact it's surprisingly simple.
+
+In terms of security, making sure that a newly generated seed is done so randomly is one of the only critical tasks needed.
+
+It's not difficult to have automated tests in place.
+
+comment: p1yv366
+parent: t1_p1pzxcz
+author: rockorangebear
+created_utc: 1785977215
+edited: false
+body:
+Why would you trust a team of clowns though? They're either complete morons or they're in on it.
+
+Maybe they'll accidentally push an update that uploads your private key next, who knows? How anyone can trust these people after this is beyond me.
