--- reddit-im-out@20260804T211845Z
+++ reddit-im-out@20260804T214836Z
@@ -303,14 +303,6 @@
 body:
 …. And I am here expecting the company to pay back the $$ I spent to get 2 CCs that I got in 2023
 
-comment: p1pvrei
-parent: t3_1vfbi4w
-author: Emotional-Run9144
-created_utc: 1785874006
-edited: false
-body:
-No form of custody is every 100% safe, not with crypto or real world assets.
-
 comment: p1pwg0u
 parent: t3_1vfbi4w
 author: hakunamatata0002
@@ -370,3 +362,11 @@
 edited: false
 body:
 As someone almost buying a coldcard Q, im kinda disappointed. Because from a hardware perspective, it seems amazing. Jade uses a shit camera, q uses a qr reader. Trezor uses a tiny touchscreen. The Q uses a full physical keyboard…like fuck man. Honestly want other companies to just use this hardware for there firmwares. Or diy builtds
+
+comment: p1qdhay
+parent: t3_1vfbi4w
+author: JumpProfessional3372
+created_utc: 1785878737
+edited: false
+body:
+To where did you move?
