--- reddit-im-out@20260805T004920Z
+++ reddit-im-out@20260805T011924Z
@@ -424,3 +424,21 @@
 edited: false
 body:
 The only information that was leaked was names, addresses, and emails from a third party hack.  If people are foolish enough to fall for scams, that’s their own fault. 
+
+comment: p1rkq8q
+parent: t3_1vfbi4w
+author: kotb0614
+created_utc: 1785891936
+edited: false
+body:
+Samesies. I just keep coming back here to try to save/warn other orangepilled bros and broettes
+
+comment: p1rku7g
+parent: t1_p1pfyij
+author: SomeGuyInOz
+created_utc: 1785891972
+edited: false
+body:
+I don’t believe that for a second. If it were intentional, it would be much smarter to simply drain a few high value wallets. There is no way they would drain this much and draw such attention to this bug.
+
+I understand how upsetting and infuriating this is, especially for those directly affected. But the majority of reactions I see are pure emotion and not much intellect.
