--- reddit-im-out@20260806T033244Z
+++ reddit-im-out@20260807T193136Z
@@ -672,3 +672,13 @@
 Why would you trust a team of clowns though? They're either complete morons or they're in on it.
 
 Maybe they'll accidentally push an update that uploads your private key next, who knows? How anyone can trust these people after this is beyond me.
+
+comment: p2arswq
+parent: t3_1vfbi4w
+author: eternal_recurrence12
+created_utc: 1786121841
+edited: false
+body:
+
+I also moved my funds to Coinbase for now, I think it's much safer. Thank God I used a passphrase and rolled dice to generate my seeds on the Coldcard, but after this event, I completely lost trust in Coinkite. Who can guarantee there aren't other bugs out there?
+
