--- reddit-im-out@20260807T193136Z
+++ reddit-im-out@20260808T150535Z
@@ -288,12 +288,11 @@
 
 comment: p1pfyij
 parent: t1_p1nlzuy
-author: anotherfroggyevening
+author: [deleted]
 created_utc: 1785869873
 edited: false
 body:
-Even scarier to think it wasnt incompetente at all. Deliberate, planned way in advance:
-https://www.reddit.com/r/CryptoCurrency/s/s5fNgg9k4X
+[removed]
 
 comment: p1prhh8
 parent: t1_p1oy5k1
@@ -532,14 +531,6 @@
 body:
 Imagine continuing to trust a CC with your hard earned bitcoin after this, you would deserve to be rugged
 
-comment: p1svl5z
-parent: t1_p1rku7g
-author: anotherfroggyevening
-created_utc: 1785909477
-edited: false
-body:
-A "bug" imo works much better according to that logic. "It can't be deliberate, because it's a bug." Sure. Seeing the general tone of comments, seems like it was a perfect game plan.
-
 comment: p1t1im3
 parent: t1_p1rd237
 author: Sammy262
