--- reddit-im-out@20260804T204829Z
+++ reddit-im-out@20260804T211845Z
@@ -346,3 +346,27 @@
 https://youtu.be/D9j5y3tyMo8
 
 Wallets that don’t offer the dice roll function could be at risk too if similar vulnerabilities are uncovered.
+
+comment: p1q6krx
+parent: t1_p1p4tx9
+author: RyzenNinja
+created_utc: 1785876874
+edited: false
+body:
+I didnt say it wouldn't happen but what was my alternative in a pinch leave it there and let it get stolen.
+
+comment: p1q6vbe
+parent: t1_p1ogj0b
+author: RyzenNinja
+created_utc: 1785876953
+edited: false
+body:
+Yes on the ColdCard I didnt erase it. I need to apply the firmware update and generate a new seed then test it out a bit but I'll never trust it to hold a lot.
+
+comment: p1qah2m
+parent: t1_p1ocvfz
+author: OddioClay
+created_utc: 1785877927
+edited: false
+body:
+As someone almost buying a coldcard Q, im kinda disappointed. Because from a hardware perspective, it seems amazing. Jade uses a shit camera, q uses a qr reader. Trezor uses a tiny touchscreen. The Q uses a full physical keyboard…like fuck man. Honestly want other companies to just use this hardware for there firmwares. Or diy builtds
