--- reddit-im-out@20260805T142811Z
+++ reddit-im-out@20260805T210343Z
@@ -596,3 +596,39 @@
 edited: false
 body:
 Of course, any rational human being should toss any coldcard to the trash bin, I’ve disposed my two mk4s and have incoming parts to build a few seed signers and a jade ordered to make a multisig setup. This has been too close of a call, if I had been drained I don’t know what would have happened to me, I’m scared just to think of it.
+
+comment: p1v78j6
+parent: t1_p1nlzuy
+author: deny_by_default
+created_utc: 1785941421
+edited: false
+body:
+Isn't the "company" only like 5 people?  How can keep up with security exploits and audit your code properly with a staff that size?  Frankly, I'm amazed it took this long for this disaster to happen.  It's a shame too.  I really like the Coldcard Q, but I've lost faith in Coinkite.
+
+comment: p1vb9cu
+parent: t1_p1owrme
+author: cworxnine
+created_utc: 1785942455
+edited: false
+body:
+I was considering the same, it's just a signing device and I used dice rolls so maybe I can continue using it right? But now their future is unclear, their team and company is very small and unlikely to absorb the loss of future income for years to come. 
+
+It's unlikely they'll double down and reinvest in a dozen more developers, or revamp their broken QA. I doubt there's more than 30 full time employees there. This company doesn't have many business skills except grass roots marketing - which is gone now. How will this affect future firmware and support? 
+
+It's just too risky to keep using their products when other viable solutions are out there who are WAY more funded and can invest more money in support and continued development.
+
+comment: p1veieb
+parent: t1_p1qah2m
+author: cworxnine
+created_utc: 1785943285
+edited: false
+body:
+Trezor Safe 7 is actually pretty big. Ledger Flex is also a great device. I agree the CCQ is a sexy device, but I only use mine a few times a year anyways. Just needs to be easy to use and stable.
+
+comment: p1w3iok
+parent: t1_p1vb9cu
+author: Stack_Sats_Often
+created_utc: 1785949479
+edited: false
+body:
+Those are valid points. I'm not expecting to rely on any future fixes or support. I am not even installing their emergency patches, so if/when they go under it won't impact my use of the cold card. But...yes, it is likely worth looking into other solutions that will be reliable and eventually take cold card out of the picture entirely. 
