--- reddit-im-out@20260804T181721Z
+++ reddit-im-out@20260804T191741Z
@@ -285,3 +285,12 @@
 edited: false
 body:
 What makes you think that another hw wallet won't get hacked in a similar way?
+
+comment: p1pfyij
+parent: t1_p1nlzuy
+author: anotherfroggyevening
+created_utc: 1785869873
+edited: false
+body:
+Even scarier to think it wasnt incompetente at all. Deliberate, planned way in advance:
+https://www.reddit.com/r/CryptoCurrency/s/s5fNgg9k4X
