--- reddit-im-out@20260808T150535Z
+++ reddit-im-out@20260808T213527Z
@@ -673,3 +673,11 @@
 
 I also moved my funds to Coinbase for now, I think it's much safer. Thank God I used a passphrase and rolled dice to generate my seeds on the Coldcard, but after this event, I completely lost trust in Coinkite. Who can guarantee there aren't other bugs out there?
 
+
+comment: p2i7983
+parent: t3_1vfbi4w
+author: carothersjoshua
+created_utc: 1786213673
+edited: false
+body:
+It sucks because I really like Coldcard and finally got used to using it and now it’s all over. Like that girlfriend you just can’t get out of your mind. 
