--- reddit-im-out@20260804T174748Z
+++ reddit-im-out@20260804T181721Z
@@ -277,3 +277,11 @@
 edited: false
 body:
 Ive seen people say theyre okay with continuing to use the coldcard with their dice generated seedphrase and passphrase, but fuck that... This company doesnt deserve anyone staying with them. If they missed this bug that was there since 2021 what else have they missed... plus they arent gonna reimburse victims of this disaster. 
+
+comment: p1p4tx9
+parent: t1_p1ocvfz
+author: mj
+created_utc: 1785867095
+edited: false
+body:
+What makes you think that another hw wallet won't get hacked in a similar way?
