--- reddit-im-out@20260804T194823Z
+++ reddit-im-out@20260804T201853Z
@@ -302,3 +302,27 @@
 edited: false
 body:
 …. And I am here expecting the company to pay back the $$ I spent to get 2 CCs that I got in 2023
+
+comment: p1pvrei
+parent: t3_1vfbi4w
+author: Emotional-Run9144
+created_utc: 1785874006
+edited: false
+body:
+No form of custody is every 100% safe, not with crypto or real world assets.
+
+comment: p1pwg0u
+parent: t3_1vfbi4w
+author: hakunamatata0002
+created_utc: 1785874187
+edited: false
+body:
+Bet they'll come out as different name for wallet soon!
+
+comment: p1pxriw
+parent: t3_1vfbi4w
+author: Aqua-Barracuda
+created_utc: 1785874536
+edited: false
+body:
+Ledger is Number One ☝️
