--- reddit-im-out@20260805T001917Z
+++ reddit-im-out@20260805T004920Z
@@ -408,3 +408,19 @@
 I’m not saying coldcard is not at fault. Their randomness was not random. It’s definitely a big flaw.
 
 I’m just saying that such attacks were used before many times with other wallet providers. I don’t think anyone should consider the 100 dice rolls “above and beyond”, regardless of the wallet maker. 
+
+comment: p1rd237
+parent: t1_p1pxriw
+author: adequate_redditor
+created_utc: 1785889465
+edited: false
+body:
+There was a big data leak a few years ago. Not directly related to the device itself, but people lost millions to phishing attempts.
+
+comment: p1ren0n
+parent: t1_p1rd237
+author: Aqua-Barracuda
+created_utc: 1785889972
+edited: false
+body:
+The only information that was leaked was names, addresses, and emails from a third party hack.  If people are foolish enough to fall for scams, that’s their own fault. 
