--- reddit-dice-wallets-at-risk@20260805T142153Z
+++ reddit-dice-wallets-at-risk@20260807T125232Z
@@ -636,3 +636,11 @@
 edited: false
 body:
 It's there a particular dice roll method that people use? I plan to do it soon. Just curious if they're is a solid guide to follow or if it's pretty straight forward. 
+
+comment: p28qr35
+parent: t3_1veixnw
+author: Makunouchiipp0
+created_utc: 1786100638
+edited: false
+body:
+Do not trust your memory. 
