--- reddit-dice-wallets-at-risk@20260804T151124Z
+++ reddit-dice-wallets-at-risk@20260804T164242Z
@@ -590,3 +590,21 @@
 exactly, using CC is simply adding new, extra bugs and vulnerabilities
 
 how do you know your CC is not leaking parts of your seed with every transaction for example
+
+comment: p1oegyp
+parent: t1_p1hde7y
+author: Scissorhat
+created_utc: 1785860568
+edited: false
+body:
+> I am in a similar position and I am not moving my Bitcoin
+
+Ballsy.
+
+comment: p1ogrty
+parent: t3_1veixnw
+author: paulm95
+created_utc: 1785861144
+edited: false
+body:
+You are all good 👍 
