--- reddit-dice-wallets-at-risk@20260804T164242Z
+++ reddit-dice-wallets-at-risk@20260804T171105Z
@@ -608,3 +608,23 @@
 edited: false
 body:
 You are all good 👍 
+
+comment: p1om17i
+parent: t3_1veixnw
+author: painfuldrp
+created_utc: 1785862454
+edited: false
+body:
+No, you should be very safe
+
+comment: p1onh59
+parent: t1_p1hsmgm
+author: painfuldrp
+created_utc: 1785862813
+edited: false
+body:
+I doubt the person with the dice roll did it properly or is confusing which device he used to dice roll.  There’s only that one case and there’s a 99% chance it was some kind of user error or the person is lying for engagement 
+
+You can cross reference dice roll seed phrases with software that confirms it (only do this on a clean computer that is not connected to the internet) or a different cold storage device
+
+You can also cross reference paraphrases by trying to set up the seed phrase containing the passphrase on a different cold storage device and see if the whole phrase is needed to gain access 
