--- reddit-dice-wallets-at-risk@20260807T125232Z
+++ reddit-dice-wallets-at-risk@20260808T015534Z
@@ -644,3 +644,19 @@
 edited: false
 body:
 Do not trust your memory. 
+
+comment: p2bxff8
+parent: t1_p1lnjst
+author: KIFulgore
+created_utc: 1786132650
+edited: false
+body:
+Yeah, that is totally reasonable. But they shouldn't deploy test harness code with the production build. I have never included mock servers or unit test code with any production service I've deployed. If code should never be reachable in production, it shouldn't be there.
+
+comment: p2cdz39
+parent: t3_1veixnw
+author: phoneguy3
+created_utc: 1786137194
+edited: false
+body:
+I am in the same situation and I am still moving my Bitcoin. Safe for now is not safe enough. You never know when single sig could fail. Not right now. Not anytime soon. But in the age of AI, I can't rule out out. So I'm going multi-sig. Not waiting for another CC exploit that hasn't been found yet; not waiting for a single-sig vulnerability whatever that might be, whether or not I or anyone else can even imagine what a single-sig vulnerability would even look like. I'm multi-sigging. As soon as I receive the additional hardware.
