--- reddit-dice-wallets-at-risk@20260804T134031Z
+++ reddit-dice-wallets-at-risk@20260804T151124Z
@@ -551,3 +551,42 @@
 edited: false
 body:
 Makes sense 
+
+comment: p1nqn8f
+parent: t1_p1liqng
+author: didnt_hodl
+created_utc: 1785854431
+edited: false
+body:
+for TRNG there are certification levels, and unlike ColdCard everyone else is fully certified I believe. meaning that their random number generator was properly tested. search for EAL5+, AIS-31.
+
+  
+the issue with TRNG certification was raised earlier with CC and they dismissed it
+
+comment: p1nr15t
+parent: t1_p1lo69c
+author: didnt_hodl
+created_utc: 1785854532
+edited: false
+body:
+right. but if I have another device, why would not I just use that then? oh ... because CC is "safer"?? please
+
+
+
+comment: p1nwevo
+parent: t1_p1nr15t
+author: _gianlucag_
+created_utc: 1785855945
+edited: false
+body:
+well, if you used another device to generate the seed and imported it into the coldcard,  the coldcard becomes just a "transaction signing" device. Which can either produce a valid transaction or an invalid one that doesnt broadcast (if more software bugs are found).
+
+comment: p1nwxcm
+parent: t1_p1nwevo
+author: didnt_hodl
+created_utc: 1785856081
+edited: false
+body:
+exactly, using CC is simply adding new, extra bugs and vulnerabilities
+
+how do you know your CC is not leaking parts of your seed with every transaction for example
