--- reddit-hacker-still-progressing@20260807T061923Z
+++ reddit-hacker-still-progressing@20260807T125837Z
@@ -1606,14 +1606,6 @@
 body:
 "Presumably" is carrying a lot of weight there. The Ledger operation could just be waiting for a bigger payday - it's even more opaque than coldcard and could run the same type of scam with even less chance of discovery.
 
-comment: p1omi3i
-parent: t1_p1o4pbz
-author: trufin2038
-created_utc: 1785862570
-edited: false
-body:
-That would be the best possible outcome.
-
 comment: p1omk0c
 parent: t1_p1nmpmg
 author: Ok-Engineering1873
@@ -3854,14 +3846,6 @@
 body:
 How does anyone know this isn’t Cold Card doing it themselves
 
-comment: p20vga4
-parent: t1_p1pf9qd
-author: Far_Professor_8254
-created_utc: 1786006784
-edited: false
-body:
-But cancer treatment is covered by basic health insurance, so they only pay the €385 annual compulsory deductible, along with the ~€180 monthly premium. Plus, sick leave guarantees their income while ill: usually 100% in the first year and 70% in the second year (up to 2 years max).
-
 comment: p214wu8
 parent: t1_p1oeovv
 author: Logical_Breadfruit36
@@ -3870,14 +3854,6 @@
 body:
 making you're own seed with dices or coins gives a peace of mind + long passphrase. Search on youtube how to make you're own seed. It took me around 60 minutes to make my own. Fully random. 
 
-comment: p217wh7
-parent: t1_p20vga4
-author: ivanjurman
-created_utc: 1786012648
-edited: false
-body:
-Maybe they don’t have basic health insurance, and they don’t have paid sick leave because they’re unemployed
-
 comment: p21cb6i
 parent: t1_p1uibgi
 author: LittleLeeno
@@ -3976,6 +3952,30 @@
 body:
 Can already see the posts 5-10 years from now. 
 
+comment: p27srqe
+parent: t3_1vfagz9
+author: keekeerun
+created_utc: 1786084715
+edited: false
+body:
+Anyone with any model coldcard should move coins. You can't trust anything that company says. 
+
+comment: p2870y1
+parent: t1_p1qg52v
+author: Nice_Statistician539
+created_utc: 1786091618
+edited: false
+body:
+i thought this one was staged?
+
+comment: p291vpw
+parent: t1_p1nijcp
+author: Rare-Mention3046
+created_utc: 1786104781
+edited: false
+body:
+Completely off topic but no one actually listens to me when I say that one of the phone and SIM provider  mobile.co.uk actually did offer a 50 pound cashback on the pixel 10a with a SIM, but is now not offering it.  I'm not getting serious responses.  Well you know I'm only one voice.   So actually, why don't people listen is the question.  People don't randomly go to the internet to vent 
+
 more-stub: parent t1_p1pf9qd count <live-count>
 
 more-stub: parent t1_p1prsbb count <live-count>
@@ -4014,6 +4014,8 @@
 
 more-stub: parent t1_p1o2d4x count <live-count>
 
+more-stub: parent t1_p1o4pbz count <live-count>
+
 more-stub: parent t1_p1nz8zy count <live-count>
 
 more-stub: parent t1_p1nolqw count <live-count>
