--- reddit-hacker-still-progressing@20260805T004930Z
+++ reddit-hacker-still-progressing@20260805T011935Z
@@ -1305,9 +1305,10 @@
 parent: t1_p1nsbwe
 author: Blbe-Check-42069
 created_utc: 1785856422
-edited: false
+edited: 1785891424
 body:
 They might not even know it's happening. I only read about it here on Reddit 
+EDIT: and today I received warning email from trezor warning about this topic.
 
 comment: p1nyzgi
 parent: t3_1vfagz9
@@ -2722,18 +2723,6 @@
 body:
 Seriously tho, ColdCard should at least figure the same process and move all the compromised funds to the safe wallets, returning them to owners later. Clearly a lot of holders are unaware of the issue.
 
-comment: p1pgovm
-parent: t3_1vfagz9
-author: SeaMisx
-created_utc: 1785870063
-edited: false
-body:
-Coinkite will go down for this, they will be sued to hell and will have to pay back the loses, their insurance will have to do it there is just no other choices, there is no need to be Clarence Darrow to know that.
-
-If they can't pay or if their insurance won't pay they will go to jail for like more than 500 years combined. 
-
-And not just the CEO but every dev that knew that fault as it was already know in 2021, that is insane.
-
 comment: p1pgqoc
 parent: t1_p1ow6uv
 author: Consider8SpeedDemon
@@ -2832,14 +2821,6 @@
 body:
 There will no longer be a CoinKite after this event. 
 
-comment: p1pkl7h
-parent: t3_1vfagz9
-author: Just_Bluebird_5268
-created_utc: 1785871068
-edited: false
-body:
-It's the future of money
-
 comment: p1pkmp4
 parent: t1_p1op2i0
 author: FlipFlopFanatic
@@ -3018,14 +2999,6 @@
 body:
 You ever notice that right towards the end of a bear cycle, there’s a sudden attack or exchange collapse causing sudden FUD and price drop? Then major governments start buying at the bottom… Maybe all anecdotal but interesting timing nonetheless. 
 
-comment: p1pq8d6
-parent: t3_1vfagz9
-author: PI-E0423
-created_utc: 1785872546
-edited: false
-body:
-It will destroy trust in Bitcoin. No matter if waranted
-
 comment: p1pqs5s
 parent: t1_p1o5tj4
 author: Professional_Golf393
@@ -3282,14 +3255,6 @@
 body:
 Wasn't links I found. I ran accross a few posts where people posted old conversations. I didn't save any of them or share them. I should have. I think back then most figured those were just user errors. 
 
-comment: p1pztw1
-parent: t3_1vfagz9
-author: Dont_Be_Sheep
-created_utc: 1785875078
-edited: false
-body:
-SELL YOUR BITCOIN WHY ARE YOU HOLDING IT?? The experiment failed.
-
 comment: p1pzuas
 parent: t1_p1prfj2
 author: burusai
@@ -3320,7 +3285,7 @@
 created_utc: 1785875767
 edited: false
 body:
-Punitive punishment is not a deterrent. What you are asking for has been proven to not work.
+[removed]
 
 comment: p1q2xp8
 parent: t1_p1pwvdh
@@ -4131,6 +4096,48 @@
 body:
 i know for a fact in the case of Indonesia if you got an interpol red notice or simply ask their immigration agency they will most likely intercept and/or track and extract them for you if they’re not an indonesian citizen. there are so many cases of these, despite no formal treaty with the US. if it’s not a citizen it’s technically expulsion, not extradition.
 
+comment: p1rikw3
+parent: t1_p1r4tgg
+author: zackel_flac
+created_utc: 1785891238
+edited: false
+body:
+> My claim is LLMs can and are being used to develop software now, today
+
+We are in agreement. LLM and all form of AI have been and are still being used. No problem with that. AI exists since the 50s, it would be silly to say it's useless.
+
+The claim of OP was that LLM would have found and fixed the issue on its own before any harm would have been done to customers.
+
+My claim is that you need a deeper knowledge about the whole product and a bug in a code is only the beginning of it. So no, it's not that easy.
+
+That's it. Regarding Mythos it has been debunked multiple times. Beware of marketing, AI companies are not out there for doing good but making cash.
+
+But let's make a deal right there. Since Mythos analyzed Firefox fully already, we should not uncover any new zero-days going forward on those old code paths, right? If we do, then we will know of Mythos was that useful or not.
+
+comment: p1rkskk
+parent: t3_1vfagz9
+author: Silent-Currency8863
+created_utc: 1785891957
+edited: false
+body:
+Anyone know how much has been stolen as of today? 
+
+comment: p1rmh8e
+parent: t1_p1o2d4x
+author: the_Elders
+created_utc: 1785892509
+edited: false
+body:
+I know we hope this person will get caught but it is funny to me people want bitcoin to become the dominant currency while simultaneously thinking this hacker has to cash out. Maybe he/she just wants bitcoin. Maybe they accidentally die in a car accident tomorrow and the coins sit there forever lost.
+
+comment: p1rmtjn
+parent: t3_1vfagz9
+author: ArtesianShiny
+created_utc: 1785892621
+edited: false
+body:
+Satoshi would want the bitcoin miners and blockchain developers to signal for the return of the coins, like banks can reverse fraudulent transactions. Im just not sure how we would implement that yet.
+
 more-stub: parent t1_p1o3rfu count 2
 
 more-stub: parent t1_p1o590j count 0
@@ -4159,8 +4166,8 @@
 
 more-stub: parent t1_p1nflwp count 1
 
-more-stub: parent t1_p1o95fd count 3
+more-stub: parent t1_p1o95fd count 4
 
 more-stub: parent t1_p1nocjn count 8
 
-more-stub: parent t3_1vfagz9 count 13
+more-stub: parent t3_1vfagz9 count 17
