--- reddit-hacker-still-progressing@20260808T213538Z
+++ reddit-hacker-still-progressing@20260809T165502Z
@@ -2724,14 +2724,6 @@
 
 WHY WOULD A THEIF DO THIS? I genuinely don’t understand.
 
-comment: p1qmti4
-parent: t1_p1pw9rz
-author: stevethewatcher
-created_utc: 1785881356
-edited: false
-body:
-Way to conflate the two lol. There are other countries beside the US you know, who use fiat just fine without starting wars or destroying environments (what do those have to do with the dollar value anyways). Btw, in order for the dollar to be worth 50% less inflation would have to be at 15%, which is not even close to reality
-
 comment: p1qnfzo
 parent: t3_1vfagz9
 author: Shwazool
@@ -2868,17 +2860,6 @@
 body:
 Dude is probably scanning through all serial numbers. 
 
-comment: p1qxikg
-parent: t1_p1pw9rz
-author: Technical_Customer_1
-created_utc: 1785884548
-edited: false
-body:
-“
- Don’t forget all that destruction to the environment and wildlife!” 
-
-Says the person who prefers money that uses gigawatt after gigawatt of juice…
-
 comment: p1qycfp
 parent: t3_1vfagz9
 author: Patrick_Atsushi
@@ -3924,6 +3905,22 @@
 body:
 If only the developers knew to use AI tools it could have been different. 
 
+comment: p2msepx
+parent: t3_1vfagz9
+author: Sebhas
+created_utc: 1786277553
+edited: false
+body:
+I genuinly dont understand this. So everyone knows which adresses that are linked to the hack? Which means you know who the hacker is. Fucking freeze the adresses and return the funds. Wth is going on?
+
+comment: p2o655g
+parent: t3_1vfagz9
+author: Only-Wonder-2610
+created_utc: 1786292898
+edited: false
+body:
+Dumb question how does one convert stolen bitcoin to cash without getting caught?
+
 more-stub: parent t1_p1pf9qd count <live-count>
 
 more-stub: parent t1_p1pjrgq count <live-count>
