--- stackernews-drains-since-2022@20260802T155530Z
+++ stackernews-drains-since-2022@20260802T200251Z
@@ -1,8 +1,8 @@
 pull down to refresh
 
 ColdCard users have allegedly been reporting drains since 2022
-2297 sats \ 21 comments \ @justin_shocknet
- 1h bitcoin -5000 sats
+2398 sats \ 21 comments \ @justin_shocknet
+ 5h bitcoin -5000 sats
 
 https://m.stacker.news/150556
 
@@ -23,12 +23,12 @@
 speak now and forever hold your keys
 reply 10 sats
 related posts
-1681 sats
+1985 sats
 LIT
 NEW
 TOP
 1 sat \ 9 replies \ @tomlaies
- 1h
+ 5h
 Raises an interesting question, if they did know, and wanted to patch it
 
 What I find particularly interesting here is who did these drains <relative-time>?
@@ -36,24 +36,24 @@
 Did an attacker know of this vulnerability and only slowly exploited it?
 Or did Coldcard users create new wallets and found somebody elses wallet?
 reply
-11 sats \ 8 replies \ @justin_shocknet
- OP 1h -420 sats
-103 sats \ 2 replies \ @optimism 1h
+254 sats \ 8 replies \ @justin_shocknet
+ OP 5h -420 sats
+103 sats \ 2 replies \ @optimism 5h
 how could they release a patch that didn't automatically highlight the issue for attack?
 
 Same way Core does it. Hide it in a refactor.
 
 reply
 126 sats \ 1 reply \ @justin_shocknet
- OP 1h -420 sats
+ OP 5h -420 sats
 1 sat \ 0 replies \ @nitter
- 1h
+ 5h
 
 https://twiiit.com/Zenul_Abidin/status/2083756420843839872
 
 reply
 18 sats \ 4 replies \ @Undisciplined
- 1h
+ 5h
 
 I was talking to @optimism about someone potentially trying to replicate the attack for the purpose of safeguarding vulnerable funds and trying to return them later.
 
@@ -61,9 +61,9 @@
 
 reply
 70 sats \ 3 replies \ @justin_shocknet
- OP 1h -420 sats
+ OP 5h -420 sats
 1 sat \ 1 reply \ @Scoresby
- 1h
+ 5h
 
 I've been looking through a number of these this morning. I'm not sure that it's very convincing that the firmware was the problem -- it is also possible users screwed up.
 
@@ -71,4 +71,4 @@
 
 reply
 103 sats \ 0 replies \ @justin_shocknet
- OP 1h -420 sats
+ OP 5h -420 sats
