--- bitcointalk-entropy-warning-design@20260809T232546Z
+++ bitcointalk-entropy-warning-design@20260812T043951Z
@@ -198,7 +198,7 @@
 If you want to have such insightful technical warnings, you can make your proposal to this site team https://walletscrutiny.com/. Ask them to add security check on wallets related to seed entropy and add it to their wallet reviews.
 Title: Re: Should wallets warn users about low-quality entropy during SEED generation?
 Post by: nc50lc on August 09, 2026, 06:15:02 AM
-Quote from: hd49728 on Today at 03:30:34 AM
+Quote from: hd49728 on August 09, 2026, 03:30:34 AM
 If you want to have such insightful technical warnings, you can make your proposal to this site team https://walletscrutiny.com/. Ask them to add security check on wallets related to seed entropy and add it to their wallet reviews.
 I doubt that it'll be 100% reliable.
 For example, the recent Coldcard MK4 warnings (https://walletscrutiny.com/hardware/coldcardMk4/) in their websites are only added after the hack while the issue was present way before their initial code audit.
