--- reddit-trezor-user-alert-email@20260806T163304Z
+++ reddit-trezor-user-alert-email@20260807T061645Z
@@ -468,3 +468,19 @@
 edited: false
 body:
 Use a passphrase or just buy the ETF… This is 2026 not 2016. 
+
+comment: p23tgph
+parent: t1_p1krtag
+author: FirmRain6748
+created_utc: 1786039064
+edited: false
+body:
+They put a massive multi post thing about it on their X basically the day it happened 
+
+comment: p2778s6
+parent: t1_p1k3kr4
+author: 7thlttd
+created_utc: 1786075389
+edited: false
+body:
+Please elaborate for us ? Very courious to hear about this 
