--- reddit-trezor-user-alert-email@20260807T061645Z
+++ reddit-trezor-user-alert-email@20260807T192848Z
@@ -484,3 +484,29 @@
 edited: false
 body:
 Please elaborate for us ? Very courious to hear about this 
+
+comment: p2a5pwe
+parent: t1_p2778s6
+author: Btcmot
+created_utc: 1786116180
+edited: false
+body:
+Over a year ago, Trezor did an update to their firmware that included what is equal to a back door to their software and your Trezor. They had some bs explanation why but I did not accept the update and as fast as I could picked a different wallet and moved away from Trezor.  Im not a techy but  i can’t afford to trust a company that wants the ability to get into my wallet stash. 
+
+And because you will ask, i moved to Passport.  Its not the most convenient but it is a Bitcoin only wallet ( my first rule) and its had a great reputation so far. 
+
+comment: p2a68yp
+parent: t1_p2a5pwe
+author: 7thlttd
+created_utc: 1786116315
+edited: false
+body:
+Thanks for the reply man appreciate it . That def sounds concerning for sure 
+
+comment: p2a7keu
+parent: t1_p2a68yp
+author: Btcmot
+created_utc: 1786116648
+edited: false
+body:
+Sorry i cant explain the specific change, but its worth not using them.
