--- reddit-trezor-user-alert-email@20260807T192848Z
+++ reddit-trezor-user-alert-email@20260809T165140Z
@@ -101,11 +101,11 @@
 
 comment: p1j7ryj
 parent: t1_p1izh9g
-author: 0fWhomIAmChief
+author: [deleted]
 created_utc: 1785794106
 edited: false
 body:
-Coldcard had 0 entropy redundancy, Trezors have 3 or 4 sources of entropy redundancy 
+[removed]
 
 comment: p1ja82c
 parent: t1_p1izlz0
@@ -193,14 +193,6 @@
 
 In the BTC world, the random variable Z generates a seed phrase, and it must have high entropy to not be reverse engineered.
 
-comment: p1jo9hp
-parent: t1_p1jd3gs
-author: 0fWhomIAmChief
-created_utc: 1785799081
-edited: false
-body:
-https://trezor.io/guides/trezor-devices/trezor-fundamentals/what-is-entropy-and-how-does-trezor-generate-your-wallet
-
 comment: p1joz0x
 parent: t1_p1j7ryj
 author: cilicia3k3
@@ -235,11 +227,11 @@
 
 comment: p1jyq0q
 parent: t1_p1joz0x
-author: 0fWhomIAmChief
+author: [deleted]
 created_utc: 1785802418
 edited: false
 body:
-https://trezor.io/guides/trezor-devices/trezor-fundamentals/what-is-entropy-and-how-does-trezor-generate-your-wallet
+[removed]
 
 comment: p1jyvzz
 parent: t1_p1jyq0q
