--- reddit-trezor-user-alert-email@20260804T214606Z
+++ reddit-trezor-user-alert-email@20260804T221628Z
@@ -456,3 +456,11 @@
 edited: false
 body:
 Yes, but you can sign up for their newsletter, which is separate from the email database used for purchase.
+
+comment: p1qi1qh
+parent: t3_1vepzf1
+author: Crypta_Silva
+created_utc: 1785880005
+edited: false
+body:
+Ha! I got 3 or 4 scam "Trezor" emails before I got the real one. Those scammers are persistent and annoying. 
