--- reddit-shipments-halted-relay@20260806T163011Z
+++ reddit-shipments-halted-relay@20260807T061356Z
@@ -336,3 +336,19 @@
 edited: false
 body:
 coldcard is junk.. ya should of bought Trezor
+
+comment: p23yp5x
+parent: t3_1vdpnza
+author: Knowledge775
+created_utc: 1786040370
+edited: false
+body:
+So the fact that they destroyed inventory instead of doing a firmware update on site makes me want a refund on my coldcard inventory.
+
+comment: p26go41
+parent: t3_1vdpnza
+author: AntZealousideal3728
+created_utc: 1786066017
+edited: false
+body:
+This should be taken down quite honestly considering they didn’t destroy remaining inventory like it says but affected devices. I cancelled my order and received 2 today in the mail even though I got my money back. 
