--- reddit-shipments-halted-relay@20260808T145934Z
+++ reddit-shipments-halted-relay@20260809T231910Z
@@ -394,3 +394,11 @@
 No. Its pure economics. A+B > C
 
 Shipping + patching costs more that producing new. Cheaper to destroy the compromised stock and produce new.
+
+comment: p2ps7a5
+parent: t1_p1dfoz0
+author: uniicorn77
+created_utc: 1786309473
+edited: false
+body:
+yes i heard people talking so much shit about nvk, and blaming him for not putting in the required attention to this 
