--- reddit-shipments-halted-relay@20260807T061356Z
+++ reddit-shipments-halted-relay@20260808T145934Z
@@ -352,3 +352,45 @@
 edited: false
 body:
 This should be taken down quite honestly considering they didn’t destroy remaining inventory like it says but affected devices. I cancelled my order and received 2 today in the mail even though I got my money back. 
+
+comment: p2fqu6b
+parent: t1_p1cr9vq
+author: Skinny_Human
+created_utc: 1786185116
+edited: false
+body:
+It’s called ‘rats deserting a sinking ship’ and covering their asses on the way out 
+
+comment: p2fqx9g
+parent: t1_p1cnrsq
+author: Skinny_Human
+created_utc: 1786185157
+edited: false
+body:
+Is it me or does it sound like you’re defending CoinKite?!!
+
+comment: p2fr3qa
+parent: t1_p2fqx9g
+author: zootreddit
+created_utc: 1786185243
+edited: false
+body:
+Its you. 
+
+comment: p2fr65k
+parent: t1_p2fr3qa
+author: Skinny_Human
+created_utc: 1786185275
+edited: false
+body:
+*kicks dirt*
+
+comment: p2gtw4s
+parent: t1_p2fqu6b
+author: Charming-Designer944
+created_utc: 1786199559
+edited: false
+body:
+No. Its pure economics. A+B > C
+
+Shipping + patching costs more that producing new. Cheaper to destroy the compromised stock and produce new.
