--- reddit-sun-will-rise@20260807T192945Z
+++ reddit-sun-will-rise@20260808T020002Z
@@ -787,3 +787,13 @@
 edited: false
 body:
 Bitcoin people are funny lol rich vs poor when everyone has the same aim. Hopefully it drops to $5 per btc and it'll equalize it for everyone. Both rich and poor. Or to 0, even better
+
+comment: p2chxn1
+parent: t3_1vdvtmu
+author: Salty-Tough-3721
+created_utc: 1786138345
+edited: false
+body:
+I once lost a 3 dollar bet to my friend, that was the first and last time I invested in something I thought I believed in. 
+Since then I only donate money straight away, feel better than losing everything on stocks and bad decisions. 
+This way someone else gets to enjoy the money.
