--- reddit-sun-will-rise@20260806T100156Z
+++ reddit-sun-will-rise@20260806T163422Z
@@ -655,14 +655,6 @@
 body:
 Cornball
 
-comment: p1ij3l2
-parent: t1_p1eb7pr
-author: CraftyBrother6974
-created_utc: 1785787433
-edited: false
-body:
-Just put the fries in the bag 
-
 comment: p1j5mna
 parent: t1_p1et857
 author: Decent_Okra_3521
@@ -771,3 +763,11 @@
 edited: false
 body:
 This was the case with Coldcard. But if randomness is generated correctly, there is no way to find "used seed". You don't comprehend the vastness of this seed space. Chances are literaly astronomical.
+
+comment: p21k4el
+parent: t3_1vdvtmu
+author: basedisciple
+created_utc: 1786017365
+edited: false
+body:
+Hate these posts. This does nothing. Life savings versus your cushion of loss porn doesn’t compare.
