--- stackernews-troll-1367btc@20260804T114137Z
+++ stackernews-troll-1367btc@20260804T124152Z
@@ -15,6 +15,13 @@
             "text": "Bob Burnett has a suggestion:\n\n![](https://m.stacker.news/150724)\n\n[source](https://x.com/boomer_btc/status/2084424317505249557)",
             "user": {
               "name": "Scoresby"
+            }
+          },
+          {
+            "createdAt": "2026-08-03T23:55:31.003Z",
+            "text": "Zap it to me and I will send you cowboy credits in return.",
+            "user": {
+              "name": "grayruby"
             }
           },
           {
@@ -36,13 +43,6 @@
             "text": "Wasabi? I don't know if they can do that much volume tho. @kruw?",
             "user": {
               "name": "wackster"
-            }
-          },
-          {
-            "createdAt": "2026-08-03T23:55:31.003Z",
-            "text": "Zap it to me and I will send you cowboy credits in return.",
-            "user": {
-              "name": "grayruby"
             }
           },
           {
