--- stackernews-troll-1367btc@20260804T101220Z
+++ stackernews-troll-1367btc@20260804T114137Z
@@ -25,6 +25,13 @@
             }
           },
           {
+            "createdAt": "2026-08-03T23:11:24.948Z",
+            "text": "I'm new here, but I believe that:\n\n\"NICE TRY FED\"\n \n... is the customary response to such questions. Do i get a biscuit?",
+            "user": {
+              "name": "anon"
+            }
+          },
+          {
             "createdAt": "2026-08-03T23:13:53.727Z",
             "text": "Wasabi? I don't know if they can do that much volume tho. @kruw?",
             "user": {
@@ -32,10 +39,10 @@
             }
           },
           {
-            "createdAt": "2026-08-03T23:11:24.948Z",
-            "text": "I'm new here, but I believe that:\n\n\"NICE TRY FED\"\n \n... is the customary response to such questions. Do i get a biscuit?",
+            "createdAt": "2026-08-03T23:55:31.003Z",
+            "text": "Zap it to me and I will send you cowboy credits in return.",
             "user": {
-              "name": "anon"
+              "name": "grayruby"
             }
           },
           {
@@ -46,10 +53,17 @@
             }
           },
           {
-            "createdAt": "2026-08-03T23:55:31.003Z",
-            "text": "Zap it to me and I will send you cowboy credits in return.",
+            "createdAt": "2026-08-04T11:27:19.982Z",
+            "text": "Birthday present 27 days late!",
             "user": {
-              "name": "grayruby"
+              "name": "teemupleb"
+            }
+          },
+          {
+            "createdAt": "2026-08-04T01:44:28.504Z",
+            "text": "NICE TRY FBI",
+            "user": {
+              "name": "magnolia_mayhem"
             }
           },
           {
@@ -57,13 +71,6 @@
             "text": "Coinjoins\n\nObviously",
             "user": {
               "name": "kilianbuhn"
-            }
-          },
-          {
-            "createdAt": "2026-08-04T01:44:28.504Z",
-            "text": "NICE TRY FBI",
-            "user": {
-              "name": "magnolia_mayhem"
             }
           },
           {
