--- stackernews-troll-1367btc@20260804T192313Z
+++ stackernews-troll-1367btc@20260806T101203Z
@@ -3,6 +3,13 @@
     "item": {
       "comments": {
         "comments": [
+          {
+            "createdAt": "2026-08-04T11:27:19.982Z",
+            "text": "Birthday present 27 days late!",
+            "user": {
+              "name": "teemupleb"
+            }
+          },
           {
             "createdAt": "2026-08-04T00:19:24.580Z",
             "text": "Seeing as you are pretty rich having won one of the biggest bounties ever in bitcon, I suggest you open a lightning channel to SN and use the bounty feature:\n![](https://m.stacker.news/150731)\n\nP.s. It wasn't given to you. You stole, so you should probably just give it back. Save your soul.",
@@ -50,13 +57,6 @@
             "text": "Put it in Mt. Gox and watch your investment grow!!!",
             "user": {
               "name": "magnolia_mayhem"
-            }
-          },
-          {
-            "createdAt": "2026-08-04T11:27:19.982Z",
-            "text": "Birthday present 27 days late!",
-            "user": {
-              "name": "teemupleb"
             }
           },
           {
