--- stackernews-troll-1367btc@20260804T041725Z
+++ stackernews-troll-1367btc@20260804T101220Z
@@ -4,10 +4,31 @@
       "comments": {
         "comments": [
           {
+            "createdAt": "2026-08-04T00:19:24.580Z",
+            "text": "Seeing as you are pretty rich having won one of the biggest bounties ever in bitcon, I suggest you open a lightning channel to SN and use the bounty feature:\n![](https://m.stacker.news/150731)\n\nP.s. It wasn't given to you. You stole, so you should probably just give it back. Save your soul.",
+            "user": {
+              "name": "billytheked"
+            }
+          },
+          {
+            "createdAt": "2026-08-03T23:51:21.455Z",
+            "text": "Bob Burnett has a suggestion:\n\n![](https://m.stacker.news/150724)\n\n[source](https://x.com/boomer_btc/status/2084424317505249557)",
+            "user": {
+              "name": "Scoresby"
+            }
+          },
+          {
             "createdAt": "2026-08-03T23:07:03.549Z",
             "text": "lol.. yeah, let's break SN's lightning channels",
             "user": {
               "name": "denlillaapan"
+            }
+          },
+          {
+            "createdAt": "2026-08-03T23:13:53.727Z",
+            "text": "Wasabi? I don't know if they can do that much volume tho. @kruw?",
+            "user": {
+              "name": "wackster"
             }
           },
           {
@@ -25,13 +46,6 @@
             }
           },
           {
-            "createdAt": "2026-08-03T23:51:21.455Z",
-            "text": "Bob Burnett has a suggestion:\n\n![](https://m.stacker.news/150724)\n\n[source](https://x.com/boomer_btc/status/2084424317505249557)",
-            "user": {
-              "name": "Scoresby"
-            }
-          },
-          {
             "createdAt": "2026-08-03T23:55:31.003Z",
             "text": "Zap it to me and I will send you cowboy credits in return.",
             "user": {
@@ -39,10 +53,10 @@
             }
           },
           {
-            "createdAt": "2026-08-03T23:13:53.727Z",
-            "text": "Wasabi? I don't know if they can do that much volume tho. @kruw?",
+            "createdAt": "2026-08-04T06:52:40.333Z",
+            "text": "Coinjoins\n\nObviously",
             "user": {
-              "name": "wackster"
+              "name": "kilianbuhn"
             }
           },
           {
@@ -74,13 +88,6 @@
             }
           },
           {
-            "createdAt": "2026-08-04T00:19:24.580Z",
-            "text": "Seeing as you are pretty rich having won one of the biggest bounties ever in bitcon, I suggest you open a lightning channel to SN and use the bounty feature:\n![](https://m.stacker.news/150731)\n\nP.s. It wasn't given to you. You stole, so you should probably just give it back. Save your soul.",
-            "user": {
-              "name": "billytheked"
-            }
-          },
-          {
             "createdAt": "2026-08-03T23:47:09.259Z",
             "text": "Use https://boltz.exchange/ ?",
             "user": {
