--- libngu-pr-58@20260801T081150Z
+++ libngu-pr-58@20260804T160834Z
@@ -1,8 +1,6 @@
 Skip to content
 Navigation Menu
-Toggle navigation
-Sign in
-Appearance settings
+Sign inAppearance settings
 Platform
 AI CODE CREATION
 GitHub CopilotWrite better code with AI
@@ -85,27 +83,9 @@
 Copilot for BusinessEnterprise-grade AI features
 Premium SupportEnterprise-grade 24/7 support
 Pricing
-Search or jump to...
-Search code, repositories, users, issues, pull requests...
-Search
-Clear
-Search syntax tips
-Provide feedback
-We read every piece of feedback, and take your input very seriously.
-Include my email address so I can be contacted
-Cancel
-Submit feedback
-Saved searches
-Use saved searches to filter your results more quickly
-Name
-Query
-To see all available qualifiers, see our documentation.
-Cancel
-Create saved search
+Type / to search
 Sign in
-Sign up
-Appearance settings
-Resetting focus
+Sign upAppearance settings
 You signed in with another tab or window. Reload to refresh your session.
 You signed out in another tab or window. Reload to refresh your session.
 You switched accounts on another tab or window. Reload to refresh your session.
