--- libngu-pr-58@20260801T035017Z
+++ libngu-pr-58@20260801T081150Z
@@ -163,11 +163,11 @@
 Uh oh!
 There was an error while loading. Please reload this page.
 👍
-1
-NickeZ reacted with thumbs up emoji
+2
+NickeZ and sutterseba reacted with thumbs up emoji
 All reactions
 👍
-1 reaction
+2 reactions
 Enforce HW TRNG
 fa09082
 This file contains hidden or bidirectional Unicode text that may be interpreted or compiled differently than what appears below. To review, open the file in an editor that reveals hidden Unicode characters.
