--- reddit-nvk-proxy-marketing-claim@20260813T015632Z
+++ reddit-nvk-proxy-marketing-claim@20260814T234705Z
@@ -24,6 +24,9 @@
 So, it was Rodolfo Novak recommending himself the Coldcard....
 
 IT IS TRULLY FUCKED UP the thing....And many of that "reputation" was build in a very deceptive way
+
+_______
+Update: website now gives 404. Rodolfo Novak is removing what it seems deceptive endorsement. 
 
 comment: p2ypf8n
 parent: t3_1vl4nly
