--- coldcard-firmware-pr-691@20260801T062115Z
+++ coldcard-firmware-pr-691@20260801T073440Z
@@ -195,6 +195,32 @@
 Thirty-two silent bytes,
 all but four were cast aside —
 now the seed stays whole.
+scgbckbone
+reviewed
+Aug 1, 2026
+View reviewed changes
+scgbckbone
+left a comment
+Copy link
+Copy Markdown
+Collaborator
+There was a problem hiding this comment.
+Choose a reason for hiding this comment
+The reason will be displayed to describe this comment to others. Learn more.
+Choose a reason
+Spam
+Abuse
+Off Topic
+Outdated
+Duplicate
+Resolved
+Low Quality
+Hide comment
+Can you please open libngu changes as separate PR against libngu repo? thanks
+Sorry, something went wrong.
+Uh oh!
+There was an error while loading. Please reload this page.
+All reactions
 This file contains hidden or bidirectional Unicode text that may be interpreted or compiled differently than what appears below. To review, open the file in an editor that reveals hidden Unicode characters.
 Learn more about bidirectional Unicode characters
 Show hidden characters
@@ -203,7 +229,10 @@
 Already have an account?
 Sign in to comment
 Reviewers
-No reviews
+1 more reviewer
+scgbckbone
+scgbckbone left review comments
+Reviewers whose approvals may not affect merge requirements
 Assignees
 No one assigned
 Labels
@@ -216,7 +245,7 @@
 Successfully merging this pull request may close these issues.
 Uh oh!
 There was an error while loading. Please reload this page.
-1 participant
+2 participants
 Add this suggestion to a batch that can be applied as a single commit.This suggestion is invalid because no changes were made to the code.Suggestions cannot be applied while the pull request is closed.Suggestions cannot be applied while viewing a subset of changes.Only one suggestion per line can be applied in a batch.Add this suggestion to a batch that can be applied as a single commit.Applying suggestions on deleted lines is not supported.You must change the existing code in this line in order to create a valid suggestion.Outdated suggestions cannot be applied.This suggestion has been applied or marked resolved.Suggestions cannot be applied from pending reviews.Suggestions cannot be applied on multi-line comments.Suggestions cannot be applied while the pull request is queued to merge.Suggestion cannot be applied right now. Please check back later.
 Footer
 © 2026 GitHub, Inc.
