--- stackernews-peerswap-disable@20260804T041721Z
+++ stackernews-peerswap-disable@20260804T101217Z
@@ -3,6 +3,13 @@
     "item": {
       "comments": {
         "comments": [
+          {
+            "createdAt": "2026-08-04T08:36:58.988Z",
+            "text": "LOL",
+            "user": {
+              "name": "DarthCoin"
+            }
+          },
           {
             "createdAt": "2026-08-03T22:28:51.747Z",
             "text": "Was checking it out today and found something critical with L-BTC swaps\n\nDidn't go deep on it though because shitcoins \n\nhttps://stacker.news/items/1539276/r/justin_shocknet?commentId=1539362",
