--- coindesk-third-wave-89m@20260803T023109Z
+++ coindesk-third-wave-89m@20260803T064809Z
@@ -42,34 +42,34 @@
 But the sweeping has not stopped almost three days later, and the falling average haul says the profitable end of that key space is already picked over.
 Latest Crypto News
 1
+Bitcoin slips under $63,000 despite Iran deal hopes as Coldcard losses rattle market
+32 minutes ago
+2
+Trump Media’s bitcoin stash may be down to loan collateral after $165 million BTC move
+55 minutes ago
+3
+Michael Saylor’s Strategy is now tracking bitcoin’s 200-week moving average
+1 hour ago
+4
 Counting down the days: State of Crypto
-8 hours ago
-2
+12 hours ago
+5
 Why a DeFi platform ditched its consumer app to become the secret backend for tech giants
-9 hours ago
-3
+13 hours ago
+6
 The future of crypto payments won't include on-ramps or bridges, Fun CEO says
-10 hours ago
-4
+14 hours ago
+7
 The reverse bridge: Crypto meets Wall Street using perps
-13 hours ago
-5
+17 hours ago
+8
 Unlike the FTX collapse, the $89 million Coldcard exploit has investors sending bitcoin back to exchanges
-14 hours ago
-6
+18 hours ago
+9
 Strategy holds STRC dividend at 12%
 Aug 1, 2026
-7
+10
 Tokenized stock trading surged 288% in July, but one QQQ token drove most of it
-Aug 1, 2026
-8
-Bank of Italy research suggests stablecoins aren't necessarily cheaper for remittances
-Aug 1, 2026
-9
-SEC to review Nasdaq bitcoin options approval after CME challenge
-Aug 1, 2026
-10
-Solana Foundation's new CISO warns AI is making crypto scams more convincing
 Aug 1, 2026
 Latest Research
 The Evolution of the Crypto CEX Landscape: A Case Study on Binance
@@ -149,4 +149,4 @@
 EthicsPrivacyTerms of UseCookie SettingsDo Not Sell My InfoBug Bounty Program
 © 2026 CoinDesk, Inc.
 Disclosure & Polices: CoinDesk is an award-winning media outlet that covers the cryptocurrency industry. Its journalists abide by a strict set of editorial policies. CoinDesk has adopted a set of principles aimed at ensuring the integrity, editorial independence and freedom from bias of its publications. CoinDesk is part of Bullish (NYSE:BLSH), an institutionally focused global digital asset platform that provides market infrastructure and information services. Bullish owns and invests in digital asset businesses and digital assets and CoinDesk employees, including journalists, may receive Bullish equity-based compensation.
-CryptoCD20$1,733.49CD20 up 0.040 percent0.040%BTC$63,084.06BTC down 0.48 percent0.48%ETH$1,864.90ETH down 0.54 percent0.54%XRP$1.07XRP down 0.17 percent0.17%SOL$72.93SOL down 0.23 percent0.23%
+CryptoCD20$1,723.56CD20 down 1.07 percent1.07%BTC$62,589.53BTC down 1.33 percent1.33%ETH$1,847.98ETH down 1.50 percent1.50%XRP$1.07XRP down 1.23 percent1.23%SOL$72.50SOL down 1.23 percent1.23%
