--- reddit-nvk-awareness-critique@20260804T172538Z
+++ reddit-nvk-awareness-critique@20260804T182518Z
@@ -258,3 +258,11 @@
 edited: false
 body:
 How do you actually know that’s true though, other than some guy said it? For example, this coldcard thing could just as easily be malice as it could be incompetence. I don’t think prioritising incompetence by default is accurate, all the time.
+
+comment: p1p2h3d
+parent: t1_p1opq1u
+author: ZachMartin
+created_utc: 1785866509
+edited: false
+body:
+It was mostly tongue in cheek ;) you may be right. Let’s see where the investigation goes.
