--- coldcard-firmware-pr-707@20260807T154535Z
+++ coldcard-firmware-pr-707@20260807T192226Z
@@ -118,12 +118,12 @@
 Insights
 Improve seed gen & require external entropy for each new wallet - #707
 #707
-Open
+Closed
 scgbckbone wants to merge 7 commits into
 Coldcard:masterColdcard/firmware:masterfrom
 scgbckbone:improve_seed_gen-publicscgbckbone/firmware:improve_seed_gen-publicCopy head branch name to clipboard
 ConversationCommits7 (7)ChecksFiles changed
-Open
+Closed
 Improve seed gen & require external entropy for each new wallet#707
 scgbckbone wants to merge 7 commits into
 Coldcard:masterColdcard/firmware:masterfrom
@@ -227,6 +227,27 @@
 Sorry, something went wrong.
 Uh oh!
 There was an error while loading. Please reload this page.
+scgbckbone
+mentioned this pull request
+Aug 7, 2026
+Improve seed generation - mash timing with microsecond-resolution
+#713
+Open
+scgbckbone
+commented
+Aug 7, 2026
+Copy link
+Copy Markdown
+Collaborator
+Author
+replaced by strictly better #713
+All reactions
+Sorry, something went wrong.
+Uh oh!
+There was an error while loading. Please reload this page.
+scgbckbone
+closed this
+Aug 7, 2026
 This file contains hidden or bidirectional Unicode text that may be interpreted or compiled differently than what appears below. To review, open the file in an editor that reveals hidden Unicode characters.
 Learn more about bidirectional Unicode characters
 Show hidden characters
