--- reddit-full-panic-drain-report@20260809T025641Z
+++ reddit-full-panic-drain-report@20260812T000539Z
@@ -1709,11 +1709,11 @@
 
 comment: p0pbvft
 parent: t1_p0ohdwk
-author: Beneficial_Past_1957
+author: [deleted]
 created_utc: 1785430395
 edited: false
 body:
-Malone found himself 
+[removed]
 
 comment: p0pc30t
 parent: t1_p0pbvft
@@ -1833,14 +1833,6 @@
 body:
 There has to be something somewhere? You must have clicked on something and opened your wallet 
 
-comment: p0phyuq
-parent: t3_1vatgl4
-author: rewiredrose
-created_utc: 1785431935
-edited: false
-body:
-a documentary I watched showed a lottery software guy used vulnerablities to win jackpot several times before he got caught. maybe someone knew the secrets if hardware wallet op used.
-
 comment: p0pi82x
 parent: t1_p0p9u5w
 author: s1ammage
@@ -2674,16 +2666,6 @@
 edited: false
 body:
 Is mij helaas ook overkomen door een valse link, 12 woorden ingevoerd en zo alles uit handen gegeven, zo vals van hun, en zo naïef van mij..
-
-comment: p0rdfln
-parent: t3_1vatgl4
-author: HayBailes69
-created_utc: 1785449625
-edited: false
-body:
-Of interest, new to bitcoin but why do people opt for wallets instead of keeping it in an exchange?
-
-I understand there’s better protection and in theory this shouldn’t happen, but this feels like a really big risk.
 
 comment: p0rdm5q
 parent: t1_p0r913j
