--- reddit-full-panic-drain-report@20260806T192547Z
+++ reddit-full-panic-drain-report@20260806T212615Z
@@ -3760,14 +3760,6 @@
 body:
 Remember when we talked about the risks of self custody? u/DJ\_Ben\_Frank
 
-comment: p0ywfg9
-parent: t3_1vatgl4
-author: Bmoses99
-created_utc: 1785538591
-edited: false
-body:
-sad to see this. i’ve heard this may have been part of broader theft 
-
 comment: p0yzrg3
 parent: t1_p0oqtqf
 author: Obvious_Present9780
