--- reddit-full-panic-drain-report@20260812T162419Z
+++ reddit-full-panic-drain-report@20260813T152330Z
@@ -2184,19 +2184,6 @@
 edited: false
 body:
 Damn I need to learn more about crypto and wallets
-
-comment: p0q3ets
-parent: t1_p0occls
-author: Important-Level6672
-created_utc: 1785437349
-edited: 1785437860
-body:
-BTC isn’t easy to steal yet the top 10 biggest financial thefts of all time are all crypto related. Keep in Mind crypto only been around a fraction of the time fiat has.  
-[https://www.visualcapitalist.com/sp/ranked-the-10-biggest-digital-heists-in01/](https://www.visualcapitalist.com/sp/ranked-the-10-biggest-digital-heists-in01/)
-
-A few millions stolen in one go used to be rare asf
-
-All this “it was the users fault” only goes a long way. BTC network it self is secure but it’s a major pain in the ass to secure your funds it 24/7 or not lose it
 
 comment: p0q476s
 parent: t1_p0p4m31
