--- reddit-full-panic-drain-report@20260814T062328Z
+++ reddit-full-panic-drain-report@20260814T132359Z
@@ -1475,6 +1475,14 @@
 edited: false
 body:
 The coldcard comes with a few safety measures to keep you from getting scammed. Did you ignore them ?
+
+comment: p0p6ih2
+parent: t1_p0omvua
+author: minimorsels
+created_utc: 1785429031
+edited: false
+body:
+I recommend trezor and setting up a hidden passphrase wallet
 
 comment: p0p6nla
 parent: t1_p0o8vw5
@@ -3839,15 +3847,6 @@
 body:
 how???
 
-comment: p12vdx8
-parent: t3_1vatgl4
-author: lemannequin
-created_utc: 1785595994
-edited: false
-body:
-If I were a victim, I'd aim to broadcast ASAP a timestamped, signed message (signed with the compromised PK) as a proof/evidence to claim legit ownership, before other bad actors would try to do the same (and they will try to, if there's a slighly chance for funds to be restored to victims).   
-I'd sign the message and paste it on a few platforms, like Github Gists, here in Reddit, Twitter. Maybe even to the Bitcoin blockchain itself, as a hash (or as arbitrary data within the limits of OP\_RETURN). 
-
 comment: p13ux08
 parent: t1_p0q7757
 author: Eastern-Try-6207
@@ -4047,6 +4046,8 @@
 
 more-stub: parent t1_p0o7000 count <live-count>
 
+more-stub: parent t1_p0p6ih2 count <live-count>
+
 more-stub: parent t1_p0omvua count <live-count>
 
 more-stub: parent t1_p0ohlvy count <live-count>
