--- reddit-full-panic-drain-report@20260807T172204Z
+++ reddit-full-panic-drain-report@20260807T182037Z
@@ -3532,14 +3532,6 @@
 
 x. Reconsider using hardware wallets. It appears this was vulnerable keygen.
 
-comment: p0u5j24
-parent: t3_1vatgl4
-author: [deleted]
-created_utc: 1785487451
-edited: false
-body:
-Yeah stay away from these scam coins. Invest in the stock market 
-
 comment: p0ueul2
 parent: t3_1vatgl4
 author: Happy-Suit-3362
