--- reddit-full-panic-drain-report@20260814T132359Z
+++ reddit-full-panic-drain-report@20260815T072348Z
@@ -395,12 +395,11 @@
 
 comment: p0o6p3l
 parent: t1_p0o3npz
-author: SharpGame83
+author: [deleted]
 created_utc: 1785419667
 edited: false
 body:
-Honestly I get this now, when I first heard about bitcoin etfs I wondered who would prefer this over actual bitcoin? Old people?  
-Now after getting half a btc stolen from my blockchain wallet I’m like , ahhh ya I get it now
+[removed]
 
 comment: p0o6p74
 parent: t1_p0o3tag
