--- reddit-full-panic-drain-report@20260812T000539Z
+++ reddit-full-panic-drain-report@20260812T162419Z
@@ -3470,21 +3470,13 @@
 body:
 It's crazy how many hacks there have been on self custody solutions the last few years. Is there anyone keeping a tally on how much in USD (at time of theft / fraud) and how much BTC has been stolen from self custody solutions vs exchanges? I'm feeling it's getting close between them, at least USD wise. 
 
-comment: p0tz4qb
-parent: t3_1vatgl4
-author: Minute-Literature-22
-created_utc: 1785484331
-edited: false
-body:
-Really sorry for this. Hope you can recover your money.
-
 comment: p0tzrf7
 parent: t1_p0ohyzy
-author: Minute-Literature-22
+author: [deleted]
 created_utc: 1785484639
 edited: false
 body:
-This is the way. People should invest more much time in learning about btc safety.
+[removed]
 
 comment: p0u3kic
 parent: t1_p0occls
