--- reddit-full-panic-drain-report@20260813T152330Z
+++ reddit-full-panic-drain-report@20260814T062328Z
@@ -2739,14 +2739,6 @@
 edited: false
 body:
 For bitcoin to become mainstream, people will have wallets where they dont own the private keys.
-
-comment: p0rj5gs
-parent: t3_1vatgl4
-author: mindylynx
-created_utc: 1785451349
-edited: false
-body:
-aaand that's why i still use paper wallets 
 
 comment: p0rj8rb
 parent: t3_1vatgl4
