--- reddit-full-panic-drain-report@20260815T072348Z
+++ reddit-full-panic-drain-report@20260815T142328Z
@@ -1981,16 +1981,6 @@
 It could be that someone bought a lot of devices, set them up with his own seedphrase and  then sold the devices as "used" and maybe some people who bought the devices did not reset them - so they used the already existing wallet of the scammer.
 
 There are many way how someone can fail, but normally if you buy a good hardware wallet directly from the company that builds it, you should be safe when you use the wallet as intended and create a fresh seedphrase & wallet if you use it the first time.
-
-comment: p0poz41
-parent: t1_p0occls
-author: Aflockofants
-created_utc: 1785433688
-edited: false
-body:
-Yet when Bitcoin defenders talk about Bitcoin, it’s almost always mixed with some bullshit about how it will give poor people a chance, all kinds of initiatives in the developing world, yadayadayada. Just admit that by now it’s a get-rich-quick scheme for people in the Western world that have some money surplus.
-
-I’ve been a software developer for over 30 years and even mined some Bitcoin with my own PC back when that was still viable. I’m fully aware of how most people won’t be able to fully secure their own wallets. You’re never gonna be able to defend this idealistic story on how it will change the world and will protect us from tyrannical governments, while at the same time claiming people are morons for not being able to secure their own wallets. I took off when I saw that all the idealistic nonsense was just that: nonsense. To justify this whole thing where a few people are getting outrageously rich.
 
 comment: p0ppfdk
 parent: t1_p0pj122
