--- reddit-lost-one-bitcoin-victim@20260804T174652Z
+++ reddit-lost-one-bitcoin-victim@20260804T184725Z
@@ -2979,3 +2979,21 @@
 edited: false
 body:
 I would say no you’re not, get your BTC off ASAP 
+
+comment: p1p5zdz
+parent: t3_1vdjlls
+author: junkhead_7
+created_utc: 1785867380
+edited: false
+body:
+>I was worried about someone physically stealing it or it being destroyed.
+
+If you were worried about your Coldcard being destroyed, you quite clearly lack a lot of understanding on how Bitcoin works except for the "line goes up" part.
+
+comment: p1pa6ir
+parent: t1_p1p5zdz
+author: nothingbutwhammies
+created_utc: 1785868422
+edited: false
+body:
+Feel free to post your seed phrase.
