--- reddit-lost-one-bitcoin-victim@20260805T001822Z
+++ reddit-lost-one-bitcoin-victim@20260805T075839Z
@@ -3013,3 +3013,29 @@
 edited: false
 body:
 But the coins itself is in the internet… 🤷‍♂️
+
+comment: p1s6ixl
+parent: t1_p1oxnqs
+author: Old-Cardiologist-545
+created_utc: 1785899216
+edited: false
+body:
+Why not tho… the hardware is completely fine. It’s an issue with entropy… if you don’t use the devices entropy then you are good. Especially with a pass-phrase.
+
+comment: p1sxo9i
+parent: t1_p1s6ixl
+author: Skinny_Human
+created_utc: 1785910437
+edited: false
+body:
+Because AI is getting exponentially better at everything very very quickly and we won’t be able to keep up. Plus people are inherently incompetent so there could definitely be more bugs and back doors
+
+This is just the beginning
+
+comment: p1syj6b
+parent: t3_1vdjlls
+author: no-bs-no-problems
+created_utc: 1785910832
+edited: false
+body:
+but I thought it was safer than using real $$ in a bank..  I don't get it??
