--- reddit-lost-one-bitcoin-victim@20260804T214741Z
+++ reddit-lost-one-bitcoin-victim@20260805T001822Z
@@ -3005,3 +3005,11 @@
 edited: false
 body:
 Coldcard website felt scammy to me. It advertises the devices on having unlimited memory with an infinity symbol which is impossible
+
+comment: p1r7w4q
+parent: t1_p1j96zj
+author: Jempol_Lele
+created_utc: 1785887807
+edited: false
+body:
+But the coins itself is in the internet… 🤷‍♂️
