--- reddit-lost-one-bitcoin-victim@20260808T083334Z
+++ reddit-lost-one-bitcoin-victim@20260809T165345Z
@@ -3018,14 +3018,6 @@
 body:
 but I thought it was safer than using real $$ in a bank..  I don't get it??
 
-comment: p1tmo00
-parent: t3_1vdjlls
-author: zerotomodd
-created_utc: 1785922501
-edited: false
-body:
-Sorry
-
 comment: p1x10gk
 parent: t3_1vdjlls
 author: After_Psychology5789
@@ -3077,3 +3069,19 @@
 Easy
 
 
+
+comment: p2nwyj3
+parent: t1_p1b4k4v
+author: AdMinute2874
+created_utc: 1786290323
+edited: false
+body:
+Oh my first wallet was ledger.. 
+
+comment: p2nx59x
+parent: t1_p1c3mx9
+author: AdMinute2874
+created_utc: 1786290377
+edited: false
+body:
+But what about fees?
