--- reddit-lost-one-bitcoin-victim@20260804T095550Z
+++ reddit-lost-one-bitcoin-victim@20260804T174652Z
@@ -2971,3 +2971,11 @@
 edited: false
 body:
 Dunno why people wouldn't diversify? If you have a lot in crypto, is it really that big a deal to have 2-3 different hardware wallets, a hot wallet and and exchange account? Or even an airgapped laptop? 
+
+comment: p1oxnqs
+parent: t1_p1h9mly
+author: Skinny_Human
+created_utc: 1785865328
+edited: false
+body:
+I would say no you’re not, get your BTC off ASAP 
