--- reddit-lost-one-bitcoin-victim@20260804T184725Z
+++ reddit-lost-one-bitcoin-victim@20260804T214741Z
@@ -2997,3 +2997,11 @@
 edited: false
 body:
 Feel free to post your seed phrase.
+
+comment: p1qex87
+parent: t3_1vdjlls
+author: False-Car-1218
+created_utc: 1785879133
+edited: false
+body:
+Coldcard website felt scammy to me. It advertises the devices on having unlimited memory with an infinity symbol which is impossible
