--- reddit-lost-one-bitcoin-victim@20260805T210229Z
+++ reddit-lost-one-bitcoin-victim@20260806T100243Z
@@ -3043,3 +3043,19 @@
 edited: false
 body:
 That sucks, but Coldcard users usually get hit when they skip the multisig setup they already knew was safer.
+
+comment: p20kdyu
+parent: t3_1vdjlls
+author: Odd-Intention7059
+created_utc: 1786001232
+edited: false
+body:
+Depp
+
+comment: p20z5bo
+parent: t3_1vdjlls
+author: myafrosheen1
+created_utc: 1786008630
+edited: false
+body:
+Things like this is exactly why the ETFs have a use case whether people want to admit it or not
