--- reddit-lost-one-bitcoin-victim@20260805T075839Z
+++ reddit-lost-one-bitcoin-victim@20260805T142712Z
@@ -1366,13 +1366,11 @@
 
 comment: p1aarp9
 parent: t1_p19kmyc
-author: OldHamburger7923
+author: [deleted]
 created_utc: 1785690207
 edited: false
 body:
-Just as much chance they put money into marketing and design and it's smoke and mirrors. 
-
-In this case they didn't make it look good and they didn't seem to handle security well either. 
+[deleted]
 
 comment: p1ab8i8
 parent: t3_1vdjlls
@@ -1381,14 +1379,6 @@
 edited: false
 body:
 No shit you lost all your money. Stop buying this scam. Good lord 
-
-comment: p1abikb
-parent: t1_p19vtop
-author: OldHamburger7923
-created_utc: 1785690408
-edited: false
-body:
-I don't think for long. Because they are making additional sweeps of accounts. 3 last I checked. If they had been working on this for a year they'd clear them all in one massive sweep instead of one every day or so. 
 
 comment: p1abs43
 parent: t1_p1a1mw8
@@ -3039,3 +3029,11 @@
 edited: false
 body:
 but I thought it was safer than using real $$ in a bank..  I don't get it??
+
+comment: p1tmo00
+parent: t3_1vdjlls
+author: zerotomodd
+created_utc: 1785922501
+edited: false
+body:
+Sorry
