--- reddit-dice-rollers-response@20260806T052509Z
+++ reddit-dice-rollers-response@20260806T100842Z
@@ -691,3 +691,69 @@
 The users who used the TRNG feature didn’t because there was literally nothing to verify..
 
 Using dice, you get the same seed output whether you do it on coldcard or on another source.  So all you would need to do is verify that using the same rolls, coldcard and a second source both yield the same result. 
+
+comment: p20h6bn
+parent: t1_p1xfbye
+author: EyesFor1
+created_utc: 1785999673
+edited: false
+body:
+Ironically, with all the eyes on the new firmware updates, cc is probably one of the safest but no one has trust
+
+
+
+comment: p20heho
+parent: t1_p20h6bn
+author: FishtailAdvice
+created_utc: 1785999782
+edited: false
+body:
+This is completely incorrect. It's clear that no one audits these things. That's what we've learned
+
+comment: p20s08t
+parent: t3_1vghhka
+author: rockpaperbanana
+created_utc: 1786005063
+edited: false
+body:
+Rolled rice over 100 times on the mk4 and the q, I’m not migrating. So many gay doomers on x
+
+comment: p20w60c
+parent: t3_1vghhka
+author: Zestyclose-Way-1351
+created_utc: 1786007143
+edited: false
+body:
+Stay for now, but it'll probably be my last device from them.  May not even be a choice if they, as a business don't survive this.  Reputation and trust is hard to win back. 
+
+comment: p20wppl
+parent: t1_p20s08t
+author: EyesFor1
+created_utc: 1786007421
+edited: false
+body:
+I tend to agree, 100 rolls is secure. 
+
+comment: p20wu41
+parent: t1_p20heho
+author: EyesFor1
+created_utc: 1786007481
+edited: false
+body:
+It was audited a few times, but it was missed. 
+
+comment: p20yl1a
+parent: t3_1vghhka
+author: 4r4nd0mninj4
+created_utc: 1786008352
+edited: false
+body:
+I can't remember how many dice rolls I did in 2021 but it was likely 100 or more. I updated the firmware on Tuesday, deleted the seed, created a new seed and added over 100 rolls and a passphrase. Moved everything to the new wallet with my backup  coldcard yesterday. Seemed to go pretty smoothly. I'm sticking with them until whatever happened is fully investigated and because I don't want to have to deal with buying and dealing with yet another cold wallet vendor. 🤷‍♂️
+
+comment: p212zg3
+parent: t1_p20yl1a
+author: EyesFor1
+created_utc: 1786010462
+edited: false
+body:
+Similar thoughts here. Funds are secure. Sit back, research new devices and in the meantime, hopefully no other flaws are found in other device's codes.
