--- reddit-dice-rollers-response@20260808T150647Z
+++ reddit-dice-rollers-response@20260809T232622Z
@@ -958,3 +958,11 @@
 edited: false
 body:
 Passphrase is not optional
+
+comment: p2psvgu
+parent: t3_1vghhka
+author: ejon7
+created_utc: 1786309671
+edited: false
+body:
+did 100 dice rolls.  unaffected.  will absolutely stay with the wallet, but concerned the company may not make it thru this crisis
