--- reddit-dice-rollers-response@20260809T232622Z
+++ reddit-dice-rollers-response@20260812T044038Z
@@ -919,14 +919,6 @@
 
 
 
-comment: p246scw
-parent: t1_p21jzjw
-author: MarkTupper9
-created_utc: 1786042403
-edited: false
-body:
-How do you prevent or watch out for this type of attack
-
 comment: p24blwq
 parent: t1_p21kboo
 author: FishtailAdvice
@@ -966,3 +958,27 @@
 edited: false
 body:
 did 100 dice rolls.  unaffected.  will absolutely stay with the wallet, but concerned the company may not make it thru this crisis
+
+comment: p2x8e4z
+parent: t1_p1x75j2
+author: consultinglove
+created_utc: 1786399419
+edited: false
+body:
+Honestly at this point you deserve to be laughed at. Hackers completely own the platform and you still decide that it's still worth using. So stubborn that you sit in a burning house, you deserve it when you lose everything hah
+
+comment: p2xl598
+parent: t1_p2x8e4z
+author: notthediz
+created_utc: 1786403246
+edited: false
+body:
+Jeeze bro. Who pissed in your cheerios this morning?
+
+comment: p2yifd9
+parent: t1_p2xl598
+author: consultinglove
+created_utc: 1786414043
+edited: false
+body:
+well based on what you’ve said, you’d happily guzzle all the piss in the world if it came from coldcard. Enjoy the technology!
