--- trezor-coldcard-not-affected@20260813T142452Z
+++ trezor-coldcard-not-affected@20260814T234825Z
@@ -28,6 +28,7 @@
 News
 Coldcard vulnerability: Trezor devices are not affected
 Trezor Team
+Sharing insights on crypto, security & self-custody
 6 mins read
 Aug 5, 2026
 Summary
@@ -89,6 +90,7 @@
 No. Trezor’s seed-generation code is continuously reviewed, publicly auditable and protected by multiple independent entropy sources, entropy checks, signed firmware and a bug bounty program.
 Share
 Trezor Team
+Sharing insights on crypto, security & self-custody
 Articles written by Trezor's team members.
 Join the Trezor Newsletter!
 Receive insider offers, product news, and crypto insights, straight to your inbox.
