--- trezor-coldcard-not-affected@20260806T080956Z
+++ trezor-coldcard-not-affected@20260806T163801Z
@@ -42,7 +42,7 @@
 Your wallet was originally generated on an affected Coldcard device
 You later recovered or imported that same backup created on the Coldcard onto a Trezor
 Moving a wallet backup to a different device does not change the backup itself.
-If this applies to you, follow Coinkite’s official guidance and migrate your funds to a newly generated wallet backup as soon as possible. Take care to verify every step before moving funds.
+If this applies to you, follow these instructions and migrate your funds to a newly generated wallet as soon as possible. Take care to verify every step before moving funds.
 You are not affected if your wallet was originally generated on a Trezor.
 Stay alert for scams
 ­Security events often lead to phishing attempts designed to create fear and urgency.
