--- trezor-coldcard-not-affected@20260812T050810Z
+++ trezor-coldcard-not-affected@20260813T142452Z
@@ -97,10 +97,10 @@
 By clicking Subscribe you agree that Trezor Company s.r.o. will use your email solely to send you its newsletter. You can unsubscribe anytime using the link in any email. To see how we handle your data, view Trezor’s Privacy Policy.
 You might also like
 News
+Recent customer data exposed in shipping provider incident
+Trezor Team
+News
 Why self-custody matters more than ever
-Henry Windle
-News
-Permissionless storage with Suite Sync
 Henry Windle
 a part of SatoshiLabs Group
 Join our newsletter
