--- reddit-ai-discovery-thread@20260804T161019Z
+++ reddit-ai-discovery-thread@20260804T174022Z
@@ -3099,11 +3099,11 @@
 
 comment: p1ap6cu
 parent: t1_p18ho21
-author: b1mm3rl1f3
+author: [deleted]
 created_utc: 1785694050
 edited: false
 body:
-Also, can someone explain why the bluetooth on the Trezor safe 7 isn't considered a vulnerability?
+[deleted]
 
 comment: p1aqpgq
 parent: t1_p18ekrz
@@ -3787,14 +3787,6 @@
 body:
 Then don’t use a styrofoam safe genius.  
 
-comment: p1d8gvr
-parent: t1_p1cqp7r
-author: b1mm3rl1f3
-created_utc: 1785721750
-edited: false
-body:
-I ordered the 7 yesterday, cancelled it, and ordered the 5 today. It isn't fully air gapped and I don't want the option there at all. I'm surprised Trezor went with that 
-
 comment: p1daxn2
 parent: t1_p1brx6e
 author: IInsulince
