--- reddit-ai-discovery-thread@20260806T162811Z
+++ reddit-ai-discovery-thread@20260807T061158Z
@@ -3007,34 +3007,6 @@
 body:
 [deleted]
 
-comment: p1aqpgq
-parent: t1_p18ekrz
-author: RollingMeteors
-created_utc: 1785694462
-edited: false
-body:
-> how fucking stupid ColdCard is and how they clearly did not test their RNG device sufficiently
-
-
-
-
-
-<priceIsRightSadTrombone.wav>
-
-comment: p1aqsh5
-parent: t1_p18ekrz
-author: RollingMeteors
-created_utc: 1785694485
-edited: false
-body:
-> how fucking stupid ColdCard is and how they clearly did not test their RNG device sufficiently
-
-
-
-
-
-<priceIsRightSadTrombone.wav>
-
 comment: p1aqvkv
 parent: t3_1vddeuy
 author: YLCZ
@@ -4185,12 +4157,30 @@
 body:
 It’s crazy that this is wasn’t caught sooner
 
+comment: p23xay4
+parent: t1_p19xhzr
+author: zinornia
+created_utc: 1786040023
+edited: false
+body:
+he did it on purpose 
+
+comment: p265lbf
+parent: t1_p18e9m0
+author: Coleyx123
+created_utc: 1786062428
+edited: false
+body:
+From my reading on this whole incident is that the team was well aware of the vulnerability, as the "bug" was there for testing purposes and was basically just left there, with no code check, even a cursory glance with AI prior to release. But like I said, just what I've read, so don't take it as the gospel until we get hard proof (if that day comes).
+
 more-stub: parent t1_p19a2fb count <live-count>
 
 more-stub: parent t1_p1922vv count <live-count>
 
 more-stub: parent t1_p18n109 count <live-count>
 
+more-stub: parent t1_p18ekrz count <live-count>
+
 more-stub: parent t1_p1ufsx6 count <live-count>
 
 more-stub: parent t1_p19jcia count <live-count>
