--- reddit-ai-discovery-thread@20260806T032432Z
+++ reddit-ai-discovery-thread@20260806T095550Z
@@ -2809,14 +2809,6 @@
 
 CC was well and truly screwed once this firmware shipped on devices.
 
-comment: p1a7tod
-parent: t1_p1937zz
-author: jannies_doit_4_free
-created_utc: 1785689413
-edited: false
-body:
-wtf are you talking about?
-
 comment: p1a82q0
 parent: t1_p18j0mj
 author: jannies_doit_4_free
@@ -4141,16 +4133,6 @@
 body:
 The guy stealing the btc himself and the guy accidentally allowing an exploit are two different things
 
-comment: p1v54ic
-parent: t1_p198qu8
-author: Napoleanna
-created_utc: 1785940869
-edited: false
-body:
-it turns out that switck is doc-hex, [proven by GPG commit signatures](https://gist.github.com/jamesob/ca9b4ca384969b4cfd62813419854d69)
-
-
-
 comment: p1vn1us
 parent: t1_p18dalk
 author: ScreenAppropriate679
@@ -4193,6 +4175,16 @@
 body:
 I trust you pay close attention to small details r/PmMeUrTinyAsianTits 
 
+comment: p20y251
+parent: t3_1vddeuy
+author: SingerLate3349
+created_utc: 1786008097
+edited: false
+body:
+No sé si es por mi profesión "Pero soy muy desconfiado" el hecho de que un dispositivo me genere su propia seed con las palabras que el quiere no me acaba de convencer. Como dice aquel "Poderoso caballero es don dinero".
+Estoy seguro al 100% que hay codigo metido en la mayoría de software actual de "día 0" que permiten estas cosas. Aunque la mayoría de la gente piense que una seed generada por el mismo sea mas insegura, os digo yo que eliminarias ese factor. De hecho creo que los que generaron su propia entropía no están afectados. 
+Con esto quiero aclarar que yo tengo una cantidad insignificante de criptos, yo prefiero tener el respaldo del banco que me asegura 100k por cuenta bancaria. Además debido a las grandes fluctuaciones que tiene me parece un producto de altisimo riesgo, soy de un perfil conservador. Para luchar contra la inflaccion lo mejor es invertir en propiedades, alquiler, airbnb. Igual no haces un x100 o x1000 pero vives tranquilo, con la certeza de que tu propiedad no va a desaparecer como "humo"
+
 more-stub: parent t1_p19a2fb count <live-count>
 
 more-stub: parent t1_p1922vv count <live-count>
@@ -4221,6 +4213,8 @@
 
 more-stub: parent t1_p184yqj count <live-count>
 
+more-stub: parent t1_p1937zz count <live-count>
+
 more-stub: parent t1_p18jy2w count <live-count>
 
 more-stub: parent t1_p18l6la count <live-count>
