--- reddit-victim-support-psa@20260804T134509Z
+++ reddit-victim-support-psa@20260804T184648Z
@@ -323,3 +323,27 @@
 edited: false
 body:
 these coins are gone for good, im sorry :/
+
+comment: p1p7eb7
+parent: t1_p1cxoqo
+author: george420
+created_utc: 1785867730
+edited: false
+body:
+I was part of the hack. Still reeling right now. I'd like to think that I could demonstrate it's was my bitcoin that was stolen because I have all transactions IDs that show the centralized exchange with KYC where I legally purchased them before moving them to the affected wallet. 
+
+comment: p1p7mhq
+parent: t1_p1dfvss
+author: george420
+created_utc: 1785867786
+edited: false
+body:
+I can show the bitcoin was purchased from the exchange legally with KYC and all transaction log. Things hacker cannot do.
+
+comment: p1pb67i
+parent: t1_p1p7mhq
+author: comp21
+created_utc: 1785868671
+edited: false
+body:
+Yes that's the only way you could prove a loss... 
