--- reddit-victim-support-psa@20260804T184648Z
+++ reddit-victim-support-psa@20260808T213338Z
@@ -262,11 +262,11 @@
 
 comment: p1ft6dj
 parent: t1_p1dfvss
-author: Expert-Evening9303
+author: [deleted]
 created_utc: 1785761794
 edited: false
 body:
-I think they mean the physical wallet itself, not the compromised seed. Simply owning a coldcard might be enough to get in on the class action.
+[deleted]
 
 comment: p1fu9wi
 parent: t1_p1ft6dj
@@ -276,14 +276,6 @@
 body:
 I doubt that. I can buy one on eBay and load it with a compromised wallet seed... Wallet seed used to be proof of ownership but now that coldcard has screwed everyone through their own pride and ego, that no longer holds true.
 
-comment: p1fv1a6
-parent: t1_p1fu9wi
-author: Expert-Evening9303
-created_utc: 1785762375
-edited: false
-body:
-I suspect that if a class action goes through, they'll just allow anyone who bought a coldcard within some time frame to join. In the end it doesn't matter much, you'll end up getting like $20 ten years later. Nobody's getting their actual funds back because, like you said, the compromised wallets can be claimed by anyone.
-
 comment: p1gj6kk
 parent: t3_1vdxmk0
 author: petateom
