--- reddit-best-way-keep-safe@20260805T075832Z
+++ reddit-best-way-keep-safe@20260805T142705Z
@@ -190,11 +190,11 @@
 
 comment: p1gdmud
 parent: t1_p1gcyqw
-author: OldHamburger7923
+author: [deleted]
 created_utc: 1785767661
 edited: false
 body:
-You roll your own dice and you make your own seed. You don't rely on the wallet to create it. 
+[deleted]
 
 comment: p1gef3t
 parent: t3_1vedae9
@@ -236,13 +236,11 @@
 
 comment: p1ghroq
 parent: t1_p1gga0h
-author: OldHamburger7923
+author: [deleted]
 created_utc: 1785768757
 edited: false
 body:
-That's why I said roll your own dice and make your own seed. Don't use the wallet to generate the words at all. Once you make your 23 words, coldcard will list the possible matching 24th word for you to select from. Should be about 7 that give you proper checksum. Pick one and you are set.
-
-You can verify it with another wallet if you think your input seed is somehow not being used by the wallet.
+[deleted]
 
 comment: p1gnjd7
 parent: t1_p1gga0h
