--- reddit-best-way-keep-safe@20260804T134539Z
+++ reddit-best-way-keep-safe@20260804T144535Z
@@ -437,3 +437,11 @@
 edited: false
 body:
 You audit their firmware thouroughly 
+
+comment: p1nri5t
+parent: t1_p1g58fk
+author: CiaranCarroll
+created_utc: 1785854657
+edited: false
+body:
+Can I generate a seed on Bitbox trustlessly, like I can on my CC Q? Can I roll dice or some other method of adding entropy that is easily accessible, and verify that the code on my device reproduces same seed from same inputs?
