--- oomahq-2085717166884618584@20260809T121400Z
+++ oomahq-2085717166884618584@20260809T185943Z
@@ -577,7 +577,7 @@
 post: 2085827728415064074
 role: reply
 author: DutyToRebel
-name: Duty to Re₿el #BIP110
+name: Duty to Re₿el
 created: 2026-08-07T20:37:37Z
 media: 0
 body:
@@ -724,15 +724,6 @@
 media: 0
 body:
 Has anyone seen this guy in the last week?
-
-post: 2085883885091741732
-role: reply
-author: Brandonwords
-name: Bangkok Carnivore - BIP110
-created: 2026-08-08T00:20:46Z
-media: 0
-body:
-This has Israel written all over it.
 
 post: 2085884563675349020
 role: reply
@@ -878,15 +869,6 @@
 @nvk
  alt account
 
-post: 2086005374360989950
-role: reply
-author: 9FigureSats
-name: DatSupraKid
-created: 2026-08-08T08:23:32Z
-media: 0
-body:
-@jackmallers
-
 post: 2086007718104473737
 role: reply
 author: MichaelRoerade
@@ -996,7 +978,7 @@
 post: 2086027079649231275
 role: reply
 author: roger__9000
-name: ROGER 9000  + ₿IP-110
+name: ROGER 9000
 created: 2026-08-08T09:49:46Z
 media: 0
 body:
@@ -1213,3 +1195,12 @@
 body:
 Someone tell the 
 @FBI
+
+post: 2086440709175787948
+role: reply
+author: LibertyTrek
+name: LibertyTrek
+created: 2026-08-09T13:13:23Z
+media: 0
+body:
+Wow... words fail me.
