--- oomahq-2085717166884618584@20260812T052232Z
+++ oomahq-2085717166884618584@20260812T120844Z
@@ -197,7 +197,7 @@
 post: 2085727326130598068
 role: reply
 author: GregTonoski
-name: Greg Tonoski, BIP-110
+name: Greg Tonoski, BIP-110, Blake2b
 created: 2026-08-07T13:58:40Z
 media: 0
 body:
@@ -399,16 +399,6 @@
 body:
 Looking like many useful idiots among podcasters
 
-post: 2085798708059459964
-role: reply
-author: MichaelDunwort1
-name: Michael Dunworth
-created: 2026-08-07T18:42:18Z
-media: 0
-body:
-@ozsats256
- 99/1 now
-
 post: 2085799334025842914
 role: reply
 author: jstamby
@@ -583,15 +573,6 @@
 body:
 He thought he was being slick.
 
-post: 2085832812280561894
-role: reply
-author: theretailbull
-name: Tim Lamb
-created: 2026-08-07T20:57:50Z
-media: 0
-body:
-Look when switck stopped posting on X.
-
 post: 2085835412090577304
 role: reply
 author: GRIDsquad
@@ -706,15 +687,6 @@
 Exhibit one.
 
 Everybody knows about this already, you know. Let's talk something else.
-
-post: 2085873969941606436
-role: reply
-author: BeachMaster893
-name: Congo_Dandy
-created: 2026-08-07T23:41:22Z
-media: 0
-body:
-He will be running to Tel Aviv
 
 post: 2085881105132233214
 role: reply
@@ -1170,7 +1142,7 @@
 post: 2086114171708682395
 role: reply
 author: DelcinMaria
-name: Delcin #BIP110
+name: Delcin #Observer
 created: 2026-08-08T15:35:51Z
 media: 0
 body:
