--- mara-slipstream-portal@20260801T091512Z
+++ mara-slipstream-portal@20260803T020201Z
@@ -15,7 +15,7 @@
 <live-fee-rate>
 Current mineable rate
 <live-fee-rate>
-Slipstreams minimum fee rate is the higher of either 2x the current mempool priority fee rate or 2 sats/vByte.
+Slipstreams minimum fee rate is the higher of either 1x the current mempool priority fee rate or 1 sats/vByte.
 Current Block Height: <live-block-height>
 ACTIVATE SLIPSTREAM
 By submitting a transaction, you agree to the Terms of Service.
