--- reddit-coldcard-future-design-philosophy@20260812T044734Z
+++ reddit-coldcard-future-design-philosophy@20260813T125908Z
@@ -207,11 +207,11 @@
 
 comment: p2o3oey
 parent: t1_p2mxm1g
-author: MiguelLancaster
+author: [deleted]
 created_utc: 1786292203
-edited: false
-body:
-https://web.archive.org/web/20200324055513/https://www.nytimes.com/2019/07/19/business/safe-deposit-box-theft.html
+edited: 1786604709
+body:
+[deleted]
 
 comment: p2o8z60
 parent: t1_p2o3oey
@@ -262,20 +262,6 @@
 edited: false
 body:
 Break bitcoin then
-
-comment: p2s9ywu
-parent: t1_p2o8z60
-author: MiguelLancaster
-created_utc: 1786342145
-edited: false
-body:
-fair, but I still think it's overkill
-
-the main reason to keep your seed in a safety deposit box would be to prevent against burglary 
-
-but a passphrase already alleviates that risk
-
-having your seed stored in a location with weekday business hours versus having it in a place accessible to you 24/7 -- the former will never be preferable, in my opinion
 
 comment: p2sljql
 parent: t1_p2l924h
