--- reddit-coldcard-future-design-philosophy@20260813T125908Z
+++ reddit-coldcard-future-design-philosophy@20260813T193044Z
@@ -263,16 +263,6 @@
 body:
 Break bitcoin then
 
-comment: p2sljql
-parent: t1_p2l924h
-author: Individual_Gate9375
-created_utc: 1786347897
-edited: 1786348079
-body:
-You can't fork the firmware because coldcards can only be updated with firmware signed by coinkite. It will refuse any other firmware. So if coinkite just dies and goes away there is no updating them ever again. These devices will end up as useless eventually because let's face it coinkite is going to go away. I doubt they will release the private key they sign the firmware with.   
-  
-Also whoever forked the firmware would open themselves up to a lawsuit because of the license coinkite uses on the firmware. You aren't allowed to use it in any other projects without permission. So they could and likely would sue due to needing money to cover the disaster. No one is going to risk their own finances forking it.
-
 comment: p2so8oz
 parent: t1_p2mpx3o
 author: ChipNDipPlus
