--- stackernews-rng-analysis-by-device@20260804T041458Z
+++ stackernews-rng-analysis-by-device@20260808T015442Z
@@ -4,17 +4,17 @@
       "comments": {
         "comments": [
           {
+            "createdAt": "2026-07-31T06:16:47.574Z",
+            "text": "The way I understand it, strong passphrase and/or human made mmnemonic are still safe, right? 🤠",
+            "user": {
+              "name": "kilianbuhn"
+            }
+          },
+          {
             "createdAt": "2026-07-31T05:14:33.372Z",
             "text": "@Lobotomite [shared](https://stacker.news/items/1536440/r/k00b?commentId=1536598) block's very thorough analysis: [https://engineering.block.xyz/blog/predictable-rng-fallback-and-32-bit-reseed-in-coldcard-firmware](https://engineering.block.xyz/blog/predictable-rng-fallback-and-32-bit-reseed-in-coldcard-firmware)",
             "user": {
               "name": "k00b"
-            }
-          },
-          {
-            "createdAt": "2026-07-31T06:16:47.574Z",
-            "text": "The way I understand it, strong passphrase and/or human made mmnemonic are still safe, right? 🤠",
-            "user": {
-              "name": "kilianbuhn"
             }
           }
         ]
