--- reddit-bitcoin-not-broken-explainer@20260812T044722Z
+++ reddit-bitcoin-not-broken-explainer@20260813T193039Z
@@ -423,18 +423,6 @@
 You have a point, I guess when I say it I actually mean “don’t invest \[in any one thing\] what you can’t afford to lose.” Which is just diversification.
 
 Out of curiosity I did some googling. The original saying is actually “you shouldn’t speculate except with money you can afford to lose” and it comes from a 20th century economist. So you are right it was meant to refer to specifically high risk investments.
-
-comment: p2lsqpo
-parent: t1_p2l2iys
-author: TheRivieraKid22
-created_utc: 1786260271
-edited: false
-body:
-You should defiantly read up on your bank too 
-
-like what happens if the bank goes under - how much of your money is insured and will be returned. 
-
-winging it in life is how shit goes wrong regardless of the industry
 
 comment: p2lw05a
 parent: t3_1vjf9jw
