--- reddit-user-entropy-options@20260804T090107Z
+++ reddit-user-entropy-options@20260804T134218Z
@@ -157,3 +157,21 @@
 Self reply: elsewhere I saw you comment about how dice rolls do let you verify the math. 
 
 I didn't think inputting mouse jiggles or keyboard mashing could be verified... but if so, I am wrong above, and in that case, yes: why not make it very convenient in the device itself to generate a big old number, then prove it did what it should. I get your point.
+
+comment: p1mu60o
+parent: t3_1vcr9uo
+author: Interesting-Gear-992
+created_utc: 1785844883
+edited: false
+body:
+I think restore the trust is an impossible mission.
+
+comment: p1mv7to
+parent: t1_p1mu60o
+author: CornFly2014
+created_utc: 1785845250
+edited: false
+body:
+Its not impossible, but its hard, and it takes hard work an dedication. 
+
+I will agree with you that i'm not sure coinkite has it in them to do what it takes, but i hope so, otherwise my investments with their devices is lost
