--- reddit-user-entropy-options@20260804T134218Z
+++ reddit-user-entropy-options@20260807T125417Z
@@ -175,3 +175,13 @@
 Its not impossible, but its hard, and it takes hard work an dedication. 
 
 I will agree with you that i'm not sure coinkite has it in them to do what it takes, but i hope so, otherwise my investments with their devices is lost
+
+comment: p289hy2
+parent: t3_1vcr9uo
+author: CornFly2014
+created_utc: 1786092823
+edited: false
+body:
+Update: from GitHub it seems this is already in progress: [https://github.com/Coldcard/firmware/pull/707](https://github.com/Coldcard/firmware/pull/707)
+
+Looks like things are improving!
