--- stackernews-losses-climb-112m-state-investigation@20260815T015831Z
+++ stackernews-losses-climb-112m-state-investigation@20260815T060002Z
@@ -18,10 +18,10 @@
             }
           },
           {
-            "createdAt": "2026-08-14T18:29:45.071Z",
-            "text": "Was it North Korea?\n\n![](https://m.stacker.news/152310)\n\nThey have the skills base and experience.",
+            "createdAt": "2026-08-14T17:52:00.543Z",
+            "text": "https://twiiit.com/intangiblecoins/status/2088305623318298903",
             "user": {
-              "name": "Solomonsatoshi"
+              "name": "nitter"
             }
           },
           {
@@ -39,10 +39,10 @@
             }
           },
           {
-            "createdAt": "2026-08-14T17:52:00.543Z",
-            "text": "https://twiiit.com/intangiblecoins/status/2088305623318298903",
+            "createdAt": "2026-08-14T18:29:45.071Z",
+            "text": "Was it North Korea?\n\n![](https://m.stacker.news/152310)\n\nThey have the skills base and experience.",
             "user": {
-              "name": "nitter"
+              "name": "Solomonsatoshi"
             }
           }
         ]
