--- reddit-ccq-dice-passphrase-risk@20260804T091223Z
+++ reddit-ccq-dice-passphrase-risk@20260807T061544Z
@@ -194,3 +194,11 @@
 edited: false
 body:
 Yes. Always double check 
+
+comment: p27jsrm
+parent: t1_p0xx15n
+author: Geekdratic
+created_utc: 1786080674
+edited: false
+body:
+Also, maybe this not how it works and this is just a weird thought, but can't imperfection itself bring a form of entropy? at least as long as its not the exact same imperfection?
