--- coinkite-blog-index@20260803T113826Z
+++ coinkite-blog-index@20260804T203614Z
@@ -20,6 +20,9 @@
 @arcasafes
 Video Tutorials
 @Coinkite
+Adding to the Public Record on Our Ongoing Investigation
+New information for affected users, the public COLDCARD security record, and lessons for teams re...
+Aug 4, 2026
 Update, Sunday.
 An update on the COLDCARD firmware vulnerability, customer support, affected inventory, migration...
 Aug 2, 2026
