--- grassfedbitcoin-2085463112048873485@20260812T183755Z
+++ grassfedbitcoin-2085463112048873485@20260813T013751Z
@@ -27,7 +27,7 @@
 post: 2085464268682469672
 role: reply
 author: BTCtoOblivion
-name: Sovereign Retard/ BIP110
+name: Hardfork Sovereign Retard/ BIP110
 created: 2026-08-06T20:33:22Z
 media: 0
 body:
@@ -213,7 +213,7 @@
 post: 2085773816958263769
 role: reply
 author: SProviso69292
-name: Sherlockean Proviso | on sabbatical | IP is theft!
+name: No Shitcoin Sherlock | on sabbatical | IP is theft
 created: 2026-08-07T17:03:24Z
 media: 0
 body:
