--- grassfedbitcoin-2085463112048873485@20260812T052131Z
+++ grassfedbitcoin-2085463112048873485@20260812T183755Z
@@ -116,7 +116,7 @@
 post: 2085504503063949810
 role: reply
 author: RNHairman
-name: Plebtitioner RN
+name: Reformed Practitioner RN
 created: 2026-08-06T23:13:14Z
 media: 0
 body:
@@ -213,7 +213,7 @@
 post: 2085773816958263769
 role: reply
 author: SProviso69292
-name: Sherlockean Proviso | BIP-110 | IP is theft!
+name: Sherlockean Proviso | on sabbatical | IP is theft!
 created: 2026-08-07T17:03:24Z
 media: 0
 body:
