--- grassfedbitcoin-2085463112048873485@20260814T101057Z
+++ grassfedbitcoin-2085463112048873485@20260815T063731Z
@@ -15,6 +15,25 @@
 @nvk
  himself is stunning.
 
+post: 2074559301796987053
+role: reply
+author: SuiNetwork
+name: Sui
+created: 
+media: 1
+body:
+Sui Basecamp is back.
+
+October 7-8. Marina Bay Sands, Singapore  with 
+@token2049
+.
+
+The next $5 trillion in transactions won’t be human.
+
+The agentic future will be built on Sui.
+
+Build with us.
+
 post: 2085463470267351090
 role: reply
 author: SchlemmerMaul5
