--- stackernews-nvk-deleted-posts@20260806T050829Z
+++ stackernews-nvk-deleted-posts@20260806T235609Z
@@ -15,6 +15,13 @@
             "text": "Deleting tweets is a bad look.",
             "user": {
               "name": "Scoresby"
+            }
+          },
+          {
+            "createdAt": "2026-08-06T18:31:05.950Z",
+            "text": "This is very odd, and  I'm not saying this is necessarily the reason, but a smart person in bad faith would  do exactly something like this if there were a few tweets that weren't obviously incriminating, but that they really wanted deleted without drawing attention to them.\n\nIt proves nothing, of course, but it is not a good look. In fact, it may be the worst possible look IMO.",
+            "user": {
+              "name": "rblb"
             }
           },
           {
