--- reddit-coinkite-refund-question@20260804T101519Z
+++ reddit-coinkite-refund-question@20260804T111509Z
@@ -64,3 +64,11 @@
 body:
 They warned about this as possible issue years ago:  
 [https://x.com/COLDCARDwallet/status/1447213375398846473](https://x.com/COLDCARDwallet/status/1447213375398846473)
+
+comment: p1mlknb
+parent: t1_p1l4ymj
+author: drumcraze92
+created_utc: 1785841701
+edited: false
+body:
+Refund for the purchase of the cold card itself. Not lost bitcoin 
