--- reddit-cancelled-order-shipped@20260807T142545Z
+++ reddit-cancelled-order-shipped@20260809T184841Z
@@ -133,4 +133,17 @@
 body:
 Did they include the stickers too?  No shitcoins Inside?
 
+comment: p2n6vqs
+parent: t3_1vhbex7
+author: Lost-Bowl3269
+created_utc: 1786282596
+edited: false
+body:
+Não faça multisig com mesma marca de hardwallet. Se a marca tiver falha (e teve) você perdera mesmo com multisig. 
+Quem tinha multisig com duas mk3 por exemplo. 
+
+Use sempre 2 marcas diferentes. 
+
+Outra questão é: pq apoiar esses malditos que destruíram a vida de varias famílias? Você deveria parar de usar os produtos dessa marca.
+
 more-stub: parent t1_p24wtuu count <live-count>
