--- reddit-jade-plus-setup-suspect@20260809T020517Z
+++ reddit-jade-plus-setup-suspect@20260812T044433Z
@@ -133,3 +133,13 @@
 edited: false
 body:
 Go to Blockstream’s official firmware site (make sure it’s http://jadefw.blockstream.com and not some other dodgy scam site!). Download the latest onto a USB-C drive. Blockstream has its own drive for this called Jadelink but you can use any old drive. Plug this into the Jade, choose the option to update firmware, and just follow the instructions on the device. There is a “no radio” version of firmware that disables bluetooth. Use that one if you want to make sure you never turn on bluetooth accidentally.
+
+comment: p2x21c0
+parent: t1_p2kcsn1
+author: TheresNoSecondBest
+created_utc: 1786397620
+edited: false
+body:
+You can use a USB stick to do so. 
+
+I prefer not to update my air-gapped setups unless there's a huge vulnerability. The wallet is like a big numbers calculator, the math never changes. 
