--- reddit-coldcard-just-drained-rug@20260804T090124Z
+++ reddit-coldcard-just-drained-rug@20260804T155609Z
@@ -251,4 +251,12 @@
 body:
 Ledger is the best no denying 
 
+comment: p1o2cjn
+parent: t1_p1joogo
+author: Highheat_10
+created_utc: 1785857494
+edited: false
+body:
+Gtfo outta here u probably own .2 BTC talking smack .. I have bitcoins not only 1 .. we assessed the situation and feel comfortable with our decision.. now do you ! Good luck kid stay in school 
+
 more-stub: parent t1_p1jzgol count 0
