--- reddit-coldcard-just-drained-rug@20260805T015544Z
+++ reddit-coldcard-just-drained-rug@20260805T080558Z
@@ -275,4 +275,12 @@
 body:
 *Press X to doubt* 
 
+comment: p1s55mi
+parent: t1_p1bs97u
+author: Perdurabo306
+created_utc: 1785898728
+edited: false
+body:
+No one deserves this man. Fing brutal. I hope they track down who did this along with the stolen BTC. 
+
 more-stub: parent t1_p1jzgol count <live-count>
