--- reddit-fourth-wave-389-btc@20260804T093444Z
+++ reddit-fourth-wave-389-btc@20260804T134434Z
@@ -1718,3 +1718,37 @@
 edited: false
 body:
 Oy. It would be like the scammer has monthly subscribers. Just a few people in the dark sending their compromised wallets money every month and having no clue. This is so upsetting to think about.  
+
+comment: p1mnvts
+parent: t1_p1dswdd
+author: Icy_Giraffe_21
+created_utc: 1785842601
+edited: false
+body:
+Someone else posted a screenshot shot of .7 BTC being drained and it was like 7-8 small transactions also, very odd.
+
+comment: p1mrbfr
+parent: t1_p1j5b90
+author: Mobe-E-Duck
+created_utc: 1785843870
+edited: false
+body:
+One thing I use is security hygiene which dictates not disclosing the specifics. 
+
+comment: p1n1vls
+parent: t1_p1mrbfr
+author: dj_destroyer
+created_utc: 1785847423
+edited: false
+body:
+lol ok
+
+comment: p1n8qzb
+parent: t1_p1k9bu6
+author: GentlemenHODL
+created_utc: 1785849500
+edited: false
+body:
+lol ok child who doesn't understand the difference between corruption and conspiracy. 
+
+
