--- reddit-fourth-wave-389-btc@20260808T150259Z
+++ reddit-fourth-wave-389-btc@20260808T213254Z
@@ -1822,3 +1822,13 @@
 edited: false
 body:
 But you’re trusting a third party here (the exchange)
+
+comment: p2j1mw8
+parent: t1_p2cyam9
+author: No_Ambassador_1299
+created_utc: 1786222585
+edited: false
+body:
+We’re talking about not trusting 3rd parties to generate your keys.
+
+You have to trust someone to buy bitcoin from unless you mine it.
