--- reddit-fourth-wave-389-btc@20260805T014724Z
+++ reddit-fourth-wave-389-btc@20260805T142559Z
@@ -55,13 +55,11 @@
 
 comment: p1e7x4l
 parent: t1_p1e3djc
-author: OldHamburger7923
+author: [deleted]
 created_utc: 1785736300
 edited: false
 body:
-People saying Ledger and the others are secure don't seem to understand that you can't know what their system is generating without the source. Yes, in this case no one spotted the RNG issue before it was exploited, but closed source can absolutely build a kleptogrum PRNG exploit and you'd never know. 
-
-Everyone needs to assume they're out to get you.
+[deleted]
 
 comment: p1e9vwo
 parent: t1_p1dswdd
@@ -174,14 +172,6 @@
 edited: false
 body:
 Its not licensed as open source but the source code is still available for anybody to see 
-
-comment: p1eiiuu
-parent: t1_p1ef2da
-author: OldHamburger7923
-created_utc: 1785741547
-edited: false
-body:
-Good idea, so lets make everything closed source and.... trust they didn't do something dumb and didn't put in backdoors? I can't tell what you're arguing for. Sounds dumb.
 
 comment: p1eiyzj
 parent: t3_1ve28wl
@@ -243,11 +233,11 @@
 
 comment: p1elf70
 parent: t1_p1ejsrh
-author: OldHamburger7923
+author: [deleted]
 created_utc: 1785743022
 edited: false
 body:
-The passphrase can be stronger than your 24 word seed too! It doesn't need to be weaker.
+[deleted]
 
 comment: p1elxoh
 parent: t1_p1e1t0t
