--- reddit-fourth-wave-389-btc@20260807T061707Z
+++ reddit-fourth-wave-389-btc@20260807T192907Z
@@ -1806,3 +1806,19 @@
 edited: false
 body:
 Got me for .17 thank God I had the majority locked up in a Bitcoin backed loan. Debt saved my ass for once. I’m truly heartbroken for those who lost so much more. This has been a lesson for me that sadly Bitcoin is not ready for mainstream. 
+
+comment: p29lo6m
+parent: t3_1ve28wl
+author: Maximum_Stretch1465
+created_utc: 1786110807
+edited: false
+body:
+I bet its the ai who escaped recently
+
+comment: p2bqtya
+parent: t1_p1dswdd
+author: mrAech219
+created_utc: 1786130888
+edited: false
+body:
+Your funds were never on coldcard
