--- reddit-fourth-wave-389-btc@20260804T134434Z
+++ reddit-fourth-wave-389-btc@20260804T154656Z
@@ -1752,3 +1752,19 @@
 lol ok child who doesn't understand the difference between corruption and conspiracy. 
 
 
+
+comment: p1o4cqh
+parent: t1_p1fyepd
+author: Guava_Poppa
+created_utc: 1785858009
+edited: false
+body:
+Yeah. But that FTX debacle has me sweating aswell. 😅
+
+comment: p1o54gc
+parent: t1_p1im4cm
+author: Guava_Poppa
+created_utc: 1785858205
+edited: false
+body:
+You know how you get that sunken feeling when you misplaced something, like your phone... This would induce a stroke.
