--- reddit-fourth-wave-389-btc@20260806T032951Z
+++ reddit-fourth-wave-389-btc@20260807T061707Z
@@ -774,11 +774,11 @@
 
 comment: p1fs0bv
 parent: t1_p1fhdrf
-author: 12ealdeal
+author: [deleted]
 created_utc: 1785761424
 edited: false
 body:
-What guy? And?
+[deleted]
 
 comment: p1fsx64
 parent: t1_p1edj3d
@@ -1790,3 +1790,19 @@
 edited: false
 body:
 Please stop .. u.u
+
+comment: p260n8g
+parent: t3_1ve28wl
+author: Trilamb22
+created_utc: 1786060830
+edited: false
+body:
+This it is hilarious. Ellipal and/or Trezor folks, the only way to go, lets u sleep very well.
+
+comment: p27mziv
+parent: t3_1ve28wl
+author: Anxious-Fondant-8929
+created_utc: 1786082109
+edited: false
+body:
+Got me for .17 thank God I had the majority locked up in a Bitcoin backed loan. Debt saved my ass for once. I’m truly heartbroken for those who lost so much more. This has been a lesson for me that sadly Bitcoin is not ready for mainstream. 
