--- stackernews-entropy-bug-history@20260804T181708Z
+++ stackernews-entropy-bug-history@20260804T184812Z
@@ -18,6 +18,13 @@
             }
           },
           {
+            "createdAt": "2026-08-04T17:54:45.626Z",
+            "text": "He has a weird obsession with \"doing the hard stuff\". He mentioned the words over and over again over the years.\n\nIt's very weird tbh. Linkedin startup bros are weirdos, they're so full of themselves.",
+            "user": {
+              "name": "tomlaies"
+            }
+          },
+          {
             "createdAt": "2026-08-04T13:54:18.496Z",
             "text": "Good work here @SimpleStacker. I was not aware of the Trezor copying thing. That really is rich for NVK. Yet another example of his ego canceling out his intelligence.",
             "user": {
@@ -29,13 +36,6 @@
             "text": "\"The feeling that this whole chain of events was triggered by NVK, who wanted to get out of GPLv3 because he felt someone had unfairly cloned his code, something he himself had been accused of just two years earlier.\"\n\nI agree with this; pure revenge.",
             "user": {
               "name": "Akg10s3"
-            }
-          },
-          {
-            "createdAt": "2026-08-04T17:54:45.626Z",
-            "text": "He has a weird obsession with \"doing the hard stuff\". He mentioned the words over and over again over the years.\n\nIt's very weird tbh. Linkedin startup bros are weirdos, they're so full of themselves.",
-            "user": {
-              "name": "tomlaies"
             }
           }
         ]
