--- stackernews-entropy-bug-history@20260804T184812Z
+++ stackernews-entropy-bug-history@20260804T191733Z
@@ -11,17 +11,17 @@
             }
           },
           {
+            "createdAt": "2026-08-04T17:54:45.626Z",
+            "text": "He has a weird obsession with \"doing the hard stuff\". He mentioned the words over and over again over the years.\n\nIt's very weird tbh. Linkedin startup bros are weirdos, they're so full of themselves.",
+            "user": {
+              "name": "tomlaies"
+            }
+          },
+          {
             "createdAt": "2026-08-04T13:39:08.347Z",
             "text": "Good writeup. Thanks.",
             "user": {
               "name": "siggy47"
-            }
-          },
-          {
-            "createdAt": "2026-08-04T17:54:45.626Z",
-            "text": "He has a weird obsession with \"doing the hard stuff\". He mentioned the words over and over again over the years.\n\nIt's very weird tbh. Linkedin startup bros are weirdos, they're so full of themselves.",
-            "user": {
-              "name": "tomlaies"
             }
           },
           {
