--- stackernews-entropy-bug-history@20260804T194815Z
+++ stackernews-entropy-bug-history@20260804T201845Z
@@ -25,6 +25,13 @@
             }
           },
           {
+            "createdAt": "2026-08-04T20:03:28.703Z",
+            "text": "\"they argue that the primary reason for migrating to `libNgU` was to use its `libsecp256k1` implementation.\"\n\n How the why \"defends” incompetent migration I'm unsure, it doesn't.  What it does do is reinforce the criticality of RNG to hide your key in the universe, not in a large virus:\n\nhttps://youtu.be/IwrKWdxt0YM?si=rPWBWgmzP8bFUOPy\n\nAnd they failed, for five years, while every OG, including me, prated on about NVK and his incredible team.",
+            "user": {
+              "name": "satyagraha"
+            }
+          },
+          {
             "createdAt": "2026-08-04T13:54:18.496Z",
             "text": "Good work here @SimpleStacker. I was not aware of the Trezor copying thing. That really is rich for NVK. Yet another example of his ego canceling out his intelligence.",
             "user": {
