--- stackernews-entropy-bug-history@20260804T191733Z
+++ stackernews-entropy-bug-history@20260804T194815Z
@@ -3,13 +3,6 @@
     "item": {
       "comments": {
         "comments": [
-          {
-            "createdAt": "2026-08-04T13:40:15.806Z",
-            "text": "So... using Trezor software good, rolling your own, not good? Rolling your own is always risky. \n\nI'm still not sure why this context makes it looks bad or could have made it look better in the end. The thing that was deemed needed was to get off GPLv3, for commercial reasons. This caused the change in libs which caused the bug.\n\nWhat it does do is highlight that a lot of people make money off of open source software. It always sucks when you see someone pull in a couple million in VC rounds by using your effort - though I think that it hurts more when you were not selling anything to anyone. Also happens in other parts of FOSS, think... wordpress.\n\nRegardless, can you explain \"how bad it is\"?",
-            "user": {
-              "name": "optimism"
-            }
-          },
           {
             "createdAt": "2026-08-04T17:54:45.626Z",
             "text": "He has a weird obsession with \"doing the hard stuff\". He mentioned the words over and over again over the years.\n\nIt's very weird tbh. Linkedin startup bros are weirdos, they're so full of themselves.",
@@ -22,6 +15,13 @@
             "text": "Good writeup. Thanks.",
             "user": {
               "name": "siggy47"
+            }
+          },
+          {
+            "createdAt": "2026-08-04T13:40:15.806Z",
+            "text": "So... using Trezor software good, rolling your own, not good? Rolling your own is always risky. \n\nI'm still not sure why this context makes it looks bad or could have made it look better in the end. The thing that was deemed needed was to get off GPLv3, for commercial reasons. This caused the change in libs which caused the bug.\n\nWhat it does do is highlight that a lot of people make money off of open source software. It always sucks when you see someone pull in a couple million in VC rounds by using your effort - though I think that it hurts more when you were not selling anything to anyone. Also happens in other parts of FOSS, think... wordpress.\n\nRegardless, can you explain \"how bad it is\"?",
+            "user": {
+              "name": "optimism"
             }
           },
           {
