--- reddit-5-13-btc-loss@20260805T205949Z
+++ reddit-5-13-btc-loss@20260806T032859Z
@@ -162,11 +162,11 @@
 
 comment: p1ih75s
 parent: t3_1vemip7
-author: dyedsthaytuned7
+author: [deleted]
 created_utc: 1785786936
 edited: false
 body:
-Losing that much, especially when it was for your daughter, is brutal.... Youre not alone in feeling angry and shaken and its okay to step back from BTC from a while. Wishing you and your dad strength !
+[removed]
 
 comment: p1ihl7n
 parent: t3_1vemip7
@@ -1022,14 +1022,6 @@
 body:
 Imagine if you just put this money and effort into a stable investment on a reputable investment platform lmao. 
 
-comment: p1pr8iv
-parent: t1_p1ijkxu
-author: dyedsthaytuned7
-created_utc: 1785872810
-edited: false
-body:
-no problem at all, I wish you the best! 
-
 comment: p1skcvk
 parent: t3_1vemip7
 author: HeyDontSkipLegDay
