--- reddit-5-13-btc-loss@20260808T213158Z
+++ reddit-5-13-btc-loss@20260809T040426Z
@@ -1157,3 +1157,19 @@
 If they could even afford to be, that is 
 
 (From an ethical perspective. Legally, they may find a way to weasel out of it).
+
+comment: p2jigqo
+parent: t1_p2iid9f
+author: Electronic_Cover_134
+created_utc: 1786227781
+edited: false
+body:
+What's odd is you believe in the white hats lol
+
+comment: p2k095a
+parent: t1_p2jigqo
+author: n8dahwgg
+created_utc: 1786233726
+edited: false
+body:
+I mean, I know the guy so I know of at least one
