--- reddit-5-13-btc-loss@20260804T161545Z
+++ reddit-5-13-btc-loss@20260804T164623Z
@@ -595,14 +595,6 @@
 body:
 Sorry man,family is the nly thing we really have…question for the community? I’m not a tech guy but iv managed to stash coins on 3 trezors..Does the community think trezors are still safe…
 
-comment: p1k2vbz
-parent: t3_1vemip7
-author: panthera_NN
-created_utc: 1785803761
-edited: false
-body:
-I've always been worried about someone creating a seed phrase similar to mine, so I used a passphrase. Theoretically, using a passphrase still doesn't prevent someone from accidentally creating a seed phrase that could potentially access my wallet, because the 24 seedphrases plus the 25th passphrase still have a chance of matching another 24 seedphrases. I did it for peace of mind. However, coincidentally, a strong 25th passphrase can save a weak/flawed 24-seedphrase, so this should be a mandatory requirement when creating a new wallet, and the passphrase should include uppercase and lowercase letters, numbers, special characters, and a minimum length to be accepted, just like when you create a password for an online account. I'm very sorry for your loss. After this incident, I've also upgraded my passphrase to be a bit longer. Previously, I split the money between two places, but I wasn't serious enough about it. Now I will be truly serious about this.
-
 comment: p1k6hgm
 parent: t1_p1j681q
 author: elfwriter
