--- reddit-5-13-btc-loss@20260807T192819Z
+++ reddit-5-13-btc-loss@20260808T015835Z
@@ -1022,14 +1022,6 @@
 body:
 Imagine if you just put this money and effort into a stable investment on a reputable investment platform lmao. 
 
-comment: p1skcvk
-parent: t3_1vemip7
-author: HeyDontSkipLegDay
-created_utc: 1785904522
-edited: false
-body:
-Sorry for your loss
-
 comment: p1x9o47
 parent: t3_1vemip7
 author: BillLow4780
@@ -1045,3 +1037,35 @@
 edited: false
 body:
 Cold card company should refund everyone that trusted then with their life savings to make things right but they wont
+
+comment: p2cdwgn
+parent: t3_1vemip7
+author: Firm_Yogurtcloset835
+created_utc: 1786137173
+edited: false
+body:
+First of all, I am very sorry for your loss; I wish your father a speedy recovery and hope everything is well with you, your daughter, and your family.
+
+Thank you for sharing your stories and thoughts.
+
+Regarding some of the replies you received: it is very easy—after the milk has already spilled—to point fingers and say you should have stayed close while it was boiling (without showing any empathy or understanding the full context of why you were away from the stove at that moment).
+
+To be realistic, hardware wallets have helped with BTC adoption, and it is highly hypocritical for anyone to say things like "it was just the dice roll" or "it was just the passphrase."
+
+If not for that extremely serious and fundamental developer error in the code (which warrants an investigation into whether it was truly just a mistake), this topic wouldn't even exist—regardless of dice rolls or passphrases—because the entropy would have met current security standards.
+
+True Bitcoin holders and members of the Bitcoin community would never accuse or blame the victim in a situation like this.
+
+To the "adventurers" and know-it-alls: I wish you no ill will, but I do ask that you rethink your values ​​and your outlook on life.
+
+Blaming the victims in this case is unacceptable.
+
+The company had a mission when selling its product; the single most important aspect of a hardware wallet is seed generation, and that has absolutely nothing to do with dice rolls... again, if the code had been correct, we wouldn't be here discussing this topic.
+
+It was a basic, grotesque, and incompetent error—setting a variable to zero in the code... it makes me choke with anger just writing about it.
+
+It needs to be investigated.
+
+I will pray that "white hat" hackers manage to return at least a portion of the funds, and that the company is held accountable and reimburses all the victims.
+
+Take care.
