--- reddit-5-13-btc-loss@20260808T150201Z
+++ reddit-5-13-btc-loss@20260808T213158Z
@@ -1103,3 +1103,57 @@
 I’m using Trezor and ledger.
 
 Can I just get a lightning wallet from you- I’d like to send you some sats! Take it as a sign from the BTC community that there are also (and mostly) very kind and friendly people who like to help each other a bit more!
+
+comment: p2h40xd
+parent: t3_1vemip7
+author: Vagelen_Von
+created_utc: 1786202546
+edited: false
+body:
+I am glad never heard of Coldcard until July 2026.
+
+comment: p2hbzzw
+parent: t3_1vemip7
+author: wangshuying
+created_utc: 1786204842
+edited: false
+body:
+I lost $500,000 on ftx. It was painful at the time. 
+
+Then I made it all back many times over
+
+You will come back stronger
+
+comment: p2hoed7
+parent: t3_1vemip7
+author: AllKnowingFlan
+created_utc: 1786208372
+edited: false
+body:
+Coinkite should be liable, full stop
+
+comment: p2i7wvr
+parent: t1_p1ifem5
+author: Electronic_Cover_134
+created_utc: 1786213858
+edited: false
+body:
+White hat hacker's lol Q 😆 
+
+comment: p2iid9f
+parent: t1_p2i7wvr
+author: n8dahwgg
+created_utc: 1786216884
+edited: false
+body:
+You havent spoken on reddit for 128 days and this js your masterful return comment? Odd…
+
+comment: p2ivtvj
+parent: t1_p2hoed7
+author: AllKnowingFlan
+created_utc: 1786220864
+edited: false
+body:
+If they could even afford to be, that is 
+
+(From an ethical perspective. Legally, they may find a way to weasel out of it).
