--- reddit-5-13-btc-loss@20260804T201553Z
+++ reddit-5-13-btc-loss@20260804T232139Z
@@ -962,6 +962,16 @@
 body:
 I just know the person on a personal level. I really admire him for taking the risk to do it. Not going to dox him unless he wants the world to know.
 
+comment: p1mzpij
+parent: t3_1vemip7
+author: BeginningBeautiful69
+created_utc: 1785846731
+edited: false
+body:
+So sorry to read your story. You did everything right. This could have been anyone. Just plain shitty luck. So many hopes and plans wrecked. Concentrating on family time now. My fingers well and truly crossed for you that those bitcoin make their way back to you some day. 🤞 
+
+Your point about the number of 'told you so' dice rolling multisig experts coming out of the woodwork resonates. I too thought coldcard to be the gold standard, obviously without verifing the TRNG code (I mean, come the fuck on). Good speed. 
+
 comment: p1n4rcy
 parent: t1_p1k6hgm
 author: Even_Routine5011
@@ -970,6 +980,14 @@
 body:
 Isnt the term "ugly on the inside" or are you saying they are physically ugly? 😂
 
+comment: p1n5o4h
+parent: t1_p1ifrjp
+author: Even_Routine5011
+created_utc: 1785848587
+edited: false
+body:
+I recently just had a 1 one ounce gold Canadian maple stolen from me so I've been more pissed with Gold recently than Bitcoin
+
 comment: p1ndohg
 parent: t1_p1l9gdr
 author: n8dahwgg
