--- reddit-open-source-missed-flaw@20260804T095542Z
+++ reddit-open-source-missed-flaw@20260806T163456Z
@@ -226,15 +226,11 @@
 
 comment: p1ezucw
 parent: t1_p1ekxtr
-author: IllllIIlIllIllllIlll
+author: [deleted]
 created_utc: 1785750319
-edited: false
-body:
-You don't need so many rolls 48 dice rolls and 12 coin flips are enough
-
-[https://bitbox.swiss/bitbox02/BitBox\_Diceware\_HowTo.pdf](https://bitbox.swiss/bitbox02/BitBox_Diceware_HowTo.pdf)
-
-[https://bitbox.swiss/bitbox02/BitBox\_Diceware\_LookupTable.pdf](https://bitbox.swiss/bitbox02/BitBox_Diceware_LookupTable.pdf)
+edited: 1786028087
+body:
+[deleted]
 
 comment: p1f5cdb
 parent: t1_p1ezucw
@@ -247,14 +243,6 @@
 
 We need to get at least random seed generation into our own hands !
 
-comment: p1f6cy2
-parent: t1_p1f5cdb
-author: IllllIIlIllIllllIlll
-created_utc: 1785753345
-edited: false
-body:
-It would be completely useless but you do you
-
 comment: p1f6kdb
 parent: t1_p1eimhe
 author: oaga_strizzi
@@ -776,3 +764,11 @@
 edited: false
 body:
 I’d guess it was a Mythos/Fable or Astra model find, and we should expect a lot more in the coming months.
+
+comment: p237jdf
+parent: t1_p1hyz3w
+author: aaj094
+created_utc: 1786033707
+edited: false
+body:
+Okay, so is Trezor fully open source by every standard of what is meant by that?
