--- reddit-coinkite-inventory-destruction-question@20260813T015623Z
+++ reddit-coinkite-inventory-destruction-question@20260813T063109Z
@@ -747,3 +747,23 @@
 edited: false
 body:
 is that right?  less than 10 employees?
+
+comment: p3d43mn
+parent: t1_p39oon3
+author: Mr_Ander5on
+created_utc: 1786586594
+edited: false
+body:
+I can’t audit code so I would have to trust them. I don’t know anyone that could perform that task and most people don’t either.
+
+99% of people are trusting, and from what I saw it didn’t even need AI to find it, no one actually audited it. No third party audits, everyone just trusted cold card.
+
+It’s still gross negligence, they should be auditing their own code to find stuff like that. 
+
+comment: p3dgm9d
+parent: t1_p3cmv3y
+author: AnthonyBTC
+created_utc: 1786590995
+edited: false
+body:
+Yes.
