--- reddit-coinkite-inventory-destruction-question@20260813T130014Z
+++ reddit-coinkite-inventory-destruction-question@20260814T234655Z
@@ -370,19 +370,11 @@
 
 comment: p31qt1u
 parent: t1_p30dac9
-author: striata
+author: [deleted]
 created_utc: 1786459787
 edited: 1786464138
 body:
-Nothing of the stuff in that post proves anything. It just reframes old Twitter posts in a malevolent light.
-
-This is, to put it mildly, pizzagate/illuminati level conspiratorial slop. Text also reeks of being written by an LLM, full of drama, which is par for the course.
-
-Of course they put "bug" in quotes when describing a retirement attack, because that's what the attack entails, intentionality. It doesn't mean they are subtly nodding to their own "bug" like some Illuminati-type villain.
-
-Of course physical dice rolls are opt-in and not the default source of entropy, because they want to make the device user-friendly. Nobody wants to do the dice rolls. Them saying that doing dice rolls is the only way to absolutely ensure you don't get affected by attacks or bugs is just a fact, again not some conspiratorial nod to their own malevolence.
-
-The part about the alias identity is the only thing that is actually weird here, but it still doesn't prove anything. It makes no sense that he'd make an alias and configure signing using the same key, but hey, maybe it's another subtle nod and "hiding in plain sight" eh?
+[deleted]
 
 comment: p31siwp
 parent: t1_p30l1bz
@@ -522,11 +514,11 @@
 
 comment: p33ttis
 parent: t1_p30u3bn
-author: Save_JR
+author: [deleted]
 created_utc: 1786478899
 edited: false
 body:
-Dumbass
+[removed]
 
 comment: p33wi8x
 parent: t1_p30mcmr
@@ -793,3 +785,11 @@
 edited: false
 body:
 same here. 
+
+comment: p3ncej2
+parent: t3_1vld3rt
+author: Levitdon
+created_utc: 1786715493
+edited: false
+body:
+Those old affected devices which haven’t been flashed with the updated software from CoinKite, are now the ONLY available hardware evidence left to further investigate what else might be further compromised leading to the known yet unknown! Beware, the real Truth is hard to Bear!
