--- hn-dettmer-writeup-thread@20260804T153908Z
+++ hn-dettmer-writeup-thread@20260804T180917Z
@@ -65,13 +65,7 @@
 2. Totally incompetent and copying code from Stack Overflow.
 I don't think it's specific to microcontrollers, either, the error from the C compiler is something that a competent programmer would be able to interpret, or at least see it as a signal to get someone more experienced in the domain to learn about.
 reply
-Scoundreller <relative-time> | root | parent | next [–]
-Hey now, I'm totally incompetent, so my code writes entries in a log file for virtually everything because otherwise I have no idea if it's doing what I expect. Would've caught some critical function not being called at all.
-reply
-eru <relative-time> | root | parent | prev | next [–]
-Of course, being a total novice and claiming to produce a super secure gadget are not to things you want to go together.
-reply
-nullc <relative-time> | root | parent | prev | next [–]
+nullc <relative-time> | root | parent | next [–]
 > the error from the C compiler is something that a competent programmer would be able to interpret,
 contrary to the post, this was almost certainly not an issue of compilers throwing errors.
 Reduced familarity with C could have played a role, but given the number of C experts that looked at this knowing there was an error and still misidentified the cause I don't think we need to reach for that powerful an explanation.
@@ -80,6 +74,12 @@
 scott_w <relative-time> | root | parent | next [–]
 Fair point: I was using the information in the article to form my opinion. I still think that, if the developer in question ignored a compiler error as described, that’s a serious competence issue. It’s harder to spot after the fact because any reviewers wouldn’t have had a chance to see the compiler output.
 The article makes the fair point of the size of the change and the complete lack of information in the commit message, which should have set off alarm bells…
+reply
+Scoundreller <relative-time> | root | parent | prev | next [–]
+Hey now, I'm totally incompetent, so my code writes entries in a log file for virtually everything because otherwise I have no idea if it's doing what I expect. Would've caught some critical function not being called at all.
+reply
+eru <relative-time> | root | parent | prev | next [–]
+Of course, being a total novice and claiming to produce a super secure gadget are not to things you want to go together.
 reply
 jgilias <relative-time> | root | parent | prev | next [–]
 The bug has been up since before the LLM shift.
