--- hn-dettmer-writeup-thread@20260803T121314Z
+++ hn-dettmer-writeup-thread@20260803T154547Z
@@ -1,6 +1,6 @@
 Hacker Newsnew | past | comments | ask | show | jobs | submitlogin
 When random.bytes() runs but doesn't work (btcpp.dev)
-93 points by Funes- <relative-time> | hide | past | favorite | 55 comments
+94 points by Funes- <relative-time> | hide | past | favorite | 55 comments
 help
 londons_explore <relative-time> | next [–]
 I am getting suspicious of "random number generator not actually returning random numbers" bugs....
