--- hn-dettmer-writeup-thread@20260804T143850Z
+++ hn-dettmer-writeup-thread@20260804T153908Z
@@ -84,6 +84,10 @@
 jgilias <relative-time> | root | parent | prev | next [–]
 The bug has been up since before the LLM shift.
 reply
+dmitrygr <relative-time> | prev | next [–]
+>Micropython creates the illusion embedded developers do not need to understand C, their CPU, or other advanced concepts to do embedded programming.
+Amen.
+reply
 PunchyHamster <relative-time> | prev | next [–]
 > The commit message is 235 characters, and the commit changes 15 lines of code. That’s a ratio of 235/15 = ~16.
 the commit message is not 235 character long, the commit message is this
@@ -91,10 +95,6 @@
 > splice-script: Round channel balances down
 author mistakes commit message for PR which doesn't get persisted into code's history and don't even contain the ID of bug being fixed.
 So yeah it is better but nothing to actually brag about
-reply
-dmitrygr <relative-time> | prev | next [–]
->Micropython creates the illusion embedded developers do not need to understand C, their CPU, or other advanced concepts to do embedded programming.
-Amen.
 reply
 hypeatei <relative-time> | prev | next [–]
 > You, as a developer working on Bitcoin, need to take your time to understand your changes, document them clearly, and verify they do what you think.
