--- hn-dettmer-writeup-thread@20260806T040624Z
+++ hn-dettmer-writeup-thread@20260807T061139Z
@@ -135,7 +135,23 @@
               "children": [
                 {
                   "author": "nullc",
-                  "children": [],
+                  "children": [
+                    {
+                      "author": "jki275",
+                      "children": [],
+                      "created_at": "2026-08-07T01:18:26.000Z",
+                      "created_at_i": 1786065506,
+                      "id": 49204786,
+                      "options": [],
+                      "parent_id": 49146607,
+                      "points": null,
+                      "story_id": 49140405,
+                      "text": "I think some things telegraph competence and some the opposite -- and our experiences over the years tend to shape what we see that way.",
+                      "title": null,
+                      "type": "comment",
+                      "url": null
+                    }
+                  ],
                   "created_at": "2026-08-02T17:46:05.000Z",
                   "created_at_i": 1785692765,
                   "id": 49146607,
