--- reddit-same-address-collision-question@20260804T094305Z
+++ reddit-same-address-collision-question@20260804T134454Z
@@ -303,3 +303,19 @@
 edited: false
 body:
 Hash collisions don't require the exact same inputs. 
+
+comment: p1mo5q0
+parent: t1_p1jciij
+author: Icy_Giraffe_21
+created_utc: 1785842707
+edited: false
+body:
+RNG wasn't weak. It was never called upon during the creation of the seed. All of coldcards hardware is top notch and sound. They chose not to use it 
+
+comment: p1moo6t
+parent: t1_p1mo5q0
+author: skr_replicator
+created_utc: 1785842903
+edited: false
+body:
+Yea, but not being called at all effectively results in a very weak RNG. Some RNG was still even in that testing PRNG version that was being called instead, just far less of it. And the device ID was one source of entropy of that weak PRG, making it output at least not always the same thing. But some small space of 2\^40, together with a couple other weak sources.
