--- reddit-same-address-collision-question@20260804T161710Z
+++ reddit-same-address-collision-question@20260805T142620Z
@@ -288,21 +288,13 @@
 body:
 lol, a typo obviously. I wrote it correctly the previous sentence.
 
-comment: p1k5nrd
-parent: t1_p1i45ke
-author: OldHamburger7923
-created_utc: 1785804673
-edited: false
-body:
-There was at least two people posting about losing their funds off coldcard in the last few years. 
-
 comment: p1k5z39
 parent: t1_p1ica5d
-author: OldHamburger7923
+author: [deleted]
 created_utc: 1785804775
 edited: false
 body:
-Hash collisions don't require the exact same inputs. 
+[deleted]
 
 comment: p1mo5q0
 parent: t1_p1jciij
