--- bitcointalk-hack-revolution-opinion@20260804T165844Z
+++ bitcointalk-hack-revolution-opinion@20260804T182609Z
@@ -233,5 +233,9 @@
 P.S. Seed phrases provide as much security as a private key (128 bits).
 I think that sometimes conversations like this are more about recovery way and less about how wallets are created. Changing the backup format is not a real solution if creation process is damaged.
 More important lesson, to me, is that users should not take anything for granted, simply because it is popular. Separate checks, open code and proper inspections are just as important as the recovery process itself.
+Title: Re: The coldcard hack will change Bitcoin, it's a revolution.
+Post by: OgNasty on August 04, 2026, 06:22:09 PM
+I don't think the coldcard hack will have any effect on Bitcoin whatsoever.  Most people don't use hardware wallets and the ones that do don't use coldcards (they were ugly as sin).  This is unfortunately a hack that hurt a very small segment of the community and while they're making a lot of noise now, it will be completely forgotten about before the end of the year.
+The thing that I'm concerned about is the introduction to my vocabulary of the phrase 'retirement attack' which has me looking at things in a different light now.  I'm not sure I want access to anyone else's funds anymore.
 Powered by SMF 1.1.19 |
 SMF © 2006-2009, Simple Machines
