--- bitcointalk-hack-revolution-opinion@20260805T012821Z
+++ bitcointalk-hack-revolution-opinion@20260805T080802Z
@@ -273,5 +273,28 @@
 Some or many people will think if they choose and use a good wallets with enough dice rolls for 128 bits of entropy, they will no longer need to use a passphrase. This is similar to security things about risk of custodial wallets and weak passwords.
 Generally, there are few people will change while the majority will not change.
 That's a valid point. The majority of people came in through use of ledger, coldcard and other hardware devices through random ads/news and mostly bought into the hype and still hold their coins on these devices. I remember when i first owned Bitcoin i spent some good amount on a store bought Ledger Nano in 2014 and continued holding until 2021 when i delved deeper to understand exactly how Bitcoin works. I suspect many other still hold their coins on such devices.
+Title: Re: The coldcard hack will change Bitcoin, it's a revolution.
+Post by: bitbollo on August 05, 2026, 02:09:41 AM
+It would be impressive to see in long term another "similar" issuue that could involve a big thirdparty (it seems most of big companies are relying to in hardware for generating keys) or just another producer of address in a largescale.
+Even with any other big advancement (AI? Quantum? whatelse?) basically this should not change the math behind btc and the complexity to guess a private key in a space of 2^256.
+It can fall a "single player"- mtgox docet just to mention another bear event - but the underlying technology would always be unbreakable .
+the same blockchain that appears "weak" since has been "breaked" by hackers.... it's actually used by them for storage of the coins!
+there is a dicotomy in what happens... and despite a common user would think that selling is the best choice due "unsecurity", it's absolutely the opposite.
+And as a general rule, bitcoin is just making what has been designed for.
+I started to use this technology few months before 2014 and I can assure that any guidance or explanation would never requires to rely on third party hardware.
+Title: Re: The coldcard hack will change Bitcoin, it's a revolution.
+Post by: Ayers on August 05, 2026, 03:57:55 AM
+Quote from: Alphakilo on August 04, 2026, 09:39:42 PM
+This coldcard issue will remain a topic of fresh debate for a long time as a reminder and a lesson and a point to develop and upgrade upon because it is such incidents that makes Bitcoin standout and stronger and self custody more aware and more secure to use that traditional banking.
+as I see it, the coldcard incident should serve as a lesson and a wake up call for the entire hw industry, but I don't think it makes Bitcoin standout and stronger. on the contrary, this incident is making more people question hardware wallet and the potential risk of digital asset.
+this has nothing to do with bank. in fact, the incident is even shaking confidence in self custody. iam concerned that more and more investors will choose etf over self custody going forward.
+Quote from: Alphakilo on August 04, 2026, 09:39:42 PM
+I think I understand why airgapped wallet is way better for self custody that others and if this is an issue from the company, that is, if this coldcard issue is from the company, there would be some kind of compensation to customers who lost and sm sure sure this would make coldcard come back stronger or reinvent itself to suit self custody demands in the future.
+as far as I know, coinkite has publicly apologized and accepted responsibility, but it has not made any concrete commitment to provide financial compensation.
+Title: Re: The coldcard hack will change Bitcoin, it's a revolution.
+Post by: red4slash on August 05, 2026, 04:40:19 AM
+This can make everyone more vigilant should even with the frills of self custody or whatever it is in the end when there is a bug that can be infiltrated then the seed that is kept safe will not be released from hacking in the end.
+This is a bit of a concern, but on the other hand, realizing that the system is 100 percent secure is not entirely true. Although we know that security is getting more and more improved but on the one hand there will always be irresponsible people trying to damage this so we are still confident about the security of bitcoin but not with hardware or software that can appear at any time like this Coldcard case.
+In the end for now we should realize that self-custody is not entirely safe but this only minimizes the worst possibility of happening.
 Powered by SMF 1.1.19 |
 SMF © 2006-2009, Simple Machines
