--- bitcointalk-hack-revolution-opinion@20260805T002806Z
+++ bitcointalk-hack-revolution-opinion@20260805T012821Z
@@ -263,5 +263,15 @@
 This coldcard issue will remain a topic of fresh debate for a long time as a reminder and a lesson and a point to develop and upgrade upon because it is such incidents that makes Bitcoin standout and stronger and self custody more aware and more secure to use that traditional banking.
 Let them say, miners will continue to exist and more mining farms will be developed with better systems because we anticipate the emergence of quantum computers that has been a looming threat to the ecosystem.
 I think I understand why airgapped wallet is way better for self custody that others and if this is an issue from the company, that is, if this coldcard issue is from the company, there would be some kind of compensation to customers who lost and sm sure sure this would make coldcard come back stronger or reinvent itself to suit self custody demands in the future.
+Title: Re: The coldcard hack will change Bitcoin, it's a revolution.
+Post by: collardelay on August 05, 2026, 01:07:40 AM
+Quote from: MusaMohamed on August 04, 2026, 11:39:29 AM
+Quote from: savetheFORUM on August 04, 2026, 08:03:31 AM
+Whether it brings a revolution or not, but one thing is for sure, for the time being, people won't be able to trust hardware wallet manufacturers, and many will probably stop using them for the short run. Some will start renewing their wallets and seed phrases and apply passphrases which should have been done at the beginning, but others might stop using hardware wallets completely for the time being.
+Before anything changed technically, people will still use what are available now. What can be changed even immediately is creating a new wallet, with a passphrase, and moving their bitcoins from current wallets without passphrase to a new wallet with passphrase.
+Not all people will change their practice, and it's similar to how they still ignore many security advice, and continue using custodial wallets, centralized exchanges, even with weak wallet passwords, weak account passwords.
+Some or many people will think if they choose and use a good wallets with enough dice rolls for 128 bits of entropy, they will no longer need to use a passphrase. This is similar to security things about risk of custodial wallets and weak passwords.
+Generally, there are few people will change while the majority will not change.
+That's a valid point. The majority of people came in through use of ledger, coldcard and other hardware devices through random ads/news and mostly bought into the hype and still hold their coins on these devices. I remember when i first owned Bitcoin i spent some good amount on a store bought Ledger Nano in 2014 and continued holding until 2021 when i delved deeper to understand exactly how Bitcoin works. I suspect many other still hold their coins on such devices.
 Powered by SMF 1.1.19 |
 SMF © 2006-2009, Simple Machines
