--- bitcointalk-hack-revolution-opinion@20260804T114456Z
+++ bitcointalk-hack-revolution-opinion@20260804T131619Z
@@ -214,5 +214,16 @@
 No it won't.
 It's like centralized exchanges. As long as they've existed, which is pretty much from the second year I think, they have been getting hacked. Some of them massive hacks (eg. Mt Gox) and yet people are still using them to this day instead of moving on to decentralized exchanges.
 P.S. Seed phrases provide as much security as a private key (128 bits).
+Title: Re: The coldcard hack will change Bitcoin, it's a revolution.
+Post by: ultrloa on August 04, 2026, 12:46:05 PM
+Quote from: pooya87 on Today at 11:41:50 AM
+Quote from: MicroGuy on August 03, 2026, 08:18:38 PM
+The Coldcard hack will change the way people store funds. Unhackable private keys instead of silly seeds.
+No it won't.
+It's like centralized exchanges. As long as they've existed, which is pretty much from the second year I think, they have been getting hacked. Some of them massive hacks (eg. Mt Gox) and yet people are still using them to this day instead of moving on to decentralized exchanges.
+P.S. Seed phrases provide as much security as a private key (128 bits).
+Those hacks happen for many times and its sad to see that despite of those big hacking issue many people still choose convenience over security.
+Lots of newbies still trust those CEX for thinking that they only put the amount which they can afford to lose, also they are afraid to lose their keys. But regret later on those decision they have done once hacking issues exist and their balances got wiped out.
+Seed phrase indeed  secure as those private keys they have, so with all of those things the real solution is to have good education about this matter and not having those excuse to justify their laziness.
 Powered by SMF 1.1.19 |
 SMF © 2006-2009, Simple Machines
