--- bitcointalk-hack-revolution-opinion@20260804T131619Z
+++ bitcointalk-hack-revolution-opinion@20260804T165844Z
@@ -225,5 +225,13 @@
 Those hacks happen for many times and its sad to see that despite of those big hacking issue many people still choose convenience over security.
 Lots of newbies still trust those CEX for thinking that they only put the amount which they can afford to lose, also they are afraid to lose their keys. But regret later on those decision they have done once hacking issues exist and their balances got wiped out.
 Seed phrase indeed  secure as those private keys they have, so with all of those things the real solution is to have good education about this matter and not having those excuse to justify their laziness.
+Title: Re: The coldcard hack will change Bitcoin, it's a revolution.
+Post by: Numan467 on August 04, 2026, 04:46:42 PM
+Quote from: pooya87 on Today at 11:41:50 AM
+No it won't.
+It's like centralized exchanges. As long as they've existed, which is pretty much from the second year I think, they have been getting hacked. Some of them massive hacks (eg. Mt Gox) and yet people are still using them to this day instead of moving on to decentralized exchanges.
+P.S. Seed phrases provide as much security as a private key (128 bits).
+I think that sometimes conversations like this are more about recovery way and less about how wallets are created. Changing the backup format is not a real solution if creation process is damaged.
+More important lesson, to me, is that users should not take anything for granted, simply because it is popular. Separate checks, open code and proper inspections are just as important as the recovery process itself.
 Powered by SMF 1.1.19 |
 SMF © 2006-2009, Simple Machines
