--- coldcard-firmware-pr-713@20260813T125418Z
+++ coldcard-firmware-pr-713@20260813T192548Z
@@ -274,19 +274,7 @@
 the
 improve_seed_gen-mash-timing
 branch
-2 times, most recently
-from
-0fb57c0 to
-e6058db
-Compare
-August 13, 2026 11:45
-review: code dedup - save flash space
-69edb16
-scgbckbone
-force-pushed
-the
-improve_seed_gen-mash-timing
-branch
+3 times, most recently
 from
 e6058db to
 69edb16
@@ -321,6 +309,18 @@
 Sorry, something went wrong.
 Uh oh!
 There was an error while loading. Please reload this page.
+review: code dedup - save flash space
+c81f966
+scgbckbone
+force-pushed
+the
+improve_seed_gen-mash-timing
+branch
+from
+69edb16 to
+c81f966
+Compare
+August 13, 2026 14:15
 This file contains hidden or bidirectional Unicode text that may be interpreted or compiled differently than what appears below. To review, open the file in an editor that reveals hidden Unicode characters.
 Learn more about bidirectional Unicode characters
 Show hidden characters
