--- reddit-hardware-wallet-comparison@20260804T141531Z
+++ reddit-hardware-wallet-comparison@20260804T144443Z
@@ -766,9 +766,11 @@
 parent: t1_p1m0kyf
 author: OldHamburger7923
 created_utc: 1785833025
-edited: false
+edited: 1785853763
 body:
 So doesn't that mean we see the associated code and this this isn't closed source with hidden logic?
+
+Edit: people on reddit think if it isn't gpl it isn't open source. Open source doesn't require gpl.
 
 comment: p1m3b5t
 parent: t1_p1klmu6
@@ -1296,3 +1298,15 @@
 edited: false
 body:
 You can roll your own rng/seed for any wallet really, you just need to either do the math yourself or use any number of open source tools to convert the dice rolls into a seed and mnemonic. Id cross reference multiple tools and see if the output is the same before trusting. Iancoleman.io is a really cool site for stuff like this. Though I would download the page, then re-open it offline on pc with no storage or network (live boot debian, tails, etc). 
+
+Then restore the wallet in your hardware of choice using the mnemonic you generated.
+
+comment: p1nqm2u
+parent: t3_1vehzv8
+author: Dextradomis
+created_utc: 1785854423
+edited: false
+body:
+End of ColdCard -shrugs-
+
+End of Ledger -real shit???-
