--- reddit-hardware-wallet-comparison@20260804T221700Z
+++ reddit-hardware-wallet-comparison@20260804T224747Z
@@ -1359,6 +1359,14 @@
 body:
 I’m old enough to remember when Trezor and Ledger were universally understood to be the only safe options for cold storage. Fascinating to watch this wisdom somehow get lost over the years.
 
+comment: p1ojdpa
+parent: t3_1vehzv8
+author: CTchippy
+created_utc: 1785861797
+edited: false
+body:
+Now convinced more people will lose money to ponzi schemes (ftx, Celcius, etc.) or crytographic incompetence than they would've from inflation.. hard to recommend this asset to anyone not deeply technical in cyber security, cryptography, and/or software engineering.
+
 comment: p1oxvo5
 parent: t1_p1l8you
 author: SmallDodgyCamel
