--- reddit-hardware-wallet-comparison@20260804T164733Z
+++ reddit-hardware-wallet-comparison@20260804T174552Z
@@ -1358,3 +1358,19 @@
 edited: false
 body:
 I’m old enough to remember when Trezor and Ledger were universally understood to be the only safe options for cold storage. Fascinating to watch this wisdom somehow get lost over the years.
+
+comment: p1oxvo5
+parent: t1_p1l8you
+author: SmallDodgyCamel
+created_utc: 1785865382
+edited: false
+body:
+Almost? You have some misgivings?
+
+comment: p1oy55i
+parent: t3_1vehzv8
+author: Saddath
+created_utc: 1785865445
+edited: false
+body:
+Bitbox missing?
