--- reddit-hardware-wallet-comparison@20260804T174552Z
+++ reddit-hardware-wallet-comparison@20260804T184623Z
@@ -1374,3 +1374,19 @@
 edited: false
 body:
 Bitbox missing?
+
+comment: p1pb81j
+parent: t3_1vehzv8
+author: Viking_13v
+created_utc: 1785868684
+edited: false
+body:
+What do you think of Bitbox 02 Nova?
+
+comment: p1pcxle
+parent: t1_p1l9rh0
+author: GettinWiggyWiddit
+created_utc: 1785869115
+edited: false
+body:
+Same. Self custody seems too risky now. I’ll trust the heavily audited giant public companies with huge anti fraud teams
