--- reddit-hardware-wallet-comparison@20260804T224747Z
+++ reddit-hardware-wallet-comparison@20260804T232248Z
@@ -591,6 +591,14 @@
 
 Trezor was the first company but Bitbox made the high standard of a super secure wallet whilest being open-source.
 
+comment: p1lplpt
+parent: t1_p1lb90a
+author: Objective-Run-9104
+created_utc: 1785826497
+edited: false
+body:
+All good and well but Ledger got hacked and doxxed every buyer
+
 comment: p1lpva0
 parent: t1_p1kydv5
 author: Objective-Run-9104
@@ -1037,6 +1045,22 @@
 body:
 Only for long term HOLDERS using ledger... If folks generate 25th Passphrase via rolling dice and keep that completely safe and secure elsewhere, engraved in a steel plate, and totally stay disconnected off ledger live or any software app, basically use seed phrase as cold paper wallet and view btc holdings on sparrow wallet should probably be a good approach? 
 
+comment: p1md0sw
+parent: t1_p1m6uqs
+author: Successful-Plenty-27
+created_utc: 1785838081
+edited: false
+body:
+The consensus for the past decade was that hardware wallets should be open source because then users can verify the code.
+
+Coldcard was "verifyable source", meaning they used an open source repository (from trezor), but the things they edited were public but not to be used by other commercial products.
+
+Well, i guess nobody bothered to actually verify the verifyable source, or coinkite straight up fooled them.
+
+Ledger allegedly is not safe because the source can't be verified.
+
+I think for a lot of the open source purists this incident with cold card was a hard reality check.
+
 comment: p1md6v4
 parent: t3_1vehzv8
 author: petitcurieux77
@@ -1060,6 +1084,23 @@
 edited: false
 body:
 This scares me and I think I’m switching to Trezor because of this
+
+comment: p1mf24l
+parent: t1_p1m8620
+author: Successful-Plenty-27
+created_utc: 1785838986
+edited: false
+body:
+I have used Ledger for years because i don't know shit about code, but they have an entire security team, the Ledger Donjon.
+I'm a bit surprised now, after the coldcard hack, you are still so fiercly advocating for open source and against Ledger.
+What is an open source worth if the majority of users is actually not literate enough to spot the issues?
+
+The choice here you have now is simple, would you choose between a company which invests in security and has an entire team trying to hack it's own products?
+Or is a repository dumped on GitHub by 1 guy enough to prove security?
+
+To be honest, the "open source is better" bubble got popped , and i know the majority of hardware wallets were following this momentum.
+But you were all wrong.
+How many of you "open source purists" can actually read and understand the code, and have you ever read it?
 
 comment: p1mgbyb
 parent: t1_p1m3b5t
