--- reddit-flag-four-addresses@20260804T145359Z
+++ reddit-flag-four-addresses@20260804T225759Z
@@ -137,3 +137,19 @@
 edited: false
 body:
 None of that is due to a bug in the bitcoin code.
+
+comment: p1qt1bk
+parent: t3_1verf1p
+author: RoyYourWorkingBoy
+created_utc: 1785883185
+edited: false
+body:
+They'll find a way to move the coins. But I think with enough money/interest the law will eventually find the hackers because of that spending. Blockchain surveillance is pretty incredible when they throw big resources into it.
+
+comment: p1qv32b
+parent: t1_p1qt1bk
+author: calos98x
+created_utc: 1785883807
+edited: false
+body:
+Same opinion
