--- reddit-security-advisory-relay@20260805T014614Z
+++ reddit-security-advisory-relay@20260809T165053Z
@@ -298,11 +298,11 @@
 
 comment: p0scr24
 parent: t1_p0s72yk
-author: 0fWhomIAmChief
+author: [deleted]
 created_utc: 1785460851
 edited: false
 body:
-Trezor models have three different sources of entropy compared to coldcards 1, the newest models have 4 each independent of one another for this exact scenario. We may as well just do what the OGs on Bitcointalk always said, 256 coin flips 🤣
+[removed]
 
 comment: p0sdq7w
 parent: t1_p0scr24
@@ -353,14 +353,6 @@
 >Or will this signal an attacker to brute force passphrases for that seed with crypto
 
 The attacker will not know if they get a hold of your seed if you are using an extended passphrase or not . Most people don't so the default assumption is not
-
-comment: p0sh0oh
-parent: t1_p0sdq7w
-author: 0fWhomIAmChief
-created_utc: 1785462265
-edited: false
-body:
-Exactly, if any one of the 3 or 4 sources of entropy provided were weak like Coldcards, the other sources provide and retain its 128 bit security,  so Trezors are unaffected by this scenario
 
 comment: p0si027
 parent: t3_1vb8taw
