--- reddit-security-advisory-relay@20260804T092525Z
+++ reddit-security-advisory-relay@20260805T014614Z
@@ -734,3 +734,13 @@
 edited: false
 body:
 The amount of schadenfreude the hackers are feeling right now is through the roof. Don't give them the satisfaction, they are reading everything and finding so much joy through your frustrations.
+
+comment: p1rp4a5
+parent: t1_p0si027
+author: Buttoshi
+created_utc: 1785893370
+edited: false
+body:
+I want to generate my own seed with dice/coins. How do I go about this? 
+
+I'm stuck on the part of generating the last word/checksum with a paper computer. 
