--- reddit-trezor-breach-coldcard-ptsd@20260814T234748Z
+++ reddit-trezor-breach-coldcard-ptsd@20260815T055950Z
@@ -328,3 +328,11 @@
 edited: false
 body:
 no because i live in a country where people punch tasty drippy holes in each other if they try something like that, and Im a dead eye. 
+
+comment: p3qz2ef
+parent: t3_1vnkt5d
+author: unstopablex15
+created_utc: 1786751489
+edited: false
+body:
+The thing is... those people's info that got leaked will definitely be targeted now via phishing attempts.  Seeing how vulnerable the human element is, there'll be a good bit of folks who will get compromised unfortunately.  My advice: Be vigilant, and on the lookout for sophisticated phishing attempts.  Also, maybe use another wallet.
