--- reddit-ceo-anthropic-hack-challenge@20260804T195539Z
+++ reddit-ceo-anthropic-hack-challenge@20260804T212559Z
@@ -3563,14 +3563,6 @@
 body:
 Anthropic could easily hack bitcoin wallets right ? The power of that machine is incredible when you consider its financial (albeit theft centric but still financial) power. This is a really exciting time to live in 
 
-comment: p1ikk7e
-parent: t1_p1c4aoi
-author: viperex
-created_utc: 1785787820
-edited: false
-body:
-At this point, I don't understand what "stealing an identity" means. It can't be just the information that's in a phonebook. Is it just the information you need to open a credit card? 
-
 comment: p1infb4
 parent: t3_1vdu0rl
 author: ChanceNecessary4172
@@ -4100,6 +4092,14 @@
 
 I mean technically it would be theft but then again I could really use 6 Million in BTC 💀
 
+comment: p1q8owv
+parent: t3_1vdu0rl
+author: HeavyBeing0_0
+created_utc: 1785877444
+edited: false
+body:
+Some rando’s gonna do it with their hive of Mac Studios running obliterated models
+
 more-stub: parent t1_p1gispf count 5
 
 more-stub: parent t1_p1g6cql count 2
@@ -4134,7 +4134,7 @@
 
 more-stub: parent t1_p1db38y count 3
 
-more-stub: parent t1_p1c4aoi count 5
+more-stub: parent t1_p1c4aoi count 6
 
 more-stub: parent t1_p1cuv5h count 1
 
