--- reddit-ceo-anthropic-hack-challenge@20260804T172448Z
+++ reddit-ceo-anthropic-hack-challenge@20260804T175452Z
@@ -3753,14 +3753,6 @@
 body:
 He is the ceo for BitGo... he put it in a BitGo wallet.  Its a publicity stunt that has obviously worked.... before today I didn't know who they were.
 
-comment: p1jfg8v
-parent: t1_p1c4aoi
-author: TaliskyeDram
-created_utc: 1785796352
-edited: false
-body:
-Experian was hacked years ago, all our info is out there already and we all just let it go by without a fight 
-
 comment: p1jgjij
 parent: t1_p1i4s6r
 author: Pick-Dapper
@@ -3885,14 +3877,6 @@
 body:
 He didn’t challenge Anthropic, he challenged everyone. 
 
-comment: p1k5x8k
-parent: t1_p1c4aoi
-author: Traditional_Yard2741
-created_utc: 1785804758
-edited: false
-body:
-What is Experian? I keep getting emails from them that are intended for someone with a similar name to me, talking about a FICO score? I haven't bothered to check it out.
-
 comment: p1kabge
 parent: t3_1vdu0rl
 author: BeastMode09-00
@@ -3977,14 +3961,6 @@
 Is a SSN meant to be secret? It seems like there's a ton of things you need a SSN to do so thousands of other people at minimum have access to it, every employer you've ever had must have it at minimum.
 Saying "I know my SSN" seems like no security at all.
 
-comment: p1l2cha
-parent: t1_p1c4aoi
-author: SeriesAo-Series
-created_utc: 1785816070
-edited: false
-body:
-Oh whaaaat? How do we protect ourselves then?
-
 comment: p1lkv9f
 parent: t3_1vdu0rl
 author: Scar3cr0w_
@@ -4097,13 +4073,30 @@
 body:
 It's all BS. Anthropic is doing that as a marketing stunt for their upcoming models, and Bitgo is doing that for their own marketing. It's just a show/marketing. 
 
-comment: p1ondah
-parent: t3_1vdu0rl
-author: btc500halving
-created_utc: 1785862786
-edited: false
-body:
-This is either the cheapest marketing campaign ever or the most expensive one.
+comment: p1othbp
+parent: t3_1vdu0rl
+author: Rough_Run_753
+created_utc: 1785864298
+edited: false
+body:
+Is this one piece?
+
+comment: p1oxxcw
+parent: t3_1vdu0rl
+author: RADITUDE__
+created_utc: 1785865393
+edited: false
+body:
+this is just gross to me imo, disgusting amount of money being played with
+
+comment: p1p0eh4
+parent: t1_p1dx6ey
+author: Andyy58
+created_utc: 1785865997
+edited: false
+body:
+well he's the ceo so...  
+[https://www.bitgo.com/mike-belshe/](https://www.bitgo.com/mike-belshe/)
 
 more-stub: parent t1_p1gispf count 5
 
@@ -4139,7 +4132,7 @@
 
 more-stub: parent t1_p1db38y count 3
 
-more-stub: parent t1_p1c4aoi count 1
+more-stub: parent t1_p1c4aoi count 4
 
 more-stub: parent t1_p1cuv5h count 1
 
