--- reddit-ceo-anthropic-hack-challenge@20260804T185559Z
+++ reddit-ceo-anthropic-hack-challenge@20260804T195539Z
@@ -3697,14 +3697,6 @@
 body:
 In this case, he is using it to your benefit in a way.  The irrationally exuberant rush for AI could very well crash the economy.  This is exposing that the emperor has no clothes.  While a recession will hurt the billionaires, it’s not the billionaires who will be losing their jobs.
 
-comment: p1j2yb7
-parent: t1_p1c4aoi
-author: theseriousman1
-created_utc: 1785792753
-edited: false
-body:
-How
-
 comment: p1j5e6v
 parent: t3_1vdu0rl
 author: Green_Material_3671
@@ -4098,6 +4090,16 @@
 well he's the ceo so...  
 [https://www.bitgo.com/mike-belshe/](https://www.bitgo.com/mike-belshe/)
 
+comment: p1piee2
+parent: t3_1vdu0rl
+author: JustASDBrainfog
+created_utc: 1785870504
+edited: false
+body:
+Am I going to make my LocalBridge try get access to the Wallet now 👀
+
+I mean technically it would be theft but then again I could really use 6 Million in BTC 💀
+
 more-stub: parent t1_p1gispf count 5
 
 more-stub: parent t1_p1g6cql count 2
@@ -4132,7 +4134,7 @@
 
 more-stub: parent t1_p1db38y count 3
 
-more-stub: parent t1_p1c4aoi count 4
+more-stub: parent t1_p1c4aoi count 5
 
 more-stub: parent t1_p1cuv5h count 1
 
@@ -4182,7 +4184,7 @@
 
 more-stub: parent t1_p1c2842 count 1
 
-more-stub: parent t1_p1dh7tp count 21
+more-stub: parent t1_p1dh7tp count 22
 
 more-stub: parent t1_p1clb2m count 12
 
