--- reddit-ceo-anthropic-hack-challenge@20260804T155837Z
+++ reddit-ceo-anthropic-hack-challenge@20260804T165651Z
@@ -3784,14 +3784,6 @@
 Don’t mess with Claude. 
 
 
-comment: p1jh4es
-parent: t1_p1c7niz
-author: mxby7e
-created_utc: 1785796863
-edited: false
-body:
-If it successfully hacks that wallet, that btc will be worth pennies within days
-
 comment: p1jhbzu
 parent: t1_p1c1h5f
 author: Consistent_Oil3428
@@ -4105,6 +4097,14 @@
 body:
 Great! Let me grab my popcorn and soda!
 
+comment: p1om7ao
+parent: t3_1vdu0rl
+author: Firm-Ad-2446
+created_utc: 1785862495
+edited: false
+body:
+It's all BS. Anthropic is doing that as a marketing stunt for their upcoming models, and Bitgo is doing that for their own marketing. It's just a show/marketing. 
+
 more-stub: parent t1_p1gispf count 5
 
 more-stub: parent t1_p1g6cql count 2
@@ -4123,7 +4123,7 @@
 
 more-stub: parent t1_p1cmdvx count 1
 
-more-stub: parent t1_p1cusv9 count 2
+more-stub: parent t1_p1cusv9 count 3
 
 more-stub: parent t1_p1clp2u count 1
 
@@ -4269,6 +4269,8 @@
 
 more-stub: parent t1_p1c2842 count 1
 
+more-stub: parent t1_p1c7niz count 1
+
 more-stub: parent t1_p1ctqfc count 2
 
 more-stub: parent t1_p1ek85w count 1
