--- a_ferron-2085731252657696911@20260815T064003Z
+++ a_ferron-2085731252657696911@20260815T130843Z
@@ -33,6 +33,15 @@
 media: 0
 body:
 A last word about the exfil protection. I see it as a complex mathematical toy that helps only to avoid very specific issues, which are rarely used by malicious actors. It adds more complexity than real security. As the author of this security mechanism wrote : "that still wouldn't stop a malicious wallet from stealing from you because it can basically just do whatever it wants." And without a SE, you have no guarantee what your hw wallet is doing.
+
+post: 2077762460157681847
+role: reply
+author: TakeProfitLLC
+name: TakeProfitTrader
+created: 
+media: 0
+body:
+There's a Trading Path Most People Don't Know Exists
 
 post: 2085835023240888816
 role: reply
@@ -101,23 +110,3 @@
 I think 
 @BTChip
  disagrees, so do I.
-
-post: 2087901443462431225
-role: reply
-author: digijunkyard
-name: Digital Junkyard
-created: 
-media: 0
-body:
-FERRARI TIER DOMAIN SALE
-
-180 aged / brandable domains.
- 
-$299 each. Last day before they're pulled.
-
-What's under the hood?
-
-• e-wills com - 2003 · GD List $3,878
-• megabookstore com - 2005 · GD List $4,144
-• yunzl com - LLLLL · 2012 · GD List $4,072
-• racehorsesale com -
