--- stackernews-exchange-loss-comparison@20260807T170107Z
+++ stackernews-exchange-loss-comparison@20260807T193449Z
@@ -11,17 +11,17 @@
             }
           },
           {
+            "createdAt": "2026-08-07T13:04:55.195Z",
+            "text": "Puts things in perspective",
+            "user": {
+              "name": "BlokchainB"
+            }
+          },
+          {
             "createdAt": "2026-08-07T15:09:24.179Z",
             "text": "I thought FTX didn't have any Bitcoin.\n\nPoint made though. That's why the market didn't budge. Maximalists are a minority in Bitcoin.",
             "user": {
               "name": "OT"
-            }
-          },
-          {
-            "createdAt": "2026-08-07T13:04:55.195Z",
-            "text": "Puts things in perspective",
-            "user": {
-              "name": "BlokchainB"
             }
           },
           {
