--- reddit-passphrase-extra-security@20260807T061849Z
+++ reddit-passphrase-extra-security@20260809T103647Z
@@ -60,14 +60,6 @@
 
 If it's 26 full random characters (ex. 4#f5q$*2P@![/1iv0$3'7:Az?) then this shit is literally impregnable. Stronger than a standard BIP39 12 word seed-phrase. 
 
-comment: p1fm0dp
-parent: t3_1ve8xf0
-author: Keefryan
-created_utc: 1785759450
-edited: false
-body:
-OP.   Try this.   https://diceware.rempe.us/
-
 comment: p1fo5fz
 parent: t3_1ve8xf0
 author: n8dahwgg
