--- reddit-saved-my-stack-thanks@20260804T083706Z
+++ reddit-saved-my-stack-thanks@20260804T225608Z
@@ -162,3 +162,11 @@
 edited: false
 body:
 I wonder if the attacker chose this week because so many people are on vacation
+
+comment: p1qsbng
+parent: t3_1vdnbk3
+author: AntZealousideal3728
+created_utc: 1785882970
+edited: false
+body:
+I would also like to thank the moderator on this subreddit who personally helped cancel my order on 2 additional cold cards before it shipped. With all they got going on I was sure after 2 days of processing they’d be sending it out and I’d be stuck with it. 
