--- trezor-2084975929948766645@20260806T164635Z
+++ trezor-2084975929948766645@20260807T081558Z
@@ -694,15 +694,6 @@
 body:
 Hi, as long as at least one source is good, it remains at full strength.
 
-post: 2085319175635943780
-role: reply
-author: CryptoJoeV1
-name: Bitcoin4all #BIP-110
-created: 2026-08-06T10:56:49Z
-media: 0
-body:
-General question. Why are some Trezors 12 word and some are 24 word seed phrases?
-
 post: 2085331051724120192
 role: reply
 author: segagz
@@ -761,4 +752,40 @@
 body:
 Your thoughts on this?
 
+post: 2085451531290054890
+role: reply
+author: Aldo35100567
+name: @Aldo35100567
+created: 2026-08-06T19:42:45Z
+media: 0
+body:
+Please increase your passhrase limit
+
+post: 2085583896658194746
+role: reply
+author: VenecoInversor
+name: Veneco Inversor
+created: 2026-08-07T04:28:43Z
+media: 1
+body:
+
+
+post: 2085588076919943328
+role: reply
+author: jdkeeolil
+name: jdkeeo
+created: 2026-08-07T04:45:20Z
+media: 0
+body:
+Hello, could you add Chinese language support to the hardware wallet and hardware display?
+
+post: 2085640217893978235
+role: reply
+author: Trezor
+name: Trezor
+created: 2026-08-07T08:12:31Z
+media: 0
+body:
+Hi, at the moment, we have Chinese as a language in the Trezor Suite app. For the device (i.e., firmware), it's a bit more challenging, but thanks for your feedback. We'll pass it along to our product team.
+
 gap: 'Show more' control present, not expanded
