--- trezor-2084975929948766645@20260815T063349Z
+++ trezor-2084975929948766645@20260815T130249Z
@@ -881,3 +881,12 @@
 media: 0
 body:
 Why does the firmware fingerprint show a different value than what’s in the releases.json when updating through Trezor suite? Makes it hard to trust anything. I don’t use the wallet because of it.
+
+post: 2088604086417416499
+role: reply
+author: MarkBeswick20
+name: Mark Beswick
+created: 2026-08-15T12:29:53Z
+media: 0
+body:
+Is it possible to move btc from a normal Trezor wallet ie seed phrase to a Trezor wallet with a pass phrase directly or do you need to transfer via an exchange?
