--- coindesk-25-minute-sweep@20260804T204034Z
+++ coindesk-25-minute-sweep@20260804T211058Z
@@ -45,35 +45,35 @@
 Bitcon traded above $64,000 in early Asian hours, with widespread drain appearing to have little impact on the market.
 Latest Crypto News
 1
+SpaceX tops Wall Street revenue forecast, posts $540 million loss on bitcoin holdings
+44 minutes ago
+2
 Open USD rattled Circle's stock, but its key backers still support USDC
-2 oras ang nakalipas
-2
+3 hours ago
+3
 Samsung is poised to become a dominant stablecoin distributor, analysts say
-3 oras ang nakalipas
-3
+3 hours ago
+4
 Clarity Act sits idle over Trump ethics question as Warren asks SEC to investigate him
-4 oras ang nakalipas
-4
+4 hours ago
+5
 Wells Fargo joins JPMorgan and Citi in the race to tokenize Wall Street’s settlement rails
-4 oras ang nakalipas
-5
+5 hours ago
+6
 Bitcoin’s $63,000 zone emerges as key battleground for buyers: Glassnode
-5 oras ang nakalipas
-6
+6 hours ago
+7
 Polymarket targets $20 billion valuation as competition heats up in prediction market sector
-6 oras ang nakalipas
-7
+6 hours ago
+8
 Intesa Sanpaolo slashed IBIT stake by 94% in 2Q, tripled ether ETF holding as crypto prices slumped
-6 oras ang nakalipas
-8
+6 hours ago
+9
 Dinari brings tokenized U.S. stocks to American investors as equity race heats up
-6 oras ang nakalipas
-9
+7 hours ago
+10
 As Clarity Act teeters, mystery group hammers away at crypto in Washington ads
-7 oras ang nakalipas
-10
-BitGo’s WBTC move pushes LayerZero-to-Chainlink tally near $15 billion
-7 oras ang nakalipas
+8 hours ago
 Latest Research
 The Evolution of the Crypto CEX Landscape: A Case Study on Binance
 The Evolution of the Crypto CEX Landscape: A Case Study on Binance
@@ -152,4 +152,4 @@
 EthicsPrivacyTerms of UseCookie SettingsDo Not Sell My InfoBug Bounty Program
 © 2026 CoinDesk, Inc.
 Disclosure & Polices: CoinDesk is an award-winning media outlet that covers the cryptocurrency industry. Its journalists abide by a strict set of editorial policies. CoinDesk has adopted a set of principles aimed at ensuring the integrity, editorial independence and freedom from bias of its publications. CoinDesk is part of Bullish (NYSE:BLSH), an institutionally focused global digital asset platform that provides market infrastructure and information services. Bullish owns and invests in digital asset businesses and digital assets and CoinDesk employees, including journalists, may receive Bullish equity-based compensation.
-CryptoCD20$1,751.31CD20 up 0.13 percent0.13%BTC$64,276.18BTC up 0.86 percent0.86%ETH$1,876.17ETH up 0.47 percent0.47%XRP$1.08XRP down 0.44 percent0.44%SOL$74.27SOL up 0.65 percent0.65%
+CryptoCD20$1,753.70CD20 up 0.47 percent0.47%BTC$64,264.53BTC up 1.13 percent1.13%ETH$1,873.57ETH up 0.60 percent0.60%XRP$1.08XRP down 0.18 percent0.18%SOL$74.10SOL up 0.62 percent0.62%
