--- coindesk-25-minute-sweep@20260803T211543Z
+++ coindesk-25-minute-sweep@20260803T214635Z
@@ -46,10 +46,10 @@
 Latest Crypto News
 1
 Trump-linked American Bitcoin president Matt Prusak departs for Giga Energy
-5 hours ago
+6 hours ago
 2
 U.S.-Japan intervention revives yen carry trade fears for bitcoin
-6 hours ago
+7 hours ago
 3
 Circle slides after Morgan Stanley downgrade, cut in price target
 7 hours ago
@@ -58,19 +58,19 @@
 7 hours ago
 5
 BlackRock expands tokenized cash with new blockchain-based money market offerings
-7 hours ago
+8 hours ago
 6
 Tom Lee's Bitmine bought more ether, added to stock buyback last week
 8 hours ago
 7
 Strategy sold another $105 million of bitcoin last week, repurchased $81.2 million of STRC
-8 hours ago
+9 hours ago
 8
 Bitget set to exit Japan, close all remaining positions by year-end
 9 hours ago
 9
 Solo Bitcoin miner nets $200,000 as Coldcard hardware wallet drains rocks sentiment
-9 hours ago
+10 hours ago
 10
 Bitcoin, ether decline as Coldcard exploit enters a fifth day
 10 hours ago
@@ -152,4 +152,4 @@
 EthicsPrivacyTerms of UseCookie SettingsDo Not Sell My InfoBug Bounty Program
 © 2026 CoinDesk, Inc.
 Disclosure & Polices: CoinDesk is an award-winning media outlet that covers the cryptocurrency industry. Its journalists abide by a strict set of editorial policies. CoinDesk has adopted a set of principles aimed at ensuring the integrity, editorial independence and freedom from bias of its publications. CoinDesk is part of Bullish (NYSE:BLSH), an institutionally focused global digital asset platform that provides market infrastructure and information services. Bullish owns and invests in digital asset businesses and digital assets and CoinDesk employees, including journalists, may receive Bullish equity-based compensation.
-CryptoCD20$1,746.27CD20 down 0.25 percent0.25%BTC$63,665.68BTC up 0.21 percent0.21%ETH$1,864.38ETH down 1.28 percent1.28%XRP$1.08XRP down 0.49 percent0.49%SOL$73.71SOL down 0.28 percent0.28%
+CryptoCD20$1,745.76CD20 down 0.27 percent0.27%BTC$63,616.29BTC up 0.13 percent0.13%ETH$1,866.24ETH down 1.19 percent1.19%XRP$1.08XRP down 0.46 percent0.46%SOL$73.71SOL down 0.28 percent0.28%
