--- coindesk-25-minute-sweep@20260803T023307Z
+++ coindesk-25-minute-sweep@20260803T030437Z
@@ -49,16 +49,16 @@
 8 hours ago
 2
 Why a DeFi platform ditched its consumer app to become the secret backend for tech giants
-9 hours ago
+10 hours ago
 3
 The future of crypto payments won't include on-ramps or bridges, Fun CEO says
-10 hours ago
+11 hours ago
 4
 The reverse bridge: Crypto meets Wall Street using perps
-13 hours ago
+14 hours ago
 5
 Unlike the FTX collapse, the $89 million Coldcard exploit has investors sending bitcoin back to exchanges
-14 hours ago
+15 hours ago
 6
 Strategy holds STRC dividend at 12%
 Aug 1, 2026
@@ -152,4 +152,4 @@
 EthicsPrivacyTerms of UseCookie SettingsDo Not Sell My InfoBug Bounty Program
 © 2026 CoinDesk, Inc.
 Disclosure & Polices: CoinDesk is an award-winning media outlet that covers the cryptocurrency industry. Its journalists abide by a strict set of editorial policies. CoinDesk has adopted a set of principles aimed at ensuring the integrity, editorial independence and freedom from bias of its publications. CoinDesk is part of Bullish (NYSE:BLSH), an institutionally focused global digital asset platform that provides market infrastructure and information services. Bullish owns and invests in digital asset businesses and digital assets and CoinDesk employees, including journalists, may receive Bullish equity-based compensation.
-CryptoCD20$1,733.42CD20 up 0.035 percent0.035%BTC$63,061.04BTC down 0.52 percent0.52%ETH$1,864.34ETH down 0.57 percent0.57%XRP$1.07XRP down 0.14 percent0.14%SOL$72.94SOL down 0.21 percent0.21%
+CryptoCD20$1,732.61CD20 down 0.20 percent0.20%BTC$63,098.44BTC down 0.48 percent0.48%ETH$1,862.85ETH down 0.69 percent0.69%XRP$1.07XRP down 0.34 percent0.34%SOL$73.02SOL down 0.17 percent0.17%
