--- coindesk-25-minute-sweep@20260804T164101Z
+++ coindesk-25-minute-sweep@20260804T170903Z
@@ -45,34 +45,34 @@
 Bitcon traded above $64,000 in early Asian hours, with widespread drain appearing to have little impact on the market.
 Latest Crypto News
 1
+Clarity Act sits idle over Trump ethics question as Warren asks SEC to investigate him
+47 minutes ago
+2
 Wells Fargo joins JPMorgan and Citi in the race to tokenize Wall Street’s settlement rails
-53 minutes ago
-2
-Bitcoin’s $63,000 zone emerges as key battleground for buyers: Glassnode
 1 hour ago
 3
+Bitcoin’s $63,000 zone emerges as key battleground for buyers: Glassnode
+2 hours ago
+4
 Polymarket targets $20 billion valuation as competition heats up in prediction market sector
 2 hours ago
-4
+5
 Intesa Sanpaolo slashed IBIT stake by 94% in 2Q, tripled ether ETF holding as crypto prices slumped
 2 hours ago
-5
+6
 Dinari brings tokenized U.S. stocks to American investors as equity race heats up
-2 hours ago
-6
-As Clarity Act teeters, mystery group hammers away at crypto in Washington ads
 3 hours ago
 7
+As Clarity Act teeters, mystery group hammers away at crypto in Washington ads
+4 hours ago
+8
 BitGo’s WBTC move pushes LayerZero-to-Chainlink tally near $15 billion
-3 hours ago
-8
+4 hours ago
+9
 BNY to add crypto staking to digital asset custody platform
 4 hours ago
-9
+10
 Coldcard urges users to move bitcoin as exploit is still in progress
-4 hours ago
-10
-The bitcoin market has plenty of reasons to freak out, yet calm pervades
 5 hours ago
 Latest Research
 The Evolution of the Crypto CEX Landscape: A Case Study on Binance
@@ -152,4 +152,4 @@
 EthicsPrivacyTerms of UseCookie SettingsDo Not Sell My InfoBug Bounty Program
 © 2026 CoinDesk, Inc.
 Disclosure & Polices: CoinDesk is an award-winning media outlet that covers the cryptocurrency industry. Its journalists abide by a strict set of editorial policies. CoinDesk has adopted a set of principles aimed at ensuring the integrity, editorial independence and freedom from bias of its publications. CoinDesk is part of Bullish (NYSE:BLSH), an institutionally focused global digital asset platform that provides market infrastructure and information services. Bullish owns and invests in digital asset businesses and digital assets and CoinDesk employees, including journalists, may receive Bullish equity-based compensation.
-CryptoCD20$1,748.68CD20 up 0.11 percent0.11%BTC$63,919.78BTC up 0.32 percent0.32%ETH$1,867.45ETH up 0.15 percent0.15%XRP$1.07XRP down 0.56 percent0.56%SOL$73.71SOL up 0.51 percent0.51%
+CryptoCD20$1,749.49CD20 up 0.10 percent0.10%BTC$64,109.68BTC up 0.41 percent0.41%ETH$1,873.92ETH up 0.37 percent0.37%XRP$1.08XRP down 0.26 percent0.26%SOL$74.12SOL up 0.69 percent0.69%
