--- coindesk-25-minute-sweep@20260804T201048Z
+++ coindesk-25-minute-sweep@20260804T204034Z
@@ -46,34 +46,34 @@
 Latest Crypto News
 1
 Open USD rattled Circle's stock, but its key backers still support USDC
-2 hours ago
+2 oras ang nakalipas
 2
 Samsung is poised to become a dominant stablecoin distributor, analysts say
-2 hours ago
+3 oras ang nakalipas
 3
 Clarity Act sits idle over Trump ethics question as Warren asks SEC to investigate him
-3 hours ago
+4 oras ang nakalipas
 4
 Wells Fargo joins JPMorgan and Citi in the race to tokenize Wall Street’s settlement rails
-4 hours ago
+4 oras ang nakalipas
 5
 Bitcoin’s $63,000 zone emerges as key battleground for buyers: Glassnode
-5 hours ago
+5 oras ang nakalipas
 6
 Polymarket targets $20 billion valuation as competition heats up in prediction market sector
-5 hours ago
+6 oras ang nakalipas
 7
 Intesa Sanpaolo slashed IBIT stake by 94% in 2Q, tripled ether ETF holding as crypto prices slumped
-5 hours ago
+6 oras ang nakalipas
 8
 Dinari brings tokenized U.S. stocks to American investors as equity race heats up
-6 hours ago
+6 oras ang nakalipas
 9
 As Clarity Act teeters, mystery group hammers away at crypto in Washington ads
-7 hours ago
+7 oras ang nakalipas
 10
 BitGo’s WBTC move pushes LayerZero-to-Chainlink tally near $15 billion
-7 hours ago
+7 oras ang nakalipas
 Latest Research
 The Evolution of the Crypto CEX Landscape: A Case Study on Binance
 The Evolution of the Crypto CEX Landscape: A Case Study on Binance
@@ -152,4 +152,4 @@
 EthicsPrivacyTerms of UseCookie SettingsDo Not Sell My InfoBug Bounty Program
 © 2026 CoinDesk, Inc.
 Disclosure & Polices: CoinDesk is an award-winning media outlet that covers the cryptocurrency industry. Its journalists abide by a strict set of editorial policies. CoinDesk has adopted a set of principles aimed at ensuring the integrity, editorial independence and freedom from bias of its publications. CoinDesk is part of Bullish (NYSE:BLSH), an institutionally focused global digital asset platform that provides market infrastructure and information services. Bullish owns and invests in digital asset businesses and digital assets and CoinDesk employees, including journalists, may receive Bullish equity-based compensation.
-CryptoCD20$1,752.55CD20 up 0.20 percent0.20%BTC$64,128.81BTC up 0.63 percent0.63%ETH$1,872.67ETH up 0.28 percent0.28%XRP$1.08XRP down 0.35 percent0.35%SOL$74.11SOL up 0.44 percent0.44%
+CryptoCD20$1,751.31CD20 up 0.13 percent0.13%BTC$64,276.18BTC up 0.86 percent0.86%ETH$1,876.17ETH up 0.47 percent0.47%XRP$1.08XRP down 0.44 percent0.44%SOL$74.27SOL up 0.65 percent0.65%
