--- coindesk-25-minute-sweep@20260803T204633Z
+++ coindesk-25-minute-sweep@20260803T211543Z
@@ -52,28 +52,28 @@
 6 hours ago
 3
 Circle slides after Morgan Stanley downgrade, cut in price target
-6 hours ago
+7 hours ago
 4
 Bernstein sees another leg lower for crypto markets if Clarity Act stalls
-6 hours ago
+7 hours ago
 5
 BlackRock expands tokenized cash with new blockchain-based money market offerings
 7 hours ago
 6
 Tom Lee's Bitmine bought more ether, added to stock buyback last week
-7 hours ago
+8 hours ago
 7
 Strategy sold another $105 million of bitcoin last week, repurchased $81.2 million of STRC
 8 hours ago
 8
 Bitget set to exit Japan, close all remaining positions by year-end
-8 hours ago
+9 hours ago
 9
 Solo Bitcoin miner nets $200,000 as Coldcard hardware wallet drains rocks sentiment
 9 hours ago
 10
 Bitcoin, ether decline as Coldcard exploit enters a fifth day
-9 hours ago
+10 hours ago
 Latest Research
 The Evolution of the Crypto CEX Landscape: A Case Study on Binance
 The Evolution of the Crypto CEX Landscape: A Case Study on Binance
@@ -152,4 +152,4 @@
 EthicsPrivacyTerms of UseCookie SettingsDo Not Sell My InfoBug Bounty Program
 © 2026 CoinDesk, Inc.
 Disclosure & Polices: CoinDesk is an award-winning media outlet that covers the cryptocurrency industry. Its journalists abide by a strict set of editorial policies. CoinDesk has adopted a set of principles aimed at ensuring the integrity, editorial independence and freedom from bias of its publications. CoinDesk is part of Bullish (NYSE:BLSH), an institutionally focused global digital asset platform that provides market infrastructure and information services. Bullish owns and invests in digital asset businesses and digital assets and CoinDesk employees, including journalists, may receive Bullish equity-based compensation.
-CryptoCD20$1,749.45CD20 up 0.25 percent0.25%BTC$63,849.85BTC up 0.64 percent0.64%ETH$1,867.99ETH down 0.76 percent0.76%XRP$1.08XRP up 0.0₃94 percent0.0₃94%SOL$73.87SOL up 0.32 percent0.32%
+CryptoCD20$1,746.27CD20 down 0.25 percent0.25%BTC$63,665.68BTC up 0.21 percent0.21%ETH$1,864.38ETH down 1.28 percent1.28%XRP$1.08XRP down 0.49 percent0.49%SOL$73.71SOL down 0.28 percent0.28%
