--- coindesk-25-minute-sweep@20260804T001627Z
+++ coindesk-25-minute-sweep@20260804T004712Z
@@ -46,13 +46,13 @@
 Latest Crypto News
 1
 U.S. FBI intelligence agent arrested in connection with theft of $1 million in crypto
-1 hour ago
+2 hours ago
 2
 Trump-linked American Bitcoin president Matt Prusak departs for Giga Energy
-8 hours ago
+9 hours ago
 3
 U.S.-Japan intervention revives yen carry trade fears for bitcoin
-9 hours ago
+10 hours ago
 4
 Circle slides after Morgan Stanley downgrade, cut in price target
 10 hours ago
@@ -67,13 +67,13 @@
 11 hours ago
 8
 Strategy sold another $105 million of bitcoin last week, repurchased $81.2 million of STRC
-11 hours ago
+12 hours ago
 9
 Bitget set to exit Japan, close all remaining positions by year-end
 12 hours ago
 10
 Solo Bitcoin miner nets $200,000 as Coldcard hardware wallet drains rocks sentiment
-12 hours ago
+13 hours ago
 Latest Research
 The Evolution of the Crypto CEX Landscape: A Case Study on Binance
 The Evolution of the Crypto CEX Landscape: A Case Study on Binance
@@ -152,4 +152,4 @@
 EthicsPrivacyTerms of UseCookie SettingsDo Not Sell My InfoBug Bounty Program
 © 2026 CoinDesk, Inc.
 Disclosure & Polices: CoinDesk is an award-winning media outlet that covers the cryptocurrency industry. Its journalists abide by a strict set of editorial policies. CoinDesk has adopted a set of principles aimed at ensuring the integrity, editorial independence and freedom from bias of its publications. CoinDesk is part of Bullish (NYSE:BLSH), an institutionally focused global digital asset platform that provides market infrastructure and information services. Bullish owns and invests in digital asset businesses and digital assets and CoinDesk employees, including journalists, may receive Bullish equity-based compensation.
-CryptoCD20$1,740.10CD20 down 0.039 percent0.039%BTC$63,394.16BTC up 0.21 percent0.21%ETH$1,856.89ETH down 0.83 percent0.83%XRP$1.07XRP down 0.59 percent0.59%SOL$73.41SOL up 0.32 percent0.32%
+CryptoCD20$1,737.50CD20 down 0.18 percent0.18%BTC$63,319.20BTC up 0.095 percent0.095%ETH$1,849.70ETH down 1.22 percent1.22%XRP$1.07XRP down 0.95 percent0.95%SOL$73.10SOL down 0.095 percent0.095%
