--- coindesk-25-minute-sweep@20260803T110844Z
+++ coindesk-25-minute-sweep@20260803T114331Z
@@ -45,35 +45,35 @@
 Bitcon traded above $64,000 in early Asian hours, with widespread drain appearing to have little impact on the market.
 Latest Crypto News
 1
+Solo Bitcoin miner nets $200,000 as Coldcard hardware wallet drains rocks sentiment
+25 minutes ago
+2
 Bitcoin, ether decline as Coldcard exploit enters a fifth day
-17 minutes ago
-2
+51 minutes ago
+3
 Bithumb now targets 2028 IP after major internal restructuring
-27 minutes ago
-3
+1 hour ago
+4
 Live updates: Traders say bitcoin sell-off from $65,000 points to thin volume, not panic selling
-39 minutes ago
-4
+1 hour ago
+5
 The bitcoin futures yield collapse: Once over 20%, now less than Treasury notes
-54 minutes ago
-5
-Robinhood wins UK crypto registration before new regulatory regime kicks off
 1 hour ago
 6
+Robinhood wins UK crypto registration before new regulatory regime kicks off
+2 hours ago
+7
 Coldcard wallet losses may near $114 million as possible fourth sweep emerges
 2 hours ago
-7
+8
 U.S. Jobs, Circle, Galaxy, American Bitcoin earnings: Crypto Week Ahead
-2 hours ago
-8
+3 hours ago
+9
 Bitcoin slips under $63,000 despite Iran deal hopes as Coldcard losses rattle market
-4 hours ago
-9
+5 hours ago
+10
 Trump Media’s bitcoin stash may be down to loan collateral after $165 million BTC move
 5 hours ago
-10
-Michael Saylor’s Strategy is now tracking bitcoin’s 200-week moving average
-6 hours ago
 Latest Research
 The Evolution of the Crypto CEX Landscape: A Case Study on Binance
 The Evolution of the Crypto CEX Landscape: A Case Study on Binance
@@ -152,4 +152,4 @@
 EthicsPrivacyTerms of UseCookie SettingsDo Not Sell My InfoBug Bounty Program
 © 2026 CoinDesk, Inc.
 Disclosure & Polices: CoinDesk is an award-winning media outlet that covers the cryptocurrency industry. Its journalists abide by a strict set of editorial policies. CoinDesk has adopted a set of principles aimed at ensuring the integrity, editorial independence and freedom from bias of its publications. CoinDesk is part of Bullish (NYSE:BLSH), an institutionally focused global digital asset platform that provides market infrastructure and information services. Bullish owns and invests in digital asset businesses and digital assets and CoinDesk employees, including journalists, may receive Bullish equity-based compensation.
-CryptoCD20$1,722.51CD20 down 0.60 percent0.60%BTC$62,557.55BTC down 0.67 percent0.67%ETH$1,838.70ETH down 0.66 percent0.66%XRP$1.07XRP down 0.99 percent0.99%SOL$72.39SOL down 0.74 percent0.74%
+CryptoCD20$1,722.39CD20 down 0.61 percent0.61%BTC$62,626.96BTC down 0.56 percent0.56%ETH$1,841.82ETH down 0.49 percent0.49%XRP$1.07XRP down 1.13 percent1.13%SOL$72.48SOL down 0.62 percent0.62%
