--- coindesk-25-minute-sweep@20260804T211058Z
+++ coindesk-25-minute-sweep@20260804T214044Z
@@ -46,16 +46,16 @@
 Latest Crypto News
 1
 SpaceX tops Wall Street revenue forecast, posts $540 million loss on bitcoin holdings
-44 minutes ago
+1 hour ago
 2
 Open USD rattled Circle's stock, but its key backers still support USDC
 3 hours ago
 3
 Samsung is poised to become a dominant stablecoin distributor, analysts say
-3 hours ago
+4 hours ago
 4
 Clarity Act sits idle over Trump ethics question as Warren asks SEC to investigate him
-4 hours ago
+5 hours ago
 5
 Wells Fargo joins JPMorgan and Citi in the race to tokenize Wall Street’s settlement rails
 5 hours ago
@@ -64,10 +64,10 @@
 6 hours ago
 7
 Polymarket targets $20 billion valuation as competition heats up in prediction market sector
-6 hours ago
+7 hours ago
 8
 Intesa Sanpaolo slashed IBIT stake by 94% in 2Q, tripled ether ETF holding as crypto prices slumped
-6 hours ago
+7 hours ago
 9
 Dinari brings tokenized U.S. stocks to American investors as equity race heats up
 7 hours ago
@@ -152,4 +152,4 @@
 EthicsPrivacyTerms of UseCookie SettingsDo Not Sell My InfoBug Bounty Program
 © 2026 CoinDesk, Inc.
 Disclosure & Polices: CoinDesk is an award-winning media outlet that covers the cryptocurrency industry. Its journalists abide by a strict set of editorial policies. CoinDesk has adopted a set of principles aimed at ensuring the integrity, editorial independence and freedom from bias of its publications. CoinDesk is part of Bullish (NYSE:BLSH), an institutionally focused global digital asset platform that provides market infrastructure and information services. Bullish owns and invests in digital asset businesses and digital assets and CoinDesk employees, including journalists, may receive Bullish equity-based compensation.
-CryptoCD20$1,753.70CD20 up 0.47 percent0.47%BTC$64,264.53BTC up 1.13 percent1.13%ETH$1,873.57ETH up 0.60 percent0.60%XRP$1.08XRP down 0.18 percent0.18%SOL$74.10SOL up 0.62 percent0.62%
+CryptoCD20$1,752.86CD20 up 0.42 percent0.42%BTC$64,192.10BTC up 1.01 percent1.01%ETH$1,871.85ETH up 0.51 percent0.51%XRP$1.08XRP down 0.25 percent0.25%SOL$74.04SOL up 0.53 percent0.53%
