--- coindesk-25-minute-sweep@20260803T030437Z
+++ coindesk-25-minute-sweep@20260803T090756Z
@@ -45,34 +45,34 @@
 Bitcon traded above $64,000 in early Asian hours, with widespread drain appearing to have little impact on the market.
 Latest Crypto News
 1
+U.S. Jobs, Circle, Galaxy, American Bitcoin earnings: Crypto Week Ahead
+51 minutes ago
+2
+Bitcoin slips under $63,000 despite Iran deal hopes as Coldcard losses rattle market
+2 hours ago
+3
+Trump Media’s bitcoin stash may be down to loan collateral after $165 million BTC move
+3 hours ago
+4
+Michael Saylor’s Strategy is now tracking bitcoin’s 200-week moving average
+4 hours ago
+5
 Counting down the days: State of Crypto
-8 hours ago
-2
+14 hours ago
+6
 Why a DeFi platform ditched its consumer app to become the secret backend for tech giants
-10 hours ago
-3
+16 hours ago
+7
 The future of crypto payments won't include on-ramps or bridges, Fun CEO says
-11 hours ago
-4
+17 hours ago
+8
 The reverse bridge: Crypto meets Wall Street using perps
-14 hours ago
-5
+20 hours ago
+9
 Unlike the FTX collapse, the $89 million Coldcard exploit has investors sending bitcoin back to exchanges
-15 hours ago
-6
+21 hours ago
+10
 Strategy holds STRC dividend at 12%
-Aug 1, 2026
-7
-Bitcoin cold-wallet attack spreads to 4,500 addresses as losses near $89 million
-Aug 1, 2026
-8
-Tokenized stock trading surged 288% in July, but one QQQ token drove most of it
-Aug 1, 2026
-9
-Bank of Italy research suggests stablecoins aren't necessarily cheaper for remittances
-Aug 1, 2026
-10
-SEC to review Nasdaq bitcoin options approval after CME challenge
 Aug 1, 2026
 Latest Research
 The Evolution of the Crypto CEX Landscape: A Case Study on Binance
@@ -152,4 +152,4 @@
 EthicsPrivacyTerms of UseCookie SettingsDo Not Sell My InfoBug Bounty Program
 © 2026 CoinDesk, Inc.
 Disclosure & Polices: CoinDesk is an award-winning media outlet that covers the cryptocurrency industry. Its journalists abide by a strict set of editorial policies. CoinDesk has adopted a set of principles aimed at ensuring the integrity, editorial independence and freedom from bias of its publications. CoinDesk is part of Bullish (NYSE:BLSH), an institutionally focused global digital asset platform that provides market infrastructure and information services. Bullish owns and invests in digital asset businesses and digital assets and CoinDesk employees, including journalists, may receive Bullish equity-based compensation.
-CryptoCD20$1,732.61CD20 down 0.20 percent0.20%BTC$63,098.44BTC down 0.48 percent0.48%ETH$1,862.85ETH down 0.69 percent0.69%XRP$1.07XRP down 0.34 percent0.34%SOL$73.02SOL down 0.17 percent0.17%
+CryptoCD20$1,717.96CD20 down 1.10 percent1.10%BTC$62,493.33BTC down 1.16 percent1.16%ETH$1,838.41ETH down 1.65 percent1.65%XRP$1.07XRP down 1.45 percent1.45%SOL$72.28SOL down 1.42 percent1.42%
