--- coindesk-25-minute-sweep@20260804T184020Z
+++ coindesk-25-minute-sweep@20260804T190929Z
@@ -46,7 +46,7 @@
 Latest Crypto News
 1
 Open USD rattled Circle's stock, but its key backers still support USDC
-44 minutes ago
+1 hour ago
 2
 Samsung is poised to become a dominant stablecoin distributor, analysts say
 1 hour ago
@@ -55,10 +55,10 @@
 2 hours ago
 4
 Wells Fargo joins JPMorgan and Citi in the race to tokenize Wall Street’s settlement rails
-2 hours ago
+3 hours ago
 5
 Bitcoin’s $63,000 zone emerges as key battleground for buyers: Glassnode
-3 hours ago
+4 hours ago
 6
 Polymarket targets $20 billion valuation as competition heats up in prediction market sector
 4 hours ago
@@ -67,13 +67,13 @@
 4 hours ago
 8
 Dinari brings tokenized U.S. stocks to American investors as equity race heats up
-4 hours ago
+5 hours ago
 9
 As Clarity Act teeters, mystery group hammers away at crypto in Washington ads
-5 hours ago
+6 hours ago
 10
 BitGo’s WBTC move pushes LayerZero-to-Chainlink tally near $15 billion
-5 hours ago
+6 hours ago
 Latest Research
 The Evolution of the Crypto CEX Landscape: A Case Study on Binance
 The Evolution of the Crypto CEX Landscape: A Case Study on Binance
@@ -152,4 +152,4 @@
 EthicsPrivacyTerms of UseCookie SettingsDo Not Sell My InfoBug Bounty Program
 © 2026 CoinDesk, Inc.
 Disclosure & Polices: CoinDesk is an award-winning media outlet that covers the cryptocurrency industry. Its journalists abide by a strict set of editorial policies. CoinDesk has adopted a set of principles aimed at ensuring the integrity, editorial independence and freedom from bias of its publications. CoinDesk is part of Bullish (NYSE:BLSH), an institutionally focused global digital asset platform that provides market infrastructure and information services. Bullish owns and invests in digital asset businesses and digital assets and CoinDesk employees, including journalists, may receive Bullish equity-based compensation.
-CryptoCD20$1,753.50CD20 up 0.28 percent0.28%BTC$64,210.29BTC up 0.64 percent0.64%ETH$1,873.42ETH up 0.33 percent0.33%XRP$1.08XRP down 0.42 percent0.42%SOL$74.05SOL up 0.19 percent0.19%
+CryptoCD20$1,753.48CD20 up 0.22 percent0.22%BTC$64,281.46BTC up 0.75 percent0.75%ETH$1,874.31ETH up 0.32 percent0.32%XRP$1.08XRP down 0.29 percent0.29%SOL$74.10SOL up 0.19 percent0.19%
