--- coindesk-25-minute-sweep@20260804T133853Z
+++ coindesk-25-minute-sweep@20260804T140937Z
@@ -46,34 +46,34 @@
 Latest Crypto News
 1
 As Clarity Act teeters, mystery group hammers away at crypto in Washington ads
-38 minutes ago
+1 hour ago
 2
 BitGo’s WBTC move pushes LayerZero-to-Chainlink tally near $15 billion
-38 minutes ago
+1 hour ago
 3
 BNY to add crypto staking to digital asset custody platform
 1 hour ago
 4
 Coldcard urges users to move bitcoin as exploit is still in progress
-1 hour ago
+2 hours ago
 5
 The bitcoin market has plenty of reasons to freak out, yet calm pervades
 2 hours ago
 6
 South African lawmakers propose draft rules on cross-border crypto transactions
-2 hours ago
+3 hours ago
 7
 Bitcoin rises toward $64,000 as Coldcard exploit, Strategy sales recede. ADA advances
-2 hours ago
+3 hours ago
 8
 BlackRock debuts tokenized access to $311 billion of money market funds in Europe
-3 hours ago
+4 hours ago
 9
 First U.S. spot bitcoin ETF to close as inflows dwindle, investors chase AI returns
-3 hours ago
+4 hours ago
 10
 Live updates: Bitcoin stuck in tight range as risk assets rise on Iran deal hopes
-4 hours ago
+5 hours ago
 Latest Research
 The Evolution of the Crypto CEX Landscape: A Case Study on Binance
 The Evolution of the Crypto CEX Landscape: A Case Study on Binance
@@ -152,4 +152,4 @@
 EthicsPrivacyTerms of UseCookie SettingsDo Not Sell My InfoBug Bounty Program
 © 2026 CoinDesk, Inc.
 Disclosure & Polices: CoinDesk is an award-winning media outlet that covers the cryptocurrency industry. Its journalists abide by a strict set of editorial policies. CoinDesk has adopted a set of principles aimed at ensuring the integrity, editorial independence and freedom from bias of its publications. CoinDesk is part of Bullish (NYSE:BLSH), an institutionally focused global digital asset platform that provides market infrastructure and information services. Bullish owns and invests in digital asset businesses and digital assets and CoinDesk employees, including journalists, may receive Bullish equity-based compensation.
-CryptoCD20$1,747.56CD20 up 1.17 percent1.17%BTC$63,920.01BTC up 1.02 percent1.02%ETH$1,871.20ETH up 0.76 percent0.76%XRP$1.08XRP up 0.18 percent0.18%SOL$73.90SOL up 0.99 percent0.99%
+CryptoCD20$1,740.87CD20 down 0.010 percent0.010%BTC$63,687.14BTC down 0.30 percent0.30%ETH$1,861.34ETH down 0.53 percent0.53%XRP$1.07XRP down 0.82 percent0.82%SOL$73.62SOL down 0.092 percent0.092%
