--- reddit-drained-timeline@20260801T025916Z
+++ reddit-drained-timeline@20260801T092130Z
@@ -68,6 +68,16 @@
 
 <engagement-count>
 <more-replies>
+ViperG
+•
+<relative-time>
+
+Someone was able to replicate the rng exploit on mk2/mk3:
+
+https://x.com/i/status/2082958675975553224
+
+<engagement-count>
+<more-replies>
 <more-replies>
 Lanky_Assist_6317
 •
@@ -130,14 +140,6 @@
 EDIT: removing my long post. It has been confirmed there was a breach on all coldcard mk3s due to weak entropy
 
 <engagement-count>
-artilekt
-•
-<relative-time>
-
-It looks like this is a wide-scale attack though with many people affected, something that would not be isolated to his SD card.
-
-<engagement-count>
-<more-replies>
 <more-replies>
 The_Bitcoin_Act
 •
