--- reddit-drained-timeline@20260801T092130Z
+++ reddit-drained-timeline@20260801T095230Z
@@ -32,6 +32,7 @@
 
 My key takeaway now never enter the seed phrase into anything. EVER. Even to restore. Always will generate new and use a 25th passphrase.
 
+Read more
 Comments Section
 s1ammage
 •
@@ -272,3 +273,85 @@
 I guess the lesson here is don’t trust the entropy of any device, and don’t store all your eggs in 1 basket
 
 <engagement-count>
+marvelish
+•
+<relative-time>
+
+Looks like it it is probably an issue with mk2/mk3 seed generation:
+
+https://x.com/i/status/2082958675975553224
+
+Coinkite also just issued a statement they are investigating and any users that generated their seed this way should migrate.
+
+<engagement-count>
+Visual_Objective537
+•
+<relative-time>
+
+https://blog.coinkite.com/coldcard-mk3-seed-generation-warning/
+
+‼️‼️‼️‼️‼️‼️‼️‼️
+
+<engagement-count>
+flowstate_research
+•
+<relative-time>
+
+So this was an issue with the Coldcard random number generator?
+
+<engagement-count>
+-M00NMAN
+•
+<relative-time>
+
+How can your wallet from 2021 be fine and your new wallet which you entered your original seedphrase from 2021 be drained? It’s the same wallet you’re telling us.
+
+<engagement-count>
+<more-replies>
+Open_Situation686
+•
+<relative-time>
+
+This is why I won’t make more than a small gamble on bitcoin.
+
+<engagement-count>
+MicroneedlingAlone2
+•
+<relative-time>
+
+Hey, it is not your fault. ColdCard fucked up. They just put out a blog post. Situation unfolding in realtime...
+
+<engagement-count>
+pizzeriagio
+•
+<relative-time>
+
+What if someone hacked the official website, stole the buyers info and send the buyer a compromised wallet?
+You buy the wallet, the hacker steals your information, sends you the compromised wallet, he waits an year and steals all the founds on the wallets. Or maybe is just an insider that switched the regular coldcards with compromised ones.
+
+Check your credit card movements and check if the payment you did in 2025 was actually sent to the official website.
+
+<engagement-count>
+Vegetable-Rabbit7503
+•
+<relative-time>
+
+Do you think they would have taken your BTC if you had done everything the same but had a passphrase?
+
+<engagement-count>
+[deleted]
+•
+<relative-time>
+
+Comment deleted by user
+
+<more-replies>
+[deleted]
+•
+<relative-time>
+
+Comment removed by moderator
+
+<engagement-count>
+<more-replies>
+View more comments
