--- reddit-drained-timeline@20260801T095801Z
+++ reddit-drained-timeline@20260801T181342Z
@@ -218,6 +218,16 @@
 
 <engagement-count>
 <more-replies>
+33halvings
+•
+<relative-time>
+
+I remember coldcard being promoted as THE hardware wallet for Bitcoin-only people since it was also open-source etc. I’m glad I never bought and used it.
+
+This is an absolute shitshow, might be the end of coldcard as a company.
+
+<engagement-count>
+<more-replies>
 0utrageousMango
 •
 <relative-time>
@@ -226,16 +236,6 @@
 
 <engagement-count>
 <more-replies>
-33halvings
-•
-<relative-time>
-
-I remember coldcard being promoted as THE hardware wallet for Bitcoin-only people since it was also open-source etc. I’m glad I never bought and used it.
-
-This is an absolute shitshow, might be the end of coldcard as a company.
-
-<engagement-count>
-<more-replies>
 thaneliness
 •
 <relative-time>
