--- reddit-drained-timeline@20260802T180053Z
+++ reddit-drained-timeline@20260802T200113Z
@@ -64,6 +64,7 @@
 <relative-time>
 •
 Edited <relative-time>
+ Top 1% Commenter
 
 This is crazy. I had never heard of such a surgical theft of bitcoin before. All bitcoin thefts are generally someone giving away their seed from a stupid manipulation.
 
@@ -157,6 +158,7 @@
 AdEuphoric5133
 •
 <relative-time>
+ Top 1% Commenter
 
 https://blog.coinkite.com/coldcard-mk3-seed-generation-warning/
 
