--- reddit-dice-rolls-saved-seedsigner@20260809T104101Z
+++ reddit-dice-rolls-saved-seedsigner@20260809T232803Z
@@ -612,3 +612,11 @@
 edited: false
 body:
 That would have included Coldcard just over a week ago though 😬 
+
+comment: p2n81ir
+parent: t1_p2m8rui
+author: ftball21
+created_utc: 1786282963
+edited: false
+body:
+I’m aware of the sub I’m in but using a one network device is nonsensical. I want something battle tested on all possibilities. Even if I only use one or two. 
