--- reddit-dice-rolls-saved-seedsigner@20260808T182913Z
+++ reddit-dice-rolls-saved-seedsigner@20260808T213823Z
@@ -569,3 +569,19 @@
 edited: false
 body:
 I did some research and apparently the encrypted SeedQR feature only exists if you’re running one of the modified/forked firmware builds, like the “Crypto Guide’s” seedsigner fork. Not the stock firmware from seedsigner.com/GitHub’s official repo.
+
+comment: p2ieesl
+parent: t3_1vie1vy
+author: Uhrenwerker
+created_utc: 1786215725
+edited: false
+body:
+I feel you bro I’m on the same boat, made my coldcard seeds with dice and dodged a terrifying bullet. I still freaked out and wiped the cold cards into a hot sparrow wallet and a exchange. I have ordered a seedsigner and a jade. Was planning to setup multisig 2-3 with these two plus maybe upgraded coldcard, I’ll anyway create new seeds with dice again.
+
+comment: p2iy6ht
+parent: t1_p2i40y3
+author: GadJedi
+created_utc: 1786221557
+edited: false
+body:
+I guess I forgot about that. Krux has the support also.
