--- stackernews-dice-attack-vectors@20260804T101223Z
+++ stackernews-dice-attack-vectors@20260804T175305Z
@@ -32,6 +32,13 @@
             }
           },
           {
+            "createdAt": "2026-08-04T17:38:56.419Z",
+            "text": "Because of an entropy vulnerability in these wallets that the attacker knew about, he can bypass all the security and steal your funds... In the end, you did the right thing 👍, it's Coldcard's fault for not discovering that vulnerability, and in the end, we users pay the price!",
+            "user": {
+              "name": "mkmloom"
+            }
+          },
+          {
             "createdAt": "2026-08-04T07:48:41.552Z",
             "text": "This guide is good: https://armantheparman.com/dicev2/ and you can use offline Tails to check the math",
             "user": {
