--- reddit-q-repurpose@20260804T165506Z
+++ reddit-q-repurpose@20260804T175325Z
@@ -179,3 +179,19 @@
 edited: false
 body:
 Do you use your computer webcam to scan the QR code on your ColdCard? Mine never seems to pick it up 
+
+comment: p1ovrwb
+parent: t1_p1olojf
+author: stargate425
+created_utc: 1785864865
+edited: 1785865418
+body:
+yes, I used my macbook pro's webcam to scan the qr code on my coldcard q. (and coldcard can also scan the qr on macbook's screen as well).
+
+comment: p1ow1b8
+parent: t1_p1lcsm4
+author: Either_Display_6624
+created_utc: 1785864930
+edited: false
+body:
+**No one would have thought that Coldcard would be susceptible to RNG failure lol**
