--- reddit-q-repurpose@20260804T101255Z
+++ reddit-q-repurpose@20260804T104140Z
@@ -139,3 +139,11 @@
 Honestly I'll probably give it a few weeks see if any further bugs are discovered and continue using it. The amount of scrutiny the codes gonna get now will prov aly make it the most secure.
 
 Also if you have a good, strong passphrase (50+ chars) that alone in my opinion is relatively decent without a 24 word seed
+
+comment: p1mfqlx
+parent: t1_p1kvd1a
+author: Intelligent_Map_246
+created_utc: 1785839283
+edited: false
+body:
+I'm on the same team here.  Its one of the few who can create a dice roll seed directly airgapoed.  And its airgapped so signed tx should be fine 
