--- reddit-q-repurpose@20260804T121310Z
+++ reddit-q-repurpose@20260804T124215Z
@@ -155,3 +155,11 @@
 edited: false
 body:
 pokemon red on a coldcard Q would go so hard lmao, imagine mining btc while grinding for a shiny
+
+comment: p1mz8zs
+parent: t3_1vex8hl
+author: F1shB0wl816
+created_utc: 1785846585
+edited: false
+body:
+I’m still using my mk4, I don’t really see a reason to drop it and it’s done everything I’ve wanted it too. 
