--- reddit-q-repurpose@20260804T212431Z
+++ reddit-q-repurpose@20260804T225552Z
@@ -215,3 +215,11 @@
 And btw, letting the hw generate the seed for you is always bad practice.
 
 And ironically, right now coldcard is probably the most secure hw out there, being subject of a massive relentless code review by experts and the general public.
+
+comment: p1qvdig
+parent: t1_p1mz8zs
+author: janpapiratie
+created_utc: 1785883895
+edited: false
+body:
+Except for creating a safe key..
