--- reddit-q-repurpose@20260804T195412Z
+++ reddit-q-repurpose@20260804T212431Z
@@ -203,3 +203,15 @@
 edited: false
 body:
 It has the most attacked firmware on earth,  battle tested and ready for whatever comes next.   
+
+comment: p1q6t57
+parent: t3_1vex8hl
+author: _gianlucag_
+created_utc: 1785876937
+edited: 1785877333
+body:
+Imho Coldcard is still the best of the best. Yes, something gone wrong with the 4.0.1+ release, they royally fu**ed up, but if you use external entropy to generate the seed, the device is still a solid hw capable of signing transactions and do all the other stuff.
+
+And btw, letting the hw generate the seed for you is always bad practice.
+
+And ironically, right now coldcard is probably the most secure hw out there, being subject of a massive relentless code review by experts and the general public.
