--- reddit-q-repurpose@20260804T082352Z
+++ reddit-q-repurpose@20260804T101255Z
@@ -129,3 +129,13 @@
 edited: false
 body:
 I hope somebody will develop a GameBoy emulator firmware for it.
+
+comment: p1m2n7n
+parent: t3_1vex8hl
+author: GreemBeam
+created_utc: 1785833014
+edited: false
+body:
+Honestly I'll probably give it a few weeks see if any further bugs are discovered and continue using it. The amount of scrutiny the codes gonna get now will prov aly make it the most secure.
+
+Also if you have a good, strong passphrase (50+ chars) that alone in my opinion is relatively decent without a 24 word seed
