--- reddit-q-repurpose@20260805T143419Z
+++ reddit-q-repurpose@20260805T210939Z
@@ -196,14 +196,6 @@
 body:
 **No one would have thought that Coldcard would be susceptible to RNG failure lol**
 
-comment: p1pmzyc
-parent: t3_1vex8hl
-author: Zealousideal-War6372
-created_utc: 1785871693
-edited: false
-body:
-It has the most attacked firmware on earth,  battle tested and ready for whatever comes next.   
-
 comment: p1q6t57
 parent: t3_1vex8hl
 author: _gianlucag_
@@ -270,3 +262,19 @@
 The only issue is if Coinkite disappears (high probability).   
   
 idk what will happen, but if they change the code from source available to open source, maybe at least the software will survive and get polished by all the parts interested that are looking at it. 
+
+comment: p1w7eks
+parent: t3_1vex8hl
+author: guuuug
+created_utc: 1785950436
+edited: false
+body:
+Update the firmware. And buys somw friggin dice, because you should have those with whichever hw signing device you use
+
+comment: p1x9w6m
+parent: t1_p1lisk6
+author: Responsible-Story260
+created_utc: 1785960123
+edited: false
+body:
+I did this with new firmware on Q and seed generated via dice roll matched to online tool using same numbers. So I’m confident that firmware doesn’t have the bug at least for now 
