--- coldcard-hack-tracker@20260809T224523Z
+++ coldcard-hack-tracker@20260812T000628Z
@@ -64,7 +64,7 @@
 
 BALANCE ON WATCH
 
-1,367.16828024 BTC
+1,367.1683017 BTC
 
 <live-fiat-value>
 <live-fiat-value>
@@ -77,7 +77,7 @@
 
 MOVED
 
-118.60746396 BTC
+118.6074425 BTC
 
 Reported consolidation has left the holding addresses
 
@@ -112,9 +112,9 @@
 LABEL	ADDRESS	BALANCE	REPORT	UTXOS	STATUS
 
 Holding 1
-	bc1qq85v2c…3fcu9r	562.02141094 BTC
-<live-fiat-value>
-	562.02021083 BTC	58	HELD
+	bc1qq85v2c…3fcu9r	562.0214324 BTC
+<live-fiat-value>
+	562.02021083 BTC	60	HELD
 
 Holding 2
 	bc1qx76cae…htfhe3	398.47576957 BTC
@@ -837,28 +837,12 @@
 
 To bc1q55ll…g4q2
 
-HOP 2
-e5530973…acbf62
-−0.00010224 BTC
-From Hop 2 from Hop 1 from Wave 4 park bc1q55ll…g4q2
-Block 961,384
-
-To bc1qcgjn…tdt7
-
 93df0d26…4eabd4
 −30.18476329 BTC
 From Wave 2 collector bc1qmd5m…8jp6
 Block 961,368
 
 To bc1qez89…dt6n
-
-HOP 2
-d05a5ac8…ac4abe
-−0.00031259 BTC
-From Hop 2 from Hop 1 from Wave 4 park bc1q55ll…g4q2
-Block 961,148
-
-To bc1qcgjn…tdt7
 
 HOP 1
 24ee02cf…7d11f2
@@ -924,7 +908,7 @@
 From Hop 2 from Hop 1 from Wave 4 park bc1q55ll…g4q2
 Block 960,898
 
-To bc1qcgjn…tdt7
+To Quidax deposit bc1qcgjn…tdt7
 
 HOP 1
 6c85668e…d30bbb
