--- coldcard-hack-tracker@20260805T175142Z
+++ coldcard-hack-tracker@20260805T185159Z
@@ -840,6 +840,14 @@
 
 To bc1qvsrp…93hs, bc1qwp64…y9kh, bc1qelpm…c5hg +332 more
 
+HOP 2
+93eeb9e7…d9fbc6
+−1.29140163 BTC
+From Hop 2 from Hop 1 from Aug 2 vault bc1qs0zc…l3n6
+Block 961,076
+
+To bc1qh6me…z56m
+
 HOP 1
 f3ee6e61…c37b24
 −64.90373154 BTC
@@ -960,14 +968,6 @@
 6c85668e…d30bbb
 −0.02374887 BTC
 From Hop 1 from Wave 4 park 3EkYzYXT…DWeF
-Block 960,794
-
-To bc1qhfdz…3wcx
-
-HOP 1
-6c85668e…d30bbb
-−0.06990788 BTC
-From Hop 1 from Wave 4 park bc1qw79v…83ha
 Block 960,794
 
 To bc1qhfdz…3wcx
