--- coldcard-hack-tracker@20260804T220914Z
+++ coldcard-hack-tracker@20260804T223919Z
@@ -815,6 +815,13 @@
 
 To 334iKwmh…J6gB
 
+f02164a9…0db281
+−0.96227191 BTC
+From Wave 4 park 34nHYNnc…4qDQ
+Block 961,064
+
+To 334iKwmh…J6gB
+
 HOP 1
 1e96c88b…91ab3a
 −0.33330008 BTC
@@ -916,12 +923,100 @@
 
 To bc1qqa7p…36pr
 
+HOP 1
+d37d6901…bac7b6
+−0.00061835 BTC
+From Hop 1 from P2TR vault bc1qqa7p…36pr
+Block 961,030
+
+To bc1qkg64…xygt
+
+HOP 1
+ee4e4c70…2105f2
+−0.00282469 BTC
+From Hop 1 from P2TR vault bc1qqa7p…36pr
+Block 961,030
+
+To bc1qz9hf…97s0
+
+HOP 1
+58932a6d…7eb177
+−0.229641 BTC
+From Hop 1 from P2TR vault bc1qqa7p…36pr
+Block 961,028
+
+To bc1q080c…wua8
+
+HOP 1
+d58b2c53…3d82a4
+−0.25476269 BTC
+From Hop 1 from P2TR vault bc1qqa7p…36pr
+Block 961,028
+
+To bc1qqm62…efvn
+
+HOP 1
+00987d33…2e738f
+−0.00231734 BTC
+From Hop 1 from P2TR vault bc1qqa7p…36pr
+Block 961,027
+
+To bc1qt6f9…8tfz
+
+HOP 1
+204afe34…a3203b
+−0.27432425 BTC
+From Hop 1 from P2TR vault bc1qqa7p…36pr
+Block 961,027
+
+To bc1qqm62…efvn
+
+HOP 1
+8458986a…c9d375
+−0.00022475 BTC
+From Hop 1 from P2TR vault bc1qqa7p…36pr
+Block 961,027
+
+To bc1qddn8…ucc7
+
+HOP 1
+ed2daae8…4e77ff
+−0.00309258 BTC
+From Hop 1 from P2TR vault bc1qqa7p…36pr
+Block 961,027
+
+To bc1q2tj0…lj63
+
 1a454301…c772c2
 −2.00017714 BTC
 From P2TR vault bc1p0l0x…lk0h
 Block 961,026
 
 To OP_RETURN messenger hub bc1q2nfx…kyyq
+
+HOP 1
+03d7ba76…5a2320
+−0.00304252 BTC
+From Hop 1 from P2TR vault bc1qqa7p…36pr
+Block 961,025
+
+To bc1qvr9d…85ay
+
+HOP 1
+8c8cd809…92b614
+−0.0030229 BTC
+From Hop 1 from P2TR vault bc1qqa7p…36pr
+Block 961,025
+
+To bc1q987r…al70
+
+HOP 1
+af2216ea…106397
+−0.04560948 BTC
+From Hop 1 from P2TR vault bc1qqa7p…36pr
+Block 961,025
+
+To bc1q5cyd…jsyq
 
 064eb61d…7ddd4f
 −1.00017823 BTC
@@ -982,44 +1077,6 @@
 Block 960,793
 
 To Bullish.com deposit 3CTpBmp8…2hBH, bc1qw79v…83ha, bc1q72k7…aamt +31 more
-
-HOP 1
-61c0ddac…98c095
-−0.509799 BTC
-From Hop 1 from Evening vault bc1qu2uq…6srw
-Block 960,668
-
-To bc1qt8ca…5y4k, bc1qzkap…6e4z
-
-ff327937…abc133
-−0.50980268 BTC
-From Evening vault bc1q7rms…rcgf
-Block 960,666
-
-To bc1qu2uq…6srw
-
-HOP 1
-a7bf30f0…92fdec
-−0.45 BTC
-From Hop 1 from Aug 1 hop vault bc1qsd9t…jah8
-Block 960,663
-
-To 3P3K2MQw…Lfgy
-
-5ecc3052…a8bf87
-−0.69135523 BTC
-From Aug 1 hop vault bc1qayw8…2u0s
-Block 960,658
-
-To bc1qsd9t…jah8, bc1qvrpc…cddc
-
-HOP 1
-135d4c2c…2d4ce9
-−0.241352 BTC
-From Hop 1 from Aug 1 hop vault bc1qvrpc…cddc
-Block 960,658
-
-To 36c8aprK…QW35
 
 Am I at risk?
 
