--- coldcard-hack-tracker@20260807T112003Z
+++ coldcard-hack-tracker@20260807T122016Z
@@ -83,9 +83,9 @@
 
 LAST MOVEMENT
 
-Block 961,404
-
-0.02324129 BTC from Hop 1 from Wave 4 park
+Block 961,428
+
+0.00046352 BTC from Hop 1 from Wave 4 park
 
 Watched holdings
 
@@ -793,7 +793,7 @@
 a2378c07…1b842f
 −0.00046352 BTC
 From Hop 1 from Wave 4 park bc1q72k7…aamt
-Unconfirmed
+Block 961,428
 
 To bc1qkc7l…qc58
 
@@ -868,6 +868,13 @@
 f02164a9…0db281
 −1.09429601 BTC
 From Wave 4 park 36XfMDAY…1x3G
+Block 961,064
+
+To 334iKwmh…J6gB
+
+f02164a9…0db281
+−0.96227191 BTC
+From Wave 4 park 34nHYNnc…4qDQ
 Block 961,064
 
 To 334iKwmh…J6gB
