--- coldcard-hack-tracker@20260805T225344Z
+++ coldcard-hack-tracker@20260805T235428Z
@@ -867,6 +867,14 @@
 HOP 2
 65c79326…638de0
 −1.29140163 BTC
+From Hop 2 from Hop 1 from Aug 2 vault bc1pxs72…0pct
+Block 961,077
+
+To bc1qvsrp…93hs, bc1qwp64…y9kh, bc1qelpm…c5hg +332 more
+
+HOP 2
+65c79326…638de0
+−1.29140163 BTC
 From Hop 2 from Hop 1 from Aug 2 vault bc1q5h2t…22mf
 Block 961,077
 
@@ -1081,13 +1089,6 @@
 Block 960,661
 
 To P2SH service hub 3KMmeqPe…qtc8
-
-5ecc3052…a8bf87
-−0.69135523 BTC
-From Aug 1 hop vault bc1qayw8…2u0s
-Block 960,658
-
-To bc1qsd9t…jah8, bc1qvrpc…cddc
 
 Am I at risk?
 
