--- coldcard-hack-tracker@20260805T155123Z
+++ coldcard-hack-tracker@20260805T165138Z
@@ -64,7 +64,7 @@
 
 BALANCE ON WATCH
 
-1,397.35235589 BTC
+1,397.35237127 BTC
 
 <live-fiat-value>
 <live-fiat-value>
@@ -77,7 +77,7 @@
 
 MOVED
 
-88.42338831 BTC
+88.42337293 BTC
 
 Reported consolidation has left the holding addresses
 
@@ -112,9 +112,9 @@
 LABEL	ADDRESS	BALANCE	REPORT	UTXOS	STATUS
 
 Holding 1
-	bc1qq85v2c…3fcu9r	562.02078435 BTC
-<live-fiat-value>
-	562.02021083 BTC	34	HELD
+	bc1qq85v2c…3fcu9r	562.02079973 BTC
+<live-fiat-value>
+	562.02021083 BTC	35	HELD
 
 Holding 2
 	bc1qx76cae…htfhe3	398.47576957 BTC
@@ -947,14 +947,6 @@
 Block 960,797
 
 To P2WPKH service hub bc1qactq…80gh, bc1qm26v…3q52, bc1qemas…ahmt +27 more
-
-HOP 1
-6c85668e…d30bbb
-−0.02374887 BTC
-From Hop 1 from Wave 4 park 3EkYzYXT…DWeF
-Block 960,794
-
-To bc1qhfdz…3wcx
 
 HOP 1
 6c85668e…d30bbb
