--- coldcard-hack-tracker@20260805T185159Z
+++ coldcard-hack-tracker@20260805T205254Z
@@ -64,7 +64,7 @@
 
 BALANCE ON WATCH
 
-1,397.35237127 BTC
+1,397.35238017 BTC
 
 <live-fiat-value>
 <live-fiat-value>
@@ -77,7 +77,7 @@
 
 MOVED
 
-88.42337293 BTC
+88.42336403 BTC
 
 Reported consolidation has left the holding addresses
 
@@ -112,9 +112,9 @@
 LABEL	ADDRESS	BALANCE	REPORT	UTXOS	STATUS
 
 Holding 1
-	bc1qq85v2c…3fcu9r	562.02079973 BTC
-<live-fiat-value>
-	562.02021083 BTC	35	HELD
+	bc1qq85v2c…3fcu9r	562.02080863 BTC
+<live-fiat-value>
+	562.02021083 BTC	36	HELD
 
 Holding 2
 	bc1qx76cae…htfhe3	398.47576957 BTC
@@ -835,6 +835,14 @@
 HOP 2
 65c79326…638de0
 −1.29140163 BTC
+From Hop 2 from Hop 1 from Aug 2 vault bc1q5h2t…22mf
+Block 961,077
+
+To bc1qvsrp…93hs, bc1qwp64…y9kh, bc1qelpm…c5hg +332 more
+
+HOP 2
+65c79326…638de0
+−1.29140163 BTC
 From Hop 2 from Hop 1 from Aug 2 vault bc1qtmwr…rwz8
 Block 961,077
 
@@ -968,6 +976,14 @@
 6c85668e…d30bbb
 −0.02374887 BTC
 From Hop 1 from Wave 4 park 3EkYzYXT…DWeF
+Block 960,794
+
+To bc1qhfdz…3wcx
+
+HOP 1
+6c85668e…d30bbb
+−0.06990788 BTC
+From Hop 1 from Wave 4 park bc1qw79v…83ha
 Block 960,794
 
 To bc1qhfdz…3wcx
