--- coldcard-hack-tracker@20260805T215323Z
+++ coldcard-hack-tracker@20260805T225344Z
@@ -809,6 +809,14 @@
 To bc1q99hg…6u46, bc1qnkre…v7z6, bc1pwh5y…fdhs +380 more
 
 HOP 2
+3e45e90f…4c11f0
+−1.29140163 BTC
+From Hop 2 from Hop 1 from Aug 2 vault bc1pr0h7…5ej8
+Block 961,103
+
+To bc1q89tx…8ue8, bc1qju80…mfh5, bc1qnq55…rkva +397 more
+
+HOP 2
 80f11c77…8cbf90
 −1.29140163 BTC
 From Hop 2 from Hop 1 from Aug 2 vault bc1qyhr4…k9jv
@@ -827,6 +835,14 @@
 HOP 2
 11a3edd2…df46c1
 −1.29140163 BTC
+From Hop 2 from Hop 1 from Aug 2 vault bc1pyym5…hfw9
+Block 961,084
+
+To bc1qqyk6…dyh2, bc1qqf2k…d8nc, bc1qz2yc…racr +380 more
+
+HOP 2
+11a3edd2…df46c1
+−1.29140163 BTC
 From Hop 2 from Hop 1 from Aug 2 vault bc1qvznm…5t7e
 Block 961,084
 
@@ -1072,14 +1088,6 @@
 Block 960,658
 
 To bc1qsd9t…jah8, bc1qvrpc…cddc
-
-HOP 1
-135d4c2c…2d4ce9
-−0.241352 BTC
-From Hop 1 from Aug 1 hop vault bc1qvrpc…cddc
-Block 960,658
-
-To 36c8aprK…QW35
 
 Am I at risk?
 
