--- reddit-self-custody-last-straw@20260804T181046Z
+++ reddit-self-custody-last-straw@20260806T095701Z
@@ -510,3 +510,18 @@
 2. It's time for people to be realistic about what would happen during a $5 wrench attack. An attacker isn't really coming after you with a wrench. It's more likely he'll have a gun. And if he has reason to suspect you gave him less than everything, you'll get beaten until the thief thinks he's gotten everything, at which point you'll be lucky if you don't get shot.
 
 Any attacker willing to physically attack you in order to steal your Bitcoin... that dude's not playing around. He knows he's risking prison if he gets caught. And really, it's probably going to be multiple attackers. If they think you're sitting on enough Bitcoin to make it worth attacking you to steal it, giving them a few sats from a decoy wallet probably also means you get shot.
+
+comment: p20epk4
+parent: t3_1vem5go
+author: Necessary_Floor_7081
+created_utc: 1785998506
+edited: false
+body:
+Could someone please offer me some peace of mind while I wait for a new wallet to arrive.
+
+My coldcard mk3 seeds were generated on firmware 3.9. So as far as I can see I’m ‘safe’ for the moment and I have time to transfer to a new set up.
+
+I’m just asking for someone with a different brain to mine to validate what I’m assuming.
+
+Wanna make sure I’m not a sitting duck.  
+Thanks legends
