--- reddit-self-custody-last-straw@20260807T192506Z
+++ reddit-self-custody-last-straw@20260808T212851Z
@@ -151,16 +151,6 @@
 
 I happened to catch it early too before my other cold card was drained. But this was my largest amount though..
 
-comment: p1iug9n
-parent: t3_1vem5go
-author: riseandride69
-created_utc: 1785790424
-edited: 1785790699
-body:
-Nobody is making anyone whole lol, it's too much money that they don't have. They will just go into Chapter 11, and bye-bye
-
-I'd rather stick with IBIT in my SIPC-insured Schwab then loosing sleep over this shit, or always staying on top of where the hell that seed backup is in my house and if my wife accidentally threw it away, lol. And don't even get me started on how much easier it will be dealing with taxes.
-
 comment: p1iuq7j
 parent: t1_p1ib1p7
 author: jdancouga
