--- reddit-retirement-attack-resurfaced@20260804T111432Z
+++ reddit-retirement-attack-resurfaced@20260804T121454Z
@@ -311,3 +311,20 @@
 >To date I have heard of zero compromised seeds that were generated using on-board RNG due to entropy issues, while there are countless examples of users losing funds due to improper dice rolls.
 
 Wow, this is your supporting reference?
+
+comment: p1mtzpq
+parent: t3_1vf0ut0
+author: NetimLabs
+created_utc: 1785844822
+edited: 1785845317
+body:
+Why bother with dice rolls?
+We have [publicly available streams](https://qrng.anu.edu.au/random-hex/) of truly [quantum] random data
+
+comment: p1mw7dg
+parent: t1_p1lp1qo
+author: CBpegasus
+created_utc: 1785845581
+edited: false
+body:
+I mean, it's a fairly well known concept and you would expect people working in a company whose product is security to know about it...
