--- reddit-retirement-attack-resurfaced@20260804T175511Z
+++ reddit-retirement-attack-resurfaced@20260804T182455Z
@@ -473,9 +473,9 @@
 parent: t1_p1n3eq9
 author: youtossershad1job2do
 created_utc: 1785860550
-edited: false
-body:
-What was the whole point of crypto was it was trust less. 
+edited: 1785867322
+body:
+It was the whole point of crypto that it was trust-less.
 
 comment: p1om0wr
 parent: t1_p1m0pf3
@@ -496,3 +496,19 @@
 edited: false
 body:
 Dice rolls are solid. No correct dice rolls have been hacked ever. If you mean human error ie what the article you posted alludes to ( not enough actual rolls) or grouping all the numbers in order( all 1's then 2' etc) then yeah thats retarded but a true 100+dice roll actually using 100 dice is effectively un-hackable which is why coldcard users using this function remain safe.
+
+comment: p1p54mw
+parent: t1_p1lka5q
+author: Express_Living2264
+created_utc: 1785867168
+edited: false
+body:
+i don't see this as incriminating at all. They are advertising to security extremists. They added a cheap to build gimmick for marketing and then advertised it to generate engagement, that's all there is to it.
+
+comment: p1p5in6
+parent: t1_p1n6apo
+author: Express_Living2264
+created_utc: 1785867265
+edited: false
+body:
+the sad thing is. It's still one of the better products due to the airgapped signing feature + electrum read only wallet. something trezor afaik cant do.
