--- reddit-retirement-attack-resurfaced@20260804T162723Z
+++ reddit-retirement-attack-resurfaced@20260804T165714Z
@@ -476,3 +476,15 @@
 edited: false
 body:
 What was the whole point of crypto was it was trust less. 
+
+comment: p1om0wr
+parent: t1_p1m0pf3
+author: anonymous-12358
+created_utc: 1785862452
+edited: false
+body:
+I would even go as far to say don’t trust Trezor or Ledger.
+
+If you’re that deep into self custodial wallets then take that little extra step to setup your own wallet hosted on a microcontroller.
+
+That way you can 100% guarantee you’re using a 100% air gapped device and you can test to your hearts content if the open-source wallet is faulty free. 
