--- reddit-retirement-attack-resurfaced@20260804T104325Z
+++ reddit-retirement-attack-resurfaced@20260804T111432Z
@@ -293,3 +293,21 @@
 edited: false
 body:
 🧡
+
+comment: p1mlbbq
+parent: t1_p1lzdzv
+author: CiaranCarroll
+created_utc: 1785841598
+edited: false
+body:
+>They used <10 dice rolls
+
+Sorry but it takes 15mins to do 99 dice rolls. Cold Card also allowed you to verify that it wasn't spoofing by using other websites like Ian Colemans on a test seed.
+
+If you cannot spare 15mins to secure your Bitcoin then you have no business in self-custody.
+
+>99.99999% of users are better off allowing good, multi-source, open-source random number generation like we do on Passport.
+
+>To date I have heard of zero compromised seeds that were generated using on-board RNG due to entropy issues, while there are countless examples of users losing funds due to improper dice rolls.
+
+Wow, this is your supporting reference?
