--- reddit-retirement-attack-resurfaced@20260804T205621Z
+++ reddit-retirement-attack-resurfaced@20260804T212618Z
@@ -561,3 +561,23 @@
 For good security you need to force the hacker to make at least 2^128 calculations. That's a lot. Dont roll your own cryptography so you dont make basic errors like this that could cost you money.
 
 You know this coldcard hack was based on the hacker doing ~2^32 calculations which is about 4 billion, still huge big number in human terms but very little for a computer.
+
+comment: p1q8d8k
+parent: t1_p1p5in6
+author: tonto515
+created_utc: 1785877355
+edited: false
+body:
+Trezor 3 and 5 are airgapped, only 7 has the Bluetooth functionality.
+
+That said, my Keystone Pro 3 is airgapped and I still moved my funds this morning from my old wallet with a single seed phrase to a new one with 3/5 Shamir sharding to protect it.
+
+comment: p1qav0u
+parent: t1_p1q8d8k
+author: Express_Living2264
+created_utc: 1785878032
+edited: false
+body:
+what im referring to is the ability to have a read only electrum wallet on an online pc. You create a transaction there, copy the transaction file via a usb stick to an offline pc sign it with electrum on the offline machine. Then take the signed file back to the online pc and send it. 
+
+The offline pc can be completely replaced with a coldwallet, afaik trezor doesn't support this.
