--- reddit-retirement-attack-resurfaced@20260804T131501Z
+++ reddit-retirement-attack-resurfaced@20260804T135352Z
@@ -386,3 +386,43 @@
 edited: false
 body:
 Well people have already said nothing is going to happen to coldkite, theyll just declare bankruptcy and get away with it. So nothing will be done even if they snitched on themselves, in a tweet.
+
+comment: p1n9nox
+parent: t1_p1mtzpq
+author: 10kpizza
+created_utc: 1785849763
+edited: false
+body:
+The important key thing with generating private keys is that nobody else knows them. The easiest way to get a number that nobody else knows is random data. However if the random data is publicly available to others then this obviously isnt secure.
+
+comment: p1nb6p1
+parent: t3_1vf0ut0
+author: crypto_chik
+created_utc: 1785850203
+edited: false
+body:
+Chances it wasn’t an inside job are dwindling…
+
+comment: p1nfcwj
+parent: t1_p1mtzpq
+author: ILurkReddi
+created_utc: 1785851379
+edited: false
+body:
+would rather [random.org](http://random.org) or cloudflare
+
+comment: p1nftns
+parent: t1_p1mtzpq
+author: tenor_tymir
+created_utc: 1785851509
+edited: false
+body:
+How is this complex hex number practical for the average user? 
+
+comment: p1ng0uf
+parent: t1_p1meno4
+author: tenor_tymir
+created_utc: 1785851564
+edited: false
+body:
+It’s a device to delicately shave your balls. 
