--- reddit-retirement-attack-resurfaced@20260804T212618Z
+++ reddit-retirement-attack-resurfaced@20260804T215615Z
@@ -581,3 +581,23 @@
 what im referring to is the ability to have a read only electrum wallet on an online pc. You create a transaction there, copy the transaction file via a usb stick to an offline pc sign it with electrum on the offline machine. Then take the signed file back to the online pc and send it. 
 
 The offline pc can be completely replaced with a coldwallet, afaik trezor doesn't support this.
+
+comment: p1qeqjq
+parent: t1_p1q4mvx
+author: NetimLabs
+created_utc: 1785879082
+edited: 1785879461
+body:
+Idk, they would have to try all the conversion methods possible anyways, that makes it much harder, if not impossible.
+
+You can come up with lots of weird ways to do that.
+
+Ideally one should DIY their own [qrng generator](https://www.youtube.com/watch?v=CE4qXo-hYng), of course.
+
+comment: p1qh0z8
+parent: t1_p1lzdzv
+author: alfredonoodles
+created_utc: 1785879719
+edited: false
+body:
+From what I have been reading the past week about this, most used WEB BASED dice rolls lololol how stupid.
