--- reddit-retirement-attack-resurfaced@20260804T135352Z
+++ reddit-retirement-attack-resurfaced@20260804T145354Z
@@ -426,3 +426,37 @@
 edited: false
 body:
 It’s a device to delicately shave your balls. 
+
+comment: p1np3g9
+parent: t3_1vf0ut0
+author: Always_working_hardd
+created_utc: 1785854019
+edited: false
+body:
+Meanwhile there's some Indian out there taking possession of a private jet so he can fly to the island he just bought in the Caribbean. 
+
+Also a former employee of coldcard.
+
+comment: p1npnij
+parent: t1_p1n3eq9
+author: Unsounded
+created_utc: 1785854167
+edited: false
+body:
+The funny thing is academia is fraught with fraudsters and bad coders.
+
+comment: p1nrrps
+parent: t1_p1m1u8n
+author: kikikza
+created_utc: 1785854728
+edited: false
+body:
+Gotta use a wall of lava lamps
+
+comment: p1nsfqo
+parent: t3_1vf0ut0
+author: beatthebook2x
+created_utc: 1785854903
+edited: false
+body:
+Retirement attack = retirement plan
