--- reddit-retirement-attack-resurfaced@20260804T195558Z
+++ reddit-retirement-attack-resurfaced@20260804T202630Z
@@ -513,6 +513,16 @@
 body:
 the sad thing is. It's still one of the better products due to the airgapped signing feature + electrum read only wallet. something trezor afaik cant do.
 
+comment: p1pq3e0
+parent: t3_1vf0ut0
+author: lysergamythical
+created_utc: 1785872511
+edited: false
+body:
+ "alternatively people" 
+
+are you being deliberately obtuse?
+
 comment: p1pri25
 parent: t1_p1n9nox
 author: NetimLabs
@@ -531,3 +541,11 @@
 body:
 The one I linked was just an example.  
 You could use any qrng source you want, of course.
+
+comment: p1pzzfh
+parent: t1_p1lj212
+author: Deto
+created_utc: 1785875119
+edited: false
+body:
+Fat fingering?  You don't have to actually remember your dice rolls for later though
