--- reddit-retirement-attack-resurfaced@20260804T202630Z
+++ reddit-retirement-attack-resurfaced@20260804T205621Z
@@ -549,3 +549,15 @@
 edited: false
 body:
 Fat fingering?  You don't have to actually remember your dice rolls for later though
+
+comment: p1q4mvx
+parent: t1_p1pri25
+author: 10kpizza
+created_utc: 1785876359
+edited: false
+body:
+No it wouldnt because theres not that many ways to partition the stream. The hacker could just try them all.
+
+For good security you need to force the hacker to make at least 2^128 calculations. That's a lot. Dont roll your own cryptography so you dont make basic errors like this that could cost you money.
+
+You know this coldcard hack was based on the hacker doing ~2^32 calculations which is about 4 billion, still huge big number in human terms but very little for a computer.
