--- reddit-retirement-attack-resurfaced@20260804T165714Z
+++ reddit-retirement-attack-resurfaced@20260804T175511Z
@@ -488,3 +488,11 @@
 If you’re that deep into self custodial wallets then take that little extra step to setup your own wallet hosted on a microcontroller.
 
 That way you can 100% guarantee you’re using a 100% air gapped device and you can test to your hearts content if the open-source wallet is faulty free. 
+
+comment: p1ou7vl
+parent: t1_p1lzdzv
+author: EyesFor1
+created_utc: 1785864479
+edited: false
+body:
+Dice rolls are solid. No correct dice rolls have been hacked ever. If you mean human error ie what the article you posted alludes to ( not enough actual rolls) or grouping all the numbers in order( all 1's then 2' etc) then yeah thats retarded but a true 100+dice roll actually using 100 dice is effectively un-hackable which is why coldcard users using this function remain safe.
