--- reddit-retirement-attack-resurfaced@20260804T215615Z
+++ reddit-retirement-attack-resurfaced@20260804T222648Z
@@ -586,13 +586,14 @@
 parent: t1_p1q4mvx
 author: NetimLabs
 created_utc: 1785879082
-edited: 1785879461
+edited: 1785881704
 body:
 Idk, they would have to try all the conversion methods possible anyways, that makes it much harder, if not impossible.
 
 You can come up with lots of weird ways to do that.
 
-Ideally one should DIY their own [qrng generator](https://www.youtube.com/watch?v=CE4qXo-hYng), of course.
+Ideally one should DIY their own [qrng generator](https://www.youtube.com/watch?v=CE4qXo-hYng), of course.  
+I guess at that point rolling dice is more practical but if you want true randomness, qrng is the way.
 
 comment: p1qh0z8
 parent: t1_p1lzdzv
