--- reddit-retirement-attack-resurfaced@20260804T124400Z
+++ reddit-retirement-attack-resurfaced@20260804T131501Z
@@ -354,3 +354,35 @@
 edited: false
 body:
 I read that at least as partially being unable to contain bragging about whatever perceived brilliance they have of themselves. They dont want people who find out to not understand their cleverness. 
+
+comment: p1n3eq9
+parent: t1_p1mxiiw
+author: bricksplus
+created_utc: 1785847901
+edited: false
+body:
+You put trust in the institution that’s being doing high level research for decades 
+
+comment: p1n3t4c
+parent: t3_1vf0ut0
+author: shadowmage666
+created_utc: 1785848023
+edited: false
+body:
+Hmm not suspicious at all 
+
+comment: p1n5dgp
+parent: t3_1vf0ut0
+author: Gooner_93
+created_utc: 1785848496
+edited: false
+body:
+And yet you will get some people calling you insane for saying it was an inside job.
+
+comment: p1n6apo
+parent: t3_1vf0ut0
+author: axb90
+created_utc: 1785848777
+edited: false
+body:
+Well people have already said nothing is going to happen to coldkite, theyll just declare bankruptcy and get away with it. So nothing will be done even if they snitched on themselves, in a tweet.
