--- reddit-retirement-attack-resurfaced@20260806T034107Z
+++ reddit-retirement-attack-resurfaced@20260806T101426Z
@@ -359,9 +359,9 @@
 parent: t1_p1mxiiw
 author: bricksplus
 created_utc: 1785847901
-edited: false
-body:
-You put trust in the institution that’s being doing high level research for decades 
+edited: 1785988016
+body:
+You put trust in the institution that’s doing high level research for decades
 
 comment: p1n3t4c
 parent: t3_1vf0ut0
