--- reddit-retirement-attack-resurfaced@20260804T222648Z
+++ reddit-retirement-attack-resurfaced@20260804T233315Z
@@ -387,6 +387,14 @@
 body:
 Well people have already said nothing is going to happen to coldkite, theyll just declare bankruptcy and get away with it. So nothing will be done even if they snitched on themselves, in a tweet.
 
+comment: p1n7h3w
+parent: t3_1vf0ut0
+author: Complete-MessUP
+created_utc: 1785849130
+edited: false
+body:
+he is the one % 99
+
 comment: p1n9nox
 parent: t1_p1mtzpq
 author: 10kpizza
@@ -602,3 +610,11 @@
 edited: false
 body:
 From what I have been reading the past week about this, most used WEB BASED dice rolls lololol how stupid.
+
+comment: p1qyab5
+parent: t1_p1lnwlq
+author: Cold_Huckleberry_633
+created_utc: 1785884786
+edited: false
+body:
+I remember when this first happened someone posted here and everyone was like “bet you posted your seed phrase online lol dumbass.” Imagine that but every few months and OP did nothing wrong and nobody believes them 💔💔
