--- reddit-retirement-attack-resurfaced@20260804T182455Z
+++ reddit-retirement-attack-resurfaced@20260804T195558Z
@@ -512,3 +512,22 @@
 edited: false
 body:
 the sad thing is. It's still one of the better products due to the airgapped signing feature + electrum read only wallet. something trezor afaik cant do.
+
+comment: p1pri25
+parent: t1_p1n9nox
+author: NetimLabs
+created_utc: 1785872880
+edited: false
+body:
+The potential attackers would have to constantly monitor the stream and save it on their devices, then figure out which method you used to convert that stream to a seed phrase.
+
+Using different parts \[same lenght\] of the stream for each word would solve this I think.
+
+comment: p1ps2r8
+parent: t1_p1nfcwj
+author: NetimLabs
+created_utc: 1785873032
+edited: false
+body:
+The one I linked was just an example.  
+You could use any qrng source you want, of course.
