--- reddit-cto-code-link@20260806T050821Z
+++ reddit-cto-code-link@20260806T100801Z
@@ -1287,3 +1287,11 @@
 edited: false
 body:
 He’s a complete jerk!
+
+comment: p202yjo
+parent: t1_p1ztvkz
+author: immersive-matthew
+created_utc: 1785993066
+edited: false
+body:
+That is debatable but I can understand why this is a view as for sure there are some benefits especially if you are a fan of Carney who essentially got voted in by a landslide thanks to Trump.  Unfortunately, Canada is now heavily investing in more fossil fuel extraction and ports to ship it all over while at the same time it’s forests are burning to the ground due to the climate change it is accelerating.  Carney is a slick leader and has many good qualities that I admire, but he is not being a leader for the future of Canada and the planet that once we are forced to moved from fossil fuels, we will be stuck with billions of legacy oil and gas infrastructure and little to no green. 
