--- reddit-cto-code-link@20260806T163636Z
+++ reddit-cto-code-link@20260807T061954Z
@@ -21,13 +21,11 @@
 
 comment: p1qysu3
 parent: t1_p1qx3mp
-author: 12ealdeal
+author: [deleted]
 created_utc: 1785884947
 edited: false
 body:
-This is crypto:
-
-I’m afraid it’s going to be much much worse.
+[deleted]
 
 comment: p1r2o7i
 parent: t1_p1qx3mp
@@ -1306,3 +1304,21 @@
 For $114 sure, not $114 million. Real exchanges can, and do, blacklist wallets. OP is also stupid because he's not a 'noob' for putting it in one wallet, it can be sent to ten thousand and it'll be tracked extremely easily by software. 
 
 Over 90% of stolen Bitcoin, from every major hacking event or fraud ever, sits dormant right now. Exchanges that have that much liquidity also have zero interest in ruining their business or being pulled into legal battles to reobtain stolen property that they just purchased. The exchanges do not operate anonymously.
+
+comment: p268fab
+parent: t1_p1scfy7
+author: Quantris
+created_utc: 1786063339
+edited: false
+body:
+you're telling me "stanley_fatmax" isn't your real name?
+
+say it ain't so
+
+comment: p26fv3p
+parent: t1_p268fab
+author: stanley_fatmax
+created_utc: 1786065747
+edited: false
+body:
+👀👀
