--- reddit-dice-bias-1200-rolls@20260809T041040Z
+++ reddit-dice-bias-1200-rolls@20260809T232742Z
@@ -821,9 +821,24 @@
 parent: t1_p2ejdfy
 author: Past_Permission_6123
 created_utc: 1786218345
-edited: false
+edited: 1786296805
 body:
 I think you make dice rolling sound a lot more complicated than it is. Any ColdCard hardware wallet user should be able to do it, the ColdCard devs even recommended to use dice rolls for the best security. It's an optional part of the setup and described in the official user guide: [https://coldcard.com/docs/master-seed/#create-a-new-master-seed](https://coldcard.com/docs/master-seed/#create-a-new-master-seed)
+
+Edit: Just to be clear I am in no way supporting ColdCard, and I've never owned one. I only mentioned them because they recommended dice rolling around the same time the seed generation vulnerability was introduced. Point being it's a simple, established method to generate entropy on your own, and it is/was easy to do on their device.
+
+>The best recommendation is to use a strong passphrase
+
+That's a good idea for sure, but the recent hack demonstrates that a lot of people simply didn't follow this advice. If you can use dice roll for the pass phrase, you can do so with the seed generation as well. Both methods are prone to be vulnerable if you don't know what you are doing. Making up your own pass phrase like many people do is well known to not be very secure.
+
+And I agree most people should not follow a complex process if they don't know what they are doing, but I think this is a given. Actually I think most "regular" people should not own a hardware wallet because of all the potential mistakes, there are many ways you can lose you keys or get hacked.
+
+>What maths qualifications does the op have to conclude that there is no bias... Let me guess none.  
+My intuition says when computers do guess in the orders of millions and billions a thousand is insignificant.
+
+This comment tells me you have no clue about the math involved, so it would be pretty bold of you to imply that dice rolling is not good enough for entropy. This is high school level math, it's really not that hard. Generating good entropy on a computer is hard to do on your own with efficient algorithms, while doing it with dice is easy - anyone can do it. That is the real point I wanted to make in this thread.
+
+I understand the analogy with ‘*Don’t roll your own crypto*’, but I don't think it's fitting to this discussion. As mentioned in [this blogpost](https://samuellucas.com/2024/08/31/debunking-dont-roll-your-own-crypto.html), "Even cryptographers working in teams come up with [broken](https://eprint.iacr.org/2022/214) [algorithms](https://eprint.iacr.org/2022/975) all the time."
 
 comment: p2kvl9n
 parent: t3_1vi20y9
