--- reddit-trezor-official-response@20260809T121942Z
+++ reddit-trezor-official-response@20260815T055714Z
@@ -819,3 +819,13 @@
 edited: false
 body:
 You need explain more. What kind of phishing, how?
+
+comment: p3r7rft
+parent: t3_1vbjtlo
+author: MaxWildman
+created_utc: 1786754451
+edited: false
+body:
+Trezor users: 
+
+Your funds are safe. Unless someone breaks into your home and starts breaking your knees with a wrench to force you to hand over your keys because trezor gave out your full name and home address to kidnappers and extortionists.\*
