--- reddit-wallet-drained-mk3@20260804T090111Z
+++ reddit-wallet-drained-mk3@20260804T134224Z
@@ -539,3 +539,11 @@
 edited: false
 body:
 Do you think quantum technology was involved to check trillions of combinations in a short time?
+
+comment: p1n034w
+parent: t1_p1m0qw6
+author: na3than
+created_utc: 1785846854
+edited: false
+body:
+I don't, because it isn't necessary for the exploits we've seen so far.
