--- reddit-fidelity-custody-debate@20260809T232756Z
+++ reddit-fidelity-custody-debate@20260812T044325Z
@@ -47,11 +47,11 @@
 
 comment: p2ds8ri
 parent: t1_p2drty2
-author: MCHappster1
+author: [deleted]
 created_utc: 1786153502
 edited: false
 body:
-You should be more worried about the government and what they can make Fidelity do with your coins. Not your keys not your coins.
+[deleted]
 
 comment: p2dsb0s
 parent: t3_1viix1b
@@ -181,11 +181,11 @@
 
 comment: p2dzh5y
 parent: t1_p2ds8ri
-author: wickedbuzzard
+author: [deleted]
 created_utc: 1786156024
 edited: false
 body:
-Turns out, your keys, also not your coin. Shrug
+[removed]
 
 comment: p2dzk9m
 parent: t3_1viix1b
@@ -648,3 +648,51 @@
 edited: false
 body:
 It has its benefits, but it basically makes you Fidelity's b****. They now control your bitcoin and get to decide what you can do with it. A lot of the benefit of Bitcoin is that you get to control your money without the permission of a financial institution.
+
+comment: p2yrk96
+parent: t1_p2kajzr
+author: MarloChrisSnoop
+created_utc: 1786417226
+edited: false
+body:
+How does that work? Do they just allow you access to buy Bitcoin as part of your contribution limit? Wondering how that looks like interface wise on the website or app.
+
+comment: p2yry7c
+parent: t1_p2epwf4
+author: MarloChrisSnoop
+created_utc: 1786417370
+edited: false
+body:
+Well said. Gives me more peace of mind. 
+
+comment: p2ys8jc
+parent: t1_p2febjc
+author: MarloChrisSnoop
+created_utc: 1786417480
+edited: false
+body:
+I think the main lesson learned about Cold Card recently is how your seed is generated. As long as it's not generated any company's TRNG and you ensure true random entropy through dice, cards, etc, then it doesn't matter what hardware wallet you use. 
+
+comment: p2ysegz
+parent: t1_p2jh8e4
+author: MarloChrisSnoop
+created_utc: 1786417544
+edited: false
+body:
+To go even safer though, SeedSigner is the ultimate peace of mind. 
+
+comment: p2zql2v
+parent: t1_p2ysegz
+author: JumpProfessional3372
+created_utc: 1786432514
+edited: false
+body:
+I might get one as second (backup) wallet in case MK4 dies or something. 
+
+comment: p315u5z
+parent: t1_p2yrk96
+author: Stryker406
+created_utc: 1786454002
+edited: false
+body:
+Not exactly sure as I just put rollover funds into it but I think you can 
