--- coldcard-store-firmware-notice@20260802T210143Z
+++ coldcard-store-firmware-notice@20260805T135014Z
@@ -15,7 +15,7 @@
 $289
 $249.21
 COLDCARD™ Q
-AFFECTED PRODUCT: Purchased units will ship with corrected firmware, but remember to upgrade older devices.
+New units ship with corrected firmware. Remember to always upgrade on receipt.
 COLDCARD Q with QR scanner, 3xAAA battery support, dual SD card slots, qwerty keyboard and huge new screen. Includes all Mk5 features. Learn more here
 or see all colours.
 Batteries & USB cable not included.
@@ -39,7 +39,7 @@
 $189
 $169.94
 COLDCARD™ Mk5
-AFFECTED PRODUCT: Purchased units will ship with corrected firmware, but remember to upgrade older devices.
+New units ship with corrected firmware. Remember to always upgrade on receipt.
 COLDCARD Mk5 with improved keyboard,
 bigger 1.54" Gorilla Glass screen, better NFC, and full backward compatibility for easy migration from older devices.
 High demand, expect some delays.
@@ -132,7 +132,7 @@
 $189
 $169.94
 Blue Mk5
-AFFECTED PRODUCT: Purchased units will ship with corrected firmware, but remember to upgrade older devices.
+New units ship with corrected firmware. Remember to always upgrade on receipt.
 Normal COLDCARD Mk5 but in  Blue!
 USB cable not included.
 Quantity 1
@@ -154,7 +154,7 @@
 $189
 $169.94
 Orange Mk5
-AFFECTED PRODUCT: Purchased units will ship with corrected firmware, but remember to upgrade older devices.
+New units ship with corrected firmware. Remember to always upgrade on receipt.
 Normal COLDCARD Mk5 but in  Orange!
 USB cable not included.
 Quantity 1
@@ -176,7 +176,7 @@
 $189
 $169.94
 Purple Mk5
-AFFECTED PRODUCT: Purchased units will ship with corrected firmware, but remember to upgrade older devices.
+New units ship with corrected firmware. Remember to always upgrade on receipt.
 Normal COLDCARD Mk5 but in  Purple!
 USB cable not included.
 Quantity 1
@@ -198,7 +198,7 @@
 $189
 $169.94
 Black Mk5
-AFFECTED PRODUCT: Purchased units will ship with corrected firmware, but remember to upgrade older devices.
+New units ship with corrected firmware. Remember to always upgrade on receipt.
 Normal COLDCARD Mk5 but in  Black!
 USB cable not included.
 Quantity 1
@@ -220,7 +220,7 @@
 $189
 $169.94
 Red Mk5
-AFFECTED PRODUCT: Purchased units will ship with corrected firmware, but remember to upgrade older devices.
+New units ship with corrected firmware. Remember to always upgrade on receipt.
 Normal COLDCARD Mk5 but in  Red!
 USB cable not included.
 Quantity 1
@@ -242,7 +242,7 @@
 $189
 $169.94
 Gold Mk5
-AFFECTED PRODUCT: Purchased units will ship with corrected firmware, but remember to upgrade older devices.
+New units ship with corrected firmware. Remember to always upgrade on receipt.
 COLDCARD "Schiff"
 Gold Edition ,
 because industrial use is not a good store-of-value.
@@ -266,7 +266,7 @@
 $189
 $169.94
 Glow-In-The-Dark Mk5
-AFFECTED PRODUCT: Purchased units will ship with corrected firmware, but remember to upgrade older devices.
+New units ship with corrected firmware. Remember to always upgrade on receipt.
 Normal COLDCARD Mk5 but in
 Glow-in-the-Dark!  Great for your
 deepest cold-storage cave. In normal light, it looks white/clear.
@@ -290,7 +290,7 @@
 $189
 $169.94
 Jade Mk5
-AFFECTED PRODUCT: Purchased units will ship with corrected firmware, but remember to upgrade older devices.
+New units ship with corrected firmware. Remember to always upgrade on receipt.
 COLDCARD Mk5  Jade  because it's a great green color!
 USB cable not included.
 Quantity 1
@@ -312,7 +312,7 @@
 $289
 $249.21
 Jade Q
-AFFECTED PRODUCT: Purchased units will ship with corrected firmware, but remember to upgrade older devices.
+New units ship with corrected firmware. Remember to always upgrade on receipt.
 COLDCARD Q  Jade  because it's a great green color!
 Batteries & USB cable not included.
 Quantity 1
@@ -334,7 +334,7 @@
 $289
 $249.21
 Blue Q
-AFFECTED PRODUCT: Purchased units will ship with corrected firmware, but remember to upgrade older devices.
+New units ship with corrected firmware. Remember to always upgrade on receipt.
 Normal COLDCARD Q but in  Blue!
 Batteries & USB cable not included.
 Quantity 1
@@ -356,7 +356,7 @@
 $289
 $249.21
 Orange Q
-AFFECTED PRODUCT: Purchased units will ship with corrected firmware, but remember to upgrade older devices.
+New units ship with corrected firmware. Remember to always upgrade on receipt.
 Normal COLDCARD Q but in  Orange!
 Batteries & USB cable not included.
 Quantity 1
@@ -378,7 +378,7 @@
 $289
 $249.21
 Black Q
-AFFECTED PRODUCT: Purchased units will ship with corrected firmware, but remember to upgrade older devices.
+New units ship with corrected firmware. Remember to always upgrade on receipt.
 Normal COLDCARD Q but in  Black!
 Batteries & USB cable not included.
 Quantity 1
@@ -400,7 +400,7 @@
 $289
 $249.21
 Red Q
-AFFECTED PRODUCT: Purchased units will ship with corrected firmware, but remember to upgrade older devices.
+New units ship with corrected firmware. Remember to always upgrade on receipt.
 Normal COLDCARD Q but in  Red!
 Batteries & USB cable not included.
 Quantity 1
@@ -422,7 +422,7 @@
 $289
 $249.21
 Gold Q
-AFFECTED PRODUCT: Purchased units will ship with corrected firmware, but remember to upgrade older devices.
+New units ship with corrected firmware. Remember to always upgrade on receipt.
 COLDCARD Q "Schiff"
 Gold Edition ,
 because industrial use is not a good store-of-value.
@@ -446,7 +446,7 @@
 $289
 $249.21
 Glow-In-The-Dark Q
-AFFECTED PRODUCT: Purchased units will ship with corrected firmware, but remember to upgrade older devices.
+New units ship with corrected firmware. Remember to always upgrade on receipt.
 Normal COLDCARD Q but in
 Glow-in-the-Dark!  Great for your
 deepest cold-storage cave. In normal light, it looks white/clear.
@@ -470,7 +470,7 @@
 $289
 $249.21
 Purple Q
-AFFECTED PRODUCT: Purchased units will ship with corrected firmware, but remember to upgrade older devices.
+New units ship with corrected firmware. Remember to always upgrade on receipt.
 Normal COLDCARD Q but in  Purple!
 Batteries & USB cable not included.
 Quantity 1
