--- chaincatcher-nunchuk-response@20260804T144820Z
+++ chaincatcher-nunchuk-response@20260804T151842Z
@@ -5,36 +5,36 @@
 Calendar
 Data
 Find
-BTC $64,053.12 +0.62%
-ETH $1,872.51 +0.54%
-BNB $593.24 +0.78%
-XRP $1.07 -0.74%
-SOL $73.79 +0.42%
-TRX $0.3294 +0.39%
-DOGE $0.0701 -0.20%
-ADA $0.1928 +1.79%
-BCH $213.91 +1.37%
-LINK $8.25 -0.43%
-HYPE $55.23 +2.62%
-AAVE $91.27 -1.93%
-SUI $0.6909 -0.50%
-XLM $0.1693 -1.57%
-ZEC $490.46 -0.23%
-BTC $64,053.12 +0.62%
-ETH $1,872.51 +0.54%
-BNB $593.24 +0.78%
-XRP $1.07 -0.74%
-SOL $73.79 +0.42%
-TRX $0.3294 +0.39%
-DOGE $0.0701 -0.20%
-ADA $0.1928 +1.79%
-BCH $213.91 +1.37%
-LINK $8.25 -0.43%
-HYPE $55.23 +2.62%
-AAVE $91.27 -1.93%
-SUI $0.6909 -0.50%
-XLM $0.1693 -1.57%
-ZEC $490.46 -0.23%
+BTC $63,982.66 +0.45%
+ETH $1,868.06 +0.27%
+BNB $592.83 +0.43%
+XRP $1.07 -0.49%
+SOL $73.79 +0.37%
+TRX $0.3293 +0.23%
+DOGE $0.0701 -0.12%
+ADA $0.1926 +0.53%
+BCH $214.33 +1.70%
+LINK $8.19 -0.72%
+HYPE $55.32 +2.36%
+AAVE $90.61 -3.32%
+SUI $0.6905 -0.38%
+XLM $0.1693 -1.24%
+ZEC $491.91 +0.53%
+BTC $63,982.66 +0.45%
+ETH $1,868.06 +0.27%
+BNB $592.83 +0.43%
+XRP $1.07 -0.49%
+SOL $73.79 +0.37%
+TRX $0.3293 +0.23%
+DOGE $0.0701 -0.12%
+ADA $0.1926 +0.53%
+BCH $214.33 +1.70%
+LINK $8.19 -0.72%
+HYPE $55.32 +2.36%
+AAVE $90.61 -3.32%
+SUI $0.6905 -0.38%
+XLM $0.1693 -1.24%
+ZEC $491.91 +0.53%
 Home
 Article
 Flash
@@ -64,12 +64,12 @@
 Related reading
 Web3 development tool provider Cubist launches hardware key management service
 2023-04-18 22:02
+Data: Polymarket's monthly visits reached 43.1 million, surpassing mainstream financial applications
+2026-08-04 23:07
 Blackstone plans to raise at least $36 billion in debt financing for Anthropic, aimed at purchasing AI chips
 2026-08-04 22:30
 Citigroup increased its holdings by 238,538 shares of Strategy, with a position value reaching 90.5 million USD
 2026-08-04 21:48
-Dinari and Circle launch a tokenized US stock platform, planning to put S&P 500 constituent stocks on the blockchain
-2026-08-04 21:39
 Related tags
 Nunchuk
 Coldcard Mk4
