--- chaincatcher-nunchuk-response@20260803T091026Z
+++ chaincatcher-nunchuk-response@20260803T093843Z
@@ -5,36 +5,36 @@
 Calendar
 Data
 Find
-BTC $62,445.37 -1.01%
-ETH $1,837.37 -1.42%
-BNB $583.33 +0.04%
-XRP $1.06 -1.06%
-SOL $72.26 -1.04%
-TRX $0.3259 -0.35%
-DOGE $0.0692 -0.95%
-ADA $0.1836 -1.02%
-BCH $209.39 -1.27%
-LINK $8.21 -1.27%
-HYPE $52.43 +0.88%
-AAVE $91.61 +0.12%
-SUI $0.6840 -0.68%
-XLM $0.1705 -2.54%
-ZEC $473.38 -0.02%
-BTC $62,445.37 -1.01%
-ETH $1,837.37 -1.42%
-BNB $583.33 +0.04%
-XRP $1.06 -1.06%
-SOL $72.26 -1.04%
-TRX $0.3259 -0.35%
-DOGE $0.0692 -0.95%
-ADA $0.1836 -1.02%
-BCH $209.39 -1.27%
-LINK $8.21 -1.27%
-HYPE $52.43 +0.88%
-AAVE $91.61 +0.12%
-SUI $0.6840 -0.68%
-XLM $0.1705 -2.54%
-ZEC $473.38 -0.02%
+BTC $62,559.80 -1.07%
+ETH $1,842.26 -1.51%
+BNB $584.37 +0.20%
+XRP $1.06 -1.31%
+SOL $72.38 -1.23%
+TRX $0.3269 +0.02%
+DOGE $0.0693 -1.06%
+ADA $0.1844 -1.23%
+BCH $209.81 -1.13%
+LINK $8.22 -1.84%
+HYPE $52.75 +1.56%
+AAVE $91.90 +0.10%
+SUI $0.6847 -0.71%
+XLM $0.1708 -2.64%
+ZEC $474.44 +0.01%
+BTC $62,559.80 -1.07%
+ETH $1,842.26 -1.51%
+BNB $584.37 +0.20%
+XRP $1.06 -1.31%
+SOL $72.38 -1.23%
+TRX $0.3269 +0.02%
+DOGE $0.0693 -1.06%
+ADA $0.1844 -1.23%
+BCH $209.81 -1.13%
+LINK $8.22 -1.84%
+HYPE $52.75 +1.56%
+AAVE $91.90 +0.10%
+SUI $0.6847 -0.71%
+XLM $0.1708 -2.64%
+ZEC $474.44 +0.01%
 Home
 Article
 Flash
@@ -64,8 +64,12 @@
 Related reading
 Web3 development tool provider Cubist launches hardware key management service
 2023-04-18 22:02
+The dark side of the moon (Kimi) responds to rumors of Hong Kong stock IPO: the news is false
+2026-08-03 17:27
 JPMorgan: The AI sector is unlikely to be the main driver of returns in the second half of the year
 2026-08-03 16:56
+Data: Over the past 7 days, Bybit's net outflow of reserve assets exceeded 126 million USD; Bithumb's BTC wallet balance increased by 2.42%
+2026-08-03 16:36
 Related tags
 Nunchuk
 Coldcard Mk4
