--- chaincatcher-nunchuk-response@20260804T141851Z
+++ chaincatcher-nunchuk-response@20260804T144820Z
@@ -5,36 +5,36 @@
 Calendar
 Data
 Find
-BTC $63,661.54 +0.53%
-ETH $1,861.98 +0.48%
-BNB $589.00 +0.40%
-XRP $1.07 -0.02%
-SOL $73.56 +0.70%
-TRX $0.3294 +0.70%
-DOGE $0.0700 -0.20%
-ADA $0.1920 -0.40%
-BCH $213.51 +1.09%
-LINK $8.19 -0.93%
-HYPE $55.38 +3.60%
-AAVE $91.31 -1.51%
-SUI $0.6895 -0.84%
-XLM $0.1691 -1.79%
-ZEC $487.03 -0.33%
-BTC $63,661.54 +0.53%
-ETH $1,861.98 +0.48%
-BNB $589.00 +0.40%
-XRP $1.07 -0.02%
-SOL $73.56 +0.70%
-TRX $0.3294 +0.70%
-DOGE $0.0700 -0.20%
-ADA $0.1920 -0.40%
-BCH $213.51 +1.09%
-LINK $8.19 -0.93%
-HYPE $55.38 +3.60%
-AAVE $91.31 -1.51%
-SUI $0.6895 -0.84%
-XLM $0.1691 -1.79%
-ZEC $487.03 -0.33%
+BTC $64,053.12 +0.62%
+ETH $1,872.51 +0.54%
+BNB $593.24 +0.78%
+XRP $1.07 -0.74%
+SOL $73.79 +0.42%
+TRX $0.3294 +0.39%
+DOGE $0.0701 -0.20%
+ADA $0.1928 +1.79%
+BCH $213.91 +1.37%
+LINK $8.25 -0.43%
+HYPE $55.23 +2.62%
+AAVE $91.27 -1.93%
+SUI $0.6909 -0.50%
+XLM $0.1693 -1.57%
+ZEC $490.46 -0.23%
+BTC $64,053.12 +0.62%
+ETH $1,872.51 +0.54%
+BNB $593.24 +0.78%
+XRP $1.07 -0.74%
+SOL $73.79 +0.42%
+TRX $0.3294 +0.39%
+DOGE $0.0701 -0.20%
+ADA $0.1928 +1.79%
+BCH $213.91 +1.37%
+LINK $8.25 -0.43%
+HYPE $55.23 +2.62%
+AAVE $91.27 -1.93%
+SUI $0.6909 -0.50%
+XLM $0.1693 -1.57%
+ZEC $490.46 -0.23%
 Home
 Article
 Flash
@@ -64,12 +64,12 @@
 Related reading
 Web3 development tool provider Cubist launches hardware key management service
 2023-04-18 22:02
+Blackstone plans to raise at least $36 billion in debt financing for Anthropic, aimed at purchasing AI chips
+2026-08-04 22:30
 Citigroup increased its holdings by 238,538 shares of Strategy, with a position value reaching 90.5 million USD
 2026-08-04 21:48
 Dinari and Circle launch a tokenized US stock platform, planning to put S&P 500 constituent stocks on the blockchain
 2026-08-04 21:39
-Stablecoin infrastructure provider Yellow Card has completed a $40 million strategic financing round, with participation from SC Ventures and others
-2026-08-04 21:07
 Related tags
 Nunchuk
 Coldcard Mk4
