--- chaincatcher-nunchuk-response@20260804T103754Z
+++ chaincatcher-nunchuk-response@20260804T110900Z
@@ -5,36 +5,36 @@
 Calendar
 Data
 Find
-BTC $63,425.63 +1.32%
-ETH $1,854.50 +0.47%
-BNB $587.67 +0.18%
-XRP $1.07 +0.19%
-SOL $73.09 +0.78%
-TRX $0.3290 +0.43%
-DOGE $0.0700 +0.97%
-ADA $0.1948 +4.09%
-BCH $212.73 +1.23%
-LINK $8.14 -0.89%
-HYPE $54.99 +4.41%
-AAVE $91.99 -0.14%
-SUI $0.6894 +0.70%
-XLM $0.1697 -0.84%
-ZEC $485.53 +1.35%
-BTC $63,425.63 +1.32%
-ETH $1,854.50 +0.47%
-BNB $587.67 +0.18%
-XRP $1.07 +0.19%
-SOL $73.09 +0.78%
-TRX $0.3290 +0.43%
-DOGE $0.0700 +0.97%
-ADA $0.1948 +4.09%
-BCH $212.73 +1.23%
-LINK $8.14 -0.89%
-HYPE $54.99 +4.41%
-AAVE $91.99 -0.14%
-SUI $0.6894 +0.70%
-XLM $0.1697 -0.84%
-ZEC $485.53 +1.35%
+BTC $63,697.24 +1.73%
+ETH $1,865.48 +1.29%
+BNB $589.42 +0.53%
+XRP $1.07 +0.63%
+SOL $73.54 +1.51%
+TRX $0.3288 +0.34%
+DOGE $0.0703 +1.35%
+ADA $0.1946 +4.20%
+BCH $213.71 +1.72%
+LINK $8.19 -0.07%
+HYPE $55.28 +4.88%
+AAVE $91.96 -0.08%
+SUI $0.6919 +1.07%
+XLM $0.1705 -0.41%
+ZEC $488.51 +1.86%
+BTC $63,697.24 +1.73%
+ETH $1,865.48 +1.29%
+BNB $589.42 +0.53%
+XRP $1.07 +0.63%
+SOL $73.54 +1.51%
+TRX $0.3288 +0.34%
+DOGE $0.0703 +1.35%
+ADA $0.1946 +4.20%
+BCH $213.71 +1.72%
+LINK $8.19 -0.07%
+HYPE $55.28 +4.88%
+AAVE $91.96 -0.08%
+SUI $0.6919 +1.07%
+XLM $0.1705 -0.41%
+ZEC $488.51 +1.86%
 Home
 Article
 Flash
@@ -64,12 +64,12 @@
 Related reading
 Web3 development tool provider Cubist launches hardware key management service
 2023-04-18 22:02
+RootData Launches Beta Version of Stock Spot Trading Exchange Rankings
+2026-08-04 18:51
 UPBIT lists Squid (QUID) and supports KRW, BTC, USDT trading
 2026-08-04 18:00
 Coinbase International Exchange will be integrated into Deribit, and institutional derivatives accounts will be unified and migrated
 2026-08-04 17:41
-Binance will delist some spot trading pairs
-2026-08-04 17:33
 Related tags
 Nunchuk
 Coldcard Mk4
