--- chaincatcher-nunchuk-response@20260804T030839Z
+++ chaincatcher-nunchuk-response@20260804T100853Z
@@ -5,36 +5,36 @@
 Calendar
 Data
 Find
-BTC $63,754.05 +1.08%
-ETH $1,864.05 +0.10%
-BNB $590.49 +0.94%
-XRP $1.07 +0.46%
-SOL $73.79 +1.11%
-TRX $0.3288 +0.83%
-DOGE $0.0703 +0.61%
-ADA $0.1954 +4.72%
-BCH $212.86 +0.69%
-LINK $8.19 -1.14%
-HYPE $54.19 +3.49%
-AAVE $92.51 +1.08%
-SUI $0.6959 +1.30%
-XLM $0.1705 -0.77%
-ZEC $484.44 +1.69%
-BTC $63,754.05 +1.08%
-ETH $1,864.05 +0.10%
-BNB $590.49 +0.94%
-XRP $1.07 +0.46%
-SOL $73.79 +1.11%
-TRX $0.3288 +0.83%
-DOGE $0.0703 +0.61%
-ADA $0.1954 +4.72%
-BCH $212.86 +0.69%
-LINK $8.19 -1.14%
-HYPE $54.19 +3.49%
-AAVE $92.51 +1.08%
-SUI $0.6959 +1.30%
-XLM $0.1705 -0.77%
-ZEC $484.44 +1.69%
+BTC $63,480.85 +1.23%
+ETH $1,854.85 +0.31%
+BNB $588.34 +0.45%
+XRP $1.07 +0.17%
+SOL $73.12 +0.70%
+TRX $0.3291 +0.48%
+DOGE $0.0700 +0.65%
+ADA $0.1960 +4.95%
+BCH $212.95 +1.18%
+LINK $8.15 -0.95%
+HYPE $54.67 +3.61%
+AAVE $91.89 -0.44%
+SUI $0.6906 +0.68%
+XLM $0.1700 -0.82%
+ZEC $485.18 +2.08%
+BTC $63,480.85 +1.23%
+ETH $1,854.85 +0.31%
+BNB $588.34 +0.45%
+XRP $1.07 +0.17%
+SOL $73.12 +0.70%
+TRX $0.3291 +0.48%
+DOGE $0.0700 +0.65%
+ADA $0.1960 +4.95%
+BCH $212.95 +1.18%
+LINK $8.15 -0.95%
+HYPE $54.67 +3.61%
+AAVE $91.89 -0.44%
+SUI $0.6906 +0.68%
+XLM $0.1700 -0.82%
+ZEC $485.18 +2.08%
 Home
 Article
 Flash
@@ -64,12 +64,12 @@
 Related reading
 Web3 development tool provider Cubist launches hardware key management service
 2023-04-18 22:02
-Telegram has resumed its availability on the Apple App Store
-2026-08-04 10:49
-SPCX rebounded by 9.5%, with large addresses opening $27.4 million in long positions in one day
-2026-08-04 10:38
-RISEx XLP vault associated RWA strategy experienced unauthorized withdrawals exceeding $670,000, and user funds have been fully compensated
-2026-08-04 10:26
+UPBIT lists Squid (QUID) and supports KRW, BTC, USDT trading
+2026-08-04 18:00
+Coinbase International Exchange will be integrated into Deribit, and institutional derivatives accounts will be unified and migrated
+2026-08-04 17:41
+Binance will delist some spot trading pairs
+2026-08-04 17:33
 Related tags
 Nunchuk
 Coldcard Mk4
@@ -78,7 +78,6 @@
 About Us
 Media Kit
 Apply column
-AWS
 Privacy Policy
 Risk Warning
 Hiring
