--- chaincatcher-nunchuk-response@20260803T204904Z
+++ chaincatcher-nunchuk-response@20260804T030839Z
@@ -5,36 +5,36 @@
 Calendar
 Data
 Find
-BTC $63,857.15 +0.78%
-ETH $1,870.21 -0.49%
-BNB $591.79 +0.64%
-XRP $1.08 +0.05%
-SOL $73.91 +0.44%
-TRX $0.3286 +0.49%
-DOGE $0.0704 -0.08%
-ADA $0.1944 +3.75%
-BCH $214.30 +0.43%
-LINK $8.23 -1.28%
-HYPE $54.66 +4.02%
-AAVE $93.21 +0.91%
-SUI $0.6939 +0.44%
-XLM $0.1722 -0.95%
-ZEC $486.47 -0.42%
-BTC $63,857.15 +0.78%
-ETH $1,870.21 -0.49%
-BNB $591.79 +0.64%
-XRP $1.08 +0.05%
-SOL $73.91 +0.44%
-TRX $0.3286 +0.49%
-DOGE $0.0704 -0.08%
-ADA $0.1944 +3.75%
-BCH $214.30 +0.43%
-LINK $8.23 -1.28%
-HYPE $54.66 +4.02%
-AAVE $93.21 +0.91%
-SUI $0.6939 +0.44%
-XLM $0.1722 -0.95%
-ZEC $486.47 -0.42%
+BTC $63,754.05 +1.08%
+ETH $1,864.05 +0.10%
+BNB $590.49 +0.94%
+XRP $1.07 +0.46%
+SOL $73.79 +1.11%
+TRX $0.3288 +0.83%
+DOGE $0.0703 +0.61%
+ADA $0.1954 +4.72%
+BCH $212.86 +0.69%
+LINK $8.19 -1.14%
+HYPE $54.19 +3.49%
+AAVE $92.51 +1.08%
+SUI $0.6959 +1.30%
+XLM $0.1705 -0.77%
+ZEC $484.44 +1.69%
+BTC $63,754.05 +1.08%
+ETH $1,864.05 +0.10%
+BNB $590.49 +0.94%
+XRP $1.07 +0.46%
+SOL $73.79 +1.11%
+TRX $0.3288 +0.83%
+DOGE $0.0703 +0.61%
+ADA $0.1954 +4.72%
+BCH $212.86 +0.69%
+LINK $8.19 -1.14%
+HYPE $54.19 +3.49%
+AAVE $92.51 +1.08%
+SUI $0.6959 +1.30%
+XLM $0.1705 -0.77%
+ZEC $484.44 +1.69%
 Home
 Article
 Flash
@@ -64,12 +64,12 @@
 Related reading
 Web3 development tool provider Cubist launches hardware key management service
 2023-04-18 22:02
-The Tie launches SEC-registered broker The Tie Capital, providing digital asset investment banking services
-2026-08-04 00:45
-The president of American Bitcoin, a Bitcoin mining company associated with Trump, has resigned to join an AI energy infrastructure company
-2026-08-03 23:51
-Willy Woo: Relying solely on ETFs and custody cannot achieve the sovereign attributes of Bitcoin; it is still recommended to allocate 15% of assets in BTC and gold
-2026-08-03 23:27
+Telegram has resumed its availability on the Apple App Store
+2026-08-04 10:49
+SPCX rebounded by 9.5%, with large addresses opening $27.4 million in long positions in one day
+2026-08-04 10:38
+RISEx XLP vault associated RWA strategy experienced unauthorized withdrawals exceeding $670,000, and user funds have been fully compensated
+2026-08-04 10:26
 Related tags
 Nunchuk
 Coldcard Mk4
