--- chaincatcher-nunchuk-response@20260804T120921Z
+++ chaincatcher-nunchuk-response@20260804T123824Z
@@ -5,36 +5,36 @@
 Calendar
 Data
 Find
-BTC $63,802.47 +2.07%
-ETH $1,869.44 +1.71%
-BNB $590.86 +0.97%
-XRP $1.07 +1.23%
-SOL $73.75 +1.97%
-TRX $0.3290 +0.57%
-DOGE $0.0704 +1.69%
-ADA $0.1939 +4.19%
-BCH $213.94 +2.06%
-LINK $8.19 +0.32%
-HYPE $55.48 +5.24%
-AAVE $91.97 +0.05%
-SUI $0.6935 +1.29%
-XLM $0.1705 -0.52%
-ZEC $486.38 +0.90%
-BTC $63,802.47 +2.07%
-ETH $1,869.44 +1.71%
-BNB $590.86 +0.97%
-XRP $1.07 +1.23%
-SOL $73.75 +1.97%
-TRX $0.3290 +0.57%
-DOGE $0.0704 +1.69%
-ADA $0.1939 +4.19%
-BCH $213.94 +2.06%
-LINK $8.19 +0.32%
-HYPE $55.48 +5.24%
-AAVE $91.97 +0.05%
-SUI $0.6935 +1.29%
-XLM $0.1705 -0.52%
-ZEC $486.38 +0.90%
+BTC $63,776.20 +1.95%
+ETH $1,870.11 +1.62%
+BNB $590.15 +0.64%
+XRP $1.07 +1.06%
+SOL $73.88 +2.07%
+TRX $0.3298 +0.80%
+DOGE $0.0703 +1.32%
+ADA $0.1944 +3.60%
+BCH $213.48 +1.76%
+LINK $8.19 +0.45%
+HYPE $55.51 +5.03%
+AAVE $91.66 -0.27%
+SUI $0.6925 +0.99%
+XLM $0.1699 -0.72%
+ZEC $485.99 +0.56%
+BTC $63,776.20 +1.95%
+ETH $1,870.11 +1.62%
+BNB $590.15 +0.64%
+XRP $1.07 +1.06%
+SOL $73.88 +2.07%
+TRX $0.3298 +0.80%
+DOGE $0.0703 +1.32%
+ADA $0.1944 +3.60%
+BCH $213.48 +1.76%
+LINK $8.19 +0.45%
+HYPE $55.51 +5.03%
+AAVE $91.66 -0.27%
+SUI $0.6925 +0.99%
+XLM $0.1699 -0.72%
+ZEC $485.99 +0.56%
 Home
 Article
 Flash
@@ -64,12 +64,12 @@
 Related reading
 Web3 development tool provider Cubist launches hardware key management service
 2023-04-18 22:02
+Polymarket seeks to raise funds with a valuation of over $20 billion
+2026-08-04 20:18
 OpenAI founders and employees collectively hold 48.93% of the company's shares
 2026-08-04 19:39
 The Trump administration drafted an order to ban the import of Chinese-made data center optical modules
 2026-08-04 19:17
-RootData Launches Beta Version of Stock Spot Trading Exchange Rankings
-2026-08-04 18:51
 Related tags
 Nunchuk
 Coldcard Mk4
