--- chaincatcher-nunchuk-response@20260803T151303Z
+++ chaincatcher-nunchuk-response@20260803T204904Z
@@ -5,36 +5,36 @@
 Calendar
 Data
 Find
-BTC $63,755.77 +1.14%
-ETH $1,865.09 +0.57%
-BNB $588.95 +0.87%
-XRP $1.07 -0.22%
-SOL $73.56 +0.66%
-TRX $0.3281 +0.26%
-DOGE $0.0702 -0.16%
-ADA $0.1904 +0.28%
-BCH $210.91 -0.49%
-LINK $8.27 -0.12%
-HYPE $54.10 +5.10%
-AAVE $93.59 +2.33%
-SUI $0.6934 +0.55%
-XLM $0.1718 -1.66%
-ZEC $489.60 +3.88%
-BTC $63,755.77 +1.14%
-ETH $1,865.09 +0.57%
-BNB $588.95 +0.87%
-XRP $1.07 -0.22%
-SOL $73.56 +0.66%
-TRX $0.3281 +0.26%
-DOGE $0.0702 -0.16%
-ADA $0.1904 +0.28%
-BCH $210.91 -0.49%
-LINK $8.27 -0.12%
-HYPE $54.10 +5.10%
-AAVE $93.59 +2.33%
-SUI $0.6934 +0.55%
-XLM $0.1718 -1.66%
-ZEC $489.60 +3.88%
+BTC $63,857.15 +0.78%
+ETH $1,870.21 -0.49%
+BNB $591.79 +0.64%
+XRP $1.08 +0.05%
+SOL $73.91 +0.44%
+TRX $0.3286 +0.49%
+DOGE $0.0704 -0.08%
+ADA $0.1944 +3.75%
+BCH $214.30 +0.43%
+LINK $8.23 -1.28%
+HYPE $54.66 +4.02%
+AAVE $93.21 +0.91%
+SUI $0.6939 +0.44%
+XLM $0.1722 -0.95%
+ZEC $486.47 -0.42%
+BTC $63,857.15 +0.78%
+ETH $1,870.21 -0.49%
+BNB $591.79 +0.64%
+XRP $1.08 +0.05%
+SOL $73.91 +0.44%
+TRX $0.3286 +0.49%
+DOGE $0.0704 -0.08%
+ADA $0.1944 +3.75%
+BCH $214.30 +0.43%
+LINK $8.23 -1.28%
+HYPE $54.66 +4.02%
+AAVE $93.21 +0.91%
+SUI $0.6939 +0.44%
+XLM $0.1722 -0.95%
+ZEC $486.47 -0.42%
 Home
 Article
 Flash
@@ -64,10 +64,12 @@
 Related reading
 Web3 development tool provider Cubist launches hardware key management service
 2023-04-18 22:02
-South Korea's increase in margin thresholds for leveraged ETFs has taken effect, with the trading volume of single-stock leveraged products dropping to one-tenth of its peak
-2026-08-03 22:53
-Data: A Bitcoin ancient miner transfers 500 BTC after 13 years of silence, currently valued at over 31 million dollars
-2026-08-03 22:05
+The Tie launches SEC-registered broker The Tie Capital, providing digital asset investment banking services
+2026-08-04 00:45
+The president of American Bitcoin, a Bitcoin mining company associated with Trump, has resigned to join an AI energy infrastructure company
+2026-08-03 23:51
+Willy Woo: Relying solely on ETFs and custody cannot achieve the sovereign attributes of Bitcoin; it is still recommended to allocate 15% of assets in BTC and gold
+2026-08-03 23:27
 Related tags
 Nunchuk
 Coldcard Mk4
