--- chaincatcher-nunchuk-response@20260805T002111Z
+++ chaincatcher-nunchuk-response@20260805T005107Z
@@ -5,36 +5,36 @@
 Calendar
 Data
 Find
-BTC $63,978.33 +0.97%
-ETH $1,865.68 +0.53%
-BNB $594.12 +0.89%
-XRP $1.07 -0.23%
-SOL $73.62 +0.26%
-TRX $0.3265 -0.71%
-DOGE $0.0699 -0.23%
-ADA $0.1925 -0.33%
-BCH $213.13 +0.03%
-LINK $8.13 -0.41%
-HYPE $55.01 +2.23%
-AAVE $90.52 -2.09%
-SUI $0.6915 +0.13%
-XLM $0.1680 -1.72%
-ZEC $503.30 +4.26%
-BTC $63,978.33 +0.97%
-ETH $1,865.68 +0.53%
-BNB $594.12 +0.89%
-XRP $1.07 -0.23%
-SOL $73.62 +0.26%
-TRX $0.3265 -0.71%
-DOGE $0.0699 -0.23%
-ADA $0.1925 -0.33%
-BCH $213.13 +0.03%
-LINK $8.13 -0.41%
-HYPE $55.01 +2.23%
-AAVE $90.52 -2.09%
-SUI $0.6915 +0.13%
-XLM $0.1680 -1.72%
-ZEC $503.30 +4.26%
+BTC $63,998.43 +1.07%
+ETH $1,864.96 +0.89%
+BNB $594.80 +0.97%
+XRP $1.07 +0.06%
+SOL $73.51 +0.55%
+TRX $0.3263 -0.69%
+DOGE $0.0698 -0.10%
+ADA $0.1915 -0.27%
+BCH $213.00 +0.62%
+LINK $8.11 +0.04%
+HYPE $54.81 +2.21%
+AAVE $90.54 -1.58%
+SUI $0.6904 +0.24%
+XLM $0.1668 -2.22%
+ZEC $503.89 +4.91%
+BTC $63,998.43 +1.07%
+ETH $1,864.96 +0.89%
+BNB $594.80 +0.97%
+XRP $1.07 +0.06%
+SOL $73.51 +0.55%
+TRX $0.3263 -0.69%
+DOGE $0.0698 -0.10%
+ADA $0.1915 -0.27%
+BCH $213.00 +0.62%
+LINK $8.11 +0.04%
+HYPE $54.81 +2.21%
+AAVE $90.54 -1.58%
+SUI $0.6904 +0.24%
+XLM $0.1668 -2.22%
+ZEC $503.89 +4.91%
 Home
 Article
 Flash
@@ -64,12 +64,12 @@
 Related reading
 Web3 development tool provider Cubist launches hardware key management service
 2023-04-18 22:02
+The blockchain game studio Proof of Play has announced the cessation of operations and will open source the Pirate Nation code and art assets
+2026-08-05 08:19
 Musk: SpaceX aims to achieve $1 trillion in revenue by 2030
 2026-08-05 05:10
 Musk: GROK 4.6 is expected to be released next week
 2026-08-05 04:41
-Transaction costs have been reduced to about 2 basis points, and BlackRock will conduct a 1-for-3 reverse split of ETHA
-2026-08-05 04:27
 Related tags
 Nunchuk
 Coldcard Mk4
