--- chaincatcher-nunchuk-response@20260803T020847Z
+++ chaincatcher-nunchuk-response@20260803T023436Z
@@ -5,36 +5,36 @@
 Calendar
 Data
 Find
-BTC $63,089.91 +0.28%
-ETH $1,863.75 +0.48%
-BNB $583.58 +1.18%
-XRP $1.07 +0.40%
-SOL $72.94 +0.13%
-TRX $0.3259 -0.61%
-DOGE $0.0699 +0.59%
-ADA $0.1853 +5.21%
-BCH $211.49 +1.90%
-LINK $8.32 +2.09%
-HYPE $52.49 +0.01%
-AAVE $91.48 +0.36%
-SUI $0.6857 +0.53%
-XLM $0.1723 +0.40%
-ZEC $479.17 +2.70%
-BTC $63,089.91 +0.28%
-ETH $1,863.75 +0.48%
-BNB $583.58 +1.18%
-XRP $1.07 +0.40%
-SOL $72.94 +0.13%
-TRX $0.3259 -0.61%
-DOGE $0.0699 +0.59%
-ADA $0.1853 +5.21%
-BCH $211.49 +1.90%
-LINK $8.32 +2.09%
-HYPE $52.49 +0.01%
-AAVE $91.48 +0.36%
-SUI $0.6857 +0.53%
-XLM $0.1723 +0.40%
-ZEC $479.17 +2.70%
+BTC $63,031.69 -0.52%
+ETH $1,864.14 -0.57%
+BNB $584.74 +0.88%
+XRP $1.07 +0.17%
+SOL $72.95 -0.03%
+TRX $0.3259 -0.64%
+DOGE $0.0700 +0.11%
+ADA $0.1864 +4.27%
+BCH $211.82 +1.28%
+LINK $8.30 +0.57%
+HYPE $52.36 -0.81%
+AAVE $91.37 -1.05%
+SUI $0.6864 -0.15%
+XLM $0.1721 -0.46%
+ZEC $477.83 +1.29%
+BTC $63,031.69 -0.52%
+ETH $1,864.14 -0.57%
+BNB $584.74 +0.88%
+XRP $1.07 +0.17%
+SOL $72.95 -0.03%
+TRX $0.3259 -0.64%
+DOGE $0.0700 +0.11%
+ADA $0.1864 +4.27%
+BCH $211.82 +1.28%
+LINK $8.30 +0.57%
+HYPE $52.36 -0.81%
+AAVE $91.37 -1.05%
+SUI $0.6864 -0.15%
+XLM $0.1721 -0.46%
+ZEC $477.83 +1.29%
 Home
 Article
 Flash
@@ -64,10 +64,12 @@
 Related reading
 Web3 development tool provider Cubist launches hardware key management service
 2023-04-18 22:02
-NVIDIA announces the mass production of Co-Packaged Optics (CPO), related concept stocks may see a rebound
-2026-08-03 09:57
-The turbulence in the Korean stock market has led to a sharp decline of nearly 20 trillion won in investor deposits
-2026-08-03 09:50
+Data: Two addresses hold a total of 1,364 BTC long positions, worth 86.2 million USD
+2026-08-03 10:28
+Arthur Hayes: Need to pay attention to the Federal Reserve H.4.1 report to confirm whether Japan is using U.S. Treasury collateral to support yen intervention
+2026-08-03 10:14
+Morgan Stanley has upgraded the rating of the South Korean stock market to "overweight," with a target price of 9,000 points
+2026-08-03 10:10
 Related tags
 Nunchuk
 Coldcard Mk4
