--- chaincatcher-nunchuk-response@20260803T134734Z
+++ chaincatcher-nunchuk-response@20260803T141806Z
@@ -5,36 +5,36 @@
 Calendar
 Data
 Find
-BTC $62,812.22 -0.32%
-ETH $1,845.55 -0.43%
-BNB $585.33 +0.58%
-XRP $1.06 -0.90%
-SOL $72.69 -0.23%
-TRX $0.3269 -0.13%
-DOGE $0.0699 -0.25%
-ADA $0.1908 +1.23%
-BCH $209.78 -0.97%
-LINK $8.23 -0.66%
-HYPE $53.21 +3.30%
-AAVE $92.19 +1.25%
-SUI $0.6904 +0.53%
-XLM $0.1716 -1.52%
-ZEC $485.58 +2.83%
-BTC $62,812.22 -0.32%
-ETH $1,845.55 -0.43%
-BNB $585.33 +0.58%
-XRP $1.06 -0.90%
-SOL $72.69 -0.23%
-TRX $0.3269 -0.13%
-DOGE $0.0699 -0.25%
-ADA $0.1908 +1.23%
-BCH $209.78 -0.97%
-LINK $8.23 -0.66%
-HYPE $53.21 +3.30%
-AAVE $92.19 +1.25%
-SUI $0.6904 +0.53%
-XLM $0.1716 -1.52%
-ZEC $485.58 +2.83%
+BTC $63,320.46 +0.48%
+ETH $1,856.39 +0.21%
+BNB $586.65 +0.77%
+XRP $1.07 -0.51%
+SOL $73.09 +0.07%
+TRX $0.3269 -0.10%
+DOGE $0.0701 +0.07%
+ADA $0.1942 +3.25%
+BCH $211.21 -0.39%
+LINK $8.27 +0.08%
+HYPE $53.41 +3.74%
+AAVE $92.67 +1.93%
+SUI $0.6951 +1.15%
+XLM $0.1722 -1.05%
+ZEC $489.17 +3.72%
+BTC $63,320.46 +0.48%
+ETH $1,856.39 +0.21%
+BNB $586.65 +0.77%
+XRP $1.07 -0.51%
+SOL $73.09 +0.07%
+TRX $0.3269 -0.10%
+DOGE $0.0701 +0.07%
+ADA $0.1942 +3.25%
+BCH $211.21 -0.39%
+LINK $8.27 +0.08%
+HYPE $53.41 +3.74%
+AAVE $92.67 +1.93%
+SUI $0.6951 +1.15%
+XLM $0.1722 -1.05%
+ZEC $489.17 +3.72%
 Home
 Article
 Flash
@@ -64,6 +64,8 @@
 Related reading
 Web3 development tool provider Cubist launches hardware key management service
 2023-04-18 22:02
+Data: A Bitcoin ancient miner transfers 500 BTC after 13 years of silence, currently valued at over 31 million dollars
+2026-08-03 22:05
 BitMine increased its holdings by 10,399 ETH last week, bringing the total holdings to approximately 5.798 million coins
 2026-08-03 20:36
 Bitget will stop providing cryptocurrency trading services for Japanese users and will close all local user positions by the end of the year
