--- chaincatcher-nunchuk-response@20260803T104241Z
+++ chaincatcher-nunchuk-response@20260803T111325Z
@@ -5,36 +5,36 @@
 Calendar
 Data
 Find
-BTC $62,603.34 -0.85%
-ETH $1,844.83 -1.18%
-BNB $586.58 +0.59%
-XRP $1.06 -1.10%
-SOL $72.53 -0.89%
-TRX $0.3274 +0.39%
-DOGE $0.0693 -0.71%
-ADA $0.1872 -0.07%
-BCH $210.03 -0.95%
-LINK $8.21 -1.16%
-HYPE $52.62 +2.17%
-AAVE $92.18 +0.00%
-SUI $0.6851 -0.30%
-XLM $0.1712 -2.04%
-ZEC $478.76 +1.04%
-BTC $62,603.34 -0.85%
-ETH $1,844.83 -1.18%
-BNB $586.58 +0.59%
-XRP $1.06 -1.10%
-SOL $72.53 -0.89%
-TRX $0.3274 +0.39%
-DOGE $0.0693 -0.71%
-ADA $0.1872 -0.07%
-BCH $210.03 -0.95%
-LINK $8.21 -1.16%
-HYPE $52.62 +2.17%
-AAVE $92.18 +0.00%
-SUI $0.6851 -0.30%
-XLM $0.1712 -2.04%
-ZEC $478.76 +1.04%
+BTC $62,563.98 -0.88%
+ETH $1,838.55 -1.53%
+BNB $585.91 +0.48%
+XRP $1.06 -1.17%
+SOL $72.39 -1.09%
+TRX $0.3275 +0.41%
+DOGE $0.0692 -1.02%
+ADA $0.1865 +0.09%
+BCH $210.05 -0.86%
+LINK $8.18 -1.42%
+HYPE $52.64 +2.23%
+AAVE $91.79 -0.39%
+SUI $0.6843 -0.36%
+XLM $0.1710 -2.06%
+ZEC $479.11 +1.15%
+BTC $62,563.98 -0.88%
+ETH $1,838.55 -1.53%
+BNB $585.91 +0.48%
+XRP $1.06 -1.17%
+SOL $72.39 -1.09%
+TRX $0.3275 +0.41%
+DOGE $0.0692 -1.02%
+ADA $0.1865 +0.09%
+BCH $210.05 -0.86%
+LINK $8.18 -1.42%
+HYPE $52.64 +2.23%
+AAVE $91.79 -0.39%
+SUI $0.6843 -0.36%
+XLM $0.1710 -2.06%
+ZEC $479.11 +1.15%
 Home
 Article
 Flash
@@ -64,10 +64,10 @@
 Related reading
 Web3 development tool provider Cubist launches hardware key management service
 2023-04-18 22:02
-Goldman Sachs: Includes Applied Materials (AMAT.O), Delta Air Lines (DAL.N), Microsoft (MSFT.O), United Parcel Service (UPS.N), etc. in the US stock conviction buy list
-2026-08-03 18:28
-Federal Reserve's Williams: If inflation does not return to 2% as expected, the Federal Reserve will take action
-2026-08-03 18:09
+Data: Giant whale Loracle increases gold long positions while simultaneously shorting ETH and SNDK
+2026-08-03 19:00
+American Bitcoin's Bitcoin reserves in the second quarter exceeded 8,000 coins, setting a record for the highest quarterly production
+2026-08-03 18:44
 Related tags
 Nunchuk
 Coldcard Mk4
