--- chaincatcher-nunchuk-response@20260803T131853Z
+++ chaincatcher-nunchuk-response@20260803T134734Z
@@ -5,36 +5,36 @@
 Calendar
 Data
 Find
-BTC $62,679.91 -0.58%
-ETH $1,841.83 -0.68%
-BNB $585.79 +0.77%
-XRP $1.06 -1.26%
-SOL $72.34 -0.73%
-TRX $0.3269 +0.17%
-DOGE $0.0695 -0.74%
-ADA $0.1899 +0.00%
-BCH $209.83 -1.03%
-LINK $8.18 -1.38%
-HYPE $52.89 +2.59%
-AAVE $92.13 +1.31%
-SUI $0.6865 -0.21%
-XLM $0.1713 -1.56%
-ZEC $483.42 +2.16%
-BTC $62,679.91 -0.58%
-ETH $1,841.83 -0.68%
-BNB $585.79 +0.77%
-XRP $1.06 -1.26%
-SOL $72.34 -0.73%
-TRX $0.3269 +0.17%
-DOGE $0.0695 -0.74%
-ADA $0.1899 +0.00%
-BCH $209.83 -1.03%
-LINK $8.18 -1.38%
-HYPE $52.89 +2.59%
-AAVE $92.13 +1.31%
-SUI $0.6865 -0.21%
-XLM $0.1713 -1.56%
-ZEC $483.42 +2.16%
+BTC $62,812.22 -0.32%
+ETH $1,845.55 -0.43%
+BNB $585.33 +0.58%
+XRP $1.06 -0.90%
+SOL $72.69 -0.23%
+TRX $0.3269 -0.13%
+DOGE $0.0699 -0.25%
+ADA $0.1908 +1.23%
+BCH $209.78 -0.97%
+LINK $8.23 -0.66%
+HYPE $53.21 +3.30%
+AAVE $92.19 +1.25%
+SUI $0.6904 +0.53%
+XLM $0.1716 -1.52%
+ZEC $485.58 +2.83%
+BTC $62,812.22 -0.32%
+ETH $1,845.55 -0.43%
+BNB $585.33 +0.58%
+XRP $1.06 -0.90%
+SOL $72.69 -0.23%
+TRX $0.3269 -0.13%
+DOGE $0.0699 -0.25%
+ADA $0.1908 +1.23%
+BCH $209.78 -0.97%
+LINK $8.23 -0.66%
+HYPE $53.21 +3.30%
+AAVE $92.19 +1.25%
+SUI $0.6904 +0.53%
+XLM $0.1716 -1.52%
+ZEC $485.58 +2.83%
 Home
 Article
 Flash
@@ -68,8 +68,6 @@
 2026-08-03 20:36
 Bitget will stop providing cryptocurrency trading services for Japanese users and will close all local user positions by the end of the year
 2026-08-03 20:19
-Data: BlackRock's IBIT transfers over 1,948 BTC to Coinbase Prime
-2026-08-03 20:09
 Related tags
 Nunchuk
 Coldcard Mk4
