--- chaincatcher-nunchuk-response@20260803T121535Z
+++ chaincatcher-nunchuk-response@20260803T124647Z
@@ -5,36 +5,36 @@
 Calendar
 Data
 Find
-BTC $62,420.64 -1.00%
-ETH $1,835.21 -0.94%
-BNB $584.78 +0.54%
-XRP $1.06 -1.51%
-SOL $72.30 -0.98%
-TRX $0.3268 +0.18%
-DOGE $0.0692 -1.12%
-ADA $0.1864 -1.39%
-BCH $209.71 -1.05%
-LINK $8.16 -1.39%
-HYPE $52.71 +2.75%
-AAVE $91.91 +0.58%
-SUI $0.6838 -0.18%
-XLM $0.1712 -1.69%
-ZEC $481.93 +2.51%
-BTC $62,420.64 -1.00%
-ETH $1,835.21 -0.94%
-BNB $584.78 +0.54%
-XRP $1.06 -1.51%
-SOL $72.30 -0.98%
-TRX $0.3268 +0.18%
-DOGE $0.0692 -1.12%
-ADA $0.1864 -1.39%
-BCH $209.71 -1.05%
-LINK $8.16 -1.39%
-HYPE $52.71 +2.75%
-AAVE $91.91 +0.58%
-SUI $0.6838 -0.18%
-XLM $0.1712 -1.69%
-ZEC $481.93 +2.51%
+BTC $62,565.82 -0.82%
+ETH $1,840.40 -0.84%
+BNB $586.22 +0.87%
+XRP $1.06 -1.23%
+SOL $72.38 -0.78%
+TRX $0.3272 +0.27%
+DOGE $0.0694 -0.73%
+ADA $0.1880 -0.48%
+BCH $209.74 -1.06%
+LINK $8.17 -1.25%
+HYPE $52.80 +2.81%
+AAVE $92.38 +1.20%
+SUI $0.6853 -0.17%
+XLM $0.1711 -1.75%
+ZEC $483.60 +2.26%
+BTC $62,565.82 -0.82%
+ETH $1,840.40 -0.84%
+BNB $586.22 +0.87%
+XRP $1.06 -1.23%
+SOL $72.38 -0.78%
+TRX $0.3272 +0.27%
+DOGE $0.0694 -0.73%
+ADA $0.1880 -0.48%
+BCH $209.74 -1.06%
+LINK $8.17 -1.25%
+HYPE $52.80 +2.81%
+AAVE $92.38 +1.20%
+SUI $0.6853 -0.17%
+XLM $0.1711 -1.75%
+ZEC $483.60 +2.26%
 Home
 Article
 Flash
@@ -64,12 +64,12 @@
 Related reading
 Web3 development tool provider Cubist launches hardware key management service
 2023-04-18 22:02
+BitMine increased its holdings by 10,399 ETH last week, bringing the total holdings to approximately 5.798 million coins
+2026-08-03 20:36
+Bitget will stop providing cryptocurrency trading services for Japanese users and will close all local user positions by the end of the year
+2026-08-03 20:19
 Data: BlackRock's IBIT transfers over 1,948 BTC to Coinbase Prime
 2026-08-03 20:09
-1confirmation founder talks about Situational Awareness crisis: "Smart money" is making the same mistakes again
-2026-08-03 20:03
-Strategy sold 1,637 bitcoins last week and repurchased STRC preferred shares, reducing the holdings to 842,138 coins
-2026-08-03 19:57
 Related tags
 Nunchuk
 Coldcard Mk4
