--- chaincatcher-nunchuk-response@20260804T134758Z
+++ chaincatcher-nunchuk-response@20260804T141851Z
@@ -5,36 +5,36 @@
 Calendar
 Data
 Find
-BTC $63,998.81 +1.89%
-ETH $1,873.17 +1.50%
-BNB $590.91 +0.96%
-XRP $1.07 +0.72%
-SOL $73.93 +1.67%
-TRX $0.3297 +0.83%
-DOGE $0.0704 +0.62%
-ADA $0.1930 +1.04%
-BCH $214.21 +2.11%
-LINK $8.19 -0.39%
-HYPE $55.59 +4.33%
-AAVE $91.95 -0.31%
-SUI $0.6946 +0.67%
-XLM $0.1700 -0.95%
-ZEC $485.96 -0.05%
-BTC $63,998.81 +1.89%
-ETH $1,873.17 +1.50%
-BNB $590.91 +0.96%
-XRP $1.07 +0.72%
-SOL $73.93 +1.67%
-TRX $0.3297 +0.83%
-DOGE $0.0704 +0.62%
-ADA $0.1930 +1.04%
-BCH $214.21 +2.11%
-LINK $8.19 -0.39%
-HYPE $55.59 +4.33%
-AAVE $91.95 -0.31%
-SUI $0.6946 +0.67%
-XLM $0.1700 -0.95%
-ZEC $485.96 -0.05%
+BTC $63,661.54 +0.53%
+ETH $1,861.98 +0.48%
+BNB $589.00 +0.40%
+XRP $1.07 -0.02%
+SOL $73.56 +0.70%
+TRX $0.3294 +0.70%
+DOGE $0.0700 -0.20%
+ADA $0.1920 -0.40%
+BCH $213.51 +1.09%
+LINK $8.19 -0.93%
+HYPE $55.38 +3.60%
+AAVE $91.31 -1.51%
+SUI $0.6895 -0.84%
+XLM $0.1691 -1.79%
+ZEC $487.03 -0.33%
+BTC $63,661.54 +0.53%
+ETH $1,861.98 +0.48%
+BNB $589.00 +0.40%
+XRP $1.07 -0.02%
+SOL $73.56 +0.70%
+TRX $0.3294 +0.70%
+DOGE $0.0700 -0.20%
+ADA $0.1920 -0.40%
+BCH $213.51 +1.09%
+LINK $8.19 -0.93%
+HYPE $55.38 +3.60%
+AAVE $91.31 -1.51%
+SUI $0.6895 -0.84%
+XLM $0.1691 -1.79%
+ZEC $487.03 -0.33%
 Home
 Article
 Flash
@@ -64,12 +64,12 @@
 Related reading
 Web3 development tool provider Cubist launches hardware key management service
 2023-04-18 22:02
+Citigroup increased its holdings by 238,538 shares of Strategy, with a position value reaching 90.5 million USD
+2026-08-04 21:48
+Dinari and Circle launch a tokenized US stock platform, planning to put S&P 500 constituent stocks on the blockchain
+2026-08-04 21:39
 Stablecoin infrastructure provider Yellow Card has completed a $40 million strategic financing round, with participation from SC Ventures and others
 2026-08-04 21:07
-Polymarket seeks to raise funds with a valuation of over $20 billion
-2026-08-04 20:18
-OpenAI founders and employees collectively hold 48.93% of the company's shares
-2026-08-04 19:39
 Related tags
 Nunchuk
 Coldcard Mk4
