--- chaincatcher-nunchuk-response@20260803T030619Z
+++ chaincatcher-nunchuk-response@20260803T091026Z
@@ -5,36 +5,36 @@
 Calendar
 Data
 Find
-BTC $63,052.47 -0.50%
-ETH $1,861.40 -0.72%
-BNB $584.94 +0.81%
-XRP $1.07 -0.15%
-SOL $72.93 -0.17%
-TRX $0.3260 -0.59%
-DOGE $0.0699 +0.01%
-ADA $0.1865 +3.57%
-BCH $211.51 +1.09%
-LINK $8.27 +0.30%
-HYPE $52.27 -1.16%
-AAVE $91.45 -0.72%
-SUI $0.6865 -0.22%
-XLM $0.1718 -1.06%
-ZEC $476.38 +0.63%
-BTC $63,052.47 -0.50%
-ETH $1,861.40 -0.72%
-BNB $584.94 +0.81%
-XRP $1.07 -0.15%
-SOL $72.93 -0.17%
-TRX $0.3260 -0.59%
-DOGE $0.0699 +0.01%
-ADA $0.1865 +3.57%
-BCH $211.51 +1.09%
-LINK $8.27 +0.30%
-HYPE $52.27 -1.16%
-AAVE $91.45 -0.72%
-SUI $0.6865 -0.22%
-XLM $0.1718 -1.06%
-ZEC $476.38 +0.63%
+BTC $62,445.37 -1.01%
+ETH $1,837.37 -1.42%
+BNB $583.33 +0.04%
+XRP $1.06 -1.06%
+SOL $72.26 -1.04%
+TRX $0.3259 -0.35%
+DOGE $0.0692 -0.95%
+ADA $0.1836 -1.02%
+BCH $209.39 -1.27%
+LINK $8.21 -1.27%
+HYPE $52.43 +0.88%
+AAVE $91.61 +0.12%
+SUI $0.6840 -0.68%
+XLM $0.1705 -2.54%
+ZEC $473.38 -0.02%
+BTC $62,445.37 -1.01%
+ETH $1,837.37 -1.42%
+BNB $583.33 +0.04%
+XRP $1.06 -1.06%
+SOL $72.26 -1.04%
+TRX $0.3259 -0.35%
+DOGE $0.0692 -0.95%
+ADA $0.1836 -1.02%
+BCH $209.39 -1.27%
+LINK $8.21 -1.27%
+HYPE $52.43 +0.88%
+AAVE $91.61 +0.12%
+SUI $0.6840 -0.68%
+XLM $0.1705 -2.54%
+ZEC $473.38 -0.02%
 Home
 Article
 Flash
@@ -64,12 +64,8 @@
 Related reading
 Web3 development tool provider Cubist launches hardware key management service
 2023-04-18 22:02
-Changxin's long position funding fee is expected to yield an annualized return of 2130%, with whales able to net a profit of 740,000 USD in a single day
-2026-08-03 11:00
-Analysis: Long-term Bitcoin holders have transferred a large amount of chips for two consecutive days, possibly to avoid macro risks in advance
-2026-08-03 10:51
-Alibaba releases Qianwen 3.8-MAX artificial intelligence model
-2026-08-03 10:48
+JPMorgan: The AI sector is unlikely to be the main driver of returns in the second half of the year
+2026-08-03 16:56
 Related tags
 Nunchuk
 Coldcard Mk4
