--- chaincatcher-nunchuk-response@20260804T113811Z
+++ chaincatcher-nunchuk-response@20260804T120921Z
@@ -5,36 +5,36 @@
 Calendar
 Data
 Find
-BTC $63,670.28 +1.62%
-ETH $1,865.87 +1.24%
-BNB $589.28 +0.49%
-XRP $1.07 +0.84%
-SOL $73.63 +1.50%
-TRX $0.3288 +0.36%
-DOGE $0.0702 +1.14%
-ADA $0.1944 +4.01%
-BCH $213.57 +1.65%
-LINK $8.18 -0.33%
-HYPE $55.32 +4.66%
-AAVE $91.89 -0.23%
-SUI $0.6917 +1.00%
-XLM $0.1701 -0.90%
-ZEC $487.67 +1.58%
-BTC $63,670.28 +1.62%
-ETH $1,865.87 +1.24%
-BNB $589.28 +0.49%
-XRP $1.07 +0.84%
-SOL $73.63 +1.50%
-TRX $0.3288 +0.36%
-DOGE $0.0702 +1.14%
-ADA $0.1944 +4.01%
-BCH $213.57 +1.65%
-LINK $8.18 -0.33%
-HYPE $55.32 +4.66%
-AAVE $91.89 -0.23%
-SUI $0.6917 +1.00%
-XLM $0.1701 -0.90%
-ZEC $487.67 +1.58%
+BTC $63,802.47 +2.07%
+ETH $1,869.44 +1.71%
+BNB $590.86 +0.97%
+XRP $1.07 +1.23%
+SOL $73.75 +1.97%
+TRX $0.3290 +0.57%
+DOGE $0.0704 +1.69%
+ADA $0.1939 +4.19%
+BCH $213.94 +2.06%
+LINK $8.19 +0.32%
+HYPE $55.48 +5.24%
+AAVE $91.97 +0.05%
+SUI $0.6935 +1.29%
+XLM $0.1705 -0.52%
+ZEC $486.38 +0.90%
+BTC $63,802.47 +2.07%
+ETH $1,869.44 +1.71%
+BNB $590.86 +0.97%
+XRP $1.07 +1.23%
+SOL $73.75 +1.97%
+TRX $0.3290 +0.57%
+DOGE $0.0704 +1.69%
+ADA $0.1939 +4.19%
+BCH $213.94 +2.06%
+LINK $8.19 +0.32%
+HYPE $55.48 +5.24%
+AAVE $91.97 +0.05%
+SUI $0.6935 +1.29%
+XLM $0.1705 -0.52%
+ZEC $486.38 +0.90%
 Home
 Article
 Flash
@@ -64,12 +64,12 @@
 Related reading
 Web3 development tool provider Cubist launches hardware key management service
 2023-04-18 22:02
+OpenAI founders and employees collectively hold 48.93% of the company's shares
+2026-08-04 19:39
 The Trump administration drafted an order to ban the import of Chinese-made data center optical modules
 2026-08-04 19:17
 RootData Launches Beta Version of Stock Spot Trading Exchange Rankings
 2026-08-04 18:51
-UPBIT lists Squid (QUID) and supports KRW, BTC, USDT trading
-2026-08-04 18:00
 Related tags
 Nunchuk
 Coldcard Mk4
