--- bitcointalk-large-scale-compromise-thread@20260804T174736Z
+++ bitcointalk-large-scale-compromise-thread@20260804T184808Z
@@ -3064,5 +3064,11 @@
 Post by: LoyceV on August 04, 2026, 05:41:33 PM
 Hypothetical: it just occurred to me that even if Coldcard knew about this vulnerability, they couldn't have warned users about it. The moment they issue a warning, potential attackers would know about it too, and their warning would have been the catalyst to losing funds.
 So once the bug was out there, all they could reasonably do was offer updated firmware and remove the vulnerability from newly sold devices.
+Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
+Post by: negotiation4 on August 04, 2026, 06:32:28 PM
+Quote from: LoyceV on Today at 05:41:33 PM
+Hypothetical: it just occurred to me that even if Coldcard knew about this vulnerability, they couldn't have warned users about it. The moment they issue a warning, potential attackers would know about it too, and their warning would have been the catalyst to losing funds.
+So once the bug was out there, all they could reasonably do was offer updated firmware and remove the vulnerability from newly sold devices.
+Not true, they could have advised moving funds without disclosing the vulnerability up front.
 Powered by SMF 1.1.19 |
 SMF © 2006-2009, Simple Machines
