--- bitcointalk-large-scale-compromise-thread@20260808T213516Z
+++ bitcointalk-large-scale-compromise-thread@20260809T040713Z
@@ -4386,7 +4386,7 @@
 And to be frank with all of you, I'm starting to think this is very justifiable to a point.
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: joker_josue on August 08, 2026, 07:39:17 AM
-Quote from: BlackHatCoiner on Today at 07:32:36 AM
+Quote from: BlackHatCoiner on August 08, 2026, 07:32:36 AM
 I'm afraid this incident does reveal why most people won't properly custody their bitcoin, ever.
 If it is required from people to roll dice, use a passphrase apart from a seed phrase, or engage in multi-vendor multi-sig setups for "extra security", I'm afraid most people who "believe" in Bitcoin's value proposition will just stick with an ETF from now on.
 And to be frank with all of you, I'm starting to think this is very justifiable to a point.
@@ -4394,7 +4394,7 @@
 What really needs to be done is for people to read or reread the Bitcoin white paper again, to understand or remember what Bitcoin is.
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: BlackHatCoiner on August 08, 2026, 07:57:42 AM
-Quote from: joker_josue on Today at 07:39:17 AM
+Quote from: joker_josue on August 08, 2026, 07:39:17 AM
 Owning a Bitcoin ETF is not the same as owning Bitcoin!
 Obviously, but what most people want to get from Bitcoin is appreciating value overtime. This is what I think is what most people want.
 I really hope people aren't into Bitcoin just for the appreciating value aspect, and they also appreciate the fact that nobody can take it away from you without your permission, but I'm just afraid that through the current chaos, it is very difficult to support this claim, considering the average person does not know 99% of what most people in here know about self-custody. Most people will just not roll a dice.
@@ -4418,7 +4418,7 @@
 [1] https://x.com/oomahq/status/2085717166884618584
 Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
 Post by: flatt on August 08, 2026, 09:55:35 AM
-Quote from: LoyceV on Today at 08:07:50 AM
+Quote from: LoyceV on August 08, 2026, 08:07:50 AM
 Quote from: vapourminer on August 04, 2026, 10:34:44 AM
 some people have a non passphrase wallet as a decoy with passphrases behind it.
 I wonder how many people realize that in this case, a "decoy" wallet have increased the risk, as it tells the attacker they found a used seed phrase. That gives them the possibility to start brute-forcing the passphrase. If you would have used the weak seed phrase only in combination with a passphrase, it would have been many orders of magnitude more difficult to find.
@@ -4427,13 +4427,13 @@
 FYI, brute-forcing passphrase for a single wallet most likely TOOK MORE TIME than brute-forcing the whole ColdCard default seed.
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: kTimesG on August 08, 2026, 10:04:42 AM
-Quote from: JayJuanGee on Today at 02:25:57 AM
+Quote from: JayJuanGee on August 08, 2026, 02:25:57 AM
 You are saying also that no entropy was added by any dice rolls, so I would need to be explained the mechanics of how that would be based on if you are saying that there might have been an override of the Cold Card software that pushed it back to the 40 bits of entropy?
 It's not 40 bits of entropy, more like 23 (or even less, CC would know better of their die cutting parameters). Watching the official YouTube easy guide "5 minutes setup" of a fresh CC adds some 6 additional bits. Each dice adds around 2 bits on top of this. So that's only around 2 to 3 billion guesses for all seeds out of all entropies + one dice rolls. A laptop can break that in an hour.
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: bitmover on August 08, 2026, 11:09:28 AM
-Quote from: joker_josue on Today at 07:39:17 AM
-Quote from: BlackHatCoiner on Today at 07:32:36 AM
+Quote from: joker_josue on August 08, 2026, 07:39:17 AM
+Quote from: BlackHatCoiner on August 08, 2026, 07:32:36 AM
 I'm afraid this incident does reveal why most people won't properly custody their bitcoin, ever.
 If it is required from people to roll dice, use a passphrase apart from a seed phrase, or engage in multi-vendor multi-sig setups for "extra security", I'm afraid most people who "believe" in Bitcoin's value proposition will just stick with an ETF from now on.
 And to be frank with all of you, I'm starting to think this is very justifiable to a point.
@@ -4445,7 +4445,7 @@
 I am also affraid that when you own a bitcoin ETF you do not own a permitionless money, but you can at least have some protection against inflation.
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: joker_josue on August 08, 2026, 11:19:59 AM
-Quote from: sergiorus on Today at 09:50:09 AM
+Quote from: sergiorus on August 08, 2026, 09:50:09 AM
 Someone on Twitter [1] did their own investigation and is claiming a possible inside job.
 It's a thread of many tweets, the link is here (https://x.com/oomahq/status/2085717166884618584)
 I had heard him talk about this investigation, but I hadn't found it yet.
@@ -4453,7 +4453,7 @@
 Things are going to get very ugly for the Coldcard team. It remains to be seen when the first lawsuits will start to come.
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: vapourminer on August 08, 2026, 11:51:30 AM
-Quote from: BlackHatCoiner on Today at 07:32:36 AM
+Quote from: BlackHatCoiner on August 08, 2026, 07:32:36 AM
 I'm afraid this incident does reveal why most people won't properly custody their bitcoin, ever.
 If it is required from people to roll dice, use a passphrase apart from a seed phrase, or engage in multi-vendor multi-sig setups for "extra security", I'm afraid most people who "believe" in Bitcoin's value proposition will just stick with an ETF from now on.
 And to be frank with all of you, I'm starting to think this is very justifiable to a point.
@@ -4463,7 +4463,7 @@
 im on team dice but only because i hang on this forum. i easily could of been a victim years ago.
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: BlackHatCoiner on August 08, 2026, 11:56:08 AM
-Quote from: vapourminer on Today at 11:51:30 AM
+Quote from: vapourminer on August 08, 2026, 11:51:30 AM
 its always been this way really.
 I know, but I used to tell to newcomers that you can "just get a hardware wallet and sleep easy" in case they didn't want to trust an exchange (and many don't trust them). Now I'm not sure what the correct suggestion is. Expecting them to use a signing device and roll dice is not something I can recommend to everyday people.
 Quote
@@ -4471,27 +4471,27 @@
 I never trusted my computer's RNG in the first place, and I remember asking in this forum how most people do trust it. (I know the Coldcard issue was not the hardware's RNG per se, but I still would rather trust something I can verify with my own eyes.)
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: Cricktor on August 08, 2026, 12:09:10 PM
-Quote from: joker_josue on Today at 07:23:16 AM
+Quote from: joker_josue on August 08, 2026, 07:23:16 AM
 The only way to be prepared anywhere in the world is to always have your seed with you.
 How many of us have all the seeds with us 365 days a year, always?
 If that were actually necessary, we should consider the whole setup to be terribly wrong and faulty. I find this very very wrong to have to carry my mnemonic seeds with me all the time. Sorry, no, this should NOT be necessary! And it is not necessary if we knew that entropy generation wasn't screwed up completely.
 Apparently there's more due diligence needed to verify that entropy generation isn't flawed. Easier said, than done, though...
 I must say, I'm genuinely surprised how badly a hardware wallet vendor screwed this up like Coinkite did. I thought light-hearted that this shouldn't be possible because it's such a basic and important functionality to NOT screw up the random stuff when you have TRNGs available. I'm still puzzled...
-Quote from: NotATether on Today at 07:26:21 AM
+Quote from: NotATether on August 08, 2026, 07:26:21 AM
 The necessary step is, and I'll keep yelling this from rooftops until it reaches mass adoption, to generate your own seed phrase yourself, without any hardware wallet.
 And also make a BIP39 passphrase for your seed.
 This can only work if users learn and understand how to safely and properly generate their own mnemonic seeds. There are many ways to screw this up without being easily able to notice it.
 A mnemonic passphrase (the additional thing) needs to be complex enough to withstand brute-force attacks (it's also computationally somewhat expensive as each guess iteration requires 2048 rounds of PBKDF2 with HMAC-SHA512), but it adds a security layer with no margin for error if you fail to document it properly and highly recommended also redundantly. Commonly you should also separate it from your mnemonic recovery words, so it needs separate secure storage places.
 That's quite a (necessary) burden with many possibilities for people to screw up.
 On the other hand, with the necessary knowledge and understanding, I'm with you. I just have doubts it will work out for the masses.
-Quote from: LoyceV on Today at 08:07:50 AM
+Quote from: LoyceV on August 08, 2026, 08:07:50 AM
 I wonder how many people realize that in this case, a "decoy" wallet have increased the risk, as it tells the attacker they found a used seed phrase. That gives them the possibility to start brute-forcing the passphrase. If you would have used the weak seed phrase only in combination with a passphrase, it would have been many orders of magnitude more difficult to find.
 If the additional mnemonic passphrase is brute-forceable, the creator has failed already badly. It does not make sense to me, e.g. to use just a few additional words from the BiP39 wordlist. The mnemonic passphrase needs to be complex enough that brute-force attacks are not feasible.
 With a complex enough mnemonic passphrase I consider the "decoy" wallet as a valuable canary indicator of compromise. Users of such a setup should of course pay very much attention to not link their "decoy" UTXOs with their "hidden" main stash(es).
 Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
 Post by: LoyceV on August 08, 2026, 12:22:57 PM
-Quote from: flatt on Today at 09:55:35 AM
-Quote from: LoyceV on Today at 08:07:50 AM
+Quote from: flatt on August 08, 2026, 09:55:35 AM
+Quote from: LoyceV on August 08, 2026, 08:07:50 AM
 Quote from: vapourminer on August 04, 2026, 10:34:44 AM
 some people have a non passphrase wallet as a decoy with passphrases behind it.
 I wonder how many people realize that in this case, a "decoy" wallet have increased the risk, as it tells the attacker they found a used seed phrase. That gives them the possibility to start brute-forcing the passphrase. If you would have used the weak seed phrase only in combination with a passphrase, it would have been many orders of magnitude more difficult to find.
@@ -4503,17 +4503,17 @@
 Quote
 FYI, brute-forcing passphrase for a single wallet most likely TOOK MORE TIME than brute-forcing the whole ColdCard default seed.
 That completely depends on the passphrase. If someone uses just 2 BIP39 words, that only gives 4 million possibilities, which is many orders of magnitude easier to brute-force than the weak seed words.
-Quote from: BlackHatCoiner on Today at 11:56:08 AM
+Quote from: BlackHatCoiner on August 08, 2026, 11:56:08 AM
 I know, but I used to tell to newcomers that you can "just get a hardware wallet and sleep easy" in case they didn't want to trust an exchange (and many don't trust them). Now I'm not sure what the correct suggestion is.
 I used to recommend Ledger or Trezor, until Ledger went nuts. Now I'd recommend Trezor only, just because they've been around for a long time. But even though I know one fuckup doesn't mean another hardware wallet will do the same, the rate at which hardware wallets are disappearing from being trusted is not comforting. I've also seen hardware wallets that simply lose support after a few years, and users will have to buy a new one again. This is something for the long run, if a simple paper wallet outlives a hardware wallet, why even bother with the hardware?.
-Quote from: Cricktor on Today at 12:09:10 PM
+Quote from: Cricktor on August 08, 2026, 12:09:10 PM
 With a complex enough mnemonic passphrase I consider the "decoy" wallet as a valuable canary indicator of compromise. Users of such a setup should of course pay very much attention to not link their "decoy" UTXOs with their "hidden" main stash(es).
 The one major reason for me to choose a hardware wallet over offline cold storage, is that it's much more convenient to make a payment. If I have to manually enter 12 randon words on a devide with 2 buttons each time I want to make a payment, I don''t really see the point of using it anymore.
 Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
 Post by: vapourminer on August 08, 2026, 12:43:16 PM
-Quote from: LoyceV on Today at 12:22:57 PM
-Quote from: flatt on Today at 09:55:35 AM
-Quote from: LoyceV on Today at 08:07:50 AM
+Quote from: LoyceV on August 08, 2026, 12:22:57 PM
+Quote from: flatt on August 08, 2026, 09:55:35 AM
+Quote from: LoyceV on August 08, 2026, 08:07:50 AM
 Quote from: vapourminer on August 04, 2026, 10:34:44 AM
 some people have a non passphrase wallet as a decoy with passphrases behind it.
 I wonder how many people realize that in this case, a "decoy" wallet have increased the risk, as it tells the attacker they found a used seed phrase. That gives them the possibility to start brute-forcing the passphrase. If you would have used the weak seed phrase only in combination with a passphrase, it would have been many orders of magnitude more difficult to find.
@@ -4521,15 +4521,15 @@
 Nope, I didn't miss that. A decoy works if someone finds your (physical) seed phrase backup. In this case, the decoy is like a big flag that says: "there could be more money here".
 perhaps there is a script that monitors the decoy addy. the decoy gets swept: tons of warnings in real life start happening. gives a head start to move the passphrased coins out.
 one works from real life attacks, one works from cyber attacks.. half dozen of one is 6 of the other, pick yer poison
-Quote from: LoyceV on Today at 12:22:57 PM
+Quote from: LoyceV on August 08, 2026, 12:22:57 PM
 [...] if a simple paper wallet outlives a hardware wallet, why even bother with the hardware?.
 pretty much this. and i started with one of the 1st trezors. i still trust trezor but i bring my own entropy now.
 Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
 Post by: philipma1957 on August 08, 2026, 01:30:05 PM
-Quote from: vapourminer on Today at 12:43:16 PM
-Quote from: LoyceV on Today at 12:22:57 PM
-Quote from: flatt on Today at 09:55:35 AM
-Quote from: LoyceV on Today at 08:07:50 AM
+Quote from: vapourminer on August 08, 2026, 12:43:16 PM
+Quote from: LoyceV on August 08, 2026, 12:22:57 PM
+Quote from: flatt on August 08, 2026, 09:55:35 AM
+Quote from: LoyceV on August 08, 2026, 08:07:50 AM
 Quote from: vapourminer on August 04, 2026, 10:34:44 AM
 some people have a non passphrase wallet as a decoy with passphrases behind it.
 I wonder how many people realize that in this case, a "decoy" wallet have increased the risk, as it tells the attacker they found a used seed phrase. That gives them the possibility to start brute-forcing the passphrase. If you would have used the weak seed phrase only in combination with a passphrase, it would have been many orders of magnitude more difficult to find.
@@ -4537,7 +4537,7 @@
 Nope, I didn't miss that. A decoy works if someone finds your (physical) seed phrase backup. In this case, the decoy is like a big flag that says: "there could be more money here".
 perhaps there is a script that monitors the decoy addy. the decoy gets swept: tons of warnings in real life start happening. gives a head start to move the passphrased coins out.
 one works from real life attacks, one works from cyber attacks.. half dozen of one is 6 of the other, pick yer poison
-Quote from: LoyceV on Today at 12:22:57 PM
+Quote from: LoyceV on August 08, 2026, 12:22:57 PM
 [...] if a simple paper wallet outlives a hardware wallet, why even bother with the hardware?.
 pretty much this. and i started with one of the 1st trezors. i still trust trezor but i bring my own entropy now.
 Yeah I ended up adding passphrase to my trezors.
@@ -4553,7 +4553,7 @@
 do I like it no
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: Woodie on August 08, 2026, 01:43:17 PM
-Quote from: sergiorus on Today at 09:50:09 AM
+Quote from: sergiorus on August 08, 2026, 09:50:09 AM
 Someone on Twitter [1] did their own investigation and is claiming a possible inside job.
 It's a thread of many tweets, the link is here (https://x.com/oomahq/status/2085717166884618584)
 ~snip~
@@ -4563,7 +4563,7 @@
 https://x.com/COLDCARDwallet/status/1447213375398846473
 Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
 Post by: flatt on August 08, 2026, 01:56:14 PM
-Quote from: LoyceV on Today at 12:22:57 PM
+Quote from: LoyceV on August 08, 2026, 12:22:57 PM
 Nope, I didn't miss that. A decoy works if someone finds your (physical) seed phrase backup. In this case, the decoy is like a big flag that says: "there could be more money here".
 There have already been reports of 2-word passphrases being compromised.
 That completely depends on the passphrase. If someone uses just 2 BIP39 words, that only gives 4 million possibilities, which is many orders of magnitude easier to brute-force than the weak seed words.
@@ -4574,11 +4574,11 @@
 I still believe in SHA256, and accessing my funds from anywhere I want without letting a company give me the keys to my funds or storing my own keys in SDB is far worse.
 Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
 Post by: flatt on August 08, 2026, 02:22:40 PM
-Quote from: philipma1957 on Today at 01:30:05 PM
-Quote from: vapourminer on Today at 12:43:16 PM
-Quote from: LoyceV on Today at 12:22:57 PM
-Quote from: flatt on Today at 09:55:35 AM
-Quote from: LoyceV on Today at 08:07:50 AM
+Quote from: philipma1957 on August 08, 2026, 01:30:05 PM
+Quote from: vapourminer on August 08, 2026, 12:43:16 PM
+Quote from: LoyceV on August 08, 2026, 12:22:57 PM
+Quote from: flatt on August 08, 2026, 09:55:35 AM
+Quote from: LoyceV on August 08, 2026, 08:07:50 AM
 Quote from: vapourminer on August 04, 2026, 10:34:44 AM
 some people have a non passphrase wallet as a decoy with passphrases behind it.
 I wonder how many people realize that in this case, a "decoy" wallet have increased the risk, as it tells the attacker they found a used seed phrase. That gives them the possibility to start brute-forcing the passphrase. If you would have used the weak seed phrase only in combination with a passphrase, it would have been many orders of magnitude more difficult to find.
@@ -4586,7 +4586,7 @@
 Nope, I didn't miss that. A decoy works if someone finds your (physical) seed phrase backup. In this case, the decoy is like a big flag that says: "there could be more money here".
 perhaps there is a script that monitors the decoy addy. the decoy gets swept: tons of warnings in real life start happening. gives a head start to move the passphrased coins out.
 one works from real life attacks, one works from cyber attacks.. half dozen of one is 6 of the other, pick yer poison
-Quote from: LoyceV on Today at 12:22:57 PM
+Quote from: LoyceV on August 08, 2026, 12:22:57 PM
 [...] if a simple paper wallet outlives a hardware wallet, why even bother with the hardware?.
 pretty much this. and i started with one of the 1st trezors. i still trust trezor but i bring my own entropy now.
 Yeah I ended up adding passphrase to my trezors.
@@ -4603,7 +4603,7 @@
 I would never make myself harder than before like that, but currently I choose my favorite signature, hashes it and put it as my extra passphrase, which I can access it from anywhere anytime.
 Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
 Post by: LoyceV on August 08, 2026, 02:24:10 PM
-Quote from: flatt on Today at 01:56:14 PM
+Quote from: flatt on August 08, 2026, 01:56:14 PM
 This still doesn't increase the risk as you claim.
 You're still missing the point: if someone added a passphrase to his weak Coldcard without funding a decoy wallet, his funds wouldn't have been taken. It's the decoy that makes the seed words and thus potential passphrase light up.
 Quote
@@ -4611,11 +4611,11 @@
 At this point it's easier to just type a private key, it's shorter.
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: joker_josue on August 08, 2026, 02:34:51 PM
-Quote from: vapourminer on Today at 11:51:30 AM
+Quote from: vapourminer on August 08, 2026, 11:51:30 AM
 im on team dice but only because i hang on this forum. i easily could of been a victim years ago.
 You build your seeds with the dice and then "retrieve" them in a hardware wallet?
-Quote from: Cricktor on Today at 12:09:10 PM
-Quote from: joker_josue on Today at 07:23:16 AM
+Quote from: Cricktor on August 08, 2026, 12:09:10 PM
+Quote from: joker_josue on August 08, 2026, 07:23:16 AM
 The only way to be prepared anywhere in the world is to always have your seed with you.
 How many of us have all the seeds with us 365 days a year, always?
 If that were actually necessary, we should consider the whole setup to be terribly wrong and faulty. I find this very very wrong to have to carry my mnemonic seeds with me all the time. Sorry, no, this should NOT be necessary! And it is not necessary if we knew that entropy generation wasn't screwed up completely.
@@ -4624,8 +4624,8 @@
 The seed is not meant to be passed around; it should be kept safe and sound in a secure place. Of course, it's good for a person to think about how they can access the seed if they can't return to its original location. But that's a different matter altogether.
 Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
 Post by: flatt on August 08, 2026, 02:40:44 PM
-Quote from: LoyceV on Today at 02:24:10 PM
-Quote from: flatt on Today at 01:56:14 PM
+Quote from: LoyceV on August 08, 2026, 02:24:10 PM
+Quote from: flatt on August 08, 2026, 01:56:14 PM
 This still doesn't increase the risk as you claim.
 You're still missing the point: if someone added a passphrase to his weak Coldcard without funding a decoy wallet, his funds wouldn't have been taken. It's the decoy that makes the seed words and thus potential passphrase light up.
 Good point, but at that point, there's no such decoy thing if you're not funding on it.
@@ -4636,7 +4636,7 @@
 Waiting for some commercial solution to implement it.
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: Meuserna on August 08, 2026, 05:33:19 PM
-Quote from: NotATether on Today at 07:26:21 AM
+Quote from: NotATether on August 08, 2026, 07:26:21 AM
 Quote from: Forsyth Jones on August 07, 2026, 03:01:28 AM
 The Coldcard case made it clear, at the very least, that creating a wallet, writing down the seed phrase, depositing some funds, and only opening the wallet 10 years later IS NOT ENOUGH. A strategy for acting during trips is more than necessary.
 The necessary step is, and I'll keep yelling this from rooftops until it reaches mass adoption, to generate your own seed phrase yourself, without any hardware wallet.
@@ -4665,8 +4665,8 @@
 I know this list sounds like a lot, but really it's not.
 Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
 Post by: Meuserna on August 08, 2026, 05:43:45 PM
-Quote from: flatt on Today at 09:55:35 AM
-Quote from: LoyceV on Today at 08:07:50 AM
+Quote from: flatt on August 08, 2026, 09:55:35 AM
+Quote from: LoyceV on August 08, 2026, 08:07:50 AM
 Quote from: vapourminer on August 04, 2026, 10:34:44 AM
 some people have a non passphrase wallet as a decoy with passphrases behind it.
 I wonder how many people realize that in this case, a "decoy" wallet have increased the risk, as it tells the attacker they found a used seed phrase. That gives them the possibility to start brute-forcing the passphrase. If you would have used the weak seed phrase only in combination with a passphrase, it would have been many orders of magnitude more difficult to find.
@@ -4680,14 +4680,14 @@
 I believe it's wiser to use a simple passphrase as a decoy wallet. One word, easily cracked, for a decoy wallet. And leave the seed-only wallet untouched, never used.
 Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
 Post by: LoyceV on August 08, 2026, 08:13:37 PM
-Quote from: Meuserna on Today at 05:43:45 PM
+Quote from: Meuserna on August 08, 2026, 05:43:45 PM
 I believe it's wiser to use a simple passphrase as a decoy wallet. One word, easily cracked, for a decoy wallet. And leave the seed-only wallet untouched, never used.
 I like this option! In Coldcard's case, it would likely have taking the attacker a very long time to search the entire seed word space for simple passphrase options.
 Question: how do brute-force attacks typically work? Do they check against the first address in a wallet, or the first N addresses?
 Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
 Post by: Meuserna on August 08, 2026, 08:53:35 PM
-Quote from: LoyceV on Today at 08:13:37 PM
-Quote from: Meuserna on Today at 05:43:45 PM
+Quote from: LoyceV on August 08, 2026, 08:13:37 PM
+Quote from: Meuserna on August 08, 2026, 05:43:45 PM
 I believe it's wiser to use a simple passphrase as a decoy wallet. One word, easily cracked, for a decoy wallet. And leave the seed-only wallet untouched, never used.
 I like this option! In Coldcard's case, it would likely have taking the attacker a very long time to search the entire seed word space for simple passphrase options.
 Question: how do brute-force attacks typically work? Do they check against the first address in a wallet, or the first N addresses?
@@ -4706,5 +4706,23 @@
 Brute force attacking passphrases is only feasible if the attacker has a reason to check that specific seed phrase.
 If a seed phrase has never been used by itself as a wallet, an attacker has no way of knowing if it's been used with a passphrase or as part of a multisig... unless the attacker found the paper/metal backup of course. That's an obvious clue the seed was probably used. But in the ColdCard attacks, the thieves are searching seed phrases based on the list of billions of possible seeds the borked ColdCard firmware could generate.
 If your seed phrase was truly random, it cannot be found by seed phrase hunting. Bu ColdCard's seeds weren't truly random, so they can be found by using the same borked method ColdCard used.
+Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
+Post by: joker_josue on August 08, 2026, 10:09:15 PM
+Quote from: Meuserna on August 08, 2026, 08:53:35 PM
+If a seed phrase has never been used by itself as a wallet, an attacker has no way of knowing if it's been used with a passphrase or as part of a multisig... unless the attacker found the paper/metal backup of course. That's an obvious clue the seed was probably used. But in the ColdCard attacks, the thieves are searching seed phrases based on the list of billions of possible seeds the borked ColdCard firmware could generate.
+If your seed phrase was truly random, it cannot be found by seed phrase hunting. Bu ColdCard's seeds weren't truly random, so they can be found by using the same borked method ColdCard used.
+In short, a passphrase doesn't necessarily need to be highly complex, because even something simple (I'm not talking about the word "bitcoin"...) will generate a very high number of probabilities, which will take a long time to discover.
+The attackers must be able to afford to search by passphrase because they already know the seeds. Since they already have a database of seeds with balances, it becomes easier for them to use techniques similar to those used to find passwords.
+They no longer have to find sequences of 24 words +1, they only have to find that one +1.
+Furthermore, the profit they've already made allows them to continue their research without significant losses or risks. Everything they find now is a bonus.
+Title: Re: Large-scale Coldcard compromise (1596 BTC stolen so far)
+Post by: CucakRowo on August 09, 2026, 02:44:40 AM
+Two OP_Return messages brought tears to my eyes.
+One said, ''Please return at least some of the stolen money".
+Another said, "Suicide tonight if you don't give me back what I worked 12 years, my 6.4 BTC".
+Who knows how many dreams these people were carrying in their heart? Maybe this was someone's dream of giving their children a secure little future. Maybe someone had been saving that money hoping they could pay for their aging parents treatment. Maybe some people spent their entire live working to achieve this 3 or 4 BTC, and then, in a matter of minute it was all gone.
+There are probably countless stories of heartbreak behind those transaction that we will never hear about.
+So many tears, so much fear and desperation, all hidden behind blockchain addresses and transaction ID.
+How long can that hacker carry the burden of all these broken lives on his conscience! Will he really close his eyes and sleep peacefully!
 Powered by SMF 1.1.19 |
 SMF © 2006-2009, Simple Machines
