--- bitcointalk-large-scale-compromise-thread@20260806T235000Z
+++ bitcointalk-large-scale-compromise-thread@20260807T061859Z
@@ -3641,7 +3641,7 @@
 [I'm not picking on anyone here, this is in fact a repost of a comment I just wrote on reddit.]
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: tvbcof on August 06, 2026, 02:44:38 AM
-Quote from: gmaxwell on Today at 01:59:15 AM
+Quote from: gmaxwell on August 06, 2026, 01:59:15 AM
 It would be prudent to remind people here that there are still a lot of vulnerable funds that could be saved and that it would be anti-social to explain how to reduce the search space of coldcard devices just to brag about how smart you are, or show that some rando was wrong on the internet.
 Plenty of other people could be posting about the exact search space needed to enumerate vulnerable seeds and are kindly refraining from doing so. Just because you could figure out that doesn't mean that every would be coin-thief is going to figure it out before the owners sweep their coins.
 [I'm not picking on anyone here, this is in fact a repost of a comment I just wrote on reddit.]
@@ -3697,7 +3697,7 @@
 Maybe you could argue that FTX is more just because the claims were paid out more promptly, and so I am not going to dispute that the forced HODL situation of MTGOX likely created a lot of injustices for those who did not have something like a 10-ish to 13-ish year timeline, to the extent that they had already been paid (maybe around 75% paid), and I am not sure if the still fucking around to pay more claims will end up getting paid, since there is some strangeness in how long it is taking with potential distributions at the end of this year, perhaps?  so yeah, delays are injust, too.
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: joker_josue on August 06, 2026, 06:42:59 AM
-Quote from: tvbcof on Today at 02:44:38 AM
+Quote from: tvbcof on August 06, 2026, 02:44:38 AM
 I found out about the 'bug', and the initial details of it, when I had about 3 days left in a foreign land.  I had to decide whether to get an emergency ticket home.  My own musings about such optimizations and growth rates had a lot to do with my catching the next plane home rather than waiting.  I'm very glad I did.
 It's vacation time in most countries in the Northern Hemisphere. I believe there are still many people in a similar situation.
 Some people tend to take a few days to be 100% offline, so they may not even be aware of this whole problem yet. They may have already run out of coins and not even know it.
@@ -3705,14 +3705,14 @@
 Fortunately for you, you were aware of the situation and had the ability to return in time (I hope). Many probably weren't so lucky.
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: NotATether on August 06, 2026, 07:06:59 AM
-Quote from: Dave1 on Today at 06:13:09 AM
+Quote from: Dave1 on August 06, 2026, 06:13:09 AM
 Or this is just for the clickbait and clout about the current incident. But this could be bad crypto media. And if confirmed, this is really a sad story and tragic. The thing is that the one that is going to suffer the most here is that family that is going to be left behind.
 CLICKBAIT until confirmed true by authorities.
 The account on X posting this is just taking screenshots and fabricating news pieces based on his imagination. Very shameful conduct by a so-called "news" account.
 Already requested Community Note for that post so that it doesn't mislead anyone.
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: Danish Ali on August 06, 2026, 08:44:38 AM
-Quote from: joker_josue on Today at 06:42:59 AM
+Quote from: joker_josue on August 06, 2026, 06:42:59 AM
 It's vacation time in most countries in the Northern Hemisphere. I believe there are still many people in a similar situation.
 Some people tend to take a few days to be 100% offline, so they may not even be aware of this whole problem yet. They may have already run out of coins and not even know it.
 I think that in a week, when many people return from vacation, they will have a sad surprise. More reports will emerge and the numbers will increase.
@@ -3731,13 +3731,13 @@
 https://pbs.twimg.com/media/HO-QEsgWYAEQbGo?format=png&name=small
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: kTimesG on August 06, 2026, 10:23:55 AM
-Quote from: gmaxwell on Today at 01:59:15 AM
+Quote from: gmaxwell on August 06, 2026, 01:59:15 AM
 It would be prudent to remind people here that there are still a lot of vulnerable funds that could be saved and that it would be anti-social to explain how to reduce the search space of coldcard devices just to brag about how smart you are, or show that some rando was wrong on the internet.
 My intention was to simply point out that the specific honeypot doesn't reflect real-life conditions.
 It's not a security breach to use the number "42" which was pretended to be some sort of a honey pot of all wallets between 1 and 100 (out of which none was actually ever created)., when in reality there's a jigsaw puzzle hidden in a maze with a billion doors.
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: Cookdata on August 06, 2026, 10:28:24 AM
-Quote from: ryzaadit on Today at 09:03:03 AM
+Quote from: ryzaadit on August 06, 2026, 09:03:03 AM
 Their device detected the sweeps and both of them are in the race for RBF transaction.
 The hacker lose the race. They ended with paying fees 9,000 sat and receiving 1,000 sat losing 90% of the fund on miners fees.
 There is a similar experimental video about Mk3 but with different objectives, the person created 5 different wallets using Mk3 Coldcard, the first wallet was generated using the insecure random number generator, the same seed phrase was used to generate 3 wallets with different passphrases and the last wallet which is the 5th was generated using the same seed phrase from a random account.
@@ -3751,7 +3751,7 @@
 I have a TREZOR? I am still save?
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: vapourminer on August 06, 2026, 10:56:46 AM
-Quote from: Bullethead21 on Today at 10:32:39 AM
+Quote from: Bullethead21 on August 06, 2026, 10:32:39 AM
 I have a TREZOR? I am still save?
 if you used a seed generated on a coldcard, no you are not safe. you need to move funds out.
 otherwise depends on how you generated the seed, but so far other wallets are not affected by the coldcard bug
@@ -3761,14 +3761,14 @@
 I am reluctant to think that a serious news program is capable of making a mistake of such magnitude, and, most likely, the person who has told me about it misunderstood what they said. But, if so, there are many more people who know less about the subject that my friend and who saw the news that must be thinking that something is wrong with Bitcoin.
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: stompix on August 06, 2026, 11:12:59 AM
-Quote from: Dave1 on Today at 06:13:09 AM
+Quote from: Dave1 on August 06, 2026, 06:13:09 AM
 Not sure if this is true or not,
 https://x.com/News_crypto/status/2084558130239664431
 Or this is just for the clickbait and clout about the current incident. But this could be bad crypto media. And if confirmed, this is really a sad story and tragic. The thing is that the one that is going to suffer the most here is that family that is going to be left behind.
 One guy sends a message and claims to commit suicide, and we already write him dead?
 Also, who uses a tumbler to split move funds 4 times before sending that message to reimburse him to an address with no activity?
 Seriously, lately the ratio of noise to actual info is going down the drain.
-Quote from: joker_josue on Today at 06:42:59 AM
+Quote from: joker_josue on August 06, 2026, 06:42:59 AM
 It's vacation time in most countries in the Northern Hemisphere. I believe there are still many people in a similar situation.
 Some people tend to take a few days to be 100% offline, so they may not even be aware of this whole problem yet. They may have already run out of coins and not even know it.
 I doubt they have planned it this way all along but indeed, it came at one of the worst times possible, at least for Christmas or some other holiday a lot of people are home with family, in summer, things are different. I'm sure there are plenty who don't even know what has happened yet.
@@ -3796,8 +3796,8 @@
 https://x.com/DylanLeClair/status/2085074383773581570
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: ultrloa on August 06, 2026, 11:27:59 AM
-Quote from: vapourminer on Today at 10:56:46 AM
-Quote from: Bullethead21 on Today at 10:32:39 AM
+Quote from: vapourminer on August 06, 2026, 10:56:46 AM
+Quote from: Bullethead21 on August 06, 2026, 10:32:39 AM
 I have a TREZOR? I am still save?
 if you used a seed generated on a coldcard, no you are not safe. you need to move funds out.
 otherwise depends on how you generated the seed, but so far other wallets are not affected by the coldcard bug
@@ -3812,30 +3812,30 @@
 Seems that it didn't get much coverage in mainstream media, though?
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: Danish Ali on August 06, 2026, 01:37:45 PM
-Quote from: vapourminer on Today at 10:56:46 AM
-Quote from: Bullethead21 on Today at 10:32:39 AM
+Quote from: vapourminer on August 06, 2026, 10:56:46 AM
+Quote from: Bullethead21 on August 06, 2026, 10:32:39 AM
 I have a TREZOR? I am still save?
 if you used a seed generated on a coldcard, no you are not safe. you need to move funds out.
 otherwise depends on how you generated the seed, but so far other wallets are not affected by the coldcard bug
 Exactly, the device brand does not matter here, the seed origin does. Any wallet holding Coldcard generated seed is equally at risk regardless of what hardware it is running on now.
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: Stalker22 on August 06, 2026, 02:02:27 PM
-Quote from: stompix on Today at 11:12:59 AM
+Quote from: stompix on August 06, 2026, 11:12:59 AM
 I doubt they have planned it this way all along but indeed, it came at one of the worst times possible,
 ~
 I think someone DID plan this carefully.  The first few batched transactions were executed almost simultaneously.  This means someone had to crack all the private keys, find the wallets with the largest balances, prepare and sign all the transactions, and just wait for the right time to execute.
-Quote from: stompix on Today at 11:12:59 AM
+Quote from: stompix on August 06, 2026, 11:12:59 AM
 I'm sure there are plenty who don't even know what has happened yet.
 Unfortunately, yes.  Mainstream coverage of this incident has been practically nonexistent, keeping the story confined strictly to the crypto space.
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: vapourminer on August 06, 2026, 02:24:29 PM
-Quote from: Stalker22 on Today at 02:02:27 PM
+Quote from: Stalker22 on August 06, 2026, 02:02:27 PM
 Mainstream coverage of this incident has been practically nonexistent, keeping the story confined strictly to the crypto space.
 just as well; the public would think its a bitcoin-the-protocol is hacked and create panic. its just one hardware wallets manufacturers idiot programming. not a big deal in the general bitcoin space as a whole.
 not to minimize the absolute suck this whole thing is for victims and thats what they were, victims. not their fault they trusted this thing it was recommended by many here and elsewhere.
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: LoyceV on August 06, 2026, 02:29:00 PM
-Quote from: ryzaadit on Today at 09:03:03 AM
+Quote from: ryzaadit on August 06, 2026, 09:03:03 AM
 Crazy to see the sweeps just need 10 seconds after the deposit.
 Was that a new hardware wallet with a newly created seed phrase, or a seed phrase that was compromised before?
 If it's the former, that means someone (or multiple entities) are now racing through all ~trillion possible seed phrases to detect any new incoming funds.
@@ -3848,7 +3848,7 @@
 I was curious about that scenario too. That would mean that (eventually) only miners profit from funds sent to addresses with leaked private keys.
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: hd49728 on August 06, 2026, 02:41:05 PM
-Quote from: ryzaadit on Today at 09:03:03 AM
+Quote from: ryzaadit on August 06, 2026, 09:03:03 AM
 Crazy to see the sweeps just need 10 seconds after the deposit.
 Some social experiment being made by Wesatoshis, who provides a Bitcoin hardware terminal for moving transaction with @Pathfinder to fill his Coldcard MK3 with 10,000 sat for the atacker to take the baits. And indeed..... just need 10 second after the deposit for the atacker trying to take those fund
 Their device detected the sweeps and both of them are in the race for RBF transaction.
@@ -3864,8 +3864,8 @@
 Who knows, it can even be an insider job. That's always the first thought I get when such hacks happen.
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: oll on August 06, 2026, 03:09:11 PM
-Quote from: vapourminer on Today at 02:24:29 PM
-Quote from: Stalker22 on Today at 02:02:27 PM
+Quote from: vapourminer on August 06, 2026, 02:24:29 PM
+Quote from: Stalker22 on August 06, 2026, 02:02:27 PM
 Mainstream coverage of this incident has been practically nonexistent, keeping the story confined strictly to the crypto space.
 just as well; the public would think its a bitcoin-the-protocol is hacked and create panic. its just one hardware wallets manufacturers idiot programming. not a big deal in the general bitcoin space as a whole.
 not to minimize the absolute suck this whole thing is for victims and thats what they were, victims. not their fault they trusted this thing it was recommended by many here and elsewhere.
@@ -3876,15 +3876,15 @@
 https://xcancel.com/callebtc/status/2085024458012586286
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: ryzaadit on August 06, 2026, 05:38:04 PM
-Quote from: Bullethead21 on Today at 10:32:39 AM
+Quote from: Bullethead21 on August 06, 2026, 10:32:39 AM
 I have a TREZOR? I am still save?
 https://pbs.twimg.com/media/HO9SYJlW0AAcuuO?format=jpg&name=large
 Best thing to do, add your own entropy my friend. Learn about entropy.
-Quote from: LoyceV on Today at 02:29:00 PM
+Quote from: LoyceV on August 06, 2026, 02:29:00 PM
 Was that a new hardware wallet with a newly created seed phrase, or a seed phrase that was compromised before?
 If it's the former, that means someone (or multiple entities) are now racing through all ~trillion possible seed phrases to detect any new incoming funds.
 Unfortunately, it's new.
-Quote from: pawanjain on Today at 02:44:08 PM
+Quote from: pawanjain on August 06, 2026, 02:44:08 PM
 At this point, I have stopped trusting these hardware wallets. Are there even any option left for really good hardware wallets?
 Just because of one stupid manufacturer mistake, don't lose your belief in hardware wallets. There are so many good hardware wallets out there.
 People forget some important things, mostly new people. They think that owning a hardware wallet their fund 100% secure, and this is the reason by using hardware wallet removes the feeling of safety, especially for anyone who has not learned more about safety risk.
@@ -3895,7 +3895,7 @@
 - Store your funds not just in one single bucket
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: Meuserna on August 06, 2026, 05:49:25 PM
-Quote from: ultrloa on Today at 11:27:59 AM
+Quote from: ultrloa on August 06, 2026, 11:27:59 AM
 I think there's no way for Trezor user to generate seed on Coldcard. So generally he's safe from this exploit.
 Sure there is.
 Generate a seed on a ColdCard. Enter the seed on a Trezor to restore the wallet. The wallet is still in danger because the seed was generated on a ColdCard.
@@ -3905,8 +3905,8 @@
 ColdCard owners are getting robbed because their devices generated predictable seeds. Moving a predictable seed to a Trezor doesn't change the fact that the seed was predictable, so the seed can be found by going through the list of all possible seeds generated by ColdCard's borked code.
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: suzanne5223 on August 06, 2026, 07:35:01 PM
-Quote from: Cookdata on Today at 10:28:24 AM
-Quote from: ryzaadit on Today at 09:03:03 AM
+Quote from: Cookdata on August 06, 2026, 10:28:24 AM
+Quote from: ryzaadit on August 06, 2026, 09:03:03 AM
 Their device detected the sweeps and both of them are in the race for RBF transaction.
 The hacker lose the race. They ended with paying fees 9,000 sat and receiving 1,000 sat losing 90% of the fund on miners fees.
 There is a similar experimental video about Mk3 but with different objectives, the person created 5 different wallets using Mk3 Coldcard, the first wallet was generated using the insecure random number generator, the same seed phrase was used to generate 3 wallets with different passphrases and the last wallet which is the 5th was generated using the same seed phrase from a random account.
@@ -3919,9 +3919,9 @@
 https://x.com/wowens/status/2084041966212591963?s=20
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: OgNasty on August 06, 2026, 07:38:06 PM
-Quote from: suzanne5223 on Today at 07:35:01 PM
-Quote from: Cookdata on Today at 10:28:24 AM
-Quote from: ryzaadit on Today at 09:03:03 AM
+Quote from: suzanne5223 on August 06, 2026, 07:35:01 PM
+Quote from: Cookdata on August 06, 2026, 10:28:24 AM
+Quote from: ryzaadit on August 06, 2026, 09:03:03 AM
 Their device detected the sweeps and both of them are in the race for RBF transaction.
 The hacker lose the race. They ended with paying fees 9,000 sat and receiving 1,000 sat losing 90% of the fund on miners fees.
 There is a similar experimental video about Mk3 but with different objectives, the person created 5 different wallets using Mk3 Coldcard, the first wallet was generated using the insecure random number generator, the same seed phrase was used to generate 3 wallets with different passphrases and the last wallet which is the 5th was generated using the same seed phrase from a random account.
@@ -3935,16 +3935,16 @@
 It isn't like this costs the attacker anything though.  It's also probably an automated process at this point, so most likely the person is just wasting their own time and money.  In this case they probably got some social media fame, followers, and maybe a boost to their X payout.  However, this isn't the huge win it may appear to be if you don't understand how the process works.  Whatever it takes to cope with the pain though.  In the end, laughter is the best medicine.
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: tvbcof on August 06, 2026, 08:57:32 PM
-Quote from: suzanne5223 on Today at 07:35:01 PM
+Quote from: suzanne5223 on August 06, 2026, 07:35:01 PM
 ...
 from image:
 imagine compromising someone's hardware wallet and still fumbling the bag
 ...
-Imagine controlling everyone who used the device's keys, dice or not, and letting a hoard of bag-fumbers fight over the the dice-less dregs for the next few years.
+Imagine controlling everyone who used the device's keys, dice or not, and letting a hoard of bag-fumblers fight over the the dice-less dregs for the next few years.
 Imagine further, whale keys sitting safely on the blockchain to be harvested as-needed in the coming decades.
 Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
 Post by: bitmover on August 06, 2026, 10:44:03 PM
-Quote from: JayJuanGee on Today at 06:28:48 AM
+Quote from: JayJuanGee on August 06, 2026, 06:28:48 AM
 Personally, I think that the earlier table that was posted by Forsyth Jones is a good guideline for at least shooting for at least minimal levels of safety, especially if we want to shoot to have our levels to at least be in the orange, and probably better to be in the lighter orange rather than the darker orange, just to be safer, yet even if we land in the darker orange, we should not need to panic, even though we might want to consider if we might want to create a wee bit stronger variation and then move our coins to that stronger variation.
 Quote from: Forsyth Jones on August 03, 2026, 11:54:10 PM
 Below is a table showing the strength of each passphrase extension:
@@ -3954,7 +3954,7 @@
 Just create a passphrase you like and can remember. I believe this is the lesson for everyone about this attack/scam
 Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
 Post by: fillippone on August 06, 2026, 10:48:48 PM
-Quote from: Pmalek on Today at 03:35:05 PM
+Quote from: Pmalek on August 06, 2026, 03:35:05 PM
 Bitcoin Red Team consists of 16 people who are working around the clock now and running security audits on Bitcoin code bases. According to their report, they have already made close to 5,000 findings and discovered what they believe are 85 critical and 635 high severity issues. They report these issues back to project owners and companies, and I think that's a great thing that has come out of this terrible situation. Whitehats and industry experts joining forces to help secure the wider Bitcoin ecosystem.
 https://xcancel.com/callebtc/status/2085024458012586286
 Bitcoin code is safe.
@@ -3962,5 +3962,101 @@
 The code is arguably kept simple just to avoid any hidden bug or negative feature.
 Interacting with this code requires using many more codes: wallets, signing devices, cryptographic libraries, exchanges....
 Those software are not secure, and prone to AI-driven exploit.
+Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
+Post by: philipma1957 on August 07, 2026, 01:53:04 AM
+Quote from: bitmover on August 06, 2026, 10:44:03 PM
+Quote from: JayJuanGee on August 06, 2026, 06:28:48 AM
+Personally, I think that the earlier table that was posted by Forsyth Jones is a good guideline for at least shooting for at least minimal levels of safety, especially if we want to shoot to have our levels to at least be in the orange, and probably better to be in the lighter orange rather than the darker orange, just to be safer, yet even if we land in the darker orange, we should not need to panic, even though we might want to consider if we might want to create a wee bit stronger variation and then move our coins to that stronger variation.
+Quote from: Forsyth Jones on August 03, 2026, 11:54:10 PM
+Below is a table showing the strength of each passphrase extension:
+https://i.bitlist.co/0IVL5jMzaudl.jpg
+I think 32 years is secure enough for me  :D
+But even the 12 years variation of 8 characters are basically invulnerable for any practical purposes.
+Just create a passphrase you like and can remember. I believe this is the lesson for everyone about this attack/scam
+buy this
+https://www.amazon.com/gp/product/B000I2FW2Q/ref=ox_sc_act_title_1?smid=ATVPDKIKX0DER&th=1
+a set of 36 punches
+next buy this
+https://www.amazon.com/gp/product/B0GMH6LGJT/ref=ox_sc_act_title_1?smid=A1MDC9YLETZE7Z&th=1
+30 washers
+https://www.amazon.com/gp/product/B0FG2LRFZ2/ref=ox_sc_act_title_1?smid=A1FA0UJGUNVMLI&th=1
+1/4 nuts and bolts
+now here comes the important part.
+go to the punch set and out of all 36 pick 20 letters and number see the photos
+https://www.talkimg.com/images/2026/08/07/UoigW2.jpg
+https://www.talkimg.com/images/2026/08/07/Uoi8Cc.jpg
+I picked those 20 because they are easy to see that they are different from each other
+so.
+A B D E F G H J M P
+R S T V W Y Z 3 4 8
+PUT THEM IN THIS
+https://www.talkimg.com/images/2026/08/07/UoiKYP.jpg
+https://www.talkimg.com/images/2026/08/07/Uoibiq.jpg
+Move and rotate it a pull a bar out with closed eyes
+pretend it was the Z    1 in 20 shot.  put it in the mixer rotate flip spin pull new letter or number say 3
+so Z3 is 1 in 400 put it back again and again
+pretend YOU do 12 times get the passphrase below
+Z3B                    1 IN                        8,000
+Z3BG                   1 IN                    160,000
+Z3BGP                 1 IN                  3,200,000
+Z3BGPT               1 IN                64,000,000
+Z3BGPT8             1 IN                1,280,000,000
+Z3BGPT8Y           1 IN                25,600,000,000
+Z3BGPT8YY           1 IN             512,000,000,000
+Z3BGPT8YYA         1 IN        10,240,000,000,000
+Z3BGPT8YYA3       1 IN      204,800,000,000,000
+Z3BGPT8YYA3S     1 IN        4,096,000,000,000,000                12 PULLS OF THE 20 PUNCHES IS       4,096 X 1 TRILLION
+Z3BGPT8YYA3S4   1 IN       81,920,000,000,000,000               13 PULLS OF THE 20 PUNCHES IS       81,920 X 1 TRILLION
+Z3BGPT8YYA3S4H   1 IN  1,638,400,000,000,000,000              14 PULLS OF THE 20 PUNCHES IS   1,638,400 X 1 TRILLION
+Z3BGPT8YYA3S4H8 1 IN 32,768,000,000,000,000,000             15 PULLS OF THE 20 PUNCHES IS  32,768,000 X 1 TRILLION
+SO A 15 character passphrase is pretty okay it is 65 bits  which is crack able but it is after the seed of 12 or 20 or 24 words
+a 20 character passphrase built with 20 punches is 85 bits
+a 24 character passphrase built wit 20 punches is 102 bits.
+cold card seed at 40 bits is around 1,099,511,627,776 when compared to the 15 character passphrase above
+15 characters is 32,768,000 harder
+Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
+Post by: Forsyth Jones on August 07, 2026, 03:01:28 AM
+Quote from: joker_josue on August 6, 2026, 03:42:59 AM
+It's vacation time in most countries in the Northern Hemisphere. I believe there are still many people in a similar situation.
+Some people tend to take a few days to be 100% offline, so they may not even be aware of this whole problem yet. They may have already run out of coins and not even know it.
+I think that in a week, when many people return from vacation, they will have a sad surprise. More reports will emerge and the numbers will increase.
+Quote from: pawanjain on August 6, 2026, 11:44:08 AM
+At this point, I have stopped trusting these hardware wallets. Are there even any option left for really good hardware wallets?
+What's the point of having one if they can have such bugs which can sweep out millions of funds in such a short time.
+I know it's not the users fault here but the company has to take the responsibility here for the mess up and ensure their users get their funds back.
+Who knows, it can even be an insider job. That's always the first thought I get when such hacks happen.
+Quote from: bitmover on August 6, 2026, 07:44:03 PM
+I think 32 years is secure enough for me
+But even the 12 years variation of 8 characters are basically invulnerable for any practical purposes.
+Just create a passphrase you like and can remember. I believe this is the lesson for everyone about this attack/scam
+The question is this: imagine you bought a hardware wallet. You depend on the developer's competence, on 3rd parties who review the code for you (if it has an open source license), meanwhilte, the manufactors pursue certification licenses for their devices (usually for secure elements and microcontrollers). The guy chose Coldcard, at this moment, he's enjoying a vacation on some paradise island (little does he know it might be the last time) and has no idea of the sea of disappointment he is about to dive into.
+A simple act that could SAVE his savings:
+- Having thought about ways to prevent himself in case of disasters like this, things like: carrying a seed phrase (encrypted, steganographic or not) with him, he could move the funds to a new wallet if he knew about this targeted hack on Coldcard.
+- He could have added a strong passphrase, but let's say he knows the passphrase by heart, but he doesn't have the seed phrase with him, he needs the both things (remembering that he is on a family trip, visiting the Eiffel Tower, etc), he failed again.
+- Memorizing the seed phrase and passphrase? You always have to maintain it (doing spaced repetition, keep reminding yourself from time to time so you don't forget, and have them written down somewhere, separately).
+How would you solve this problem if you had a coldcard and haven't been affected yet? Imagine yourself in his shoes, what would you do? How to solve this problem?
+The passphrase, you either need to keep it somewhere (geographically far from the seed phrase) or have it memorized. Another way is to keep it discreetly in your agenda (paper) or in your leather wallet.
+Many people (myself included) own more than one hardware wallet or some form of airgapped storage (even better, as long as you know what you're doing). This would also prevent a lot of problems, since people practically entrusted their lives to it.
+The Coldcard case made it clear, at the very least, that creating a wallet, writing down the seed phrase, depositing some funds, and only opening the wallet 10 years later IS NOT ENOUGH. A strategy for acting during trips is more than necessary.
+Quote from: philipma1957 on August 6, 2026, 10:53:04 PMLast edit: Today at 11:24:46 PM by philipma1957
+buy this
+https://www.amazon.com/gp/product/B000I2FW2Q/ref=ox_sc_act_title_1?smid=ATVPDKIKX0DER&th=1
+a set of 36 punches
+This is also good, as long as it's kept at home, in a safe, on the property, etc. but what about when traveling? What about the risk of someone who knows what this is about seizing it and asking questions in a not-so-friendly way?  ::)
+Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
+Post by: JayJuanGee on August 07, 2026, 03:46:15 AM
+Quote from: bitmover on August 06, 2026, 10:44:03 PM
+Quote from: JayJuanGee on August 06, 2026, 06:28:48 AM
+Personally, I think that the earlier table that was posted by Forsyth Jones is a good guideline for at least shooting for at least minimal levels of safety, especially if we want to shoot to have our levels to at least be in the orange, and probably better to be in the lighter orange rather than the darker orange, just to be safer, yet even if we land in the darker orange, we should not need to panic, even though we might want to consider if we might want to create a wee bit stronger variation and then move our coins to that stronger variation.
+Quote from: Forsyth Jones on August 03, 2026, 11:54:10 PM
+Below is a table showing the strength of each passphrase extension:
+https://i.bitlist.co/0IVL5jMzaudl.jpg
+I think 32 years is secure enough for me  :D
+But even the 12 years variation of 8 characters are basically invulnerable for any practical purposes.
+Just create a passphrase you like and can remember. I believe this is the lesson for everyone about this attack/scam
+For sure, we probably do not want to give away our exact set up, except maybe amongst friends and sometimes even on the internet we have to be a bit careful in terms of describing our own set-up, security or our inclinations of what we believe might be "good enough."
+I have been warming up more towards the ideas of several members who proclaim that there is quite a bit of value in future-proofing my own chosen set up a bit more, yet I am not going to point out which areas, exactly, in which I might be currently vulnerable - since even with my own set ups, I have variations in their level of security, and some of them I am considering whether to upgrade sooner or later, and it can surely take a long time to upgrade, as we mentioned in another post in which there was a description of maybe doing 50-ish different transfers after the set up had been made.
+At the same time, we might have our own ways of keeping back up records in regards to what our set-ups might be, so then we might have to update our various back up records too.  It can be difficult (and problematic) to keep too much information in our heads, even if we might think that some of the information is basic, such as the location of each of the pieces of information that might be needed to reconstruct our seed... and then are those pieces of information complete enough or do they include any changes that  we might have had chosen to make.
+This particular attack had a bit of its own uniqueness in terms of potentially creating a bit of a time-buffer for anyone who had any amount of security beyond the various defaults that were built into Cold Card, and that seemed to be one of the problems with the assumptions around cold card, since they seemed to have so many security features within their device.  Accordingly, there were likely a lot of normies (and even somewhat seemingly sophisticated bitcoiners) presuming that since Cold Card had so many various security options that their default must have had at least had some levels of protection, which that was a "HOLY SHIT!!!!" kind of a revelation to find out that Cold Card's default protections (at such a low level as the random number generator) was so damned vulnerable.
 Powered by SMF 1.1.19 |
 SMF © 2006-2009, Simple Machines
