--- bitcointalk-large-scale-compromise-thread@20260808T083433Z
+++ bitcointalk-large-scale-compromise-thread@20260808T150524Z
@@ -3154,6 +3154,8 @@
 Code:
 https://t.me/coldcardREAL
 https://t.me/coldcardMigration
+https://t.me/ColdcardwalletTG
+https://t.me/ColdCardDev
 These groups are swaming with scammers that want victims to use their malicious links:
 Code:
 https://swiftprotocolresolvers.web.app/
@@ -3166,8 +3168,11 @@
 https://dapplogin-connect.com
 https://t.me/AiCustomerSupport247_bot
 https://onchains-hub.vercel.app
+https://meta-masks.vercel.app
+http://multichaindecent.web.app/
 Do not enter seed words there! Do not send funds there! Do not download any software from there!
 Edit. Added new scams
+Edit 2. Added new scams
 Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
 Post by: NUCLEAR7.1 on August 04, 2026, 10:13:31 PM
 Quote from: shahzadafzal on August 04, 2026, 09:12:58 PM
@@ -4397,47 +4402,232 @@
 Post by: LoyceV on August 08, 2026, 08:07:50 AM
 Quote from: vapourminer on August 04, 2026, 10:34:44 AM
 some people have a non passphrase wallet as a decoy with passphrases behind it.
-I wonder how many people realize that in this case, a "decoy" wallet actually increased the risk, as it tells the attacker they found a used seed phrase. That gives them the possibility to start brute-forcing the passphrase. If you would have used the weak seed phrase only in combination with a passphrase, it would have been many orders of magnitude more difficult to find.
-Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
-Post by: Kalicharan on August 08, 2026, 08:28:41 AM
-WEEK 1 (02.08 - 08.08)
-Telegram Profile Link https://t.me/Kalicharan44
-Twitter profile link https://twitter.com/kalicharan44
-Twitter campaign
-Twitter Profile Link https://twitter.com/kalicharan44
-Followers:
-Retweet
-1.  https://x.com/i/status/2084472592816042180
-2. https://x.com/i/status/2085190036769873937
-3. https://x.com/i/status/2086002779735405013
-4. https://x.com/i/status/2086002868147093896
-5. https://x.com/i/status/2086002936044507487
-Tweet
-1. https://x.com/i/status/2084476655976972778
-Facebook campaign
-Facebook Profile Link: https://www.facebook.com/Kalicharan44
-Friends:
-Spreadsheet #:
-Shares
-1. https://www.facebook.com/100083630666228/posts/1019528570844822/?app=fbl
-2. https://www.facebook.com/100083630666228/posts/1021225910675088/?app=fbl
-3. https://www.facebook.com/100083630666228/posts/1023146377149708/?app=fbl
-4. https://www.facebook.com/100083630666228/posts/1023146587149687/?app=fbl
-5. https://www.facebook.com/100083630666228/posts/1023146653816347/?app=fbl
-post
-1. https://www.facebook.com/100083630666228/posts/pfbid037en1TBb1bVp8yNJwpmLBfktSLgF3zf33icNXHcvFuBLe5BCCJzRzUE9gU5zRA7Vjl/
-Shilling: campaign
-Telegram shilling:
-1. https://t.me/ICOCryptoReview/275888
-2. https://t.me/CryptoInvestorsNews2020/501915
-3. https://t.me/BestCryptoInvestments2020/307746
-4. https://x.com/i/status/2086005537494425782
-5. https://x.com/i/status/2086006097052393533
-Twitter shilling:
-1. https://x.com/i/status/2084476655976972778
-2. https://x.com/i/status/2085191004135862508
-3. https://x.com/i/status/2086004652911554719
-4. https://t.me/CryptoAirdropBounties/530979
-5. https://t.me/ICOCryptoReview/275957
+I wonder how many people realize that in this case, a "decoy" wallet would have increased the risk, as it tells the attacker they found a used seed phrase. That gives them the possibility to start brute-forcing the passphrase. If you would have used the weak seed phrase only in combination with a passphrase, it would have been many orders of magnitude more difficult to find.
+Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
+Post by: sergiorus on August 08, 2026, 09:50:09 AM
+Someone on Twitter [1] did their own investigation and is claiming a possible inside job.
+It's a thread of many tweets, the link is here (https://x.com/oomahq/status/2085717166884618584)
+The OP tweet:
+Quote
+I did my own investigation because I obviously don't trust them.
+What I found is that the external dependency of the firmware with the critical vulnerability hidden in it was written by CoinKite's CTO
+@DocHex
+pretending to be someone else.
+All of the following can be verified:
+https://pbs.twimg.com/media/HPH0qqaXoAI_7FI?format=jpg&name=medium
+[1] https://x.com/oomahq/status/2085717166884618584
+Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
+Post by: flatt on August 08, 2026, 09:55:35 AM
+Quote from: LoyceV on Today at 08:07:50 AM
+Quote from: vapourminer on August 04, 2026, 10:34:44 AM
+some people have a non passphrase wallet as a decoy with passphrases behind it.
+I wonder how many people realize that in this case, a "decoy" wallet have increased the risk, as it tells the attacker they found a used seed phrase. That gives them the possibility to start brute-forcing the passphrase. If you would have used the weak seed phrase only in combination with a passphrase, it would have been many orders of magnitude more difficult to find.
+You missed 2 real things: the perspective of hacker & the function of decoy.
+it's called decoy for a reason. once the owner see their decoy is gone , the owner can immediately move the funds. it does not increased the risk at all, it literally does increased the security for the funds. and in this case (the worst case ever displayed on this day UNTIL probably 100 next years), the hacker knows well brute-forcing into passphrase is such a waste of time because the owner could be already moved the real funds within no-time.
+FYI, brute-forcing passphrase for a single wallet most likely TOOK MORE TIME than brute-forcing the whole ColdCard default seed.
+Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
+Post by: kTimesG on August 08, 2026, 10:04:42 AM
+Quote from: JayJuanGee on Today at 02:25:57 AM
+You are saying also that no entropy was added by any dice rolls, so I would need to be explained the mechanics of how that would be based on if you are saying that there might have been an override of the Cold Card software that pushed it back to the 40 bits of entropy?
+It's not 40 bits of entropy, more like 23 (or even less, CC would know better of their die cutting parameters). Watching the official YouTube easy guide "5 minutes setup" of a fresh CC adds some 6 additional bits. Each dice adds around 2 bits on top of this. So that's only around 2 to 3 billion guesses for all seeds out of all entropies + one dice rolls. A laptop can break that in an hour.
+Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
+Post by: bitmover on August 08, 2026, 11:09:28 AM
+Quote from: joker_josue on Today at 07:39:17 AM
+Quote from: BlackHatCoiner on Today at 07:32:36 AM
+I'm afraid this incident does reveal why most people won't properly custody their bitcoin, ever.
+If it is required from people to roll dice, use a passphrase apart from a seed phrase, or engage in multi-vendor multi-sig setups for "extra security", I'm afraid most people who "believe" in Bitcoin's value proposition will just stick with an ETF from now on.
+And to be frank with all of you, I'm starting to think this is very justifiable to a point.
+Owning a Bitcoin ETF is not the same as owning Bitcoin!
+What really needs to be done is for people to read or reread the Bitcoin white paper again, to understand or remember what Bitcoin is.
+I think both points are valid.
+But it is much better to own a bitcoin ETF than to own a hacked wallet.
+Maybe it will make sense for some people to have both.
+I am also affraid that when you own a bitcoin ETF you do not own a permitionless money, but you can at least have some protection against inflation.
+Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
+Post by: joker_josue on August 08, 2026, 11:19:59 AM
+Quote from: sergiorus on Today at 09:50:09 AM
+Someone on Twitter [1] did their own investigation and is claiming a possible inside job.
+It's a thread of many tweets, the link is here (https://x.com/oomahq/status/2085717166884618584)
+I had heard him talk about this investigation, but I hadn't found it yet.
+Thank you for sharing.
+Things are going to get very ugly for the Coldcard team. It remains to be seen when the first lawsuits will start to come.
+Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
+Post by: vapourminer on August 08, 2026, 11:51:30 AM
+Quote from: BlackHatCoiner on Today at 07:32:36 AM
+I'm afraid this incident does reveal why most people won't properly custody their bitcoin, ever.
+If it is required from people to roll dice, use a passphrase apart from a seed phrase, or engage in multi-vendor multi-sig setups for "extra security", I'm afraid most people who "believe" in Bitcoin's value proposition will just stick with an ETF from now on.
+And to be frank with all of you, I'm starting to think this is very justifiable to a point.
+its always been this way really. ive always recommended (shudder) a registered exchange for noobs. because they expect to be able to "reset" passwords (pins, passphrases) on demand. the fact that they lose a piece of paper and it gone for good? thats not for most of the people i meet.
+so only after they fully understand self custody do i recommend hardware airgaped wallets etc.
+most people in general arent ready for self custody and it seems many here werent ready either.
+im on team dice but only because i hang on this forum. i easily could of been a victim years ago.
+Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
+Post by: BlackHatCoiner on August 08, 2026, 11:56:08 AM
+Quote from: vapourminer on Today at 11:51:30 AM
+its always been this way really.
+I know, but I used to tell to newcomers that you can "just get a hardware wallet and sleep easy" in case they didn't want to trust an exchange (and many don't trust them). Now I'm not sure what the correct suggestion is. Expecting them to use a signing device and roll dice is not something I can recommend to everyday people.
+Quote
+im on team dice but only because i hang on this forum. i easily could of been a victim years ago.
+I never trusted my computer's RNG in the first place, and I remember asking in this forum how most people do trust it. (I know the Coldcard issue was not the hardware's RNG per se, but I still would rather trust something I can verify with my own eyes.)
+Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
+Post by: Cricktor on August 08, 2026, 12:09:10 PM
+Quote from: joker_josue on Today at 07:23:16 AM
+The only way to be prepared anywhere in the world is to always have your seed with you.
+How many of us have all the seeds with us 365 days a year, always?
+If that were actually necessary, we should consider the whole setup to be terribly wrong and faulty. I find this very very wrong to have to carry my mnemonic seeds with me all the time. Sorry, no, this should NOT be necessary! And it is not necessary if we knew that entropy generation wasn't screwed up completely.
+Apparently there's more due diligence needed to verify that entropy generation isn't flawed. Easier said, than done, though...
+I must say, I'm genuinely surprised how badly a hardware wallet vendor screwed this up like Coinkite did. I thought light-hearted that this shouldn't be possible because it's such a basic and important functionality to NOT screw up the random stuff when you have TRNGs available. I'm still puzzled...
+Quote from: NotATether on Today at 07:26:21 AM
+The necessary step is, and I'll keep yelling this from rooftops until it reaches mass adoption, to generate your own seed phrase yourself, without any hardware wallet.
+And also make a BIP39 passphrase for your seed.
+This can only work if users learn and understand how to safely and properly generate their own mnemonic seeds. There are many ways to screw this up without being easily able to notice it.
+A mnemonic passphrase (the additional thing) needs to be complex enough to withstand brute-force attacks (it's also computationally somewhat expensive as each guess iteration requires 2048 rounds of PBKDF2 with HMAC-SHA512), but it adds a security layer with no margin for error if you fail to document it properly and highly recommended also redundantly. Commonly you should also separate it from your mnemonic recovery words, so it needs separate secure storage places.
+That's quite a (necessary) burden with many possibilities for people to screw up.
+On the other hand, with the necessary knowledge and understanding, I'm with you. I just have doubts it will work out for the masses.
+Quote from: LoyceV on Today at 08:07:50 AM
+I wonder how many people realize that in this case, a "decoy" wallet have increased the risk, as it tells the attacker they found a used seed phrase. That gives them the possibility to start brute-forcing the passphrase. If you would have used the weak seed phrase only in combination with a passphrase, it would have been many orders of magnitude more difficult to find.
+If the additional mnemonic passphrase is brute-forceable, the creator has failed already badly. It does not make sense to me, e.g. to use just a few additional words from the BiP39 wordlist. The mnemonic passphrase needs to be complex enough that brute-force attacks are not feasible.
+With a complex enough mnemonic passphrase I consider the "decoy" wallet as a valuable canary indicator of compromise. Users of such a setup should of course pay very much attention to not link their "decoy" UTXOs with their "hidden" main stash(es).
+Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
+Post by: LoyceV on August 08, 2026, 12:22:57 PM
+Quote from: flatt on Today at 09:55:35 AM
+Quote from: LoyceV on Today at 08:07:50 AM
+Quote from: vapourminer on August 04, 2026, 10:34:44 AM
+some people have a non passphrase wallet as a decoy with passphrases behind it.
+I wonder how many people realize that in this case, a "decoy" wallet have increased the risk, as it tells the attacker they found a used seed phrase. That gives them the possibility to start brute-forcing the passphrase. If you would have used the weak seed phrase only in combination with a passphrase, it would have been many orders of magnitude more difficult to find.
+You missed 2 real things: the perspective of hacker & the function of decoy.
+Nope, I didn't miss that. A decoy works if someone finds your (physical) seed phrase backup. In this case, the decoy is like a big flag that says: "there could be more money here".
+Quote
+the hacker knows well brute-forcing into passphrase is such a waste of time because the owner could be already moved the real funds within no-time.
+There have already been reports of 2-word passphrases being compromised.
+Quote
+FYI, brute-forcing passphrase for a single wallet most likely TOOK MORE TIME than brute-forcing the whole ColdCard default seed.
+That completely depends on the passphrase. If someone uses just 2 BIP39 words, that only gives 4 million possibilities, which is many orders of magnitude easier to brute-force than the weak seed words.
+Quote from: BlackHatCoiner on Today at 11:56:08 AM
+I know, but I used to tell to newcomers that you can "just get a hardware wallet and sleep easy" in case they didn't want to trust an exchange (and many don't trust them). Now I'm not sure what the correct suggestion is.
+I used to recommend Ledger or Trezor, until Ledger went nuts. Now I'd recommend Trezor only, just because they've been around for a long time. But even though I know one fuckup doesn't mean another hardware wallet will do the same, the rate at which hardware wallets are disappearing from being trusted is not comforting. I've also seen hardware wallets that simply lose support after a few years, and users will have to buy a new one again. This is something for the long run, if a simple paper wallet outlives a hardware wallet, why even bother with the hardware?.
+Quote from: Cricktor on Today at 12:09:10 PM
+With a complex enough mnemonic passphrase I consider the "decoy" wallet as a valuable canary indicator of compromise. Users of such a setup should of course pay very much attention to not link their "decoy" UTXOs with their "hidden" main stash(es).
+The one major reason for me to choose a hardware wallet over offline cold storage, is that it's much more convenient to make a payment. If I have to manually enter 12 randon words on a devide with 2 buttons each time I want to make a payment, I don''t really see the point of using it anymore.
+Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
+Post by: vapourminer on August 08, 2026, 12:43:16 PM
+Quote from: LoyceV on Today at 12:22:57 PM
+Quote from: flatt on Today at 09:55:35 AM
+Quote from: LoyceV on Today at 08:07:50 AM
+Quote from: vapourminer on August 04, 2026, 10:34:44 AM
+some people have a non passphrase wallet as a decoy with passphrases behind it.
+I wonder how many people realize that in this case, a "decoy" wallet have increased the risk, as it tells the attacker they found a used seed phrase. That gives them the possibility to start brute-forcing the passphrase. If you would have used the weak seed phrase only in combination with a passphrase, it would have been many orders of magnitude more difficult to find.
+You missed 2 real things: the perspective of hacker & the function of decoy.
+Nope, I didn't miss that. A decoy works if someone finds your (physical) seed phrase backup. In this case, the decoy is like a big flag that says: "there could be more money here".
+perhaps there is a script that monitors the decoy addy. the decoy gets swept: tons of warnings in real life start happening. gives a head start to move the passphrased coins out.
+one works from real life attacks, one works from cyber attacks.. half dozen of one is 6 of the other, pick yer poison
+Quote from: LoyceV on Today at 12:22:57 PM
+[...] if a simple paper wallet outlives a hardware wallet, why even bother with the hardware?.
+pretty much this. and i started with one of the 1st trezors. i still trust trezor but i bring my own entropy now.
+Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
+Post by: philipma1957 on August 08, 2026, 01:30:05 PM
+Quote from: vapourminer on Today at 12:43:16 PM
+Quote from: LoyceV on Today at 12:22:57 PM
+Quote from: flatt on Today at 09:55:35 AM
+Quote from: LoyceV on Today at 08:07:50 AM
+Quote from: vapourminer on August 04, 2026, 10:34:44 AM
+some people have a non passphrase wallet as a decoy with passphrases behind it.
+I wonder how many people realize that in this case, a "decoy" wallet have increased the risk, as it tells the attacker they found a used seed phrase. That gives them the possibility to start brute-forcing the passphrase. If you would have used the weak seed phrase only in combination with a passphrase, it would have been many orders of magnitude more difficult to find.
+You missed 2 real things: the perspective of hacker & the function of decoy.
+Nope, I didn't miss that. A decoy works if someone finds your (physical) seed phrase backup. In this case, the decoy is like a big flag that says: "there could be more money here".
+perhaps there is a script that monitors the decoy addy. the decoy gets swept: tons of warnings in real life start happening. gives a head start to move the passphrased coins out.
+one works from real life attacks, one works from cyber attacks.. half dozen of one is 6 of the other, pick yer poison
+Quote from: LoyceV on Today at 12:22:57 PM
+[...] if a simple paper wallet outlives a hardware wallet, why even bother with the hardware?.
+pretty much this. and i started with one of the 1st trezors. i still trust trezor but i bring my own entropy now.
+Yeah I ended up adding passphrase to my trezors.
+3 washers with 8 characters each
+I decided on 32 of my 36 punches knock out 0OIL
+so my passphrases are 32 to the 8th cubed or
+1099511627776 x 1099511627776 x 109951162776
+that's 121 bits and the 1099511627776 is all the cold card had as that is 40 bits.
+as for convinent well the trezor has a standard wallet with say 0.04 BTC and the each of the 3
+passphrases have say 0.32BTC total 1 of  btc
+what's nice is keeping the passphrase in my banks safety deposit box slows .96 btc from my own spending
+issues.
+do I like it no
+Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
+Post by: Woodie on August 08, 2026, 01:43:17 PM
+Quote from: sergiorus on Today at 09:50:09 AM
+Someone on Twitter [1] did their own investigation and is claiming a possible inside job.
+It's a thread of many tweets, the link is here (https://x.com/oomahq/status/2085717166884618584)
+~snip~
+Inside job would be an understatement, see shared tweet.. is this possible that this was left as a backdoor for themselves ?
+They had a light moment about the entropy bug 🐛 and some years pass they get hit by the very entropy generation bug. Definitely pulled a retirement job on their clients and it was just a matter of time!
+https://www.talkimg.com/images/2026/08/08/Uoeig3.jpg
+https://x.com/COLDCARDwallet/status/1447213375398846473
+Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
+Post by: flatt on August 08, 2026, 01:56:14 PM
+Quote from: LoyceV on Today at 12:22:57 PM
+Nope, I didn't miss that. A decoy works if someone finds your (physical) seed phrase backup. In this case, the decoy is like a big flag that says: "there could be more money here".
+There have already been reports of 2-word passphrases being compromised.
+That completely depends on the passphrase. If someone uses just 2 BIP39 words, that only gives 4 million possibilities, which is many orders of magnitude easier to brute-force than the weak seed words.
+In this case, and many similar cases to come, a passphrase is still a good way to secure your funds, and letting hackers eat the decoy as a warning to move your funds immediately. This still doesn't increase the risk as you claim. The rest depends on how the user uses the "passphrase." The user could even use a one-word passphrase with 64 hexadecimal characters, and the funds would still be there until next Christmas when the user is ready to move the funds.
+In short, it depends on:
+1. How the attacker detects whether it's a decoy address and decides to invest all their resources in a single wallet that they believe "It's a decoy, got a correct passphrase and you'll get the whale." which there's no script / tools exist for detecting such things.
+2. (MOST IMPORTANTLY) how the user leverages the passphrase instead of entering their Instagram password to secure their real funds.
+I still believe in SHA256, and accessing my funds from anywhere I want without letting a company give me the keys to my funds or storing my own keys in SDB is far worse.
+Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
+Post by: flatt on August 08, 2026, 02:22:40 PM
+Quote from: philipma1957 on Today at 01:30:05 PM
+Quote from: vapourminer on Today at 12:43:16 PM
+Quote from: LoyceV on Today at 12:22:57 PM
+Quote from: flatt on Today at 09:55:35 AM
+Quote from: LoyceV on Today at 08:07:50 AM
+Quote from: vapourminer on August 04, 2026, 10:34:44 AM
+some people have a non passphrase wallet as a decoy with passphrases behind it.
+I wonder how many people realize that in this case, a "decoy" wallet have increased the risk, as it tells the attacker they found a used seed phrase. That gives them the possibility to start brute-forcing the passphrase. If you would have used the weak seed phrase only in combination with a passphrase, it would have been many orders of magnitude more difficult to find.
+You missed 2 real things: the perspective of hacker & the function of decoy.
+Nope, I didn't miss that. A decoy works if someone finds your (physical) seed phrase backup. In this case, the decoy is like a big flag that says: "there could be more money here".
+perhaps there is a script that monitors the decoy addy. the decoy gets swept: tons of warnings in real life start happening. gives a head start to move the passphrased coins out.
+one works from real life attacks, one works from cyber attacks.. half dozen of one is 6 of the other, pick yer poison
+Quote from: LoyceV on Today at 12:22:57 PM
+[...] if a simple paper wallet outlives a hardware wallet, why even bother with the hardware?.
+pretty much this. and i started with one of the 1st trezors. i still trust trezor but i bring my own entropy now.
+Yeah I ended up adding passphrase to my trezors.
+3 washers with 8 characters each
+I decided on 32 of my 36 punches knock out 0OIL
+so my passphrases are 32 to the 8th cubed or
+1099511627776 x 1099511627776 x 109951162776
+that's 121 bits and the 1099511627776 is all the cold card had as that is 40 bits.
+as for convinent well the trezor has a standard wallet with say 0.04 BTC and the each of the 3
+passphrases have say 0.32BTC total 1 of  btc
+what's nice is keeping the passphrase in my banks safety deposit box slows .96 btc from my own spending
+issues.
+do I like it no
+I would never make myself harder than before like that, but currently I choose my favorite signature, hashes it and put it as my extra passphrase, which I can access it from anywhere anytime.
+Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
+Post by: LoyceV on August 08, 2026, 02:24:10 PM
+Quote from: flatt on Today at 01:56:14 PM
+This still doesn't increase the risk as you claim.
+You're still missing the point: if someone added a passphrase to his weak Coldcard without funding a decoy wallet, his funds wouldn't have been taken. It's the decoy that makes the seed words and thus potential passphrase light up.
+Quote
+The user could even use a one-word passphrase with 64 hexadecimal characters
+At this point it's easier to just type a private key, it's shorter.
+Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
+Post by: joker_josue on August 08, 2026, 02:34:51 PM
+Quote from: vapourminer on Today at 11:51:30 AM
+im on team dice but only because i hang on this forum. i easily could of been a victim years ago.
+You build your seeds with the dice and then "retrieve" them in a hardware wallet?
+Quote from: Cricktor on Today at 12:09:10 PM
+Quote from: joker_josue on Today at 07:23:16 AM
+The only way to be prepared anywhere in the world is to always have your seed with you.
+How many of us have all the seeds with us 365 days a year, always?
+If that were actually necessary, we should consider the whole setup to be terribly wrong and faulty. I find this very very wrong to have to carry my mnemonic seeds with me all the time. Sorry, no, this should NOT be necessary! And it is not necessary if we knew that entropy generation wasn't screwed up completely.
+Apparently there's more due diligence needed to verify that entropy generation isn't flawed. Easier said, than done, though...
+Exactly, I agree.
+The seed is not meant to be passed around; it should be kept safe and sound in a secure place. Of course, it's good for a person to think about how they can access the seed if they can't return to its original location. But that's a different matter altogether.
+Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
+Post by: flatt on August 08, 2026, 02:40:44 PM
+Quote from: LoyceV on Today at 02:24:10 PM
+Quote from: flatt on Today at 01:56:14 PM
+This still doesn't increase the risk as you claim.
+You're still missing the point: if someone added a passphrase to his weak Coldcard without funding a decoy wallet, his funds wouldn't have been taken. It's the decoy that makes the seed words and thus potential passphrase light up.
+Good point, but at that point, there's no such decoy thing if you're not funding on it.
 Powered by SMF 1.1.19 |
 SMF © 2006-2009, Simple Machines
