--- bitcointalk-large-scale-compromise-thread@20260809T165441Z
+++ bitcointalk-large-scale-compromise-thread@20260809T232458Z
@@ -4572,8 +4572,6 @@
 1. How the attacker detects whether it's a decoy address and decides to invest all their resources in a single wallet that they believe "It's a decoy, got a correct passphrase and you'll get the whale." which there's no script / tools exist for detecting such things.
 2. (MOST IMPORTANTLY) how the user leverages the passphrase instead of entering their Instagram password to secure their real funds.
 I still believe in SHA256, and accessing my funds from anywhere I want without letting a company give me the keys to my funds or storing my own keys in SDB is far worse.
-Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
-Post by: flatt on August 08, 2026, 02:22:40 PM
 Quote from: philipma1957 on August 08, 2026, 01:30:05 PM
 Quote from: vapourminer on August 08, 2026, 12:43:16 PM
 Quote from: LoyceV on August 08, 2026, 12:22:57 PM
@@ -4596,7 +4594,7 @@
 1099511627776 x 1099511627776 x 109951162776
 that's 121 bits and the 1099511627776 is all the cold card had as that is 40 bits.
 as for convinent well the trezor has a standard wallet with say 0.04 BTC and the each of the 3
-passphrases have say 0.32BTC total 1 of  btc
+passphrases have say 0.32BTC total 1 of  btc
 what's nice is keeping the passphrase in my banks safety deposit box slows .96 btc from my own spending
 issues.
 do I like it no
@@ -4646,7 +4644,7 @@
 It's perfectly safe to let a hardware calculate the checksum word. Out of the entire list of 2048 words, only 7 or 8 will form a checksum for a 24 word seed. So, you're not sacrificing any randomness by letting a hardware wallet calculate the checksum word.
 You've now got a totally random 24 word seed phrase. And you know it's random because you did the work.
 Pick another 10 words from the bowl of paper strips. That's your passphrase.
-I'm sure somebody is thinking "But the paper strips don't weigh exactly the same!" That's right. Any imperfections  just add to the randomness since those imperfections can't be predicted or calculated.
+I'm sure somebody is thinking "But the paper strips don't weigh exactly the same!" That's right. Any imperfections  just add to the randomness since those imperfections can't be predicted or calculated.
 I started doing this after Ledger added key extraction to their firmware. I was a Ledger user back then, and their actions made me question how much I could trust the firmware for any device. So, I figured, if I don't let a device generate my seed phrase, I don't have to trust the device.
 I also swear by ShieldSigner, which is a fork of SeedSigner that adds encrypted Seed QR, passphrase QR, and smartcard support (SeedKeeper and Satochip).
 This gives me the ability to use my seed on a device which is airgapped and stateless.
@@ -4663,8 +4661,6 @@
 6. Get a small safe to keep in your home, where you'll store any documentation that needs to be written down. Document everything for your setup, even if only to help yourself remember "How'd I generate this seed? Why'd I set it up this way?"
 7. Get home automation and put a sensor on the safe, to send you instant notifications if it is opened or moved. Aqara makes this easy and cheap. On sale, you can get an Aqara hub & sensors for under $50.
 I know this list sounds like a lot, but really it's not.
-Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
-Post by: Meuserna on August 08, 2026, 05:43:45 PM
 Quote from: flatt on August 08, 2026, 09:55:35 AM
 Quote from: LoyceV on August 08, 2026, 08:07:50 AM
 Quote from: vapourminer on August 04, 2026, 10:34:44 AM
@@ -4931,5 +4927,170 @@
 I'm very sure Coldcard lawyers are going to be busy with the rest of the year with different lawsuits.
 Aren't we underestimating the hacker here? He has been so clever to find this bug in Coldcard that wasn't detected before, and now we are assuming that the hacker will have a hard time cashing out those stolen BTC, and he will either end up in the wrong hands or get traced???
 I am sure he must have already figured out how to get out of all this without getting noticed. He has a criminal mind, and we are just thinking that he will find trouble spending those bitcoins  ???  I think we are too innocent to think like this, and if he is reading this, he must be laughing at us.
+Title: Re: Large-scale Coldcard compromise (1596 BTC stolen so far)
+Post by: Webetcoins on August 09, 2026, 05:21:59 PM
+Quote from: joker_josue on Today at 08:05:56 AM
+it seems that the coins, after being stolen, were not touched again.
+Is the hacker still thinking about what to do, or is he feeling guilty?
+The thing is, at first it all looks very nice and generates a lot of excitement, but when you start to see the real impact, things change a bit.
+I'm not saying he's going to return the stolen coins, but at least he might be going through a period of reflection.
+He is probably thinking of the best way for him to cash out all or at least some of those assets so that he can use them in real life. Just because he isn't touching the coins doesn't mean he is feeling remorse for what he did, because such people barely care about these things or have such feelings, if they did, they wouldn't do it in the first place. One could argue that those who get into doing such things are the ones who have suffered a lot themselves in life, but doesn't that make it even more logical for them to understand what it means to have nothing or to lose everything you have or have workeed for your entire life?
+So, I think the reason why there is a delay in moving those funds is because the attacker knows that he has so many eyes on him, and he probably wouldn't want to make a silly mistake and get caught somehow, so he is surely exploring and researching all his options that he can use to convert his bitcoins either to another untraceable asset, or simply convert it to fiat and use it, because it obviously isn't his goal to just steal the funds and let it lying around in a wallet, but the most difficult thing in such attacks is not the attack itself but it is to use the funds after it.
+Title: Re: Large-scale Coldcard compromise (1596 BTC stolen so far)
+Post by: pbies on August 09, 2026, 05:33:02 PM
+XMR (Monero) is no longer usable on exchanges, so he could use ZCash or just swap chunks in Exodus wallet to other crypto, mix it and pay out.
+Title: Re: Large-scale Coldcard compromise (1596 BTC stolen so far)
+Post by: sergiorus on August 09, 2026, 05:44:57 PM
+Quote from: pbies on Today at 05:33:02 PM
+XMR (Monero) is no longer usable on exchanges, so he could use ZCash or just swap chunks in Exodus wallet to other crypto, mix it and pay out.
+What do you mean by "not usable"?
+It's still listed on a plenty of centralised exchanges with decent liquidity as well as on DEXs but with questionable liquidity if that's what you meant.
+Title: Re: Large-scale Coldcard compromise (1596 BTC stolen so far)
+Post by: joker_josue on August 09, 2026, 06:03:25 PM
+Quote from: Lucius on Today at 01:19:29 PM
+Do you think someone who clearly doesn't know what passphrase or multi-sig is knows how to insert a message into their transaction? My opinion is that most of these messages come from scammers who play the victim card in the hope that some rich donor will see it and send a big donation.
+I think one thing has nothing to do with the other.
+Aside from the most paranoid (and rightfully so), the vast majority of people didn't add a 25th word to their seed created in a hardware wallet, considered reliable by the overwhelming majority of the community.
+Otherwise, we'll be jeopardizing the entire seed system, which also doesn't make sense.
+Therefore, a person not having the passphrase does not mean that they had little knowledge about Bitcoin.
+Quote from: Lucius on Today at 01:19:29 PM
+Most people assume that it is a person, but isn't it possible that it is not a person but an advanced AI that was tasked with trying to hack a hardware wallet? Most people think that AI capabilities are what we see in popular models mainly from US companies - but the same companies and the US government itself have admitted that China, for example, has far more advanced AI models that are capable of much more than we can even imagine.
+That's a plausible scenario. I'd never thought of that. Could some AI be running loose, stealing money, and even the AI ​​creators themselves don't know about it?
+We're already entering the realm of conspiracy theory.   ::)
+Quote from: UmerIdrees on Today at 04:52:20 PM
+Aren't we underestimating the hacker here? He has been so clever to find this bug in Coldcard that wasn't detected before, and now we are assuming that the hacker will have a hard time cashing out those stolen BTC, and he will either end up in the wrong hands or get traced???
+I am sure he must have already figured out how to get out of all this without getting noticed. He has a criminal mind, and we are just thinking that he will find trouble spending those bitcoins  ???  I think we are too innocent to think like this, and if he is reading this, he must be laughing at us.
+Quote from: Webetcoins on Today at 05:21:59 PM
+So, I think the reason why there is a delay in moving those funds is because the attacker knows that he has so many eyes on him, and he probably wouldn't want to make a silly mistake and get caught somehow, so he is surely exploring and researching all his options that he can use to convert his bitcoins either to another untraceable asset, or simply convert it to fiat and use it, because it obviously isn't his goal to just steal the funds and let it lying around in a wallet, but the most difficult thing in such attacks is not the attack itself but it is to use the funds after it.
+Time is everything with this type of hack. The more time passes, the harder it is to get rid of the coins, not the other way around.
+The longer he waits, the less things get "forgotten," and the better prepared blockchain analysts are to track these coins. The entire community, including exchanges, swaps, and the like, has become better prepared to catch any movement of these coins.
+He's already missed the window of opportunity to move without being caught. Now, every step he takes is under surveillance.
+Of course, this doesn't mean they won't try to withdraw the money at any time, but the longer it goes on, the more difficult it will be.
+That's why I'm saying this: even the hacker himself wasn't expecting to be so successful. Otherwise, as soon as he had acquired the first coins, he would have immediately sent them to mixes and swaps to break their trail.
+Perhaps he had already planned this, but the news broke too quickly, leaving him with no room for maneuver.
+I'm not saying that he is naive or ingenuous. I'm simply presenting the scenario where the hacker who carried out the attack wasn't expecting the success they had, and was left unsure of what to do. Because stealing 10 BTC is one thing, stealing 1000 BTC is quite another; the whole plan changes.
+Title: Re: Large-scale Coldcard compromise (1596 BTC stolen so far)
+Post by: Stalker22 on August 09, 2026, 06:40:47 PM
+Quote from: fillippone on Today at 11:15:33 AM
+Quote from: LoyceV on Today at 10:48:34 AM
+Quote from: philipma1957 on Today at 10:38:43 AM
+He could keep them still for a month or two and then send them back
+To return the stolen funds, he'll need to identify each owner first, and you can bet many scammers will claim to be the owners. No single person can do that on his own.
+The addresses are now compromised.
+Everyone could claim the ownership of the sending addresses (the private keys).
+Identifiying the original and authentic owner is almost impossible (it would require checking upon the possession of the physical hw, something that looks very impractical to me).
+Even verifying possession of the physical HW will not be definitive proof.  It is theoretically possible for someone in possession of the ColdCard HW to recreate the wallet by importing a compromised seed.  How would that differ from the original wallet of the real victim?
+I think on-chain history is the only real proof.  To verify a claim, someone would have to trace where the funds originally came from - like an CEX withdrawal, or from some other KYC-ed account.  Doing that manual forensics for thousands of separate victim addresses is an insane amount of work though.
+Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
+Post by: JayJuanGee on August 09, 2026, 06:51:48 PM
+Quote from: kTimesG on August 08, 2026, 10:04:42 AM
+Quote from: JayJuanGee on August 08, 2026, 02:25:57 AM
+You are saying also that no entropy was added by any dice rolls, so I would need to be explained the mechanics of how that would be based on if you are saying that there might have been an override of the Cold Card software that pushed it back to the 40 bits of entropy?
+It's not 40 bits of entropy, more like 23 (or even less, CC would know better of their die cutting parameters). Watching the official YouTube easy guide "5 minutes setup" of a fresh CC adds some 6 additional bits. Each dice adds around 2 bits on top of this. So that's only around 2 to 3 billion guesses for all seeds out of all entropies + one dice rolls. A laptop can break that in an hour.
+You are probably not wrong in your presumption, that even the guys who did dice rolls did not do a sufficient quantity of dice rolls, and I already mentioned my understanding that 100 plus dice rolls adds 256 bits of entropy, 50 plus dice rolls adds 128 bits of entropy, 25 plus dice rolls adds 64 bits of entropy and lower quantity of rolls would add lower amounts of entropy... ~2.5 bits per dice roll.
+And, yeah, Cold Card software probably should not have allowed the setting up a wallet without a minimum quantity of extra entropy, perhaps even 25 plus might have had been enough, but maybe they should have had set the bare minimum quantity of dice rolls to a higher number like 30 plus, which would be around 72 bits of entropy, and even that might not have had been enough as a bare minimum amount, since allowing people to do it would mean that people would do the bare minimum, which is what happened, since there are examples of people not doing any dice rolls or just doing a few dice rolls or just making up their dice roll numbers.
+Quote from: Cricktor on August 08, 2026, 12:09:10 PM
+Quote from: joker_josue on August 08, 2026, 07:23:16 AM
+The only way to be prepared anywhere in the world is to always have your seed with you.
+How many of us have all the seeds with us 365 days a year, always?
+If that were actually necessary, we should consider the whole setup to be terribly wrong and faulty. I find this very very wrong to have to carry my mnemonic seeds with me all the time. Sorry, no, this should NOT be necessary! And it is not necessary if we knew that entropy generation wasn't screwed up completely.
+Apparently there's more due diligence needed to verify that entropy generation isn't flawed. Easier said, than done, though...
+I must say, I'm genuinely surprised how badly a hardware wallet vendor screwed this up like Coinkite did. I thought light-hearted that this shouldn't be possible because it's such a basic and important functionality to NOT screw up the random stuff when you have TRNGs available. I'm still puzzled...
+Another reason that a person might need to carry their seed words would be if there might end up being som difficulties crossing a border or maybe some other way that a person's hardware wallet might become compromised, in the event that they are traveling with their hardware wallet.  There may be instances that some guys might want to wipe their device when crossing certain borders.  There also could be instances in which their device might get stolen, lost or confiscated, so then they might want to be able to race to get  to their seed words to be able to sweep the funds in any main wallet or in any hidden wallet(s).
+Quote from: Meuserna on August 08, 2026, 05:33:19 PM
+Pick another 10 words from the bowl of paper strips. That's your passphrase.
+Have you ever tried to transact with a hardware wallet?  It will frequently require the passphrase to be re-entered, especially if a certain amount of time passes between transactions.
+Accordingly, I am not sure what advantage comes from having to type 10 words each time as compared with maybe having a passphrase that might have some quasi-random string of around 15-ish characters that include numbers, letters, cap letters and symbols.
+Quote from: Meuserna on August 08, 2026, 05:33:19 PM
+3. Back up your seed and passphrase on metal.
+4. Store your seed and passphrase somewhere secure. Somewhere only you have access to. Preferably 2 places, separate.
+I am perfectly fine with the two places idea, even though practically, there are likely some guys who are quite challenged to have two places that are sufficiently within their control.
+Quote from: Meuserna on August 08, 2026, 05:33:19 PM
+6. Get a small safe to keep in your home, where you'll store any documentation that needs to be written down. Document everything for your setup, even if only to help yourself remember "How'd I generate this seed? Why'd I set it up this way?"
+7. Get home automation and put a sensor on the safe, to send you instant notifications if it is opened or moved. Aqara makes this easy and cheap. On sale, you can get an Aqara hub & sensors for under $50.
+What if your house burns down?  I understand the document is separate from the seed, yet if you have the seed without the documents, is the seed still going to be useful?
+Title: Re: Large-scale Coldcard compromise (1596 BTC stolen so far)
+Post by: asUHWEceyc on August 09, 2026, 07:06:15 PM
+Quote from: LoyceV on Today at 08:27:01 AM
+Quote from: joker_josue on Today at 08:05:56 AM
+The thing is, at first it all looks very nice and generates a lot of excitement, but when you start to see the real impact, things change a bit.
+I'm not saying he's going to return the stolen coins, but at least he might be going through a period of reflection.
+That's why I came up with this theory:
+Quote from: LoyceV on August 01, 2026, 06:24:45 AM
+Or could it just be an amateur who's in way over his head? Someone who accidentally stumbled upon this weakness, and suddenly ended up with $70 million in stolen funds?
+Quote from: joker_josue on Today at 08:05:56 AM
+Perhaps we need to look again at the master Satoshi and see how he created his wallets.
+They've been sitting there for over 15 years without suffering any problems, despite probably being the most attacked wallets in the world.
+Satoshi used Send to Pubkey transactions, whilch may become quantum vulnerable in the future. That's not the best example for the long term. But his random generation must have been pretty good :)
+Surprisingly good for a Windows XP VM https://bitcointalk.org/Smileys/default/grin.gif
+But, a big bag of private keys also has certain advantages
+Also-
+How many NVK proximal influencer-podcaster-types (bent, odell, etc) and their investment projects (1031) that funded coinkite lost money in this thing? What about secondary corporate operators that used coldcards for key handholding services?
+Did they "know better" or get a heads up?
+Title: Re: Large-scale Coldcard compromise (1485.77 BTC stolen so far)
+Post by: Meuserna on August 09, 2026, 07:33:22 PM
+Quote from: JayJuanGee on Today at 06:51:48 PM
+Quote from: Meuserna on August 08, 2026, 05:33:19 PM
+Pick another 10 words from the bowl of paper strips. That's your passphrase.
+Have you ever tried to transact with a hardware wallet?  It will frequently require the passphrase to be re-entered, especially if a certain amount of time passes between transactions.
+Accordingly, I am not sure what advantage comes from having to type 10 words each time as compared with maybe having a passphrase that might have some quasi-random string of around 15-ish characters that include numbers, letters, cap letters and symbols.
+That's a huge flaw in most hardware wallets.
+Most hardware wallets make entering a passphrase so cumbersome that people use a short and weak passphrase instead.
+These days, my hardware wallets are Krux and ShieldSigner. Both include the option for Passphrase QR, which makes using very strong passphrases really easy. Save your passphrase as a QR code. To load the wallet, scan the QR with the passphrase.
+Before I switched to Krux, I was a Ledger user. I hate Ledger, but one thing they did right was make it easy to use a strong passphrase. Ledger devices allowed setting up 2 PIN codes to unlock the device.
+PIN 1 unlocked the device to the seed-only wallet.
+PIN 2 unlocked the device to the seed+passphrase wallet.
+That made using a very strong passphrase really easy. Unlock the device using PIN 2. Done.
+For years, I've been using passphrases that are over 50 characters long.
+My hope is that the ColdCard flaw teaches more people (especially devs) the importance of making it easier for users to quickly enter strong passphrases.
+Quote from: JayJuanGee on Today at 06:51:48 PM
+Quote from: Meuserna on August 08, 2026, 05:33:19 PM
+3. Back up your seed and passphrase on metal.
+4. Store your seed and passphrase somewhere secure. Somewhere only you have access to. Preferably 2 places, separate.
+I am perfectly fine with the two places idea, even though practically, there are likely some guys who are quite challenged to have two places that are sufficiently within their control.
+2 places: A safe in your home and a safe deposit box at the bank.
+Quote from: JayJuanGee on Today at 06:51:48 PM
+Quote from: Meuserna on August 08, 2026, 05:33:19 PM
+6. Get a small safe to keep in your home, where you'll store any documentation that needs to be written down. Document everything for your setup, even if only to help yourself remember "How'd I generate this seed? Why'd I set it up this way?"
+7. Get home automation and put a sensor on the safe, to send you instant notifications if it is opened or moved. Aqara makes this easy and cheap. On sale, you can get an Aqara hub & sensors for under $50.
+What if your house burns down?  I understand the document is separate from the seed, yet if you have the seed without the documents, is the seed still going to be useful?
+Your seed and passphrase should be backed up on metal in 2 places: A safe in your home and a safe deposit box at the bank. Your documentation for your wallet should be in 2 places too. This is especially true for anybody doing multisig.
+Bitcoin self custody comes with self responsibility. Those who aren't prepared to do it right or don't have the means to do it right should buy ETFs instead. It makes me sad to say that, but self custody needs to be done right.
+Title: Re: Large-scale Coldcard compromise (1596 BTC stolen so far)
+Post by: Meuserna on August 09, 2026, 09:23:14 PM
+Sometimes, this forum is bizarre.
+Mods deleted a comment where I explained how ColdCard wallet attackers are now targeting seed-only wallets they suspect are only decoys. Mods said the comment was off topic.
+Using a seed-only wallet as a decoy is how ColdCard attackers know which seeds might be hiding coins behind a passphrase. There are often clues that a wallet is actually just a decoy.
+How is that not relevant to this conversation?
+I believe the safe way to set up a decoy is to leave the seed-only wallet unused, and set up a decoy using a weak passphrase.
+An empty seed-only wallet gives anyone who finds it no clue that the wallet has ever been used. Adding a few sats there as a decoy gave the ColdCard attackers reason to see if they could find more hidden behind a passphrase. Now, if the passphrase for the real wallet is strong enough, those coins are safe (but should still be moved to a new seed+passphrase or multisig wallet for long term safety).
+Title: Re: Large-scale Coldcard compromise (1596 BTC stolen so far)
+Post by: Stalker22 on August 09, 2026, 09:33:12 PM
+Quote from: Meuserna on Today at 09:23:14 PM
+Sometimes, this forum is bizarre.
+Mods deleted a comment where I explained how ColdCard wallet attackers are now targeting seed-only wallets they suspect are only decoys. Mods said the comment was off topic.
+~
+The mods did not delete your post, they just merged your comment with your previous post.
+You can see it here: https://bitcointalk.org/index.php?topic=5589927.msg67024234#msg67024234
+Just try not to make multiple posts in a row!  It is against the forum rules (https://bitcointalk.org/index.php?topic=703657.0).  You can always use the edit button if you need to add something later.
+Title: Re: Large-scale Coldcard compromise (1596 BTC stolen so far)
+Post by: PrivacyG on August 09, 2026, 09:39:25 PM
+Quote from: Meuserna on Today at 09:23:14 PM
+I believe the safe way to set up a decoy is to leave the seed-only wallet unused, and set up a decoy using a weak passphrase.
+This is actually a fantastic idea.  Although it is an unnecessary effort in this Coldcard case.  If you have a strong Pass phrase, they would not find it any way no matter how long they search.  The weak decoy would only be useful in a 5 Dollar wrench attack.  And even then.  It is only useful if the attacker does not believe you may be hiding more.
+Which is what I believe would come after a short search for weak Pass phrases in the Coldcard case too.  Look for weak ones and then move on and try to find treasures.  They have all the time to do it.  And money to purchase better equipment now too.  Unless they are caught.
+Title: Re: Large-scale Coldcard compromise (1596 BTC stolen so far)
+Post by: Meuserna on August 09, 2026, 10:36:16 PM
+Quote from: PrivacyG on Today at 09:39:25 PM
+Quote from: Meuserna on Today at 09:23:14 PM
+I believe the safe way to set up a decoy is to leave the seed-only wallet unused, and set up a decoy using a weak passphrase.
+This is actually a fantastic idea.  Although it is an unnecessary effort in this Coldcard case.  If you have a strong Pass phrase, they would not find it any way no matter how long they search.  The weak decoy would only be useful in a 5 Dollar wrench attack.  And even then.  It is only useful if the attacker does not believe you may be hiding more.
+Which is what I believe would come after a short search for weak Pass phrases in the Coldcard case too.  Look for weak ones and then move on and try to find treasures.  They have all the time to do it.  And money to purchase better equipment now too.  Unless they are caught.
+Yeah.
+Decoy wallets aren't effective for a $5 wrench attack unless you have enough Bitcoin in the decoy wallet to convince the attacker they got everything. And, really, at the point when you've become a target for a violent in-person attack, your life is what you need to worry about.
+But for a ColdCard-style seed-hunting attack, I think a decoy wallet set up using a weak passphrase is a good idea because it becomes like an alarm that tells you someone is trying to crack your passphrase. Even if your passphrase is strong enough to prevent the attack (mine is well over 50 characters) it's still good to know someone is trying. Of course, the attacker has to take the decoy coins in order for you to know they found 'em and are trying to crack your passphrase.
+Using a weak passphrase as a decoy can also be a great form of security if your main wallet is set up some other way, such as multisig. A weak singlesig passphrase might convince an attacker to keep churning through passphrases to see if they can find your real wallet... which is especially useful if that's not how your real wallet is set up.
+This whole ColdCard attack is terrible, and my heart breaks for everyone who lost coins. But one sliver of good news is that the attack made everyone start looking for vulnerabilities to patch or improve on.
+I didn't lose any coins in this attack, but it did make me rethink every aspect of my own setup. We should all be doing that, because there's always room to improve even if it doesn't lead to changing your setup. I've been focusing on how I document mine for future reference, and how I teach the basics.
 Powered by SMF 1.1.19 |
 SMF © 2006-2009, Simple Machines
