--- bitcointalk-large-scale-compromise-thread@20260804T154904Z
+++ bitcointalk-large-scale-compromise-thread@20260804T164924Z
@@ -3027,5 +3027,16 @@
 so  back to core only?
 since if core is bad it is all dead.
 The Coldcard hacking incident is actually challenging the entire Bitcoin ecosystem, no one can say exactly what will happen in the future, but self-custody, which was people's last resort to keep their digital assets Bitcoin safe, is now distrusted. You are right that there is no guarantee that other companies will not do the same in the near future. Without people's trust and confidence, no security system is effectively secure, no matter how much companies say "your assets are 100% safe through our products", the public and companies must understand these things.
+Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
+Post by: Danish Ali on August 04, 2026, 04:29:39 PM
+Quote from: Catenaccio on Today at 12:44:35 PM
+Seedsigner looks good by passing 10 tests there.
+https://walletscrutiny.com/hardware/seedsigner/
+It is recommended by Jameson Lopp in his list of recommended wallets (https://www.lopp.net/bitcoin-information/recommended-wallets.html) but honestly I don't know and I am unsure because he recommended Coldcard and has yet removed this terrible hardware wallet from the list.
+In this What are best Bitcoin wallets (https://learnmeabitcoin.com/beginners/wallets/#best-wallets), there is no recommendation for SeedSigner.
+Personally I don't consider saving cost is important when I buy a hardware wallet because I understand its importance and there are free open source software wallets to use. If I spend money to buy a hardware wallet, I buy a best one and don't mind to save cost that possibly puts my fund at risk if I buy a bad hardware wallet.
+That recommendation list on Lopp's is losing trust is fair concern. However, SeedSigner trust model is quite different from that of Coldcard. Risk of Coldcard was centralized decision of a company owned maker. SeedSigner does not have maker, it's made out of everyday parts, so that type of failure is not relevant.
+Price factor plays a different role here as well. It is not about cost of the product you are buying, it is about removing risk from your shipping and parts supply. That is different type of benefit.
+It is still on Lopp's list and WalletScrutiny passes it, that is pretty good starting point for open source hardware.
 Powered by SMF 1.1.19 |
 SMF © 2006-2009, Simple Machines
