--- bitcointalk-large-scale-compromise-thread@20260804T151714Z
+++ bitcointalk-large-scale-compromise-thread@20260804T154904Z
@@ -3012,5 +3012,20 @@
 right down the road.
 so  back to core only?
 since if core is bad it is all dead.
+Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
+Post by: abaeze on August 04, 2026, 03:39:59 PM
+Quote from: philipma1957 on Today at 02:47:28 PM
+Quote from: asUHWEceyc on Today at 01:18:16 PM
+This certainly appears to be an exit scam with good plausible deniability when you look back at JWWeatherman_ badgering coldcard to add external entropy via dice rolls to their quick start guide in the 2020-2021 period, which they refused to do.
+It also casts a shadow on their podcast promoters, who were most likely mere useful idiots.
+The company is obviously done, through reputational damage and/or lawsuits, but someone has the coins
+lets say inside job.
+the issue is any other wallet company can do the same.
+firmware 2027 for ledger makes a bug
+and in 2028 ledger hacked.
+right down the road.
+so  back to core only?
+since if core is bad it is all dead.
+The Coldcard hacking incident is actually challenging the entire Bitcoin ecosystem, no one can say exactly what will happen in the future, but self-custody, which was people's last resort to keep their digital assets Bitcoin safe, is now distrusted. You are right that there is no guarantee that other companies will not do the same in the near future. Without people's trust and confidence, no security system is effectively secure, no matter how much companies say "your assets are 100% safe through our products", the public and companies must understand these things.
 Powered by SMF 1.1.19 |
 SMF © 2006-2009, Simple Machines
