--- bitcointalk-large-scale-compromise-thread@20260804T164924Z
+++ bitcointalk-large-scale-compromise-thread@20260804T171737Z
@@ -3038,5 +3038,22 @@
 That recommendation list on Lopp's is losing trust is fair concern. However, SeedSigner trust model is quite different from that of Coldcard. Risk of Coldcard was centralized decision of a company owned maker. SeedSigner does not have maker, it's made out of everyday parts, so that type of failure is not relevant.
 Price factor plays a different role here as well. It is not about cost of the product you are buying, it is about removing risk from your shipping and parts supply. That is different type of benefit.
 It is still on Lopp's list and WalletScrutiny passes it, that is pretty good starting point for open source hardware.
+Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
+Post by: asUHWEceyc on August 04, 2026, 04:53:22 PM
+Quote from: philipma1957 on Today at 02:47:28 PM
+Quote from: asUHWEceyc on Today at 01:18:16 PM
+This certainly appears to be an exit scam with good plausible deniability when you look back at JWWeatherman_ badgering coldcard to add external entropy via dice rolls to their quick start guide in the 2020-2021 period, which they refused to do.
+It also casts a shadow on their podcast promoters, who were most likely mere useful idiots.
+The company is obviously done, through reputational damage and/or lawsuits, but someone has the coins
+lets say inside job.
+the issue is any other wallet company can do the same.
+firmware 2027 for ledger makes a bug
+and in 2028 ledger hacked.
+right down the road.
+so  back to core only?
+since if core is bad it is all dead.
+I can only recommend storing coins safely at MtGox, BTC-e or FTX at this time
+All wallets are hardware wallets; some consist of electronic, analog and/or auditable components. Bitcoin has been more or less "working" for a long time and is highly scrutinized, with fewer LOC and less stuff bolted on the farther you go back. Intel CPUs commonly relied on, on the other hand, are implicitly un-examinable.
+If someone's selling a cheap turnkey solution it might be too good to be true. It seems reasonable that a company marketing such a product in the future may need to insure every device sold with an implementation loss warranty up to some nominal fiat amount.
 Powered by SMF 1.1.19 |
 SMF © 2006-2009, Simple Machines
