--- bitcointalk-large-scale-compromise-thread@20260804T191729Z
+++ bitcointalk-large-scale-compromise-thread@20260804T201842Z
@@ -3096,5 +3096,25 @@
 https://www.talkimg.com/images/2026/08/04/UoYhdj.png
 For instance, giving a response from a solution aspect to the firmware bugs. They respond to the bug on the latest firmware by giving a new device COLDCARD. Man, you own COLDCARD.... these what you should do on COLDCARD.
 https://i.bitlist.co/bJbyj2VAGyt1.gif
+Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
+Post by: Forsyth Jones on August 04, 2026, 07:49:23 PM
+Quote from: Cookdata on Today at 01:49:35 PM
+I have seen some creepy tweets of old posts from Coldcard but I don't think this company is worth defending.
+https://talkimg.com/images/2026/08/04/UoJbgc.jpeg
+https://x.com/coldcardwallet/status/1447213375398846473
+I don't think I'm overreacting right!
+He is very arrogant, which makes it harder not to think it was intentional, because when dealing with a data security device, the first thing a expert cryptography/dev must know is how to generate sensitive data with a good source of entropy. Coldcard is a hardware wallet, which has all the security features we can imagine, but they failed in the most critical aspect: the entropy used to generate wallets. Only those who used a strong passphrase were safe.
+Owners of hardware wallets should know what passphrase is, this also makes me have the following question: and other devices from other brands, how can we guarantee that such devices do not have some other flaw that could cause potential losses, we are in the age of AI guys, while there are millions of pcs using AI to search for hardware/software failures out there and they are evolving every day, we cannot do what the arrogant Nvk from coldcard did, which was ignore all the warnings from 2021 (like this tweet on the image for example).
+Ledger, its competitor that also did a lot of shit (like Recover), has a team of white hat who scour their competitors' devices in search of bugs, and mainly, very serious vulnerabilities, while nvk didn't review its own code, and if it did, it probably used vibe coding that did a lousy analysis, at the very least, as a developer, he should have a team responsible for reviewing the all code.
+I just haven't bought a coldcard yet, besides its price, I would have to pay between 60% - 300% in customs fees (yes, I live in a country that outrageously abuses taxing its own citizens).
+Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
+Post by: Alone055 on August 04, 2026, 08:16:12 PM
+Quote from: BlackHatCoiner on Today at 06:51:25 PM
+Quote from: LoyceV on Today at 05:41:33 PM
+Hypothetical: it just occurred to me that even if Coldcard knew about this vulnerability, they couldn't have warned users about it. The moment they issue a warning, potential attackers would know about it too, and their warning would have been the catalyst to losing funds.
+So once the bug was out there, all they could reasonably do was offer updated firmware and remove the vulnerability from newly sold devices.
+They could have taken most of the coins though, and give it back to the soon-to-be-victims though. Regardless, however, I agree that either way their business would be completely over.
+And yes, they could have warned the user of a vulnerability, but that should be worded in a way that it does not reveal your cold storage is at risk, or attackers would have got sooner than the victims by just scanning the codebase.
+Would it be illogical to think that this might not be just a random attack but a preplanned one? For which the bug or vulnerability was planted years ago, and was never found or patched on purpose to keep collecting information for a major attack, but doing it in a way that shouldn't make it suspicious for those in control of everything? Because there are cases unfolding where users reported similar problems years ago, but the developers or their support never took them seriously or looked into it. Maybe those few people who got their funds stolen back then were just victims of a few test tries if it actually works or not?  ::)
 Powered by SMF 1.1.19 |
 SMF © 2006-2009, Simple Machines
