--- bitcointalk-large-scale-compromise-thread@20260804T141715Z
+++ bitcointalk-large-scale-compromise-thread@20260804T151714Z
@@ -2999,5 +2999,18 @@
 What's wrong with a laminated paper wallet rolled up in a sealed PVC pipe filled with rice and buried in backyard?
 It's okay if you don't have a dog that likes to dig, and you haven't dug deep enough - or if you have a neighbor who watches you just for fun and decides to dig around your yard when you're not home. In addition, in some countries there are laws that say that the owner of the land is the owner of what is found up to a certain depth, and everything else is owned by the state. This is how they protect oil and gas deposits from ordinary people.
 The ground settles over time, so if you bury something to a depth of, say, half a meter, it will slowly sink over time, especially if the soil is moist and there is a lot of rainfall.
+Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
+Post by: philipma1957 on August 04, 2026, 02:47:28 PM
+Quote from: asUHWEceyc on Today at 01:18:16 PM
+This certainly appears to be an exit scam with good plausible deniability when you look back at JWWeatherman_ badgering coldcard to add external entropy via dice rolls to their quick start guide in the 2020-2021 period, which they refused to do.
+It also casts a shadow on their podcast promoters, who were most likely mere useful idiots.
+The company is obviously done, through reputational damage and/or lawsuits, but someone has the coins
+lets say inside job.
+the issue is any other wallet company can do the same.
+firmware 2027 for ledger makes a bug
+and in 2028 ledger hacked.
+right down the road.
+so  back to core only?
+since if core is bad it is all dead.
 Powered by SMF 1.1.19 |
 SMF © 2006-2009, Simple Machines
