--- bitcointalk-large-scale-compromise-thread@20260804T201842Z
+++ bitcointalk-large-scale-compromise-thread@20260804T204817Z
@@ -3116,5 +3116,8 @@
 They could have taken most of the coins though, and give it back to the soon-to-be-victims though. Regardless, however, I agree that either way their business would be completely over.
 And yes, they could have warned the user of a vulnerability, but that should be worded in a way that it does not reveal your cold storage is at risk, or attackers would have got sooner than the victims by just scanning the codebase.
 Would it be illogical to think that this might not be just a random attack but a preplanned one? For which the bug or vulnerability was planted years ago, and was never found or patched on purpose to keep collecting information for a major attack, but doing it in a way that shouldn't make it suspicious for those in control of everything? Because there are cases unfolding where users reported similar problems years ago, but the developers or their support never took them seriously or looked into it. Maybe those few people who got their funds stolen back then were just victims of a few test tries if it actually works or not?  ::)
+Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
+Post by: CryptoCrookz on August 04, 2026, 08:25:31 PM
+Seems diversification is the only real answer, with a preselection of suppliers based on some code aspects like rng characteristics - though let's see what the next 'hack of the month' will be. Including custodial options in the approach is also not bad idea, 'not your keys not your coins' is not the answer to everything.
 Powered by SMF 1.1.19 |
 SMF © 2006-2009, Simple Machines
