--- bitcointalk-large-scale-compromise-thread@20260804T171737Z
+++ bitcointalk-large-scale-compromise-thread@20260804T174736Z
@@ -3055,5 +3055,14 @@
 I can only recommend storing coins safely at MtGox, BTC-e or FTX at this time
 All wallets are hardware wallets; some consist of electronic, analog and/or auditable components. Bitcoin has been more or less "working" for a long time and is highly scrutinized, with fewer LOC and less stuff bolted on the farther you go back. Intel CPUs commonly relied on, on the other hand, are implicitly un-examinable.
 If someone's selling a cheap turnkey solution it might be too good to be true. It seems reasonable that a company marketing such a product in the future may need to insure every device sold with an implementation loss warranty up to some nominal fiat amount.
+Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
+Post by: vapourminer on August 04, 2026, 05:28:10 PM
+Quote from: asUHWEceyc on Today at 04:53:22 PM
+I can only recommend storing coins safely at MtGox, BTC-e or FTX at this time
+you jest but at least mtgox people got ~20% of their btc/bcash back
+Title: Re: Large-scale Coldcard compromise (1360.23 BTC stolen so far)
+Post by: LoyceV on August 04, 2026, 05:41:33 PM
+Hypothetical: it just occurred to me that even if Coldcard knew about this vulnerability, they couldn't have warned users about it. The moment they issue a warning, potential attackers would know about it too, and their warning would have been the catalyst to losing funds.
+So once the bug was out there, all they could reasonably do was offer updated firmware and remove the vulnerability from newly sold devices.
 Powered by SMF 1.1.19 |
 SMF © 2006-2009, Simple Machines
