--- reddit-nvk-ai-audit-podcast@20260806T163719Z
+++ reddit-nvk-ai-audit-podcast@20260807T193326Z
@@ -45,6 +45,9 @@
 
 Well, no one will say hardware wallets are bad, but it is time to make accountable hardware wallet vendors selling you unaudited shit and with no security in dept design. What is security in dept design or foolproof design? Well do not tell that rolling dice is optional but recommended, fucking do not allow the seed generation step to be completed without it. Specially if you ar going to say that AI reported bugs are crap and you have no external verified audit history of source code.
 
+_____
+https://gitlab.com/walletscrutiny/walletScrutinyCom/-/work_items/340
+
 comment: p1xm2dz
 parent: t3_1vgj17y
 author: Few_Response_7028
