--- bitcoinmag-closed-source-era@20260807T205411Z
+++ bitcoinmag-closed-source-era@20260812T051154Z
@@ -110,7 +110,7 @@
 Working toward a citadel archipelago, powered by bitcoin mining, regenerative farming, and hacking the fiat system. Find his Twitter here.
 RELATED ARTICLES
 TECHNICAL
-Bitcoin’s BIP110 Moment: Three Possible Scenarios
+Bitcoin’s BIP110 Moment: Three Possible Scenarios (Updated)
 August 7, 2026
 BUSINESS
 Bitcoin Red Team Finds 85 Critical Flaws Across 390 Open Source Repos After Coldcard Exploit
@@ -130,14 +130,14 @@
 Error loading data. Check console for details.
 VIEW 150+ BITCOIN CHARTS
 LATEST NEWS
-Coldcard Bitcoin Hack: Victims Report Median Loss of 1 BTC as Theft Tops $111 Million
-August 7, 2026
-Bitcoin Shrugs off Coldcard Hack and Clarity Act Delays, Price Chops Higher as Investors Buy ETFs
-August 7, 2026
-Bitcoin ETFs Add Nearly $800 Million in the Wake of Coldcard Exploit
-August 7, 2026
-Crucial Vote On Crypto Clarity Act Delayed Until September, Trump Says More People Are Paying in Bitcoin: Reports
-August 7, 2026
+Bank of Russia Approves Bitcoin Trading For Retail Investors
+August 11, 2026
+Luke Dashjr Removed as Bitcoin BIP Editor Following (BIP)-110 Fork Failure
+August 11, 2026
+Blockstream Debuts Swaps, Allowing Bitcoiners to Move Between Lightning and Main Network With Ease
+August 10, 2026
+BTCPay Server Offers a 3 Bitcoin Bounty for Recovery of Stolen Funds After Wallet Exploit
+August 10, 2026
 Load more
 EDITOR PICKS
 President Harris Should Buy Bitcoin to Pay Black Americans Reparations
@@ -157,7 +157,7 @@
 BUSINESS4306
 CULTURE3586
 MARKETS2428
-NEWS1499
+NEWS1509
 TECHNICAL1342
 INDUSTRY EVENTS366
 PRESS RELEASES292
