--- bitcoinmag-closed-source-era@20260807T133139Z
+++ bitcoinmag-closed-source-era@20260807T205411Z
@@ -109,15 +109,15 @@
 Colin Crossman
 Working toward a citadel archipelago, powered by bitcoin mining, regenerative farming, and hacking the fiat system. Find his Twitter here.
 RELATED ARTICLES
+TECHNICAL
+Bitcoin’s BIP110 Moment: Three Possible Scenarios
+August 7, 2026
 BUSINESS
 Bitcoin Red Team Finds 85 Critical Flaws Across 390 Open Source Repos After Coldcard Exploit
 August 5, 2026
 CULTURE
 Self Custody Is Dead. Long Live Self Custody
 August 4, 2026
-BUSINESS
-Coinkite Releases Fixed Firmware After Coldcard Bug; AI Likely Involved In The Breach
-July 31, 2026
 Bitcoin Portfolio Tracker & Media Updates
 Bitcoin BTC/USD
 $0.00
@@ -130,14 +130,14 @@
 Error loading data. Check console for details.
 VIEW 150+ BITCOIN CHARTS
 LATEST NEWS
-Breez Announces Glow, an Open Source Bitcoin to Stablecoins Progressive Web App
-August 6, 2026
-Senate Whip Barrasso Becomes Latest Lawmaker to Support Crypto Clarity Act, But Time May Be Running Out
-August 6, 2026
-Breez Drops New Bitcoin App Which Doubles As Wallet and Developer Toolkit
-August 6, 2026
-Bitcoin ETF Inflows Surge Following $130M Coldcard Hack
-August 6, 2026
+Coldcard Bitcoin Hack: Victims Report Median Loss of 1 BTC as Theft Tops $111 Million
+August 7, 2026
+Bitcoin Shrugs off Coldcard Hack and Clarity Act Delays, Price Chops Higher as Investors Buy ETFs
+August 7, 2026
+Bitcoin ETFs Add Nearly $800 Million in the Wake of Coldcard Exploit
+August 7, 2026
+Crucial Vote On Crypto Clarity Act Delayed Until September, Trump Says More People Are Paying in Bitcoin: Reports
+August 7, 2026
 Load more
 EDITOR PICKS
 President Harris Should Buy Bitcoin to Pay Black Americans Reparations
@@ -157,8 +157,8 @@
 BUSINESS4306
 CULTURE3586
 MARKETS2428
-NEWS1494
-TECHNICAL1341
+NEWS1499
+TECHNICAL1342
 INDUSTRY EVENTS366
 PRESS RELEASES292
 LEGAL206
