--- bitcoinmag-closed-source-era@20260812T051154Z
+++ bitcoinmag-closed-source-era@20260812T115816Z
@@ -130,14 +130,14 @@
 Error loading data. Check console for details.
 VIEW 150+ BITCOIN CHARTS
 LATEST NEWS
-Bank of Russia Approves Bitcoin Trading For Retail Investors
+Regulators To Push Pro-Crypto Initiatives Following Clarity Act Delay
 August 11, 2026
-Luke Dashjr Removed as Bitcoin BIP Editor Following (BIP)-110 Fork Failure
+Should Bitcoin Companies Build USD Reserves? Understanding The Truth
 August 11, 2026
-Blockstream Debuts Swaps, Allowing Bitcoiners to Move Between Lightning and Main Network With Ease
-August 10, 2026
-BTCPay Server Offers a 3 Bitcoin Bounty for Recovery of Stolen Funds After Wallet Exploit
-August 10, 2026
+CFTC Charges Goliath Ventures With $400M Bitcoin Fraud
+August 11, 2026
+Casa CEO Nick Neuman: 233k BTC Moved To Safety After Coldcard Exploit Proves Self-Custody Resilience
+August 11, 2026
 Load more
 EDITOR PICKS
 President Harris Should Buy Bitcoin to Pay Black Americans Reparations
@@ -157,7 +157,7 @@
 BUSINESS4306
 CULTURE3586
 MARKETS2428
-NEWS1509
+NEWS1513
 TECHNICAL1342
 INDUSTRY EVENTS366
 PRESS RELEASES292
