--- coldcard-firmware-pr-691-patch@20260801T063934Z
+++ coldcard-firmware-pr-691-patch@20260806T032247Z
@@ -23,14 +23,14 @@
  shared/mk4.py   | 6 ++++--
  2 files changed, 5 insertions(+), 3 deletions(-)
 diff --git a/external/libngu b/external/libngu
-index 537519a8..a5121743 160000
+index 537519a82..a51217432 160000
 --- a/external/libngu
 +++ b/external/libngu
 @@ -1 +1 @@
 -Subproject commit 537519a829259622ea6b0334fbafd6cae852852f
 +Subproject commit a51217432308fd9ebc7c3d54b96a5af72b71b2e9
 diff --git a/shared/mk4.py b/shared/mk4.py
-index 3a3e229d..8a4b1572 100644
+index 3a3e229d3..8a4b15724 100644
 --- a/shared/mk4.py
 +++ b/shared/mk4.py
 @@ -38,13 +38,15 @@ def make_psram_fs():
