--- reddit-coinkite-silence-unprofessional@20260813T133346Z
+++ reddit-coinkite-silence-unprofessional@20260813T193220Z
@@ -173,3 +173,27 @@
 body:
 Bought a wallet not long ago.  Now it's useless.  Looking for a refund but they just point to terms of sales. 
 Guess I'll have to use it as offline password storage then. 
+
+comment: p3fxhcg
+parent: t3_1vm708g
+author: Search327
+created_utc: 1786628201
+edited: false
+body:
+Same here buddy.  I reserved the ARCA on 28-Jul-2026. I asked for a refund 12 days ago. I have no heard a word. FUCK THEM!
+
+comment: p3gqlir
+parent: t3_1vm708g
+author: 712Jefferson
+created_utc: 1786635871
+edited: false
+body:
+Probably desperately trying not to get sued.
+
+comment: p3h6v0n
+parent: t1_p3di6qt
+author: xirvin
+created_utc: 1786639986
+edited: false
+body:
+Power cycle should get rid of brick status. Remove batteries and leave the device de energized. The energize it after 1min. What device you had ? If you have the Q  can I buy it from you ?
