--- reddit-coinkite-silence-unprofessional@20260813T193220Z
+++ reddit-coinkite-silence-unprofessional@20260815T123007Z
@@ -197,3 +197,13 @@
 edited: false
 body:
 Power cycle should get rid of brick status. Remove batteries and leave the device de energized. The energize it after 1min. What device you had ? If you have the Q  can I buy it from you ?
+
+comment: p3sxv3m
+parent: t3_1vm708g
+author: Mission-Disaster-447
+created_utc: 1786780741
+edited: false
+body:
+They have created a whole website an published multiple blog posts on the matter. Its understandable that they cannot answer every query individually.
+
+[https://coldcard.com/security/status](https://coldcard.com/security/status)
