--- reddit-device-useful-if-company-closes@20260806T095740Z
+++ reddit-device-useful-if-company-closes@20260808T145929Z
@@ -347,3 +347,19 @@
 edited: false
 body:
 I updated and moved my funds to new wallet without issue 
+
+comment: p2gsmpx
+parent: t1_p1e31yy
+author: dadlif3
+created_utc: 1786199178
+edited: false
+body:
+I will take the Q off your hands for you, will pay shipping in bitcoin!
+
+comment: p2gu8hv
+parent: t1_p2gsmpx
+author: BTC_MYTH44
+created_utc: 1786199662
+edited: false
+body:
+No thanks, I’m keeping it as memorabilia 
