--- reddit-device-useful-if-company-closes@20260804T083712Z
+++ reddit-device-useful-if-company-closes@20260804T134107Z
@@ -323,3 +323,11 @@
 edited: false
 body:
 I was who reported this. The bug was confirmed yesterday. So yes the new firmware can brick your coldcard. I’ve added the github link to the pull requests that shows the problem on the original post.
+
+comment: p1m7rbl
+parent: t3_1ve3vnu
+author: RedReadRedemption
+created_utc: 1785835554
+edited: false
+body:
+For the right price, and with the right technical expertise of the user, I would consider them a bargain. For regular users, I would base it on the availability of the updates - which would most likely stop if/when the company closes. 
