--- checkonchain-coldcard-psa@20260803T091024Z
+++ checkonchain-coldcard-psa@20260803T111322Z
@@ -3,8 +3,8 @@
 If you use a Coldcard in your Bitcoin security system, please read this post.
 James Check (Checkmatey)
 Aug 01, 2026
-76
-22
+78
+24
 13
 Share
 G’day Folks,
@@ -96,8 +96,8 @@
 Please reach out if you have any issues, I am happy to assist however I can.
 Thanks for reading,
 James
-76
-22
+78
+24
 13
 Share
 Discussion about this post
@@ -108,7 +108,7 @@
 Um, which risk category does someone fall into if they used the full number of recommended dice rolls, as in the coldcard paranoid setup guide? I would argue 🟢 but I think it’s a weird omission in your post.
 Reply
 Share
-1 reply by James Check (Checkmatey)
+3 replies by James Check (Checkmatey) and others
 Ricardo Santiago
 2d
 Liked by James Check (Checkmatey)
@@ -116,7 +116,7 @@
 Reply
 Share
 1 reply
-20 more comments...
+22 more comments...
 TopLatestDiscussions
 No posts
 Ready for more?
