--- checkonchain-coldcard-psa@20260808T162652Z
+++ checkonchain-coldcard-psa@20260809T052957Z
@@ -103,14 +103,14 @@
 Discussion about this post
 CommentsRestacks
 loniousmonk
-8d
+Aug 1
 Liked by James Check (Checkmatey)
 Um, which risk category does someone fall into if they used the full number of recommended dice rolls, as in the coldcard paranoid setup guide? I would argue 🟢 but I think it’s a weird omission in your post.
 Reply
 Share
 3 replies by James Check (Checkmatey) and others
 Ricardo Santiago
-8d
+Aug 1
 Liked by James Check (Checkmatey)
 Great write up. My heart breaks for those who lost coins. I do see so many folks who didn’t lose coins completely ditching the Cold Card Q and other newer models. I was lucky enough to have generated an old school dice roll and make my seed phrase. The Cold Card Q is still a fantastic air gapped cold storage device. Some folks act like it can never be used ever again. What they did with this bug though can not go without some type of consequence (little to no sales or winding down of the company) trust has clearly been lost and it will be very interesting to see how Coin kite and Cold Card handle this going forward.
 Reply
