--- checkonchain-coldcard-psa@20260804T030837Z
+++ checkonchain-coldcard-psa@20260804T100851Z
@@ -3,9 +3,9 @@
 If you use a Coldcard in your Bitcoin security system, please read this post.
 James Check (Checkmatey)
 Aug 01, 2026
-81
+83
 24
-13
+14
 Share
 G’day Folks,
 The last two days have been uniquely challenging for many of us grizzled and tenured Bitcoiners.
@@ -96,9 +96,9 @@
 Please reach out if you have any issues, I am happy to assist however I can.
 Thanks for reading,
 James
-81
+83
 24
-13
+14
 Share
 Discussion about this post
 CommentsRestacks
