--- checkonchain-coldcard-psa@20260807T142341Z
+++ checkonchain-coldcard-psa@20260807T205239Z
@@ -103,7 +103,7 @@
 Discussion about this post
 CommentsRestacks
 loniousmonk
-6d
+7d
 Liked by James Check (Checkmatey)
 Um, which risk category does someone fall into if they used the full number of recommended dice rolls, as in the coldcard paranoid setup guide? I would argue 🟢 but I think it’s a weird omission in your post.
 Reply
