--- checkonchain-coldcard-psa@20260809T052957Z
+++ checkonchain-coldcard-psa@20260812T051028Z
@@ -4,7 +4,7 @@
 James Check (Checkmatey)
 Aug 01, 2026
 84
-24
+26
 14
 Share
 G’day Folks,
@@ -97,7 +97,7 @@
 Thanks for reading,
 James
 84
-24
+26
 14
 Share
 Discussion about this post
@@ -108,7 +108,7 @@
 Um, which risk category does someone fall into if they used the full number of recommended dice rolls, as in the coldcard paranoid setup guide? I would argue 🟢 but I think it’s a weird omission in your post.
 Reply
 Share
-3 replies by James Check (Checkmatey) and others
+5 replies by James Check (Checkmatey) and others
 Ricardo Santiago
 Aug 1
 Liked by James Check (Checkmatey)
@@ -116,7 +116,7 @@
 Reply
 Share
 1 reply
-22 more comments...
+24 more comments...
 TopLatestDiscussions
 No posts
 Ready for more?
