--- reddit-coinkite-options-after-disclosure@20260804T174722Z
+++ reddit-coinkite-options-after-disclosure@20260805T142744Z
@@ -155,19 +155,11 @@
 
 comment: p1ed0nj
 parent: t1_p1dyd6l
-author: OldHamburger7923
+author: [deleted]
 created_utc: 1785738807
 edited: false
 body:
-more likely a wallet collision than a hack.
-
-comment: p1edgwu
-parent: t1_p1e36i7
-author: OldHamburger7923
-created_utc: 1785739032
-edited: false
-body:
-i really don't see why multisig. A passphrase can have more entropy than the 24 seed itself with sufficient length (40 characters). If you randomly generate the seed yourself, it's already secure. Multi sig does what? Relies on you not trusting either wallet? But then you're just hoping at least one of them was generated properly. You may as well generate your own properly and not hope and pray.
+[deleted]
 
 comment: p1efdei
 parent: t1_p1dps95
