--- reddit-hacker-address-message-board@20260804T154504Z
+++ reddit-hacker-address-message-board@20260804T164630Z
@@ -642,3 +642,11 @@
 edited: false
 body:
 like getting punched twice in the balls to make sure
+
+comment: p1og4mj
+parent: t3_1ved66r
+author: Cute_Style3598
+created_utc: 1785860982
+edited: false
+body:
+I really think someone is gonna get caught or the whole thing is some kind of hoax/statement. The latest from Matthew Kratter and others is that most of the stolen coins are all in one wallet. That's either really stupid or intentionally done.
