--- reddit-hacker-address-message-board@20260804T092557Z
+++ reddit-hacker-address-message-board@20260804T134344Z
@@ -569,3 +569,52 @@
 edited: false
 body:
 Ah alright then lol
+
+comment: p1m9ttl
+parent: t3_1ved66r
+author: LornaQin
+created_utc: 1785836568
+edited: false
+body:
+Interesting, I only knew EVM's blockchat. TIL you can leave messages on BTC too
+
+comment: p1mh2et
+parent: t3_1ved66r
+author: AdLeft7000
+created_utc: 1785839849
+edited: false
+body:
+What ist this Adresse? It uses vanity to communicate. Have never seen so long vanity adresses....wtf?! :D
+1888888LXhoiiitdkxNErUNzeupsZrNhaZ
+
+comment: p1mquzg
+parent: t1_p1lns1h
+author: Leto33
+created_utc: 1785843708
+edited: false
+body:
+People who invest in BTC have their money in BTC, and may use it for a number of purposes which -surprise- they cannot foresee for lack of prescience skills. Luckily, vilent\_sibrate is here to remind them of the errors of their ways. Only on Reddit 🙄
+
+comment: p1n8z7q
+parent: t1_p1mquzg
+author: tenor_tymir
+created_utc: 1785849566
+edited: false
+body:
+Yes, that’s why I call it gravely irresponsible. You cannot hold money you need in high risk assets and then be surprised when it’s gone. That’s the absolute basics of common investing.
+
+comment: p1n9wmj
+parent: t1_p1l46iv
+author: razvanciuy
+created_utc: 1785849835
+edited: false
+body:
+more like their savings. Some had lots others saved 2000$ in sats and now gone after doing everything right using an actual dedicated device for storage. Total trash
+
+comment: p1ndy41
+parent: t3_1ved66r
+author: Mission_Sir2220
+created_utc: 1785850987
+edited: false
+body:
+So is this the nail in the coffin ? 
