--- reddit-hacker-address-message-board@20260804T151432Z
+++ reddit-hacker-address-message-board@20260804T154504Z
@@ -626,3 +626,19 @@
 edited: false
 body:
 Seems like it'll be a former employee
+
+comment: p1o0nie
+parent: t1_p1n9wmj
+author: anonymousopsec1337
+created_utc: 1785857058
+edited: false
+body:
+The first wave was wallets with .15 btc or higher which is a nice chunk of change to lose when the avg American can’t afford a $1000 emergency.
+
+comment: p1o11au
+parent: t1_p1o0nie
+author: razvanciuy
+created_utc: 1785857158
+edited: false
+body:
+like getting punched twice in the balls to make sure
