--- reddit-hacker-address-message-board@20260804T201600Z
+++ reddit-hacker-address-message-board@20260805T142509Z
@@ -688,3 +688,11 @@
 I feel sorry for anyone foolish enough to invest their money in risky investments. That's poor risk management. Money you need should be in low-risk investments like savings accounts, fixed-term deposits, or bonds. Available cash should not be invested in stocks, let alone cryptocurrencies. This is the most basic form of risk management that anyone even considering investing should follow.
 
 Anything else is simply stupid.
+
+comment: p1uezol
+parent: t3_1ved66r
+author: Correct_Priority_617
+created_utc: 1785933528
+edited: false
+body:
+I hope the MFs get caught - only if people fighting this care. So many lives destroyed and all that they say is “you should have been more careful”.  Self custody and all that. Coldcard should be made to pay the victims. 
