--- reddit-june-letter-report@20260803T141418Z
+++ reddit-june-letter-report@20260803T144328Z
@@ -13,10 +13,9 @@
 Endowus
 •
 Promoted
-Stay invested through every market cycle. The Endowus Stable Income Portfolio targets monthly payouts of 5–6% p.a.*, designed to preserve capital and deliver stable income in different market conditions.
+Maximise the potential of your CPF savings and secure a better retirement by investing with Endowus.
 Sign Up
 endowus.com
-Collapse video player
 Sort by:
 Comments Section
 HodlDee
@@ -128,12 +127,13 @@
 
 My suspicion, (and I'm only speculating) is FedEx being the information source. There were news stories of hackers getting access to their tracking software before and apple products being stolen from the leaks of the origin and destination addresses.
 
-u/Kilde_Singapore
+u/IBKR_Official
 •
 Promoted
-Earn 10-15% annually with monthly coupons and flexible terms.
-Learn More
-kilde.sg
+Uncover undervalued stocks and stay ahead with relevant insights. Trade smarter with AI built into IBKR Desktop.
+Sign Up
+interactivebrokers.com.sg
+Collapse video player
 RlzJohnnyM
 •
 1mo ago
