--- reddit-ledger-article-relay@20260804T134423Z
+++ reddit-ledger-article-relay@20260805T142549Z
@@ -315,36 +315,6 @@
 body:
 passphrase with high entropy. Pass phrases with low entropy are already being drained
 
-comment: p1jvn6t
-parent: t1_p1iyq5t
-author: SideOk1317
-created_utc: 1785801417
-edited: false
-body:
-you didn't lose money though 
-
-comment: p1k1inn
-parent: t1_p1i33zu
-author: OldHamburger7923
-created_utc: 1785803318
-edited: false
-body:
-The current issue doesn't mean the previous complaint isn't valid and justified. 
-
-Also the secure element is closed source so you can't know if there is a backdoor in their implementation. 
-
-Remember the secure element was supposed to prevent seed export then we found out it can export the seed, to the cloud no less.
-
-Only thing op proves is that people are boneheads.
-
-comment: p1k1yi1
-parent: t1_p1iqhnm
-author: OldHamburger7923
-created_utc: 1785803461
-edited: false
-body:
-Open source doesn't guarantee exploit and bug free. But it does mean backdoors are less easy/likely since any code change is in public view.
-
 comment: p1kgpvw
 parent: t1_p1iqhnm
 author: TheSlame
