--- reddit-ledger-article-relay@20260805T142549Z
+++ reddit-ledger-article-relay@20260806T163319Z
@@ -338,3 +338,13 @@
 edited: false
 body:
 This might be cover from Ledger. The situation can be a person migrating weak CC keys directly to a Ledger device, without generating new keys.  Next thing you know everything blows up on Ledger for the same people losing their stash. 
+
+comment: p22dixu
+parent: t1_p1i6afu
+author: BruceLee2112
+created_utc: 1786026033
+edited: false
+body:
+what is the current go to?
+
+
