--- reddit-nvk-scammer-regulation-call@20260812T173312Z
+++ reddit-nvk-scammer-regulation-call@20260814T234618Z
@@ -66,11 +66,11 @@
 
 comment: p2lnwig
 parent: t3_1vji2nz
-author: cleankiwii
+author: [deleted]
 created_utc: 1786257883
 edited: false
 body:
-what’s going on, are they selling closed source HW with a backdoor for their exit strategy?
+[deleted]
 
 comment: p2lo14c
 parent: t3_1vji2nz
