--- reddit-70m-41-minutes-report@20260804T095524Z
+++ reddit-70m-41-minutes-report@20260806T100228Z
@@ -173,13 +173,11 @@
 
 comment: p1e7sz9
 parent: t3_1vdt92h
-author: DonTheHolder
+author: [deleted]
 created_utc: 1785736243
 edited: false
 body:
-They are broken and broke. They trusted their bag with a hardware wallet creator instead of security professionals and other safety factors and switches that come into play that are secured with stocks as ETFS and CEX COINBASES.
-
-You're not nerd enough to protect your coin. Let a professional company manage them properly. 
+[removed]
 
 comment: p1e9pj3
 parent: t3_1vdt92h
