--- reddit-passphrase-vulnerable-question@20260805T210053Z
+++ reddit-passphrase-vulnerable-question@20260808T150306Z
@@ -2285,3 +2285,11 @@
 Add in mixed case, a few numbers and symbols, and it's basically uncrackable. 
 
 People seem to think passphrases are just ok security but you can have a passphrase with as much if not more security as the 24 word seed.
+
+comment: p2ftz5n
+parent: t1_p1f5nh3
+author: TechnologyGrouchy679
+created_utc: 1786186596
+edited: false
+body:
+no. it can be anything including spaces and special characters.  also an empty passphrase is also valid, which is what you're effectively using  when you use the seed phrase without an additional passphrase.
