--- reddit-passphrase-vulnerable-question@20260804T134440Z
+++ reddit-passphrase-vulnerable-question@20260804T144437Z
@@ -1505,9 +1505,7 @@
 created_utc: 1785743657
 edited: 1785842462
 body:
-Saffron ribbon paddle noodle breezy kettle
-
-This post was anonymized with Redact.dev
+[removed]
 
 comment: p1emwu7
 parent: t1_p1elhar
@@ -2279,3 +2277,11 @@
 edited: false
 body:
 ledger leaked their customers' information, twice. And their source isn't fully open
+
+comment: p1npysk
+parent: t3_1vdzo8p
+author: FigAggressive237
+created_utc: 1785854250
+edited: false
+body:
+2 word passphrase? That is like... 25 bits of entropy.... even with how many rounds on your preferred PBKDF this is beatable with consumer grade hardware....
