--- reddit-passphrase-vulnerable-question@20260804T171540Z
+++ reddit-passphrase-vulnerable-question@20260804T174547Z
@@ -71,11 +71,11 @@
 
 comment: p1d86u9
 parent: t1_p1d6yjj
-author: SmokeAndSkate
+author: [deleted]
 created_utc: 1785721651
 edited: false
 body:
-Could be that the wallet without passphrase showed some previous activity. Those wallets are more likely to have a passphrase wallet associated. 
+[deleted]
 
 comment: p1d88d4
 parent: t3_1vdzo8p
