--- reddit-passphrase-vulnerable-question@20260805T142603Z
+++ reddit-passphrase-vulnerable-question@20260805T210053Z
@@ -2275,3 +2275,13 @@
 edited: false
 body:
 I'll hazard a guess and say yes, but who knows what this guy was using. All I heard about his passphrase was "two words". 
+
+comment: p1vhwon
+parent: t1_p1dzv08
+author: read_more_comments
+created_utc: 1785944135
+edited: false
+body:
+Add in mixed case, a few numbers and symbols, and it's basically uncrackable. 
+
+People seem to think passphrases are just ok security but you can have a passphrase with as much if not more security as the 24 word seed.
