--- reddit-passphrase-vulnerable-question@20260808T150306Z
+++ reddit-passphrase-vulnerable-question@20260809T040527Z
@@ -2293,3 +2293,11 @@
 edited: false
 body:
 no. it can be anything including spaces and special characters.  also an empty passphrase is also valid, which is what you're effectively using  when you use the seed phrase without an additional passphrase.
+
+comment: p2kba5n
+parent: t1_p1elhar
+author: Suibeam
+created_utc: 1786237625
+edited: false
+body:
+Why would the US government do this lol. That money is peanuts for countries like USA, China and Germany.
