--- reddit-passphrase-vulnerable-question@20260804T144437Z
+++ reddit-passphrase-vulnerable-question@20260804T171540Z
@@ -2285,3 +2285,11 @@
 edited: false
 body:
 2 word passphrase? That is like... 25 bits of entropy.... even with how many rounds on your preferred PBKDF this is beatable with consumer grade hardware....
+
+comment: p1oqhwz
+parent: t1_p1e3666
+author: Candid-Walk-6762
+created_utc: 1785863569
+edited: false
+body:
+Apparently these guys cared a lot. I’m with you, just get what works the best. It’s just that a lot of these people deliberately ignored companies like Trezor because it wasn’t cypherphunky enough for them.
