--- reddit-passphrase-vulnerable-question@20260804T174547Z
+++ reddit-passphrase-vulnerable-question@20260804T181516Z
@@ -2293,3 +2293,35 @@
 edited: false
 body:
 Apparently these guys cared a lot. I’m with you, just get what works the best. It’s just that a lot of these people deliberately ignored companies like Trezor because it wasn’t cypherphunky enough for them.
+
+comment: p1p2kfb
+parent: t3_1vdzo8p
+author: Javanaut018
+created_utc: 1785866533
+edited: false
+body:
+Ya, 2 dictionary words adding roughly 32 bits of entropy. Less so if these were on a word list.
+
+comment: p1p2zi0
+parent: t1_p1ixvd7
+author: Javanaut018
+created_utc: 1785866639
+edited: false
+body:
+Smart thought...
+
+comment: p1p45hw
+parent: t1_p1d8pcq
+author: Javanaut018
+created_utc: 1785866927
+edited: false
+body:
+Are there really people using bip-39 words as passphrase anywhere?
+
+comment: p1p4ktr
+parent: t1_p1p45hw
+author: so7ow
+created_utc: 1785867033
+edited: false
+body:
+I'll hazard a guess and say yes, but who knows what this guy was using. All I heard about his passphrase was "two words". 
