--- theblock-galaxy-total@20260801T030828Z
+++ theblock-galaxy-total@20260801T032237Z
@@ -13,11 +13,11 @@
 Elections Hub
 NEW
 Live
-BTCUSD$62,999.750.17%
-ETHUSD$1,866.700.31%
-SOLUSD$72.970.07%
+BTCUSD$62,929.500.06%
+ETHUSD$1,865.980.27%
+SOLUSD$72.85-0.09%
 PYTHUSD$0.03970.00%
-LINKUSD$8.211.04%
+LINKUSD$8.190.81%
 Latest Crypto News
 Former Rep. George Santos agrees to pay $35,000 in CFTC settlement over Kalshi SOTU bet
 Jul 31, 2026, 4:25PM EDT • Regulation
