--- theblock-galaxy-total@20260801T222620Z
+++ theblock-galaxy-total@20260802T150203Z
@@ -14,7 +14,7 @@
 NEW
 <live-market-ticker-region>
 Latest Crypto News
-Iran-linked exchange sent $676 million to Binance in alleged sanctions-evasion operation: Reuters
+Iran-linked exchange sent $676 million to Binance wallets in alleged sanctions-evasion operation: Reuters
 Aug 01, 2026, 5:18PM EDT • Regulation
 BNB Chain 'pursuing legal action' against former employee over memecoin launched from tutorial wallet
 Aug 01, 2026, 3:09PM EDT • DeFi
