--- reddit-device-repurpose@20260804T090128Z
+++ reddit-device-repurpose@20260804T222600Z
@@ -130,3 +130,19 @@
 edited: false
 body:
 Checksum creator
+
+comment: p1qlwze
+parent: t3_1vckh2n
+author: thelaurelhighlander
+created_utc: 1785881096
+edited: false
+body:
+You can still use the device to sign transactions. Blow away any pre-existing seeds, update the firmware, do dice rolls and confirm it via separate source online to make sure the cold card is giving you the same values, repeat but don't check online and use that seed and then set a passphrase.... You should be golden.
+
+comment: p1qm8ya
+parent: t3_1vckh2n
+author: thelaurelhighlander
+created_utc: 1785881191
+edited: false
+body:
+It's still a good air-gapped solution.
